<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<!-- saved from url=(0051)https://www.bioconductor.org/packages/release/bioc/ -->
<html xmlns="http://www.w3.org/1999/xhtml" dir="ltr" lang="en-US" prefix="og: http://ogp.me/ns#"><head><meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
  
  <meta http-equiv="Content-Language" content="en-us">
  <meta name="robots" content="all">
  



  
  
  
    <script type="text/javascript" async="" src="./links2_files/ga.js"></script><script type="text/javascript" src="./links2_files/jquery.min.js"></script>
    <script type="text/javascript">
        if (!window.location.href.match(/bioconductor\.org|\/\/localhost/i)) {
            var mirror = true;
            document.write('<base href="//www.bioconductor.org"/>\n');
        } 
    </script>
  
  
  <script src="./links2_files/jquery.tools.min.js"></script>

  
  
  <title>Bioconductor - 3.10 Software Packages</title>
  <link rel="stylesheet" type="text/css" href="./links2_files/bioconductor.css" media="screen">
  <link rel="SHORTCUT ICON" type="image/x-icon" href="https://www.bioconductor.org/favicon.ico">
  <link rel="ICON" type="image/x-icon" href="https://www.bioconductor.org/favicon.ico">
  <script type="text/javascript" src="./links2_files/bioconductor.js"></script>
 
  <script type="text/javascript" src="./links2_files/jquery.corner.js"></script>
  <script type="text/javascript" src="./links2_files/jquery.timeago.js"></script>
  <script type="text/javascript" src="./links2_files/bioc-style.js"></script>
  <script type="text/javascript" src="./links2_files/versions.js"></script>

  <!-- Google Analytics script -->
  <script type="text/javascript">
    var _gaq = _gaq || [];
    _gaq.push(['_setAccount', 'UA-9338486-1']);
    _gaq.push(['_trackPageview']);

    (function() {
      var ga = document.createElement('script'); ga.type = 'text/javascript'; ga.async = true;
      ga.src = ('https:' == document.location.protocol ? 'https://ssl' : 'http://www') + '.google-analytics.com/ga.js';
      var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(ga, s);
    })();
  </script>
</head>

  <body>
    <a name="top"></a>

<!-- a few hooks for screen readers -->
<a href="https://www.bioconductor.org/packages/release/bioc/#site-navigation" title="Jump to site nav"></a>

<a href="https://www.bioconductor.org/packages/release/bioc/#site-map" title="Jump to site map"></a>


    <div id="SiteContainer" class="SiteContainer">
      
      <div id="PageContent" class="PageContent WithRightRail">
      


	
	  <div id="PageBreadcrumbs" class="PageBreadcrumbs">
  <ul>
    
    
    
      
	  	
        <li><a href="https://www.bioconductor.org/">Home</a></li>
      
    
    
      
	  	
        <li><a href="https://www.bioconductor.org/packages/3.10/BiocViews.html">Bioconductor 3.10</a></li>
      
    
    
      
        <li>3.10 Software Packages</li>
      
    
  </ul>
</div>

	

	
	    <h1>Bioconductor Software Packages</h1>







<p>
	Bioconductor version: Release (3.10)
</p>

<div class="do_not_rebase">
    <table>
        <tbody><tr> 
            <th>Package</th> 
            <th>Maintainer</th> 
            <th>Title</th> 
        </tr>        
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/a4.html">a4</a></td>
            <td>Tobias Verbeke, Willem Ligtenberg </td>
            <td>Automated Affymetrix Array Analysis Umbrella Package</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/a4Base.html">a4Base</a></td>
            <td>Tobias Verbeke, Willem Ligtenberg </td>
            <td>Automated Affymetrix Array Analysis Base Package</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/a4Classif.html">a4Classif</a></td>
            <td>Tobias Verbeke, Willem Ligtenberg </td>
            <td>Automated Affymetrix Array Analysis Classification Package</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/a4Core.html">a4Core</a></td>
            <td>Tobias Verbeke, Willem Ligtenberg </td>
            <td>Automated Affymetrix Array Analysis Core Package</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/a4Preproc.html">a4Preproc</a></td>
            <td>Tobias Verbeke, Willem Ligtenberg </td>
            <td>Automated Affymetrix Array Analysis Preprocessing Package</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/a4Reporting.html">a4Reporting</a></td>
            <td>Tobias Verbeke, Willem Ligtenberg </td>
            <td>Automated Affymetrix Array Analysis Reporting Package</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/ABAEnrichment.html">ABAEnrichment</a></td>
            <td>Steffi Grote </td>
            <td>Gene expression enrichment in human brain regions</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/ABarray.html">ABarray</a></td>
            <td>Yongming Andrew Sun </td>
            <td>Microarray QA and statistical data analysis for Applied Biosystems Genome Survey Microrarray (AB1700) gene expression data.</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/abseqR.html">abseqR</a></td>
            <td>JiaHong Fong </td>
            <td>Reporting and data analysis functionalities for Rep-Seq datasets of antibody libraries</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/ABSSeq.html">ABSSeq</a></td>
            <td>Wentao Yang </td>
            <td>ABSSeq: a new RNA-Seq analysis method based on modelling absolute expression differences</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/acde.html">acde</a></td>
            <td>Juan Pablo Acosta </td>
            <td>Artificial Components Detection of Differentially Expressed Genes</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/ACE.html">ACE</a></td>
            <td>Jos B Poell </td>
            <td>Absolute Copy Number Estimation from Low-coverage Whole Genome Sequencing</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/aCGH.html">aCGH</a></td>
            <td>Peter Dimitrov </td>
            <td>Classes and functions for Array Comparative Genomic Hybridization data.</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/ACME.html">ACME</a></td>
            <td>Sean Davis </td>
            <td>Algorithms for Calculating Microarray Enrichment (ACME)</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/ADaCGH2.html">ADaCGH2</a></td>
            <td>Ramon Diaz-Uriarte </td>
            <td>Analysis of big data from aCGH experiments using parallel computing and ff objects</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/ADAM.html">ADAM</a></td>
            <td>Jose Luiz Rybarczyk Filho </td>
            <td>ADAM: Activity and Diversity Analysis Module</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/ADAMgui.html">ADAMgui</a></td>
            <td>Jose Luiz Rybarczyk Filho </td>
            <td>Activity and Diversity Analysis Module Graphical User Interface</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/adaptest.html">adaptest</a></td>
            <td>Weixin Cai </td>
            <td>Data-Adaptive Statistics for High-Dimensional Multiple Testing</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/adductomicsR.html">adductomicsR</a></td>
            <td>Josie Hayes </td>
            <td>Processing of adductomic mass spectral datasets</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/adSplit.html">adSplit</a></td>
            <td>Claudio Lottaz </td>
            <td>Annotation-Driven Clustering</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/AffiXcan.html">AffiXcan</a></td>
            <td>Alessandro Lussana </td>
            <td>A Functional Approach To Impute Genetically Regulated Expression</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/affxparser.html">affxparser</a></td>
            <td>Kasper Daniel Hansen </td>
            <td>Affymetrix File Parsing SDK</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/affy.html">affy</a></td>
            <td>Rafael A. Irizarry </td>
            <td>Methods for Affymetrix Oligonucleotide Arrays</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/affycomp.html">affycomp</a></td>
            <td>Rafael A. Irizarry </td>
            <td>Graphics Toolbox for Assessment of Affymetrix Expression Measures</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/AffyCompatible.html">AffyCompatible</a></td>
            <td>Martin Morgan </td>
            <td>Affymetrix GeneChip software compatibility</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/affyContam.html">affyContam</a></td>
            <td>V. Carey </td>
            <td>structured corruption of affymetrix cel file data</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/affycoretools.html">affycoretools</a></td>
            <td>James W. MacDonald </td>
            <td>Functions useful for those doing repetitive analyses with Affymetrix GeneChips</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/AffyExpress.html">AffyExpress</a></td>
            <td>Xuejun Arthur Li </td>
            <td>Affymetrix Quality Assessment and Analysis Tool</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/affyILM.html">affyILM</a></td>
            <td>Myriam Kroll and Fabrice Berger </td>
            <td>Linear Model of background subtraction and the Langmuir isotherm</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/affyio.html">affyio</a></td>
            <td>Ben Bolstad </td>
            <td>Tools for parsing Affymetrix data files</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/affylmGUI.html">affylmGUI</a></td>
            <td>Gordon Smyth </td>
            <td>GUI for limma Package with Affymetrix Microarrays</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/affyPara.html">affyPara</a></td>
            <td>Markus Schmidberger </td>
            <td>Parallelized preprocessing methods for Affymetrix Oligonucleotide Arrays</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/affypdnn.html">affypdnn</a></td>
            <td>Laurent Gautier</td>
            <td>Probe Dependent Nearest Neighbours (PDNN) for the affy package</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/affyPLM.html">affyPLM</a></td>
            <td>Ben Bolstad </td>
            <td>Methods for fitting probe-level models</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/affyQCReport.html">affyQCReport</a></td>
            <td>Craig Parman </td>
            <td>QC Report Generation for affyBatch objects</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/AffyRNADegradation.html">AffyRNADegradation</a></td>
            <td>Mario Fasold </td>
            <td>Analyze and correct probe positional bias in microarray data due to RNA degradation</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/AGDEX.html">AGDEX</a></td>
            <td>Cuilan lani Gao </td>
            <td>Agreement of Differential Expression Analysis</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/agilp.html">agilp</a></td>
            <td>Benny Chain </td>
            <td>Agilent expression array processing package</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/AgiMicroRna.html">AgiMicroRna</a></td>
            <td>Pedro Lopez-Romero </td>
            <td>Processing and Differential Expression Analysis of Agilent microRNA chips</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/AIMS.html">AIMS</a></td>
            <td>Eric R Paquet </td>
            <td>AIMS : Absolute Assignment of Breast Cancer Intrinsic Molecular Subtype</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/ALDEx2.html">ALDEx2</a></td>
            <td>Greg Gloor </td>
            <td>Analysis Of Differential Abundance Taking Sample Variation Into Account</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/alevinQC.html">alevinQC</a></td>
            <td>Charlotte Soneson </td>
            <td>Generate QC Reports For Alevin Output</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/AllelicImbalance.html">AllelicImbalance</a></td>
            <td>Jesper R Gadin </td>
            <td>Investigates Allele Specific Expression</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/AlphaBeta.html">AlphaBeta</a></td>
            <td>Yadollah Shahryary Dizaji </td>
            <td>Computational inference of epimutation rates and spectra from high-throughput DNA methylation data in plants</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/alpine.html">alpine</a></td>
            <td>Michael Love </td>
            <td>alpine</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/ALPS.html">ALPS</a></td>
            <td>Venu Thatikonda </td>
            <td>AnaLysis routines for ePigenomicS data</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/alsace.html">alsace</a></td>
            <td>Ron Wehrens </td>
            <td>ALS for the Automatic Chemical Exploration of mixtures</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/altcdfenvs.html">altcdfenvs</a></td>
            <td>Laurent Gautier </td>
            <td>alternative CDF environments (aka probeset mappings)</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/AMARETTO.html">AMARETTO</a></td>
            <td>Olivier Gevaert </td>
            <td>Regulatory Network Inference and Driver Gene Evaluation using Integrative Multi-Omics Analysis and Penalized Regression</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/AMOUNTAIN.html">AMOUNTAIN</a></td>
            <td>Dong Li </td>
            <td>Active modules for multilayer weighted gene co-expression networks: a continuous optimization approach</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/amplican.html">amplican</a></td>
            <td>Eivind Valen </td>
            <td>Automated analysis of CRISPR experiments</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/AnalysisPageServer.html">AnalysisPageServer</a></td>
            <td>Brad Friedman </td>
            <td>A framework for sharing interactive data and plots from R through the web</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/anamiR.html">anamiR</a></td>
            <td>Ti-Tai Wang </td>
            <td>An integrated analysis package of miRNA and mRNA expression data</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/Anaquin.html">Anaquin</a></td>
            <td>Ted Wong </td>
            <td>Statistical analysis of sequins</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/AneuFinder.html">AneuFinder</a></td>
            <td>Aaron Taudt </td>
            <td>Analysis of Copy Number Variation in Single-Cell-Sequencing Data</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/ANF.html">ANF</a></td>
            <td>Tianle Ma </td>
            <td>Affinity Network Fusion for Complex Patient Clustering</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/animalcules.html">animalcules</a></td>
            <td>Yue Zhao </td>
            <td>Interactive microbiome analysis toolkit</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/annaffy.html">annaffy</a></td>
            <td>Colin A. Smith </td>
            <td>Annotation tools for Affymetrix biological metadata</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/annmap.html">annmap</a></td>
            <td>Chris Wirth </td>
            <td>Genome annotation and visualisation package pertaining to Affymetrix arrays and NGS analysis.</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/annotate.html">annotate</a></td>
            <td>Bioconductor Package Maintainer </td>
            <td>Annotation for microarrays</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/AnnotationDbi.html">AnnotationDbi</a></td>
            <td>Bioconductor Package Maintainer </td>
            <td>Manipulation of SQLite-based annotations in Bioconductor</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/AnnotationFilter.html">AnnotationFilter</a></td>
            <td>Bioconductor Package Maintainer </td>
            <td>Facilities for Filtering Bioconductor Annotation Resources</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/AnnotationForge.html">AnnotationForge</a></td>
            <td>Bioconductor Package Maintainer </td>
            <td>Tools for building SQLite-based annotation data packages</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/AnnotationFuncs.html">AnnotationFuncs</a></td>
            <td>Stefan McKinnon Edwards </td>
            <td>Annotation translation functions</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/AnnotationHub.html">AnnotationHub</a></td>
            <td>Bioconductor Package Maintainer </td>
            <td>Client to access AnnotationHub resources</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/AnnotationHubData.html">AnnotationHubData</a></td>
            <td>Bioconductor Package Maintainer </td>
            <td>Transform public data resources into Bioconductor Data Structures</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/annotationTools.html">annotationTools</a></td>
            <td>Alexandre Kuhn </td>
            <td>Annotate microarrays and perform cross-species gene expression analyses using flat file databases.</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/annotatr.html">annotatr</a></td>
            <td>Raymond G. Cavalcante </td>
            <td>Annotation of Genomic Regions to Genomic Annotations</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/anota.html">anota</a></td>
            <td>Ola Larsson </td>
            <td>ANalysis Of Translational Activity (ANOTA).</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/anota2seq.html">anota2seq</a></td>
            <td>Christian Oertlin, Julie Lorent </td>
            <td>Generally applicable transcriptome-wide analysis of translational efficiency using anota2seq</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/antiProfiles.html">antiProfiles</a></td>
            <td>Hector Corrada Bravo </td>
            <td>Implementation of gene expression anti-profiles</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/APAlyzer.html">APAlyzer</a></td>
            <td>Ruijia Wang </td>
            <td>A toolkit for APA analysis using RNA-seq data</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/apComplex.html">apComplex</a></td>
            <td>Denise Scholtens </td>
            <td>Estimate protein complex membership using AP-MS protein data</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/apeglm.html">apeglm</a></td>
            <td>Anqi Zhu </td>
            <td>Approximate posterior estimation for GLM coefficients</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/appreci8R.html">appreci8R</a></td>
            <td>Sarah Sandmann </td>
            <td>appreci8R: an R/Bioconductor package for filtering SNVs and short indels with high sensitivity and high PPV</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/aroma.light.html">aroma.light</a></td>
            <td>Henrik Bengtsson </td>
            <td>Light-Weight Methods for Normalization and Visualization of Microarray Data using Only Basic R Data Types</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/ArrayExpress.html">ArrayExpress</a></td>
            <td>Suhaib Mohammed </td>
            <td>Access the ArrayExpress Microarray Database at EBI and build Bioconductor data structures: ExpressionSet, AffyBatch, NChannelSet</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/ArrayExpressHTS.html">ArrayExpressHTS</a></td>
            <td>Angela Goncalves, Andrew Tikhonov </td>
            <td>ArrayExpress High Throughput Sequencing Processing Pipeline</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/arrayMvout.html">arrayMvout</a></td>
            <td>V. Carey </td>
            <td>multivariate outlier detection for expression array QA</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/arrayQuality.html">arrayQuality</a></td>
            <td>Agnes Paquet </td>
            <td>Assessing array quality on spotted arrays</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/arrayQualityMetrics.html">arrayQualityMetrics</a></td>
            <td>Mike Smith </td>
            <td>Quality metrics report for microarray data sets</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/ArrayTools.html">ArrayTools</a></td>
            <td>Arthur Li </td>
            <td>geneChip Analysis Package</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/ArrayTV.html">ArrayTV</a></td>
            <td>Eitan Halper-Stromberg </td>
            <td>Implementation of wave correction for arrays</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/ARRmNormalization.html">ARRmNormalization</a></td>
            <td>Jean-Philippe Fortin </td>
            <td>Adaptive Robust Regression normalization for Illumina methylation data</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/artMS.html">artMS</a></td>
            <td>David Jimenez-Morales </td>
            <td>Analytical R tools for Mass Spectrometry</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/ASAFE.html">ASAFE</a></td>
            <td>Qian Zhang </td>
            <td>Ancestry Specific Allele Frequency Estimation</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/ASEB.html">ASEB</a></td>
            <td>Likun Wang </td>
            <td>Predict Acetylated Lysine Sites</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/ASGSCA.html">ASGSCA</a></td>
            <td>Hela Romdhani </td>
            <td>Association Studies for multiple SNPs and multiple traits using Generalized Structured Equation Models</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/ASICS.html">ASICS</a></td>
            <td>Gaëlle Lefort </td>
            <td>Automatic Statistical Identification in Complex Spectra</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/ASpediaFI.html">ASpediaFI</a></td>
            <td>Doyeong Yu </td>
            <td>ASpedia-FI: Functional Interaction Analysis of Alternative Splicing Events</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/ASpli.html">ASpli</a></td>
            <td>Estefania Mancini </td>
            <td>Analysis of alternative splicing using RNA-Seq</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/AssessORF.html">AssessORF</a></td>
            <td>Deepank Korandla </td>
            <td>Assess Gene Predictions Using Proteomics and Evolutionary Conservation</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/ASSET.html">ASSET</a></td>
            <td>Samsiddhi Bhattacharjee </td>
            <td>An R package for subset-based association analysis of heterogeneous traits and subtypes</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/ASSIGN.html">ASSIGN</a></td>
            <td>Ying Shen, W. Evan Johnson, David Jenkins, Mumtehena Rahman </td>
            <td>Adaptive Signature Selection and InteGratioN (ASSIGN)</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/ATACseqQC.html">ATACseqQC</a></td>
            <td>Jianhong Ou </td>
            <td>ATAC-seq Quality Control</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/atSNP.html">atSNP</a></td>
            <td>Sunyoung Shin </td>
            <td>Affinity test for identifying regulatory SNPs</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/attract.html">attract</a></td>
            <td>Samuel Zimmerman </td>
            <td>Methods to Find the Gene Expression Modules that Represent the Drivers of Kauffman's Attractor Landscape</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/AUCell.html">AUCell</a></td>
            <td>Sara Aibar </td>
            <td>AUCell: Analysis of 'gene set' activity in single-cell RNA-seq data (e.g. identify cells with specific gene signatures)</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/Autotuner.html">Autotuner</a></td>
            <td>Craig McLean </td>
            <td>Automated parameter selection for untargeted metabolomics data processing</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/AWFisher.html">AWFisher</a></td>
            <td>Zhiguang Huo </td>
            <td>An R package for fast computing for adaptively weighted fisher's method</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/BaalChIP.html">BaalChIP</a></td>
            <td>Ines de Santiago </td>
            <td>BaalChIP: Bayesian analysis of allele-specific transcription factor binding in cancer genomes</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/BAC.html">BAC</a></td>
            <td>Raphael Gottardo </td>
            <td>Bayesian Analysis of Chip-chip experiment</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/bacon.html">bacon</a></td>
            <td>Maarten van Iterson </td>
            <td>Controlling bias and inflation in association studies using the empirical null distribution</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/BADER.html">BADER</a></td>
            <td>Andreas Neudecker </td>
            <td>Bayesian Analysis of Differential Expression in RNA Sequencing Data</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/BadRegionFinder.html">BadRegionFinder</a></td>
            <td>Sarah Sandmann </td>
            <td>BadRegionFinder: an R/Bioconductor package for identifying regions with bad coverage</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/BAGS.html">BAGS</a></td>
            <td>Alejandro Quiroz-Zarate </td>
            <td>A Bayesian Approach for Geneset Selection</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/ballgown.html">ballgown</a></td>
            <td>Jack Fu </td>
            <td>Flexible, isoform-level differential expression analysis</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/bamsignals.html">bamsignals</a></td>
            <td>Johannes Helmuth </td>
            <td>Extract read count signals from bam files</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/BANDITS.html">BANDITS</a></td>
            <td>Simone Tiberi </td>
            <td>BANDITS: Bayesian ANalysis of DIfferenTial Splicing</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/banocc.html">banocc</a></td>
            <td>George Weingart, Curtis Huttenhower </td>
            <td>Bayesian ANalysis Of Compositional Covariance</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/basecallQC.html">basecallQC</a></td>
            <td>Thomas Carroll </td>
            <td>Working with Illumina Basecalling and Demultiplexing input and output files</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/BaseSpaceR.html">BaseSpaceR</a></td>
            <td>Jared O'Connell </td>
            <td>R SDK for BaseSpace RESTful API</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/Basic4Cseq.html">Basic4Cseq</a></td>
            <td>Carolin Walter </td>
            <td>Basic4Cseq: an R/Bioconductor package for analyzing 4C-seq data</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/BASiCS.html">BASiCS</a></td>
            <td>Catalina Vallejos </td>
            <td>Bayesian Analysis of Single-Cell Sequencing data</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/BasicSTARRseq.html">BasicSTARRseq</a></td>
            <td>Annika Buerger </td>
            <td>Basic peak calling on STARR-seq data</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/batchelor.html">batchelor</a></td>
            <td>Aaron Lun </td>
            <td>Single-Cell Batch Correction Methods</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/BatchQC.html">BatchQC</a></td>
            <td>Solaiappan Manimaran </td>
            <td>Batch Effects Quality Control Software</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/BayesKnockdown.html">BayesKnockdown</a></td>
            <td>William Chad Young </td>
            <td>BayesKnockdown: Posterior Probabilities for Edges from Knockdown Data</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/BayesPeak.html">BayesPeak</a></td>
            <td>Jonathan Cairns </td>
            <td>Bayesian Analysis of ChIP-seq Data</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/bayNorm.html">bayNorm</a></td>
            <td>Wenhao Tang </td>
            <td>Single-cell RNA sequencing data normalization</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/baySeq.html">baySeq</a></td>
            <td>Thomas J. Hardcastle </td>
            <td>Empirical Bayesian analysis of patterns of differential expression in count data</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/BBCAnalyzer.html">BBCAnalyzer</a></td>
            <td>Sarah Sandmann </td>
            <td>BBCAnalyzer: an R/Bioconductor package for visualizing base counts</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/BCRANK.html">BCRANK</a></td>
            <td>Adam Ameur </td>
            <td>Predicting binding site consensus from ranked DNA sequences</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/bcSeq.html">bcSeq</a></td>
            <td>Jiaxing Lin </td>
            <td>Fast Sequence Mapping in High-Throughput shRNA and CRISPR Screens</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/BDMMAcorrect.html">BDMMAcorrect</a></td>
            <td>ZHENWEI DAI </td>
            <td>Meta-analysis for the metagenomic read counts data from different cohorts</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/beachmat.html">beachmat</a></td>
            <td>Aaron Lun </td>
            <td>Compiling Bioconductor to Handle Each Matrix Type</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/beadarray.html">beadarray</a></td>
            <td>Mark Dunning </td>
            <td>Quality assessment and low-level analysis for Illumina BeadArray data</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/beadarraySNP.html">beadarraySNP</a></td>
            <td>Jan Oosting </td>
            <td>Normalization and reporting of Illumina SNP bead arrays</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/BeadDataPackR.html">BeadDataPackR</a></td>
            <td>Mike Smith </td>
            <td>Compression of Illumina BeadArray data</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/BEARscc.html">BEARscc</a></td>
            <td>Benjamin Schuster-Boeckler </td>
            <td>BEARscc (Bayesian ERCC Assesstment of Robustness of Single Cell Clusters)</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/BEAT.html">BEAT</a></td>
            <td>Kemal Akman </td>
            <td>BEAT - BS-Seq Epimutation Analysis Toolkit</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/BEclear.html">BEclear</a></td>
            <td>David Rasp </td>
            <td>Correction of batch effects in DNA methylation data</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/bgafun.html">bgafun</a></td>
            <td>Iain Wallace </td>
            <td>BGAfun A method to identify specifity determining residues in protein families</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/BgeeDB.html">BgeeDB</a></td>
            <td>Julien Wollbrett, Julien Roux, Andrea Komljenovic, Frederic Bastian </td>
            <td>Annotation and gene expression data retrieval from Bgee database</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/BGmix.html">BGmix</a></td>
            <td>Alex Lewin </td>
            <td>Bayesian models for differential gene expression</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/bgx.html">bgx</a></td>
            <td>Ernest Turro </td>
            <td>Bayesian Gene eXpression</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/BHC.html">BHC</a></td>
            <td>Rich Savage </td>
            <td>Bayesian Hierarchical Clustering</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/BicARE.html">BicARE</a></td>
            <td>Pierre Gestraud </td>
            <td>Biclustering Analysis and Results Exploration</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/BiFET.html">BiFET</a></td>
            <td>Ahrim Youn </td>
            <td>Bias-free Footprint Enrichment Test</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/BiGGR.html">BiGGR</a></td>
            <td>Anand K. Gavai, Hannes Hettling </td>
            <td>Constraint based modeling in R using metabolic reconstruction databases</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/bigmelon.html">bigmelon</a></td>
            <td>Tyler J. Gorrie-Stone </td>
            <td>Illumina methylation array analysis for large experiments</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/bigmemoryExtras.html">bigmemoryExtras</a></td>
            <td>Peter M. Haverty </td>
            <td>An extension of the bigmemory package with added safety, convenience, and a factor class</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/bigPint.html">bigPint</a></td>
            <td>Lindsay Rutter </td>
            <td>Big multivariate data plotted interactively</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/bioassayR.html">bioassayR</a></td>
            <td>Tyler Backman </td>
            <td>Cross-target analysis of small molecule bioactivity</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/Biobase.html">Biobase</a></td>
            <td>Bioconductor Package Maintainer </td>
            <td>Biobase: Base functions for Bioconductor</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/biobroom.html">biobroom</a></td>
            <td>John D. Storey and Andrew J. Bass </td>
            <td>Turn Bioconductor objects into tidy data frames</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/bioCancer.html">bioCancer</a></td>
            <td>Karim Mezhoud </td>
            <td>Interactive Multi-Omics Cancers Data Visualization and Analysis</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/BiocCaseStudies.html">BiocCaseStudies</a></td>
            <td>Bioconductor Package Maintainer </td>
            <td>BiocCaseStudies: Support for the Case Studies Monograph</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/BiocCheck.html">BiocCheck</a></td>
            <td>Bioconductor Package Maintainer </td>
            <td>Bioconductor-specific package checks</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/BiocFileCache.html">BiocFileCache</a></td>
            <td>Lori Shepherd </td>
            <td>Manage Files Across Sessions</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/BiocGenerics.html">BiocGenerics</a></td>
            <td>Bioconductor Package Maintainer </td>
            <td>S4 generic functions used in Bioconductor</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/biocGraph.html">biocGraph</a></td>
            <td>Florian Hahne </td>
            <td>Graph examples and use cases in Bioinformatics</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/BiocNeighbors.html">BiocNeighbors</a></td>
            <td>Aaron Lun </td>
            <td>Nearest Neighbor Detection for Bioconductor Packages</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/BiocOncoTK.html">BiocOncoTK</a></td>
            <td>VJ Carey </td>
            <td>Bioconductor components for general cancer genomics</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/BioCor.html">BioCor</a></td>
            <td>Lluís Revilla Sancho </td>
            <td>Functional similarities</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/BiocParallel.html">BiocParallel</a></td>
            <td>Bioconductor Package Maintainer </td>
            <td>Bioconductor facilities for parallel evaluation</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/BiocPkgTools.html">BiocPkgTools</a></td>
            <td>Sean Davis </td>
            <td>Collection of simple tools for learning about Bioc Packages</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/BiocSet.html">BiocSet</a></td>
            <td>Kayla Morrell </td>
            <td>Representing Different Biological Sets</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/BiocSingular.html">BiocSingular</a></td>
            <td>Aaron Lun </td>
            <td>Singular Value Decomposition for Bioconductor Packages</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/BiocSklearn.html">BiocSklearn</a></td>
            <td>VJ Carey </td>
            <td>interface to python sklearn via Rstudio reticulate</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/BiocStyle.html">BiocStyle</a></td>
            <td>Bioconductor Package Maintainer </td>
            <td>Standard styles for vignettes and other Bioconductor documents</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/BiocVersion.html">BiocVersion</a></td>
            <td>Bioconductor Package Maintainer </td>
            <td>Set the appropriate version of Bioconductor packages</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/biocViews.html">biocViews</a></td>
            <td>Bioconductor Package Maintainer </td>
            <td>Categorized views of R package repositories</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/BiocWorkflowTools.html">BiocWorkflowTools</a></td>
            <td>Mike Smith </td>
            <td>Tools to aid the development of Bioconductor Workflow packages</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/bioDist.html">bioDist</a></td>
            <td>Bioconductor Package Maintainer </td>
            <td>Different distance measures</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/biomaRt.html">biomaRt</a></td>
            <td>Mike Smith </td>
            <td>Interface to BioMart databases (i.e. Ensembl)</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/biomformat.html">biomformat</a></td>
            <td>Paul J. McMurdie </td>
            <td>An interface package for the BIOM file format</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/BioMM.html">BioMM</a></td>
            <td>Junfang Chen </td>
            <td>BioMM: Biological-informed Multi-stage Machine learning framework for phenotype prediction using omics data</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/BioMVCClass.html">BioMVCClass</a></td>
            <td>Elizabeth Whalen </td>
            <td>Model-View-Controller (MVC) Classes That Use Biobase</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/biomvRCNS.html">biomvRCNS</a></td>
            <td>Yang Du </td>
            <td>Copy Number study and Segmentation for multivariate biological data</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/BioNet.html">BioNet</a></td>
            <td>Marcus Dittrich </td>
            <td>Routines for the functional analysis of biological networks</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/BioNetStat.html">BioNetStat</a></td>
            <td>Vinicius Jardim </td>
            <td>Biological Network Analysis</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/BioQC.html">BioQC</a></td>
            <td>Jitao David Zhang </td>
            <td>Detect tissue heterogeneity in expression profiles with gene sets</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/BioSeqClass.html">BioSeqClass</a></td>
            <td>Li Hong </td>
            <td>Classification for Biological Sequences</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/biosigner.html">biosigner</a></td>
            <td>Philippe Rinaudo, Etienne Thevenot </td>
            <td>Signature discovery from omics data</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/Biostrings.html">Biostrings</a></td>
            <td>H. Pagès </td>
            <td>Efficient manipulation of biological strings</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/biosvd.html">biosvd</a></td>
            <td>Anneleen Daemen, Matthew Brauer </td>
            <td>Package for high-throughput data processing, outlier detection, noise removal and dynamic modeling</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/BioTIP.html">BioTIP</a></td>
            <td>Zhezhen Wang, X Holly Yang and Yuxi (Jennifer) Sun </td>
            <td>BioTIP: An R package for characterization of Biological Tipping-Point</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/biotmle.html">biotmle</a></td>
            <td>Nima Hejazi </td>
            <td>Targeted Learning with Moderated Statistics for Biomarker Discovery</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/biovizBase.html">biovizBase</a></td>
            <td>Michael Lawrence </td>
            <td>Basic graphic utilities for visualization of genomic data.</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/BiRewire.html">BiRewire</a></td>
            <td>Andrea Gobbi </td>
            <td>High-performing routines for the randomization of a bipartite graph (or a binary event matrix), undirected and directed signed graph preserving degree distribution (or marginal totals)</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/birta.html">birta</a></td>
            <td>Benedikt Zacher, Holger Froehlich </td>
            <td>Bayesian Inference of Regulation of Transcriptional Activity</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/birte.html">birte</a></td>
            <td>Holger Froehlich </td>
            <td>Bayesian Inference of Regulatory Influence on Expression (biRte)</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/biscuiteer.html">biscuiteer</a></td>
            <td>"Tim Triche, Jr." </td>
            <td>Convenience Functions for Biscuit</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/BiSeq.html">BiSeq</a></td>
            <td>Katja Hebestreit </td>
            <td>Processing and analyzing bisulfite sequencing data</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/BitSeq.html">BitSeq</a></td>
            <td>Antti Honkela, Panagiotis Papastamoulis </td>
            <td>Transcript expression inference and differential expression analysis for RNA-seq data</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/blacksheepr.html">blacksheepr</a></td>
            <td>RugglesLab </td>
            <td>Outlier Analysis for pairwise differential comparison</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/blima.html">blima</a></td>
            <td>Vojtěch Kulvait </td>
            <td>Tools for the preprocessing and analysis of the Illumina microarrays on the detector (bead) level</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/BLMA.html">BLMA</a></td>
            <td>Tin Nguyen </td>
            <td>BLMA: A package for bi-level meta-analysis</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/bnbc.html">bnbc</a></td>
            <td>Kipper Fletez-Brant </td>
            <td>Bandwise normalization and batch correction of Hi-C data</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/BPRMeth.html">BPRMeth</a></td>
            <td>Chantriolnt-Andreas Kapourani </td>
            <td>Model higher-order methylation profiles</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/BRAIN.html">BRAIN</a></td>
            <td>Piotr Dittwald </td>
            <td>Baffling Recursive Algorithm for Isotope distributioN calculations</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/brainflowprobes.html">brainflowprobes</a></td>
            <td>Amanda Price </td>
            <td>Plots and annotation for choosing BrainFlow target probe sequence</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/brainImageR.html">brainImageR</a></td>
            <td>Sara B Linker </td>
            <td>A Framework for visualizing gene set enrichment throughout neurodevelopment</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/BrainStars.html">BrainStars</a></td>
            <td>Itoshi NIKAIDO </td>
            <td>query gene expression data and plots from BrainStars (B*)</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/branchpointer.html">branchpointer</a></td>
            <td>Beth Signal </td>
            <td>Prediction of intronic splicing branchpoints</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/breakpointR.html">breakpointR</a></td>
            <td>David Porubsky </td>
            <td>Find breakpoints in Strand-seq data</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/brendaDb.html">brendaDb</a></td>
            <td>Yi Zhou </td>
            <td>The BRENDA Enzyme Database</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/bridge.html">bridge</a></td>
            <td>Raphael Gottardo </td>
            <td>Bayesian Robust Inference for Differential Gene Expression</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/BridgeDbR.html">BridgeDbR</a></td>
            <td>Egon Willighagen </td>
            <td>Code for using BridgeDb identifier mapping framework from within R</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/BrowserViz.html">BrowserViz</a></td>
            <td>Paul Shannon </td>
            <td>BrowserViz: interactive R/browser graphics using websockets and JSON</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/BSgenome.html">BSgenome</a></td>
            <td>H. Pagès </td>
            <td>Software infrastructure for efficient representation of full genomes and their SNPs</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/bsseq.html">bsseq</a></td>
            <td>Kasper Daniel Hansen </td>
            <td>Analyze, manage and store bisulfite sequencing data</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/BubbleTree.html">BubbleTree</a></td>
            <td>Todd Creasy, Wei Zhu </td>
            <td>BubbleTree: an intuitive visualization to elucidate tumoral aneuploidy and clonality in somatic mosaicism using next generation sequencing data</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/BufferedMatrix.html">BufferedMatrix</a></td>
            <td>Ben Bolstad </td>
            <td>A matrix data storage object held in temporary files</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/BufferedMatrixMethods.html">BufferedMatrixMethods</a></td>
            <td>Ben Bolstad </td>
            <td>Microarray Data related methods that utlize BufferedMatrix objects</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/BUMHMM.html">BUMHMM</a></td>
            <td>Alina Selega </td>
            <td>Computational pipeline for computing probability of modification from structure probing experiment data</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/bumphunter.html">bumphunter</a></td>
            <td>Rafael A. Irizarry </td>
            <td>Bump Hunter</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/BUS.html">BUS</a></td>
            <td>Yuanhua Liu </td>
            <td>Gene network reconstruction</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/BUScorrect.html">BUScorrect</a></td>
            <td>Xiangyu Luo </td>
            <td>Batch Effects Correction with Unknown Subtypes</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/BUSpaRse.html">BUSpaRse</a></td>
            <td>Lambda Moses </td>
            <td>kallisto | bustools R utilities</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/CAFE.html">CAFE</a></td>
            <td>Sander Bollen </td>
            <td>Chromosmal Aberrations Finder in Expression data</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/CAGEfightR.html">CAGEfightR</a></td>
            <td>Malte Thodberg </td>
            <td>Analysis of Cap Analysis of Gene Expression (CAGE) data using Bioconductor</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/CAGEr.html">CAGEr</a></td>
            <td>Vanja Haberle, Charles Plessy, Damir Baranasic, Sarvesh Nikumbh </td>
            <td>Analysis of CAGE (Cap Analysis of Gene Expression) sequencing data for precise mapping of transcription start sites and promoterome mining</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/CALIB.html">CALIB</a></td>
            <td>Hui Zhao </td>
            <td>Calibration model for estimating absolute expression levels from microarray data</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/calm.html">calm</a></td>
            <td>Kun Liang </td>
            <td>Covariate Assisted Large-scale Multiple testing</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/CAMERA.html">CAMERA</a></td>
            <td>Steffen Neumann </td>
            <td>Collection of annotation related methods for mass spectrometry data</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/CAMTHC.html">CAMTHC</a></td>
            <td>Lulu Chen </td>
            <td>Convex Analysis of Mixtures for Tissue Heterogeneity Characterization</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/canceR.html">canceR</a></td>
            <td>Karim Mezhoud </td>
            <td>A Graphical User Interface for accessing and modeling the Cancer Genomics Data of MSKCC</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/cancerclass.html">cancerclass</a></td>
            <td>Daniel Kosztyla </td>
            <td>Development and validation of diagnostic tests from high-dimensional molecular data</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/CancerInSilico.html">CancerInSilico</a></td>
            <td>Thomas D. Sherman, Elana J. Fertig </td>
            <td>An R interface for computational modeling of tumor progression</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/CancerMutationAnalysis.html">CancerMutationAnalysis</a></td>
            <td>Simina M. Boca </td>
            <td>Cancer mutation analysis</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/CancerSubtypes.html">CancerSubtypes</a></td>
            <td>Taosheng Xu</td>
            <td>Cancer subtypes identification, validation and visualization based on multiple genomic data sets</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/CAnD.html">CAnD</a></td>
            <td>Caitlin McHugh </td>
            <td>Perform Chromosomal Ancestry Differences (CAnD) Analyses</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/caOmicsV.html">caOmicsV</a></td>
            <td>Henry Zhang </td>
            <td>Visualization of multi-dimentional cancer genomics data</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/Cardinal.html">Cardinal</a></td>
            <td>Kylie A. Bemis </td>
            <td>A mass spectrometry imaging toolbox for statistical analysis</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/casper.html">casper</a></td>
            <td>David Rossell </td>
            <td>Characterization of Alternative Splicing based on Paired-End Reads</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/CATALYST.html">CATALYST</a></td>
            <td>Helena L. Crowell </td>
            <td>Cytometry dATa anALYSis Tools</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/Category.html">Category</a></td>
            <td>Bioconductor Package Maintainer </td>
            <td>Category Analysis</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/categoryCompare.html">categoryCompare</a></td>
            <td>Robert M. Flight </td>
            <td>Meta-analysis of high-throughput experiments using feature annotations</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/CausalR.html">CausalR</a></td>
            <td>Glyn Bradley, Steven Barrett </td>
            <td>Causal network analysis methods</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/cbaf.html">cbaf</a></td>
            <td>Arman Shahrisa </td>
            <td>Automated functions for comparing various data from cbioportal.org</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/ccfindR.html">ccfindR</a></td>
            <td>Jun Woo </td>
            <td>Cancer Clone Finder</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/ccmap.html">ccmap</a></td>
            <td>Alex Pickering </td>
            <td>Combination Connectivity Mapping</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/CCPROMISE.html">CCPROMISE</a></td>
            <td>Xueyuan Cao </td>
            <td>PROMISE analysis with Canonical Correlation for Two Forms of High Dimensional Genetic Data</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/ccrepe.html">ccrepe</a></td>
            <td>Emma Schwager,Craig Bielski, George Weingart</td>
            <td>ccrepe_and_nc.score</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/celaref.html">celaref</a></td>
            <td>Sarah Williams </td>
            <td>Single-cell RNAseq cell cluster labelling by reference</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/celda.html">celda</a></td>
            <td>Joshua Campbell </td>
            <td>CEllular Latent Dirichlet Allocation</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/cellbaseR.html">cellbaseR</a></td>
            <td>Mohammed OE Abdallah </td>
            <td>Querying annotation data from the high performance Cellbase web</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/CellBench.html">CellBench</a></td>
            <td>Shian Su </td>
            <td>Construct Benchmarks for Single Cell Analysis Methods</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/cellGrowth.html">cellGrowth</a></td>
            <td>Julien Gagneur </td>
            <td>Fitting cell population growth models</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/cellHTS2.html">cellHTS2</a></td>
            <td>Joseph Barry </td>
            <td>Analysis of cell-based screens - revised version of cellHTS</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/cellity.html">cellity</a></td>
            <td>Tomislav Ilicic </td>
            <td>Quality Control for Single-Cell RNA-seq Data</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/CellMapper.html">CellMapper</a></td>
            <td>Brad Nelms </td>
            <td>Predict genes expressed selectively in specific cell types</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/CellMixS.html">CellMixS</a></td>
            <td>Almut Lütge </td>
            <td>Evaluate Cellspecific Mixing</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/CellNOptR.html">CellNOptR</a></td>
            <td>A.Gabor </td>
            <td>Training of boolean logic models of signalling networks using prior knowledge networks and perturbation data</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/cellscape.html">cellscape</a></td>
            <td>Maia Smith </td>
            <td>Explores single cell copy number profiles in the context of a single cell tree</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/CellScore.html">CellScore</a></td>
            <td>Nancy Mah </td>
            <td>Tool for Evaluation of Cell Identity from Transcription Profiles</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/CellTrails.html">CellTrails</a></td>
            <td>Daniel Ellwanger </td>
            <td>Reconstruction, visualization and analysis of branching trajectories</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/cellTree.html">cellTree</a></td>
            <td>David duVerle </td>
            <td>Inference and visualisation of Single-Cell RNA-seq data as a hierarchical tree structure</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/CEMiTool.html">CEMiTool</a></td>
            <td>Helder Nakaya </td>
            <td>Co-expression Modules identification Tool</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/CexoR.html">CexoR</a></td>
            <td>Pedro Madrigal </td>
            <td>An R package to uncover high-resolution protein-DNA interactions in ChIP-exo replicates</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/CFAssay.html">CFAssay</a></td>
            <td>Herbert Braselmann </td>
            <td>Statistical analysis for the Colony Formation Assay</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/CGEN.html">CGEN</a></td>
            <td>William Wheeler </td>
            <td>An R package for analysis of case-control studies in genetic epidemiology</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/CGHbase.html">CGHbase</a></td>
            <td>Mark van de Wiel </td>
            <td>CGHbase: Base functions and classes for arrayCGH data analysis.</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/CGHcall.html">CGHcall</a></td>
            <td>Mark van de Wiel </td>
            <td>Calling aberrations for array CGH tumor profiles.</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/cghMCR.html">cghMCR</a></td>
            <td>J. Zhang </td>
            <td>Find chromosome regions showing common gains/losses</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/CGHnormaliter.html">CGHnormaliter</a></td>
            <td>Bart P.P. van Houte </td>
            <td>Normalization of array CGH data with imbalanced aberrations.</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/CGHregions.html">CGHregions</a></td>
            <td>Sjoerd Vosse </td>
            <td>Dimension Reduction for Array CGH Data with Minimal Information Loss.</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/ChAMP.html">ChAMP</a></td>
            <td>Yuan Tian </td>
            <td>Chip Analysis Methylation Pipeline for Illumina HumanMethylation450 and EPIC</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/CHARGE.html">CHARGE</a></td>
            <td>Benjamin Mayne </td>
            <td>CHARGE: CHromosome Assessment in R from Gene Expression data</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/charm.html">charm</a></td>
            <td>Peter Murakami </td>
            <td>Analysis of DNA methylation data from CHARM microarrays</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/ChemmineOB.html">ChemmineOB</a></td>
            <td>Thomas Girke </td>
            <td>R interface to a subset of OpenBabel functionalities</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/ChemmineR.html">ChemmineR</a></td>
            <td>Thomas Girke </td>
            <td>Cheminformatics Toolkit for R</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/CHETAH.html">CHETAH</a></td>
            <td>Jurrian de Kanter </td>
            <td>Fast and accurate scRNA-seq cell type identification</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/ChIC.html">ChIC</a></td>
            <td>Carmen Maria Livi </td>
            <td>Quality Control Pipeline for ChIP-Seq Data</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/Chicago.html">Chicago</a></td>
            <td>Mikhail Spivakov </td>
            <td>CHiCAGO: Capture Hi-C Analysis of Genomic Organization</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/chimera.html">chimera</a></td>
            <td>Raffaele A Calogero </td>
            <td>A package for secondary analysis of fusion products</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/chimeraviz.html">chimeraviz</a></td>
            <td>Stian Lågstad </td>
            <td>Visualization tools for gene fusions</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/ChIPanalyser.html">ChIPanalyser</a></td>
            <td>Patrick C.N. Martin </td>
            <td>ChIPanalyser: Predicting Transcription Factor Binding Sites</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/ChIPComp.html">ChIPComp</a></td>
            <td>Li Chen </td>
            <td>Quantitative comparison of multiple ChIP-seq datasets</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/chipenrich.html">chipenrich</a></td>
            <td>Raymond G. Cavalcante </td>
            <td>Gene Set Enrichment For ChIP-seq Peak Data</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/ChIPexoQual.html">ChIPexoQual</a></td>
            <td>Rene Welch </td>
            <td>ChIPexoQual</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/ChIPpeakAnno.html">ChIPpeakAnno</a></td>
            <td>Lihua Julie Zhu, Jianhong Ou </td>
            <td>Batch annotation of the peaks identified from either ChIP-seq, ChIP-chip experiments or any experiments resulted in large number of chromosome ranges</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/ChIPQC.html">ChIPQC</a></td>
            <td>Tom Carroll, Rory Stark </td>
            <td>Quality metrics for ChIPseq data</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/ChIPseeker.html">ChIPseeker</a></td>
            <td>Guangchuang Yu </td>
            <td>ChIPseeker for ChIP peak Annotation, Comparison, and Visualization</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/chipseq.html">chipseq</a></td>
            <td>Bioconductor Package Maintainer </td>
            <td>chipseq: A package for analyzing chipseq data</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/ChIPseqR.html">ChIPseqR</a></td>
            <td>Peter Humburg </td>
            <td>Identifying Protein Binding Sites in High-Throughput Sequencing Data</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/ChIPSeqSpike.html">ChIPSeqSpike</a></td>
            <td>Nicolas Descostes </td>
            <td>ChIP-Seq data scaling according to spike-in control</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/ChIPsim.html">ChIPsim</a></td>
            <td>Peter Humburg </td>
            <td>Simulation of ChIP-seq experiments</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/ChIPXpress.html">ChIPXpress</a></td>
            <td>George Wu </td>
            <td>ChIPXpress: enhanced transcription factor target gene identification from ChIP-seq and ChIP-chip data using publicly available gene expression profiles</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/chopsticks.html">chopsticks</a></td>
            <td>Hin-Tak Leung </td>
            <td>The 'snp.matrix' and 'X.snp.matrix' Classes</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/chroGPS.html">chroGPS</a></td>
            <td>Oscar Reina </td>
            <td>chroGPS2: Generation, visualization and differential analysis of epigenome maps</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/chromDraw.html">chromDraw</a></td>
            <td>Jan Janecka </td>
            <td>chromDraw is a R package for drawing the schemes of karyotypes in the linear and circular fashion.</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/ChromHeatMap.html">ChromHeatMap</a></td>
            <td>Tim F. Rayner </td>
            <td>Heat map plotting by genome coordinate</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/chromPlot.html">chromPlot</a></td>
            <td>Karen Y. Orostica </td>
            <td>Global visualization tool of genomic data</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/chromstaR.html">chromstaR</a></td>
            <td>Aaron Taudt </td>
            <td>Combinatorial and Differential Chromatin State Analysis for ChIP-Seq Data</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/chromswitch.html">chromswitch</a></td>
            <td>Selin Jessa </td>
            <td>An R package to detect chromatin state switches from epigenomic data</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/chromVAR.html">chromVAR</a></td>
            <td>Alicia Schep </td>
            <td>Chromatin Variation Across Regions</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/CHRONOS.html">CHRONOS</a></td>
            <td>Panos Balomenos </td>
            <td>CHRONOS: A time-varying method for microRNA-mediated sub-pathway enrichment analysis</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/cicero.html">cicero</a></td>
            <td>Hannah Pliner </td>
            <td>Precict cis-co-accessibility from single-cell chromatin accessibility data</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/CINdex.html">CINdex</a></td>
            <td>Yuriy Gusev </td>
            <td>Chromosome Instability Index</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/circRNAprofiler.html">circRNAprofiler</a></td>
            <td>Simona Aufiero </td>
            <td>circRNAprofiler: An R-Based Computational Framework for the Downstream Analysis of Circular RNAs</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/cisPath.html">cisPath</a></td>
            <td>Likun Wang </td>
            <td>Visualization and management of the protein-protein interaction networks.</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/ClassifyR.html">ClassifyR</a></td>
            <td>Dario Strbenac </td>
            <td>A framework for cross-validated classification problems, with applications to differential variability and differential distribution testing</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/cleanUpdTSeq.html">cleanUpdTSeq</a></td>
            <td>Jianhong Ou ; Lihua Julie Zhu </td>
            <td>This package classifies putative polyadenylation sites as true or false/internally oligodT primed</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/cleaver.html">cleaver</a></td>
            <td>Sebastian Gibb </td>
            <td>Cleavage of Polypeptide Sequences</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/clippda.html">clippda</a></td>
            <td>Stephen Nyangoma </td>
            <td>A package for the clinical proteomic profiling data analysis</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/clipper.html">clipper</a></td>
            <td>Paolo Martini </td>
            <td>Gene Set Analysis Exploiting Pathway Topology</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/cliqueMS.html">cliqueMS</a></td>
            <td>Oriol Senan Campos </td>
            <td>Annotation of Isotopes, Adducts and Fragmentation Adducts for in-Source LC/MS Metabolomics Data</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/Clomial.html">Clomial</a></td>
            <td>Habil Zare </td>
            <td>Infers clonal composition of a tumor</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/Clonality.html">Clonality</a></td>
            <td>Irina Ostrovnaya </td>
            <td>Clonality testing</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/clonotypeR.html">clonotypeR</a></td>
            <td>Charles Plessy </td>
            <td>High throughput analysis of T cell antigen receptor sequences</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/clst.html">clst</a></td>
            <td>Noah Hoffman </td>
            <td>Classification by local similarity threshold</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/clstutils.html">clstutils</a></td>
            <td>Noah Hoffman </td>
            <td>Tools for performing taxonomic assignment.</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/CluMSID.html">CluMSID</a></td>
            <td>Tobias Depke </td>
            <td>Clustering of MS2 Spectra for Metabolite Identification</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/clustComp.html">clustComp</a></td>
            <td>Aurora Torrente </td>
            <td>Clustering Comparison Package</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/clusterExperiment.html">clusterExperiment</a></td>
            <td>Elizabeth Purdom </td>
            <td>Compare Clusterings for Single-Cell Sequencing</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/ClusterJudge.html">ClusterJudge</a></td>
            <td>Adrian Pasculescu </td>
            <td>Judging Quality of Clustering Methods using Mutual Information</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/clusterProfiler.html">clusterProfiler</a></td>
            <td>Guangchuang Yu </td>
            <td>statistical analysis and visualization of functional profiles for genes and gene clusters</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/clusterSeq.html">clusterSeq</a></td>
            <td>Thomas J. Hardcastle </td>
            <td>Clustering of high-throughput sequencing data by identifying co-expression patterns</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/ClusterSignificance.html">ClusterSignificance</a></td>
            <td>Jason T Serviss </td>
            <td>The ClusterSignificance package provides tools to assess if class clusters in dimensionality reduced data representations have a separation different from permuted data</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/clusterStab.html">clusterStab</a></td>
            <td>James W. MacDonald </td>
            <td>Compute cluster stability scores for microarray data</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/CMA.html">CMA</a></td>
            <td>Roman Hornung </td>
            <td>Synthesis of microarray-based classification</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/cn.farms.html">cn.farms</a></td>
            <td>Andreas Mitterecker </td>
            <td>cn.FARMS - factor analysis for copy number estimation</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/cn.mops.html">cn.mops</a></td>
            <td>Guenter Klambauer </td>
            <td>cn.mops - Mixture of Poissons for CNV detection in NGS data</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/CNAnorm.html">CNAnorm</a></td>
            <td>Stefano Berri </td>
            <td>A normalization method for Copy Number Aberration in cancer samples</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/CNEr.html">CNEr</a></td>
            <td>Ge Tan </td>
            <td>CNE Detection and Visualization</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/CNORdt.html">CNORdt</a></td>
            <td>A. MacNamara </td>
            <td>Add-on to CellNOptR: Discretized time treatments</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/CNORfeeder.html">CNORfeeder</a></td>
            <td>F.Eduati </td>
            <td>Integration of CellNOptR to add missing links</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/CNORfuzzy.html">CNORfuzzy</a></td>
            <td>T. Cokelaer </td>
            <td>Addon to CellNOptR: Fuzzy Logic</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/CNORode.html">CNORode</a></td>
            <td>David Henriques </td>
            <td>ODE add-on to CellNOptR</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/CNPBayes.html">CNPBayes</a></td>
            <td>Jacob Carey </td>
            <td>Bayesian mixture models for copy number polymorphisms</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/CNTools.html">CNTools</a></td>
            <td>J. Zhang </td>
            <td>Convert segment data into a region by sample matrix to allow for other high level computational analyses.</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/CNVfilteR.html">CNVfilteR</a></td>
            <td>Jose Marcos Moreno-Cabrera </td>
            <td>Identifies false positives of CNV calling tools by using SNV calls</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/cnvGSA.html">cnvGSA</a></td>
            <td>Joseph Lugo </td>
            <td>Gene Set Analysis of (Rare) Copy Number Variants</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/CNVPanelizer.html">CNVPanelizer</a></td>
            <td>Thomas Wolf </td>
            <td>Reliable CNV detection in targeted sequencing applications</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/CNVRanger.html">CNVRanger</a></td>
            <td>Ludwig Geistlinger </td>
            <td>Summarization and expression/phenotype association of CNV ranges</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/CNVrd2.html">CNVrd2</a></td>
            <td>Hoang Tan Nguyen </td>
            <td>CNVrd2: a read depth-based method to detect and genotype complex common copy number variants from next generation sequencing data.</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/CNVtools.html">CNVtools</a></td>
            <td>Chris Barnes </td>
            <td>A package to test genetic association with CNV data</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/cobindR.html">cobindR</a></td>
            <td>Manuela Benary </td>
            <td>Finding Co-occuring motifs of transcription factor binding sites</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/CoCiteStats.html">CoCiteStats</a></td>
            <td>Bioconductor Package Maintainer </td>
            <td>Different test statistics based on co-citation.</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/COCOA.html">COCOA</a></td>
            <td>John Lawson </td>
            <td>Coordinate Covariation Analysis</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/codelink.html">codelink</a></td>
            <td>Diego Diez </td>
            <td>Manipulation of Codelink microarray data</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/CODEX.html">CODEX</a></td>
            <td>Yuchao Jiang </td>
            <td>A Normalization and Copy Number Variation Detection Method for Whole Exome Sequencing</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/coexnet.html">coexnet</a></td>
            <td>Juan David Henao </td>
            <td>coexnet: An R package to build CO-EXpression NETworks from Microarray Data</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/CoGAPS.html">CoGAPS</a></td>
            <td>Elana J. Fertig, Thomas D. Sherman </td>
            <td>Coordinated Gene Activity in Pattern Sets</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/cogena.html">cogena</a></td>
            <td>Zhilong Jia </td>
            <td>co-expressed gene-set enrichment analysis</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/coGPS.html">coGPS</a></td>
            <td>Yingying Wei </td>
            <td>cancer outlier Gene Profile Sets</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/COHCAP.html">COHCAP</a></td>
            <td>Charles Warden </td>
            <td>CpG Island Analysis Pipeline for Illumina Methylation Array and Targeted BS-Seq Data</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/cola.html">cola</a></td>
            <td>Zuguang Gu </td>
            <td>A Framework for Consensus Partitioning</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/coMET.html">coMET</a></td>
            <td>Tiphaine Martin </td>
            <td>coMET: visualisation of regional epigenome-wide association scan (EWAS) results and DNA co-methylation patterns</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/compartmap.html">compartmap</a></td>
            <td>Benjamin Johnson </td>
            <td>A/B compartment inference from ATAC-seq and methylation array data</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/COMPASS.html">COMPASS</a></td>
            <td>Greg Finak </td>
            <td>Combinatorial Polyfunctionality Analysis of Single Cells</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/compcodeR.html">compcodeR</a></td>
            <td>Charlotte Soneson </td>
            <td>RNAseq data simulation, differential expression analysis and performance comparison of differential expression methods</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/compEpiTools.html">compEpiTools</a></td>
            <td>Kamal Kishore </td>
            <td>Tools for computational epigenomics</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/CompGO.html">CompGO</a></td>
            <td>Ashley J. Waardenberg </td>
            <td>An R pipeline for .bed file annotation, comparing GO term enrichment between gene sets and data visualisation</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/ComplexHeatmap.html">ComplexHeatmap</a></td>
            <td>Zuguang Gu </td>
            <td>Make Complex Heatmaps</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/condcomp.html">condcomp</a></td>
            <td></td>
            <td>Condition Comparison in scRNA-seq Data</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/CONFESS.html">CONFESS</a></td>
            <td>Diana LOW </td>
            <td>Cell OrderiNg by FluorEScence Signal</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/consensus.html">consensus</a></td>
            <td>Tim Peters </td>
            <td>Cross-platform consensus analysis of genomic measurements via interlaboratory testing method</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/ConsensusClusterPlus.html">ConsensusClusterPlus</a></td>
            <td>Matt Wilkerson </td>
            <td>ConsensusClusterPlus</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/consensusDE.html">consensusDE</a></td>
            <td>Ashley J. Waardenberg </td>
            <td>RNA-seq analysis using multiple algorithms</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/consensusOV.html">consensusOV</a></td>
            <td>Benjamin Haibe-Kains </td>
            <td>Gene expression-based subtype classification for high-grade serous ovarian cancer</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/consensusSeekeR.html">consensusSeekeR</a></td>
            <td>Astrid Deschenes </td>
            <td>Detection of consensus regions inside a group of experiences using genomic positions and genomic ranges</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/contiBAIT.html">contiBAIT</a></td>
            <td>Kieran O'Neill </td>
            <td>Improves Early Build Genome Assemblies using Strand-Seq Data</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/conumee.html">conumee</a></td>
            <td>Volker Hovestadt </td>
            <td>Enhanced copy-number variation analysis using Illumina DNA methylation arrays</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/convert.html">convert</a></td>
            <td>Yee Hwa (Jean) Yang </td>
            <td>Convert Microarray Data Objects</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/copa.html">copa</a></td>
            <td>James W. MacDonald </td>
            <td>Functions to perform cancer outlier profile analysis.</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/copynumber.html">copynumber</a></td>
            <td>Gro Nilsen </td>
            <td>Segmentation of single- and multi-track copy number data by penalized least squares regression.</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/CopyNumberPlots.html">CopyNumberPlots</a></td>
            <td>Bernat Gel </td>
            <td>Create Copy-Number Plots using karyoploteR functionality</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/CopywriteR.html">CopywriteR</a></td>
            <td>Oscar Krijgsman </td>
            <td>Copy number information from targeted sequencing using off-target reads</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/coRdon.html">coRdon</a></td>
            <td>Anamaria Elek </td>
            <td>Codon Usage Analysis and Prediction of Gene Expressivity</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/CoRegFlux.html">CoRegFlux</a></td>
            <td>Pauline Trébulle and Mohamed Elati </td>
            <td>CoRegFlux</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/CoRegNet.html">CoRegNet</a></td>
            <td>Remy Nicolle </td>
            <td>CoRegNet : reconstruction and integrated analysis of co-regulatory networks</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/Cormotif.html">Cormotif</a></td>
            <td>Yingying Wei </td>
            <td>Correlation Motif Fit</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/CorMut.html">CorMut</a></td>
            <td>Zhenpeng Li</td>
            <td>Detect the correlated mutations based on selection pressure</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/CORREP.html">CORREP</a></td>
            <td>Dongxiao Zhu </td>
            <td>Multivariate Correlation Estimator and Statistical Inference Procedures.</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/coseq.html">coseq</a></td>
            <td>Andrea Rau </td>
            <td>Co-Expression Analysis of Sequencing Data</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/cosmiq.html">cosmiq</a></td>
            <td>David Fischer </td>
            <td>cosmiq - COmbining Single Masses Into Quantities</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/COSNet.html">COSNet</a></td>
            <td>Marco Frasca</td>
            <td>Cost Sensitive Network for node label prediction on graphs with highly unbalanced labelings</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/CountClust.html">CountClust</a></td>
            <td>Kushal Dey </td>
            <td>Clustering and Visualizing RNA-Seq Expression Data using Grade of Membership Models</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/countsimQC.html">countsimQC</a></td>
            <td>Charlotte Soneson </td>
            <td>Compare Characteristic Features of Count Data Sets</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/covEB.html">covEB</a></td>
            <td>C. Pacini </td>
            <td>Empirical Bayes estimate of block diagonal covariance matrices</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/CoverageView.html">CoverageView</a></td>
            <td>Ernesto Lowy </td>
            <td>Coverage visualization package for R</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/covRNA.html">covRNA</a></td>
            <td>Lara Urban </td>
            <td>Multivariate Analysis of Transcriptomic Data</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/cpvSNP.html">cpvSNP</a></td>
            <td>Caitlin McHugh </td>
            <td>Gene set analysis methods for SNP association p-values that lie in genes in given gene sets</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/cqn.html">cqn</a></td>
            <td>Kasper Daniel Hansen </td>
            <td>Conditional quantile normalization</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/CRImage.html">CRImage</a></td>
            <td>Henrik Failmezger, Yinyin Yuan </td>
            <td>CRImage a package to classify cells and calculate tumour cellularity</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/CRISPRseek.html">CRISPRseek</a></td>
            <td>Lihua Julie Zhu </td>
            <td>Design of target-specific guide RNAs in CRISPR-Cas9, genome-editing systems</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/crisprseekplus.html">crisprseekplus</a></td>
            <td>Alper Kucukural </td>
            <td>crisprseekplus</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/CrispRVariants.html">CrispRVariants</a></td>
            <td>Helen Lindsay </td>
            <td>Tools for counting and visualising mutations in a target location</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/crlmm.html">crlmm</a></td>
            <td>Benilton S Carvalho, Robert Scharpf, Matt Ritchie </td>
            <td>Genotype Calling (CRLMM) and Copy Number Analysis tool for Affymetrix SNP 5.0 and 6.0 and Illumina arrays</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/CrossICC.html">CrossICC</a></td>
            <td>Yu Sun </td>
            <td>An Interactive Consensus Clustering Framework for Multi-platform Data Analysis</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/crossmeta.html">crossmeta</a></td>
            <td>Alex Pickering </td>
            <td>Cross Platform Meta-Analysis of Microarray Data</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/CSAR.html">CSAR</a></td>
            <td>Jose M Muino </td>
            <td>Statistical tools for the analysis of ChIP-seq data</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/csaw.html">csaw</a></td>
            <td>Aaron Lun </td>
            <td>ChIP-Seq Analysis with Windows</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/CSSP.html">CSSP</a></td>
            <td>Chandler Zuo</td>
            <td>ChIP-Seq Statistical Power</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/ctc.html">ctc</a></td>
            <td>Antoine Lucas </td>
            <td>Cluster and Tree Conversion.</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/CTDquerier.html">CTDquerier</a></td>
            <td>Carles Hernandez-Ferrer </td>
            <td>Package for CTDbase data query, visualization and downstream analysis</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/cTRAP.html">cTRAP</a></td>
            <td>Nuno Saraiva-Agostinho </td>
            <td>Identification of candidate causal perturbations from differential gene expression data</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/ctsGE.html">ctsGE</a></td>
            <td>Michal Sharabi-Schwager </td>
            <td>Clustering of Time Series Gene Expression data</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/cummeRbund.html">cummeRbund</a></td>
            <td>Loyal A. Goff </td>
            <td>Analysis, exploration, manipulation, and visualization of Cufflinks high-throughput sequencing data.</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/customProDB.html">customProDB</a></td>
            <td>Xiaojing Wang Bo Wen </td>
            <td>Generate customized protein database from NGS data, with a focus on RNA-Seq data, for proteomics search</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/CVE.html">CVE</a></td>
            <td>Andreas Mock </td>
            <td>Cancer Variant Explorer</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/cycle.html">cycle</a></td>
            <td>Matthias Futschik </td>
            <td>Significance of periodic expression pattern in time-series data</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/cydar.html">cydar</a></td>
            <td>Aaron Lun </td>
            <td>Using Mass Cytometry for Differential Abundance Analyses</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/CytoDx.html">CytoDx</a></td>
            <td>Zicheng Hu </td>
            <td>Robust prediction of clinical outcomes using cytometry data without cell gating</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/cytofast.html">cytofast</a></td>
            <td>K.A. Stam </td>
            <td>cytofast - A quick visualization and analysis tool for CyTOF data</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/cytolib.html">cytolib</a></td>
            <td>Mike Jiang </td>
            <td>C++ infrastructure for representing and interacting with the gated cytometry</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/CytoML.html">CytoML</a></td>
            <td>Mike Jiang </td>
            <td>A GatingML Interface for Cross Platform Cytometry Data Sharing</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/dada2.html">dada2</a></td>
            <td>Benjamin Callahan </td>
            <td>Accurate, high-resolution sample inference from amplicon sequencing data</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/dagLogo.html">dagLogo</a></td>
            <td>Jianhong Ou </td>
            <td>dagLogo: a bioconductor package for visualizeing conserved amino acid sequence pattern in groups based on probability theory</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/daMA.html">daMA</a></td>
            <td>Jobst Landgrebe </td>
            <td>Efficient design and analysis of factorial two-colour microarray data</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/DaMiRseq.html">DaMiRseq</a></td>
            <td>Mattia Chiesa </td>
            <td>Data Mining for RNA-seq data: normalization, feature selection and classification</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/DAPAR.html">DAPAR</a></td>
            <td>Samuel Wieczorek </td>
            <td>Tools for the Differential Analysis of Proteins Abundance with R</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/DART.html">DART</a></td>
            <td>Charles Shijie Zheng </td>
            <td>Denoising Algorithm based on Relevance network Topology</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/DBChIP.html">DBChIP</a></td>
            <td>Kun Liang </td>
            <td>Differential Binding of Transcription Factor with ChIP-seq</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/dcanr.html">dcanr</a></td>
            <td>Dharmesh D. Bhuva </td>
            <td>Differential co-expression/association network analysis</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/dcGSA.html">dcGSA</a></td>
            <td>Jiehuan sun </td>
            <td>Distance-correlation based Gene Set Analysis for longitudinal gene expression profiles</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/DChIPRep.html">DChIPRep</a></td>
            <td>Bernd Klaus </td>
            <td>DChIPRep - Analysis of chromatin modification ChIP-Seq data with replication</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/ddCt.html">ddCt</a></td>
            <td>Jitao David Zhang </td>
            <td>The ddCt Algorithm for the Analysis of Quantitative Real-Time PCR (qRT-PCR)</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/ddPCRclust.html">ddPCRclust</a></td>
            <td>Benedikt G. Brink </td>
            <td>Clustering algorithm for ddPCR data</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/debCAM.html">debCAM</a></td>
            <td>Lulu Chen </td>
            <td>Deconvolution by Convex Analysis of Mixtures</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/debrowser.html">debrowser</a></td>
            <td>Alper Kucukural </td>
            <td>Interactive Differential Expresion Analysis Browser</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/DECIPHER.html">DECIPHER</a></td>
            <td>Erik Wright </td>
            <td>Tools for curating, analyzing, and manipulating biological sequences</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/deco.html">deco</a></td>
            <td>Francisco Jose Campos Laborie </td>
            <td>Decomposing Heterogeneous Cohorts using Omic Data Profiling</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/DEComplexDisease.html">DEComplexDisease</a></td>
            <td>Guofeng Meng </td>
            <td>A tool for differential expression analysis and DEGs based investigation to complex diseases by bi-clustering analysis</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/decompTumor2Sig.html">decompTumor2Sig</a></td>
            <td>Rosario M. Piro </td>
            <td>Decomposition of individual tumors into mutational signatures by signature refitting</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/DeconRNASeq.html">DeconRNASeq</a></td>
            <td>Ting Gong </td>
            <td>Deconvolution of Heterogeneous Tissue Samples for mRNA-Seq data</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/decontam.html">decontam</a></td>
            <td>Benjamin Callahan </td>
            <td>Identify Contaminants in Marker-gene and Metagenomics Sequencing Data</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/DEDS.html">DEDS</a></td>
            <td>Yuanyuan Xiao </td>
            <td>Differential Expression via Distance Summary for Microarray Data</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/DeepBlueR.html">DeepBlueR</a></td>
            <td>Felipe Albrecht, Markus List </td>
            <td>DeepBlueR</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/deepSNV.html">deepSNV</a></td>
            <td>Moritz Gerstung </td>
            <td>Detection of subclonal SNVs in deep sequencing data.</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/DEFormats.html">DEFormats</a></td>
            <td>Andrzej Oleś </td>
            <td>Differential gene expression data formats converter</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/DEGraph.html">DEGraph</a></td>
            <td>Laurent Jacob </td>
            <td>Two-sample tests on a graph</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/DEGreport.html">DEGreport</a></td>
            <td>Lorena Pantano </td>
            <td>Report of DEG analysis</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/DEGseq.html">DEGseq</a></td>
            <td>Likun Wang </td>
            <td>Identify Differentially Expressed Genes from RNA-seq data</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/DelayedArray.html">DelayedArray</a></td>
            <td>Hervé Pagès </td>
            <td>A unified framework for working transparently with on-disk and in-memory array-like datasets</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/DelayedDataFrame.html">DelayedDataFrame</a></td>
            <td>Qian Liu </td>
            <td>Delayed operation on DataFrame using standard DataFrame metaphor</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/DelayedMatrixStats.html">DelayedMatrixStats</a></td>
            <td>Peter Hickey </td>
            <td>Functions that Apply to Rows and Columns of 'DelayedMatrix' Objects</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/deltaCaptureC.html">deltaCaptureC</a></td>
            <td>Michael Shapiro </td>
            <td>This Package Discovers Meso-scale Chromatin Remodeling from 3C Data</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/deltaGseg.html">deltaGseg</a></td>
            <td>Diana Low </td>
            <td>deltaGseg</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/DeMAND.html">DeMAND</a></td>
            <td>Jung Hoon Woo, Mariano Alvarez </td>
            <td>DeMAND</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/DeMixT.html">DeMixT</a></td>
            <td>Shaolong Cao, Peng Yang </td>
            <td>Cell type-specific deconvolution of heterogeneous tumor samples with two or three components using expression data from RNAseq or microarray platforms</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/DEP.html">DEP</a></td>
            <td>Arne Smits </td>
            <td>Differential Enrichment analysis of Proteomics data</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/DepecheR.html">DepecheR</a></td>
            <td>Jakob Theorell </td>
            <td>Determination of essential phenotypic elements of clusters in high-dimensional entities</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/DEqMS.html">DEqMS</a></td>
            <td>Yafeng Zhu </td>
            <td>a tool to perform statistical analysis of differential protein expression for quantitative proteomics data.</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/derfinder.html">derfinder</a></td>
            <td>Leonardo Collado-Torres </td>
            <td>Annotation-agnostic differential expression analysis of RNA-seq data at base-pair resolution via the DER Finder approach</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/derfinderHelper.html">derfinderHelper</a></td>
            <td>Leonardo Collado-Torres </td>
            <td>derfinder helper package</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/derfinderPlot.html">derfinderPlot</a></td>
            <td>Leonardo Collado-Torres </td>
            <td>Plotting functions for derfinder</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/DEScan2.html">DEScan2</a></td>
            <td>Dario Righelli </td>
            <td>Differential Enrichment Scan 2</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/DESeq.html">DESeq</a></td>
            <td>Simon Anders </td>
            <td>Differential gene expression analysis based on the negative binomial distribution</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/DESeq2.html">DESeq2</a></td>
            <td>Michael Love </td>
            <td>Differential gene expression analysis based on the negative binomial distribution</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/DEsingle.html">DEsingle</a></td>
            <td>Zhun Miao </td>
            <td>DEsingle for detecting three types of differential expression in single-cell RNA-seq data</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/destiny.html">destiny</a></td>
            <td>Philipp Angerer </td>
            <td>Creates diffusion maps</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/DEsubs.html">DEsubs</a></td>
            <td>Aristidis G. Vrahatis, Panos Balomenos </td>
            <td>DEsubs: an R package for flexible identification of differentially expressed subpathways using RNA-seq expression experiments</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/DEWSeq.html">DEWSeq</a></td>
            <td>Hentze bioinformatics team </td>
            <td>Differential Expressed Windows Based on Negative Binomial Distribution</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/DEXSeq.html">DEXSeq</a></td>
            <td>Alejandro Reyes </td>
            <td>Inference of differential exon usage in RNA-Seq</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/dexus.html">dexus</a></td>
            <td>Guenter Klambauer </td>
            <td>DEXUS - Identifying Differential Expression in RNA-Seq Studies with Unknown Conditions or without Replicates</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/DFP.html">DFP</a></td>
            <td>Rodrigo Alvarez-Glez </td>
            <td>Gene Selection</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/DiffBind.html">DiffBind</a></td>
            <td>Rory Stark</td>
            <td>Differential Binding Analysis of ChIP-Seq Peak Data</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/diffcoexp.html">diffcoexp</a></td>
            <td>Wenbin Wei </td>
            <td>Differential Co-expression Analysis</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/diffcyt.html">diffcyt</a></td>
            <td>Lukas M. Weber </td>
            <td>Differential discovery in high-dimensional cytometry via high-resolution clustering</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/diffGeneAnalysis.html">diffGeneAnalysis</a></td>
            <td>Choudary Jagarlamudi </td>
            <td>Performs differential gene expression Analysis</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/diffHic.html">diffHic</a></td>
            <td>Aaron Lun </td>
            <td>Differential Analyis of Hi-C Data</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/DiffLogo.html">DiffLogo</a></td>
            <td>Hendrik Treutler</td>
            <td>DiffLogo: A comparative visualisation of biooligomer motifs</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/diffloop.html">diffloop</a></td>
            <td>Caleb Lareau </td>
            <td>Identifying differential DNA loops from chromatin topology data</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/diffuStats.html">diffuStats</a></td>
            <td>Sergio Picart-Armada </td>
            <td>Diffusion scores on biological networks</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/diggit.html">diggit</a></td>
            <td>Mariano J Alvarez </td>
            <td>Inference of Genetic Variants Driving Cellular Phenotypes</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/Director.html">Director</a></td>
            <td>Katherine Icay </td>
            <td>A dynamic visualization tool of multi-level data</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/DirichletMultinomial.html">DirichletMultinomial</a></td>
            <td>Martin Morgan </td>
            <td>Dirichlet-Multinomial Mixture Model Machine Learning for Microbiome Data</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/discordant.html">discordant</a></td>
            <td>Charlotte Siska </td>
            <td>The Discordant Method: A Novel Approach for Differential Correlation</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/DiscoRhythm.html">DiscoRhythm</a></td>
            <td>Matthew Carlucci </td>
            <td>Interactive Workflow for Discovering Rhythmicity in Biological Data</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/divergence.html">divergence</a></td>
            <td>Wikum Dinalankara, Luigi Marchionni </td>
            <td>Divergence: Functionality for assessing omics data by divergence with respect to a baseline</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/dks.html">dks</a></td>
            <td>Jeffrey T. Leek </td>
            <td>The double Kolmogorov-Smirnov package for evaluating multiple testing procedures.</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/DMCFB.html">DMCFB</a></td>
            <td>Farhad Shokoohi </td>
            <td>Differentially Methylated Cytosines via a Bayesian Functional Approach</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/DMCHMM.html">DMCHMM</a></td>
            <td>Farhad Shokoohi </td>
            <td>Differentially Methylated CpG using Hidden Markov Model</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/DMRcaller.html">DMRcaller</a></td>
            <td>Nicolae Radu Zabet </td>
            <td>Differentially Methylated Regions caller</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/DMRcate.html">DMRcate</a></td>
            <td>Tim Peters </td>
            <td>Methylation array and sequencing spatial analysis methods</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/DMRforPairs.html">DMRforPairs</a></td>
            <td>Martin Rijlaarsdam </td>
            <td>DMRforPairs: identifying Differentially Methylated Regions between unique samples using array based methylation profiles</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/DMRScan.html">DMRScan</a></td>
            <td>Christian M Page </td>
            <td>Detection of Differentially Methylated Regions</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/dmrseq.html">dmrseq</a></td>
            <td>Keegan Korthauer </td>
            <td>Detection and inference of differentially methylated regions from Whole Genome Bisulfite Sequencing</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/DNABarcodeCompatibility.html">DNABarcodeCompatibility</a></td>
            <td>Céline Trébeau </td>
            <td>A Tool for Optimizing Combinations of DNA Barcodes Used in Multiplexed Experiments on Next Generation Sequencing Platforms</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/DNABarcodes.html">DNABarcodes</a></td>
            <td>Tilo Buschmann </td>
            <td>A tool for creating and analysing DNA barcodes used in Next Generation Sequencing multiplexing experiments</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/DNAcopy.html">DNAcopy</a></td>
            <td>Venkatraman E. Seshan </td>
            <td>DNA copy number data analysis</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/DNAshapeR.html">DNAshapeR</a></td>
            <td>Tsu-Pei Chiu </td>
            <td>High-throughput prediction of DNA shape features</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/DominoEffect.html">DominoEffect</a></td>
            <td>Marija Buljan, Peter Blattmann </td>
            <td>Identification and Annotation of Protein Hotspot Residues</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/doppelgangR.html">doppelgangR</a></td>
            <td>Levi Waldron </td>
            <td>Identify likely duplicate samples from genomic or meta-data</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/Doscheda.html">Doscheda</a></td>
            <td>Bruno Contrino </td>
            <td>A DownStream Chemo-Proteomics Analysis Pipeline</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/DOSE.html">DOSE</a></td>
            <td>Guangchuang Yu </td>
            <td>Disease Ontology Semantic and Enrichment analysis</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/doseR.html">doseR</a></td>
            <td>ake.vastermark </td>
            <td>doseR</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/drawProteins.html">drawProteins</a></td>
            <td>Paul Brennan </td>
            <td>Package to Draw Protein Schematics from Uniprot API output</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/DRIMSeq.html">DRIMSeq</a></td>
            <td>Malgorzata Nowicka </td>
            <td>Differential transcript usage and tuQTL analyses with Dirichlet-multinomial model in RNA-seq</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/DriverNet.html">DriverNet</a></td>
            <td>Jiarui Ding </td>
            <td>Drivernet: uncovering somatic driver mutations modulating transcriptional networks in cancer</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/DropletUtils.html">DropletUtils</a></td>
            <td>Aaron Lun </td>
            <td>Utilities for Handling Single-Cell Droplet Data</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/DrugVsDisease.html">DrugVsDisease</a></td>
            <td>j. Saez-Rodriguez </td>
            <td>Comparison of disease and drug profiles using Gene set Enrichment Analysis</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/dSimer.html">dSimer</a></td>
            <td>Peng Ni </td>
            <td>Integration of Disease Similarity Methods</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/DSS.html">DSS</a></td>
            <td>Hao Wu </td>
            <td>Dispersion shrinkage for sequencing data</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/DTA.html">DTA</a></td>
            <td>Bjoern Schwalb </td>
            <td>Dynamic Transcriptome Analysis</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/dualKS.html">dualKS</a></td>
            <td>Eric J. Kort, Yarong Yang </td>
            <td>Dual KS Discriminant Analysis and Classification</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/DupChecker.html">DupChecker</a></td>
            <td>"Quanhu SHENG" </td>
            <td>a package for checking high-throughput genomic data redundancy in meta-analysis</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/dupRadar.html">dupRadar</a></td>
            <td>Sergi Sayols, Holger Klein </td>
            <td>Assessment of duplication rates in RNA-Seq datasets</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/dyebias.html">dyebias</a></td>
            <td>Philip Lijnzaad </td>
            <td>The GASSCO method for correcting for slide-dependent gene-specific dye bias</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/DynDoc.html">DynDoc</a></td>
            <td>Bioconductor Package Maintainer </td>
            <td>Dynamic document tools</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/EasyqpcR.html">EasyqpcR</a></td>
            <td>Le Pape Sylvain </td>
            <td>EasyqpcR for low-throughput real-time quantitative PCR data analysis</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/easyRNASeq.html">easyRNASeq</a></td>
            <td>Nicolas Delhomme </td>
            <td>Count summarization and normalization for RNA-Seq data</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/EBarrays.html">EBarrays</a></td>
            <td>Ming Yuan </td>
            <td>Unified Approach for Simultaneous Gene Clustering and Differential Expression Identification</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/EBcoexpress.html">EBcoexpress</a></td>
            <td>John A. Dawson </td>
            <td>EBcoexpress for Differential Co-Expression Analysis</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/EBImage.html">EBImage</a></td>
            <td>Andrzej Oleś </td>
            <td>Image processing and analysis toolbox for R</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/EBSEA.html">EBSEA</a></td>
            <td>Arfa Mehmood </td>
            <td>Exon Based Strategy for Expression Analysis of genes</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/EBSeq.html">EBSeq</a></td>
            <td>Ning Leng </td>
            <td>An R package for gene and isoform differential expression analysis of RNA-seq data</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/EBSeqHMM.html">EBSeqHMM</a></td>
            <td>Ning Leng </td>
            <td>Bayesian analysis for identifying gene or isoform expression changes in ordered RNA-seq experiments</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/ecolitk.html">ecolitk</a></td>
            <td>Laurent Gautier </td>
            <td>Meta-data and tools for E. coli</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/EDASeq.html">EDASeq</a></td>
            <td>Davide Risso </td>
            <td>Exploratory Data Analysis and Normalization for RNA-Seq</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/EDDA.html">EDDA</a></td>
            <td>Chia Kuan Hui Burton, Niranjan Nagarajan </td>
            <td>Experimental Design in Differential Abundance analysis</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/edge.html">edge</a></td>
            <td>John D. Storey, Andrew J. Bass </td>
            <td>Extraction of Differential Gene Expression</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/edgeR.html">edgeR</a></td>
            <td>Yunshun Chen, Aaron Lun, Mark Robinson, Gordon Smyth </td>
            <td>Empirical Analysis of Digital Gene Expression Data in R</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/eegc.html">eegc</a></td>
            <td>Xiaoyuan Zhou </td>
            <td>Engineering Evaluation by Gene Categorization (eegc)</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/EGAD.html">EGAD</a></td>
            <td>Sara Ballouz </td>
            <td>Extending guilt by association by degree</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/EGSEA.html">EGSEA</a></td>
            <td>Monther Alhamdoosh </td>
            <td>Ensemble of Gene Set Enrichment Analyses</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/eiR.html">eiR</a></td>
            <td>Thomas Girke </td>
            <td>Accelerated similarity searching of small molecules</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/eisa.html">eisa</a></td>
            <td>Gabor Csardi </td>
            <td>Expression data analysis via the Iterative Signature Algorithm</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/ELBOW.html">ELBOW</a></td>
            <td>Graham Alvare, Xiangli Zhang </td>
            <td>ELBOW - Evaluating foLd change By the lOgit Way</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/ELMER.html">ELMER</a></td>
            <td>Tiago Chedraoui Silva </td>
            <td>Inferring Regulatory Element Landscapes and Transcription Factor Networks Using Cancer Methylomes</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/EMDomics.html">EMDomics</a></td>
            <td>Sadhika Malladi and Daniel Schmolze </td>
            <td>Earth Mover's Distance for Differential Analysis of Genomics Data</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/EmpiricalBrownsMethod.html">EmpiricalBrownsMethod</a></td>
            <td>David Gibbs </td>
            <td>Uses Brown's method to combine p-values from dependent tests</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/ENCODExplorer.html">ENCODExplorer</a></td>
            <td>Charles Joly Beauparlant </td>
            <td>A compilation of ENCODE metadata</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/EnhancedVolcano.html">EnhancedVolcano</a></td>
            <td>Kevin Blighe </td>
            <td>Publication-ready volcano plots with enhanced colouring and labeling</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/ENmix.html">ENmix</a></td>
            <td>Zongli Xu </td>
            <td>Data preprocessing and quality control for Illumina HumanMethylation450 and MethylationEPIC BeadChip</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/EnrichedHeatmap.html">EnrichedHeatmap</a></td>
            <td>Zuguang Gu </td>
            <td>Making Enriched Heatmaps</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/EnrichmentBrowser.html">EnrichmentBrowser</a></td>
            <td>Ludwig Geistlinger </td>
            <td>Seamless navigation through combined results of set-based and network-based enrichment analysis</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/enrichplot.html">enrichplot</a></td>
            <td>Guangchuang Yu </td>
            <td>Visualization of Functional Enrichment Result</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/enrichTF.html">enrichTF</a></td>
            <td>Zheng Wei </td>
            <td>Transcription Factors Enrichment Analysis</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/ensembldb.html">ensembldb</a></td>
            <td>Johannes Rainer </td>
            <td>Utilities to create and use Ensembl-based annotation databases</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/ensemblVEP.html">ensemblVEP</a></td>
            <td>Bioconductor Package Maintainer </td>
            <td>R Interface to Ensembl Variant Effect Predictor</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/ENVISIONQuery.html">ENVISIONQuery</a></td>
            <td>Alex Lisovich, Roger Day </td>
            <td>Retrieval from the ENVISION bioinformatics data portal into R</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/EpiDISH.html">EpiDISH</a></td>
            <td>Shijie Charles Zheng</td>
            <td>Epigenetic Dissection of Intra-Sample-Heterogeneity</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/epigenomix.html">epigenomix</a></td>
            <td>Hans-Ulrich Klein </td>
            <td>Epigenetic and gene transcription data normalization and integration with mixture models</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/epihet.html">epihet</a></td>
            <td>Xiaowen Chen </td>
            <td>Determining Epigenetic Heterogeneity from Bisulfite Sequencing Data</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/epiNEM.html">epiNEM</a></td>
            <td>Martin Pirkl </td>
            <td>epiNEM</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/epivizr.html">epivizr</a></td>
            <td>Hector Corrada Bravo </td>
            <td>R Interface to epiviz web app</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/epivizrChart.html">epivizrChart</a></td>
            <td>Hector Corrada Bravo </td>
            <td>R interface to epiviz web components</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/epivizrData.html">epivizrData</a></td>
            <td>Hector Corrada Bravo </td>
            <td>Data Management API for epiviz interactive visualization app</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/epivizrServer.html">epivizrServer</a></td>
            <td>Hector Corrada Bravo </td>
            <td>WebSocket server infrastructure for epivizr apps and packages</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/epivizrStandalone.html">epivizrStandalone</a></td>
            <td>Hector Corrada Bravo </td>
            <td>Run Epiviz Interactive Genomic Data Visualization App within R</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/erccdashboard.html">erccdashboard</a></td>
            <td>Sarah Munro </td>
            <td>Assess Differential Gene Expression Experiments with ERCC Controls</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/erma.html">erma</a></td>
            <td>VJ Carey </td>
            <td>epigenomic road map adventures</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/ERSSA.html">ERSSA</a></td>
            <td>Zixuan Shao </td>
            <td>Empirical RNA-seq Sample Size Analysis</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/esATAC.html">esATAC</a></td>
            <td>Zheng Wei </td>
            <td>An Easy-to-use Systematic pipeline for ATACseq data analysis</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/esetVis.html">esetVis</a></td>
            <td>Laure Cougnaud </td>
            <td>Visualizations of expressionSet Bioconductor object</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/eudysbiome.html">eudysbiome</a></td>
            <td>Xiaoyuan Zhou </td>
            <td>Cartesian plot and contingency test on 16S Microbial data</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/evaluomeR.html">evaluomeR</a></td>
            <td>José Antonio Bernabé-Díaz </td>
            <td>Evaluation of Bioinformatics Metrics</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/EventPointer.html">EventPointer</a></td>
            <td>Juan Pablo Romero </td>
            <td>An effective identification of alternative splicing events using junction arrays and RNA-Seq data</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/ExCluster.html">ExCluster</a></td>
            <td>R. Matthew Tanner </td>
            <td>ExCluster robustly detects differentially expressed exons between two conditions of RNA-seq data, requiring at least two independent biological replicates per condition</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/ExiMiR.html">ExiMiR</a></td>
            <td>Sylvain Gubian </td>
            <td>R functions for the normalization of Exiqon miRNA array data</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/exomeCopy.html">exomeCopy</a></td>
            <td>Michael Love </td>
            <td>Copy number variant detection from exome sequencing read depth</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/exomePeak.html">exomePeak</a></td>
            <td>Lin Zhang, Lian Liu, Jia Meng </td>
            <td>exome-based anlaysis of MeRIP-Seq data: peak calling and differential analysis</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/ExperimentHub.html">ExperimentHub</a></td>
            <td>Bioconductor Package Maintainer </td>
            <td>Client to access ExperimentHub resources</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/ExperimentHubData.html">ExperimentHubData</a></td>
            <td>Bioconductor Package Maintainer </td>
            <td>Add resources to ExperimentHub</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/explorase.html">explorase</a></td>
            <td>Michael Lawrence </td>
            <td>GUI for exploratory data analysis of systems biology data</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/ExpressionAtlas.html">ExpressionAtlas</a></td>
            <td>Suhaib Mohammed </td>
            <td>Download datasets from EMBL-EBI Expression Atlas</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/ExpressionView.html">ExpressionView</a></td>
            <td>Gabor Csardi </td>
            <td>Visualize biclusters identified in gene expression data</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/fabia.html">fabia</a></td>
            <td>Andreas Mitterecker </td>
            <td>FABIA: Factor Analysis for Bicluster Acquisition</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/factDesign.html">factDesign</a></td>
            <td>Denise Scholtens </td>
            <td>Factorial designed microarray experiment analysis</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/FamAgg.html">FamAgg</a></td>
            <td>Johannes Rainer </td>
            <td>Pedigree Analysis and Familial Aggregation</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/farms.html">farms</a></td>
            <td>Djork-Arne Clevert </td>
            <td>FARMS - Factor Analysis for Robust Microarray Summarization</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/fastLiquidAssociation.html">fastLiquidAssociation</a></td>
            <td>Tina Gunderson </td>
            <td>functions for genome-wide application of Liquid Association</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/FastqCleaner.html">FastqCleaner</a></td>
            <td>Leandro Roser </td>
            <td>A Shiny Application for Quality Control, Filtering and Trimming of FASTQ Files</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/fastseg.html">fastseg</a></td>
            <td>Guenter Klambauer </td>
            <td>fastseg - a fast segmentation algorithm</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/FCBF.html">FCBF</a></td>
            <td>Tiago Lubiana </td>
            <td>Fast Correlation Based Filter for Feature Selection</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/fCCAC.html">fCCAC</a></td>
            <td>Pedro Madrigal </td>
            <td>functional Canonical Correlation Analysis to evaluate Covariance between nucleic acid sequencing datasets</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/fCI.html">fCI</a></td>
            <td>Shaojun Tang </td>
            <td>f-divergence Cutoff Index for Differential Expression Analysis in Transcriptomics and Proteomics</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/fcoex.html">fcoex</a></td>
            <td>Tiago Lubiana </td>
            <td>FCBF-based Co-Expression Networks for Single Cells</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/fcScan.html">fcScan</a></td>
            <td>Pierre Khoueiry Abdallah El-Kurdi </td>
            <td>fcScan for detecting clusters of coordinates with user defined options</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/fdrame.html">fdrame</a></td>
            <td>Effi Kenigsberg </td>
            <td>FDR adjustments of Microarray Experiments (FDR-AME)</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/FELLA.html">FELLA</a></td>
            <td>Sergio Picart-Armada </td>
            <td>Interpretation and enrichment for metabolomics data</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/FEM.html">FEM</a></td>
            <td>Zhen Yang </td>
            <td>Identification of Functional Epigenetic Modules</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/ffpe.html">ffpe</a></td>
            <td>Levi Waldron </td>
            <td>Quality assessment and control for FFPE microarray expression data</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/FGNet.html">FGNet</a></td>
            <td>Sara Aibar </td>
            <td>Functional Gene Networks derived from biological enrichment analyses</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/fgsea.html">fgsea</a></td>
            <td>Alexey Sergushichev </td>
            <td>Fast Gene Set Enrichment Analysis</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/FindMyFriends.html">FindMyFriends</a></td>
            <td>Thomas Lin Pedersen </td>
            <td>Microbial Comparative Genomics in R</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/FISHalyseR.html">FISHalyseR</a></td>
            <td>Karesh Arunakirinathan, Andreas Heindl </td>
            <td>FISHalyseR a package for automated FISH quantification</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/fishpond.html">fishpond</a></td>
            <td>Michael Love </td>
            <td>Fishpond: differential transcript and gene expression with inferential replicates</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/FitHiC.html">FitHiC</a></td>
            <td>Ruyu Tan </td>
            <td>Confidence estimation for intra-chromosomal contact maps</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/flagme.html">flagme</a></td>
            <td>Mark Robinson, Riccardo Romoli </td>
            <td>Analysis of Metabolomics GC/MS Data</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/flipflop.html">flipflop</a></td>
            <td>Elsa Bernard </td>
            <td>Fast lasso-based isoform prediction as a flow problem</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/flowAI.html">flowAI</a></td>
            <td>Gianni Monaco </td>
            <td>Automatic and interactive quality control for flow cytometry data</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/flowBeads.html">flowBeads</a></td>
            <td>Nikolas Pontikos </td>
            <td>flowBeads: Analysis of flow bead data</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/flowBin.html">flowBin</a></td>
            <td>Kieran O'Neill </td>
            <td>Combining multitube flow cytometry data by binning</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/flowcatchR.html">flowcatchR</a></td>
            <td>Federico Marini </td>
            <td>Tools to analyze in vivo microscopy imaging data focused on tracking flowing blood cells</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/flowCHIC.html">flowCHIC</a></td>
            <td>Author: Joachim Schumann </td>
            <td>Analyze flow cytometric data using histogram information</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/flowCL.html">flowCL</a></td>
            <td>Justin Meskas </td>
            <td>Semantic labelling of flow cytometric cell populations</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/flowClean.html">flowClean</a></td>
            <td>Kipper Fletez-Brant </td>
            <td>flowClean</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/flowClust.html">flowClust</a></td>
            <td>Greg Finak, Mike Jiang </td>
            <td>Clustering for Flow Cytometry</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/flowCore.html">flowCore</a></td>
            <td>Mike Jiang </td>
            <td>flowCore: Basic structures for flow cytometry data</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/flowCyBar.html">flowCyBar</a></td>
            <td>Joachim Schumann </td>
            <td>Analyze flow cytometric data using gate information</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/flowDensity.html">flowDensity</a></td>
            <td>Mehrnoush Malek </td>
            <td>Sequential Flow Cytometry Data Gating</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/flowFit.html">flowFit</a></td>
            <td>Davide Rambaldi </td>
            <td>Estimate proliferation in cell-tracking dye studies</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/flowFP.html">flowFP</a></td>
            <td>Herb Holyst </td>
            <td>Fingerprinting for Flow Cytometry</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/flowMap.html">flowMap</a></td>
            <td>Chiaowen Joyce Hsiao </td>
            <td>Mapping cell populations in flow cytometry data for cross-sample comparisons using the Friedman-Rafsky Test</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/flowMatch.html">flowMatch</a></td>
            <td>Ariful Azad </td>
            <td>Matching and meta-clustering in flow cytometry</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/flowMeans.html">flowMeans</a></td>
            <td>Nima Aghaeepour </td>
            <td>Non-parametric Flow Cytometry Data Gating</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/flowMerge.html">flowMerge</a></td>
            <td>Greg Finak </td>
            <td>Cluster Merging for Flow Cytometry Data</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/flowPeaks.html">flowPeaks</a></td>
            <td>Yongchao Ge</td>
            <td>An R package for flow data clustering</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/flowPloidy.html">flowPloidy</a></td>
            <td>Tyler Smith </td>
            <td>Analyze flow cytometer data to determine sample ploidy</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/flowPlots.html">flowPlots</a></td>
            <td>N. Hawkins </td>
            <td>flowPlots: analysis plots and data class for gated flow cytometry data</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/FlowRepositoryR.html">FlowRepositoryR</a></td>
            <td>Josef Spidlen </td>
            <td>FlowRepository R Interface</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/FlowSOM.html">FlowSOM</a></td>
            <td>Sofie Van Gassen </td>
            <td>Using self-organizing maps for visualization and interpretation of cytometry data</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/flowSpecs.html">flowSpecs</a></td>
            <td>Jakob Theorell </td>
            <td>Tools for processing of high-dimensional cytometry data</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/flowSpy.html">flowSpy</a></td>
            <td>Yuting Dai </td>
            <td>A Toolkit for Flow And Mass Cytometry Data</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/flowStats.html">flowStats</a></td>
            <td>Greg Finak and Mike Jiang </td>
            <td>Statistical methods for the analysis of flow cytometry data</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/flowTime.html">flowTime</a></td>
            <td>R. Clay Wright </td>
            <td>Annotation and analysis of biological dynamical systems using flow cytometry</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/flowTrans.html">flowTrans</a></td>
            <td>Greg Finak </td>
            <td>Parameter Optimization for Flow Cytometry Data Transformation</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/flowType.html">flowType</a></td>
            <td>Nima Aghaeepour </td>
            <td>Phenotyping Flow Cytometry Assays</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/flowUtils.html">flowUtils</a></td>
            <td>Josef Spidlen </td>
            <td>Utilities for flow cytometry</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/flowViz.html">flowViz</a></td>
            <td>Mike Jiang </td>
            <td>Visualization for flow cytometry</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/flowVS.html">flowVS</a></td>
            <td>Ariful Azad </td>
            <td>Variance stabilization in flow cytometry (and microarrays)</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/flowWorkspace.html">flowWorkspace</a></td>
            <td>Greg Finak,Mike Jiang </td>
            <td>Infrastructure for representing and interacting with gated and ungated cytometry data sets.</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/fmcsR.html">fmcsR</a></td>
            <td>Thomas Girke </td>
            <td>Mismatch Tolerant Maximum Common Substructure Searching</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/focalCall.html">focalCall</a></td>
            <td>Oscar Krijgsman </td>
            <td>Detection of focal aberrations in DNA copy number data</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/FoldGO.html">FoldGO</a></td>
            <td>Daniil Wiebe </td>
            <td>Package for Fold-specific GO Terms Recognition</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/FourCSeq.html">FourCSeq</a></td>
            <td>Felix A. Klein </td>
            <td>Package analyse 4C sequencing data</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/FRGEpistasis.html">FRGEpistasis</a></td>
            <td>Futao Zhang </td>
            <td>Epistasis Analysis for Quantitative Traits by Functional Regression Model</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/frma.html">frma</a></td>
            <td>Matthew N. McCall </td>
            <td>Frozen RMA and Barcode</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/frmaTools.html">frmaTools</a></td>
            <td>Matthew N. McCall </td>
            <td>Frozen RMA Tools</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/FunChIP.html">FunChIP</a></td>
            <td>Alice Parodi </td>
            <td>Clustering and Alignment of ChIP-Seq peaks based on their shapes</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/FunciSNP.html">FunciSNP</a></td>
            <td>Simon G. Coetzee </td>
            <td>Integrating Functional Non-coding Datasets with Genetic Association Studies to Identify Candidate Regulatory SNPs</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/funtooNorm.html">funtooNorm</a></td>
            <td>Kathleen Klein </td>
            <td>Normalization Procedure for Infinium HumanMethylation450 BeadChip Kit</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/GA4GHclient.html">GA4GHclient</a></td>
            <td>Welliton Souza </td>
            <td>A Bioconductor package for accessing GA4GH API data servers</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/GA4GHshiny.html">GA4GHshiny</a></td>
            <td>Welliton Souza </td>
            <td>Shiny application for interacting with GA4GH-based data servers</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/gaga.html">gaga</a></td>
            <td>David Rossell </td>
            <td>GaGa hierarchical model for high-throughput data analysis</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/gage.html">gage</a></td>
            <td>Weijun Luo </td>
            <td>Generally Applicable Gene-set Enrichment for Pathway Analysis</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/gaggle.html">gaggle</a></td>
            <td>Christopher Bare </td>
            <td>Broadcast data between R and Gaggle</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/gaia.html">gaia</a></td>
            <td>S. Morganella </td>
            <td>GAIA: An R package for genomic analysis of significant chromosomal aberrations.</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/GAPGOM.html">GAPGOM</a></td>
            <td>Casper Peters </td>
            <td>GAPGOM (novel Gene Annotation Prediction and other GO Metrics)</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/GAprediction.html">GAprediction</a></td>
            <td>Jon Bohlin </td>
            <td>Prediction of gestational age with Illumina HumanMethylation450 data</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/garfield.html">garfield</a></td>
            <td>Valentina Iotchkova </td>
            <td>GWAS Analysis of Regulatory or Functional Information Enrichment with LD correction</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/GARS.html">GARS</a></td>
            <td>Mattia Chiesa </td>
            <td>GARS: Genetic Algorithm for the identification of Robust Subsets of variables in high-dimensional and challenging datasets</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/GateFinder.html">GateFinder</a></td>
            <td>Nima Aghaeepour </td>
            <td>Projection-based Gating Strategy Optimization for Flow and Mass Cytometry</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/gcapc.html">gcapc</a></td>
            <td>Mingxiang Teng </td>
            <td>GC Aware Peak Caller</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/gcatest.html">gcatest</a></td>
            <td>Wei Hao, John D. Storey </td>
            <td>Genotype Conditional Association TEST</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/gCMAP.html">gCMAP</a></td>
            <td>Thomas Sandmann </td>
            <td>Tools for Connectivity Map-like analyses</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/gCMAPWeb.html">gCMAPWeb</a></td>
            <td>Thomas Sandmann </td>
            <td>A web interface for gene-set enrichment analyses</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/gCrisprTools.html">gCrisprTools</a></td>
            <td>Peter Haverty </td>
            <td>Suite of Functions for Pooled Crispr Screen QC and Analysis</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/gcrma.html">gcrma</a></td>
            <td>Z. Wu </td>
            <td>Background Adjustment Using Sequence Information</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/GCSscore.html">GCSscore</a></td>
            <td>Guy M. Harris </td>
            <td>GCSscore: an R package for microarray analysis for Affymetrix/Thermo Fisher arrays</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/GDCRNATools.html">GDCRNATools</a></td>
            <td>Ruidong Li, Han Qu </td>
            <td>GDCRNATools: an R/Bioconductor package for integrative analysis of lncRNA, mRNA, and miRNA data in GDC</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/GDSArray.html">GDSArray</a></td>
            <td>Qian Liu </td>
            <td>Representing GDS files as array-like objects</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/gdsfmt.html">gdsfmt</a></td>
            <td>Xiuwen Zheng </td>
            <td>R Interface to CoreArray Genomic Data Structure (GDS) Files</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/geecc.html">geecc</a></td>
            <td>Markus Boenn </td>
            <td>Gene Set Enrichment Analysis Extended to Contingency Cubes</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/GEM.html">GEM</a></td>
            <td>Hong Pan </td>
            <td>GEM: fast association study for the interplay of Gene, Environment and Methylation</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/gemini.html">gemini</a></td>
            <td>Sidharth Jain </td>
            <td>GEMINI: Variational inference approach to infer genetic interactions from pairwise CRISPR screens</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/genArise.html">genArise</a></td>
            <td>IFC Development Team </td>
            <td>Microarray Analysis tool</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/genbankr.html">genbankr</a></td>
            <td>Gabriel Becker </td>
            <td>Parsing GenBank files into semantically useful objects</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/GeneAccord.html">GeneAccord</a></td>
            <td>Ariane L. Moore </td>
            <td>Detection of clonally exclusive gene or pathway pairs in a cohort of cancer patients</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/GeneAnswers.html">GeneAnswers</a></td>
            <td>Lei Huang and Gang Feng </td>
            <td>Integrated Interpretation of Genes</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/geneAttribution.html">geneAttribution</a></td>
            <td>Arthur Wuster </td>
            <td>Identification of candidate genes associated with genetic variation</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/GeneBreak.html">GeneBreak</a></td>
            <td>Evert van den Broek </td>
            <td>Gene Break Detection</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/geneClassifiers.html">geneClassifiers</a></td>
            <td>R Kuiper </td>
            <td>Application of gene classifiers</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/GeneExpressionSignature.html">GeneExpressionSignature</a></td>
            <td>Yang Cao, Fei Li,Lu Han </td>
            <td>Gene Expression Signature based Similarity Metric</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/genefilter.html">genefilter</a></td>
            <td>Bioconductor Package Maintainer </td>
            <td>genefilter: methods for filtering genes from high-throughput experiments</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/genefu.html">genefu</a></td>
            <td>Benjamin Haibe-Kains </td>
            <td>Computation of Gene Expression-Based Signatures in Breast Cancer</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/GeneGA.html">GeneGA</a></td>
            <td>Zhenpeng Li </td>
            <td>Design gene based on both mRNA secondary structure and codon usage bias using Genetic algorithm</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/GeneGeneInteR.html">GeneGeneInteR</a></td>
            <td>Mathieu Emily, Magalie Houee-Bigot </td>
            <td>Tools for Testing Gene-Gene Interaction at the Gene Level</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/GeneMeta.html">GeneMeta</a></td>
            <td>Bioconductor Package Maintainer </td>
            <td>MetaAnalysis for High Throughput Experiments</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/GeneNetworkBuilder.html">GeneNetworkBuilder</a></td>
            <td>Jianhong Ou </td>
            <td>GeneNetworkBuilder: a bioconductor package for building regulatory network using ChIP-chip/ChIP-seq data and Gene Expression Data</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/GeneOverlap.html">GeneOverlap</a></td>
            <td>Li Shen, Mount Sinai </td>
            <td>Test and visualize gene overlaps</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/geneplast.html">geneplast</a></td>
            <td>Mauro Castro </td>
            <td>Evolutionary and plasticity analysis of orthologous groups</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/geneplotter.html">geneplotter</a></td>
            <td>Bioconductor Package Maintainer </td>
            <td>Graphics related functions for Bioconductor</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/geneRecommender.html">geneRecommender</a></td>
            <td>Greg Hather </td>
            <td>A gene recommender algorithm to identify genes coexpressed with a query set of genes</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/GeneRegionScan.html">GeneRegionScan</a></td>
            <td>Lasse Folkersen </td>
            <td>GeneRegionScan</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/geneRxCluster.html">geneRxCluster</a></td>
            <td>Charles Berry </td>
            <td>gRx Differential Clustering</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/GeneSelectMMD.html">GeneSelectMMD</a></td>
            <td>Weiliang Qiu </td>
            <td>Gene selection based on the marginal distributions of gene profiles that characterized by a mixture of three-component multivariate distributions</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/GENESIS.html">GENESIS</a></td>
            <td>Stephanie M. Gogarten </td>
            <td>GENetic EStimation and Inference in Structured samples (GENESIS): Statistical methods for analyzing genetic data from samples with population structure and/or relatedness</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/GeneStructureTools.html">GeneStructureTools</a></td>
            <td>Beth Signal </td>
            <td>Tools for spliced gene structure manipulation and analysis</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/geNetClassifier.html">geNetClassifier</a></td>
            <td>Sara Aibar </td>
            <td>Classify diseases and build associated gene networks using gene expression profiles</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/GeneticsDesign.html">GeneticsDesign</a></td>
            <td>The R Genetics Project </td>
            <td>Functions for designing genetics studies</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/GeneticsPed.html">GeneticsPed</a></td>
            <td>David Henderson </td>
            <td>Pedigree and genetic relationship functions</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/geneXtendeR.html">geneXtendeR</a></td>
            <td>Bohdan Khomtchouk </td>
            <td>Optimized Functional Annotation Of ChIP-seq Data</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/GENIE3.html">GENIE3</a></td>
            <td>Van Anh Huynh-Thu </td>
            <td>GEne Network Inference with Ensemble of trees</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/genoCN.html">genoCN</a></td>
            <td>Wei Sun </td>
            <td>genotyping and copy number study tools</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/GenoGAM.html">GenoGAM</a></td>
            <td>Georg Stricker </td>
            <td>A GAM based framework for analysis of ChIP-Seq data</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/genomation.html">genomation</a></td>
            <td>Altuna Akalin, Vedran Franke, Katarzyna Wreczycka </td>
            <td>Summary, annotation and visualization of genomic data</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/GenomeGraphs.html">GenomeGraphs</a></td>
            <td>Steffen Durinck </td>
            <td>Plotting genomic information from Ensembl</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/GenomeInfoDb.html">GenomeInfoDb</a></td>
            <td>Bioconductor Package Maintainer </td>
            <td>Utilities for manipulating chromosome names, including modifying them to follow a particular naming style</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/genomeIntervals.html">genomeIntervals</a></td>
            <td>Julien Gagneur </td>
            <td>Operations on genomic intervals</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/genomes.html">genomes</a></td>
            <td>Chris Stubben </td>
            <td>Genome sequencing project metadata</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/GenomicAlignments.html">GenomicAlignments</a></td>
            <td>Bioconductor Package Maintainer </td>
            <td>Representation and manipulation of short genomic alignments</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/GenomicDataCommons.html">GenomicDataCommons</a></td>
            <td>Sean Davis </td>
            <td>NIH / NCI Genomic Data Commons Access</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/GenomicFeatures.html">GenomicFeatures</a></td>
            <td>Bioconductor Package Maintainer </td>
            <td>Conveniently import and query gene models</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/GenomicFiles.html">GenomicFiles</a></td>
            <td>Bioconductor Package Maintainer </td>
            <td>Distributed computing by file or by range</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/GenomicInteractions.html">GenomicInteractions</a></td>
            <td>Liz Ing-Simmons </td>
            <td>R package for handling genomic interaction data</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/GenomicOZone.html">GenomicOZone</a></td>
            <td>Hua Zhong, Mingzhou Song </td>
            <td>Delineate outstanding genomic zones of differential gene activity</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/GenomicRanges.html">GenomicRanges</a></td>
            <td>Bioconductor Package Maintainer </td>
            <td>Representation and manipulation of genomic intervals</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/GenomicScores.html">GenomicScores</a></td>
            <td>Robert Castelo </td>
            <td>Infrastructure to work with genomewide position-specific scores</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/GenomicTuples.html">GenomicTuples</a></td>
            <td>Peter Hickey </td>
            <td>Representation and Manipulation of Genomic Tuples</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/Genominator.html">Genominator</a></td>
            <td>James Bullard </td>
            <td>Analyze, manage and store genomic data</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/genoset.html">genoset</a></td>
            <td>Peter M. Haverty </td>
            <td>A RangedSummarizedExperiment with methods for copy number analysis</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/genotypeeval.html">genotypeeval</a></td>
            <td>Jennifer Tom </td>
            <td>QA/QC of a gVCF or VCF file</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/genphen.html">genphen</a></td>
            <td>Simo Kitanovski </td>
            <td>A tool for quantification of associations between genotypes and phenotypes in genome wide association studies (GWAS) with Bayesian inference and statistical learning</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/GenRank.html">GenRank</a></td>
            <td>Chakravarthi Kanduri </td>
            <td>Candidate gene prioritization based on convergent evidence</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/GenVisR.html">GenVisR</a></td>
            <td>Zachary Skidmore </td>
            <td>Genomic Visualizations in R</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/GEOmetadb.html">GEOmetadb</a></td>
            <td>Jack Zhu </td>
            <td>A compilation of metadata from NCBI GEO</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/GEOquery.html">GEOquery</a></td>
            <td>Sean Davis </td>
            <td>Get data from NCBI Gene Expression Omnibus (GEO)</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/GEOsubmission.html">GEOsubmission</a></td>
            <td>Alexandre Kuhn </td>
            <td>Prepares microarray data for submission to GEO</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/gep2pep.html">gep2pep</a></td>
            <td>Francesco Napolitano </td>
            <td>Creation and Analysis of Pathway Expression Profiles (PEPs)</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/gespeR.html">gespeR</a></td>
            <td>Fabian Schmich </td>
            <td>Gene-Specific Phenotype EstimatoR</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/GEWIST.html">GEWIST</a></td>
            <td>Wei Q. Deng </td>
            <td>Gene Environment Wide Interaction Search Threshold</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/GGBase.html">GGBase</a></td>
            <td>VJ Carey </td>
            <td>GGBase infrastructure for genetics of gene expression package GGtools</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/ggbio.html">ggbio</a></td>
            <td>Michael Lawrence </td>
            <td>Visualization tools for genomic data</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/ggcyto.html">ggcyto</a></td>
            <td>Mike Jiang </td>
            <td>Visualize Cytometry data with ggplot</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/GGtools.html">GGtools</a></td>
            <td>VJ Carey </td>
            <td>software and data for analyses in genetics of gene expression</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/ggtree.html">ggtree</a></td>
            <td>Guangchuang Yu </td>
            <td>an R package for visualization of tree and annotation data</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/GIGSEA.html">GIGSEA</a></td>
            <td>Shijia Zhu </td>
            <td>Genotype Imputed Gene Set Enrichment Analysis</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/girafe.html">girafe</a></td>
            <td>J. Toedling </td>
            <td>Genome Intervals and Read Alignments for Functional Exploration</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/GISPA.html">GISPA</a></td>
            <td>Bhakti Dwivedi </td>
            <td>GISPA: Method for Gene Integrated Set Profile Analysis</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/GLAD.html">GLAD</a></td>
            <td>Philippe Hupe </td>
            <td>Gain and Loss Analysis of DNA</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/GladiaTOX.html">GladiaTOX</a></td>
            <td>PMP S.A. R Support </td>
            <td>R Package for Processing High Content Screening data</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/Glimma.html">Glimma</a></td>
            <td>Shian Su </td>
            <td>Interactive HTML graphics</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/glmSparseNet.html">glmSparseNet</a></td>
            <td>André Veríssimo </td>
            <td>Network Centrality Metrics for Elastic-Net Regularized Models</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/GlobalAncova.html">GlobalAncova</a></td>
            <td>Manuela Hummel </td>
            <td>Global test for groups of variables via model comparisons</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/globalSeq.html">globalSeq</a></td>
            <td>Armin Rauschenberger </td>
            <td>Global Test for Counts</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/globaltest.html">globaltest</a></td>
            <td>Jelle Goeman </td>
            <td>Testing Groups of Covariates/Features for Association with a Response Variable, with Applications to Gene Set Testing</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/gmapR.html">gmapR</a></td>
            <td>Michael Lawrence </td>
            <td>An R interface to the GMAP/GSNAP/GSTRUCT suite</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/GmicR.html">GmicR</a></td>
            <td>The package maintainer </td>
            <td>Combines WGCNA and xCell readouts with bayesian network learrning to generate a Gene-Module Immune-Cell network (GMIC)</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/GMRP.html">GMRP</a></td>
            <td>Yuan-De Tan </td>
            <td>GWAS-based Mendelian Randomization and Path Analyses</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/GNET2.html">GNET2</a></td>
            <td>Chen Chen </td>
            <td>Constructing gene regulatory networks from expression data through functional module inference</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/GOexpress.html">GOexpress</a></td>
            <td>Kevin Rue-Albrecht </td>
            <td>Visualise microarray and RNAseq data using gene ontology annotations</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/GOfuncR.html">GOfuncR</a></td>
            <td>Steffi Grote </td>
            <td>Gene ontology enrichment using FUNC</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/GOFunction.html">GOFunction</a></td>
            <td>Jing Wang </td>
            <td>GO-function: deriving biologcially relevant functions from statistically significant functions</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/GOpro.html">GOpro</a></td>
            <td>Lidia Chrabaszcz </td>
            <td>Find the most characteristic gene ontology terms for groups of human genes</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/goProfiles.html">goProfiles</a></td>
            <td>Alex Sanchez </td>
            <td>goProfiles: an R package for the statistical analysis of functional profiles</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/GOSemSim.html">GOSemSim</a></td>
            <td>Guangchuang Yu </td>
            <td>GO-terms Semantic Similarity Measures</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/goseq.html">goseq</a></td>
            <td>Matthew Young, Nadia Davidson </td>
            <td>Gene Ontology analyser for RNA-seq and other length biased data</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/GOSim.html">GOSim</a></td>
            <td>Holger Froehlich </td>
            <td>Computation of functional similarities between GO terms and gene products; GO enrichment analysis</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/goSTAG.html">goSTAG</a></td>
            <td>Brian D. Bennett </td>
            <td>A tool to use GO Subtrees to Tag and Annotate Genes within a set</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/GOstats.html">GOstats</a></td>
            <td>Bioconductor Package Maintainer </td>
            <td>Tools for manipulating GO and microarrays</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/GOsummaries.html">GOsummaries</a></td>
            <td>Raivo Kolde </td>
            <td>Word cloud summaries of GO enrichment analysis</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/GOTHiC.html">GOTHiC</a></td>
            <td>Borbala Mifsud </td>
            <td>Binomial test for Hi-C data analysis</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/goTools.html">goTools</a></td>
            <td>Agnes Paquet </td>
            <td>Functions for Gene Ontology database</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/gpart.html">gpart</a></td>
            <td>Sun Ah Kim </td>
            <td>Human genome partitioning of dense sequencing data by identifying haplotype blocks</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/gpls.html">gpls</a></td>
            <td>Bioconductor Package Maintainer </td>
            <td>Classification using generalized partial least squares</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/gprege.html">gprege</a></td>
            <td>Alfredo Kalaitzis </td>
            <td>Gaussian Process Ranking and Estimation of Gene Expression time-series</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/gpuMagic.html">gpuMagic</a></td>
            <td>Jiefei Wang </td>
            <td>An openCL compiler with the capacity to compile R functions and run the code on GPU</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/gQTLBase.html">gQTLBase</a></td>
            <td>VJ Carey </td>
            <td>gQTLBase: infrastructure for eQTL, mQTL and similar studies</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/gQTLstats.html">gQTLstats</a></td>
            <td>VJ Carey </td>
            <td>gQTLstats: computationally efficient analysis for eQTL and allied studies</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/gramm4R.html">gramm4R</a></td>
            <td>Tianlu Chen </td>
            <td>Generalized correlation analysis and model construction strategy for metabolome and microbiome</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/graper.html">graper</a></td>
            <td>Britta Velten </td>
            <td>Adaptive penalization in high-dimensional regression and classification with external covariates using variational Bayes</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/graph.html">graph</a></td>
            <td>Bioconductor Package Maintainer </td>
            <td>graph: A package to handle graph data structures</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/GraphAlignment.html">GraphAlignment</a></td>
            <td>Joern P. Meier </td>
            <td>GraphAlignment</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/GraphAT.html">GraphAT</a></td>
            <td>Thomas LaFramboise </td>
            <td>Graph Theoretic Association Tests</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/graphite.html">graphite</a></td>
            <td>Gabriele Sales </td>
            <td>GRAPH Interaction from pathway Topological Environment</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/GraphPAC.html">GraphPAC</a></td>
            <td>Gregory Ryslik </td>
            <td>Identification of Mutational Clusters in Proteins via a Graph Theoretical Approach.</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/GRENITS.html">GRENITS</a></td>
            <td>Edward Morrissey </td>
            <td>Gene Regulatory Network Inference Using Time Series</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/GreyListChIP.html">GreyListChIP</a></td>
            <td>Gordon Brown </td>
            <td>Grey Lists -- Mask Artefact Regions Based on ChIP Inputs</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/GRmetrics.html">GRmetrics</a></td>
            <td>Nicholas Clark, Mario Medvedovic </td>
            <td>Calculate growth-rate inhibition (GR) metrics</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/groHMM.html">groHMM</a></td>
            <td>Anusha Nagari, Tulip Nandu, W. Lee Kraus </td>
            <td>GRO-seq Analysis Pipeline</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/GRridge.html">GRridge</a></td>
            <td>Mark A. van de Wiel </td>
            <td>Better prediction by use of co-data: Adaptive group-regularized ridge regression</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/GSALightning.html">GSALightning</a></td>
            <td>Billy Heung Wing Chang </td>
            <td>Fast Permutation-based Gene Set Analysis</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/GSAR.html">GSAR</a></td>
            <td>Yasir Rahmatallah, Galina Glazko </td>
            <td>Gene Set Analysis in R</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/GSCA.html">GSCA</a></td>
            <td>Zhicheng Ji </td>
            <td>GSCA: Gene Set Context Analysis</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/gscreend.html">gscreend</a></td>
            <td>Katharina Imkeller </td>
            <td>Analysis of pooled genetic screens</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/GSEABase.html">GSEABase</a></td>
            <td>Bioconductor Package Maintainer </td>
            <td>Gene set enrichment data structures and methods</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/GSEABenchmarkeR.html">GSEABenchmarkeR</a></td>
            <td>Ludwig Geistlinger </td>
            <td>Reproducible GSEA Benchmarking</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/GSEAlm.html">GSEAlm</a></td>
            <td>Assaf Oron </td>
            <td>Linear Model Toolset for Gene Set Enrichment Analysis</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/gsean.html">gsean</a></td>
            <td>Dongmin Jung </td>
            <td>Gene Set Enrichment Analysis with Networks</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/GSReg.html">GSReg</a></td>
            <td>Bahman Afsari, Elana J. Fertig </td>
            <td>Gene Set Regulation (GS-Reg)</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/GSRI.html">GSRI</a></td>
            <td>Julian Gehring </td>
            <td>Gene Set Regulation Index</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/GSVA.html">GSVA</a></td>
            <td>Justin Guinney </td>
            <td>Gene Set Variation Analysis for microarray and RNA-seq data</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/gtrellis.html">gtrellis</a></td>
            <td>Zuguang Gu </td>
            <td>Genome Level Trellis Layout</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/GUIDEseq.html">GUIDEseq</a></td>
            <td>Lihua Julie Zhu </td>
            <td>GUIDE-seq analysis pipeline</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/Guitar.html">Guitar</a></td>
            <td>Jia Meng </td>
            <td>Guitar</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/Gviz.html">Gviz</a></td>
            <td>Robert Ivanek </td>
            <td>Plotting data and annotation information along genomic coordinates</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/gwascat.html">gwascat</a></td>
            <td>VJ Carey </td>
            <td>representing and modeling data in the EMBL-EBI GWAS catalog</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/GWASTools.html">GWASTools</a></td>
            <td>Stephanie M. Gogarten </td>
            <td>Tools for Genome Wide Association Studies</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/gwasurvivr.html">gwasurvivr</a></td>
            <td>Abbas Rizvi </td>
            <td>gwasurvivr: an R package for genome wide survival analysis</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/h5vc.html">h5vc</a></td>
            <td>Paul Theodor Pyl </td>
            <td>Managing alignment tallies using a hdf5 backend</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/hapFabia.html">hapFabia</a></td>
            <td>Andreas Mitterecker </td>
            <td>hapFabia: Identification of very short segments of identity by descent (IBD) characterized by rare variants in large sequencing data</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/Harman.html">Harman</a></td>
            <td>Jason Ross </td>
            <td>The removal of batch effects from datasets using a PCA and constrained optimisation based technique</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/Harshlight.html">Harshlight</a></td>
            <td>Maurizio Pellegrino </td>
            <td>A "corrective make-up" program for microarray chips</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/HCABrowser.html">HCABrowser</a></td>
            <td>Bioconductor Package Maintainer </td>
            <td>Browse the Human Cell Atlas data portal</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/HCAExplorer.html">HCAExplorer</a></td>
            <td>Bioconductor Package Maintainer </td>
            <td>Browse the Human Cell Atlas data portal</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/HDF5Array.html">HDF5Array</a></td>
            <td>Hervé Pagès </td>
            <td>HDF5 backend for DelayedArray objects</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/HDTD.html">HDTD</a></td>
            <td>Anestis Touloumis </td>
            <td>Statistical Inference about the Mean Matrix and the Covariance Matrices in High-Dimensional Transposable Data (HDTD)</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/heatmaps.html">heatmaps</a></td>
            <td>Malcolm Perry </td>
            <td>Flexible Heatmaps for Functional Genomics and Sequence Features</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/Heatplus.html">Heatplus</a></td>
            <td>Alexander Ploner </td>
            <td>Heatmaps with row and/or column covariates and colored clusters</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/HelloRanges.html">HelloRanges</a></td>
            <td>Michael Lawrence </td>
            <td>Introduce *Ranges to bedtools users</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/HELP.html">HELP</a></td>
            <td>Reid F. Thompson </td>
            <td>Tools for HELP data analysis</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/HEM.html">HEM</a></td>
            <td>HyungJun Cho </td>
            <td>Heterogeneous error model for identification of differentially expressed genes under multiple conditions</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/hiAnnotator.html">hiAnnotator</a></td>
            <td>Nirav V Malani </td>
            <td>Functions for annotating GRanges objects</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/HIBAG.html">HIBAG</a></td>
            <td>Xiuwen Zheng </td>
            <td>HLA Genotype Imputation with Attribute Bagging</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/HiCBricks.html">HiCBricks</a></td>
            <td>Koustav Pal </td>
            <td>Framework for Storing and Accessing Hi-C Data Through HDF Files</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/HiCcompare.html">HiCcompare</a></td>
            <td>John Stansfield </td>
            <td>HiCcompare: Joint normalization and comparative analysis of multiple Hi-C datasets</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/hicrep.html">hicrep</a></td>
            <td>Tao Yang </td>
            <td>Measuring the reproducibility of Hi-C data</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/hierGWAS.html">hierGWAS</a></td>
            <td>Laura Buzdugan </td>
            <td>Asessing statistical significance in predictive GWA studies</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/hierinf.html">hierinf</a></td>
            <td>Claude Renaux </td>
            <td>Hierarchical Inference</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/HilbertCurve.html">HilbertCurve</a></td>
            <td>Zuguang Gu </td>
            <td>Making 2D Hilbert Curve</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/HilbertVis.html">HilbertVis</a></td>
            <td>Simon Anders </td>
            <td>Hilbert curve visualization</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/HilbertVisGUI.html">HilbertVisGUI</a></td>
            <td>Simon Anders </td>
            <td>HilbertVisGUI</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/HiLDA.html">HiLDA</a></td>
            <td>Zhi Yang </td>
            <td>Conducting statistical inference on comparing the mutational exposures of mutational signatures by using hierarchical latent Dirichlet allocation</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/hipathia.html">hipathia</a></td>
            <td>Marta R. Hidalgo </td>
            <td>HiPathia: High-throughput Pathway Analysis</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/hiReadsProcessor.html">hiReadsProcessor</a></td>
            <td>Nirav V Malani </td>
            <td>Functions to process LM-PCR reads from 454/Illumina data</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/HIREewas.html">HIREewas</a></td>
            <td>Xiangyu Luo </td>
            <td>Detection of cell-type-specific risk-CpG sites in epigenome-wide association studies</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/HiTC.html">HiTC</a></td>
            <td>Nicolas Servant </td>
            <td>High Throughput Chromosome Conformation Capture analysis</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/hmdbQuery.html">hmdbQuery</a></td>
            <td>VJ Carey </td>
            <td>utilities for exploration of human metabolome database</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/HMMcopy.html">HMMcopy</a></td>
            <td>Daniel Lai, Sohrab Shah </td>
            <td>Copy number prediction with correction for GC and mappability bias for HTS data</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/hopach.html">hopach</a></td>
            <td>Katherine S. Pollard </td>
            <td>Hierarchical Ordered Partitioning and Collapsing Hybrid (HOPACH)</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/HPAanalyze.html">HPAanalyze</a></td>
            <td>Anh Nhat Tran </td>
            <td>Retrieve and analyze data from the Human Protein Atlas</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/hpar.html">hpar</a></td>
            <td>Laurent Gatto </td>
            <td>Human Protein Atlas in R</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/HTqPCR.html">HTqPCR</a></td>
            <td>Heidi Dvinge </td>
            <td>Automated analysis of high-throughput qPCR data</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/HTSanalyzeR.html">HTSanalyzeR</a></td>
            <td>Xin Wang </td>
            <td>Gene set over-representation, enrichment and network analyses for high-throughput screens</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/HTSeqGenie.html">HTSeqGenie</a></td>
            <td>Jens Reeder </td>
            <td>A NGS analysis pipeline.</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/HTSFilter.html">HTSFilter</a></td>
            <td>Andrea Rau </td>
            <td>Filter replicated high-throughput transcriptome sequencing data</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/HumanTranscriptomeCompendium.html">HumanTranscriptomeCompendium</a></td>
            <td>VJ Carey </td>
            <td>Tools to work with a Compendium of 181000 human transcriptome sequencing studies</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/HybridMTest.html">HybridMTest</a></td>
            <td>Demba Fofana </td>
            <td>Hybrid Multiple Testing</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/hypeR.html">hypeR</a></td>
            <td>Anthony Federico </td>
            <td>Hyper Enrichment</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/hyperdraw.html">hyperdraw</a></td>
            <td>Paul Murrell </td>
            <td>Visualizing Hypergaphs</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/hypergraph.html">hypergraph</a></td>
            <td>Bioconductor Package Maintainer </td>
            <td>A package providing hypergraph data structures</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/iASeq.html">iASeq</a></td>
            <td>Yingying Wei </td>
            <td>iASeq: integrating multiple sequencing datasets for detecting allele-specific events</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/iasva.html">iasva</a></td>
            <td>Donghyung Lee, Anthony Cheng </td>
            <td>Iteratively Adjusted Surrogate Variable Analysis</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/iBBiG.html">iBBiG</a></td>
            <td>Aedin Culhane </td>
            <td>Iterative Binary Biclustering of Genesets</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/ibh.html">ibh</a></td>
            <td>Kircicegi Korkmaz </td>
            <td>Interaction Based Homogeneity for Evaluating Gene Lists</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/iBMQ.html">iBMQ</a></td>
            <td>Greg Imholte </td>
            <td>integrated Bayesian Modeling of eQTL data</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/iCARE.html">iCARE</a></td>
            <td>Bill Wheeler </td>
            <td>A Tool for Individualized Coherent Absolute Risk Estimation (iCARE)</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/Icens.html">Icens</a></td>
            <td>Bioconductor Package Maintainer </td>
            <td>NPMLE for Censored and Truncated Data</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/icetea.html">icetea</a></td>
            <td>Vivek Bhardwaj </td>
            <td>Integrating Cap Enrichment with Transcript Expression Analysis</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/iCheck.html">iCheck</a></td>
            <td>Weiliang Qiu </td>
            <td>QC Pipeline and Data Analysis Tools for High-Dimensional Illumina mRNA Expression Data</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/iChip.html">iChip</a></td>
            <td>Qianxing Mo </td>
            <td>Bayesian Modeling of ChIP-chip Data Through Hidden Ising Models</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/iClusterPlus.html">iClusterPlus</a></td>
            <td>Qianxing Mo, Ronglai Shen </td>
            <td>Integrative clustering of multi-type genomic data</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/iCNV.html">iCNV</a></td>
            <td>Zilu Zhou </td>
            <td>Integrated Copy Number Variation detection</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/iCOBRA.html">iCOBRA</a></td>
            <td>Charlotte Soneson </td>
            <td>Comparison and Visualization of Ranking and Assignment Methods</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/ideal.html">ideal</a></td>
            <td>Federico Marini </td>
            <td>Interactive Differential Expression AnaLysis</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/IdeoViz.html">IdeoViz</a></td>
            <td>Shraddha Pai </td>
            <td>Plots data (continuous/discrete) along chromosomal ideogram</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/idiogram.html">idiogram</a></td>
            <td>Karl J. Dykema </td>
            <td>idiogram</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/IdMappingAnalysis.html">IdMappingAnalysis</a></td>
            <td>Alex Lisovich, Roger Day </td>
            <td>ID Mapping Analysis</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/IdMappingRetrieval.html">IdMappingRetrieval</a></td>
            <td>Alex Lisovich, Roger Day </td>
            <td>ID Mapping Data Retrieval</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/idr2d.html">idr2d</a></td>
            <td>Konstantin Krismer </td>
            <td>Irreproducible Discovery Rate for Genomic Interactions Data</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/iGC.html">iGC</a></td>
            <td>Liang-Bo Wang </td>
            <td>An integrated analysis package of Gene expression and Copy number alteration</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/IgGeneUsage.html">IgGeneUsage</a></td>
            <td>Simo Kitanovski </td>
            <td>Differential gene usage in immune repertoires</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/igvR.html">igvR</a></td>
            <td>Paul Shannon </td>
            <td>igvR: integrative genomics viewer</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/IHW.html">IHW</a></td>
            <td>Nikos Ignatiadis </td>
            <td>Independent Hypothesis Weighting</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/illuminaio.html">illuminaio</a></td>
            <td>Kasper Daniel Hansen </td>
            <td>Parsing Illumina Microarray Output Files</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/imageHTS.html">imageHTS</a></td>
            <td>Joseph Barry </td>
            <td>Analysis of high-throughput microscopy-based screens</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/IMAS.html">IMAS</a></td>
            <td>Seonggyun Han </td>
            <td>Integrative analysis of Multi-omics data for Alternative Splicing</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/Imetagene.html">Imetagene</a></td>
            <td>Audrey Lemacon </td>
            <td>A graphical interface for the metagene package</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/IMMAN.html">IMMAN</a></td>
            <td>Minoo Ashtiani </td>
            <td>Interlog protein network reconstruction by Mapping and Mining ANalysis</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/ImmuneSpaceR.html">ImmuneSpaceR</a></td>
            <td>ImmuneSpace Package Maintainer </td>
            <td>A Thin Wrapper around the ImmuneSpace Database</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/immunoClust.html">immunoClust</a></td>
            <td>Till Soerensen </td>
            <td>immunoClust - Automated Pipeline for Population Detection in Flow Cytometry</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/IMPCdata.html">IMPCdata</a></td>
            <td>Jeremy Mason </td>
            <td>Retrieves data from IMPC database</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/ImpulseDE.html">ImpulseDE</a></td>
            <td>Jil Sander, Nir Yosef </td>
            <td>Detection of DE genes in time series data using impulse models</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/ImpulseDE2.html">ImpulseDE2</a></td>
            <td>David S Fischer </td>
            <td>Differential expression analysis of longitudinal count data sets</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/impute.html">impute</a></td>
            <td>Balasubramanian Narasimhan </td>
            <td>impute: Imputation for microarray data</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/INDEED.html">INDEED</a></td>
            <td>Ressom group, Yiming Zuo </td>
            <td>Interactive Visualization of Integrated Differential Expression and Differential Network Analysis for Biomarker Candidate Selection Package</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/infercnv.html">infercnv</a></td>
            <td>Christophe Georgescu </td>
            <td>Infer Copy Number Variation from Single-Cell RNA-Seq Data</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/InPAS.html">InPAS</a></td>
            <td>Jianhong Ou, Lihua Julie Zhu </td>
            <td>InPAS: a bioconductor package for the identification of novel alternative PolyAdenylation Sites (PAS) using RNA-seq data</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/INPower.html">INPower</a></td>
            <td>Bill Wheeler </td>
            <td>An R package for computing the number of susceptibility SNPs</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/INSPEcT.html">INSPEcT</a></td>
            <td>Stefano de Pretis, Mattia Furlan </td>
            <td>Modeling RNA synthesis, processing and degradation with RNA-seq data</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/InTAD.html">InTAD</a></td>
            <td>Konstantin Okonechnikov </td>
            <td>Search for correlation between epigenetic signals and gene expression in TADs</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/intansv.html">intansv</a></td>
            <td>Wen Yao </td>
            <td>Integrative analysis of structural variations</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/InteractionSet.html">InteractionSet</a></td>
            <td>Aaron Lun </td>
            <td>Base Classes for Storing Genomic Interaction Data</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/interactiveDisplay.html">interactiveDisplay</a></td>
            <td>Shawn Balcome </td>
            <td>Package for enabling powerful shiny web displays of Bioconductor objects</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/interactiveDisplayBase.html">interactiveDisplayBase</a></td>
            <td>Shawn Balcome </td>
            <td>Base package for enabling powerful shiny web displays of Bioconductor objects</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/IntEREst.html">IntEREst</a></td>
            <td>Ali Oghabian, Mikko Frilander </td>
            <td>Intron-Exon Retention Estimator</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/InterMineR.html">InterMineR</a></td>
            <td>InterMine Team </td>
            <td>R Interface with InterMine-Powered Databases</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/IntramiRExploreR.html">IntramiRExploreR</a></td>
            <td>Surajit Bhattacharya </td>
            <td>Predicting Targets for Drosophila Intragenic miRNAs</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/inveRsion.html">inveRsion</a></td>
            <td>Alejandro Caceres </td>
            <td>Inversions in genotype data</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/IONiseR.html">IONiseR</a></td>
            <td>Mike Smith </td>
            <td>Quality Assessment Tools for Oxford Nanopore MinION data</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/iPAC.html">iPAC</a></td>
            <td>Gregory Ryslik </td>
            <td>Identification of Protein Amino acid Clustering</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/ipdDb.html">ipdDb</a></td>
            <td>Steffen Klasberg </td>
            <td>IPD IMGT/HLA and IPD KIR database for Homo sapiens</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/IPO.html">IPO</a></td>
            <td>Thomas Riebenbauer </td>
            <td>Automated Optimization of XCMS Data Processing parameters</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/IPPD.html">IPPD</a></td>
            <td>Martin Slawski </td>
            <td>Isotopic peak pattern deconvolution for Protein Mass Spectrometry by template matching</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/IRanges.html">IRanges</a></td>
            <td>Bioconductor Package Maintainer </td>
            <td>Foundation of integer range manipulation in Bioconductor</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/iSEE.html">iSEE</a></td>
            <td>Charlotte Soneson </td>
            <td>Interactive SummarizedExperiment Explorer</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/iSeq.html">iSeq</a></td>
            <td>Qianxing Mo </td>
            <td>Bayesian Hierarchical Modeling of ChIP-seq Data Through Hidden Ising Models</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/isobar.html">isobar</a></td>
            <td>Florian P Breitwieser </td>
            <td>Analysis and quantitation of isobarically tagged MSMS proteomics data</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/IsoCorrectoR.html">IsoCorrectoR</a></td>
            <td>Christian Kohler </td>
            <td>Correction for natural isotope abundance and tracer purity in MS and MS/MS data from stable isotope labeling experiments</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/IsoCorrectoRGUI.html">IsoCorrectoRGUI</a></td>
            <td>Christian Kohler </td>
            <td>Graphical User Interface for IsoCorrectoR</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/IsoformSwitchAnalyzeR.html">IsoformSwitchAnalyzeR</a></td>
            <td>Kristoffer Vitting-Seerup </td>
            <td>Identify, Annotate and Visualize Alternative Splicing and Isoform Switches with Functional Consequences from both short- and long-read RNA-seq data.</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/IsoGeneGUI.html">IsoGeneGUI</a></td>
            <td>Setia Pramana </td>
            <td>A graphical user interface to conduct a dose-response analysis of microarray data</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/ISoLDE.html">ISoLDE</a></td>
            <td>Christelle Reynès </td>
            <td>Integrative Statistics of alleLe Dependent Expression</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/isomiRs.html">isomiRs</a></td>
            <td>Lorena Pantano </td>
            <td>Analyze isomiRs and miRNAs from small RNA-seq</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/ITALICS.html">ITALICS</a></td>
            <td>Guillem Rigaill </td>
            <td>ITALICS</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/iterativeBMA.html">iterativeBMA</a></td>
            <td>Ka Yee Yeung </td>
            <td>The Iterative Bayesian Model Averaging (BMA) algorithm</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/iterativeBMAsurv.html">iterativeBMAsurv</a></td>
            <td>Ka Yee Yeung </td>
            <td>The Iterative Bayesian Model Averaging (BMA) Algorithm For Survival Analysis</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/iterClust.html">iterClust</a></td>
            <td>Hongxu Ding </td>
            <td>Iterative Clustering</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/iteremoval.html">iteremoval</a></td>
            <td>Jiacheng Chuan </td>
            <td>Iteration removal method for feature selection</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/IVAS.html">IVAS</a></td>
            <td>Seonggyun Han </td>
            <td>Identification of genetic Variants affecting Alternative Splicing</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/ivygapSE.html">ivygapSE</a></td>
            <td>VJ Carey </td>
            <td>A SummarizedExperiment for Ivy-GAP data</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/IWTomics.html">IWTomics</a></td>
            <td>Marzia A Cremona </td>
            <td>Interval-Wise Testing for Omics Data</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/joda.html">joda</a></td>
            <td>Ewa Szczurek </td>
            <td>JODA algorithm for quantifying gene deregulation using knowledge</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/JunctionSeq.html">JunctionSeq</a></td>
            <td>Stephen Hartley </td>
            <td>JunctionSeq: A Utility for Detection of Differential Exon and Splice-Junction Usage in RNA-Seq data</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/karyoploteR.html">karyoploteR</a></td>
            <td>Bernat Gel </td>
            <td>Plot customizable linear genomes displaying arbitrary data</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/KCsmart.html">KCsmart</a></td>
            <td>Jorma de Ronde </td>
            <td>Multi sample aCGH analysis package using kernel convolution</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/kebabs.html">kebabs</a></td>
            <td>Ulrich Bodenhofer </td>
            <td>Kernel-Based Analysis Of Biological Sequences</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/KEGGgraph.html">KEGGgraph</a></td>
            <td>Jitao David Zhang </td>
            <td>KEGGgraph: A graph approach to KEGG PATHWAY in R and Bioconductor</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/KEGGlincs.html">KEGGlincs</a></td>
            <td>Shana White, Mario Medvedovic </td>
            <td>Visualize all edges within a KEGG pathway and overlay LINCS data</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/keggorthology.html">keggorthology</a></td>
            <td>VJ Carey </td>
            <td>graph support for KO, KEGG Orthology</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/KEGGprofile.html">KEGGprofile</a></td>
            <td>Shilin Zhao </td>
            <td>An annotation and visualization package for multi-types and multi-groups expression data in KEGG pathway</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/KEGGREST.html">KEGGREST</a></td>
            <td>Bioconductor Package Maintainer </td>
            <td>Client-side REST access to KEGG</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/kimod.html">kimod</a></td>
            <td>M L Zingaretti </td>
            <td>A k-tables approach to integrate multiple Omics-Data</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/KinSwingR.html">KinSwingR</a></td>
            <td>Ashley J. Waardenberg </td>
            <td>KinSwingR: network-based kinase activity prediction</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/kissDE.html">kissDE</a></td>
            <td>Aurélie Siberchicot </td>
            <td>Retrieves Condition-Specific Variants in RNA-Seq Data</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/KnowSeq.html">KnowSeq</a></td>
            <td>Daniel Castillo Secilla </td>
            <td>A R package to extract knowledge by using RNA-seq raw files</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/lapmix.html">lapmix</a></td>
            <td>Yann Ruffieux </td>
            <td>Laplace Mixture Model in Microarray Experiments</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/LBE.html">LBE</a></td>
            <td>Cyril Dalmasso </td>
            <td>Estimation of the false discovery rate.</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/ldblock.html">ldblock</a></td>
            <td>VJ Carey </td>
            <td>data structures for linkage disequilibrium measures in populations</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/LEA.html">LEA</a></td>
            <td>Eric Frichot, Olivier Francois </td>
            <td>LEA: an R package for Landscape and Ecological Association Studies</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/LedPred.html">LedPred</a></td>
            <td>Aitor Gonzalez </td>
            <td>Learning from DNA to Predict Enhancers</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/les.html">les</a></td>
            <td>Julian Gehring </td>
            <td>Identifying Differential Effects in Tiling Microarray Data</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/levi.html">levi</a></td>
            <td>Jose Luiz Rybarczyk Filho </td>
            <td>Landscape Expression Visualization Interface</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/lfa.html">lfa</a></td>
            <td>Wei Hao, John D. Storey </td>
            <td>Logistic Factor Analysis for Categorical Data</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/limma.html">limma</a></td>
            <td>Gordon Smyth </td>
            <td>Linear Models for Microarray Data</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/limmaGUI.html">limmaGUI</a></td>
            <td>Gordon Smyth </td>
            <td>GUI for limma Package With Two Color Microarrays</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/LINC.html">LINC</a></td>
            <td>Manuel Goepferich </td>
            <td>co-expression of lincRNAs and protein-coding genes</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/LineagePulse.html">LineagePulse</a></td>
            <td>David S Fischer </td>
            <td>Differential expression analysis and model fitting for single-cell RNA-seq data</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/LinkHD.html">LinkHD</a></td>
            <td>"Laura M Zingaretti" </td>
            <td>LinkHD: a versatile framework to explore and integrate heterogeneous data</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/Linnorm.html">Linnorm</a></td>
            <td>Ken Shun Hang Yip </td>
            <td>Linear model and normality based normalization and transformation method (Linnorm)</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/lionessR.html">lionessR</a></td>
            <td>Ping-Han Hsieh </td>
            <td>Modeling networks for individual samples using LIONESS</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/lipidr.html">lipidr</a></td>
            <td>Ahmed Mohamed </td>
            <td>Data Mining and Analysis of Lipidomics Datasets</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/LiquidAssociation.html">LiquidAssociation</a></td>
            <td>Yen-Yi Ho </td>
            <td>LiquidAssociation</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/lmdme.html">lmdme</a></td>
            <td>Cristobal Fresno </td>
            <td>Linear Model decomposition for Designed Multivariate Experiments</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/LMGene.html">LMGene</a></td>
            <td>Blythe Durbin-Johnson </td>
            <td>LMGene Software for Data Transformation and Identification of Differentially Expressed Genes in Gene Expression Arrays</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/LOBSTAHS.html">LOBSTAHS</a></td>
            <td>James Collins </td>
            <td>Lipid and Oxylipin Biomarker Screening through Adduct Hierarchy Sequences</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/loci2path.html">loci2path</a></td>
            <td>Tianlei Xu </td>
            <td>Loci2path: regulatory annotation of genomic intervals based on tissue-specific expression QTLs</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/logicFS.html">logicFS</a></td>
            <td>Holger Schwender </td>
            <td>Identification of SNP Interactions</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/logitT.html">logitT</a></td>
            <td>Tobias Guennel </td>
            <td>logit-t Package</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/Logolas.html">Logolas</a></td>
            <td>Kushal Dey </td>
            <td>EDLogo Plots Featuring String Logos and Adaptive Scaling of Position-Weight Matrices</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/lol.html">lol</a></td>
            <td>Yinyin Yuan </td>
            <td>Lots Of Lasso</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/LOLA.html">LOLA</a></td>
            <td>Nathan Sheffield </td>
            <td>Locus overlap analysis for enrichment of genomic ranges</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/LoomExperiment.html">LoomExperiment</a></td>
            <td>Bioconductor Package Maintainer </td>
            <td>LoomExperiment container</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/LowMACA.html">LowMACA</a></td>
            <td>Stefano de Pretis, Giorgio Melloni </td>
            <td>LowMACA - Low frequency Mutation Analysis via Consensus Alignment</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/LPE.html">LPE</a></td>
            <td>Nitin Jain </td>
            <td>Methods for analyzing microarray data using Local Pooled Error (LPE) method</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/LPEadj.html">LPEadj</a></td>
            <td>Carl Murie </td>
            <td>A correction of the local pooled error (LPE) method to replace the asymptotic variance adjustment with an unbiased adjustment based on sample size.</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/lpNet.html">lpNet</a></td>
            <td>Lars Kaderali </td>
            <td>Linear Programming Model for Network Inference</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/lpsymphony.html">lpsymphony</a></td>
            <td>Vladislav Kim </td>
            <td>Symphony integer linear programming solver in R</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/LRBaseDbi.html">LRBaseDbi</a></td>
            <td>Koki Tsuyuzaki </td>
            <td>DBI to construct LRBase-related package</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/lumi.html">lumi</a></td>
            <td>Pan Du, Lei Huang, Gang Feng </td>
            <td>BeadArray Specific Methods for Illumina Methylation and Expression Microarrays</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/LVSmiRNA.html">LVSmiRNA</a></td>
            <td>Stefano Calza </td>
            <td>LVS normalization for Agilent miRNA data</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/LymphoSeq.html">LymphoSeq</a></td>
            <td>David Coffey </td>
            <td>Analyze high-throughput sequencing of T and B cell receptors</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/M3C.html">M3C</a></td>
            <td>Christopher John </td>
            <td>Monte Carlo Reference-based Consensus Clustering</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/M3D.html">M3D</a></td>
            <td>Tom Mayo </td>
            <td>Identifies differentially methylated regions across testing groups</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/M3Drop.html">M3Drop</a></td>
            <td>Tallulah Andrews </td>
            <td>Michaelis-Menten Modelling of Dropouts in single-cell RNASeq</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/maanova.html">maanova</a></td>
            <td>Keith Sheppard </td>
            <td>Tools for analyzing Micro Array experiments</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/Maaslin2.html">Maaslin2</a></td>
            <td>Lauren McIver </td>
            <td>Maaslin2</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/macat.html">macat</a></td>
            <td>Joern Toedling </td>
            <td>MicroArray Chromosome Analysis Tool</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/maCorrPlot.html">maCorrPlot</a></td>
            <td>Alexander Ploner </td>
            <td>Visualize artificial correlation in microarray data</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/MACPET.html">MACPET</a></td>
            <td>Ioannis Vardaxis </td>
            <td>Model based analysis for paired-end data</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/MACSQuantifyR.html">MACSQuantifyR</a></td>
            <td>Raphaël Bonnet </td>
            <td>Fast treatment of MACSQuantify FACS data</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/made4.html">made4</a></td>
            <td>Aedin Culhane </td>
            <td>Multivariate analysis of microarray data using ADE4</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/MADSEQ.html">MADSEQ</a></td>
            <td>Yu Kong </td>
            <td>Mosaic Aneuploidy Detection and Quantification using Massive Parallel Sequencing Data</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/maftools.html">maftools</a></td>
            <td>Anand Mayakonda </td>
            <td>Summarize, Analyze and Visualize MAF Files</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/MAGeCKFlute.html">MAGeCKFlute</a></td>
            <td>Wubing Zhang</td>
            <td>Integrative analysis pipeline for pooled CRISPR functional genetic screens</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/maigesPack.html">maigesPack</a></td>
            <td>Gustavo H. Esteves </td>
            <td>Functions to handle cDNA microarray data, including several methods of data analysis</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/MAIT.html">MAIT</a></td>
            <td>Pol Sola-Santos </td>
            <td>Statistical Analysis of Metabolomic Data</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/makecdfenv.html">makecdfenv</a></td>
            <td>James W. MacDonald </td>
            <td>CDF Environment Maker</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/MANOR.html">MANOR</a></td>
            <td>Pierre Neuvial </td>
            <td>CGH Micro-Array NORmalization</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/manta.html">manta</a></td>
            <td>Chris Berthiaume, Adrian Marchetti </td>
            <td>Microbial Assemblage Normalized Transcript Analysis</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/MantelCorr.html">MantelCorr</a></td>
            <td>Brian Steinmeyer </td>
            <td>Compute Mantel Cluster Correlations</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/mAPKL.html">mAPKL</a></td>
            <td>Argiris Sakellariou </td>
            <td>A Hybrid Feature Selection method for gene expression data</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/maPredictDSC.html">maPredictDSC</a></td>
            <td>Adi Laurentiu Tarca </td>
            <td>Phenotype prediction using microarray data: approach of the best overall team in the IMPROVER Diagnostic Signature Challenge</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/mapscape.html">mapscape</a></td>
            <td>Maia Smith </td>
            <td>mapscape</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/marray.html">marray</a></td>
            <td>Yee Hwa (Jean) Yang </td>
            <td>Exploratory analysis for two-color spotted microarray data</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/martini.html">martini</a></td>
            <td>Hector Climente-Gonzalez </td>
            <td>GWAS Incorporating Networks</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/maser.html">maser</a></td>
            <td>Diogo F.T. Veiga </td>
            <td>Mapping Alternative Splicing Events to pRoteins</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/maSigPro.html">maSigPro</a></td>
            <td>Maria Jose Nueda </td>
            <td>Significant Gene Expression Profile Differences in Time Course Gene Expression Data</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/maskBAD.html">maskBAD</a></td>
            <td>Michael Dannemann </td>
            <td>Masking probes with binding affinity differences</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/MassArray.html">MassArray</a></td>
            <td>Reid F. Thompson </td>
            <td>Analytical Tools for MassArray Data</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/massiR.html">massiR</a></td>
            <td>Sam Buckberry </td>
            <td>massiR: MicroArray Sample Sex Identifier</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/MassSpecWavelet.html">MassSpecWavelet</a></td>
            <td>Pan Du </td>
            <td>Mass spectrum processing by wavelet-based algorithms</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/MAST.html">MAST</a></td>
            <td>Andrew McDavid </td>
            <td>Model-based Analysis of Single Cell Transcriptomics</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/matchBox.html">matchBox</a></td>
            <td>Luigi Marchionni, Anuj Gupta </td>
            <td>Utilities to compute, compare, and plot the agreement between ordered vectors of features (ie. distinct genomic experiments). The package includes Correspondence-At-the-TOP (CAT) analysis.</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/MatrixRider.html">MatrixRider</a></td>
            <td>Elena Grassi </td>
            <td>Obtain total affinity and occupancies for binding site matrices on a given sequence</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/matter.html">matter</a></td>
            <td>Kylie A. Bemis </td>
            <td>A framework for rapid prototyping with file-based data structures</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/MaxContrastProjection.html">MaxContrastProjection</a></td>
            <td>Jan Sauer </td>
            <td>Perform a maximum contrast projection of 3D images along the z-dimension into 2D</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/MBAmethyl.html">MBAmethyl</a></td>
            <td>Tao Wang </td>
            <td>Model-based analysis of DNA methylation data</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/MBASED.html">MBASED</a></td>
            <td>Oleg Mayba </td>
            <td>Package containing functions for ASE analysis using Meta-analysis Based Allele-Specific Expression Detection</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/MBCB.html">MBCB</a></td>
            <td>Jeff Allen </td>
            <td>MBCB (Model-based Background Correction for Beadarray)</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/mbkmeans.html">mbkmeans</a></td>
            <td>Davide Risso </td>
            <td>Mini-batch K-means Clustering for Single-Cell RNA-seq</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/mBPCR.html">mBPCR</a></td>
            <td>P.M.V. Rancoita </td>
            <td>Bayesian Piecewise Constant Regression for DNA copy number estimation</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/MBQN.html">MBQN</a></td>
            <td>Ariane Schad </td>
            <td>Mean/Median-balanced quantile normalization</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/MBttest.html">MBttest</a></td>
            <td>Yuan-De Tan </td>
            <td>Multiple Beta t-Tests</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/mcaGUI.html">mcaGUI</a></td>
            <td>Wade K. Copeland </td>
            <td>Microbial Community Analysis GUI</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/MCbiclust.html">MCbiclust</a></td>
            <td>Robert Bentham </td>
            <td>Massive correlating biclusters for gene expression data and associated methods</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/MCRestimate.html">MCRestimate</a></td>
            <td>Marc Johannes </td>
            <td>Misclassification error estimation with cross-validation</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/mCSEA.html">mCSEA</a></td>
            <td>Jordi Martorell-Marugán </td>
            <td>Methylated CpGs Set Enrichment Analysis</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/mdgsa.html">mdgsa</a></td>
            <td>David Montaner </td>
            <td>Multi Dimensional Gene Set Analysis.</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/mdp.html">mdp</a></td>
            <td>Helder Nakaya </td>
            <td>Molecular Degree of Perturbation calculates scores for transcriptome data samples based on their perturbation from controls</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/mdqc.html">mdqc</a></td>
            <td>Gabriela Cohen-Freue </td>
            <td>Mahalanobis Distance Quality Control for microarrays</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/MDTS.html">MDTS</a></td>
            <td>Jack M.. Fu </td>
            <td>Detection of de novo deletion in targeted sequencing trios</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/MEAL.html">MEAL</a></td>
            <td>Carlos Ruiz-Arenas </td>
            <td>Perform methylation analysis</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/MeasurementError.cor.html">MeasurementError.cor</a></td>
            <td>Beiying Ding </td>
            <td>Measurement Error model estimate for correlation coefficient</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/MEB.html">MEB</a></td>
            <td>Jiadi Zhu, Yan Zhou </td>
            <td>A Minimum Enclosing Ball (MEB) method to detect differential expression genes for RNA-seq data</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/MEDIPS.html">MEDIPS</a></td>
            <td>Lukas Chavez </td>
            <td>DNA IP-seq data analysis</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/MEDME.html">MEDME</a></td>
            <td>Mattia Pelizzola </td>
            <td>Modelling Experimental Data from MeDIP Enrichment</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/MEIGOR.html">MEIGOR</a></td>
            <td>Jose Egea </td>
            <td>MEIGO - MEtaheuristics for bIoinformatics Global Optimization</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/Melissa.html">Melissa</a></td>
            <td>C. A. Kapourani </td>
            <td>Bayesian clustering and imputationa of single cell methylomes</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/MergeMaid.html">MergeMaid</a></td>
            <td>Xiaogang Zhong </td>
            <td>Merge Maid</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/Mergeomics.html">Mergeomics</a></td>
            <td>Zeyneb Kurt </td>
            <td>Integrative network analysis of omics data</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/MeSHDbi.html">MeSHDbi</a></td>
            <td>Koki Tsuyuzaki </td>
            <td>DBI to construct MeSH-related package from sqlite file</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/meshes.html">meshes</a></td>
            <td>Guangchuang Yu </td>
            <td>MeSH Enrichment and Semantic analyses</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/meshr.html">meshr</a></td>
            <td>Koki Tsuyuzaki </td>
            <td>Tools for conducting enrichment analysis of MeSH</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/messina.html">messina</a></td>
            <td>Mark Pinese </td>
            <td>Single-gene classifiers and outlier-resistant detection of differential expression for two-group and survival problems.</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/metaArray.html">metaArray</a></td>
            <td>Hyungwon Choi </td>
            <td>Integration of Microarray Data for Meta-analysis</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/Metab.html">Metab</a></td>
            <td>Raphael Aggio </td>
            <td>Metab: An R Package for a High-Throughput Analysis of Metabolomics Data Generated by GC-MS.</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/metabomxtr.html">metabomxtr</a></td>
            <td>Michael Nodzenski </td>
            <td>A package to run mixture models for truncated metabolomics data with normal or lognormal distributions</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/MetaboSignal.html">MetaboSignal</a></td>
            <td>Andrea Rodriguez-Martinez, Rafael Ayala </td>
            <td>MetaboSignal: a network-based approach to overlay and explore metabolic and signaling KEGG pathways</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/metaCCA.html">metaCCA</a></td>
            <td>Anna Cichonska </td>
            <td>Summary Statistics-Based Multivariate Meta-Analysis of Genome-Wide Association Studies Using Canonical Correlation Analysis</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/MetaCyto.html">MetaCyto</a></td>
            <td>Zicheng Hu </td>
            <td>MetaCyto: A package for meta-analysis of cytometry data</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/metagene.html">metagene</a></td>
            <td>Charles Joly Beauparlant </td>
            <td>A package to produce metagene plots</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/metagene2.html">metagene2</a></td>
            <td>Eric Fournier </td>
            <td>A package to produce metagene plots</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/metagenomeFeatures.html">metagenomeFeatures</a></td>
            <td>Nathan D. Olson </td>
            <td>Exploration of marker-gene sequence taxonomic annotations</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/metagenomeSeq.html">metagenomeSeq</a></td>
            <td>Joseph N. Paulson </td>
            <td>Statistical analysis for sparse high-throughput sequencing</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/metahdep.html">metahdep</a></td>
            <td>John R. Stevens </td>
            <td>Hierarchical Dependence in Meta-Analysis</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/metaMS.html">metaMS</a></td>
            <td>Yann Guitton </td>
            <td>MS-based metabolomics annotation pipeline</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/MetaNeighbor.html">MetaNeighbor</a></td>
            <td>Manthan Shah </td>
            <td>Single cell replicability analysis</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/metaSeq.html">metaSeq</a></td>
            <td>Koki Tsuyuzaki </td>
            <td>Meta-analysis of RNA-Seq count data in multiple studies</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/metaseqR.html">metaseqR</a></td>
            <td>Panagiotis Moulos </td>
            <td>An R package for the analysis and result reporting of RNA-Seq data by combining multiple statistical algorithms.</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/metavizr.html">metavizr</a></td>
            <td>Hector Corrada Bravo </td>
            <td>R Interface to the metaviz web app for interactive metagenomics data analysis and visualization</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/MetaVolcanoR.html">MetaVolcanoR</a></td>
            <td>Cesar Prada </td>
            <td>Gene Expression Meta-analysis Visualization Tool</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/MetCirc.html">MetCirc</a></td>
            <td>Thomas Naake </td>
            <td>Navigating mass spectral similarity in high-resolution MS/MS metabolomics data</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/MethCP.html">MethCP</a></td>
            <td>Boying Gong </td>
            <td>Differential methylation anlsysis for bisulfite sequencing data</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/methimpute.html">methimpute</a></td>
            <td>Aaron Taudt </td>
            <td>Imputation-guided re-construction of complete methylomes from WGBS data</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/methInheritSim.html">methInheritSim</a></td>
            <td>Pascal Belleau </td>
            <td>Simulating Whole-Genome Inherited Bisulphite Sequencing Data</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/MethPed.html">MethPed</a></td>
            <td>Helena Carén </td>
            <td>A DNA methylation classifier tool for the identification of pediatric brain tumor subtypes</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/methrix.html">methrix</a></td>
            <td>Anand Mayakonda </td>
            <td>Fast and efficient summarization of generic bedGraph files from Bisufite sequencing</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/MethTargetedNGS.html">MethTargetedNGS</a></td>
            <td>Muhammad Ahmer Jamil </td>
            <td>Perform Methylation Analysis on Next Generation Sequencing Data</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/methVisual.html">methVisual</a></td>
            <td>Arie Zackay </td>
            <td>Methods for visualization and statistics on DNA methylation data</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/methyAnalysis.html">methyAnalysis</a></td>
            <td>Pan Du, Lei Huang, Gang Feng </td>
            <td>DNA methylation data analysis and visualization</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/MethylAid.html">MethylAid</a></td>
            <td>L.J.Sinke </td>
            <td>Visual and interactive quality control of large Illumina DNA Methylation array data sets</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/methylCC.html">methylCC</a></td>
            <td>Stephanie C. Hicks </td>
            <td>Estimate the cell composition of whole blood in DNA methylation samples</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/methylGSA.html">methylGSA</a></td>
            <td>Xu Ren </td>
            <td>Gene Set Analysis Using the Outcome of Differential Methylation</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/methylInheritance.html">methylInheritance</a></td>
            <td>Astrid Deschenes </td>
            <td>Permutation-Based Analysis associating Conserved Differentially Methylated Elements Across Multiple Generations to a Treatment Effect</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/methylKit.html">methylKit</a></td>
            <td>Altuna Akalin, Alexander Gosdschan </td>
            <td>DNA methylation analysis from high-throughput bisulfite sequencing results</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/MethylMix.html">MethylMix</a></td>
            <td>Olivier Gevaert </td>
            <td>MethylMix: Identifying methylation driven cancer genes</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/methylMnM.html">methylMnM</a></td>
            <td>Yan Zhou</td>
            <td>detect different methylation level (DMR)</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/methylPipe.html">methylPipe</a></td>
            <td>Kamal Kishore </td>
            <td>Base resolution DNA methylation data analysis</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/MethylSeekR.html">MethylSeekR</a></td>
            <td>Lukas Burger </td>
            <td>Segmentation of Bis-seq data</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/methylumi.html">methylumi</a></td>
            <td>Sean Davis </td>
            <td>Handle Illumina methylation data</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/methyvim.html">methyvim</a></td>
            <td>Nima Hejazi </td>
            <td>Targeted, Robust, and Model-free Differential Methylation Analysis</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/MetID.html">MetID</a></td>
            <td>Zhenzhi Li </td>
            <td>Network-based prioritization of putative metabolite IDs</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/MetNet.html">MetNet</a></td>
            <td>Thomas Naake </td>
            <td>Inferring metabolic networks from untargeted high-resolution mass spectrometry data</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/mfa.html">mfa</a></td>
            <td>Kieran Campbell </td>
            <td>Bayesian hierarchical mixture of factor analyzers for modelling genomic bifurcations</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/Mfuzz.html">Mfuzz</a></td>
            <td>Matthias Futschik </td>
            <td>Soft clustering of time series gene expression data</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/MGFM.html">MGFM</a></td>
            <td>Khadija El Amrani </td>
            <td>Marker Gene Finder in Microarray gene expression data</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/MGFR.html">MGFR</a></td>
            <td>Khadija El Amrani </td>
            <td>Marker Gene Finder in RNA-seq data</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/mgsa.html">mgsa</a></td>
            <td>Sebastian Bauer </td>
            <td>Model-based gene set analysis</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/MiChip.html">MiChip</a></td>
            <td>Jonathon Blake </td>
            <td>MiChip Parsing and Summarizing Functions</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/microbiome.html">microbiome</a></td>
            <td>Leo Lahti </td>
            <td>Microbiome Analytics</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/microbiomeDASim.html">microbiomeDASim</a></td>
            <td>Justin Williams </td>
            <td>Microbiome Differential Abundance Simulation</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/microRNA.html">microRNA</a></td>
            <td>"James F. Reid" </td>
            <td>Data and functions for dealing with microRNAs</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/MIGSA.html">MIGSA</a></td>
            <td>Juan C. Rodriguez </td>
            <td>Massive and Integrative Gene Set Analysis</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/mimager.html">mimager</a></td>
            <td>Aaron Wolen </td>
            <td>mimager: The Microarray Imager</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/MIMOSA.html">MIMOSA</a></td>
            <td>Greg Finak </td>
            <td>Mixture Models for Single-Cell Assays</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/MineICA.html">MineICA</a></td>
            <td>Anne Biton </td>
            <td>Analysis of an ICA decomposition obtained on genomics data</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/minet.html">minet</a></td>
            <td>Patrick E. Meyer </td>
            <td>Mutual Information NETworks</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/minfi.html">minfi</a></td>
            <td>Kasper Daniel Hansen </td>
            <td>Analyze Illumina Infinium DNA methylation arrays</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/MinimumDistance.html">MinimumDistance</a></td>
            <td>Robert B Scharpf </td>
            <td>A Package for De Novo CNV Detection in Case-Parent Trios</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/MiPP.html">MiPP</a></td>
            <td>Sukwoo Kim </td>
            <td>Misclassification Penalized Posterior Classification</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/MIRA.html">MIRA</a></td>
            <td>John Lawson </td>
            <td>Methylation-Based Inference of Regulatory Activity</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/MiRaGE.html">MiRaGE</a></td>
            <td>Y-h. Taguchi </td>
            <td>MiRNA Ranking by Gene Expression</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/miRBaseConverter.html">miRBaseConverter</a></td>
            <td>Taosheng Xu</td>
            <td>A comprehensive and high-efficiency tool for converting and retrieving the information of miRNAs in different miRBase versions</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/miRcomp.html">miRcomp</a></td>
            <td>Matthew N. McCall </td>
            <td>Tools to assess and compare miRNA expression estimatation methods</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/mirIntegrator.html">mirIntegrator</a></td>
            <td>Diana Diaz </td>
            <td>Integrating microRNA expression into signaling pathways for pathway analysis</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/miRLAB.html">miRLAB</a></td>
            <td>Thuc Duy Le </td>
            <td>Dry lab for exploring miRNA-mRNA relationships</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/miRmine.html">miRmine</a></td>
            <td>Dusan Randjelovic </td>
            <td>Data package with miRNA-seq datasets from miRmine database as RangedSummarizedExperiment</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/miRNAmeConverter.html">miRNAmeConverter</a></td>
            <td>Stefan J. Haunsberger </td>
            <td>Convert miRNA Names to Different miRBase Versions</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/miRNApath.html">miRNApath</a></td>
            <td>James M. Ward </td>
            <td>miRNApath: Pathway Enrichment for miRNA Expression Data</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/miRNAtap.html">miRNAtap</a></td>
            <td>T. Ian Simpson </td>
            <td>miRNAtap: microRNA Targets - Aggregated Predictions</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/miRSM.html">miRSM</a></td>
            <td>Junpeng Zhang </td>
            <td>Inferring miRNA sponge modules by integrating expression data and miRNA-target binding information</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/miRspongeR.html">miRspongeR</a></td>
            <td>Junpeng Zhang </td>
            <td>Identification and analysis of miRNA sponge interaction networks and modules</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/Mirsynergy.html">Mirsynergy</a></td>
            <td>Yue Li </td>
            <td>Mirsynergy</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/missMethyl.html">missMethyl</a></td>
            <td>Belinda Phipson, Jovana Maksimovic, Andrew Lonsdale </td>
            <td>Analysing Illumina HumanMethylation BeadChip Data</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/missRows.html">missRows</a></td>
            <td>Gonzalez Ignacio </td>
            <td>Handling Missing Individuals in Multi-Omics Data Integration</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/mitoODE.html">mitoODE</a></td>
            <td>Gregoire Pau </td>
            <td>Implementation of the differential equation model described in "Dynamical modelling of phenotypes in a genome-wide RNAi live-cell imaging assay"</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/mixOmics.html">mixOmics</a></td>
            <td>Kim-Anh Le Cao </td>
            <td>Omics Data Integration Project</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/MLInterfaces.html">MLInterfaces</a></td>
            <td>V. Carey </td>
            <td>Uniform interfaces to R machine learning procedures for data in Bioconductor containers</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/mlm4omics.html">mlm4omics</a></td>
            <td>Irene SL Zeng </td>
            <td>Multilevel Model for Multivariate Responses with Missing Values</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/MLP.html">MLP</a></td>
            <td>Tobias Verbeke </td>
            <td>MLP</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/MLSeq.html">MLSeq</a></td>
            <td>Gokmen Zararsiz </td>
            <td>Machine Learning Interface for RNA-Seq Data</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/MMAPPR2.html">MMAPPR2</a></td>
            <td>Jonathon Hill </td>
            <td>Mutation Mapping Analysis Pipeline for Pooled RNA-Seq</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/MMDiff2.html">MMDiff2</a></td>
            <td>Gabriele Schweikert </td>
            <td>Statistical Testing for ChIP-Seq data sets</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/MmPalateMiRNA.html">MmPalateMiRNA</a></td>
            <td>Guy Brock </td>
            <td>Murine Palate miRNA Expression Analysis</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/MMUPHin.html">MMUPHin</a></td>
            <td>Siyuan MA </td>
            <td>Meta-analysis Methods with Uniform Pipeline for Heterogeneity in Microbiome Studies</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/mnem.html">mnem</a></td>
            <td>Martin Pirkl </td>
            <td>Mixture Nested Effects Models</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/MODA.html">MODA</a></td>
            <td>Dong Li </td>
            <td>MODA: MOdule Differential Analysis for weighted gene co-expression network</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/Modstrings.html">Modstrings</a></td>
            <td>Felix G.M. Ernst </td>
            <td>Implementation of Biostrings to work with nucleotide sequences containing modified nucleotides</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/MOFA.html">MOFA</a></td>
            <td>Britta Velten </td>
            <td>Multi-Omics Factor Analysis (MOFA)</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/mogsa.html">mogsa</a></td>
            <td>Chen Meng </td>
            <td>Multiple omics data integrative clustering and gene set analysis</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/monocle.html">monocle</a></td>
            <td>Cole Trapnell </td>
            <td>Clustering, differential expression, and trajectory analysis for single- cell RNA-Seq</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/MoonlightR.html">MoonlightR</a></td>
            <td>Antonio Colaprico, Catharina Olsen </td>
            <td>Identify oncogenes and tumor suppressor genes from omics data</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/MoPS.html">MoPS</a></td>
            <td>Philipp Eser </td>
            <td>MoPS - Model-based Periodicity Screening</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/mosaics.html">mosaics</a></td>
            <td>Dongjun Chung </td>
            <td>MOSAiCS (MOdel-based one and two Sample Analysis and Inference for ChIP-Seq)</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/MOSim.html">MOSim</a></td>
            <td>Carlos Martínez </td>
            <td>Multi-Omics Simulation (MOSim)</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/motifbreakR.html">motifbreakR</a></td>
            <td>Simon Gert Coetzee </td>
            <td>A Package For Predicting The Disruptiveness Of Single Nucleotide Polymorphisms On Transcription Factor Binding Sites</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/motifcounter.html">motifcounter</a></td>
            <td>Wolfgang Kopp </td>
            <td>R package for analysing TFBSs in DNA sequences</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/MotifDb.html">MotifDb</a></td>
            <td>Paul Shannon </td>
            <td>An Annotated Collection of Protein-DNA Binding Sequence Motifs</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/motifmatchr.html">motifmatchr</a></td>
            <td>Alicia Schep </td>
            <td>Fast Motif Matching in R</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/motifRG.html">motifRG</a></td>
            <td>Zizhen Yao </td>
            <td>A package for discriminative motif discovery, designed for high throughput sequencing dataset</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/motifStack.html">motifStack</a></td>
            <td>Jianhong Ou </td>
            <td>Plot stacked logos for single or multiple DNA, RNA and amino acid sequence</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/MotIV.html">MotIV</a></td>
            <td>Eloi Mercier, Raphael Gottardo </td>
            <td>Motif Identification and Validation</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/MPFE.html">MPFE</a></td>
            <td>Conrad Burden </td>
            <td>Estimation of the amplicon methylation pattern distribution from bisulphite sequencing data</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/mpra.html">mpra</a></td>
            <td>Leslie Myint </td>
            <td>Analyze massively parallel reporter assays</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/MPRAnalyze.html">MPRAnalyze</a></td>
            <td>Tal Ashuach </td>
            <td>Statistical Analysis of MPRA data</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/msa.html">msa</a></td>
            <td>Ulrich Bodenhofer </td>
            <td>Multiple Sequence Alignment</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/msgbsR.html">msgbsR</a></td>
            <td>Benjamin Mayne </td>
            <td>msgbsR: methylation sensitive genotyping by sequencing (MS-GBS) R functions</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/MSGFgui.html">MSGFgui</a></td>
            <td>Thomas Lin Pedersen </td>
            <td>A shiny GUI for MSGFplus</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/MSGFplus.html">MSGFplus</a></td>
            <td>Thomas Lin Pedersen </td>
            <td>An interface between R and MS-GF+</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/msmsEDA.html">msmsEDA</a></td>
            <td>Josep Gregori </td>
            <td>Exploratory Data Analysis of LC-MS/MS data by spectral counts</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/msmsTests.html">msmsTests</a></td>
            <td>Josep Gregori i Font </td>
            <td>LC-MS/MS Differential Expression Tests</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/MSnbase.html">MSnbase</a></td>
            <td>Laurent Gatto </td>
            <td>Base Functions and Classes for Mass Spectrometry and Proteomics</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/MSnID.html">MSnID</a></td>
            <td>Vlad Petyuk </td>
            <td>Utilities for Exploration and Assessment of Confidence of LC-MSn Proteomics Identifications</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/msPurity.html">msPurity</a></td>
            <td>Thomas N. Lawson </td>
            <td>Automated Evaluation of Precursor Ion Purity for Mass Spectrometry Based Fragmentation in Metabolomics</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/MSstats.html">MSstats</a></td>
            <td>Meena Choi </td>
            <td>Protein Significance Analysis in DDA, SRM and DIA for Label-free or Label-based Proteomics Experiments</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/MSstatsQC.html">MSstatsQC</a></td>
            <td>Eralp Dogu </td>
            <td>Longitudinal system suitability monitoring and quality control for proteomic experiments</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/MSstatsQCgui.html">MSstatsQCgui</a></td>
            <td>Eralp Dogu </td>
            <td>A graphical user interface for MSstatsQC package</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/MSstatsSampleSize.html">MSstatsSampleSize</a></td>
            <td>Ting Huang </td>
            <td>Simulation tool for optimal design of high-dimensional MS-based proteomics experiment</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/MSstatsTMT.html">MSstatsTMT</a></td>
            <td>Ting Huang </td>
            <td>Protein Significance Analysis in shotgun mass spectrometry-based proteomic experiments with tandem mass tag (TMT) labeling</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/MTseeker.html">MTseeker</a></td>
            <td>Tim Triche </td>
            <td>Bioconductor Tools for Human Mitochondrial Variant Analysis</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/Mulcom.html">Mulcom</a></td>
            <td>Claudio Isella </td>
            <td>Calculates Mulcom test</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/MultiAssayExperiment.html">MultiAssayExperiment</a></td>
            <td>Marcel Ramos </td>
            <td>Software for the integration of multi-omics experiments in Bioconductor</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/multiClust.html">multiClust</a></td>
            <td>Nathan Lawlor </td>
            <td>multiClust: An R-package for Identifying Biologically Relevant Clusters in Cancer Transcriptome Profiles</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/MultiDataSet.html">MultiDataSet</a></td>
            <td>Carlos Ruiz-Arenas </td>
            <td>Implementation of MultiDataSet and ResultSet</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/multiHiCcompare.html">multiHiCcompare</a></td>
            <td>John Stansfield </td>
            <td>Normalize and detect differences between Hi-C datasets when replicates of each experimental condition are available</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/MultiMed.html">MultiMed</a></td>
            <td>Simina M. Boca </td>
            <td>Testing multiple biological mediators simultaneously</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/multiMiR.html">multiMiR</a></td>
            <td>Matt Mulvahill </td>
            <td>Integration of multiple microRNA-target databases with their disease and drug associations</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/multiOmicsViz.html">multiOmicsViz</a></td>
            <td>Jing Wang </td>
            <td>Plot the effect of one omics data on other omics data along the chromosome</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/multiscan.html">multiscan</a></td>
            <td>Mizanur Khondoker </td>
            <td>R package for combining multiple scans</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/multtest.html">multtest</a></td>
            <td>Katherine S. Pollard </td>
            <td>Resampling-based multiple hypothesis testing</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/muscat.html">muscat</a></td>
            <td>Helena L. Crowell </td>
            <td>Multi-sample multi-group scRNA-seq data analysis tools</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/muscle.html">muscle</a></td>
            <td>Alex T. Kalinka </td>
            <td>Multiple Sequence Alignment with MUSCLE</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/MutationalPatterns.html">MutationalPatterns</a></td>
            <td>Roel Janssen, Francis Blokzijl </td>
            <td>Comprehensive genome-wide analysis of mutational processes</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/MVCClass.html">MVCClass</a></td>
            <td>Elizabeth Whalen </td>
            <td>Model-View-Controller (MVC) Classes</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/MWASTools.html">MWASTools</a></td>
            <td>Andrea Rodriguez-Martinez, Rafael Ayala </td>
            <td>MWASTools: an integrated pipeline to perform metabolome-wide association studies</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/mygene.html">mygene</a></td>
            <td>Adam Mark, Cyrus Afrasiabi, Chunlei Wu </td>
            <td>Access MyGene.Info_ services</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/myvariant.html">myvariant</a></td>
            <td>Adam Mark, Chunlei Wu </td>
            <td>Accesses MyVariant.info variant query and annotation services</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/mzID.html">mzID</a></td>
            <td>Laurent Gatto </td>
            <td>An mzIdentML parser for R</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/mzR.html">mzR</a></td>
            <td>Steffen Neumann, Laurent Gatto, Qiang Kou </td>
            <td>parser for netCDF, mzXML, mzData and mzML and mzIdentML files (mass spectrometry data)</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/NADfinder.html">NADfinder</a></td>
            <td>Jianhong Ou, Lihua Julie Zhu </td>
            <td>Call wide peaks for sequencing data</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/NanoStringDiff.html">NanoStringDiff</a></td>
            <td>tingting zhai,hong wang </td>
            <td>Differential Expression Analysis of NanoString nCounter Data</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/NanoStringQCPro.html">NanoStringQCPro</a></td>
            <td>Robert Ziman </td>
            <td>Quality metrics and data processing methods for NanoString mRNA gene expression data</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/nanotatoR.html">nanotatoR</a></td>
            <td>Surajit Bhattacharya </td>
            <td>nanotatoR: next generation structural variant annotation and classification</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/NarrowPeaks.html">NarrowPeaks</a></td>
            <td>Pedro Madrigal </td>
            <td>Shape-based Analysis of Variation in ChIP-seq using Functional PCA</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/NBAMSeq.html">NBAMSeq</a></td>
            <td>Xu Ren </td>
            <td>Negative Binomial Additive Model for RNA-Seq Data</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/NBSplice.html">NBSplice</a></td>
            <td>Gabriela Merino </td>
            <td>Negative Binomial Models to detect Differential Splicing</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/ncdfFlow.html">ncdfFlow</a></td>
            <td>Mike Jiang </td>
            <td>ncdfFlow: A package that provides HDF5 based storage for flow cytometry data.</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/ncGTW.html">ncGTW</a></td>
            <td>Chiung-Ting Wu </td>
            <td>Alignment of LC-MS Profiles by Neighbor-wise Compound-specific Graphical Time Warping with Misalignment Detection</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/NCIgraph.html">NCIgraph</a></td>
            <td>Laurent Jacob </td>
            <td>Pathways from the NCI Pathways Database</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/ndexr.html">ndexr</a></td>
            <td>Florian Auer </td>
            <td>NDEx R client library</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/NeighborNet.html">NeighborNet</a></td>
            <td>Sahar Ansari </td>
            <td>Neighbor_net analysis</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/nem.html">nem</a></td>
            <td>Holger Froehlich </td>
            <td>(Dynamic) Nested Effects Models and Deterministic Effects Propagation Networks to reconstruct phenotypic hierarchies</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/netbenchmark.html">netbenchmark</a></td>
            <td>Pau Bellot </td>
            <td>Benchmarking of several gene network inference methods</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/netbiov.html">netbiov</a></td>
            <td>Shailesh tripathi </td>
            <td>A package for visualizing complex biological network</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/netboost.html">netboost</a></td>
            <td>Pascal Schlosser </td>
            <td>Network Analysis Supported by Boosting</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/nethet.html">nethet</a></td>
            <td>Nicolas Staedler, Frank Dondelinger </td>
            <td>A bioconductor package for high-dimensional exploration of biological network heterogeneity</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/NetPathMiner.html">NetPathMiner</a></td>
            <td>Ahmed Mohamed </td>
            <td>NetPathMiner for Biological Network Construction, Path Mining and Visualization</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/netprioR.html">netprioR</a></td>
            <td>Fabian Schmich </td>
            <td>A model for network-based prioritisation of genes</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/netReg.html">netReg</a></td>
            <td>Simon Dirmeier </td>
            <td>Network-Regularized Regression Models</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/netresponse.html">netresponse</a></td>
            <td>Leo Lahti </td>
            <td>Functional Network Analysis</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/NetSAM.html">NetSAM</a></td>
            <td>Bing Zhang </td>
            <td>Network Seriation And Modularization</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/netSmooth.html">netSmooth</a></td>
            <td>Jonathan Ronen </td>
            <td>Network smoothing for scRNAseq</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/networkBMA.html">networkBMA</a></td>
            <td>Ka Yee Yeung </td>
            <td>Regression-based network inference using Bayesian Model Averaging</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/ngsReports.html">ngsReports</a></td>
            <td>Steve Pederson </td>
            <td>Load FastqQC reports and other NGS related files</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/nnNorm.html">nnNorm</a></td>
            <td>Adi Laurentiu Tarca </td>
            <td>Spatial and intensity based normalization of cDNA microarray data based on robust neural nets</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/NOISeq.html">NOISeq</a></td>
            <td>Sonia Tarazona </td>
            <td>Exploratory analysis and differential expression for RNA-seq data</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/nondetects.html">nondetects</a></td>
            <td>Valeriia Sherina </td>
            <td>Non-detects in qPCR data</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/normalize450K.html">normalize450K</a></td>
            <td>Jonathan Alexander Heiss </td>
            <td>Preprocessing of Illumina Infinium 450K data</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/NormalyzerDE.html">NormalyzerDE</a></td>
            <td>Jakob Willforss </td>
            <td>Evaluation of normalization methods and calculation of differential expression analysis statistics</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/NormqPCR.html">NormqPCR</a></td>
            <td>James Perkins </td>
            <td>Functions for normalisation of RT-qPCR data</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/normr.html">normr</a></td>
            <td>Johannes Helmuth </td>
            <td>Normalization and difference calling in ChIP-seq data</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/npGSEA.html">npGSEA</a></td>
            <td>Jessica Larson </td>
            <td>Permutation approximation methods for gene set enrichment analysis (non-permutation GSEA)</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/NTW.html">NTW</a></td>
            <td>Yuanhua Liu </td>
            <td>Predict gene network using an Ordinary Differential Equation (ODE) based method</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/nucleoSim.html">nucleoSim</a></td>
            <td>Astrid Deschenes </td>
            <td>Generate synthetic nucleosome maps</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/nucleR.html">nucleR</a></td>
            <td>Diego Gallego </td>
            <td>Nucleosome positioning package for R</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/nuCpos.html">nuCpos</a></td>
            <td>Hiroaki Kato </td>
            <td>An R package for prediction of nucleosome positions</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/NuPoP.html">NuPoP</a></td>
            <td>Ji-Ping Wang</td>
            <td>An R package for nucleosome positioning prediction</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/occugene.html">occugene</a></td>
            <td>Oliver Will </td>
            <td>Functions for Multinomial Occupancy Distribution</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/OCplus.html">OCplus</a></td>
            <td>Alexander Ploner </td>
            <td>Operating characteristics plus sample size and local fdr for microarray experiments</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/odseq.html">odseq</a></td>
            <td>José Jiménez </td>
            <td>Outlier detection in multiple sequence alignments</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/OGSA.html">OGSA</a></td>
            <td>Michael F. Ochs </td>
            <td>Outlier Gene Set Analysis</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/oligo.html">oligo</a></td>
            <td>Benilton Carvalho </td>
            <td>Preprocessing tools for oligonucleotide arrays</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/oligoClasses.html">oligoClasses</a></td>
            <td>Benilton Carvalho and Robert Scharpf </td>
            <td>Classes for high-throughput arrays supported by oligo and crlmm</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/OLIN.html">OLIN</a></td>
            <td>Matthias Futschik </td>
            <td>Optimized local intensity-dependent normalisation of two-color microarrays</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/OLINgui.html">OLINgui</a></td>
            <td>Matthias Futschik </td>
            <td>Graphical user interface for OLIN</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/OmaDB.html">OmaDB</a></td>
            <td>Klara Kaleb, Adrian Altenhoff </td>
            <td>R wrapper for the OMA REST API</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/omicade4.html">omicade4</a></td>
            <td>Chen Meng </td>
            <td>Multiple co-inertia analysis of omics datasets</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/OmicCircos.html">OmicCircos</a></td>
            <td>Ying Hu </td>
            <td>High-quality circular visualization of omics data</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/omicplotR.html">omicplotR</a></td>
            <td>Daniel Giguere </td>
            <td>Visual Exploration of Omic Datasets Using a Shiny App</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/omicRexposome.html">omicRexposome</a></td>
            <td>Carles Hernandez-Ferrer </td>
            <td>Exposome and omic data associatin and integration analysis</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/OmicsLonDA.html">OmicsLonDA</a></td>
            <td>Ahmed A. Metwally </td>
            <td>Omics Longitudinal Differential Analysis</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/OmicsMarkeR.html">OmicsMarkeR</a></td>
            <td>Charles E. Determan Jr. </td>
            <td>Classification and Feature Selection for 'Omics' Datasets</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/OMICsPCA.html">OMICsPCA</a></td>
            <td>Subhadeep Das </td>
            <td>An R package for quantitative integration and analysis of multiple omics assays from heterogeneous samples</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/omicsPrint.html">omicsPrint</a></td>
            <td>Davy Cats </td>
            <td>Cross omic genetic fingerprinting</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/OmnipathR.html">OmnipathR</a></td>
            <td>Alberto Valdeolivas Urbelz </td>
            <td>Import Omnipath network</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/Onassis.html">Onassis</a></td>
            <td>Eugenia Galeota </td>
            <td>OnASSIs Ontology Annotation and Semantic SImilarity software</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/oncomix.html">oncomix</a></td>
            <td>Daniel Pique </td>
            <td>Identifying Genes Overexpressed in Subsets of Tumors from Tumor-Normal mRNA Expression Data</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/OncoScore.html">OncoScore</a></td>
            <td>Luca De Sano </td>
            <td>A tool to identify potentially oncogenic genes</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/OncoSimulR.html">OncoSimulR</a></td>
            <td>Ramon Diaz-Uriarte </td>
            <td>Forward Genetic Simulation of Cancer Progression with Epistasis</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/oneSENSE.html">oneSENSE</a></td>
            <td>Yong Kee Tan </td>
            <td>One-Dimensional Soli-Expression by Nonlinear Stochastic Embedding (OneSENSE)</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/onlineFDR.html">onlineFDR</a></td>
            <td>David S. Robertson </td>
            <td>Online error control</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/ontoProc.html">ontoProc</a></td>
            <td>VJ Carey </td>
            <td>processing of ontologies of anatomy, cell lines, and so on</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/openCyto.html">openCyto</a></td>
            <td>Mike Jiang </td>
            <td>Hierarchical Gating Pipeline for flow cytometry data</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/openPrimeR.html">openPrimeR</a></td>
            <td>Matthias Döring </td>
            <td>Multiplex PCR Primer Design and Analysis</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/openPrimeRui.html">openPrimeRui</a></td>
            <td>Matthias Döring </td>
            <td>Shiny Application for Multiplex PCR Primer Design and Analysis</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/oposSOM.html">oposSOM</a></td>
            <td>Henry Loeffler-Wirth </td>
            <td>Comprehensive analysis of transciptome data</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/oppar.html">oppar</a></td>
            <td>Soroor Hediyeh zadeh </td>
            <td>Outlier profile and pathway analysis in R</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/oppti.html">oppti</a></td>
            <td>Abdulkadir Elmas </td>
            <td>Outlier Protein and Phosphosite Target Identifier</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/OPWeight.html">OPWeight</a></td>
            <td>Mohamad Hasan </td>
            <td>Optimal p-value weighting with independent information</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/OrderedList.html">OrderedList</a></td>
            <td>Claudio Lottaz </td>
            <td>Similarities of Ordered Gene Lists</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/ORFik.html">ORFik</a></td>
            <td>Haakon Tjeldnes </td>
            <td>Open Reading Frames in Genomics</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/Organism.dplyr.html">Organism.dplyr</a></td>
            <td>Martin Morgan </td>
            <td>dplyr-based Access to Bioconductor Annotation Resources</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/OrganismDbi.html">OrganismDbi</a></td>
            <td>Bioconductor Package Maintainer </td>
            <td>Software to enable the smooth interfacing of different database packages</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/OSAT.html">OSAT</a></td>
            <td>Li Yan </td>
            <td>OSAT: Optimal Sample Assignment Tool</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/Oscope.html">Oscope</a></td>
            <td>Ning Leng </td>
            <td>Oscope - A statistical pipeline for identifying oscillatory genes in unsynchronized single cell RNA-seq</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/OTUbase.html">OTUbase</a></td>
            <td>Daniel Beck </td>
            <td>Provides structure and functions for the analysis of OTU data</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/OutlierD.html">OutlierD</a></td>
            <td>Sukwoo Kim </td>
            <td>Outlier detection using quantile regression on the M-A scatterplots of high-throughput data</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/OUTRIDER.html">OUTRIDER</a></td>
            <td>Christian Mertes </td>
            <td>OUTRIDER - OUTlier in RNA-Seq fInDER</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/OVESEG.html">OVESEG</a></td>
            <td>Lulu Chen </td>
            <td>OVESEG-test to detect tissue/cell-specific markers</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/PAA.html">PAA</a></td>
            <td>Michael Turewicz, Martin Eisenacher </td>
            <td>PAA (Protein Array Analyzer)</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/PADOG.html">PADOG</a></td>
            <td>Adi Laurentiu Tarca </td>
            <td>Pathway Analysis with Down-weighting of Overlapping Genes (PADOG)</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/PAIRADISE.html">PAIRADISE</a></td>
            <td>Qiang Hu, Levon Demirdjian </td>
            <td>PAIRADISE: Paired analysis of differential isoform expression</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/paircompviz.html">paircompviz</a></td>
            <td>Michal Burda </td>
            <td>Multiple comparison test visualization</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/pandaR.html">pandaR</a></td>
            <td>Joseph N. Paulson, Dan Schlauch </td>
            <td>PANDA Algorithm</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/panelcn.mops.html">panelcn.mops</a></td>
            <td>Gundula Povysil </td>
            <td>CNV detection tool for targeted NGS panel data</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/panp.html">panp</a></td>
            <td>Peter Warren </td>
            <td>Presence-Absence Calls from Negative Strand Matching Probesets</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/PANR.html">PANR</a></td>
            <td>Xin Wang </td>
            <td>Posterior association networks and functional modules inferred from rich phenotypes of gene perturbations</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/PanVizGenerator.html">PanVizGenerator</a></td>
            <td>Thomas Lin Pedersen </td>
            <td>Generate PanViz visualisations from your pangenome</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/PAPi.html">PAPi</a></td>
            <td>Raphael Aggio </td>
            <td>Predict metabolic pathway activity based on metabolomics data</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/parglms.html">parglms</a></td>
            <td>VJ Carey </td>
            <td>support for parallelized estimation of GLMs/GEEs</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/parody.html">parody</a></td>
            <td>VJ Carey </td>
            <td>Parametric And Resistant Outlier DYtection</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/PAST.html">PAST</a></td>
            <td>Thrash Adam </td>
            <td>Pathway Association Study Tool (PAST)</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/Path2PPI.html">Path2PPI</a></td>
            <td>Oliver Philipp </td>
            <td>Prediction of pathway-related protein-protein interaction networks</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/pathifier.html">pathifier</a></td>
            <td>Assif Yitzhaky </td>
            <td>Quantify deregulation of pathways in cancer</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/PathNet.html">PathNet</a></td>
            <td>Jason B. Smith </td>
            <td>An R package for pathway analysis using topological information</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/PathoStat.html">PathoStat</a></td>
            <td>Solaiappan Manimaran, Yue Zhao </td>
            <td>PathoStat Statistical Microbiome Analysis Package</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/pathprint.html">pathprint</a></td>
            <td>Sokratis Kariotis </td>
            <td>Pathway fingerprinting for analysis of gene expression arrays</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/pathRender.html">pathRender</a></td>
            <td>Vince Carey </td>
            <td>Render molecular pathways</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/pathVar.html">pathVar</a></td>
            <td>Samuel Zimmerman </td>
            <td>Methods to Find Pathways with Significantly Different Variability</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/pathview.html">pathview</a></td>
            <td>Weijun Luo </td>
            <td>a tool set for pathway based data integration and visualization</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/pathwayPCA.html">pathwayPCA</a></td>
            <td>Gabriel Odom </td>
            <td>Integrative Pathway Analysis with Modern PCA Methodology and Gene Selection</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/PathwaySplice.html">PathwaySplice</a></td>
            <td>Aimin Yan </td>
            <td>An R Package for Unbiased Splicing Pathway Analysis</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/paxtoolsr.html">paxtoolsr</a></td>
            <td>Augustin Luna </td>
            <td>PaxtoolsR: Access Pathways from Multiple Databases through BioPAX and Pathway Commons</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/Pbase.html">Pbase</a></td>
            <td>Laurent Gatto </td>
            <td>Manipulating and exploring protein and proteomics data</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/pcaExplorer.html">pcaExplorer</a></td>
            <td>Federico Marini </td>
            <td>Interactive Visualization of RNA-seq Data Using a Principal Components Approach</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/pcaGoPromoter.html">pcaGoPromoter</a></td>
            <td>Morten Hansen </td>
            <td>pcaGoPromoter is used to analyze DNA micro array data</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/pcaMethods.html">pcaMethods</a></td>
            <td>Henning Redestig </td>
            <td>A collection of PCA methods</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/PCAN.html">PCAN</a></td>
            <td>Matthew Page and Patrice Godard </td>
            <td>Phenotype Consensus ANalysis (PCAN)</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/PCAtools.html">PCAtools</a></td>
            <td>Kevin Blighe </td>
            <td>PCAtools: Everything Principal Components Analysis</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/pcot2.html">pcot2</a></td>
            <td>Sarah Song </td>
            <td>Principal Coordinates and Hotelling's T-Square method</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/PCpheno.html">PCpheno</a></td>
            <td>Nolwenn Le Meur </td>
            <td>Phenotypes and cellular organizational units</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/pcxn.html">pcxn</a></td>
            <td>Sokratis Kariotis </td>
            <td>Exploring, analyzing and visualizing functions utilizing the pcxnData package</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/pdInfoBuilder.html">pdInfoBuilder</a></td>
            <td>Benilton Carvalho </td>
            <td>Platform Design Information Package Builder</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/peakPantheR.html">peakPantheR</a></td>
            <td>Arnaud Wolfer </td>
            <td>Peak Picking and Annotation of High Resolution Experiments</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/PECA.html">PECA</a></td>
            <td>Tomi Suomi </td>
            <td>Probe-level Expression Change Averaging</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/PepsNMR.html">PepsNMR</a></td>
            <td>Manon Martin </td>
            <td>Pre-process 1H-NMR FID signals</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/pepStat.html">pepStat</a></td>
            <td>Gregory C Imholte </td>
            <td>Statistical analysis of peptide microarrays</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/pepXMLTab.html">pepXMLTab</a></td>
            <td>Xiaojing Wang </td>
            <td>Parsing pepXML files and filter based on peptide FDR.</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/PERFect.html">PERFect</a></td>
            <td>Quy Cao </td>
            <td>Permutation filtration for microbiome data</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/perturbatr.html">perturbatr</a></td>
            <td>Simon Dirmeier </td>
            <td>Statistical Analysis of High-Throughput Genetic Perturbation Screens</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/PGA.html">PGA</a></td>
            <td>Bo Wen, Shaohang Xu </td>
            <td>An package for identification of novel peptides by customized database derived from RNA-Seq</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/pgca.html">pgca</a></td>
            <td>Gabriela Cohen-Freue </td>
            <td>PGCA: An Algorithm to Link Protein Groups Created from MS/MS Data</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/PGSEA.html">PGSEA</a></td>
            <td>Karl Dykema </td>
            <td>Parametric Gene Set Enrichment Analysis</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/phantasus.html">phantasus</a></td>
            <td>Alexey Sergushichev </td>
            <td>Visual and interactive gene expression analysis</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/PharmacoGx.html">PharmacoGx</a></td>
            <td>Benjamin Haibe-Kains </td>
            <td>Analysis of Large-Scale Pharmacogenomic Data</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/phemd.html">phemd</a></td>
            <td>William S Chen </td>
            <td>Phenotypic EMD for comparison of single-cell samples</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/phenopath.html">phenopath</a></td>
            <td>Kieran Campbell </td>
            <td>Genomic trajectories with heterogeneous genetic and environmental backgrounds</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/phenoTest.html">phenoTest</a></td>
            <td>Evarist Planet </td>
            <td>Tools to test association between gene expression and phenotype in a way that is efficient, structured, fast and scalable. We also provide tools to do GSEA (Gene set enrichment analysis) and copy number variation.</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/PhenStat.html">PhenStat</a></td>
            <td>Hamed Haselimashhadi </td>
            <td>Statistical analysis of phenotypic data</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/philr.html">philr</a></td>
            <td>Justin Silverman </td>
            <td>Phylogenetic partitioning based ILR transform for metagenomics data</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/phosphonormalizer.html">phosphonormalizer</a></td>
            <td>Sohrab Saraei </td>
            <td>Compensates for the bias introduced by median normalization in phosphoproteomics</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/PhyloProfile.html">PhyloProfile</a></td>
            <td>Vinh Tran </td>
            <td>PhyloProfile</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/phyloseq.html">phyloseq</a></td>
            <td>Paul J. McMurdie </td>
            <td>Handling and analysis of high-throughput microbiome census data</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/Pi.html">Pi</a></td>
            <td>Hai Fang </td>
            <td>Leveraging Genetic Evidence to Prioritise Drug Targets at the Gene and Pathway Level</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/piano.html">piano</a></td>
            <td>Leif Varemo Wigge </td>
            <td>Platform for integrative analysis of omics data</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/pickgene.html">pickgene</a></td>
            <td>Brian S. Yandell </td>
            <td>Adaptive Gene Picking for Microarray Expression Data Analysis</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/PICS.html">PICS</a></td>
            <td>Renan Sauteraud </td>
            <td>Probabilistic inference of ChIP-seq</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/Pigengene.html">Pigengene</a></td>
            <td>Habil Zare </td>
            <td>Infers biological signatures from gene expression data</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/PING.html">PING</a></td>
            <td>Renan Sauteraud </td>
            <td>Probabilistic inference for Nucleosome Positioning with MNase-based or Sonicated Short-read Data</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/pint.html">pint</a></td>
            <td>Olli-Pekka Huovilainen </td>
            <td>Pairwise INTegration of functional genomics data</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/pipeFrame.html">pipeFrame</a></td>
            <td>Zheng Wei </td>
            <td>Pipeline framework for bioinformatics in R</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/pkgDepTools.html">pkgDepTools</a></td>
            <td>Bioconductor Package Maintainer </td>
            <td>Package Dependency Tools</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/plateCore.html">plateCore</a></td>
            <td>Errol Strain </td>
            <td>Statistical tools and data structures for plate-based flow cytometry</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/plethy.html">plethy</a></td>
            <td>Daniel Bottomly </td>
            <td>R framework for exploration and analysis of respirometry data</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/plgem.html">plgem</a></td>
            <td>Norman Pavelka </td>
            <td>Detect differential expression in microarray and proteomics datasets with the Power Law Global Error Model (PLGEM)</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/plier.html">plier</a></td>
            <td>Crispin Miller </td>
            <td>Implements the Affymetrix PLIER algorithm</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/plotGrouper.html">plotGrouper</a></td>
            <td>John D. Gagnon </td>
            <td>Shiny app GUI wrapper for ggplot with built-in statistical analysis</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/PLPE.html">PLPE</a></td>
            <td>Soo-heang Eo </td>
            <td>Local Pooled Error Test for Differential Expression with Paired High-throughput Data</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/plrs.html">plrs</a></td>
            <td>Gwenael G.R. Leday to </td>
            <td>Piecewise Linear Regression Splines (PLRS) for the association between DNA copy number and gene expression</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/plw.html">plw</a></td>
            <td>Magnus Astrand </td>
            <td>Probe level Locally moderated Weighted t-tests.</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/plyranges.html">plyranges</a></td>
            <td>Stuart Lee </td>
            <td>A fluent interface for manipulating GenomicRanges</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/pmm.html">pmm</a></td>
            <td>Anna Drewek </td>
            <td>Parallel Mixed Model</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/podkat.html">podkat</a></td>
            <td>Ulrich Bodenhofer </td>
            <td>Position-Dependent Kernel Association Test</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/pogos.html">pogos</a></td>
            <td>VJ Carey </td>
            <td>PharmacOGenomics Ontology Support</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/polyester.html">polyester</a></td>
            <td>Jack Fu, Jeff Leek </td>
            <td>Simulate RNA-seq reads</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/Polyfit.html">Polyfit</a></td>
            <td>Conrad Burden </td>
            <td>Add-on to DESeq to improve p-values and q-values</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/POST.html">POST</a></td>
            <td>Xueyuan Cao </td>
            <td>Projection onto Orthogonal Space Testing for High Dimensional Data</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/PoTRA.html">PoTRA</a></td>
            <td>Valentin Dinu </td>
            <td>PoTRA: Pathways of Topological Rank Analysis</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/PowerExplorer.html">PowerExplorer</a></td>
            <td>Xu Qiao </td>
            <td>Power Estimation Tool for RNA-Seq and proteomics data</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/powerTCR.html">powerTCR</a></td>
            <td>Hillary Koch </td>
            <td>Model-Based Comparative Analysis of the TCR Repertoire</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/PPInfer.html">PPInfer</a></td>
            <td>Dongmin Jung </td>
            <td>Inferring functionally related proteins using protein interaction networks</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/ppiStats.html">ppiStats</a></td>
            <td>Bioconductor Package Maintainer </td>
            <td>Protein-Protein Interaction Statistical Package</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/pqsfinder.html">pqsfinder</a></td>
            <td>Jiri Hon </td>
            <td>Identification of potential quadruplex forming sequences</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/prada.html">prada</a></td>
            <td>Florian Hahne </td>
            <td>Data analysis for cell-based functional assays</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/pram.html">pram</a></td>
            <td>Peng Liu </td>
            <td>Pooling RNA-seq datasets for assembling transcript models</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/prebs.html">prebs</a></td>
            <td>Karolis Uziela </td>
            <td>Probe region expression estimation for RNA-seq data for improved microarray comparability</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/PrecisionTrialDrawer.html">PrecisionTrialDrawer</a></td>
            <td>Giorgio Melloni </td>
            <td>A Tool to Analyze and Design NGS Based Custom Gene Panels</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/PREDA.html">PREDA</a></td>
            <td>Francesco Ferrari </td>
            <td>Position Related Data Analysis</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/predictionet.html">predictionet</a></td>
            <td>Benjamin Haibe-Kains, Catharina Olsen </td>
            <td>Inference for predictive networks designed for (but not limited to) genomic data</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/preprocessCore.html">preprocessCore</a></td>
            <td>Ben Bolstad </td>
            <td>A collection of pre-processing functions</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/primirTSS.html">primirTSS</a></td>
            <td>Pumin Li </td>
            <td>Prediction of pri-miRNA Transcription Start Site</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/PrInCE.html">PrInCE</a></td>
            <td>Michael Skinnider </td>
            <td>Predicting Interactomes from Co-Elution</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/Prize.html">Prize</a></td>
            <td>Daryanaz Dargahi </td>
            <td>Prize: an R package for prioritization estimation based on analytic hierarchy process</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/proBAMr.html">proBAMr</a></td>
            <td>Xiaojing Wang </td>
            <td>Generating SAM file for PSMs in shotgun proteomics data</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/proBatch.html">proBatch</a></td>
            <td>Jelena Cuklina </td>
            <td>Tools for Diagnostics and Corrections of Batch Effects in Proteomics</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/PROcess.html">PROcess</a></td>
            <td>Xiaochun Li </td>
            <td>Ciphergen SELDI-TOF Processing</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/procoil.html">procoil</a></td>
            <td>Ulrich Bodenhofer </td>
            <td>Prediction of Oligomerization of Coiled Coil Proteins</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/proDA.html">proDA</a></td>
            <td>Constantin Ahlmann-Eltze </td>
            <td>Differential Abundance Analysis of Label-Free Mass Spectrometry Data</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/proFIA.html">proFIA</a></td>
            <td>Alexis Delabriere </td>
            <td>Preprocessing of FIA-HRMS data</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/profileplyr.html">profileplyr</a></td>
            <td>Tom Carroll, Doug Barrows </td>
            <td>Visualization and annotation of read signal over genomic ranges with profileplyr</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/profileScoreDist.html">profileScoreDist</a></td>
            <td>Paal O. Westermark </td>
            <td>Profile score distributions</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/progeny.html">progeny</a></td>
            <td>Michael Schubert </td>
            <td>Pathway RespOnsive GENes for activity inference from gene expression</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/projectR.html">projectR</a></td>
            <td>Genevieve Stein-O'Brien </td>
            <td>Functions for the projection of weights from PCA, CoGAPS, NMF, correlation, and clustering</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/pRoloc.html">pRoloc</a></td>
            <td>Laurent Gatto </td>
            <td>A unifying bioinformatics framework for spatial proteomics</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/pRolocGUI.html">pRolocGUI</a></td>
            <td>Laurent Gatto, Lisa M Breckels </td>
            <td>Interactive visualisation of spatial proteomics data</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/PROMISE.html">PROMISE</a></td>
            <td>Stan Pounds, Xueyuan Cao </td>
            <td>PRojection Onto the Most Interesting Statistical Evidence</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/PROPER.html">PROPER</a></td>
            <td>Hao Wu </td>
            <td>PROspective Power Evaluation for RNAseq</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/PROPS.html">PROPS</a></td>
            <td>Lichy Han </td>
            <td>PRObabilistic Pathway Score (PROPS)</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/Prostar.html">Prostar</a></td>
            <td>Samuel Wieczorek </td>
            <td>Provides a GUI for DAPAR</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/proteinProfiles.html">proteinProfiles</a></td>
            <td>Julian Gehring </td>
            <td>Protein Profiling</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/ProteomicsAnnotationHubData.html">ProteomicsAnnotationHubData</a></td>
            <td>Laurent Gatto </td>
            <td>Transform public proteomics data resources into Bioconductor Data Structures</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/ProteoMM.html">ProteoMM</a></td>
            <td>Yuliya V Karpievitch </td>
            <td>Multi-Dataset Model-based Differential Expression Proteomics Analysis Platform</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/proteoQC.html">proteoQC</a></td>
            <td>Bo Wen </td>
            <td>An R package for proteomics data quality control</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/ProtGenerics.html">ProtGenerics</a></td>
            <td>Laurent Gatto </td>
            <td>S4 generic functions for Bioconductor proteomics infrastructure</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/PSEA.html">PSEA</a></td>
            <td>Alexandre Kuhn </td>
            <td>Population-Specific Expression Analysis.</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/psichomics.html">psichomics</a></td>
            <td>Nuno Saraiva-Agostinho </td>
            <td>Graphical Interface for Alternative Splicing Quantification, Analysis and Visualisation</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/PSICQUIC.html">PSICQUIC</a></td>
            <td>Paul Shannon</td>
            <td>Proteomics Standard Initiative Common QUery InterfaCe</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/psygenet2r.html">psygenet2r</a></td>
            <td>Alba Gutierrez-Sacristan </td>
            <td>psygenet2r - An R package for querying PsyGeNET and to perform comorbidity studies in psychiatric disorders</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/pulsedSilac.html">pulsedSilac</a></td>
            <td>Marc Pagès-Gallego </td>
            <td>Analysis of pulsed-SILAC quantitative proteomics data</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/puma.html">puma</a></td>
            <td>Xuejun Liu </td>
            <td>Propagating Uncertainty in Microarray Analysis(including Affymetrix tranditional 3' arrays and exon arrays and Human Transcriptome Array 2.0)</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/PureCN.html">PureCN</a></td>
            <td>Markus Riester </td>
            <td>Copy number calling and SNV classification using targeted short read sequencing</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/pvac.html">pvac</a></td>
            <td>Jun Lu, Pierre R. Bushel </td>
            <td>PCA-based gene filtering for Affymetrix arrays</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/pvca.html">pvca</a></td>
            <td>Jianying LI </td>
            <td>Principal Variance Component Analysis (PVCA)</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/Pviz.html">Pviz</a></td>
            <td>Renan Sauteraud </td>
            <td>Peptide Annotation and Data Visualization using Gviz</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/PWMEnrich.html">PWMEnrich</a></td>
            <td>Robert Stojnic </td>
            <td>PWM enrichment analysis</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/pwOmics.html">pwOmics</a></td>
            <td>Maren Sitte </td>
            <td>Pathway-based data integration of omics data</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/pwrEWAS.html">pwrEWAS</a></td>
            <td>Stefan Graw </td>
            <td>A user-friendly tool for comprehensive power estimation for epigenome wide association studies (EWAS)</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/qckitfastq.html">qckitfastq</a></td>
            <td>August Guang </td>
            <td>FASTQ Quality Control</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/qcmetrics.html">qcmetrics</a></td>
            <td>Laurent Gatto </td>
            <td>A Framework for Quality Control</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/QDNAseq.html">QDNAseq</a></td>
            <td>Daoud Sie </td>
            <td>Quantitative DNA Sequencing for Chromosomal Aberrations</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/qpcrNorm.html">qpcrNorm</a></td>
            <td>Jessica Mar </td>
            <td>Data-driven normalization strategies for high-throughput qPCR data.</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/qpgraph.html">qpgraph</a></td>
            <td>Robert Castelo </td>
            <td>Estimation of genetic and molecular regulatory networks from high-throughput genomics data</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/qPLEXanalyzer.html">qPLEXanalyzer</a></td>
            <td>Ashley Sawle </td>
            <td>Tools for qPLEX-RIME data analysis</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/qrqc.html">qrqc</a></td>
            <td>Vince Buffalo </td>
            <td>Quick Read Quality Control</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/qsea.html">qsea</a></td>
            <td>Matthias Lienhard </td>
            <td>IP-seq data analysis and vizualization</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/qsmooth.html">qsmooth</a></td>
            <td>Stephanie C. Hicks </td>
            <td>Smooth quantile normalization</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/QSutils.html">QSutils</a></td>
            <td>Mercedes Guerrero-Murillo </td>
            <td>Quasispecies Diversity</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/Qtlizer.html">Qtlizer</a></td>
            <td>Matthias Munz </td>
            <td>Qtlizer: comprehensive QTL annotation of GWAS results</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/QUALIFIER.html">QUALIFIER</a></td>
            <td>Mike Jiang </td>
            <td>Quality Control of Gated Flow Cytometry Experiments</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/quantro.html">quantro</a></td>
            <td>Stephanie Hicks </td>
            <td>A test for when to use quantile normalization</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/quantsmooth.html">quantsmooth</a></td>
            <td>Jan Oosting </td>
            <td>Quantile smoothing and genomic visualization of array data</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/QuartPAC.html">QuartPAC</a></td>
            <td>Gregory Ryslik </td>
            <td>Identification of mutational clusters in protein quaternary structures.</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/QuasR.html">QuasR</a></td>
            <td>Michael Stadler </td>
            <td>Quantify and Annotate Short Reads in R</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/QuaternaryProd.html">QuaternaryProd</a></td>
            <td>Carl Tony Fakhry </td>
            <td>Computes the Quaternary Dot Product Scoring Statistic for Signed and Unsigned Causal Graphs</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/QUBIC.html">QUBIC</a></td>
            <td>Yu Zhang </td>
            <td>An R package for qualitative biclustering in support of gene co-expression analyses</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/qusage.html">qusage</a></td>
            <td>Christopher Bolen </td>
            <td>qusage: Quantitative Set Analysis for Gene Expression</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/qvalue.html">qvalue</a></td>
            <td>John D. Storey, Andrew J. Bass </td>
            <td>Q-value estimation for false discovery rate control</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/R3CPET.html">R3CPET</a></td>
            <td>Mohamed Nadhir Djekidel </td>
            <td>3CPET: Finding Co-factor Complexes in Chia-PET experiment using a Hierarchical Dirichlet Process</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/r3Cseq.html">r3Cseq</a></td>
            <td>Supat Thongjuea or </td>
            <td>Analysis of Chromosome Conformation Capture and Next-generation Sequencing (3C-seq)</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/R453Plus1Toolbox.html">R453Plus1Toolbox</a></td>
            <td>Hans-Ulrich Klein </td>
            <td>A package for importing and analyzing data from Roche's Genome Sequencer System</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/R4RNA.html">R4RNA</a></td>
            <td>Daniel Lai </td>
            <td>An R package for RNA visualization and analysis</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/RaggedExperiment.html">RaggedExperiment</a></td>
            <td>Martin Morgan </td>
            <td>Representation of Sparse Experiments and Assays Across Samples</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/rain.html">rain</a></td>
            <td>Paul F. Thaben </td>
            <td>Rhythmicity Analysis Incorporating Non-parametric Methods</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/rama.html">rama</a></td>
            <td>Raphael Gottardo </td>
            <td>Robust Analysis of MicroArrays</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/ramwas.html">ramwas</a></td>
            <td>Andrey A Shabalin </td>
            <td>Fast Methylome-Wide Association Study Pipeline for Enrichment Platforms</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/RandomWalkRestartMH.html">RandomWalkRestartMH</a></td>
            <td>Alberto Valdeolivas Urbelz </td>
            <td>Random walk with restart on multiplex and heterogeneous Networks</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/randPack.html">randPack</a></td>
            <td>Robert Gentleman </td>
            <td>Randomization routines for Clinical Trials</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/RankProd.html">RankProd</a></td>
            <td>Francesco Del Carratore </td>
            <td>Rank Product method for identifying differentially expressed genes with application in meta-analysis</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/RareVariantVis.html">RareVariantVis</a></td>
            <td>Tomasz Stokowy </td>
            <td>A suite for analysis of rare genomic variants in whole genome sequencing data</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/Rariant.html">Rariant</a></td>
            <td>Julian Gehring </td>
            <td>Identification and Assessment of Single Nucleotide Variants through Shifts in Non-Consensus Base Call Frequencies</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/RbcBook1.html">RbcBook1</a></td>
            <td>Vince Carey </td>
            <td>Support for Springer monograph on Bioconductor</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/RBGL.html">RBGL</a></td>
            <td>Bioconductor Package Maintainer </td>
            <td>An interface to the BOOST graph library</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/RBioinf.html">RBioinf</a></td>
            <td>Robert Gentleman </td>
            <td>RBioinf</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/rBiopaxParser.html">rBiopaxParser</a></td>
            <td>Frank Kramer </td>
            <td>Parses BioPax files and represents them in R</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/RBM.html">RBM</a></td>
            <td>Dongmei Li </td>
            <td>RBM: a R package for microarray and RNA-Seq data analysis</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/Rbowtie.html">Rbowtie</a></td>
            <td>Michael Stadler </td>
            <td>R bowtie wrapper</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/Rbowtie2.html">Rbowtie2</a></td>
            <td>Zheng Wei </td>
            <td>An R Wrapper for Bowtie2 and AdapterRemoval</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/rbsurv.html">rbsurv</a></td>
            <td>Soo-heang Eo </td>
            <td>Robust likelihood-based survival modeling with microarray data</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/Rcade.html">Rcade</a></td>
            <td>Jonathan Cairns </td>
            <td>R-based analysis of ChIP-seq And Differential Expression - a tool for integrating a count-based ChIP-seq analysis with differential expression summary data</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/RCAS.html">RCAS</a></td>
            <td>Bora Uyar </td>
            <td>RNA Centric Annotation System</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/RCASPAR.html">RCASPAR</a></td>
            <td>Douaa Mugahid, Lars Kaderali </td>
            <td>A package for survival time prediction based on a piecewise baseline hazard Cox regression model.</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/rcellminer.html">rcellminer</a></td>
            <td>Augustin Luna, Vinodh Rajapakse, Fathi Elloumi </td>
            <td>rcellminer: Molecular Profiles and Drug Response for the NCI-60 Cell Lines</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/rCGH.html">rCGH</a></td>
            <td>Frederic Commo </td>
            <td>Comprehensive Pipeline for Analyzing and Visualizing Array-Based CGH Data</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/Rchemcpp.html">Rchemcpp</a></td>
            <td>Guenter Klambauer </td>
            <td>Similarity measures for chemical compounds</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/RchyOptimyx.html">RchyOptimyx</a></td>
            <td>Adrin Jalali, Nima Aghaeepour </td>
            <td>Optimyzed Cellular Hierarchies for Flow Cytometry</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/RcisTarget.html">RcisTarget</a></td>
            <td>Sara Aibar </td>
            <td>RcisTarget: Identify transcription factor binding motifs enriched on a gene list</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/RCM.html">RCM</a></td>
            <td>Joris Meys </td>
            <td>Fit row-column association models with the negative binomial distribution for the microbiome</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/Rcpi.html">Rcpi</a></td>
            <td>Nan Xiao </td>
            <td>Molecular Informatics Toolkit for Compound-Protein Interaction in Drug Discovery</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/Rcwl.html">Rcwl</a></td>
            <td>Qiang Hu </td>
            <td>Wrap Command Tools and Pipelines Using CWL</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/RcwlPipelines.html">RcwlPipelines</a></td>
            <td>Qiang Hu </td>
            <td>Bioinformatics pipelines based on Rcwl</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/RCy3.html">RCy3</a></td>
            <td>Alex Pico </td>
            <td>Functions to Access and Control Cytoscape</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/RCyjs.html">RCyjs</a></td>
            <td>Paul Shannon </td>
            <td>Display and manipulate graphs in cytoscape.js</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/RDAVIDWebService.html">RDAVIDWebService</a></td>
            <td>Cristobal Fresno </td>
            <td>An R Package for retrieving data from DAVID into R objects using Web Services API.</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/rDGIdb.html">rDGIdb</a></td>
            <td>Thomas Thurnherr </td>
            <td>R Wrapper for DGIdb</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/Rdisop.html">Rdisop</a></td>
            <td>Steffen Neumann </td>
            <td>Decomposition of Isotopic Patterns</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/RDRToolbox.html">RDRToolbox</a></td>
            <td>Christoph Bartenhagen </td>
            <td>A package for nonlinear dimension reduction with Isomap and LLE.</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/ReactomeGSA.html">ReactomeGSA</a></td>
            <td>Johannes Griss </td>
            <td>Client for the Reactome Analysis Service for comparative multi-omics gene set analysis</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/ReactomePA.html">ReactomePA</a></td>
            <td>Guangchuang Yu </td>
            <td>Reactome Pathway Analysis</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/readat.html">readat</a></td>
            <td>Richard Cotton </td>
            <td>Functionality to Read and Manipulate SomaLogic ADAT files</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/ReadqPCR.html">ReadqPCR</a></td>
            <td>James Perkins </td>
            <td>Read qPCR data</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/reb.html">reb</a></td>
            <td>Karl J. Dykema </td>
            <td>Regional Expression Biases</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/REBET.html">REBET</a></td>
            <td>Bill Wheeler </td>
            <td>The subREgion-based BurdEn Test (REBET)</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/recount.html">recount</a></td>
            <td>Leonardo Collado-Torres </td>
            <td>Explore and download data from the recount project</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/recoup.html">recoup</a></td>
            <td>Panagiotis Moulos </td>
            <td>An R package for the creation of complex genomic profile plots</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/RedeR.html">RedeR</a></td>
            <td>Mauro Castro </td>
            <td>Interactive visualization and manipulation of nested networks</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/REDseq.html">REDseq</a></td>
            <td>Lihua Julie Zhu </td>
            <td>Analysis of high-throughput sequencing data processed by restriction enzyme digestion</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/RefNet.html">RefNet</a></td>
            <td>Paul Shannon </td>
            <td>A queryable collection of molecular interactions, from many sources</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/RefPlus.html">RefPlus</a></td>
            <td>Kai-Ming Chang </td>
            <td>A function set for the Extrapolation Strategy (RMA+) and Extrapolation Averaging (RMA++) methods.</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/regioneR.html">regioneR</a></td>
            <td>Bernat Gel </td>
            <td>Association analysis of genomic regions based on permutation tests</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/regionReport.html">regionReport</a></td>
            <td>Leonardo Collado-Torres </td>
            <td>Generate HTML or PDF reports for a set of genomic regions or DESeq2/edgeR results</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/regsplice.html">regsplice</a></td>
            <td>Lukas M. Weber </td>
            <td>L1-regularization based methods for detection of differential splicing</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/REMP.html">REMP</a></td>
            <td>Yinan Zheng </td>
            <td>Repetitive Element Methylation Prediction</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/Repitools.html">Repitools</a></td>
            <td>Mark Robinson </td>
            <td>Epigenomic tools</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/ReportingTools.html">ReportingTools</a></td>
            <td>Jason A. Hackney, Gabriel Becker, Jessica L. Larson </td>
            <td>Tools for making reports in various formats</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/RepViz.html">RepViz</a></td>
            <td>Thomas Faux, Asta Laiho </td>
            <td>Replicate oriented Visualization of a genomic region</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/ReQON.html">ReQON</a></td>
            <td>Christopher Cabanski </td>
            <td>Recalibrating Quality Of Nucleotides</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/restfulSE.html">restfulSE</a></td>
            <td>Shweta Gopaulakrishnan </td>
            <td>Access matrix-like HDF5 server content or BigQuery content through a SummarizedExperiment interface</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/rexposome.html">rexposome</a></td>
            <td>Carles Hernandez-Ferrer </td>
            <td>Exposome exploration and outcome data analysis</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/rfPred.html">rfPred</a></td>
            <td>Hugo Varet </td>
            <td>Assign rfPred functional prediction scores to a missense variants list</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/rGADEM.html">rGADEM</a></td>
            <td>Arnaud Droit </td>
            <td>de novo motif discovery</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/RGalaxy.html">RGalaxy</a></td>
            <td>Bioconductor Package Maintainer </td>
            <td>Make an R function available in the Galaxy web platform</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/Rgin.html">Rgin</a></td>
            <td>Hector Climente-Gonzalez </td>
            <td>gin in R</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/RGMQL.html">RGMQL</a></td>
            <td>Simone Pallotta </td>
            <td>GenoMetric Query Language for R/Bioconductor</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/RGraph2js.html">RGraph2js</a></td>
            <td>Stephane Cano </td>
            <td>Convert a Graph into a D3js Script</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/Rgraphviz.html">Rgraphviz</a></td>
            <td>Kasper Daniel Hansen </td>
            <td>Provides plotting capabilities for R graph objects</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/rGREAT.html">rGREAT</a></td>
            <td>Zuguang Gu </td>
            <td>Client for GREAT Analysis</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/RGSEA.html">RGSEA</a></td>
            <td>Chengcheng Ma </td>
            <td>Random Gene Set Enrichment Analysis</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/rgsepd.html">rgsepd</a></td>
            <td>Karl Stamm </td>
            <td>Gene Set Enrichment / Projection Displays</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/rhdf5.html">rhdf5</a></td>
            <td>Mike Smith </td>
            <td>R Interface to HDF5</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/rhdf5client.html">rhdf5client</a></td>
            <td>Vincent Carey </td>
            <td>Access HDF5 content from h5serv</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/Rhdf5lib.html">Rhdf5lib</a></td>
            <td>Mike Smith </td>
            <td>hdf5 library as an R package</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/Rhisat2.html">Rhisat2</a></td>
            <td>Charlotte Soneson </td>
            <td>R Wrapper for HISAT2 Aligner</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/Rhtslib.html">Rhtslib</a></td>
            <td>Bioconductor Package Maintainer </td>
            <td>HTSlib high-throughput sequencing library as an R package</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/rHVDM.html">rHVDM</a></td>
            <td>Martino Barenco</td>
            <td>Hidden Variable Dynamic Modeling</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/RiboProfiling.html">RiboProfiling</a></td>
            <td>A. Popa </td>
            <td>Ribosome Profiling Data Analysis: from BAM to Data Representation and Interpretation</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/riboSeqR.html">riboSeqR</a></td>
            <td>Thomas J. Hardcastle </td>
            <td>Analysis of sequencing data from ribosome profiling experiments</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/RImmPort.html">RImmPort</a></td>
            <td>Zicheng Hu, Ravi Shankar </td>
            <td>RImmPort: Enabling Ready-for-analysis Immunology Research Data</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/Ringo.html">Ringo</a></td>
            <td>J. Toedling </td>
            <td>R Investigation of ChIP-chip Oligoarrays</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/RIPSeeker.html">RIPSeeker</a></td>
            <td>Yue Li </td>
            <td>RIPSeeker: a statistical package for identifying protein-associated transcripts from RIP-seq experiments</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/Risa.html">Risa</a></td>
            <td>Alejandra Gonzalez-Beltran </td>
            <td>Converting experimental metadata from ISA-tab into Bioconductor data structures</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/RITAN.html">RITAN</a></td>
            <td>Michael Zimmermann </td>
            <td>Rapid Integration of Term Annotation and Network resources</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/RIVER.html">RIVER</a></td>
            <td>Yungil Kim </td>
            <td>R package for RIVER (RNA-Informed Variant Effect on Regulation)</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/RJMCMCNucleosomes.html">RJMCMCNucleosomes</a></td>
            <td>Astrid Deschênes </td>
            <td>Bayesian hierarchical model for genome-wide nucleosome positioning with high-throughput short-read data (MNase-Seq)</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/RLMM.html">RLMM</a></td>
            <td>Nusrat Rabbee </td>
            <td>A Genotype Calling Algorithm for Affymetrix SNP Arrays</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/Rmagpie.html">Rmagpie</a></td>
            <td>Camille Maumet </td>
            <td>MicroArray Gene-expression-based Program In Error rate estimation</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/RMassBank.html">RMassBank</a></td>
            <td>RMassBank at Eawag </td>
            <td>Workflow to process tandem MS files and build MassBank records</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/rmelting.html">rmelting</a></td>
            <td>J. Aravind </td>
            <td>R Interface to MELTING 5</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/RmiR.html">RmiR</a></td>
            <td>Francesco Favero </td>
            <td>Package to work with miRNAs and miRNA targets with R</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/Rmmquant.html">Rmmquant</a></td>
            <td>Zytnicki Matthias </td>
            <td>RNA-Seq multi-mapping Reads Quantification Tool</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/RNAdecay.html">RNAdecay</a></td>
            <td>Reed Sorenson </td>
            <td>Maximum Likelihood Decay Modeling of RNA Degradation Data</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/RNAinteract.html">RNAinteract</a></td>
            <td>Bernd Fischer </td>
            <td>Estimate Pairwise Interactions from multidimensional features</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/RNAither.html">RNAither</a></td>
            <td>Lars Kaderali </td>
            <td>Statistical analysis of high-throughput RNAi screens</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/RNAmodR.html">RNAmodR</a></td>
            <td>Felix G.M. Ernst </td>
            <td>Detection of post-transcriptional modifications in high throughput sequencing data</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/RNAmodR.AlkAnilineSeq.html">RNAmodR.AlkAnilineSeq</a></td>
            <td>Felix G.M. Ernst </td>
            <td>Detection of m7G, m3C and D modification by AlkAnilineSeq</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/RNAmodR.ML.html">RNAmodR.ML</a></td>
            <td>Felix G.M. Ernst </td>
            <td>Detecting patterns of post-transcriptional modifications using machine learning</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/RNAmodR.RiboMethSeq.html">RNAmodR.RiboMethSeq</a></td>
            <td>Felix G.M. Ernst </td>
            <td>Detection of 2'-O methylations by RiboMethSeq</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/RNAprobR.html">RNAprobR</a></td>
            <td>Nikos Sidiropoulos </td>
            <td>An R package for analysis of massive parallel sequencing based RNA structure probing data</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/RNAsense.html">RNAsense</a></td>
            <td>Marcus Rosenblatt </td>
            <td>Analysis of Time-Resolved RNA-Seq Data</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/rnaseqcomp.html">rnaseqcomp</a></td>
            <td>Mingxiang Teng </td>
            <td>Benchmarks for RNA-seq Quantification Pipelines</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/rnaSeqMap.html">rnaSeqMap</a></td>
            <td>Michal Okoniewski </td>
            <td>rnaSeq secondary analyses</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/RNASeqPower.html">RNASeqPower</a></td>
            <td>Terry M Therneau </td>
            <td>Sample size for RNAseq studies</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/RNASeqR.html">RNASeqR</a></td>
            <td>Kuan-Hao Chao </td>
            <td>RNASeqR: an R package for automated two-group RNA-Seq analysis workflow</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/RnaSeqSampleSize.html">RnaSeqSampleSize</a></td>
            <td>Shilin Zhao </td>
            <td>RnaSeqSampleSize</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/RnBeads.html">RnBeads</a></td>
            <td>Fabian Mueller </td>
            <td>RnBeads</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/Rnits.html">Rnits</a></td>
            <td>Dipen P. Sangurdekar </td>
            <td>R Normalization and Inference of Time Series data</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/roar.html">roar</a></td>
            <td>Elena Grassi </td>
            <td>Identify differential APA usage from RNA-seq alignments</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/ROC.html">ROC</a></td>
            <td>Vince Carey </td>
            <td>utilities for ROC, with microarray focus</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/Roleswitch.html">Roleswitch</a></td>
            <td>Yue Li </td>
            <td>Infer miRNA-mRNA interactions using paired expression data from a single sample</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/rols.html">rols</a></td>
            <td>Laurent Gatto </td>
            <td>An R interface to the Ontology Lookup Service</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/ROntoTools.html">ROntoTools</a></td>
            <td>Calin Voichita </td>
            <td>R Onto-Tools suite</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/ropls.html">ropls</a></td>
            <td>Etienne A. Thevenot </td>
            <td>PCA, PLS(-DA) and OPLS(-DA) for multivariate analysis and feature selection of omics data</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/ROTS.html">ROTS</a></td>
            <td>Tomi Suomi </td>
            <td>Reproducibility-Optimized Test Statistic</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/RPA.html">RPA</a></td>
            <td>Leo Lahti </td>
            <td>RPA: Robust Probabilistic Averaging for probe-level analysis</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/RProtoBufLib.html">RProtoBufLib</a></td>
            <td>Mike Jiang </td>
            <td>C++ headers and static libraries of Protocol buffers</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/RpsiXML.html">RpsiXML</a></td>
            <td>Jitao David Zhang </td>
            <td>R interface to PSI-MI 2.5 files</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/rpx.html">rpx</a></td>
            <td>Laurent Gatto </td>
            <td>R Interface to the ProteomeXchange Repository</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/Rqc.html">Rqc</a></td>
            <td>Welliton Souza </td>
            <td>Quality Control Tool for High-Throughput Sequencing Data</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/rqt.html">rqt</a></td>
            <td>Ilya Y. Zhbannikov </td>
            <td>rqt: utilities for gene-level meta-analysis</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/rqubic.html">rqubic</a></td>
            <td>Jitao David Zhang </td>
            <td>Qualitative biclustering algorithm for expression data analysis in R</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/rRDP.html">rRDP</a></td>
            <td>Michael Hahsler </td>
            <td>Interface to the RDP Classifier</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/RRHO.html">RRHO</a></td>
            <td>Jonathan Rosenblatt </td>
            <td>Inference on agreement between ordered lists</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/Rsamtools.html">Rsamtools</a></td>
            <td>Bioconductor Package Maintainer </td>
            <td>Binary alignment (BAM), FASTA, variant call (BCF), and tabix file import</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/rsbml.html">rsbml</a></td>
            <td>Michael Lawrence </td>
            <td>R support for SBML, using libsbml</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/rScudo.html">rScudo</a></td>
            <td>Matteo Ciciani </td>
            <td>Signature-based Clustering for Diagnostic Purposes</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/RSeqAn.html">RSeqAn</a></td>
            <td>August Guang </td>
            <td>R SeqAn</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/Rsubread.html">Rsubread</a></td>
            <td>Wei Shi, Yang Liao and Gordon K Smyth </td>
            <td>Subread Sequence Alignment and Counting for R</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/RSVSim.html">RSVSim</a></td>
            <td>Christoph Bartenhagen </td>
            <td>RSVSim: an R/Bioconductor package for the simulation of structural variations</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/rTANDEM.html">rTANDEM</a></td>
            <td>Frederic Fournier </td>
            <td>Interfaces the tandem protein identification algorithm in R</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/RTCA.html">RTCA</a></td>
            <td>Jitao David Zhang </td>
            <td>Open-source toolkit to analyse data from xCELLigence System (RTCA)</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/RTCGA.html">RTCGA</a></td>
            <td>Marcin Kosinski </td>
            <td>The Cancer Genome Atlas Data Integration</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/RTCGAToolbox.html">RTCGAToolbox</a></td>
            <td>Marcel Ramos </td>
            <td>A new tool for exporting TCGA Firehose data</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/RTN.html">RTN</a></td>
            <td>Mauro Castro </td>
            <td>RTN: Reconstruction of Transcriptional regulatory Networks and analysis of regulons</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/RTNduals.html">RTNduals</a></td>
            <td>Mauro Castro, Clarice Groeneveld </td>
            <td>Analysis of co-regulation and inference of 'dual regulons'</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/RTNsurvival.html">RTNsurvival</a></td>
            <td>Clarice Groeneveld, Mauro A. A. Castro </td>
            <td>Survival analysis using transcriptional networks inferred by the RTN package</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/RTopper.html">RTopper</a></td>
            <td>Luigi Marchionni </td>
            <td>This package is designed to perform Gene Set Analysis across multiple genomic platforms</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/rtracklayer.html">rtracklayer</a></td>
            <td>Michael Lawrence </td>
            <td>R interface to genome annotation files and the UCSC genome browser</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/Rtreemix.html">Rtreemix</a></td>
            <td>Jasmina Bogojeska </td>
            <td>Rtreemix: Mutagenetic trees mixture models.</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/rTRM.html">rTRM</a></td>
            <td>Diego Diez </td>
            <td>Identification of transcriptional regulatory modules from PPI networks</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/rTRMui.html">rTRMui</a></td>
            <td>Diego Diez </td>
            <td>A shiny user interface for rTRM</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/runibic.html">runibic</a></td>
            <td>Patryk Orzechowski </td>
            <td>runibic: row-based biclustering algorithm for analysis of gene expression data in R</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/RUVcorr.html">RUVcorr</a></td>
            <td>Saskia Freytag </td>
            <td>Removal of unwanted variation for gene-gene correlations and related analysis</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/RUVnormalize.html">RUVnormalize</a></td>
            <td>Laurent Jacob </td>
            <td>RUV for normalization of expression array data</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/RUVSeq.html">RUVSeq</a></td>
            <td>Davide Risso </td>
            <td>Remove Unwanted Variation from RNA-Seq Data</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/RVS.html">RVS</a></td>
            <td>Thomas Sherman </td>
            <td>Computes estimates of the probability of related individuals sharing a rare variant</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/rWikiPathways.html">rWikiPathways</a></td>
            <td>Egon Willighagen </td>
            <td>rWikiPathways - R client library for the WikiPathways API</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/S4Vectors.html">S4Vectors</a></td>
            <td>Bioconductor Package Maintainer </td>
            <td>Foundation of vector-like and list-like containers in Bioconductor</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/safe.html">safe</a></td>
            <td>William T. Barry </td>
            <td>Significance Analysis of Function and Expression</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/sagenhaft.html">sagenhaft</a></td>
            <td>Tim Beissbarth </td>
            <td>Collection of functions for reading and comparing SAGE libraries</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/SAGx.html">SAGx</a></td>
            <td>Per Broberg, </td>
            <td>Statistical Analysis of the GeneChip</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/SAIGEgds.html">SAIGEgds</a></td>
            <td>Xiuwen Zheng </td>
            <td>Scalable Implementation of Generalized mixed models using GDS files in PheWAS</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/samExploreR.html">samExploreR</a></td>
            <td>shailesh tripathi </td>
            <td>samExploreR package: high-performance read summarisation to count vectors with avaliability of sequencing depth reduction simulation</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/sampleClassifier.html">sampleClassifier</a></td>
            <td>Khadija El Amrani </td>
            <td>Sample Classifier</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/SamSPECTRAL.html">SamSPECTRAL</a></td>
            <td>Habil </td>
            <td>Identifies cell population in flow cytometry data.</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/sangerseqR.html">sangerseqR</a></td>
            <td>Jonathon Hill </td>
            <td>Tools for Sanger Sequencing Data in R</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/SANTA.html">SANTA</a></td>
            <td>Alex J. Cornish </td>
            <td>Spatial Analysis of Network Associations</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/sapFinder.html">sapFinder</a></td>
            <td>Shaohang Xu, Bo Wen </td>
            <td>A package for variant peptides detection and visualization in shotgun proteomics.</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/savR.html">savR</a></td>
            <td>R. Brent Calder </td>
            <td>Parse and analyze Illumina SAV files</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/SBGNview.html">SBGNview</a></td>
            <td>Xiaoxi Dong </td>
            <td>Overlay omics data onto SBGN pathway diagram</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/SBMLR.html">SBMLR</a></td>
            <td>Tomas Radivoyevitch </td>
            <td>SBML-R Interface and Analysis Tools</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/SC3.html">SC3</a></td>
            <td>Vladimir Kiselev </td>
            <td>Single-Cell Consensus Clustering</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/Scale4C.html">Scale4C</a></td>
            <td>Carolin Walter </td>
            <td>Scale4C: an R/Bioconductor package for scale-space transformation of 4C-seq data</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/scAlign.html">scAlign</a></td>
            <td>Nelson Johansen </td>
            <td>An alignment and integration method for single cell genomics</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/SCAN.UPC.html">SCAN.UPC</a></td>
            <td>Stephen R. Piccolo </td>
            <td>Single-channel array normalization (SCAN) and Universal exPression Codes (UPC)</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/SCANVIS.html">SCANVIS</a></td>
            <td>Phaedra Agius </td>
            <td>SCANVIS - a tool for SCoring, ANnotating and VISualizing splice junctions</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/scater.html">scater</a></td>
            <td>Davis McCarthy </td>
            <td>Single-Cell Analysis Toolkit for Gene Expression Data in R</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/scBFA.html">scBFA</a></td>
            <td>Ruoxin Li </td>
            <td>A dimensionality reduction tool using gene detection pattern to mitigate noisy expression profile of scRNA-seq</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/SCBN.html">SCBN</a></td>
            <td>Yan Zhou </td>
            <td>A statistical normalization method and differential expression analysis for RNA-seq data between different species</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/scDblFinder.html">scDblFinder</a></td>
            <td>Pierre-Luc Germain </td>
            <td>scDblFinder</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/scDD.html">scDD</a></td>
            <td>Keegan Korthauer </td>
            <td>Mixture modeling of single-cell RNA-seq data to identify genes with differential distributions</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/scde.html">scde</a></td>
            <td>Jean Fan </td>
            <td>Single Cell Differential Expression</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/scds.html">scds</a></td>
            <td>Dennis Kostka </td>
            <td>In-Silico Annotation of Doublets for Single Cell RNA Sequencing Data</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/scFeatureFilter.html">scFeatureFilter</a></td>
            <td>Guillaume Devailly </td>
            <td>A correlation-based method for quality filtering of single-cell RNAseq data</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/scfind.html">scfind</a></td>
            <td>Vladimir Kiselev </td>
            <td>A search tool for single cell RNA-seq data by gene lists</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/scGPS.html">scGPS</a></td>
            <td>Quan Nguyen </td>
            <td>A complete analysis of single cell subpopulations, from identifying subpopulations to analysing their relationship (scGPS = single cell Global Predictions of Subpopulation)</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/schex.html">schex</a></td>
            <td>Saskia Freytag </td>
            <td>Hexbin plots for single cell omics data</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/ScISI.html">ScISI</a></td>
            <td>Tony Chiang </td>
            <td>In Silico Interactome</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/scmap.html">scmap</a></td>
            <td>Vladimir Kiselev </td>
            <td>A tool for unsupervised projection of single cell RNA-seq data</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/scMerge.html">scMerge</a></td>
            <td>Kevin Wang </td>
            <td>scMerge: Merging multiple batches of scRNA-seq data</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/scmeth.html">scmeth</a></td>
            <td>Divy Kangeyan </td>
            <td>Functions to conduct quality control analysis in methylation data</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/SCnorm.html">SCnorm</a></td>
            <td>Rhonda Bacher </td>
            <td>Normalization of single cell RNA-seq data</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/scone.html">scone</a></td>
            <td>Davide Risso </td>
            <td>Single Cell Overview of Normalized Expression data</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/Sconify.html">Sconify</a></td>
            <td>Tyler J Burns </td>
            <td>A toolkit for performing KNN-based statistics for flow and mass cytometry data</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/scoreInvHap.html">scoreInvHap</a></td>
            <td>Carlos Ruiz </td>
            <td>Get inversion status in predefined regions</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/scPCA.html">scPCA</a></td>
            <td>Philippe Boileau </td>
            <td>Sparse Contrastive Principal Component Analysis</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/scPipe.html">scPipe</a></td>
            <td>Luyi Tian </td>
            <td>pipeline for single cell RNA-seq data analysis</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/scran.html">scran</a></td>
            <td>Aaron Lun </td>
            <td>Methods for Single-Cell RNA-Seq Data Analysis</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/scRecover.html">scRecover</a></td>
            <td>Zhun Miao </td>
            <td>scRecover for imputation of single-cell RNA-seq data</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/scruff.html">scruff</a></td>
            <td>Zhe Wang </td>
            <td>Single Cell RNA-Seq UMI Filtering Facilitator (scruff)</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/scsR.html">scsR</a></td>
            <td>Andrea Franceschini, Roger Meier, Christian von Mering </td>
            <td>SiRNA correction for seed mediated off-target effect</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/scTensor.html">scTensor</a></td>
            <td>Koki Tsuyuzaki </td>
            <td>Detection of cell-cell interaction from single-cell RNA-seq dataset by tensor decomposition</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/scTGIF.html">scTGIF</a></td>
            <td>Koki Tsuyuzaki </td>
            <td>Cell type annotation for unannotated single-cell RNA-Seq data</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/SDAMS.html">SDAMS</a></td>
            <td>Yuntong Li </td>
            <td>Differential Abundant Analysis for Metabolomics and Proteomics Data</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/segmentSeq.html">segmentSeq</a></td>
            <td>Thomas J. Hardcastle </td>
            <td>Methods for identifying small RNA loci from high-throughput sequencing data</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/SELEX.html">SELEX</a></td>
            <td>Harmen Bussemaker </td>
            <td>Functions for analyzing SELEX-seq data</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/SemDist.html">SemDist</a></td>
            <td>Ian Gonzalez </td>
            <td>Information Accretion-based Function Predictor Evaluation</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/semisup.html">semisup</a></td>
            <td>Armin Rauschenberger </td>
            <td>Semi-Supervised Mixture Model</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/SEPA.html">SEPA</a></td>
            <td>Zhicheng Ji </td>
            <td>SEPA</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/SEPIRA.html">SEPIRA</a></td>
            <td>Yuting Chen </td>
            <td>Systems EPigenomics Inference of Regulatory Activity</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/seq2pathway.html">seq2pathway</a></td>
            <td>Xinan Yang with contribution from Zhezhen Wang </td>
            <td>a novel tool for functional gene-set (or termed as pathway) analysis of next-generation sequencing data</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/SeqArray.html">SeqArray</a></td>
            <td>Xiuwen Zheng </td>
            <td>Data Management of Large-scale Whole-genome Sequence Variant Calls</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/seqbias.html">seqbias</a></td>
            <td>Daniel Jones </td>
            <td>Estimation of per-position bias in high-throughput sequencing data</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/seqCAT.html">seqCAT</a></td>
            <td>Erik Fasterius </td>
            <td>High Throughput Sequencing Cell Authentication Toolkit</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/seqCNA.html">seqCNA</a></td>
            <td>David Mosen-Ansorena </td>
            <td>Copy number analysis of high-throughput sequencing cancer data</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/seqcombo.html">seqcombo</a></td>
            <td>Guangchuang Yu </td>
            <td>Visualization Tool for Sequence Recombination and Reassortment</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/SeqGSEA.html">SeqGSEA</a></td>
            <td>Xi Wang </td>
            <td>Gene Set Enrichment Analysis (GSEA) of RNA-Seq Data: integrating differential expression and splicing</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/seqLogo.html">seqLogo</a></td>
            <td>Oliver Bembom </td>
            <td>Sequence logos for DNA sequence alignments</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/seqPattern.html">seqPattern</a></td>
            <td>Vanja Haberle </td>
            <td>Visualising oligonucleotide patterns and motif occurrences across a set of sorted sequences</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/seqplots.html">seqplots</a></td>
            <td>Przemyslaw Stempor </td>
            <td>An interactive tool for visualizing NGS signals and sequence motif densities along genomic features using average plots and heatmaps</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/seqsetvis.html">seqsetvis</a></td>
            <td>Joseph R Boyd </td>
            <td>Set Based Visualizations for Next-Gen Sequencing Data</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/SeqSQC.html">SeqSQC</a></td>
            <td>Qian Liu </td>
            <td>A bioconductor package for sample quality check with next generation sequencing data</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/seqTools.html">seqTools</a></td>
            <td>Wolfgang Kaisers </td>
            <td>Analysis of nucleotide, sequence and quality content on fastq files</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/SeqVarTools.html">SeqVarTools</a></td>
            <td>Stephanie M. Gogarten </td>
            <td>Tools for variant data</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/sesame.html">sesame</a></td>
            <td>Wanding Zhou </td>
            <td>Tools For Analyzing Illumina Infinium DNA Methylation Arrays</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/SEtools.html">SEtools</a></td>
            <td>Pierre-Luc Germain </td>
            <td>SEtools: tools for working with SummarizedExperiment</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/sevenbridges.html">sevenbridges</a></td>
            <td>Nan Xiao </td>
            <td>Seven Bridges Platform API Client and Common Workflow Language Tool Builder in R</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/sevenC.html">sevenC</a></td>
            <td>Jonas Ibn-Salem </td>
            <td>Computational Chromosome Conformation Capture by Correlation of ChIP-seq at CTCF motifs</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/SGSeq.html">SGSeq</a></td>
            <td>Leonard Goldstein </td>
            <td>Splice event prediction and quantification from RNA-seq data</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/SharedObject.html">SharedObject</a></td>
            <td>Jiefei Wang </td>
            <td>Sharing R objects across multiple R processes without memory duplication</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/shinyMethyl.html">shinyMethyl</a></td>
            <td>Jean-Philippe Fortin </td>
            <td>Interactive visualization for Illumina methylation arrays</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/shinyTANDEM.html">shinyTANDEM</a></td>
            <td>Frederic Fournier </td>
            <td>Provides a GUI for rTANDEM</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/ShortRead.html">ShortRead</a></td>
            <td>Bioconductor Package Maintainer </td>
            <td>FASTQ input and manipulation</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/SIAMCAT.html">SIAMCAT</a></td>
            <td>Konrad Zych </td>
            <td>Statistical Inference of Associations between Microbial Communities And host phenoTypes</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/SICtools.html">SICtools</a></td>
            <td>Xiaobin Xing </td>
            <td>Find SNV/Indel differences between two bam files with near relationship</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/sigaR.html">sigaR</a></td>
            <td>Wessel N. van Wieringen </td>
            <td>Statistics for Integrative Genomics Analyses in R</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/SigCheck.html">SigCheck</a></td>
            <td>Rory Stark </td>
            <td>Check a gene signature's prognostic performance against random signatures, known signatures, and permuted data/metadata</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/sigFeature.html">sigFeature</a></td>
            <td>Pijush Das Developer </td>
            <td>sigFeature: Significant feature selection using SVM-RFE &amp; t-statistic</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/SigFuge.html">SigFuge</a></td>
            <td>Patrick Kimes </td>
            <td>SigFuge</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/siggenes.html">siggenes</a></td>
            <td>Holger Schwender </td>
            <td>Multiple Testing using SAM and Efron's Empirical Bayes Approaches</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/sights.html">sights</a></td>
            <td>Elika Garg </td>
            <td>Statistics and dIagnostic Graphs for HTS</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/signatureSearch.html">signatureSearch</a></td>
            <td>Yuzhu Duan </td>
            <td>Environment for Gene Expression Searching Combined with Functional Enrichment Analysis</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/signeR.html">signeR</a></td>
            <td>Renan Valieris </td>
            <td>Empirical Bayesian approach to mutational signature discovery</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/signet.html">signet</a></td>
            <td>Alexandre Gouy </td>
            <td>signet: Selection Inference in Gene NETworks</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/sigPathway.html">sigPathway</a></td>
            <td>Weil Lai </td>
            <td>Pathway Analysis</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/SigsPack.html">SigsPack</a></td>
            <td>Franziska Schumann </td>
            <td>Mutational Signature Estimation for Single Samples</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/sigsquared.html">sigsquared</a></td>
            <td>UnJin Lee </td>
            <td>Gene signature generation for functionally validated signaling pathways</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/SIM.html">SIM</a></td>
            <td>Renee X. de Menezes </td>
            <td>Integrated Analysis on two human genomic datasets</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/SIMAT.html">SIMAT</a></td>
            <td>M. R. Nezami Ranjbar </td>
            <td>GC-SIM-MS data processing and alaysis tool</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/SimBindProfiles.html">SimBindProfiles</a></td>
            <td>Bettina Fischer </td>
            <td>Similar Binding Profiles</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/SIMD.html">SIMD</a></td>
            <td>Jiadi Zhu </td>
            <td>Statistical Inferences with MeDIP-seq Data (SIMD) to infer the methylation level for each CpG site</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/similaRpeak.html">similaRpeak</a></td>
            <td>Astrid Deschenes </td>
            <td>Metrics to estimate a level of similarity between two ChIP-Seq profiles</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/SIMLR.html">SIMLR</a></td>
            <td>Luca De Sano </td>
            <td>Single-cell Interpretation via Multi-kernel LeaRning (SIMLR)</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/simpleaffy.html">simpleaffy</a></td>
            <td>Crispin Miller </td>
            <td>Very simple high level analysis of Affymetrix data</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/simulatorZ.html">simulatorZ</a></td>
            <td>Yuqing Zhang </td>
            <td>Simulator for Collections of Independent Genomic Data Sets</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/sincell.html">sincell</a></td>
            <td>Miguel Julia, Antonio Rausell</td>
            <td>R package for the statistical assessment of cell state hierarchies from single-cell RNA-seq data</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/SingleCellExperiment.html">SingleCellExperiment</a></td>
            <td>Davide Risso </td>
            <td>S4 Classes for Single Cell Data</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/singleCellTK.html">singleCellTK</a></td>
            <td>David Jenkins </td>
            <td>Interactive Analysis of Single Cell RNA-Seq Data</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/SingleR.html">SingleR</a></td>
            <td>Aaron Lun </td>
            <td>Reference-Based Single-Cell RNA-Seq Annotation</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/singscore.html">singscore</a></td>
            <td>Dharmesh D. Bhuva </td>
            <td>Rank-based single-sample gene set scoring method</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/SISPA.html">SISPA</a></td>
            <td>Bhakti Dwivedi </td>
            <td>SISPA: Method for Sample Integrated Set Profile Analysis</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/sitePath.html">sitePath</a></td>
            <td>Chengyang Ji </td>
            <td>Detection of sites with fixation of amino acid substitutions in protein evolution</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/sizepower.html">sizepower</a></td>
            <td>Weiliang Qiu </td>
            <td>Sample Size and Power Calculation in Micorarray Studies</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/skewr.html">skewr</a></td>
            <td>Ryan Putney </td>
            <td>Visualize Intensities Produced by Illumina's Human Methylation 450k BeadChip</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/slalom.html">slalom</a></td>
            <td>Davis McCarthy </td>
            <td>Factorial Latent Variable Modeling of Single-Cell RNA-Seq Data</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/SLGI.html">SLGI</a></td>
            <td>Nolwenn Le Meur </td>
            <td>Synthetic Lethal Genetic Interaction</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/slingshot.html">slingshot</a></td>
            <td>Kelly Street </td>
            <td>Tools for ordering single-cell sequencing</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/slinky.html">slinky</a></td>
            <td>Eric J. Kort </td>
            <td>Putting the fun in LINCS L1000 data analysis</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/SLqPCR.html">SLqPCR</a></td>
            <td>Matthias Kohl </td>
            <td>Functions for analysis of real-time quantitative PCR data at SIRS-Lab GmbH</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/SMAD.html">SMAD</a></td>
            <td>Qingzhou Zhang </td>
            <td>Statistical Modelling of AP-MS Data (SMAD)</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/SMAP.html">SMAP</a></td>
            <td>Robin Andersson </td>
            <td>A Segmental Maximum A Posteriori Approach to Array-CGH Copy Number Profiling</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/SMITE.html">SMITE</a></td>
            <td>Neil Ari Wijetunga, Andrew Damon Johnston </td>
            <td>Significance-based Modules Integrating the Transcriptome and Epigenome</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/SNAGEE.html">SNAGEE</a></td>
            <td>David Venet </td>
            <td>Signal-to-Noise applied to Gene Expression Experiments</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/snapCGH.html">snapCGH</a></td>
            <td>John Marioni </td>
            <td>Segmentation, normalisation and processing of aCGH data</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/snm.html">snm</a></td>
            <td>John D. Storey </td>
            <td>Supervised Normalization of Microarrays</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/SNPchip.html">SNPchip</a></td>
            <td>Robert Scharpf </td>
            <td>Visualizations for copy number alterations</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/SNPediaR.html">SNPediaR</a></td>
            <td>David Montaner </td>
            <td>Query data from SNPedia</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/SNPhood.html">SNPhood</a></td>
            <td>Christian Arnold </td>
            <td>SNPhood: Investigate, quantify and visualise the epigenomic neighbourhood of SNPs using NGS data</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/SNPRelate.html">SNPRelate</a></td>
            <td>Xiuwen Zheng </td>
            <td>Parallel Computing Toolset for Relatedness and Principal Component Analysis of SNP Data</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/snpStats.html">snpStats</a></td>
            <td>David Clayton </td>
            <td>SnpMatrix and XSnpMatrix classes and methods</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/soGGi.html">soGGi</a></td>
            <td>Tom Carroll </td>
            <td>Visualise ChIP-seq, MNase-seq and motif occurrence as aggregate plots Summarised Over Grouped Genomic Intervals</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/sojourner.html">sojourner</a></td>
            <td>Sojourner Developer </td>
            <td>sojourner: An R package for statistical analysis of single molecule trajectories</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/SomaticSignatures.html">SomaticSignatures</a></td>
            <td>Julian Gehring </td>
            <td>Somatic Signatures</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/SpacePAC.html">SpacePAC</a></td>
            <td>Gregory Ryslik </td>
            <td>Identification of Mutational Clusters in 3D Protein Space via Simulation.</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/Spaniel.html">Spaniel</a></td>
            <td>Rachel Queen </td>
            <td>Spatial Transcriptomics Analysis</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/sparseDOSSA.html">sparseDOSSA</a></td>
            <td>Boyu Ren, Emma Schwager, George Weingart</td>
            <td>Sparse Data Observations for Simulating Synthetic Abundance</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/sparsenetgls.html">sparsenetgls</a></td>
            <td>Irene Zeng </td>
            <td>Using Gaussian graphical structue learning estimation in generalized least squared regression for multivariate normal regression</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/SparseSignatures.html">SparseSignatures</a></td>
            <td>Luca De Sano </td>
            <td>SparseSignatures</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/SpatialCPie.html">SpatialCPie</a></td>
            <td>Joseph Bergenstraahle </td>
            <td>Cluster analysis of Spatial Transcriptomics data</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/specL.html">specL</a></td>
            <td>Christian Panse </td>
            <td>specL - Prepare Peptide Spectrum Matches for Use in Targeted Proteomics</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/SpeCond.html">SpeCond</a></td>
            <td>Florence Cavalli </td>
            <td>Condition specific detection from expression data</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/SpectralTAD.html">SpectralTAD</a></td>
            <td>Kellen Cresswell </td>
            <td>SpectralTAD: Hierarchical TAD detection using spectral clustering</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/SPEM.html">SPEM</a></td>
            <td>Xinyi YANG </td>
            <td>S-system parameter estimation method</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/SPIA.html">SPIA</a></td>
            <td>Adi Laurentiu Tarca </td>
            <td>Signaling Pathway Impact Analysis (SPIA) using combined evidence of pathway over-representation and unusual signaling perturbations</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/SpidermiR.html">SpidermiR</a></td>
            <td>Claudia Cava </td>
            <td>SpidermiR: An R/Bioconductor package for integrative network analysis with miRNA data</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/spikeLI.html">spikeLI</a></td>
            <td>Enrico Carlon </td>
            <td>Affymetrix Spike-in Langmuir Isotherm Data Analysis Tool</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/spkTools.html">spkTools</a></td>
            <td>Matthew N McCall </td>
            <td>Methods for Spike-in Arrays</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/splatter.html">splatter</a></td>
            <td>Luke Zappia </td>
            <td>Simple Simulation of Single-cell RNA Sequencing Data</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/splicegear.html">splicegear</a></td>
            <td>Laurent Gautier </td>
            <td>splicegear</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/SplicingGraphs.html">SplicingGraphs</a></td>
            <td>H. Pagès </td>
            <td>Create, manipulate, visualize splicing graphs, and assign RNA-seq reads to them</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/splineTimeR.html">splineTimeR</a></td>
            <td>Herbert Braselmann </td>
            <td>Time-course differential gene expression data analysis using spline regression models followed by gene association network reconstruction</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/SPLINTER.html">SPLINTER</a></td>
            <td>Diana Low </td>
            <td>Splice Interpreter of Transcripts</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/splots.html">splots</a></td>
            <td>Wolfgang Huber </td>
            <td>Visualization of high-throughput assays in microtitre plate or slide format</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/SPONGE.html">SPONGE</a></td>
            <td>Markus List </td>
            <td>Sparse Partial Correlations On Gene Expression</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/spotSegmentation.html">spotSegmentation</a></td>
            <td>Chris Fraley </td>
            <td>Microarray Spot Segmentation and Gridding for Blocks of Microarray Spots</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/SQLDataFrame.html">SQLDataFrame</a></td>
            <td>Qian Liu </td>
            <td>Representation of SQL database in DataFrame metaphor</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/SQUADD.html">SQUADD</a></td>
            <td>Martial Sankar </td>
            <td>Add-on of the SQUAD Software</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/sRACIPE.html">sRACIPE</a></td>
            <td>Vivek Kohar </td>
            <td>Systems biology tool to simulate gene regulatory circuits</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/SRAdb.html">SRAdb</a></td>
            <td>Jack Zhu </td>
            <td>A compilation of metadata from NCBI SRA and tools</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/sRAP.html">sRAP</a></td>
            <td>Charles Warden </td>
            <td>Simplified RNA-Seq Analysis Pipeline</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/SRGnet.html">SRGnet</a></td>
            <td>Isar Nassiri </td>
            <td>SRGnet: An R package for studying synergistic response to gene mutations from transcriptomics data</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/srnadiff.html">srnadiff</a></td>
            <td>Zytnicki Matthias </td>
            <td>Finding differentially expressed unannotated genomic regions from RNA-seq data</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/sscore.html">sscore</a></td>
            <td>Richard Kennedy </td>
            <td>S-Score Algorithm for Affymetrix Oligonucleotide Microarrays</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/sscu.html">sscu</a></td>
            <td>Yu Sun </td>
            <td>Strength of Selected Codon Usage</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/sSeq.html">sSeq</a></td>
            <td>Danni Yu </td>
            <td>Shrinkage estimation of dispersion in Negative Binomial models for RNA-seq experiments with small sample size</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/ssize.html">ssize</a></td>
            <td>Gregory R. Warnes </td>
            <td>Estimate Microarray Sample Size</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/SSPA.html">SSPA</a></td>
            <td>Maarten van Iterson </td>
            <td>General Sample Size and Power Analysis for Microarray and Next-Generation Sequencing Data</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/ssPATHS.html">ssPATHS</a></td>
            <td>Natalie R. Davidson </td>
            <td>ssPATHS: Single Sample PATHway Score</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/ssrch.html">ssrch</a></td>
            <td>VJ Carey </td>
            <td>a simple search engine</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/ssviz.html">ssviz</a></td>
            <td>Diana Low </td>
            <td>A small RNA-seq visualizer and analysis toolkit</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/stageR.html">stageR</a></td>
            <td>Koen Van den Berge </td>
            <td>stageR: stage-wise analysis of high throughput gene expression data in R</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/STAN.html">STAN</a></td>
            <td>Rafael Campos-Martin </td>
            <td>The Genomic STate ANnotation Package</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/staRank.html">staRank</a></td>
            <td>Juliane Siebourg </td>
            <td>Stability Ranking</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/StarBioTrek.html">StarBioTrek</a></td>
            <td>Claudia Cava </td>
            <td>StarBioTrek</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/Starr.html">Starr</a></td>
            <td>Benedikt Zacher </td>
            <td>Simple tiling array analysis of Affymetrix ChIP-chip data</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/STATegRa.html">STATegRa</a></td>
            <td>David Gomez-Cabrero, Núria Planell </td>
            <td>Classes and methods for multi-omics data integration</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/statTarget.html">statTarget</a></td>
            <td>Hemi Luan </td>
            <td>Statistical Analysis of Molecular Profiles</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/stepNorm.html">stepNorm</a></td>
            <td>Yuanyuan Xiao </td>
            <td>Stepwise normalization functions for cDNA microarrays</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/strandCheckR.html">strandCheckR</a></td>
            <td>Thu-Hien To </td>
            <td>Calculate strandness information of a bam file</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/Streamer.html">Streamer</a></td>
            <td>Martin Morgan </td>
            <td>Enabling stream processing of large files</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/STRINGdb.html">STRINGdb</a></td>
            <td>Damian Szklarczyk </td>
            <td>STRINGdb (Search Tool for the Retrieval of Interacting proteins database)</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/STROMA4.html">STROMA4</a></td>
            <td>Sadiq Saleh </td>
            <td>Assign Properties to TNBC Patients</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/Structstrings.html">Structstrings</a></td>
            <td>Felix G.M. Ernst </td>
            <td>Implementation of the dot bracket annotations with Biostrings</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/StructuralVariantAnnotation.html">StructuralVariantAnnotation</a></td>
            <td>Daniel Cameron </td>
            <td>Variant annotations for structural variants</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/SubCellBarCode.html">SubCellBarCode</a></td>
            <td>Taner Arslan </td>
            <td>SubCellBarCode: Integrated workflow for robust mapping and visualizing whole human spatial proteome</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/subSeq.html">subSeq</a></td>
            <td>Andrew J. Bass, John D. Storey </td>
            <td>Subsampling of high-throughput sequencing count data</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/SummarizedBenchmark.html">SummarizedBenchmark</a></td>
            <td>Patrick Kimes </td>
            <td>Classes and methods for performing benchmark comparisons</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/SummarizedExperiment.html">SummarizedExperiment</a></td>
            <td>Bioconductor Package Maintainer </td>
            <td>SummarizedExperiment container</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/supraHex.html">supraHex</a></td>
            <td>Hai Fang </td>
            <td>supraHex: a supra-hexagonal map for analysing tabular omics data</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/survcomp.html">survcomp</a></td>
            <td>Benjamin Haibe-Kains, Markus Schroeder, Catharina Olsen </td>
            <td>Performance Assessment and Comparison for Survival Analysis</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/survtype.html">survtype</a></td>
            <td>Dongmin Jung </td>
            <td>Subtype Identification with Survival Data</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/Sushi.html">Sushi</a></td>
            <td>Douglas H Phanstiel </td>
            <td>Tools for visualizing genomics data</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/sva.html">sva</a></td>
            <td>Jeffrey T. Leek, John D. Storey, W. Evan Johnson </td>
            <td>Surrogate Variable Analysis</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/SVAPLSseq.html">SVAPLSseq</a></td>
            <td>Sutirtha Chakraborty </td>
            <td>SVAPLSseq-An R package to estimate the hidden factors of unwanted variability and adjust for them to enable a more powerful and accurate differential expression analysis based on RNAseq data</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/SWATH2stats.html">SWATH2stats</a></td>
            <td>Peter Blattmann </td>
            <td>Transform and Filter SWATH Data for Statistical Packages</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/SwathXtend.html">SwathXtend</a></td>
            <td>Jemma Wu </td>
            <td>SWATH extended library generation and statistical data analysis</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/swfdr.html">swfdr</a></td>
            <td>Simina M. Boca, Jeffrey T. Leek </td>
            <td>Science-wise false discovery rate and proportion of true null hypotheses estimation</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/SwimR.html">SwimR</a></td>
            <td>Randy Blakely </td>
            <td>SwimR: A Suite of Analytical Tools for Quantification of C. elegans Swimming Behavior</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/switchBox.html">switchBox</a></td>
            <td>Bahman Afsari, Luigi Marchionni, Wikum Dinalankara </td>
            <td>Utilities to train and validate classifiers based on pair switching using the K-Top-Scoring-Pair (KTSP) algorithm</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/switchde.html">switchde</a></td>
            <td>Kieran Campbell </td>
            <td>Switch-like differential expression across single-cell trajectories</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/synapter.html">synapter</a></td>
            <td>Laurent Gatto and Sebastian Gibb </td>
            <td>Label-free data analysis pipeline for optimal identification and quantitation</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/synergyfinder.html">synergyfinder</a></td>
            <td>Shuyu Zheng </td>
            <td>Calculate and Visualize Synergy Scores for Drug Combinations</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/synlet.html">synlet</a></td>
            <td>Chunxuan Shao </td>
            <td>Hits Selection for Synthetic Lethal RNAi Screen Data</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/SynMut.html">SynMut</a></td>
            <td>Haogao Gu </td>
            <td>SynMut: Designing Synonymously Mutated Sequences with Different Genomic Signatures</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/systemPipeR.html">systemPipeR</a></td>
            <td>Thomas Girke </td>
            <td>systemPipeR: NGS workflow and report generation environment</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/target.html">target</a></td>
            <td>Mahmoud Ahmed </td>
            <td>Predict Combined Function of Transcription Factors</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/TargetScore.html">TargetScore</a></td>
            <td>Yue Li </td>
            <td>TargetScore: Infer microRNA targets using microRNA-overexpression data and sequence information</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/TargetSearch.html">TargetSearch</a></td>
            <td>Alvaro Cuadros-Inostroza </td>
            <td>A package for the analysis of GC-MS metabolite profiling data</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/TarSeqQC.html">TarSeqQC</a></td>
            <td>Gabriela Merino </td>
            <td>TARgeted SEQuencing Experiment Quality Control</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/TCC.html">TCC</a></td>
            <td>Jianqiang Sun, Tomoaki Nishiyama </td>
            <td>TCC: Differential expression analysis for tag count data with robust normalization strategies</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/TCGAbiolinks.html">TCGAbiolinks</a></td>
            <td>Antonio Colaprico, Tiago Chedraoui Silva </td>
            <td>TCGAbiolinks: An R/Bioconductor package for integrative analysis with GDC data</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/TCGAbiolinksGUI.html">TCGAbiolinksGUI</a></td>
            <td>Tiago C. Silva </td>
            <td>"TCGAbiolinksGUI: A Graphical User Interface to analyze cancer molecular and clinical data"</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/TCGAutils.html">TCGAutils</a></td>
            <td>Marcel Ramos </td>
            <td>TCGA utility functions for data management</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/TCseq.html">TCseq</a></td>
            <td>Mengjun Wu </td>
            <td>Time course sequencing data analysis</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/TDARACNE.html">TDARACNE</a></td>
            <td>Zoppoli Pietro </td>
            <td>Network reverse engineering from time course data.</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/tenXplore.html">tenXplore</a></td>
            <td>VJ Carey </td>
            <td>ontological exploration of scRNA-seq of 1.3 million mouse neurons from 10x genomics</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/TEQC.html">TEQC</a></td>
            <td>Manuela Hummel </td>
            <td>Quality control for target capture experiments</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/ternarynet.html">ternarynet</a></td>
            <td>Matthew N. McCall </td>
            <td>Ternary Network Estimation</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/TFARM.html">TFARM</a></td>
            <td>Liuba Nausicaa Martino </td>
            <td>Transcription Factors Association Rules Miner</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/TFBSTools.html">TFBSTools</a></td>
            <td>Ge Tan </td>
            <td>Software Package for Transcription Factor Binding Site (TFBS) Analysis</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/TFEA.ChIP.html">TFEA.ChIP</a></td>
            <td>Laura Puente Santamaría </td>
            <td>Analyze Transcription Factor Enrichment</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/TFHAZ.html">TFHAZ</a></td>
            <td>Alberto Marchesi </td>
            <td>Transcription Factor High Accumulation Zones</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/TFutils.html">TFutils</a></td>
            <td>Shweta Gopaulakrishnan </td>
            <td>TFutils</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/tigre.html">tigre</a></td>
            <td>Antti Honkela </td>
            <td>Transcription factor Inference through Gaussian process Reconstruction of Expression</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/tilingArray.html">tilingArray</a></td>
            <td>Zhenyu Xu </td>
            <td>Transcript mapping with high-density oligonucleotide tiling arrays</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/timecourse.html">timecourse</a></td>
            <td>Yu Chuan Tai </td>
            <td>Statistical Analysis for Developmental Microarray Time Course Data</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/timescape.html">timescape</a></td>
            <td>Maia Smith </td>
            <td>Patient Clonal Timescapes</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/TimeSeriesExperiment.html">TimeSeriesExperiment</a></td>
            <td>Lan Huong Nguyen </td>
            <td>Analysis for short time-series data</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/TIN.html">TIN</a></td>
            <td>Bjarne Johannessen </td>
            <td>Transcriptome instability analysis</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/TissueEnrich.html">TissueEnrich</a></td>
            <td>Ashish Jain </td>
            <td>Tissue-specific gene enrichment analysis</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/TitanCNA.html">TitanCNA</a></td>
            <td>Gavin Ha </td>
            <td>Subclonal copy number and LOH prediction from whole genome sequencing of tumours</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/tkWidgets.html">tkWidgets</a></td>
            <td>J. Zhang </td>
            <td>R based tk widgets</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/TMixClust.html">TMixClust</a></td>
            <td>Monica Golumbeanu </td>
            <td>Time Series Clustering of Gene Expression with Gaussian Mixed-Effects Models and Smoothing Splines</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/TNBC.CMS.html">TNBC.CMS</a></td>
            <td>Doyeong Yu </td>
            <td>TNBC.CMS: Prediction of TNBC Consensus Molecular Subtypes</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/TnT.html">TnT</a></td>
            <td>Jialin Ma </td>
            <td>Interactive Visualization for Genomic Features</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/TOAST.html">TOAST</a></td>
            <td>Ziyi Li </td>
            <td>Tools for the analysis of heterogeneous tissues</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/tofsims.html">tofsims</a></td>
            <td>Lorenz Gerber </td>
            <td>Import, process and analysis of Time-of-Flight Secondary Ion Mass Spectrometry (ToF-SIMS) imaging data</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/ToPASeq.html">ToPASeq</a></td>
            <td>Ivana Ihnatova </td>
            <td>Topology-based pathway analysis of RNA-seq data</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/topconfects.html">topconfects</a></td>
            <td>Paul Harrison </td>
            <td>Top Confident Effect Sizes</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/topdownr.html">topdownr</a></td>
            <td>Sebastian Gibb </td>
            <td>Investigation of Fragmentation Conditions in Top-Down Proteomics</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/topGO.html">topGO</a></td>
            <td>Adrian Alexa </td>
            <td>Enrichment Analysis for Gene Ontology</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/TPP.html">TPP</a></td>
            <td>Dorothee Childs </td>
            <td>Analyze thermal proteome profiling (TPP) experiments</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/TPP2D.html">TPP2D</a></td>
            <td>Nils Kurzawa </td>
            <td>FDR-controlled analysis of 2D-TPP experiments</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/tracktables.html">tracktables</a></td>
            <td>Tom Carroll </td>
            <td>Build IGV tracks and HTML reports</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/trackViewer.html">trackViewer</a></td>
            <td>Jianhong Ou </td>
            <td>A R/Bioconductor package with web interface for drawing elegant interactive tracks or lollipop plot to facilitate integrated analysis of multi-omics data</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/tradeSeq.html">tradeSeq</a></td>
            <td>Hector Roux de Bezieux </td>
            <td>trajectory-based differential expression analysis for sequencing data</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/transcriptogramer.html">transcriptogramer</a></td>
            <td>Diego Morais </td>
            <td>Transcriptional analysis based on transcriptograms</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/transcriptR.html">transcriptR</a></td>
            <td>Armen R. Karapetyan </td>
            <td>An Integrative Tool for ChIP- And RNA-Seq Based Primary Transcripts Detection and Quantification</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/transite.html">transite</a></td>
            <td>Konstantin Krismer </td>
            <td>RNA-binding protein motif analysis</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/tRanslatome.html">tRanslatome</a></td>
            <td>Toma Tebaldi, Erik Dassi </td>
            <td>Comparison between multiple levels of gene expression</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/TransView.html">TransView</a></td>
            <td>Julius Muller </td>
            <td>Read density map construction and accession. Visualization of ChIPSeq and RNASeq data sets</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/traseR.html">traseR</a></td>
            <td>li chen</td>
            <td>GWAS trait-associated SNP enrichment analyses in genomic intervals</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/treeio.html">treeio</a></td>
            <td>Guangchuang Yu </td>
            <td>Base Classes and Functions for Phylogenetic Tree Input and Output</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/TreeSummarizedExperiment.html">TreeSummarizedExperiment</a></td>
            <td>Ruizhu Huang </td>
            <td>TreeSummarizedExperiment: a S4 Class for Data with Tree Structures</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/trena.html">trena</a></td>
            <td>Paul Shannon </td>
            <td>Fit transcriptional regulatory networks using gene expression, priors, machine learning</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/Trendy.html">Trendy</a></td>
            <td>Rhonda Bacher </td>
            <td>Breakpoint analysis of time-course expression data</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/triform.html">triform</a></td>
            <td>Thomas Carroll </td>
            <td>Triform finds enriched regions (peaks) in transcription factor ChIP-sequencing data</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/trigger.html">trigger</a></td>
            <td>John D. Storey </td>
            <td>Transcriptional Regulatory Inference from Genetics of Gene ExpRession</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/trio.html">trio</a></td>
            <td>Holger Schwender </td>
            <td>Testing of SNPs and SNP Interactions in Case-Parent Trio Studies</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/triplex.html">triplex</a></td>
            <td>Jiri Hon </td>
            <td>Search and visualize intramolecular triplex-forming sequences in DNA</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/tRNA.html">tRNA</a></td>
            <td>Felix GM Ernst </td>
            <td>Analyzing tRNA sequences and structures</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/tRNAdbImport.html">tRNAdbImport</a></td>
            <td>Felix G.M. Ernst </td>
            <td>Importing from tRNAdb and mitotRNAdb as GRanges objects</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/tRNAscanImport.html">tRNAscanImport</a></td>
            <td>Felix G.M. Ernst </td>
            <td>Importing a tRNAscan-SE result file as GRanges object</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/TRONCO.html">TRONCO</a></td>
            <td>Luca De Sano </td>
            <td>TRONCO, an R package for TRanslational ONCOlogy</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/TSCAN.html">TSCAN</a></td>
            <td>Zhicheng Ji </td>
            <td>TSCAN: Tools for Single-Cell ANalysis</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/tspair.html">tspair</a></td>
            <td>Jeffrey T. Leek </td>
            <td>Top Scoring Pairs for Microarray Classification</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/TSRchitect.html">TSRchitect</a></td>
            <td>R. Taylor Raborn </td>
            <td>Promoter identification from large-scale TSS profiling data</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/TTMap.html">TTMap</a></td>
            <td>Rachel Jeitziner </td>
            <td>Two-Tier Mapper: a clustering tool based on topological data analysis</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/TurboNorm.html">TurboNorm</a></td>
            <td>Maarten van Iterson </td>
            <td>A fast scatterplot smoother suitable for microarray normalization</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/TVTB.html">TVTB</a></td>
            <td>Kevin Rue-Albrecht </td>
            <td>TVTB: The VCF Tool Box</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/tweeDEseq.html">tweeDEseq</a></td>
            <td>Juan R Gonzalez </td>
            <td>RNA-seq data analysis using the Poisson-Tweedie family of distributions</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/twilight.html">twilight</a></td>
            <td>Stefanie Scheid </td>
            <td>Estimation of local false discovery rate</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/twoddpcr.html">twoddpcr</a></td>
            <td>Anthony Chiu </td>
            <td>Classify 2-d Droplet Digital PCR (ddPCR) data and quantify the number of starting molecules</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/tximeta.html">tximeta</a></td>
            <td>Michael Love </td>
            <td>Transcript Quantification Import with Automatic Metadata</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/tximport.html">tximport</a></td>
            <td>Michael Love </td>
            <td>Import and summarize transcript-level estimates for transcript- and gene-level analysis</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/TxRegInfra.html">TxRegInfra</a></td>
            <td>VJ Carey </td>
            <td>Metadata management for multiomic specification of transcriptional regulatory networks</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/TypeInfo.html">TypeInfo</a></td>
            <td>Duncan Temple Lang </td>
            <td>Optional Type Specification Prototype</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/Ularcirc.html">Ularcirc</a></td>
            <td>David Humphreys </td>
            <td>Shiny app for canonical and back splicing analysis (i.e. circular and mRNA analysis)</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/UNDO.html">UNDO</a></td>
            <td>Niya Wang </td>
            <td>Unsupervised Deconvolution of Tumor-Stromal Mixed Expressions</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/unifiedWMWqPCR.html">unifiedWMWqPCR</a></td>
            <td>Joris Meys </td>
            <td>Unified Wilcoxon-Mann Whitney Test for testing differential expression in qPCR data</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/UniProt.ws.html">UniProt.ws</a></td>
            <td>Bioconductor Package Maintainer </td>
            <td>R Interface to UniProt Web Services</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/Uniquorn.html">Uniquorn</a></td>
            <td>'Raik Otto' </td>
            <td>Identification of cancer cell lines based on their weighted mutational/ variational fingerprint</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/universalmotif.html">universalmotif</a></td>
            <td>Benjamin Jean-Marie Tremblay </td>
            <td>Import, Modify, and Export Motifs with R</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/uSORT.html">uSORT</a></td>
            <td>Hao Chen </td>
            <td>uSORT: A self-refining ordering pipeline for gene selection</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/VanillaICE.html">VanillaICE</a></td>
            <td>Robert Scharpf </td>
            <td>A Hidden Markov Model for high throughput genotyping arrays</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/variancePartition.html">variancePartition</a></td>
            <td>Gabriel E. Hoffman </td>
            <td>Quantify and interpret divers of variation in multilevel gene expression experiments</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/VariantAnnotation.html">VariantAnnotation</a></td>
            <td>Valerie Obenchain </td>
            <td>Annotation of Genetic Variants</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/VariantExperiment.html">VariantExperiment</a></td>
            <td>Qian Liu </td>
            <td>A RangedSummarizedExperiment Container for VCF/GDS Data with GDS Backend</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/VariantFiltering.html">VariantFiltering</a></td>
            <td>Robert Castelo </td>
            <td>Filtering of coding and non-coding genetic variants</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/VariantTools.html">VariantTools</a></td>
            <td>Michael Lawrence </td>
            <td>Tools for Exploratory Analysis of Variant Calls</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/vbmp.html">vbmp</a></td>
            <td>Nicola Lama </td>
            <td>Variational Bayesian Multinomial Probit Regression</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/VCFArray.html">VCFArray</a></td>
            <td>Qian Liu </td>
            <td>Representing on-disk / remote VCF files as array-like objects</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/Vega.html">Vega</a></td>
            <td>Sandro Morganella </td>
            <td>An R package for copy number data segmentation</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/VegaMC.html">VegaMC</a></td>
            <td>Sandro Morganella </td>
            <td>VegaMC: A Package Implementing a Variational Piecewise Smooth Model for Identification of Driver Chromosomal Imbalances in Cancer</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/VennDetail.html">VennDetail</a></td>
            <td>Kai Guo </td>
            <td>A package for visualization and extract details</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/vidger.html">vidger</a></td>
            <td>Brandon Monier </td>
            <td>Create rapid visualizations of RNAseq data in R</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/viper.html">viper</a></td>
            <td>Mariano J Alvarez </td>
            <td>Virtual Inference of Protein-activity by Enriched Regulon analysis</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/ViSEAGO.html">ViSEAGO</a></td>
            <td>Aurelien Brionne </td>
            <td>ViSEAGO: a Bioconductor package for clustering biological functions using Gene Ontology and semantic similarity</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/vsn.html">vsn</a></td>
            <td>Wolfgang Huber </td>
            <td>Variance stabilization and calibration for microarray data</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/vtpnet.html">vtpnet</a></td>
            <td>VJ Carey </td>
            <td>variant-transcription factor-phenotype networks</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/vulcan.html">vulcan</a></td>
            <td>Federico M. Giorgi </td>
            <td>VirtUaL ChIP-Seq data Analysis using Networks</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/waddR.html">waddR</a></td>
            <td>Julian Flesch </td>
            <td>Statistical Test for Detecting Differential Distributions Based on the Wasserstein Metric</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/wateRmelon.html">wateRmelon</a></td>
            <td>Leo </td>
            <td>Illumina 450 methylation array normalization and metrics</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/wavClusteR.html">wavClusteR</a></td>
            <td>Federico Comoglio </td>
            <td>Sensitive and highly resolved identification of RNA-protein interaction sites in PAR-CLIP data</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/waveTiling.html">waveTiling</a></td>
            <td>Kristof De Beuf </td>
            <td>Wavelet-Based Models for Tiling Array Transcriptome Analysis</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/weaver.html">weaver</a></td>
            <td>Seth Falcon </td>
            <td>Tools and extensions for processing Sweave documents</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/webbioc.html">webbioc</a></td>
            <td>Colin A. Smith </td>
            <td>Bioconductor Web Interface</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/widgetTools.html">widgetTools</a></td>
            <td>Jianhua Zhang </td>
            <td>Creates an interactive tcltk widget</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/wiggleplotr.html">wiggleplotr</a></td>
            <td>Kaur Alasoo </td>
            <td>Make read coverage plots from BigWig files</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/Wrench.html">Wrench</a></td>
            <td>Hector Corrada Bravo </td>
            <td>Wrench normalization for sparse count data</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/XBSeq.html">XBSeq</a></td>
            <td>Yuanhang Liu </td>
            <td>Test for differential expression for RNA-seq data</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/XCIR.html">XCIR</a></td>
            <td>Renan Sauteraud </td>
            <td>XCI-inference</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/xcms.html">xcms</a></td>
            <td>Steffen Neumann </td>
            <td>LC-MS and GC-MS Data Analysis</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/XDE.html">XDE</a></td>
            <td>Robert Scharpf </td>
            <td>XDE: a Bayesian hierarchical model for cross-study analysis of differential gene expression</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/Xeva.html">Xeva</a></td>
            <td>Benjamin Haibe-Kains </td>
            <td>Analysis of patient-derived xenograft (PDX) data</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/XINA.html">XINA</a></td>
            <td>Lang Ho Lee and Sasha A. Singh </td>
            <td>Multiplexes isobaric mass tagged-based kinetics data for network analysis</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/xmapbridge.html">xmapbridge</a></td>
            <td>Chris Wirth </td>
            <td>Export plotting files to the xmapBridge for visualisation in X:Map</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/xps.html">xps</a></td>
            <td>Christian Stratowa </td>
            <td>Processing and Analysis of Affymetrix Oligonucleotide Arrays including Exon Arrays, Whole Genome Arrays and Plate Arrays</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/XVector.html">XVector</a></td>
            <td>Hervé Pagès </td>
            <td>Foundation of external vector representation and manipulation in Bioconductor</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/yamss.html">yamss</a></td>
            <td>Leslie Myint </td>
            <td>Tools for high-throughput metabolomics</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/YAPSA.html">YAPSA</a></td>
            <td>Daniel Huebschmann </td>
            <td>Yet Another Package for Signature Analysis</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/yaqcaffy.html">yaqcaffy</a></td>
            <td>Laurent Gatto </td>
            <td>Affymetrix expression data quality control and reproducibility analysis</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/yarn.html">yarn</a></td>
            <td>Joseph N Paulson </td>
            <td>YARN: Robust Multi-Condition RNA-Seq Preprocessing and Normalization</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/zFPKM.html">zFPKM</a></td>
            <td>Ron Ammar </td>
            <td>A suite of functions to facilitate zFPKM transformations</td>
        </tr>
        
        

        <tr class="row_even">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/zinbwave.html">zinbwave</a></td>
            <td>Davide Risso </td>
            <td>Zero-Inflated Negative Binomial Model for RNA-Seq Data</td>
        </tr>
        
        

        <tr class="row_odd">
            <td><a href="https://www.bioconductor.org/packages/release/bioc/html/zlibbioc.html">zlibbioc</a></td>
            <td>Bioconductor Package Maintainer </td>
            <td>An R packaged zlib-1.2.5</td>
        </tr>
        
    </tbody></table>
</div>
	
	
	

	
	
	  </div>
	  <div id="RightRail" class="RightRail">
	  <a name="section-navigation"></a> <!-- accessibility anchor -->
	  
  


  
    <div class=""><table cellspacing="0" cellpadding="0" class="sb"><tbody><tr><td class="sb1"></td><td class="sb2"></td><td class="sb3"></td></tr><tr><td class="sb4">&nbsp;</td><td class="sb5">
  <img src="./links2_files/inv.gif" align="right" height="200" width="1" alt="">
  <p class="box_header">Packages <span class="olive">»</span></p>
  <p><em>Bioconductor's</em> stable, semi-annual release:</p>
  <ul>
    <li>Analysis <a class="symlink" href="https://www.bioconductor.org/packages/release/BiocViews.html#___Software">software</a> packages.</li>
    <li><a class="symlink" href="https://www.bioconductor.org/packages/release/BiocViews.html#___AnnotationData">Annotation</a> packages.</li>
    <li>Illustrative <a class="symlink" href="https://www.bioconductor.org/packages/release/BiocViews.html#___ExperimentData">experiment data</a> packages.</li>
    <li><a class="symlink" href="https://www.bioconductor.org/packages/release/BiocViews.html#___Workflow">Workflow</a> packages.</li>
    <li>Latest <a href="https://www.bioconductor.org/news/bioc_3_10_release">release announcement</a>.</li>
  </ul>
  <p><em>Bioconductor</em> is also available via
    <a href="https://www.bioconductor.org/help/docker/">Docker</a>
    and <a href="https://www.bioconductor.org/help/bioconductor-cloud-ami/">Amazon Machine
      Images</a>.</p>
</td><td class="sb6">&nbsp;</td></tr><tr><td class="sb7"></td><td class="sb8"></td><td class="sb9"></td></tr></tbody></table></div>

  
    <div class=""><table cellspacing="0" cellpadding="0" class="sb"><tbody><tr><td class="sb1"></td><td class="sb2"></td><td class="sb3"></td></tr><tr><td class="sb4">&nbsp;</td><td class="sb5">
  <img src="./links2_files/inv.gif" align="right" height="200" width="1" alt="">
  <p class="box_header">Development Version
    <span class="olive">»</span></p>
  <p>Bioconductor packages under development:</p>
  <ul>
    
    
    <li>Analysis <a class="symlink" href="https://www.bioconductor.org/packages/devel/BiocViews.html#___Software">software</a> packages.</li>
    
    
    <li><a class="symlink" href="https://www.bioconductor.org/packages/devel/BiocViews.html#___AnnotationData">Annotation</a> packages</li>
    
    
    <li>Illustrative <a class="symlink" href="https://www.bioconductor.org/packages/devel/BiocViews.html#___ExperimentData">experiment data</a> packages</li>
    
  </ul>
  <p>Developer Resources:</p>
  <ul>
    <li><a href="https://www.bioconductor.org/developers/gitlog">GIT Log</a></li>
  </ul>
</td><td class="sb6">&nbsp;</td></tr><tr><td class="sb7"></td><td class="sb8"></td><td class="sb9"></td></tr></tbody></table></div>

  
    <div class=""><table cellspacing="0" cellpadding="0" class="sb"><tbody><tr><td class="sb1"></td><td class="sb2"></td><td class="sb3"></td></tr><tr><td class="sb4">&nbsp;</td><td class="sb5">
  <img src="./links2_files/inv.gif" align="right" height="200" width="1" alt="">
  <p class="box_header">
    <a href="https://www.bioconductor.org/about/release-announcements/">Previous Releases</a>
    <span class="olive">»</span>
  </p>
  <p><em>Bioconductor</em>
    (<em>R</em>) <a href="https://www.bioconductor.org/about/release-announcements/#release-versions">versions</a>:
  </p>
  <ul class="inline_list">
    <li><a href="https://www.bioconductor.org/packages/3.9/BiocViews.html#___Software">3.9&nbsp;(3.6)</a></li>
    <li><a href="https://www.bioconductor.org/packages/3.8/BiocViews.html#___Software">3.8&nbsp;(3.5)</a></li>
    <li><a href="https://www.bioconductor.org/packages/3.7/BiocViews.html#___Software">3.7&nbsp;(3.5)</a></li>
    <li><a href="https://www.bioconductor.org/packages/3.6/BiocViews.html#___Software">3.6&nbsp;(3.4)</a></li>
    <li><a href="https://www.bioconductor.org/packages/3.5/BiocViews.html#___Software">3.5&nbsp;(3.4)</a></li>
    <li><a href="https://www.bioconductor.org/packages/3.4/BiocViews.html#___Software">3.4&nbsp;(3.3)</a></li>
    <li><a href="https://www.bioconductor.org/packages/3.3/BiocViews.html#___Software">3.3&nbsp;(3.3)</a></li>
    <li><a href="https://www.bioconductor.org/packages/3.2/BiocViews.html#___Software">3.2&nbsp;(3.2)</a></li>
    <li><a href="https://www.bioconductor.org/packages/3.1/BiocViews.html#___Software">3.1&nbsp;(3.2)</a></li>
    <li><a href="https://www.bioconductor.org/packages/3.0/BiocViews.html#___Software">3.0&nbsp;(3.1)</a></li>
    <li><a href="https://www.bioconductor.org/packages/2.14/BiocViews.html#___Software">2.14&nbsp;(3.1)</a></li>
    <li><a href="https://www.bioconductor.org/packages/2.13/BiocViews.html#___Software">2.13&nbsp;(3.0)</a></li>
    <li><a href="https://www.bioconductor.org/packages/2.12/BiocViews.html#___Software">2.12&nbsp;(3.0)</a></li>
    <li><a href="https://www.bioconductor.org/packages/2.11/BiocViews.html#___Software">2.11&nbsp;(2.15)</a></li>
    <li><a href="https://www.bioconductor.org/packages/2.10/BiocViews.html#___Software">2.10&nbsp;(2.15)</a></li>
    <li><a href="https://www.bioconductor.org/packages/2.9/BiocViews.html#___Software">2.9&nbsp;(2.14)</a></li>
    <li><a href="https://www.bioconductor.org/packages/2.8/BiocViews.html#___Software">2.8&nbsp;(2.13)</a></li>
    <li><a href="https://www.bioconductor.org/packages/2.7/BiocViews.html#___Software">2.7&nbsp;(2.12)</a></li>
    <li><a href="https://www.bioconductor.org/packages/2.6/BiocViews.html#___Software">2.6&nbsp;(2.11)</a></li>
    <li><a href="https://www.bioconductor.org/packages/2.5/BiocViews.html#___Software">2.5&nbsp;(2.10)</a></li>
    <li><a href="https://www.bioconductor.org/packages/2.4/BiocViews.html">2.4&nbsp;(2.9)</a></li>
    <li><a href="https://www.bioconductor.org/packages/2.3/BiocViews.html">2.3&nbsp;(2.8)</a></li>
    <li><a href="https://www.bioconductor.org/packages/2.2/BiocViews.html">2.2&nbsp;(2.7)</a></li>
    <li><a href="https://www.bioconductor.org/packages/2.1/BiocViews.html">2.1&nbsp;(2.6)</a></li>
    <li><a href="https://www.bioconductor.org/packages/2.0/BiocViews.html">2.0&nbsp;(2.5)</a></li>
    <li><a href="https://www.bioconductor.org/packages/1.9/BiocViews.html">1.9&nbsp;(2.4)</a></li>
    <li><a href="https://www.bioconductor.org/packages/1.8/BiocViews.html">1.8&nbsp;(2.3)</a></li>
    <li><a href="https://www.bioconductor.org/packages/bioc/1.7/src/contrib/html/">1.7&nbsp;(2.2)</a></li>
    <li><a href="https://www.bioconductor.org/packages/bioc/1.6/src/contrib/html/">1.6&nbsp;(2.1)</a></li>
  </ul>
  <p>Release
    <a href="https://www.bioconductor.org/about/release-announcements/#release-announcements">announcements</a>.
  </p>
</td><td class="sb6">&nbsp;</td></tr><tr><td class="sb7"></td><td class="sb8"></td><td class="sb9"></td></tr></tbody></table></div>

  


	

      </div>

      <div id="SiteGlobalFooter" class="SiteGlobalFooter">

  <span class="global_footer_logo_bc noprint"><img src="./links2_files/logo_global_footer_bioconductor.gif" height="51" width="154" alt=""></span>

  <div class="attribution_copyright">
    <p>Contact us: <a href="https://support.bioconductor.org/">support.bioconductor.org</a></p>
    <p>Copyright © 2003 - 2020, Bioconductor</p>
  </div>

  <a name="site-map"></a> <!-- accessibility anchor -->
  <ul class="footer_nav_list noprint">
    <li class="footer_nav_list_element footer_nav_list_element_0">
      <b><a href="https://www.bioconductor.org/index.html">Home</a></b>
    </li>
    <li class="footer_nav_list_element footer_nav_list_element_1">
      <ul>
        <li><b><a href="https://www.bioconductor.org/install/index.html">Install</a></b></li>
        <li><a href="https://www.bioconductor.org/install/index.html#install-R">Install R</a></li>
        <li><a href="https://www.bioconductor.org/install/index.html#find-bioconductor-packages">Find Bioconductor Packages</a></li>
        <li><a href="https://www.bioconductor.org/install/index.html#install-bioconductor-packages">Install Bioconductor Packages</a></li>
        <li><a href="https://www.bioconductor.org/install/index.html#update-bioconductor-packages">Update Bioconductor Packages</a></li>
      </ul>
    </li>
    <li class="footer_nav_list_element footer_nav_list_element_2">
      <ul>
        <li><b><a href="https://www.bioconductor.org/help/index.html">Help</a></b></li>
        <li><a href="https://www.bioconductor.org/packages/release/workflows/">Workflows</a></li>
        <li><a href="https://www.bioconductor.org/help/package-vignettes/">Package Vignettes</a></li>
        <li><a href="https://www.bioconductor.org/help/faq/">FAQ</a></li>
        <li><a href="https://www.bioconductor.org/help/support/">Support</a></li>
        <li><a href="http://cran.r-project.org/">Using R</a></li>
        <li><a href="https://www.bioconductor.org/help/course-materials/">Courses</a></li>
        <li><a href="https://www.bioconductor.org/help/publications/">Publications</a></li>
        <li><a href="https://www.bioconductor.org/help/bioconductor-cloud-ami/">Cloud AMI</a></li>
        <li><a href="https://www.bioconductor.org/help/community/">Community Resources</a></li>

      </ul>
    </li>
    <li class="footer_nav_list_element footer_nav_list_element_3">
      <ul>
        <li><b><a href="https://www.bioconductor.org/developers/index.html">Developers</a></b></li>
		<li><a href="https://www.bioconductor.org/developers/package-guidelines/">Package Guidelines</a></li>
		<li><a href="https://www.bioconductor.org/developers/package-submission/">Package Submission</a></li>
		<li><a href="https://www.bioconductor.org/developers/release-schedule/">Release Schedule</a></li>
		<li><a href="https://www.bioconductor.org/developers/source-control/">Source Control</a></li>
      </ul>
    </li>
    <li class="footer_nav_list_element footer_nav_list_element_4">
      <ul>
        <li><b><a href="https://www.bioconductor.org/about/index.html">About</a></b></li>
        <li><a href="https://www.bioconductor.org/about/annual-reports/">Annual Reports</a></li>
        <li><a href="https://www.bioconductor.org/about/core-team/">Core Team</a></li>
        <li><a href="https://www.bioconductor.org/about/mirrors/">Mirrors</a></li>
        <li><a href="https://www.bioconductor.org/about/related-projects/">Related Projects</a></li>
      </ul>
    </li>
  </ul>
  <br style="clear:both">
  &nbsp;

</div>


<div id="SiteMasthead" class="SiteMasthead">

  <a name="site-navigation"></a> <!-- accessibility anchor -->

  <span class="logo_vanity_bar noprint"></span>

  <a href="https://www.bioconductor.org/">
	<img src="./links2_files/logo_bioconductor.gif" border="0" class="masthead_logo" height="78" width="260" alt="Bioconductor - open source software for bioinformatics">
  </a>

  <div id="SiteMastheadRight" class="SiteMastheadRight">
    <div id="SiteMastheadRightBackground" class="SiteMastheadRightBackground">

      <a name="site-search"></a> <!-- accessibility anchor -->
      <form class="site_search" id="search_form" method="GET" action="https://www.bioconductor.org/help/search/index.html">
        Search: <input id="q" name="q">
      </form>

      <ul class="masthead_nav noprint">
        <li class="masthead_nav_element" id="masthead_nav_element_1">
          <a href="https://www.bioconductor.org/">Home</a>
        </li>
        <li class="masthead_nav_element" id="masthead_nav_element_2">
          <a href="https://www.bioconductor.org/install/">Install</a>
        </li>
        <li class="masthead_nav_element" id="masthead_nav_element_3">
          <a href="https://www.bioconductor.org/help/">Help</a>
        </li>
        <li class="masthead_nav_element" id="masthead_nav_element_4">
          <a href="https://www.bioconductor.org/developers/">Developers</a>
        </li>
        <li class="masthead_nav_element" id="masthead_nav_element_5">
          <a href="https://www.bioconductor.org/about/">About</a>
        </li>
      </ul>

    </div>
  </div>

</div>

    </div>
  

</body></html>