<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<!-- saved from url=(0072)https://www.bioconductor.org/packages/release/BiocViews.html#___Software -->
<html xmlns="http://www.w3.org/1999/xhtml" dir="ltr" lang="en-US" prefix="og: http://ogp.me/ns#"><head><meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
  
  <meta http-equiv="Content-Language" content="en-us">
  <meta name="robots" content="all">
  



  
  
  
    <script type="text/javascript" async="" src="./Bioconductor - BiocViews_files/ga.js"></script><script type="text/javascript" src="./Bioconductor - BiocViews_files/jquery.js"></script>
  
  
  <script src="./Bioconductor - BiocViews_files/jquery.tools.min.js"></script>

  
  
  <title>Bioconductor - BiocViews</title>
  <link rel="stylesheet" type="text/css" href="./Bioconductor - BiocViews_files/bioconductor.css" media="screen">
  <link rel="SHORTCUT ICON" type="image/x-icon" href="https://www.bioconductor.org/favicon.ico">
  <link rel="ICON" type="image/x-icon" href="https://www.bioconductor.org/favicon.ico">
  <script type="text/javascript" src="./Bioconductor - BiocViews_files/bioconductor.js"></script>
 
  <script type="text/javascript" src="./Bioconductor - BiocViews_files/jquery.corner.js"></script>
  <script type="text/javascript" src="./Bioconductor - BiocViews_files/jquery.timeago.js"></script>
  <script type="text/javascript" src="./Bioconductor - BiocViews_files/bioc-style.js"></script>
  <script type="text/javascript" src="./Bioconductor - BiocViews_files/versions.js"></script>

  <!-- Google Analytics script -->
  <script type="text/javascript">
    var _gaq = _gaq || [];
    _gaq.push(['_setAccount', 'UA-9338486-1']);
    _gaq.push(['_trackPageview']);

    (function() {
      var ga = document.createElement('script'); ga.type = 'text/javascript'; ga.async = true;
      ga.src = ('https:' == document.location.protocol ? 'https://ssl' : 'http://www') + '.google-analytics.com/ga.js';
      var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(ga, s);
    })();
  </script>
<style type="text/css" id="vakata-stylesheet">#vakata-dragged { display:block; margin:0 0 0 0; padding:4px 4px 4px 24px; position:absolute; top:-2000px; line-height:16px; z-index:10000; } #vakata-contextmenu { display:block; visibility:hidden; left:0; top:-200px; position:absolute; margin:0; padding:0; min-width:180px; background:#ebebeb; border:1px solid silver; z-index:10000; *width:180px; } #vakata-contextmenu ul { min-width:180px; *width:180px; } #vakata-contextmenu ul, #vakata-contextmenu li { margin:0; padding:0; list-style-type:none; display:block; } #vakata-contextmenu li { line-height:20px; min-height:20px; position:relative; padding:0px; } #vakata-contextmenu li a { padding:1px 6px; line-height:17px; display:block; text-decoration:none; margin:1px 1px 0 1px; } #vakata-contextmenu li ins { float:left; width:16px; height:16px; text-decoration:none; margin-right:2px; } #vakata-contextmenu li a:hover, #vakata-contextmenu li.vakata-hover > a { background:gray; color:white; } #vakata-contextmenu li ul { display:none; position:absolute; top:-2px; left:100%; background:#ebebeb; border:1px solid gray; } #vakata-contextmenu .right { right:100%; left:auto; } #vakata-contextmenu .bottom { bottom:-1px; top:auto; } #vakata-contextmenu li.vakata-separator { min-height:0; height:1px; line-height:1px; font-size:1px; overflow:hidden; margin:0 2px; background:silver; /* border-top:1px solid #fefefe; */ padding:0; } </style><style type="text/css" id="jstree-stylesheet">.jstree ul, .jstree li { display:block; margin:0 0 0 0; padding:0 0 0 0; list-style-type:none; } .jstree li { display:block; min-height:18px; line-height:18px; white-space:nowrap; margin-left:18px; min-width:18px; } .jstree-rtl li { margin-left:0; margin-right:18px; } .jstree > ul > li { margin-left:0px; } .jstree-rtl > ul > li { margin-right:0px; } .jstree ins { display:inline-block; text-decoration:none; width:18px; height:18px; margin:0 0 0 0; padding:0; } .jstree a { display:inline-block; line-height:16px; height:16px; color:black; white-space:nowrap; text-decoration:none; padding:1px 2px; margin:0; } .jstree a:focus { outline: none; } .jstree a > ins { height:16px; width:16px; } .jstree a > .jstree-icon { margin-right:3px; } .jstree-rtl a > .jstree-icon { margin-left:3px; margin-right:0; } li.jstree-open > ul { display:block; } li.jstree-closed > ul { display:none; } #vakata-dragged ins { display:block; text-decoration:none; width:16px; height:16px; margin:0 0 0 0; padding:0; position:absolute; top:4px; left:4px;  -moz-border-radius:4px; border-radius:4px; -webkit-border-radius:4px; } #vakata-dragged .jstree-ok { background:green; } #vakata-dragged .jstree-invalid { background:red; } #jstree-marker { padding:0; margin:0; font-size:12px; overflow:hidden; height:12px; width:8px; position:absolute; top:-30px; z-index:10001; background-repeat:no-repeat; display:none; background-color:transparent; text-shadow:1px 1px 1px white; color:black; line-height:10px; } #jstree-marker-line { padding:0; margin:0; line-height:0%; font-size:1px; overflow:hidden; height:1px; width:100px; position:absolute; top:-30px; z-index:10000; background-repeat:no-repeat; display:none; background-color:#456c43;  cursor:pointer; border:1px solid #eeeeee; border-left:0; -moz-box-shadow: 0px 0px 2px #666; -webkit-box-shadow: 0px 0px 2px #666; box-shadow: 0px 0px 2px #666;  -moz-border-radius:1px; border-radius:1px; -webkit-border-radius:1px; }.jstree .jstree-real-checkbox { display:none; } .jstree-themeroller .ui-icon { overflow:visible; } .jstree-themeroller a { padding:0 2px; } .jstree-themeroller .jstree-no-icon { display:none; }.jstree .jstree-wholerow-real { position:relative; z-index:1; } .jstree .jstree-wholerow-real li { cursor:pointer; } .jstree .jstree-wholerow-real a { border-left-color:transparent !important; border-right-color:transparent !important; } .jstree .jstree-wholerow { position:relative; z-index:0; height:0; } .jstree .jstree-wholerow ul, .jstree .jstree-wholerow li { width:100%; } .jstree .jstree-wholerow, .jstree .jstree-wholerow ul, .jstree .jstree-wholerow li, .jstree .jstree-wholerow a { margin:0 !important; padding:0 !important; } .jstree .jstree-wholerow, .jstree .jstree-wholerow ul, .jstree .jstree-wholerow li { background:transparent !important; }.jstree .jstree-wholerow ins, .jstree .jstree-wholerow span, .jstree .jstree-wholerow input { display:none !important; }.jstree .jstree-wholerow a, .jstree .jstree-wholerow a:hover { text-indent:-9999px; !important; width:100%; padding:0 !important; border-right-width:0px !important; border-left-width:0px !important; } .jstree .jstree-wholerow-span { position:absolute; left:0; margin:0px; padding:0; height:18px; border-width:0; padding:0; z-index:0; }</style><link rel="stylesheet" type="text/css" media="all" href="./Bioconductor - BiocViews_files/style.css"></head>

  <body>
    <a name="top"></a>

<!-- a few hooks for screen readers -->
<a href="https://www.bioconductor.org/packages/release/BiocViews.html#site-navigation" title="Jump to site nav"></a>

<a href="https://www.bioconductor.org/packages/release/BiocViews.html#site-map" title="Jump to site map"></a>


    <div id="SiteContainer" class="SiteContainer">
      
      <div id="PageContent" class="PageContent">
      


	
	  <div id="PageBreadcrumbs" class="PageBreadcrumbs">
  <ul>
    
    
    
      
	  	
        <li><a href="https://www.bioconductor.org/">Home</a></li>
      
    
    
    
    
      
        <li>BiocViews</li>
      
    
  </ul>
</div>

	

	
	
	

	
<script type="text/javascript" src="./Bioconductor - BiocViews_files/tree.js"></script>
<script type="text/javascript" src="./Bioconductor - BiocViews_files/packages.js"></script>
<script type="text/javascript" src="./Bioconductor - BiocViews_files/packages(1).js"></script>
<script type="text/javascript" src="./Bioconductor - BiocViews_files/packages(2).js"></script>
<script type="text/javascript" src="./Bioconductor - BiocViews_files/packages(3).js"></script>


<script type="text/javascript" src="./Bioconductor - BiocViews_files/jquery.cookie.js"></script>
<script type="text/javascript" src="./Bioconductor - BiocViews_files/jquery.hotkeys.js"></script>
<script type="text/javascript" src="./Bioconductor - BiocViews_files/jquery.jstree.js"></script>
<script type="text/javascript" src="./Bioconductor - BiocViews_files/jsnetworkx.js"></script>
<script type="text/javascript" src="./Bioconductor - BiocViews_files/bioc_views.js"></script>

<!-- DataTables CSS -->
<link rel="stylesheet" type="text/css" href="./Bioconductor - BiocViews_files/jquery.dataTables.css">
 
<!-- DataTables -->
<script type="text/javascript" charset="utf8" src="./Bioconductor - BiocViews_files/jquery.dataTables.min.js"></script>


<link rel="stylesheet" href="./Bioconductor - BiocViews_files/jquery-ui.css">
<script src="./Bioconductor - BiocViews_files/jquery-ui.js"></script>
<!--<link rel="stylesheet" href="/resources/demos/style.css">-->


<h1>All Packages</h1>



<link type="text/css" rel="stylesheet" href="./Bioconductor - BiocViews_files/!style.css">


<div id="tree_outer" class="biocViewsTreeContainer">
    <div class="do_not_rebase">
        <div style="float:left;width:33%;">
            <h3>Bioconductor version <span id="bioc_version">3.8 (Release)</span></h3>
            <div id="tree-toggler-div" style="display: none;">
                <p>&nbsp;</p>
                <p>
                    <input id="tree-toggler" type="checkbox">
                    Developers: check this box
                    to toggle the visibility of childless biocViews.
                </p>
            </div>

            <div class="ui-widget">
                <label for="autocompleter">Autocomplete biocViews search:</label>
                <span role="status" aria-live="polite" class="ui-helper-hidden-accessible"></span><input id="autocompleter" size="35" class="ui-autocomplete-input" autocomplete="off">
                <p>&nbsp;</p>
            </div>

            <div id="tree" class="jstree jstree-0 biocViewsTree jstree-focused jstree-apple"><ul class="jstree-no-dots jstree-no-icons"><li packagelist="a4,a4Base,a4Classif,a4Core,a4Preproc,a4Reporting,ABAEnrichment,ABarray,abseqR,ABSSeq,acde,ACE,aCGH,ACME,ADaCGH2,adaptest,adSplit,AffiXcan,affxparser,affy,affycomp,AffyCompatible,affyContam,affycoretools,AffyExpress,affyILM,affyio,affylmGUI,affyPara,affypdnn,affyPLM,affyQCReport,AffyRNADegradation,AGDEX,agilp,AgiMicroRna,AIMS,ALDEx2,AllelicImbalance,alpine,alsace,altcdfenvs,AMOUNTAIN,amplican,ampliQueso,AnalysisPageServer,anamiR,Anaquin,AneuFinder,ANF,annaffy,annmap,annotate,AnnotationDbi,AnnotationFilter,AnnotationForge,AnnotationFuncs,AnnotationHub,AnnotationHubData,annotationTools,annotatr,anota,anota2seq,antiProfiles,apComplex,apeglm,appreci8R,aroma.light,ArrayExpress,ArrayExpressHTS,arrayMvout,arrayQuality,arrayQualityMetrics,ArrayTools,ArrayTV,ARRmNormalization,artMS,ASAFE,ASEB,ASGSCA,ASICS,ASpli,AssessORF,ASSET,ASSIGN,ATACseqQC,attract,AUCell,BaalChIP,BAC,bacon,BADER,BadRegionFinder,BAGS,ballgown,bamsignals,banocc,basecallQC,BaseSpaceR,Basic4Cseq,BASiCS,BasicSTARRseq,BatchQC,BayesKnockdown,BayesPeak,bayNorm,baySeq,BBCAnalyzer,BCRANK,bcSeq,BDMMAcorrect,beachmat,beadarray,beadarraySNP,BeadDataPackR,BEARscc,BEAT,BEclear,bgafun,BgeeDB,BGmix,bgx,BHC,BicARE,BiFET,BiGGR,bigmelon,bigmemoryExtras,bioassayR,Biobase,biobroom,bioCancer,BiocCaseStudies,BiocCheck,BiocFileCache,BiocGenerics,biocGraph,BiocInstaller,BiocNeighbors,BiocOncoTK,BioCor,BiocParallel,BiocPkgTools,BiocSklearn,BiocStyle,BiocVersion,biocViews,BiocWorkflowTools,bioDist,biomaRt,biomformat,BioMVCClass,biomvRCNS,BioNet,BioNetStat,BioQC,BioSeqClass,biosigner,Biostrings,biosvd,biotmle,biovizBase,BiRewire,birta,birte,BiSeq,BitSeq,blima,BLMA,bnbc,BPRMeth,BRAIN,brainImageR,BrainStars,branchpointer,breakpointR,bridge,BridgeDbR,BrowserViz,BSgenome,bsseq,BubbleTree,BufferedMatrix,BufferedMatrixMethods,BUMHMM,bumphunter,BUS,BUScorrect,CAFE,CAGEfightR,CAGEr,CALIB,CAMERA,CAMTHC,canceR,cancerclass,CancerInSilico,CancerMutationAnalysis,CancerSubtypes,CAnD,caOmicsV,Cardinal,casper,CATALYST,Category,categoryCompare,CausalR,cbaf,ccfindR,ccmap,CCPROMISE,ccrepe,celaref,cellbaseR,cellGrowth,cellHTS2,cellity,CellMapper,CellNOptR,cellscape,CellScore,CellTrails,cellTree,CEMiTool,CexoR,CFAssay,CGEN,CGHbase,CGHcall,cghMCR,CGHnormaliter,CGHregions,ChAMP,CHARGE,charm,ChemmineOB,ChemmineR,ChIC,Chicago,chimera,chimeraviz,ChIPanalyser,ChIPComp,chipenrich,ChIPexoQual,ChIPpeakAnno,ChIPQC,ChIPseeker,chipseq,ChIPseqR,ChIPSeqSpike,ChIPsim,ChIPXpress,chopsticks,chroGPS,chromDraw,ChromHeatMap,chromPlot,chromstaR,chromswitch,chromVAR,CHRONOS,cicero,CINdex,cisPath,ClassifyR,cleanUpdTSeq,cleaver,clippda,clipper,Clomial,Clonality,clonotypeR,clst,clstutils,clustComp,clusterExperiment,ClusterJudge,clusterProfiler,clusterSeq,ClusterSignificance,clusterStab,CMA,cn.farms,cn.mops,CNAnorm,CNEr,CNORdt,CNORfeeder,CNORfuzzy,CNORode,CNPBayes,CNTools,cnvGSA,CNVPanelizer,CNVrd2,CNVtools,cobindR,CoCiteStats,COCOA,codelink,CODEX,coexnet,CoGAPS,cogena,coGPS,COHCAP,coMET,compartmap,COMPASS,compcodeR,compEpiTools,CompGO,ComplexHeatmap,condcomp,CONFESS,consensus,ConsensusClusterPlus,consensusDE,consensusOV,consensusSeekeR,contiBAIT,conumee,convert,copa,copynumber,CopywriteR,coRdon,CoRegNet,Cormotif,CorMut,CORREP,coseq,cosmiq,COSNet,CountClust,countsimQC,covEB,CoverageView,covRNA,cpvSNP,cqn,CRImage,CRISPRseek,crisprseekplus,CrispRVariants,crlmm,crossmeta,CSAR,csaw,CSSP,ctc,CTDquerier,cTRAP,ctsGE,cummeRbund,customProDB,CVE,cycle,cydar,CytoDx,cytofkit,cytolib,CytoML,dada2,dagLogo,daMA,DaMiRseq,DAPAR,DART,DBChIP,dcGSA,DChIPRep,ddCt,ddPCRclust,debrowser,DECIPHER,DEComplexDisease,DeconRNASeq,decontam,DEDS,DeepBlueR,deepSNV,DEFormats,DEGraph,DEGreport,DEGseq,DelayedArray,DelayedMatrixStats,deltaGseg,DeMAND,DEP,DEqMS,derfinder,derfinderHelper,derfinderPlot,DEScan2,DESeq,DESeq2,DEsingle,destiny,DEsubs,DEXSeq,dexus,DFP,DiffBind,diffcoexp,diffcyt,diffGeneAnalysis,diffHic,DiffLogo,diffloop,diffuStats,diggit,Director,DirichletMultinomial,discordant,dks,DMCHMM,DMRcaller,DMRcate,DMRforPairs,DMRScan,dmrseq,DNABarcodes,DNAcopy,DNAshapeR,DominoEffect,doppelgangR,DOQTL,Doscheda,DOSE,drawProteins,DRIMSeq,DriverNet,DropletUtils,DrugVsDisease,dSimer,DSS,DTA,dualKS,DupChecker,dupRadar,dyebias,DynDoc,EasyqpcR,easyRNASeq,EBarrays,EBcoexpress,EBImage,EBSEA,EBSeq,EBSeqHMM,ecolitk,EDASeq,EDDA,edge,edgeR,eegc,EGAD,EGSEA,eiR,eisa,ELBOW,ELMER,EMDomics,EmpiricalBrownsMethod,ENCODExplorer,EnhancedVolcano,ENmix,EnrichedHeatmap,EnrichmentBrowser,enrichplot,ensembldb,ensemblVEP,ENVISIONQuery,EpiDISH,epigenomix,epiNEM,epivizr,epivizrChart,epivizrData,epivizrServer,epivizrStandalone,erccdashboard,erma,ERSSA,esATAC,esetVis,eudysbiome,EventPointer,ExCluster,ExiMiR,exomeCopy,exomePeak,ExperimentHub,ExperimentHubData,explorase,ExpressionAtlas,ExpressionView,fabia,facopy,factDesign,FamAgg,farms,fastLiquidAssociation,FastqCleaner,fastseg,FCBF,fCCAC,fCI,fdrame,FELLA,FEM,ffpe,FGNet,fgsea,FindMyFriends,FISHalyseR,FitHiC,flagme,flipflop,flowAI,flowBeads,flowBin,flowcatchR,flowCHIC,flowCL,flowClean,flowClust,flowCore,flowCyBar,flowDensity,flowFit,flowFP,flowMap,flowMatch,flowMeans,flowMerge,flowPeaks,flowPloidy,flowPlots,flowQ,flowQB,FlowRepositoryR,FlowSOM,flowStats,flowTime,flowTrans,flowType,flowUtils,flowViz,flowVS,flowWorkspace,fmcsR,focalCall,FoldGO,FourCSeq,FRGEpistasis,frma,frmaTools,FunChIP,FunciSNP,funtooNorm,GA4GHclient,GA4GHshiny,gaga,gage,gaggle,gaia,GAprediction,garfield,GARS,GateFinder,gaucho,gcapc,gcatest,gCMAP,gCMAPWeb,gCrisprTools,gcrma,GDCRNATools,GDSArray,gdsfmt,geecc,GEM,genArise,genbankr,GeneAccord,GeneAnswers,geneAttribution,GeneBreak,geneClassifiers,GeneExpressionSignature,genefilter,genefu,GeneGA,GeneGeneInteR,GeneMeta,GeneNetworkBuilder,GeneOverlap,geneplast,geneplotter,geneRecommender,GeneRegionScan,geneRxCluster,GeneSelectMMD,GeneSelector,GENESIS,GeneStructureTools,geNetClassifier,GeneticsDesign,GeneticsPed,geneXtendeR,GENIE3,genoCN,GenoGAM,genomation,GenomeGraphs,GenomeInfoDb,genomeIntervals,genomes,GenomicAlignments,GenomicDataCommons,GenomicFeatures,GenomicFiles,GenomicInteractions,GenomicRanges,GenomicScores,GenomicTuples,Genominator,genoset,genotypeeval,genphen,GenRank,GenVisR,GEOmetadb,GEOquery,GEOsubmission,gep2pep,gespeR,GEWIST,GGBase,ggbio,ggcyto,GGtools,ggtree,GIGSEA,girafe,GISPA,GLAD,Glimma,glmSparseNet,GlobalAncova,globalSeq,globaltest,gmapR,GMRP,GOexpress,GOfuncR,GOFunction,GoogleGenomics,GOpro,goProfiles,GOSemSim,goseq,GOSim,goSTAG,GOstats,GOsummaries,GOTHiC,goTools,gpart,gpls,gprege,gQTLBase,gQTLstats,graph,GraphAlignment,GraphAT,graphite,GraphPAC,GRENITS,GreyListChIP,GRmetrics,groHMM,GRridge,GSALightning,GSAR,GSCA,GSEABase,GSEABenchmarkeR,GSEAlm,gsean,GSReg,GSRI,GSVA,gtrellis,GUIDEseq,Guitar,Gviz,gwascat,GWASTools,gwasurvivr,h5vc,hapFabia,Harman,Harshlight,HDF5Array,HDTD,heatmaps,Heatplus,HelloRanges,HELP,HEM,hiAnnotator,HIBAG,HiCBricks,HiCcompare,hicrep,hierGWAS,hierinf,HilbertCurve,HilbertVis,HilbertVisGUI,hipathia,hiReadsProcessor,HIREewas,HiTC,hmdbQuery,HMMcopy,hopach,HPAanalyze,hpar,HTqPCR,HTSanalyzeR,HTSeqGenie,htSeqTools,HTSFilter,HybridMTest,hyperdraw,hypergraph,iASeq,iasva,iBBiG,ibh,iBMQ,iCARE,Icens,icetea,iCheck,iChip,iClusterPlus,iCNV,iCOBRA,ideal,IdeoViz,idiogram,IdMappingAnalysis,IdMappingRetrieval,iGC,igvR,IHW,illuminaio,imageHTS,IMAS,Imetagene,IMMAN,ImmuneSpaceR,immunoClust,IMPCdata,ImpulseDE,ImpulseDE2,impute,INDEED,InPAS,INPower,INSPEcT,InTAD,intansv,InteractionSet,interactiveDisplay,interactiveDisplayBase,IntEREst,InterMineR,IntramiRExploreR,inveRsion,IONiseR,iPAC,ipdDb,IPO,IPPD,IRanges,IrisSpatialFeatures,iSEE,iSeq,isobar,IsoCorrectoR,IsoformSwitchAnalyzeR,IsoGeneGUI,ISoLDE,isomiRs,ITALICS,iterativeBMA,iterativeBMAsurv,iterClust,iteremoval,IVAS,ivygapSE,IWTomics,joda,JunctionSeq,karyoploteR,KCsmart,kebabs,KEGGgraph,KEGGlincs,keggorthology,KEGGprofile,KEGGREST,kimod,KinSwingR,kissDE,lapmix,LBE,ldblock,LEA,LedPred,les,levi,lfa,limma,limmaGUI,LINC,LineagePulse,Linnorm,LiquidAssociation,lmdme,LMGene,LOBSTAHS,loci2path,logicFS,logitT,Logolas,lol,LOLA,LoomExperiment,LowMACA,LPE,LPEadj,lpNet,lpsymphony,LRBaseDbi,lumi,LVSmiRNA,LymphoSeq,M3C,M3D,M3Drop,maanova,macat,maCorrPlot,MACPET,made4,MADSEQ,maftools,MAGeCKFlute,maigesPack,MAIT,makecdfenv,MANOR,manta,MantelCorr,mAPKL,maPredictDSC,mapscape,marray,martini,maser,maSigPro,maskBAD,MassArray,massiR,MassSpecWavelet,MAST,matchBox,MatrixRider,matter,MaxContrastProjection,MBAmethyl,MBASED,MBCB,mBPCR,MBttest,mcaGUI,MCbiclust,MCRestimate,mCSEA,mdgsa,mdp,mdqc,MDTS,MEAL,MeasurementError.cor,MEDIPS,MEDME,MEIGOR,MergeMaid,Mergeomics,MeSHDbi,meshes,meshr,messina,metaArray,Metab,metabomxtr,MetaboSignal,metaCCA,MetaCyto,metagene,metagenomeFeatures,metagenomeSeq,metahdep,metaMS,MetaNeighbor,metaSeq,metaseqR,metavizr,MetCirc,methimpute,methInheritSim,MethPed,MethTargetedNGS,methVisual,methyAnalysis,MethylAid,methylGSA,methylInheritance,methylKit,MethylMix,methylMnM,methylPipe,MethylSeekR,methylumi,methyvim,MetID,MetNet,mfa,Mfuzz,MGFM,MGFR,mgsa,MiChip,microbiome,microRNA,MIGSA,mimager,MIMOSA,MineICA,minet,minfi,MinimumDistance,MiPP,MIRA,MiRaGE,miRBaseConverter,miRcomp,mirIntegrator,miRLAB,miRmine,miRNAmeConverter,miRNApath,miRNAtap,miRSM,miRsponge,Mirsynergy,missMethyl,missRows,mitoODE,mixOmics,MLInterfaces,mlm4omics,MLP,MLSeq,MMDiff2,MmPalateMiRNA,MODA,mogsa,monocle,MoonlightR,MoPS,mosaics,motifbreakR,motifcounter,MotifDb,motifmatchr,motifRG,motifStack,MotIV,MPFE,mpra,MPRAnalyze,mQTL.NMR,msa,msgbsR,MSGFgui,MSGFplus,msmsEDA,msmsTests,MSnbase,MSnID,msPurity,MSstats,MSstatsQC,MSstatsQCgui,MSstatsTMT,MTseeker,Mulcom,MultiAssayExperiment,multiClust,MultiDataSet,multiHiCcompare,MultiMed,multiMiR,multiOmicsViz,multiscan,multtest,muscle,MutationalPatterns,MVCClass,MWASTools,mygene,myvariant,mzID,mzR,NADfinder,NanoStringDiff,NanoStringQCPro,NarrowPeaks,NBSplice,ncdfFlow,NCIgraph,ndexr,NeighborNet,nem,netbenchmark,netbiov,nethet,NetPathMiner,netprioR,netReg,netresponse,NetSAM,netSmooth,networkBMA,NGScopy,nnNorm,NOISeq,nondetects,normalize450K,NormalyzerDE,NormqPCR,normr,npGSEA,NTW,nucleoSim,nucleR,nuCpos,nudge,NuPoP,occugene,OCplus,odseq,OGSA,oligo,oligoClasses,OLIN,OLINgui,OmaDB,omicade4,OmicCircos,omicplotR,omicRexposome,OmicsMarkeR,OMICsPCA,omicsPrint,Onassis,oncomix,OncoScore,OncoSimulR,oneSENSE,onlineFDR,ontoProc,openCyto,openPrimeR,openPrimeRui,oposSOM,oppar,OPWeight,OrderedList,ORFik,Organism.dplyr,OrganismDbi,OSAT,Oscope,OTUbase,OutlierD,OUTRIDER,PAA,PADOG,paircompviz,pandaR,panelcn.mops,panp,PANR,PanVizGenerator,PAPi,parglms,parody,Path2PPI,pathifier,PathNet,PathoStat,pathprint,pathRender,pathVar,pathview,PathwaySplice,paxtoolsr,Pbase,pbcmc,pcaExplorer,pcaGoPromoter,pcaMethods,PCAN,pcot2,PCpheno,pcxn,pdInfoBuilder,PECA,PepsNMR,pepStat,pepXMLTab,perturbatr,PGA,pgca,PGSEA,phantasus,PharmacoGx,phenopath,phenoTest,PhenStat,philr,phosphonormalizer,phyloseq,Pi,piano,pickgene,PICS,Pigengene,PING,pint,pkgDepTools,plateCore,plethy,plgem,plier,plotGrouper,PLPE,plrs,plw,plyranges,pmm,podkat,pogos,polyester,Polyfit,POST,PowerExplorer,powerTCR,PPInfer,ppiStats,pqsfinder,prada,prebs,PREDA,predictionet,preprocessCore,primirTSS,Prize,proBAMr,PROcess,procoil,ProCoNA,proFIA,profileScoreDist,progeny,pRoloc,pRolocGUI,PROMISE,PROPER,PROPS,Prostar,prot2D,proteinProfiles,ProteomicsAnnotationHubData,ProteoMM,proteoQC,ProtGenerics,PSEA,psichomics,PSICQUIC,psygenet2r,puma,PureCN,pvac,pvca,Pviz,PWMEnrich,pwOmics,qcmetrics,QDNAseq,qpcrNorm,qpgraph,qPLEXanalyzer,qrqc,qsea,QSutils,QUALIFIER,quantro,quantsmooth,QuartPAC,QuasR,QuaternaryProd,QUBIC,qusage,qvalue,R3CPET,r3Cseq,R453Plus1Toolbox,R4RNA,RaggedExperiment,rain,rama,RamiGO,ramwas,RandomWalkRestartMH,randPack,RankProd,RareVariantVis,Rariant,RbcBook1,RBGL,RBioinf,rBiopaxParser,RBM,Rbowtie,Rbowtie2,rbsurv,Rcade,RCAS,RCASPAR,rcellminer,rCGH,Rchemcpp,RchyOptimyx,RcisTarget,Rcpi,RCy3,RCyjs,RDAVIDWebService,rDGIdb,Rdisop,RDRToolbox,ReactomePA,readat,ReadqPCR,reb,REBET,recount,recoup,RedeR,REDseq,RefNet,RefPlus,regioneR,regionReport,regsplice,REMP,Repitools,ReportingTools,ReQON,restfulSE,rexposome,rfPred,rGADEM,RGalaxy,Rgin,RGMQL,RGraph2js,Rgraphviz,rGREAT,RGSEA,rgsepd,rhdf5,rhdf5client,Rhdf5lib,Rhtslib,rHVDM,RiboProfiling,riboSeqR,RImmPort,Ringo,RIPSeeker,Risa,RITAN,RIVER,RJMCMCNucleosomes,RLMM,Rmagpie,RMassBank,rMAT,RmiR,Rmmquant,RNAdecay,RNAinteract,RNAither,RNAprobR,rnaseqcomp,rnaSeqMap,RNASeqPower,RNASeqR,RnaSeqSampleSize,RnBeads,Rnits,roar,ROC,Roleswitch,rols,ROntoTools,ropls,ROTS,RPA,RProtoBufLib,RpsiXML,rpx,Rqc,rqt,rqubic,rRDP,RRHO,Rsamtools,rsbml,RSeqAn,rSFFreader,Rsubread,RSVSim,rTANDEM,RTCA,RTCGA,RTCGAToolbox,RTN,RTNduals,RTNsurvival,RTopper,rtracklayer,Rtreemix,rTRM,rTRMui,runibic,RUVcorr,RUVnormalize,RUVSeq,RVS,rWikiPathways,S4Vectors,safe,sagenhaft,SAGx,samExploreR,sampleClassifier,SamSPECTRAL,sangerseqR,SANTA,sapFinder,savR,SBMLR,SC3,Scale4C,SCAN.UPC,scater,SCBN,scDD,scde,scFeatureFilter,scfind,ScISI,scmap,scmeth,SCnorm,scone,Sconify,scoreInvHap,scPipe,scran,scruff,scsR,SDAMS,segmentSeq,SELEX,SemDist,semisup,SEPA,SEPIRA,seq2pathway,SeqArray,seqbias,seqCAT,seqCNA,seqcombo,SeqGSEA,seqLogo,seqPattern,seqplots,seqsetvis,SeqSQC,seqTools,SeqVarTools,sesame,sevenbridges,sevenC,SGSeq,shinyMethyl,shinyTANDEM,ShortRead,SIAMCAT,SICtools,sigaR,SigCheck,sigFeature,SigFuge,siggenes,sights,signeR,signet,sigPathway,sigsquared,SIM,SIMAT,SimBindProfiles,SIMD,similaRpeak,SIMLR,simpleaffy,simulatorZ,sincell,SingleCellExperiment,singleCellTK,singscore,SISPA,sizepower,skewr,slalom,SLGI,slingshot,slinky,SLqPCR,SMAP,SMITE,SNAGEE,snapCGH,snm,SNPchip,SNPediaR,SNPhood,SNPRelate,snpStats,soGGi,SomaticSignatures,SpacePAC,sparseDOSSA,sparsenetgls,SparseSignatures,specL,SpeCond,SPEM,SPIA,SpidermiR,spikeLI,spkTools,splatter,splicegear,spliceSites,SplicingGraphs,splineTimeR,SPLINTER,splots,SPONGE,spotSegmentation,SQUADD,SRAdb,sRAP,SRGnet,srnadiff,sscore,sscu,sSeq,ssize,SSPA,ssviz,stageR,STAN,staRank,StarBioTrek,Starr,STATegRa,statTarget,stepNorm,strandCheckR,Streamer,STRINGdb,STROMA4,subSeq,SummarizedBenchmark,SummarizedExperiment,supraHex,survcomp,Sushi,sva,SVAPLSseq,SVM2CRM,SWATH2stats,SwathXtend,swfdr,SwimR,switchBox,switchde,synapter,synergyfinder,synlet,systemPipeR,TargetScore,TargetSearch,TarSeqQC,TCC,TCGAbiolinks,TCGAbiolinksGUI,TCGAutils,TCseq,TDARACNE,tenXplore,TEQC,ternarynet,TFARM,TFBSTools,TFEA.ChIP,TFHAZ,TFutils,tigre,tilingArray,timecourse,timescape,TimeSeriesExperiment,TIN,TissueEnrich,TitanCNA,tkWidgets,TMixClust,TnT,tofsims,ToPASeq,topdownr,topGO,TPP,tracktables,trackViewer,transcriptogramer,transcriptR,transite,tRanslatome,TransView,traseR,treeio,trena,Trendy,triform,trigger,trio,triplex,tRNA,tRNAdbImport,tRNAscanImport,TRONCO,TSCAN,tspair,TSRchitect,TSSi,TTMap,TurboNorm,TVTB,tweeDEseq,twilight,twoddpcr,tximeta,tximport,TxRegInfra,TypeInfo,Ularcirc,UNDO,unifiedWMWqPCR,UniProt.ws,Uniquorn,universalmotif,uSORT,VanillaICE,variancePartition,VariantAnnotation,VariantFiltering,VariantTools,vbmp,Vega,VegaMC,vidger,viper,vsn,vtpnet,vulcan,wateRmelon,wavClusteR,waveTiling,weaver,webbioc,widgetTools,wiggleplotr,Wrench,XBSeq,xcms,XDE,XINA,xmapbridge,xps,XVector,yamss,YAPSA,yaqcaffy,yarn,zFPKM,zinbwave,zlibbioc" id="Software" class="jstree-open"><ins class="jstree-icon">&nbsp;</ins><a href="https://www.bioconductor.org/packages/release/BiocViews.html#" class="jstree-clicked"><ins class="jstree-icon">&nbsp;</ins>Software (1649)</a><ul><li packagelist="anamiR,iGC,MetID,nondetects,pgca" id="AssayDomain" class="jstree-closed"><ins class="jstree-icon">&nbsp;</ins><a href="https://www.bioconductor.org/packages/release/BiocViews.html#" class=""><ins class="jstree-icon">&nbsp;</ins>AssayDomain (661)</a><ul><li packagelist="biomvRCNS,CINdex,Clonality,copynumber,gaia,GeneBreak,KCsmart,mBPCR,pint,RCASPAR,rcellminer,rCGH,sigaR,Vega,VegaMC" id="aCGH" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="https://www.bioconductor.org/packages/release/BiocViews.html#"><ins class="jstree-icon">&nbsp;</ins>aCGH (15)</a></li><li packagelist="bioassayR,cellGrowth,cellHTS2,CellNOptR,CFAssay,ChemmineOB,ChemmineR,cicero,CNORdt,CNORfeeder,CNORode,contiBAIT,CytoDx,destiny,diffcyt,eiR,flowBeads,flowBin,flowCHIC,flowCore,flowCyBar,flowFit,flowFP,flowPlots,flowQ,FlowSOM,flowStats,flowTime,flowUtils,flowViz,flowVS,fmcsR,gespeR,ggcyto,GRmetrics,HTSanalyzeR,imageHTS,immunoClust,ImpulseDE2,iSEE,LineagePulse,MaxContrastProjection,MetaCyto,MIMOSA,mitoODE,MPRAnalyze,netprioR,PANR,perturbatr,plateCore,prada,QUALIFIER,rcellminer,Rchemcpp,RNAinteract,RNAither,RTCA,sights,staRank,synlet" id="CellBasedAssays" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="https://www.bioconductor.org/packages/release/BiocViews.html#"><ins class="jstree-icon">&nbsp;</ins>CellBasedAssays (60)</a></li><li packagelist="ChIPpeakAnno,ChIPXpress,chroGPS,iChip,les,motifStack,MotIV,rGADEM,sevenC,Starr" id="ChIPchip" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="https://www.bioconductor.org/packages/release/BiocViews.html#"><ins class="jstree-icon">&nbsp;</ins>ChIPchip (10)</a></li><li packagelist="ACE,aCGH,AneuFinder,ArrayTV,beadarraySNP,BiocOncoTK,biomvRCNS,BubbleTree,CGHbase,cghMCR,CGHregions,CINdex,Clonality,cn.farms,cn.mops,CNAnorm,CNPBayes,CNTools,CNVPanelizer,CNVrd2,CODEX,conumee,copynumber,CopywriteR,crlmm,DNAcopy,exomeCopy,facopy,fastseg,gaia,GeneBreak,genoset,GenRank,GLAD,HMMcopy,iCNV,iGC,ITALICS,karyoploteR,KCsmart,MADSEQ,MANOR,mBPCR,MEDIPS,MinimumDistance,mlm4omics,NGScopy,OGSA,panelcn.mops,PREDA,PureCN,QDNAseq,qsea,quantsmooth,rcellminer,rCGH,reb,regioneR,seqCNA,SMAP,snapCGH,SNPchip,sparsenetgls,TitanCNA,VanillaICE,Vega,VegaMC" id="CopyNumberVariation" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="https://www.bioconductor.org/packages/release/BiocViews.html#"><ins class="jstree-icon">&nbsp;</ins>CopyNumberVariation (67)</a></li><li packagelist="bigmelon,BiocOncoTK,DeepBlueR,genomation,HELP,M3D,MEDIPS,MEDME,methylumi,qsea,RnBeads,TurboNorm" id="CpGIsland" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="https://www.bioconductor.org/packages/release/BiocViews.html#"><ins class="jstree-icon">&nbsp;</ins>CpGIsland (12)</a></li><li packagelist="ARRmNormalization,BEAT,BEclear,bigmelon,BiocOncoTK,BiSeq,BPRMeth,bsseq,bumphunter,ChAMP,charm,COCOA,COHCAP,coMET,conumee,DeepBlueR,DMRcaller,DMRcate,DMRforPairs,dmrseq,DSS,edgeR,ELMER,ENmix,EpiDISH,funtooNorm,GAprediction,GEM,Harman,HELP,HIREewas,iCheck,les,lumi,M3D,MassArray,MBAmethyl,mCSEA,MEAL,MEDIPS,MEDME,methimpute,methInheritSim,MethPed,methVisual,methyAnalysis,MethylAid,methylGSA,methylInheritance,methylKit,MethylMix,methylMnM,methylPipe,MethylSeekR,methylumi,methyvim,minfi,MIRA,missMethyl,MoonlightR,MPFE,normalize450K,omicsPrint,qsea,ramwas,REMP,Repitools,RnBeads,scmeth,sesame,shinyMethyl,skewr,TCGAbiolinks,TCGAbiolinksGUI,TransView,TurboNorm,wateRmelon" id="DNAMethylation" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="https://www.bioconductor.org/packages/release/BiocViews.html#"><ins class="jstree-icon">&nbsp;</ins>DNAMethylation (77)</a></li><li packagelist="globalSeq,limma,oligo,PECA,puma,regsplice,siggenes,snm,TIN,xps" id="ExonArray" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="https://www.bioconductor.org/packages/release/BiocViews.html#"><ins class="jstree-icon">&nbsp;</ins>ExonArray (10)</a></li><li packagelist="ABAEnrichment,acde,adaptest,AffiXcan,affycoretools,affylmGUI,AffyRNADegradation,AGDEX,AIMS,ALDEx2,alpine,AMOUNTAIN,ampliQueso,anamiR,Anaquin,anota,anota2seq,antiProfiles,apeglm,ASpli,ASSIGN,attract,AUCell,BASiCS,BayesKnockdown,BgeeDB,biobroom,bioCancer,BiocOncoTK,BioCor,BioNet,BioNetStat,BioQC,biotmle,birta,birte,BitSeq,blima,BPRMeth,brainImageR,BUMHMM,BUScorrect,CAFE,CAGEfightR,CAGEr,CAMTHC,canceR,CancerInSilico,CancerSubtypes,casper,categoryCompare,ccmap,CCPROMISE,cellity,CellMapper,CellScore,CellTrails,cellTree,CEMiTool,CHARGE,chromswitch,clustComp,ClusterJudge,clusterSeq,coexnet,CoGAPS,cogena,compEpiTools,CONFESS,consensus,consensusOV,coRdon,CoRegNet,coseq,CountClust,covEB,covRNA,crossmeta,cTRAP,ctsGE,cummeRbund,DART,dcGSA,ddCt,debrowser,DEComplexDisease,DEFormats,DEGreport,DEGseq,DeMAND,DESeq2,DEsingle,DEsubs,DEXSeq,dexus,diffcoexp,diffuStats,diggit,DMRcate,doppelgangR,DRIMSeq,DropletUtils,DrugVsDisease,DSS,DTA,EasyqpcR,easyRNASeq,EBSEA,EBSeqHMM,edge,edgeR,eegc,EGSEA,eisa,ELBOW,ELMER,EMDomics,EmpiricalBrownsMethod,EnhancedVolcano,EnrichmentBrowser,epigenomix,erccdashboard,ERSSA,ExiMiR,ExpressionView,farms,fastLiquidAssociation,FCBF,ffpe,fgsea,FoldGO,gage,gCMAPWeb,GDCRNATools,geecc,GEM,geneClassifiers,GeneExpressionSignature,genefu,GeneGA,GeneMeta,GENIE3,GenRank,gep2pep,GGtools,GIGSEA,Glimma,globalSeq,GOexpress,GOpro,goProfiles,goseq,goSTAG,GOstats,GOsummaries,GRENITS,GRridge,GSALightning,GSCA,GSEABase,GSEABenchmarkeR,gsean,GSReg,HDTD,hipathia,HTqPCR,HTSFilter,HybridMTest,iBMQ,icetea,iCheck,ideal,iGC,IMAS,ImpulseDE2,InTAD,interactiveDisplay,interactiveDisplayBase,InterMineR,IntramiRExploreR,iSEE,IsoformSwitchAnalyzeR,ISoLDE,IVAS,ivygapSE,KEGGlincs,kimod,levi,limma,limmaGUI,LINC,LineagePulse,Linnorm,LiquidAssociation,loci2path,M3C,M3Drop,manta,mAPKL,massiR,MAST,MBASED,MBttest,MCbiclust,messina,MetaNeighbor,metaseqR,MethylMix,mfa,MGFM,MGFR,MIGSA,MiRaGE,miRLAB,miRSM,miRsponge,MODA,mogsa,monocle,MoonlightR,MPRAnalyze,Mulcom,multiClust,NeighborNet,netbenchmark,netresponse,netSmooth,networkBMA,nondetects,NormqPCR,OGSA,oligo,omicplotR,omicRexposome,oncomix,oposSOM,oppar,Oscope,OUTRIDER,pandaR,pathprint,pathview,pbcmc,pcaGoPromoter,pcot2,PECA,phantasus,PharmacoGx,phenopath,Pi,piano,Pigengene,pint,plgem,Polyfit,POST,prebs,PREDA,Prize,progeny,PROMISE,PROPS,psichomics,puma,qpcrNorm,qpgraph,QuaternaryProd,QUBIC,R3CPET,RankProd,RBioinf,Rcade,RCASPAR,rcellminer,readat,ReadqPCR,recount,recoup,Repitools,RIVER,RmiR,Rmmquant,RNAdecay,rnaSeqMap,RNASeqR,RnaSeqSampleSize,Rnits,ROTS,RPA,Rsubread,RTCGAToolbox,RTN,RTNduals,runibic,RUVcorr,SAGx,samExploreR,sampleClassifier,scater,SCBN,scFeatureFilter,scfind,scmap,scone,scPipe,scran,SEPA,SEPIRA,SeqGSEA,sigaR,SigCheck,sigFeature,siggenes,signet,SIMLR,sincell,singleCellTK,singscore,slalom,slingshot,slinky,snm,splatter,spliceSites,SplicingGraphs,splineTimeR,SPLINTER,SPONGE,sRAP,srnadiff,sscu,SSPA,STROMA4,subSeq,supraHex,survcomp,SVAPLSseq,switchde,systemPipeR,TCGAbiolinks,TCGAbiolinksGUI,tigre,TimeSeriesExperiment,TIN,TissueEnrich,TMixClust,ToPASeq,transcriptogramer,transite,tRanslatome,TransView,trena,trigger,TSCAN,TSRchitect,tximeta,tximport,unifiedWMWqPCR,variancePartition,vidger,viper,vulcan,waveTiling,wiggleplotr,xps,yarn,zFPKM,zinbwave" id="GeneExpression" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="https://www.bioconductor.org/packages/release/BiocViews.html#"><ins class="jstree-icon">&nbsp;</ins>GeneExpression (368)</a></li><li packagelist="appreci8R,BBCAnalyzer,beadarraySNP,BiocOncoTK,BUMHMM,CAnD,chopsticks,Clomial,CNVtools,CrispRVariants,deepSNV,DOQTL,FRGEpistasis,GeneAccord,GeneGeneInteR,GENESIS,GGtools,GSReg,GWASTools,gwasurvivr,hapFabia,ISoLDE,martini,metagenomeSeq,missMethyl,OMICsPCA,omicsPrint,pathVar,phyloseq,qpgraph,QSutils,Rgin,RIVER,RLMM,Rsubread,RTN,samExploreR,SeqVarTools,SIAMCAT,SNPchip,snpStats,trigger,trio,TVTB,VariantTools" id="GeneticVariability" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="https://www.bioconductor.org/packages/release/BiocViews.html#"><ins class="jstree-icon">&nbsp;</ins>GeneticVariability (45)</a></li><li packagelist="appreci8R,ASAFE,BBCAnalyzer,beadarraySNP,BiocOncoTK,BSgenome,CAnD,CGEN,chopsticks,CNVrd2,copynumber,crlmm,customProDB,deepSNV,DOQTL,DRIMSeq,ensemblVEP,gaucho,gcatest,GEM,geneAttribution,GeneGeneInteR,GeneRegionScan,GENESIS,GenomicAlignments,genoset,genotypeeval,GenRank,GGtools,GIGSEA,GMRP,GWASTools,gwasurvivr,hapFabia,hierGWAS,hierinf,iASeq,iBMQ,iCNV,IMAS,INPower,InterMineR,inveRsion,ISoLDE,IVAS,lfa,logicFS,martini,MassArray,mBPCR,metaCCA,MinimumDistance,mQTL.NMR,oligo,OPWeight,qpgraph,REBET,Rgin,RIVER,RLMM,Rsubread,RTN,sangerseqR,sapFinder,scoreInvHap,semisup,SeqVarTools,SICtools,siggenes,SNPchip,SNPediaR,snpStats,systemPipeR,trigger,trio,VariantAnnotation,VariantFiltering" id="SNP" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="https://www.bioconductor.org/packages/release/BiocViews.html#"><ins class="jstree-icon">&nbsp;</ins>SNP (77)</a></li><li packagelist="AffiXcan,affylmGUI,alpine,altcdfenvs,ampliQueso,annmap,ASpli,AUCell,BAC,BicARE,BiFET,BiocOncoTK,birta,birte,BitSeq,brainImageR,BUMHMM,CAGEfightR,CAGEr,casper,ccmap,CellScore,CexoR,ChIPanalyser,ChIPComp,ChIPexoQual,ChIPSeqSpike,chromswitch,CoGAPS,consensusSeekeR,CoRegNet,covRNA,crossmeta,CSAR,ctsGE,customProDB,DBChIP,DEFormats,DESeq2,diffcoexp,DTA,edgeR,ELMER,EnhancedVolcano,erccdashboard,ERSSA,EventPointer,ExiMiR,fCCAC,gCMAPWeb,geecc,GOexpress,goseq,GSALightning,GSRI,Guitar,Harman,icetea,IMAS,InPAS,iSEE,IsoformSwitchAnalyzeR,ISoLDE,IVAS,ivygapSE,les,limma,Linnorm,M3C,MBASED,MBttest,mosaics,motifcounter,motifRG,NarrowPeaks,netresponse,OMICsPCA,omicsPrint,pandaR,pathprint,primirTSS,psichomics,pwOmics,qpgraph,QuaternaryProd,Rcade,RcisTarget,regionReport,rHVDM,RIVER,Rmmquant,RNAdecay,rnaSeqMap,roar,RRHO,rTRM,rTRMui,SC3,scde,scfind,scmap,SELEX,SEPIRA,SGSeq,sigFeature,simpleaffy,snm,splicegear,SplicingGraphs,SPONGE,STAN,subSeq,TFARM,TFBSTools,TFEA.ChIP,TFHAZ,tigre,TimeSeriesExperiment,transcriptogramer,transcriptR,transite,TransView,trena,TSRchitect,tximeta,tximport,variancePartition,wiggleplotr,xps" id="Transcription" class="jstree-last jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="https://www.bioconductor.org/packages/release/BiocViews.html#"><ins class="jstree-icon">&nbsp;</ins>Transcription (129)</a></li></ul></li><li packagelist="anamiR,ChIPanalyser,consensusSeekeR,discordant,geecc,GSALightning,INDEED,LINC,methInheritSim,methylInheritance,MetID,OMICsPCA,OncoSimulR,rexposome,RJMCMCNucleosomes,similaRpeak,TFARM,TFHAZ,YAPSA" id="BiologicalQuestion" class="jstree-closed"><ins class="jstree-icon">&nbsp;</ins><a href="https://www.bioconductor.org/packages/release/BiocViews.html#"><ins class="jstree-icon">&nbsp;</ins>BiologicalQuestion (668)</a><ul><li packagelist="alpine,ASpli,BitSeq,customProDB,DEXSeq,DRIMSeq,edgeR,erccdashboard,EventPointer,flipflop,GeneStructureTools,GSReg,IMAS,InPAS,IntEREst,InterMineR,IsoformSwitchAnalyzeR,IVAS,kissDE,limma,maser,MBttest,NBSplice,PathwaySplice,psichomics,puma,regsplice,SGSeq,SplicingGraphs,SPLINTER,TIN,Ularcirc,wiggleplotr" id="AlternativeSplicing" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="https://www.bioconductor.org/packages/release/BiocViews.html#"><ins class="jstree-icon">&nbsp;</ins>AlternativeSplicing (33)</a></li><li packagelist="ACE,alpine,ASpli,ATACseqQC,BadRegionFinder,bamsignals,Basic4Cseq,BasicSTARRseq,BBCAnalyzer,BPRMeth,breakpointR,BubbleTree,BUMHMM,CAGEfightR,ChIPanalyser,ChIPComp,ChIPexoQual,chipseq,ChIPSeqSpike,CNAnorm,CNVPanelizer,CNVrd2,compEpiTools,consensusSeekeR,CopywriteR,csaw,DEScan2,diffcoexp,diffHic,DMCHMM,DMRcaller,DropletUtils,edgeR,EnrichedHeatmap,ensembldb,esATAC,fCCAC,geneXtendeR,GenomicAlignments,GenomicFiles,GenomicRanges,GenomicScores,GreyListChIP,Guitar,HDF5Array,HelloRanges,HilbertCurve,Imetagene,InPAS,IntEREst,karyoploteR,loci2path,M3D,MADSEQ,MBttest,MDTS,MEDIPS,metagene,methimpute,methylPipe,MIRA,nucleR,ORFik,plyranges,PowerExplorer,PureCN,QuasR,ramwas,RCAS,recount,recoup,regionReport,restfulSE,rGREAT,RiboProfiling,RJMCMCNucleosomes,RNAprobR,Rsamtools,runibic,Scale4C,scater,scone,seqCAT,sevenC,SICtools,SMITE,soGGi,srnadiff,strandCheckR,SummarizedExperiment,systemPipeR,TarSeqQC,TFHAZ,topdownr,transcriptR,traseR,TVTB,Ularcirc,wiggleplotr" id="Coverage" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="https://www.bioconductor.org/packages/release/BiocViews.html#"><ins class="jstree-icon">&nbsp;</ins>Coverage (99)</a></li><li class="jstree-leaf" style="display: none;"><ins class="jstree-icon">&nbsp;</ins><a href="https://www.bioconductor.org/packages/release/BiocViews.html#"><ins class="jstree-icon">&nbsp;</ins>DemethylateRegionDetection</a></li><li class="jstree-leaf" style="display: none;"><ins class="jstree-icon">&nbsp;</ins><a href="https://www.bioconductor.org/packages/release/BiocViews.html#"><ins class="jstree-icon">&nbsp;</ins>DenovoAssembler</a></li><li packagelist="ABSSeq,acde,adaptest,AffyExpress,affylmGUI,AgiMicroRna,ALDEx2,AMOUNTAIN,ampliQueso,anamiR,Anaquin,anota,anota2seq,apeglm,ArrayTools,artMS,ASpli,BADER,ballgown,BASiCS,BatchQC,baySeq,BGmix,bgx,biobroom,BioNet,BioNetStat,biotmle,BitSeq,blima,BLMA,bridge,casper,CATALYST,CausalR,ccmap,CellTrails,clippda,clusterSeq,coexnet,CoGAPS,coGPS,compcodeR,consensusOV,copa,Cormotif,cqn,crossmeta,cTRAP,ctsGE,cummeRbund,daMA,DART,ddCt,debrowser,DEComplexDisease,DeconRNASeq,DEDS,DEFormats,DEGraph,DEGreport,DEGseq,DEP,DEqMS,derfinder,derfinderHelper,derfinderPlot,DESeq,DESeq2,DEsingle,DEsubs,DEXSeq,dexus,DFP,diffcoexp,diffGeneAnalysis,DMRcate,DRIMSeq,DSS,DTA,EBarrays,EBSEA,EBSeq,EBSeqHMM,EDASeq,edge,edgeR,eegc,EGSEA,EMDomics,EnhancedVolcano,EnrichmentBrowser,epigenomix,erccdashboard,ERSSA,fabia,factDesign,fdrame,FEM,fgsea,flagme,FoldGO,gaga,gage,gCMAPWeb,gCrisprTools,GDCRNATools,genefu,GeneSelectMMD,GeneSelector,geNetClassifier,GenoGAM,gep2pep,GIGSEA,Glimma,GlobalAncova,globalSeq,globaltest,GOexpress,goSTAG,gprege,GSALightning,GSAR,GSEABenchmarkeR,gsean,GSRI,HDTD,HEM,HTqPCR,HTSFilter,icetea,iCheck,ideal,iGC,IMAS,ImpulseDE2,INDEED,IntEREst,InterMineR,IsoformSwitchAnalyzeR,IsoGeneGUI,isomiRs,IVAS,IWTomics,JunctionSeq,lapmix,les,limma,limmaGUI,LineagePulse,Linnorm,lmdme,LMGene,logitT,LPE,M3Drop,maanova,macat,maigesPack,manta,mAPKL,MAST,MBttest,MEDIPS,MergeMaid,messina,metaArray,metagenomeSeq,metahdep,metaSeq,metaseqR,MethylMix,miRNApath,MmPalateMiRNA,MODA,monocle,MoonlightR,MPRAnalyze,Mulcom,multtest,NanoStringDiff,NBSplice,netresponse,NOISeq,NormalyzerDE,nudge,OCplus,oligo,omicplotR,omicRexposome,oposSOM,OrderedList,PathNet,pathview,pbcmc,pcot2,PECA,phantasus,phenoTest,piano,pickgene,pint,plgem,PLPE,plw,polyester,Polyfit,PowerExplorer,PROPER,prot2D,ProteoMM,psichomics,puma,QUBIC,RankProd,RBM,Rcade,RDAVIDWebService,recount,regionReport,regsplice,rgsepd,RNAdecay,rnaSeqMap,RNASeqR,RnaSeqSampleSize,Rnits,ROC,ROTS,RTCGAToolbox,RUVSeq,safe,SAGx,SC3,SCBN,scDD,scde,SDAMS,segmentSeq,SeqGSEA,sigaR,siggenes,signet,sigPathway,SIMD,similaRpeak,simpleaffy,singleCellTK,slingshot,slinky,SMITE,snm,SpeCond,spliceSites,splineTimeR,sRAP,srnadiff,sscore,ssize,statTarget,subSeq,survcomp,switchde,TCC,TCGAbiolinks,TCGAbiolinksGUI,TCseq,timecourse,TimeSeriesExperiment,ToPASeq,transcriptogramer,transite,tRanslatome,TTMap,tweeDEseq,twilight,unifiedWMWqPCR,variancePartition,vidger,waveTiling,webbioc,XBSeq,XDE,xps" id="DifferentialExpression" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="https://www.bioconductor.org/packages/release/BiocViews.html#"><ins class="jstree-icon">&nbsp;</ins>DifferentialExpression (284)</a></li><li packagelist="ChIPSeqSpike,COHCAP,coMET,DMCHMM,DMRcaller,DMRcate,DMRforPairs,dmrseq,DSS,edgeR,ENmix,EpiDISH,FEM,HIREewas,IWTomics,M3D,mCSEA,methInheritSim,methylGSA,methylInheritance,MethylMix,methyvim,minfi,missMethyl,MoonlightR,msgbsR,omicRexposome,qsea,ramwas,regionReport,REMP,RnBeads,SIMD,SMITE,TCGAbiolinks,TCGAbiolinksGUI" id="DifferentialMethylation" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="https://www.bioconductor.org/packages/release/BiocViews.html#"><ins class="jstree-icon">&nbsp;</ins>DifferentialMethylation (36)</a></li><li packagelist="chromstaR,chromswitch,csaw,derfinder,DiffBind,geneXtendeR,GenoGAM,GreyListChIP,IWTomics,MMDiff2,normr,regionReport" id="DifferentialPeakCalling" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="https://www.bioconductor.org/packages/release/BiocViews.html#"><ins class="jstree-icon">&nbsp;</ins>DifferentialPeakCalling (12)</a></li><li packagelist="alpine,ASpli,BitSeq,DEXSeq,DRIMSeq,edgeR,erccdashboard,EventPointer,ExCluster,GeneStructureTools,IMAS,InPAS,IntEREst,IsoformSwitchAnalyzeR,IVAS,kissDE,limma,MBttest,NBSplice,PathwaySplice,PECA,psichomics,puma,regsplice,TIN,Ularcirc" id="DifferentialSplicing" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="https://www.bioconductor.org/packages/release/BiocViews.html#"><ins class="jstree-icon">&nbsp;</ins>DifferentialSplicing (26)</a></li><li packagelist="DNAshapeR,FitHiC,MACPET,sevenC" id="DNA3DStructure" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="https://www.bioconductor.org/packages/release/BiocViews.html#"><ins class="jstree-icon">&nbsp;</ins>DNA3DStructure (4)</a></li><li packagelist="maftools" id="DriverMutation" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="https://www.bioconductor.org/packages/release/BiocViews.html#"><ins class="jstree-icon">&nbsp;</ins>DriverMutation (1)</a></li><li packagelist="BasicSTARRseq,diggit,drawProteins,EGAD,garfield,GeneStructureTools,InterMineR,IsoformSwitchAnalyzeR,OmaDB,PCAN,progeny,RIVER,RNASeqR,viper" id="FunctionalPrediction" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="https://www.bioconductor.org/packages/release/BiocViews.html#"><ins class="jstree-icon">&nbsp;</ins>FunctionalPrediction (14)</a></li><li class="jstree-leaf" style="display: none;"><ins class="jstree-icon">&nbsp;</ins><a href="https://www.bioconductor.org/packages/release/BiocViews.html#"><ins class="jstree-icon">&nbsp;</ins>GeneFusionDetection</a></li><li packagelist="AssessORF,brainImageR,coRdon,EGAD,geneAttribution,GRridge,IntramiRExploreR,IsoformSwitchAnalyzeR,mixOmics,R3CPET,RandomWalkRestartMH,REMP,sigFeature" id="GenePrediction" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="https://www.bioconductor.org/packages/release/BiocViews.html#"><ins class="jstree-icon">&nbsp;</ins>GenePrediction (13)</a></li><li packagelist="AffiXcan,anamiR,anota2seq,ATACseqQC,BasicSTARRseq,BCRANK,BiFET,blima,BPRMeth,BUMHMM,CAGEfightR,chromVAR,cleanUpdTSeq,CNEr,COCOA,CoRegNet,CRISPRseek,crisprseekplus,DeepBlueR,diggit,eegc,ELMER,GDCRNATools,geneplast,GIGSEA,GRENITS,GSReg,GSRI,GUIDEseq,IMAS,InPAS,INSPEcT,InterMineR,ISoLDE,IVAS,LINC,LOLA,MatrixRider,methylGSA,MethylMix,MIRA,miRSM,MoonlightR,MoPS,mpra,NADfinder,omicRexposome,pandaR,pqsfinder,primirTSS,profileScoreDist,progeny,qpgraph,RcisTarget,RIVER,RNAdecay,Rsubread,RTN,RTNduals,RTNsurvival,rTRM,rTRMui,samExploreR,SELEX,SEPIRA,SpidermiR,SPONGE,StarBioTrek,TCGAbiolinks,TCGAbiolinksGUI,TFBSTools,TFEA.ChIP,tigre,tRanslatome,trena,triplex,TSRchitect,universalmotif,viper" id="GeneRegulation" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="https://www.bioconductor.org/packages/release/BiocViews.html#"><ins class="jstree-icon">&nbsp;</ins>GeneRegulation (79)</a></li><li packagelist="ABAEnrichment,anamiR,artMS,attract,AUCell,BgeeDB,BioNetStat,BLMA,brainImageR,CAFE,Category,cellTree,chipenrich,clusterProfiler,CoGAPS,cogena,CompGO,coRdon,cpvSNP,CTDquerier,ctsGE,dcGSA,DOSE,edgeR,eegc,EGSEA,EnrichmentBrowser,enrichplot,facopy,FGNet,fgsea,gage,gCMAPWeb,gCrisprTools,GDCRNATools,geecc,gep2pep,GIGSEA,GISPA,GOexpress,GOfuncR,GOFunction,GOpro,goProfiles,goSTAG,GOstats,GRridge,GSALightning,GSEABase,GSEABenchmarkeR,gsean,GSReg,GSRI,GSVA,iBBiG,ideal,InterMineR,ISoLDE,limma,loci2path,LOLA,MAGeCKFlute,MAST,mdgsa,methylGSA,mgsa,MIGSA,miRSM,missMethyl,MoonlightR,npGSEA,oposSOM,oppar,pathVar,pathview,PathwaySplice,paxtoolsr,piano,PPInfer,qusage,RCAS,RcisTarget,ReactomePA,ReportingTools,rGREAT,RGSEA,rgsepd,RITAN,RNASeqR,RTN,RTNsurvival,safe,SAGx,SeqGSEA,SigCheck,sincell,singscore,SISPA,slinky,splineTimeR,sRAP,systemPipeR,TFEA.ChIP,TissueEnrich,transcriptogramer,transite" id="GeneSetEnrichment" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="https://www.bioconductor.org/packages/release/BiocViews.html#"><ins class="jstree-icon">&nbsp;</ins>GeneSetEnrichment (106)</a></li><li packagelist="EGSEA,hipathia,MetaboSignal,pwOmics,RankProd" id="GeneSignaling" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="https://www.bioconductor.org/packages/release/BiocViews.html#"><ins class="jstree-icon">&nbsp;</ins>GeneSignaling (5)</a></li><li packagelist="anamiR,BayesKnockdown,bioCancer,cicero,eegc,EGSEA,FCBF,GDCRNATools,gespeR,IntramiRExploreR,MetaboSignal,miRSM,networkBMA,Onassis,Pi,pwOmics,RCAS,RcisTarget,samExploreR,SEPIRA" id="GeneTarget" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="https://www.bioconductor.org/packages/release/BiocViews.html#"><ins class="jstree-icon">&nbsp;</ins>GeneTarget (20)</a></li><li packagelist="AnnotationDbi,annotatr,ASpli,AssessORF,branchpointer,COCOA,compEpiTools,DeepBlueR,DelayedArray,DMRcate,EnrichedHeatmap,garfield,geneXtendeR,GenomeInfoDb,GenomicFeatures,GenomicRanges,GreyListChIP,HDF5Array,HelloRanges,HilbertCurve,InterMineR,IWTomics,LOLA,mCSEA,MEDIPS,microRNA,MIRA,Organism.dplyr,QDNAseq,RCAS,rGREAT,RNAprobR,Rsubread,samExploreR,scPipe,SMITE,STAN,SummarizedExperiment,TSRchitect" id="GenomeAnnotation" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="https://www.bioconductor.org/packages/release/BiocViews.html#"><ins class="jstree-icon">&nbsp;</ins>GenomeAnnotation (39)</a></li><li packagelist="contiBAIT" id="GenomeAssembly" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="https://www.bioconductor.org/packages/release/BiocViews.html#"><ins class="jstree-icon">&nbsp;</ins>GenomeAssembly (1)</a></li><li packagelist="anota2seq,ASAFE,bacon,coMET,gcatest,GEM,geneAttribution,GeneGeneInteR,GENESIS,genphen,GISPA,globalSeq,gwasurvivr,hierinf,iCARE,InterMineR,martini,metaCCA,Pi,Rgin,rqt,RVS,SISPA" id="GenomeWideAssociation" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="https://www.bioconductor.org/packages/release/BiocViews.html#"><ins class="jstree-icon">&nbsp;</ins>GenomeWideAssociation (23)</a></li><li packagelist="AneuFinder,branchpointer,CINdex,CNAnorm,COCOA,cpvSNP,CrispRVariants,facopy,FindMyFriends,GeneAccord,geneAttribution,iGC,ipdDb,kissDE,MADSEQ,missMethyl,myvariant,panelcn.mops,RareVariantVis,Rariant,RIVER,scoreInvHap,semisup,seqCAT,signeR,SomaticSignatures,TitanCNA,TVTB,YAPSA" id="GenomicVariation" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="https://www.bioconductor.org/packages/release/BiocViews.html#"><ins class="jstree-icon">&nbsp;</ins>GenomicVariation (29)</a></li><li class="jstree-leaf" style="display: none;"><ins class="jstree-icon">&nbsp;</ins><a href="https://www.bioconductor.org/packages/release/BiocViews.html#"><ins class="jstree-icon">&nbsp;</ins>GermlineMutation</a></li><li packagelist="chipenrich,ChIPSeqSpike,chroGPS,chromstaR,chromswitch,geneXtendeR" id="HistoneModification" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="https://www.bioconductor.org/packages/release/BiocViews.html#"><ins class="jstree-icon">&nbsp;</ins>HistoneModification (6)</a></li><li class="jstree-leaf" style="display: none;"><ins class="jstree-icon">&nbsp;</ins><a href="https://www.bioconductor.org/packages/release/BiocViews.html#"><ins class="jstree-icon">&nbsp;</ins>IndelDetection</a></li><li packagelist="ASAFE,CNVrd2,hierGWAS,hierinf,Pi" id="LinkageDisequilibrium" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="https://www.bioconductor.org/packages/release/BiocViews.html#"><ins class="jstree-icon">&nbsp;</ins>LinkageDisequilibrium (5)</a></li><li class="jstree-leaf" style="display: none;"><ins class="jstree-icon">&nbsp;</ins><a href="https://www.bioconductor.org/packages/release/BiocViews.html#"><ins class="jstree-icon">&nbsp;</ins>MetagenomeAssembly</a></li><li class="jstree-leaf" style="display: none;"><ins class="jstree-icon">&nbsp;</ins><a href="https://www.bioconductor.org/packages/release/BiocViews.html#"><ins class="jstree-icon">&nbsp;</ins>MicrosatelliteDetection</a></li><li packagelist="branchpointer,coMET,DiffLogo,ELMER,LedPred,MatrixRider,motifbreakR,motifcounter,MotifDb,motifmatchr,motifStack,MotIV,PWMEnrich,RCAS,RcisTarget,SELEX,TFBSTools,universalmotif" id="MotifAnnotation" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="https://www.bioconductor.org/packages/release/BiocViews.html#"><ins class="jstree-icon">&nbsp;</ins>MotifAnnotation (18)</a></li><li packagelist="BCRANK,motifRG,pqsfinder,RCAS,rGADEM,SELEX,TFBSTools,universalmotif" id="MotifDiscovery" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="https://www.bioconductor.org/packages/release/BiocViews.html#"><ins class="jstree-icon">&nbsp;</ins>MotifDiscovery (8)</a></li><li packagelist="BioCor,BioNet,CEMiTool,CoRegNet,CTDquerier,DEGraph,DeMAND,DEsubs,diggit,EGAD,EGSEA,EnrichmentBrowser,FELLA,FEM,FGNet,GDCRNATools,GSEABenchmarkeR,gsean,ibh,miRNApath,miRsponge,MoonlightR,PathNet,PathwaySplice,paxtoolsr,PPInfer,RITAN,RTN,RTNduals,RTNsurvival,SANTA,signet,SMITE,splineTimeR,ToPASeq,viper,vulcan" id="NetworkEnrichment" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="https://www.bioconductor.org/packages/release/BiocViews.html#"><ins class="jstree-icon">&nbsp;</ins>NetworkEnrichment (37)</a></li><li packagelist="apComplex,BayesKnockdown,BioNetStat,birte,CNORfeeder,coexnet,CoRegNet,epiNEM,FRGEpistasis,GDCRNATools,GENIE3,GRENITS,joda,KEGGlincs,lpNet,minet,miRLAB,nem,netbenchmark,networkBMA,pandaR,PANR,Path2PPI,Pigengene,ppiStats,predictionet,qpgraph,R3CPET,RandomWalkRestartMH,rHVDM,RITAN,RTN,RTNduals,RTNsurvival,ScISI,SEPIRA,SPEM,splineTimeR,SPONGE,tigre,trena,TRONCO" id="NetworkInference" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="https://www.bioconductor.org/packages/release/BiocViews.html#"><ins class="jstree-icon">&nbsp;</ins>NetworkInference (42)</a></li><li packagelist="ATACseqQC,nucleR,nuCpos,RJMCMCNucleosomes" id="NucleosomePositioning" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="https://www.bioconductor.org/packages/release/BiocViews.html#"><ins class="jstree-icon">&nbsp;</ins>NucleosomePositioning (4)</a></li><li packagelist="BasicSTARRseq,CAGEfightR,ChIPanalyser,chromstaR,consensusSeekeR,DEScan2,flowQB,gcapc,geneXtendeR,Linnorm,MACPET,NADfinder,normr,proFIA" id="PeakDetection" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="https://www.bioconductor.org/packages/release/BiocViews.html#"><ins class="jstree-icon">&nbsp;</ins>PeakDetection (14)</a></li><li class="jstree-leaf" style="display: none;"><ins class="jstree-icon">&nbsp;</ins><a href="https://www.bioconductor.org/packages/release/BiocViews.html#"><ins class="jstree-icon">&nbsp;</ins>QuantitativeTrailLocus</a></li><li class="jstree-leaf" style="display: none;"><ins class="jstree-icon">&nbsp;</ins><a href="https://www.bioconductor.org/packages/release/BiocViews.html#"><ins class="jstree-icon">&nbsp;</ins>Scaffolding</a></li><li packagelist="Basic4Cseq,bcSeq,Biostrings,BSgenome,ChIPanalyser,cleanUpdTSeq,cobindR,CRISPRseek,crisprseekplus,dagLogo,DiffLogo,DominoEffect,FastqCleaner,FindMyFriends,FunciSNP,genphen,hapFabia,heatmaps,HelloRanges,IMMAN,ipdDb,KinSwingR,Logolas,LowMACA,MEDIPS,MethTargetedNGS,microRNA,MmPalateMiRNA,motifcounter,motifStack,muscle,pqsfinder,PWMEnrich,QSutils,REDseq,RRHO,Rsubread,samExploreR,scPipe,seqLogo,seqPattern,SICtools,triplex,TVTB" id="SequenceMatching" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="https://www.bioconductor.org/packages/release/BiocViews.html#"><ins class="jstree-icon">&nbsp;</ins>SequenceMatching (44)</a></li><li packagelist="DominoEffect,gaucho,GeneAccord,LowMACA,MADSEQ,maftools,MutationalPatterns,OncoSimulR,QuartPAC,RandomWalkRestartMH,Rariant,semisup,signeR,SomaticSignatures,SparseSignatures,TRONCO,YAPSA" id="SomaticMutation" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="https://www.bioconductor.org/packages/release/BiocViews.html#"><ins class="jstree-icon">&nbsp;</ins>SomaticMutation (17)</a></li><li packagelist="Guitar" id="SplicedAlignment" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="https://www.bioconductor.org/packages/release/BiocViews.html#"><ins class="jstree-icon">&nbsp;</ins>SplicedAlignment (1)</a></li><li class="jstree-leaf" style="display: none;"><ins class="jstree-icon">&nbsp;</ins><a href="https://www.bioconductor.org/packages/release/BiocViews.html#"><ins class="jstree-icon">&nbsp;</ins>StructuralVariation</a></li><li packagelist="IsoformSwitchAnalyzeR,RIVER" id="TranscriptomeVariant" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="https://www.bioconductor.org/packages/release/BiocViews.html#"><ins class="jstree-icon">&nbsp;</ins>TranscriptomeVariant (2)</a></li><li packagelist="appreci8R,cellbaseR,ensemblVEP,geneAttribution,genotypeeval,GIGSEA,HelloRanges,ipdDb,maftools,MTseeker,myvariant,PCAN,podkat,PureCN,REBET,seqCAT,SNPediaR,TVTB,VariantAnnotation" id="VariantAnnotation" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="https://www.bioconductor.org/packages/release/BiocViews.html#"><ins class="jstree-icon">&nbsp;</ins>VariantAnnotation (19)</a></li><li packagelist="appreci8R,CrispRVariants,MADSEQ,panelcn.mops,PureCN,Rariant,RVS,SICtools" id="VariantDetection" class="jstree-last jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="https://www.bioconductor.org/packages/release/BiocViews.html#"><ins class="jstree-icon">&nbsp;</ins>VariantDetection (8)</a></li></ul></li><li packagelist="affxparser,AffyCompatible,affyContam,affyio,AllelicImbalance,AnnotationFilter,AnnotationForge,AnnotationHub,aroma.light,arrayMvout,basecallQC,BaseSpaceR,beachmat,bigmemoryExtras,bioassayR,Biobase,BiocCaseStudies,BiocCheck,BiocGenerics,BiocInstaller,BiocParallel,BiocPkgTools,BiocSklearn,BiocVersion,biocViews,BioMVCClass,Biostrings,biovizBase,BSgenome,BufferedMatrix,BufferedMatrixMethods,bumphunter,Cardinal,CGHbase,ChemmineOB,ChemmineR,chimera,chimeraviz,ChIPseqR,ChIPsim,convert,CTDquerier,cummeRbund,DelayedArray,DelayedMatrixStats,DynDoc,eiR,ENCODExplorer,epivizr,epivizrData,epivizrServer,epivizrStandalone,ExperimentHub,ExperimentHubData,flowBeads,flowcatchR,flowCore,flowQ,FlowRepositoryR,flowUtils,flowViz,fmcsR,FunciSNP,GDSArray,gdsfmt,genbankr,GeneAnswers,genomeIntervals,GenomicAlignments,GenomicFeatures,GenomicFiles,GenomicInteractions,GenomicRanges,GenomicScores,GenomicTuples,Genominator,genoset,GenVisR,GEOmetadb,GGBase,ggbio,ggcyto,HDF5Array,HiCBricks,hmdbQuery,Icens,illuminaio,InteractionSet,IRanges,LoomExperiment,lpsymphony,LRBaseDbi,matter,MeSHDbi,metagenomeFeatures,metavizr,MetID,microRNA,mimager,monocle,MSnbase,MultiAssayExperiment,MVCClass,mzR,oligoClasses,ontoProc,OrganismDbi,panp,Pbase,pdInfoBuilder,pkgDepTools,plateCore,plyranges,preprocessCore,ProtGenerics,psygenet2r,QUALIFIER,R453Plus1Toolbox,RaggedExperiment,Rchemcpp,RedeR,restfulSE,rexposome,RGalaxy,RGMQL,rhdf5,Rhdf5lib,RNASeqR,RProtoBufLib,RpsiXML,rRDP,RSeqAn,RTCA,S4Vectors,scater,SeqArray,SingleCellExperiment,SNPRelate,splicegear,SRAdb,Streamer,SummarizedBenchmark,SummarizedExperiment,Sushi,systemPipeR,TFARM,tkWidgets,TnT,tofsims,topdownr,TypeInfo,UniProt.ws,weaver,webbioc,widgetTools,XVector,zlibbioc" id="Infrastructure" class="jstree-closed"><ins class="jstree-icon">&nbsp;</ins><a href="https://www.bioconductor.org/packages/release/BiocViews.html#"><ins class="jstree-icon">&nbsp;</ins>Infrastructure (360)</a><ul><li packagelist="aCGH,affxparser,affyio,affylmGUI,AnnotationHub,AnnotationHubData,ArrayExpress,ASICS,bamsignals,basecallQC,BaseSpaceR,Basic4Cseq,beachmat,beadarraySNP,BgeeDB,bigmelon,bioassayR,biobroom,BiocFileCache,biomformat,BioNet,Biostrings,BrainStars,CAGEfightR,cellGrowth,CellScore,ChemmineOB,ChemmineR,ChIPanalyser,ChIPComp,chipseq,ChIPSeqSpike,CNEr,codelink,cogena,CONFESS,ctc,CTDquerier,cummeRbund,cytolib,CytoML,DAPAR,DECIPHER,DeepBlueR,deepSNV,diffloop,Doscheda,DropletUtils,edge,eiR,ENCODExplorer,ENmix,ExperimentHub,ExperimentHubData,flowTime,flowWorkspace,fmcsR,FunciSNP,gaggle,gdsfmt,genbankr,GeneRegionScan,geneXtendeR,genomeIntervals,GenomicAlignments,GenomicDataCommons,GenomicFiles,GenomicInteractions,genotypeeval,GEOquery,ggtree,GoogleGenomics,Guitar,HDF5Array,HelloRanges,HELP,HiCBricks,HTqPCR,ibh,illuminaio,ImmuneSpaceR,IONiseR,IsoformSwitchAnalyzeR,IWTomics,karyoploteR,limma,limmaGUI,LOBSTAHS,LoomExperiment,LowMACA,makecdfenv,MANOR,manta,MassArray,MDTS,MethTargetedNGS,minfi,monocle,motifStack,msgbsR,MSnbase,muscle,mzID,mzR,ndexr,normr,nucleR,oligo,Onassis,openCyto,ORFik,OTUbase,Pbase,PepsNMR,plethy,plotGrouper,podkat,ProteomicsAnnotationHubData,PSICQUIC,psygenet2r,puma,qPLEXanalyzer,qrqc,QSutils,R453Plus1Toolbox,R4RNA,Rchemcpp,Rcpi,readat,ReadqPCR,recount,REMP,rexposome,RGMQL,rhdf5,rhdf5client,Rhtslib,RImmPort,Ringo,Risa,RNASeqR,RnBeads,rpx,Rqc,Rsamtools,rSFFreader,RTCGA,rtracklayer,SAGx,Scale4C,scater,scmap,segmentSeq,sevenbridges,sevenC,ShortRead,SICtools,simpleaffy,SingleCellExperiment,slinky,SRAdb,Starr,Streamer,systemPipeR,TarSeqQC,tofsims,TransView,traseR,treeio,tRNAdbImport,tRNAscanImport,TRONCO,TVTB,tximeta,tximport,universalmotif,VariantAnnotation,xps" id="DataImport" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="https://www.bioconductor.org/packages/release/BiocViews.html#"><ins class="jstree-icon">&nbsp;</ins>DataImport (178)</a></li><li packagelist="AnalysisPageServer,BaseSpaceR,beachmat,bigmemoryExtras,bioCancer,Biostrings,BSgenome,CAGEfightR,CellTrails,chroGPS,chromPlot,cogena,consensus,CrispRVariants,CTDquerier,cummeRbund,cytolib,CytoML,DeepBlueR,DelayedArray,DelayedMatrixStats,DEP,diffloop,esetVis,FindMyFriends,flowPlots,flowWorkspace,FunciSNP,GA4GHclient,GDSArray,GeneAnswers,GenomeInfoDb,GenomicInteractions,GenomicTuples,genoset,GenVisR,GOexpress,HDF5Array,ImmuneSpaceR,InteractionSet,interactiveDisplay,interactiveDisplayBase,ipdDb,IRanges,KEGGlincs,LoomExperiment,maftools,monocle,MultiAssayExperiment,MultiDataSet,OMICsPCA,openCyto,oposSOM,OSAT,Pbase,phantasus,plethy,plyranges,psygenet2r,R453Plus1Toolbox,R4RNA,RaggedExperiment,rBiopaxParser,Rcpi,RedeR,ReportingTools,rexposome,RImmPort,RTCGA,S4Vectors,SC3,scater,scfind,scmap,SeqArray,SingleCellExperiment,SplicingGraphs,Sushi,treeio,TVTB,Ularcirc,XVector" id="DataRepresentation" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="https://www.bioconductor.org/packages/release/BiocViews.html#"><ins class="jstree-icon">&nbsp;</ins>DataRepresentation (82)</a></li><li packagelist="affylmGUI,AnalysisPageServer,AnnotationHub,bioCancer,canceR,crossmeta,cytofkit,epivizr,epivizrChart,epivizrStandalone,ExperimentHub,ExperimentHubData,explorase,flowcatchR,flowPloidy,GA4GHshiny,gCMAPWeb,GEM,GSCA,ideal,interactiveDisplay,interactiveDisplayBase,iSEE,IsoGeneGUI,limmaGUI,mcaGUI,metavizr,MethylAid,MSGFgui,MSstatsQCgui,omicplotR,OMICsPCA,oneSENSE,PanVizGenerator,pcaExplorer,phantasus,plotGrouper,pRolocGUI,Prostar,psichomics,rTRMui,SC3,SEPA,statTarget,TCGAbiolinksGUI,TSCAN,TVTB,uSORT" id="GUI" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="https://www.bioconductor.org/packages/release/BiocViews.html#"><ins class="jstree-icon">&nbsp;</ins>GUI (48)</a></li><li packagelist="AnnotationHub,BrowserViz,DeepBlueR,ExperimentHub,ExperimentHubData,GA4GHclient,gaggle,GoogleGenomics,graphite,igvR,ImmuneSpaceR,KEGGlincs,KEGGREST,lpsymphony,maigesPack,PSICQUIC,RamiGO,RCy3,RCyjs,RGraph2js,rpx,rWikiPathways,sevenbridges,slinky" id="ThirdPartyClient" class="jstree-last jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="https://www.bioconductor.org/packages/release/BiocViews.html#"><ins class="jstree-icon">&nbsp;</ins>ThirdPartyClient (24)</a></li></ul></li><li packagelist="anamiR,iGC,INDEED,MethTargetedNGS,MetID,RankProd,rDGIdb" id="ResearchField" class="jstree-closed"><ins class="jstree-icon">&nbsp;</ins><a href="https://www.bioconductor.org/packages/release/BiocViews.html#"><ins class="jstree-icon">&nbsp;</ins>ResearchField (728)</a><ul><li class="jstree-leaf" style="display: none;"><ins class="jstree-icon">&nbsp;</ins><a href="https://www.bioconductor.org/packages/release/BiocViews.html#"><ins class="jstree-icon">&nbsp;</ins>Agroinformatics</a></li><li packagelist="artMS,ASAFE,CancerInSilico,cellTree,ChemmineOB,ChemmineR,coRdon,CTDquerier,CVE,cytofkit,edgeR,eiR,flowAI,fmcsR,GAprediction,gCrisprTools,GeneAccord,geneClassifiers,gwasurvivr,IsoformSwitchAnalyzeR,limma,mdp,messina,miRSM,miRsponge,OMICsPCA,OncoScore,OPWeight,Pigengene,powerTCR,psichomics,psygenet2r,RandomWalkRestartMH,Rcpi,rexposome,RImmPort,RIVER,sincell,SparseSignatures,STROMA4,timescape,TRONCO" id="BiomedicalInformatics" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="https://www.bioconductor.org/packages/release/BiocViews.html#"><ins class="jstree-icon">&nbsp;</ins>BiomedicalInformatics (42)</a></li><li class="jstree-leaf" style="display: none;"><ins class="jstree-icon">&nbsp;</ins><a href="https://www.bioconductor.org/packages/release/BiocViews.html#"><ins class="jstree-icon">&nbsp;</ins>Biophysics</a></li><li packagelist="BASiCS,CAMTHC,CancerInSilico,CellNOptR,cellTree,CFAssay,cn.mops,CNORdt,CNORfeeder,CNORode,cobindR,CONFESS,CRImage,CytoDx,cytofkit,destiny,dexus,diffcyt,edgeR,fastLiquidAssociation,FISHalyseR,flowcatchR,flowDensity,flowMeans,FlowSOM,GateFinder,gCrisprTools,GRmetrics,HPAanalyze,hpar,ImpulseDE2,KEGGlincs,limma,LineagePulse,LiquidAssociation,MetaCyto,MPRAnalyze,netresponse,panelcn.mops,Prize,SamSPECTRAL,sincell,staRank,ternarynet,tRanslatome,uSORT" id="CellBiology" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="https://www.bioconductor.org/packages/release/BiocViews.html#"><ins class="jstree-icon">&nbsp;</ins>CellBiology (46)</a></li><li packagelist="ASICS,BridgeDbR,ChemmineOB,ChemmineR,eiR,fmcsR,limma,mQTL.NMR,MWASTools,rcellminer,Rcpi" id="Cheminformatics" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="https://www.bioconductor.org/packages/release/BiocViews.html#"><ins class="jstree-icon">&nbsp;</ins>Cheminformatics (11)</a></li><li packagelist="AssessORF,FindMyFriends,InterMineR,OmaDB,PanVizGenerator" id="ComparativeGenomics" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="https://www.bioconductor.org/packages/release/BiocViews.html#"><ins class="jstree-icon">&nbsp;</ins>ComparativeGenomics (5)</a></li><li class="jstree-leaf" style="display: none;"><ins class="jstree-icon">&nbsp;</ins><a href="https://www.bioconductor.org/packages/release/BiocViews.html#"><ins class="jstree-icon">&nbsp;</ins>ComputationalChemistry</a></li><li packagelist="BEAT,BiFET,BPRMeth,bumphunter,Chicago,chipenrich,ChIPSeqSpike,chroGPS,chromswitch,cicero,COCOA,COHCAP,compartmap,DeepBlueR,DEScan2,dmrseq,edgeR,ENmix,EpiDISH,GAprediction,GenoGAM,GOTHiC,InTAD,ISoLDE,limma,mCSEA,methimpute,methInheritSim,MethPed,methylInheritance,minfi,MIRA,msgbsR,nuCpos,omicRexposome,OMICsPCA,omicsPrint,REMP,RnBeads,sevenC,srnadiff,TCseq" id="Epigenetics" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="https://www.bioconductor.org/packages/release/BiocViews.html#"><ins class="jstree-icon">&nbsp;</ins>Epigenetics (42)</a></li><li packagelist="annotatr,BasicSTARRseq,CAGEr,cellTree,chipenrich,chromPlot,chromswitch,COCOA,coMET,customProDB,DEComplexDisease,dmrseq,edgeR,EGAD,FGNet,gCrisprTools,GeneAccord,heatmaps,InterMineR,IsoformSwitchAnalyzeR,limma,loci2path,LOLA,maftools,MAGeCKFlute,MIRA,mpra,normr,OmaDB,OMICsPCA,ORFik,rDGIdb,RITAN,RIVER,sevenC,sincell,TSRchitect" id="FunctionalGenomics" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="https://www.bioconductor.org/packages/release/BiocViews.html#"><ins class="jstree-icon">&nbsp;</ins>FunctionalGenomics (37)</a></li><li packagelist="aCGH,adaptest,AGDEX,AllelicImbalance,anamiR,ASAFE,AssessORF,BaseSpaceR,BEAT,BiFET,biomvRCNS,Biostrings,BiSeq,BPRMeth,BSgenome,BUMHMM,CancerMutationAnalysis,CAnD,CexoR,ChIPComp,chromPlot,CINdex,cleanUpdTSeq,Clomial,cn.mops,coMET,compartmap,consensusSeekeR,contiBAIT,copynumber,cpvSNP,CSAR,csaw,ctsGE,DBChIP,DECIPHER,deepSNV,discordant,DMRcate,DOQTL,DRIMSeq,easyRNASeq,edgeR,EGSEA,ensembldb,erccdashboard,exomeCopy,facopy,FamAgg,fastLiquidAssociation,fCCAC,FRGEpistasis,gage,gaucho,gCrisprTools,GeneAccord,GeneBreak,GeneGeneInteR,geneplast,geneRxCluster,GENESIS,GeneticsDesign,GeneticsPed,geneXtendeR,genoCN,GenoGAM,GenomeInfoDb,genomeIntervals,genomes,GenomicAlignments,GenomicFeatures,GenomicFiles,GenomicRanges,GenomicScores,genotypeeval,genphen,GenRank,GEWIST,GGBase,GGtools,GoogleGenomics,gwascat,gwasurvivr,hapFabia,HDTD,HIBAG,HybridMTest,iGC,Imetagene,intansv,interactiveDisplay,interactiveDisplayBase,ISoLDE,limma,Linnorm,LiquidAssociation,loci2path,logicFS,manta,martini,MassArray,MatrixRider,mCSEA,MEDIPS,metaCCA,metagene,MethTargetedNGS,MGFM,MGFR,MLP,mosaics,mQTL.NMR,MTseeker,muscle,MutationalPatterns,NarrowPeaks,netresponse,normr,nucleoSim,nucleR,nuCpos,NuPoP,OmaDB,omicsPrint,OUTRIDER,panelcn.mops,pathview,PCAN,Pi,pint,podkat,PREDA,primirTSS,psygenet2r,QDNAseq,qpgraph,QSutils,QuasR,rain,Rcade,RCASPAR,regioneR,Rgin,riboSeqR,RIVER,RNASeqR,rRDP,RRHO,Rsubread,RVS,scoreInvHap,semisup,SeqArray,seqCNA,SeqVarTools,similaRpeak,SLGI,SNPRelate,SplicingGraphs,sscu,ssviz,SummarizedExperiment,Sushi,systemPipeR,TCGAbiolinksGUI,TEQC,TIN,TitanCNA,transite,traseR,trigger,trio,TSSi,TVTB,Ularcirc,VariantAnnotation,VariantFiltering,VariantTools" id="Genetics" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="https://www.bioconductor.org/packages/release/BiocViews.html#"><ins class="jstree-icon">&nbsp;</ins>Genetics (178)</a></li><li packagelist="AIMS,ALDEx2,amplican,Anaquin,AneuFinder,anota2seq,apComplex,apeglm,ArrayExpressHTS,artMS,ASpli,ATACseqQC,attract,AUCell,bacon,BADER,ballgown,banocc,Basic4Cseq,BASiCS,BatchQC,bayNorm,bcSeq,BDMMAcorrect,BEARscc,BEAT,BiFET,bioassayR,BiocOncoTK,biomformat,BioNetStat,BitSeq,BPRMeth,BRAIN,BUMHMM,bumphunter,CAMERA,CancerInSilico,caOmicsV,Cardinal,casper,CATALYST,CausalR,ccfindR,ccrepe,celaref,cellGrowth,cellHTS2,cellity,CellNOptR,CellTrails,cellTree,CEMiTool,CFAssay,Chicago,chipenrich,ChIPSeqSpike,chroGPS,chromstaR,chromswitch,chromVAR,Clomial,CNORdt,CNORfeeder,CNORode,COCOA,CODEX,CoGAPS,compartmap,COMPASS,compcodeR,condcomp,CONFESS,consensus,contiBAIT,CopywriteR,coRdon,coseq,cosmiq,CountClust,countsimQC,covEB,CoverageView,cqn,CRISPRseek,CrispRVariants,cTRAP,ctsGE,customProDB,cydar,CytoDx,cytolib,CytoML,dada2,DaMiRseq,DAPAR,dcGSA,debrowser,DECIPHER,decontam,DEFormats,DEGreport,DEGseq,DEP,DEqMS,derfinder,derfinderHelper,derfinderPlot,DEScan2,DESeq,DESeq2,DEsingle,DEsubs,DEXSeq,dexus,diffcyt,DirichletMultinomial,discordant,dmrseq,doppelgangR,Doscheda,DRIMSeq,DropletUtils,DSS,dupRadar,easyRNASeq,EBSeq,EBSeqHMM,EDASeq,EDDA,edgeR,eegc,EGSEA,ELBOW,EnhancedVolcano,ENmix,EnrichmentBrowser,EpiDISH,erccdashboard,ERSSA,esATAC,EventPointer,ExCluster,FCBF,flagme,flowAI,flowBeads,flowBin,flowCHIC,flowCL,flowClust,flowCore,flowCyBar,flowDensity,flowFit,flowFP,flowMap,flowMatch,flowMeans,flowMerge,flowPeaks,flowPlots,flowQB,FlowRepositoryR,FlowSOM,flowStats,flowTime,flowTrans,flowType,flowUtils,flowViz,flowVS,flowWorkspace,gaga,gage,GAprediction,GateFinder,gCrisprTools,GDCRNATools,geecc,GeneStructureTools,GenoGAM,GenomicAlignments,gespeR,ggcyto,Glimma,globalSeq,GOexpress,goseq,goSTAG,GOTHiC,GRmetrics,GSEABenchmarkeR,GUIDEseq,Guitar,HDF5Array,HTSanalyzeR,HTSFilter,iASeq,iasva,icetea,iCNV,ideal,IHW,imageHTS,IMAS,immunoClust,ImpulseDE2,INDEED,InPAS,InTAD,IntEREst,IPO,iSEE,isobar,IsoCorrectoR,ISoLDE,isomiRs,IVAS,JunctionSeq,limma,LineagePulse,Linnorm,LOBSTAHS,LoomExperiment,M3C,M3Drop,MAGeCKFlute,MAIT,manta,MassArray,MassSpecWavelet,MaxContrastProjection,MCbiclust,mCSEA,Metab,metabomxtr,MetaCyto,metagenomeFeatures,metagenomeSeq,metaMS,MetaNeighbor,metaSeq,metaseqR,metavizr,MetCirc,methimpute,methInheritSim,MethPed,methylInheritance,MetNet,mfa,MGFR,microbiome,MIMOSA,minfi,MIRA,MiRaGE,mitoODE,mixOmics,mlm4omics,MLSeq,monocle,MPRAnalyze,msgbsR,MSGFgui,MSGFplus,msmsEDA,msmsTests,MSnbase,MSnID,msPurity,MSstats,MSstatsQC,MSstatsQCgui,MSstatsTMT,mzID,mzR,NBSplice,ncdfFlow,netprioR,NGScopy,NOISeq,nuCpos,omicplotR,omicRexposome,OMICsPCA,omicsPrint,oneSENSE,openCyto,OPWeight,ORFik,Oscope,OUTRIDER,PANR,PAPi,PathoStat,pathview,PathwaySplice,Pbase,pcaExplorer,pepXMLTab,perturbatr,PGA,pgca,phantasus,philr,phyloseq,Pigengene,plateCore,plgem,plotGrouper,pogos,Polyfit,PowerExplorer,prada,prebs,primirTSS,Prize,proBAMr,PROcess,proFIA,pRoloc,PROPER,Prostar,ProteoMM,proteoQC,psichomics,PureCN,qcmetrics,qPLEXanalyzer,QUALIFIER,QuasR,qusage,Rchemcpp,Rdisop,recount,recoup,regionReport,regsplice,REMP,ReportingTools,restfulSE,rgsepd,RMassBank,RNAdecay,RNAinteract,RNAither,rnaSeqMap,RNASeqPower,RNASeqR,RnaSeqSampleSize,RnBeads,rols,ropls,ROTS,rpx,rRDP,Rsubread,rTANDEM,RTCA,RTCGA,RUVSeq,RVS,samExploreR,sampleClassifier,SamSPECTRAL,sapFinder,SC3,SCAN.UPC,scater,scDD,scde,scFeatureFilter,scfind,scmap,scmeth,SCnorm,scone,Sconify,scPipe,scran,scruff,SDAMS,SeqGSEA,seqplots,SGSeq,shinyTANDEM,SIAMCAT,sights,SIMAT,SIMLR,sincell,SingleCellExperiment,singleCellTK,slalom,SMITE,sparseDOSSA,sparsenetgls,specL,splatter,SplicingGraphs,SPLINTER,srnadiff,sSeq,SSPA,ssviz,STAN,staRank,statTarget,STROMA4,subSeq,sva,SVAPLSseq,SWATH2stats,switchde,synapter,synlet,systemPipeR,TargetSearch,TCC,TCseq,tenXplore,TFEA.ChIP,TimeSeriesExperiment,TitanCNA,tofsims,ToPASeq,topdownr,TPP,transcriptogramer,transcriptR,TransView,Trendy,TRONCO,tweeDEseq,tximeta,tximport,Uniquorn,uSORT,variancePartition,vidger,wavClusteR,wiggleplotr,XBSeq,xcms,XINA,yamss,zFPKM,zinbwave" id="ImmunoOncology" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="https://www.bioconductor.org/packages/release/BiocViews.html#"><ins class="jstree-icon">&nbsp;</ins>ImmunoOncology (450)</a></li><li packagelist="biosigner,Cardinal,LOBSTAHS,MWASTools,proFIA,RankProd,ropls" id="Lipidomics" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="https://www.bioconductor.org/packages/release/BiocViews.html#"><ins class="jstree-icon">&nbsp;</ins>Lipidomics (7)</a></li><li packagelist="CancerInSilico,GeneAccord,ISoLDE,missRows,RandomWalkRestartMH" id="MathematicalBiology" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="https://www.bioconductor.org/packages/release/BiocViews.html#"><ins class="jstree-icon">&nbsp;</ins>MathematicalBiology (5)</a></li><li packagelist="ASICS,BiGGR,bioassayR,BioNetStat,biosigner,BridgeDbR,CAMERA,ChemmineOB,ChemmineR,cosmiq,EGSEA,eiR,FELLA,fmcsR,graphite,hmdbQuery,INDEED,IPO,IsoCorrectoR,limma,LOBSTAHS,MAIT,Metab,metabomxtr,metaMS,MetCirc,MetNet,mixOmics,mQTL.NMR,msPurity,MTseeker,MWASTools,mzR,NormalyzerDE,OmicsMarkeR,PAPi,pathview,PepsNMR,proFIA,RankProd,Rdisop,RMassBank,ropls,rWikiPathways,SDAMS,SIMAT,sparsenetgls,statTarget,tofsims,xcms,yamss" id="Metabolomics" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="https://www.bioconductor.org/packages/release/BiocViews.html#"><ins class="jstree-icon">&nbsp;</ins>Metabolomics (51)</a></li><li packagelist="ALDEx2,banocc,biomformat,ccrepe,coRdon,dada2,decontam,DirichletMultinomial,metagenomeFeatures,metagenomeSeq,metavizr,microbiome,mixOmics,omicplotR,PathoStat,philr,phyloseq,SIAMCAT,sparseDOSSA" id="Metagenomics" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="https://www.bioconductor.org/packages/release/BiocViews.html#"><ins class="jstree-icon">&nbsp;</ins>Metagenomics (19)</a></li><li packagelist="ChemmineOB,ChemmineR,eiR,fmcsR,gCrisprTools,PharmacoGx,rcellminer,rDGIdb" id="Pharmacogenetics" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="https://www.bioconductor.org/packages/release/BiocViews.html#"><ins class="jstree-icon">&nbsp;</ins>Pharmacogenetics (8)</a></li><li packagelist="ChemmineOB,ChemmineR,eiR,fmcsR,gCrisprTools,gwasurvivr,PharmacoGx,pogos,rcellminer,rDGIdb" id="Pharmacogenomics" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="https://www.bioconductor.org/packages/release/BiocViews.html#"><ins class="jstree-icon">&nbsp;</ins>Pharmacogenomics (10)</a></li><li packagelist="artMS,ASEB,AssessORF,bioassayR,biobroom,BioNetStat,biosigner,BRAIN,Cardinal,CausalR,CellNOptR,ChemmineOB,ChemmineR,cisPath,cleaver,clippda,CNORdt,CNORfeeder,CNORode,customProDB,cydar,DAPAR,deltaGseg,DEP,DEqMS,diffcyt,DominoEffect,Doscheda,drawProteins,EGSEA,eiR,fCI,fmcsR,GraphPAC,HPAanalyze,hpar,IMMAN,InterMineR,iPAC,IPPD,isobar,kimod,KinSwingR,limma,LPEadj,MassSpecWavelet,mixOmics,mlm4omics,MSGFgui,MSGFplus,msmsEDA,msmsTests,MSnbase,MSnID,MSstats,MSstatsQC,MSstatsQCgui,MSstatsTMT,mzID,mzR,NormalyzerDE,omicRexposome,PAA,Path2PPI,pathview,Pbase,PCpheno,PECA,pepXMLTab,PGA,pgca,phosphonormalizer,plgem,PLPE,PowerExplorer,ppiStats,proBAMr,PROcess,procoil,ProCoNA,proFIA,pRoloc,pRolocGUI,prot2D,ProteomicsAnnotationHubData,ProteoMM,proteoQC,ProtGenerics,Pviz,qcmetrics,qPLEXanalyzer,QuartPAC,rain,RankProd,RCASPAR,Rchemcpp,Rcpi,readat,ropls,ROTS,RpsiXML,rpx,rTANDEM,sapFinder,ScISI,SDAMS,shinyTANDEM,SLGI,SpacePAC,sparsenetgls,specL,spliceSites,statTarget,SWATH2stats,synapter,tofsims,topdownr,TPP,XINA" id="Proteomics" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="https://www.bioconductor.org/packages/release/BiocViews.html#"><ins class="jstree-icon">&nbsp;</ins>Proteomics (119)</a></li><li class="jstree-leaf" style="display: none;"><ins class="jstree-icon">&nbsp;</ins><a href="https://www.bioconductor.org/packages/release/BiocViews.html#"><ins class="jstree-icon">&nbsp;</ins>StructuralGenomics</a></li><li packagelist="BUMHMM,DNAshapeR" id="StructuralPrediction" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="https://www.bioconductor.org/packages/release/BiocViews.html#"><ins class="jstree-icon">&nbsp;</ins>StructuralPrediction (2)</a></li><li packagelist="artMS,BioCor,BioNetStat,CancerInSilico,CausalR,cellTree,CHRONOS,COCOA,coexnet,cogena,CoRegNet,DeMAND,DEsubs,diggit,edgeR,EGAD,EGSEA,epiNEM,FEM,gage,gCrisprTools,GeneAccord,geneplast,GENIE3,IMMAN,INSPEcT,InterMineR,IsoformSwitchAnalyzeR,limma,LOLA,mdp,MEIGOR,microbiome,MIRA,multiOmicsViz,MWASTools,nem,oppar,Path2PPI,pathview,paxtoolsr,Pigengene,pmm,progeny,pwOmics,rain,RandomWalkRestartMH,RankProd,rcellminer,Rcpi,RIVER,sincell,SMITE,SPONGE,transcriptogramer,trena,viper,vulcan,XINA" id="SystemsBiology" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="https://www.bioconductor.org/packages/release/BiocViews.html#"><ins class="jstree-icon">&nbsp;</ins>SystemsBiology (59)</a></li><li packagelist="AUCell,bacon,BASiCS,BEARscc,biosigner,BUMHMM,CAGEfightR,CausalR,ccfindR,cellity,CEMiTool,consensusDE,consensusOV,cTRAP,DEsingle,DropletUtils,edgeR,GeneStructureTools,globalSeq,icetea,iSEE,IsoformSwitchAnalyzeR,kissDE,limma,M3Drop,maser,MAST,mdp,MetaNeighbor,netSmooth,omicplotR,omicRexposome,omicsPrint,OUTRIDER,phantasus,Pigengene,psichomics,RCAS,RcisTarget,restfulSE,RIVER,RNAdecay,ropls,SC3,scater,scfind,scmap,scone,scran,slalom,slingshot,splatter,SPONGE,switchde,tenXplore,TFEA.ChIP,TFutils,tximeta,zinbwave" id="Transcriptomics" class="jstree-last jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="https://www.bioconductor.org/packages/release/BiocViews.html#"><ins class="jstree-icon">&nbsp;</ins>Transcriptomics (59)</a></li></ul></li><li packagelist="ArrayTools,bacon,ballgown,BioCor,BiocSklearn,BioQC,BUScorrect,CAnD,CATALYST,ClusterJudge,ClusterSignificance,CountClust,covEB,cpvSNP,CytoDx,dcGSA,DeMAND,discordant,EBSeq,EBSeqHMM,EmpiricalBrownsMethod,fabia,FunChIP,garfield,geecc,GeneSelector,GENESIS,GISPA,glmSparseNet,GSAR,gsean,Harman,HDTD,HIBAG,iasva,iCARE,ImpulseDE,ImpulseDE2,INDEED,IntramiRExploreR,IsoformSwitchAnalyzeR,iterClust,iteremoval,IWTomics,joda,LEA,LineagePulse,lol,MACPET,MCbiclust,MDTS,MeasurementError.cor,metaCCA,MetaCyto,methInheritSim,methylInheritance,MethylMix,MetID,missRows,mogsa,MPRAnalyze,Mulcom,MultiMed,NBSplice,NeighborNet,netReg,npGSEA,nucleoSim,omicRexposome,onlineFDR,Oscope,pandaR,PhenStat,phosphonormalizer,PING,plotGrouper,PPInfer,profileScoreDist,QUBIC,randPack,RankProd,Rariant,RGSEA,RJMCMCNucleosomes,rqt,Rtreemix,RUVnormalize,safe,scde,signeR,SNPRelate,SomaticSignatures,SRGnet,srnadiff,SSPA,stageR,STATegRa,sva,swfdr,switchBox,TCGAbiolinksGUI,TFARM,TitanCNA,TMixClust,tweeDEseq,Uniquorn,YAPSA" id="StatisticalMethod" class="jstree-closed"><ins class="jstree-icon">&nbsp;</ins><a href="https://www.bioconductor.org/packages/release/BiocViews.html#"><ins class="jstree-icon">&nbsp;</ins>StatisticalMethod (572)</a><ul><li packagelist="affylmGUI,ALDEx2,apeglm,ASSIGN,BaalChIP,bacon,BAGS,banocc,BASiCS,BayesKnockdown,BDMMAcorrect,birte,BitSeq,BPRMeth,BUMHMM,BUScorrect,ccfindR,Clomial,CNPBayes,CoGAPS,covEB,CSSP,ctsGE,DEqMS,DESeq2,EBcoexpress,EBSeqHMM,edgeR,genphen,GRENITS,GRridge,limma,limmaGUI,Logolas,MADSEQ,mfa,mlm4omics,networkBMA,NormalyzerDE,normr,omicplotR,pcaMethods,phenopath,PROPS,puma,R3CPET,RIVER,RJMCMCNucleosomes,scDD,scde,sparseDOSSA,tigre,TRONCO,wavClusteR" id="Bayesian" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="https://www.bioconductor.org/packages/release/BiocViews.html#"><ins class="jstree-icon">&nbsp;</ins>Bayesian (54)</a></li><li packagelist="AIMS,antiProfiles,BadRegionFinder,bgafun,BiocNeighbors,bioDist,BioSeqClass,biosigner,BUMHMM,cancerclass,Cardinal,ClassifyR,Clonality,clst,clstutils,ClusterSignificance,CMA,CNVPanelizer,CONFESS,consensusOV,COSNet,CRImage,ctc,CytoDx,dada2,DaMiRseq,decontam,dexus,DirichletMultinomial,dualKS,EGSEA,eisa,epigenomix,ExpressionView,fastseg,FCBF,flowcatchR,gaga,GARS,geneClassifiers,genefu,geNetClassifier,genphen,GenVisR,glmSparseNet,gpls,GRridge,iCOBRA,interactiveDisplay,interactiveDisplayBase,iterativeBMA,kebabs,LedPred,logicFS,MACPET,made4,maftools,maigesPack,maPredictDSC,massiR,MCRestimate,messina,metagenomeSeq,MethPed,methVisual,MiPP,miRNAtap,mixOmics,MLInterfaces,mlm4omics,MLSeq,MoPS,NetPathMiner,normr,NuPoP,omicade4,OmicsMarkeR,PAA,pbcmc,PharmacoGx,phenoTest,phyloseq,Pigengene,procoil,pRoloc,PROPS,RamiGO,RBioinf,rexposome,rfPred,RGSEA,rHVDM,Rmagpie,ropls,rRDP,sampleClassifier,SC3,scmap,SemDist,semisup,sevenC,SIAMCAT,SigCheck,sigFeature,STROMA4,switchBox,TTMap,twoddpcr,vbmp" id="Classification" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="https://www.bioconductor.org/packages/release/BiocViews.html#"><ins class="jstree-icon">&nbsp;</ins>Classification (109)</a></li><li packagelist="adSplit,ANF,artMS,BEARscc,BHC,BicARE,BiocNeighbors,BioCor,bioDist,BPRMeth,BUScorrect,CAGEr,CancerSubtypes,ccfindR,CellTrails,cellTree,CGEN,CHARGE,ChemmineOB,ChemmineR,chroGPS,chromswitch,cicero,Clomial,clustComp,clusterExperiment,ClusterJudge,clusterProfiler,clusterSeq,ClusterSignificance,clusterStab,CoGAPS,cogena,condcomp,CONFESS,ConsensusClusterPlus,consensusDE,consensusOV,CORREP,CountClust,ctc,ctsGE,cummeRbund,cytofkit,ddPCRclust,debrowser,DECIPHER,DEComplexDisease,deltaGseg,DESeq2,destiny,diffcyt,diffHic,DirichletMultinomial,DrugVsDisease,EBarrays,edgeR,eiR,fabia,FGNet,FindMyFriends,flowCHIC,flowClust,flowCyBar,flowDensity,flowFP,flowMatch,flowMeans,flowMerge,flowPeaks,FlowSOM,fmcsR,FunChIP,GARS,GateFinder,genefu,geneRecommender,geneRxCluster,ggtree,GOexpress,GOpro,GOSemSim,GOSim,goSTAG,GOsummaries,gpart,GraphPAC,hapFabia,hierGWAS,hierinf,hopach,iBBiG,iClusterPlus,immunoClust,iPAC,iSEE,isomiRs,iterClust,kebabs,LEA,limma,Linnorm,M3C,maanova,MACPET,made4,maigesPack,MantelCorr,massiR,mcaGUI,MCbiclust,meshes,MetaCyto,metagenomeSeq,methVisual,methyvim,Mfuzz,microbiome,miRSM,Mirsynergy,MLInterfaces,MLSeq,MmPalateMiRNA,mogsa,monocle,multiClust,nethet,NetPathMiner,netresponse,netSmooth,omicade4,OMICsPCA,Onassis,PANR,pbcmc,phantasus,phenoTest,phyloseq,PICS,Pigengene,PING,powerTCR,pRoloc,puma,QuartPAC,QUBIC,RBioinf,Rchemcpp,RDRToolbox,rexposome,RIVER,RNASeqR,rqubic,runibic,SAGx,SamSPECTRAL,SANTA,SC3,scDD,scran,semisup,SIAMCAT,SigFuge,SIMLR,sincell,singleCellTK,slingshot,SomaticSignatures,SpacePAC,SpeCond,STATegRa,supraHex,TCseq,TimeSeriesExperiment,TMixClust,transcriptogramer,TransView,treeio,TRONCO,TSRchitect,TTMap,YAPSA,yarn" id="Clustering" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="https://www.bioconductor.org/packages/release/BiocViews.html#"><ins class="jstree-icon">&nbsp;</ins>Clustering (183)</a></li><li packagelist="CMA,DEGraph,flowUtils,GENIE3,Pigengene,ScISI,TargetSearch" id="DecisionTree" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="https://www.bioconductor.org/packages/release/BiocViews.html#"><ins class="jstree-icon">&nbsp;</ins>DecisionTree (7)</a></li><li packagelist="adaptest,AffiXcan,BiocSklearn,cellity,CellTrails,CoGAPS,cytofkit,esetVis,gcatest,GENESIS,gep2pep,glmSparseNet,globalSeq,iSEE,lfa,M3Drop,missRows,netSmooth,OMICsPCA,oneSENSE,pcaExplorer,Pigengene,RDRToolbox,SC3,scater,slalom,STATegRa,tenXplore,zinbwave" id="DimensionReduction" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="https://www.bioconductor.org/packages/release/BiocViews.html#"><ins class="jstree-icon">&nbsp;</ins>DimensionReduction (29)</a></li><li packagelist="biosigner,BPRMeth,BUMHMM,BUScorrect,diffcyt,FCBF,GARS,GeneAccord,genphen,HIREewas,iasva,IrisSpatialFeatures,iSEE,M3Drop,maftools,mAPKL,martini,multiClust,netReg,OmicsMarkeR,rCGH,Rcpi,RDRToolbox,Rgin,SIAMCAT,sigFeature,synlet,trena,zFPKM" id="FeatureExtraction" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="https://www.bioconductor.org/packages/release/BiocViews.html#"><ins class="jstree-icon">&nbsp;</ins>FeatureExtraction (29)</a></li><li packagelist="ANF,apComplex,BatchQC,BiGGR,biocGraph,BioMVCClass,BioNet,BioNetStat,birta,CausalR,cellTree,CEMiTool,CHRONOS,coexnet,CoRegNet,CORREP,COSNet,DART,DEGraph,DEsubs,diffuStats,EGAD,EGSEA,EnrichmentBrowser,FELLA,FindMyFriends,gaggle,GeneNetworkBuilder,GENIE3,glmSparseNet,goProfiles,GOstats,graph,GraphAlignment,GraphAT,graphite,GRENITS,GRridge,GSEABase,GSEABenchmarkeR,gsean,hipathia,hyperdraw,hypergraph,ibh,IMMAN,joda,KEGGgraph,KEGGlincs,keggorthology,maigesPack,martini,MetaboSignal,minet,mirIntegrator,MVCClass,NCIgraph,NeighborNet,netbenchmark,netbiov,nethet,NetPathMiner,netReg,netresponse,netSmooth,paircompviz,pandaR,PANR,PathoStat,pathRender,pathview,PathwaySplice,paxtoolsr,PCpheno,Pi,Pigengene,pkgDepTools,plotGrouper,PPInfer,ppiStats,predictionet,ProCoNA,PSICQUIC,qpgraph,QuaternaryProd,R3CPET,RamiGO,RandomWalkRestartMH,RBGL,RCy3,RCyjs,RDAVIDWebService,RedeR,RefNet,Rgin,RGraph2js,Rgraphviz,rHVDM,rsbml,RTN,RTNduals,RTNsurvival,rTRM,rTRMui,rWikiPathways,SBMLR,ScISI,signet,sincell,singscore,SLGI,sparsenetgls,SPIA,splineTimeR,SQUADD,ternarynet,ToPASeq,transcriptogramer,TRONCO" id="GraphAndNetwork" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="https://www.bioconductor.org/packages/release/BiocViews.html#"><ins class="jstree-icon">&nbsp;</ins>GraphAndNetwork (119)</a></li><li packagelist="AneuFinder,BUMHMM,chromstaR,DMCHMM,DOQTL,EBSeqHMM,iCNV,methimpute,nuCpos,STAN,TitanCNA" id="HiddenMarkovModel" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="https://www.bioconductor.org/packages/release/BiocViews.html#"><ins class="jstree-icon">&nbsp;</ins>HiddenMarkovModel (11)</a></li><li class="jstree-leaf" style="display: none;"><ins class="jstree-icon">&nbsp;</ins><a href="https://www.bioconductor.org/packages/release/BiocViews.html#"><ins class="jstree-icon">&nbsp;</ins>MultidimensionalScaling</a></li><li packagelist="COSNet" id="NeuralNetwork" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="https://www.bioconductor.org/packages/release/BiocViews.html#"><ins class="jstree-icon">&nbsp;</ins>NeuralNetwork (1)</a></li><li packagelist="GeneAccord,PathoStat,slinky" id="PatternLogic" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="https://www.bioconductor.org/packages/release/BiocViews.html#"><ins class="jstree-icon">&nbsp;</ins>PatternLogic (3)</a></li><li packagelist="acde,BatchQC,ClusterSignificance,COCOA,DESeq2,ENmix,esetVis,gcatest,GENESIS,Harman,lfa,missRows,mogsa,OMICsPCA,PathoStat,pcaExplorer,phantasus,phenopath,Pigengene,psichomics,ramwas,RiboProfiling,ropls,rqt,SNPRelate,STATegRa,tenXplore,TimeSeriesExperiment" id="PrincipalComponent" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="https://www.bioconductor.org/packages/release/BiocViews.html#"><ins class="jstree-icon">&nbsp;</ins>PrincipalComponent (28)</a></li><li packagelist="adaptest,AffiXcan,affylmGUI,anota2seq,bacon,biobroom,birte,BPRMeth,BUMHMM,CFAssay,chipenrich,CONFESS,consensus,CytoDx,DESeq2,dmrseq,Doscheda,edge,edgeR,ENmix,flipflop,flowPloidy,flowQB,GAprediction,gCrisprTools,GEM,GENIE3,GenoGAM,genphen,gespeR,GIGSEA,glmSparseNet,GlobalAncova,globalSeq,GMRP,gpls,GRridge,gwasurvivr,hierinf,IMAS,IVAS,kebabs,LEA,limma,limmaGUI,metaCCA,methylGSA,MetNet,mixOmics,mlm4omics,MoPS,NBSplice,netReg,omicRexposome,OPWeight,PathwaySplice,perturbatr,plrs,pmm,RIVER,RNAdecay,ropls,rqt,SAGx,sevenC,SIAMCAT,sparsenetgls,splineTimeR,SPONGE,SRGnet,switchde,trena,Trendy,variancePartition" id="Regression" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="https://www.bioconductor.org/packages/release/BiocViews.html#"><ins class="jstree-icon">&nbsp;</ins>Regression (74)</a></li><li packagelist="ASGSCA" id="StructuralEquationModels" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="https://www.bioconductor.org/packages/release/BiocViews.html#"><ins class="jstree-icon">&nbsp;</ins>StructuralEquationModels (1)</a></li><li packagelist="cellity,genphen,kebabs,LedPred,procoil,SC3,scmap,sigFeature,SVM2CRM" id="SupportVectorMachine" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="https://www.bioconductor.org/packages/release/BiocViews.html#"><ins class="jstree-icon">&nbsp;</ins>SupportVectorMachine (9)</a></li><li packagelist="CFAssay,ClassifyR,CytoDx,GDCRNATools,geneClassifiers,glmSparseNet,GRridge,gwasurvivr,ivygapSE,maftools,messina,miRsponge,MoonlightR,multiClust,psichomics,rqt,RTNsurvival,simulatorZ,TCGAbiolinks" id="Survival" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="https://www.bioconductor.org/packages/release/BiocViews.html#"><ins class="jstree-icon">&nbsp;</ins>Survival (19)</a></li><li packagelist="acde,affylmGUI,ASpli,biosvd,cellGrowth,CellNOptR,CellTrails,cellTree,CNORdt,CNORode,CoGAPS,CONFESS,ctsGE,cycle,deltaGseg,DMRcate,EBSeqHMM,edge,edgeR,EventPointer,flowTime,GOexpress,gprege,GRENITS,GRmetrics,ImpulseDE,ImpulseDE2,INSPEcT,limma,limmaGUI,LineagePulse,LiquidAssociation,maSigPro,Mfuzz,mitoODE,MoPS,MSstats,plethy,rain,RmiR,RNAdecay,Rnits,RTCA,splineTimeR,TCseq,TDARACNE,tigre,timecourse,TimeSeriesExperiment,TMixClust,Trendy,waveTiling" id="TimeCourse" class="jstree-last jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="https://www.bioconductor.org/packages/release/BiocViews.html#"><ins class="jstree-icon">&nbsp;</ins>TimeCourse (52)</a></li></ul></li><li packagelist="ACME,amplican,anamiR,cbaf,CoverageView,DMRScan,dupRadar,ELBOW,HiCBricks,iGC,LymphoSeq,MDTS,MetID,nondetects,openPrimeR,openPrimeRui,scruff,spkTools,wavClusteR" id="Technology" class="jstree-closed"><ins class="jstree-icon">&nbsp;</ins><a href="https://www.bioconductor.org/packages/release/BiocViews.html#"><ins class="jstree-icon">&nbsp;</ins>Technology (1049)</a><ul><li packagelist="amplican,bcSeq,CRISPRseek,CrispRVariants,gCrisprTools,GUIDEseq,MAGeCKFlute" id="CRISPR" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="https://www.bioconductor.org/packages/release/BiocViews.html#"><ins class="jstree-icon">&nbsp;</ins>CRISPR (7)</a></li><li packagelist="ddPCRclust,twoddpcr" id="ddPCR" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="https://www.bioconductor.org/packages/release/BiocViews.html#"><ins class="jstree-icon">&nbsp;</ins>ddPCR (2)</a></li><li packagelist="CATALYST,COMPASS,cydar,CytoDx,cytofkit,cytolib,CytoML,diffcyt,flowAI,flowBeads,flowBin,flowCHIC,flowCL,flowClean,flowClust,flowCore,flowCyBar,flowDensity,flowFit,flowFP,flowMap,flowMatch,flowMeans,flowMerge,flowPeaks,flowPloidy,flowPlots,flowQ,flowQB,FlowRepositoryR,FlowSOM,flowStats,flowTime,flowTrans,flowType,flowUtils,flowViz,flowVS,flowWorkspace,GateFinder,ggcyto,immunoClust,MetaCyto,MIMOSA,ncdfFlow,oneSENSE,openCyto,plateCore,plotGrouper,QUALIFIER,RchyOptimyx,SamSPECTRAL,Sconify" id="FlowCytometry" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="https://www.bioconductor.org/packages/release/BiocViews.html#"><ins class="jstree-icon">&nbsp;</ins>FlowCytometry (53)</a></li><li packagelist="apComplex,artMS,BRAIN,CAMERA,Cardinal,CATALYST,cosmiq,customProDB,DAPAR,DEP,DEqMS,Doscheda,flagme,gaga,INDEED,IPO,isobar,IsoCorrectoR,LOBSTAHS,MAIT,MassArray,MassSpecWavelet,Metab,metabomxtr,metaMS,MetCirc,MetNet,mlm4omics,MSGFgui,MSGFplus,msmsEDA,msmsTests,MSnbase,MSnID,msPurity,MSstats,MSstatsQC,MSstatsQCgui,MSstatsTMT,mzID,mzR,PAPi,Pbase,pepXMLTab,PGA,pgca,plgem,proBAMr,PROcess,proFIA,pRoloc,Prostar,ProteoMM,proteoQC,ProtGenerics,qcmetrics,qPLEXanalyzer,Rdisop,Risa,RMassBank,rols,ropls,rpx,rTANDEM,sapFinder,SDAMS,shinyTANDEM,SIMAT,sparsenetgls,specL,statTarget,SWATH2stats,synapter,TargetSearch,tofsims,topdownr,TPP,xcms,yamss" id="MassSpectrometry" class="jstree-closed"><ins class="jstree-icon">&nbsp;</ins><a href="https://www.bioconductor.org/packages/release/BiocViews.html#"><ins class="jstree-icon">&nbsp;</ins>MassSpectrometry (79)</a><ul><li packagelist="tofsims" id="ImagingMassSpectrometry" class="jstree-last jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="https://www.bioconductor.org/packages/release/BiocViews.html#"><ins class="jstree-icon">&nbsp;</ins>ImagingMassSpectrometry (1)</a></li></ul></li><li packagelist="a4,a4Base,a4Classif,a4Core,a4Preproc,a4Reporting,ABarray,acde,ACME,ADaCGH2,adaptest,adSplit,affxparser,affy,affycomp,AffyCompatible,affycoretools,AffyExpress,affyILM,affyio,affylmGUI,affyPara,affypdnn,affyPLM,affyQCReport,AffyRNADegradation,AGDEX,AgiMicroRna,AIMS,altcdfenvs,AMOUNTAIN,anamiR,annaffy,annmap,AnnotationDbi,annotationTools,anota,anota2seq,aroma.light,ArrayExpress,arrayMvout,arrayQuality,arrayQualityMetrics,ArrayTools,ARRmNormalization,attract,BAC,BatchQC,beadarray,BeadDataPackR,BgeeDB,BGmix,bgx,BHC,BicARE,bigmelon,biomvRCNS,BioNet,BioNetStat,biotmle,birta,birte,blima,BLMA,BrainStars,bridge,CAFE,CALIB,cancerclass,CausalR,ccmap,CCPROMISE,CellMapper,CellScore,cellTree,CGHbase,CGHcall,cghMCR,CGHnormaliter,CGHregions,ChAMP,charm,chopsticks,CINdex,Clonality,cn.farms,CNTools,codelink,coexnet,CoGAPS,cogena,coGPS,COHCAP,coMET,consensus,consensusOV,conumee,convert,Cormotif,CORREP,covEB,crlmm,crossmeta,ctc,cycle,daMA,dcGSA,DECIPHER,DEDS,DEGraph,DFP,diffcoexp,diffGeneAnalysis,discordant,DMRcate,DMRforPairs,DNAcopy,doppelgangR,DrugVsDisease,DTA,dualKS,dyebias,eegc,EGSEA,eisa,ELBOW,EMDomics,ENmix,EnrichmentBrowser,erccdashboard,ExiMiR,explorase,ExpressionView,fabia,facopy,factDesign,farms,fdrame,ffpe,flowVS,focalCall,frma,frmaTools,gage,gCMAP,gCMAPWeb,gcrma,geecc,genArise,geneClassifiers,genefilter,GeneMeta,GeneNetworkBuilder,geneRecommender,GeneRegionScan,geNetClassifier,genoCN,GenomeGraphs,genoset,GenRank,GEOquery,GEOsubmission,GLAD,Glimma,GlobalAncova,globaltest,GOexpress,GOFunction,goProfiles,goSTAG,GOstats,goTools,gpls,gprege,GSEABenchmarkeR,GSEAlm,GSRI,GSVA,Gviz,GWASTools,Harman,Harshlight,HDTD,Heatplus,HELP,HEM,HMMcopy,HybridMTest,iBMQ,iCheck,iChip,iClusterPlus,IdeoViz,iGC,illuminaio,impute,interactiveDisplay,interactiveDisplayBase,InterMineR,IntramiRExploreR,inveRsion,IsoGeneGUI,ITALICS,iterativeBMA,iterativeBMAsurv,joda,KCsmart,kimod,lapmix,les,limma,limmaGUI,lmdme,LMGene,logitT,LPE,LPEadj,lumi,LVSmiRNA,maanova,macat,maCorrPlot,maigesPack,MANOR,mAPKL,maPredictDSC,marray,maSigPro,maskBAD,massiR,matchBox,MBCB,mBPCR,MCbiclust,mCSEA,mdp,mdqc,MEAL,MEDIPS,MEDME,MergeMaid,metaArray,metahdep,methyAnalysis,MethylAid,Mfuzz,MGFM,MiChip,MIGSA,mimager,minet,minfi,MinimumDistance,MiPP,MiRaGE,mirIntegrator,miRNAtap,miRSM,miRsponge,mixOmics,MmPalateMiRNA,MODA,motifStack,MotIV,Mulcom,multiscan,multtest,NanoStringQCPro,nem,netbenchmark,netresponse,nnNorm,normalize450K,npGSEA,nucleR,nudge,OCplus,OGSA,oligo,OLIN,OLINgui,OrderedList,OutlierD,PAA,PADOG,pandaR,PathoStat,pathview,pbcmc,pcaGoPromoter,pcot2,PECA,pepStat,PGSEA,phantasus,phenoTest,piano,pickgene,Pigengene,pint,plgem,PLPE,plw,POST,prebs,PROMISE,puma,pvac,pvca,Pviz,qcmetrics,qpgraph,quantro,QUBIC,qusage,rain,rama,RankProd,RBioinf,RBM,rbsurv,RDRToolbox,reb,RefPlus,regsplice,REMP,ReportingTools,rGADEM,rHVDM,Ringo,RLMM,Rmagpie,rMAT,RmiR,Rnits,ROntoTools,ROTS,RPA,rqubic,RRHO,RTopper,runibic,SAGx,sampleClassifier,SCAN.UPC,semisup,shinyMethyl,sigaR,sigFeature,siggenes,SIM,SimBindProfiles,simpleaffy,sizepower,SMAP,SNAGEE,snapCGH,snm,snpStats,SpeCond,SPIA,spikeLI,spkTools,spotSegmentation,sRAP,ssize,SSPA,STAN,Starr,stepNorm,STROMA4,sva,TDARACNE,TEQC,tigre,tilingArray,timecourse,TIN,topGO,transcriptogramer,transite,tRanslatome,TransView,trigger,trio,tspair,TTMap,TurboNorm,twilight,vsn,wateRmelon,waveTiling,webbioc,XDE,xps,yaqcaffy" id="Microarray" class="jstree-closed"><ins class="jstree-icon">&nbsp;</ins><a href="https://www.bioconductor.org/packages/release/BiocViews.html#"><ins class="jstree-icon">&nbsp;</ins>Microarray (427)</a><ul><li packagelist="ChIPanalyser,geneXtendeR,GenoGAM,puma,qsea,STAN,SVM2CRM" id="ChipOnChip" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="https://www.bioconductor.org/packages/release/BiocViews.html#"><ins class="jstree-icon">&nbsp;</ins>ChipOnChip (7)</a></li><li packagelist="GDSArray" id="GenotypingArray" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="https://www.bioconductor.org/packages/release/BiocViews.html#"><ins class="jstree-icon">&nbsp;</ins>GenotypingArray (1)</a></li><li packagelist="bacon,bigmelon,ChAMP,coMET,compartmap,conumee,DMRcate,ENmix,EpiDISH,GEM,MBAmethyl,mCSEA,MethylAid,MethylMix,methyvim,minfi,missMethyl,MoonlightR,normalize450K,REMP,RnBeads,sesame,TCGAbiolinks,TCGAbiolinksGUI,TurboNorm" id="MethylationArray" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="https://www.bioconductor.org/packages/release/BiocViews.html#"><ins class="jstree-icon">&nbsp;</ins>MethylationArray (25)</a></li><li packagelist="limma" id="MicroRNAArray" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="https://www.bioconductor.org/packages/release/BiocViews.html#"><ins class="jstree-icon">&nbsp;</ins>MicroRNAArray (1)</a></li><li packagelist="acde,affylmGUI,CEMiTool,discordant,erccdashboard,EventPointer,goSTAG,limma,limmaGUI,NanoStringQCPro,pbcmc,puma,sigFeature,transite" id="mRNAMicroarray" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="https://www.bioconductor.org/packages/release/BiocViews.html#"><ins class="jstree-icon">&nbsp;</ins>mRNAMicroarray (14)</a></li><li packagelist="aroma.light,ELBOW,ENmix,flowQB,minfi,REMP,snm" id="MultiChannel" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="https://www.bioconductor.org/packages/release/BiocViews.html#"><ins class="jstree-icon">&nbsp;</ins>MultiChannel (7)</a></li><li packagelist="ABarray,affxparser,affy,affycomp,AffyCompatible,affycoretools,AffyExpress,affyILM,affylmGUI,affypdnn,affyPLM,affyQCReport,AffyRNADegradation,AgiMicroRna,altcdfenvs,annaffy,annmap,aroma.light,ArrayExpress,arrayQualityMetrics,ArrayTools,beadarray,BrainStars,bridge,CAFE,clippda,codelink,copa,crossmeta,diffcoexp,DMRcate,EGSEA,ELBOW,ENmix,ExiMiR,flowQB,gaga,gage,gcrma,GeneRegionScan,GEOquery,GlobalAncova,globaltest,Harshlight,iCheck,iChip,karyoploteR,lapmix,limma,lmdme,lumi,LVSmiRNA,makecdfenv,oligo,PAA,PADOG,pbcmc,plw,PROMISE,puma,pvac,readat,RefPlus,RLMM,SAGx,SCAN.UPC,simpleaffy,SNAGEE,snm,Starr,tilingArray,TurboNorm,vsn,webbioc,xps,yaqcaffy" id="OneChannel" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="https://www.bioconductor.org/packages/release/BiocViews.html#"><ins class="jstree-icon">&nbsp;</ins>OneChannel (76)</a></li><li packagelist="affxparser,affylmGUI,altcdfenvs,illuminaio,limma,readat" id="ProprietaryPlatforms" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="https://www.bioconductor.org/packages/release/BiocViews.html#"><ins class="jstree-icon">&nbsp;</ins>ProprietaryPlatforms (6)</a></li><li class="jstree-leaf" style="display: none;"><ins class="jstree-icon">&nbsp;</ins><a href="https://www.bioconductor.org/packages/release/BiocViews.html#"><ins class="jstree-icon">&nbsp;</ins>ReversePhaseProteinArray</a></li><li packagelist="crossmeta" id="TissueMicroarray" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="https://www.bioconductor.org/packages/release/BiocViews.html#"><ins class="jstree-icon">&nbsp;</ins>TissueMicroarray (1)</a></li><li packagelist="aroma.light,ArrayExpress,arrayQuality,arrayQualityMetrics,ARRmNormalization,beadarraySNP,bigmelon,bridge,CALIB,ChAMP,convert,copa,daMA,diffcoexp,DMRcate,dyebias,EGSEA,ELBOW,ENmix,ExiMiR,gage,genArise,GEOquery,HELP,iCheck,limma,limmaGUI,lmdme,lumi,maigesPack,MANOR,marray,mCSEA,MethylAid,methylumi,minfi,MmPalateMiRNA,nnNorm,normalize450K,nudge,oligo,OLIN,OLINgui,PADOG,pbcmc,plw,puma,rama,REMP,Ringo,RnBeads,SCAN.UPC,shinyMethyl,skewr,SMAP,SNAGEE,snapCGH,snm,spotSegmentation,stepNorm,TurboNorm,vsn,wateRmelon" id="TwoChannel" class="jstree-last jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="https://www.bioconductor.org/packages/release/BiocViews.html#"><ins class="jstree-icon">&nbsp;</ins>TwoChannel (63)</a></li></ul></li><li packagelist="bioassayR,cellGrowth,ChemmineOB,ChemmineR,ddCt,eiR,fmcsR,HTqPCR,NormqPCR,Rchemcpp,ReadqPCR,sights,SLqPCR,splots,staRank,unifiedWMWqPCR" id="MicrotitrePlateAssay" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="https://www.bioconductor.org/packages/release/BiocViews.html#"><ins class="jstree-icon">&nbsp;</ins>MicrotitrePlateAssay (16)</a></li><li packagelist="amplican,ddCt,DECIPHER,EasyqpcR,HTqPCR,miRcomp,nondetects,NormqPCR,ReadqPCR,SLqPCR,unifiedWMWqPCR" id="qPCR" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="https://www.bioconductor.org/packages/release/BiocViews.html#"><ins class="jstree-icon">&nbsp;</ins>qPCR (11)</a></li><li packagelist="BADER,baySeq,DESeq,edgeR,GEOquery,MBttest,MiRaGE,rnaSeqMap,sagenhaft" id="SAGE" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="https://www.bioconductor.org/packages/release/BiocViews.html#"><ins class="jstree-icon">&nbsp;</ins>SAGE (9)</a></li><li packagelist="abseqR,ACE,adaptest,ALDEx2,AllelicImbalance,alpine,ampliQueso,anamiR,AneuFinder,AnnotationDbi,anota,anota2seq,apeglm,appreci8R,ArrayExpressHTS,ASpli,ATACseqQC,BaalChIP,BADER,BadRegionFinder,bamsignals,basecallQC,BaseSpaceR,Basic4Cseq,BASiCS,BatchQC,bayNorm,baySeq,BBCAnalyzer,bcSeq,BgeeDB,biomvRCNS,BioNetStat,Biostrings,biotmle,birta,birte,BiSeq,BitSeq,BPRMeth,breakpointR,BubbleTree,BUMHMM,CAGEfightR,CAGEr,casper,cellity,CellTrails,cellTree,CexoR,Chicago,ChIPanalyser,ChIPComp,ChIPexoQual,ChIPQC,chipseq,ChIPSeqSpike,chromPlot,chromstaR,chromVAR,cicero,CINdex,cleanUpdTSeq,Clomial,clonotypeR,clstutils,clusterExperiment,clusterSeq,cn.mops,CNAnorm,CNVPanelizer,CNVrd2,COCOA,cogena,coMET,compEpiTools,ComplexHeatmap,consensusSeekeR,CorMut,coseq,CountClust,CoverageView,csaw,CSSP,ctsGE,customProDB,dada2,DaMiRseq,DBChIP,dcGSA,DChIPRep,debrowser,DECIPHER,decontam,deepSNV,DEFormats,derfinder,derfinderHelper,derfinderPlot,DEScan2,DESeq,DESeq2,DEsingle,DEXSeq,dexus,DiffBind,diffcoexp,diffHic,DirichletMultinomial,DMCHMM,DMRcaller,DMRScan,dmrseq,DNABarcodes,DRIMSeq,DropletUtils,DSS,dupRadar,EBSEA,EBSeq,EBSeqHMM,EDASeq,EDDA,edgeR,eegc,EGSEA,ELBOW,EnrichedHeatmap,ensembldb,esATAC,EventPointer,exomeCopy,exomePeak,facopy,FastqCleaner,fCCAC,focalCall,FourCSeq,gage,gaucho,gcapc,GDSArray,GeneBreak,GeneMeta,GeneNetworkBuilder,geneRxCluster,genomation,GenomicAlignments,GenomicDataCommons,GenomicFeatures,GenomicFiles,GenomicRanges,GenomicScores,GenomicTuples,genotypeeval,genphen,GenRank,girafe,Glimma,globalSeq,GMRP,GOexpress,goseq,GOTHiC,GreyListChIP,groHMM,gtrellis,GUIDEseq,Guitar,hapFabia,HDF5Array,HDTD,HelloRanges,HiCBricks,HiCcompare,hicrep,HilbertCurve,hiReadsProcessor,HiTC,HMMcopy,htSeqTools,HTSFilter,icetea,ideal,iGC,IMAS,Imetagene,ImpulseDE2,InPAS,INSPEcT,InTAD,intansv,interactiveDisplay,interactiveDisplayBase,IntEREst,IONiseR,iSeq,ISoLDE,IVAS,ivygapSE,JunctionSeq,karyoploteR,LedPred,levi,limma,LineagePulse,Linnorm,loci2path,LOLA,LowMACA,LymphoSeq,M3C,M3Drop,MADSEQ,maftools,manta,MBASED,MBttest,mcaGUI,MDTS,MEDIPS,metagene,metagenomeFeatures,metagenomeSeq,metaSeq,methimpute,methInheritSim,MethTargetedNGS,methylInheritance,methylKit,methylMnM,methylPipe,MethylSeekR,microbiome,MIRA,MiRaGE,miRNAtap,mixOmics,MLSeq,MMDiff2,monocle,mosaics,motifStack,mpra,MPRAnalyze,msa,multiHiCcompare,muscle,NADfinder,NarrowPeaks,netSmooth,NGScopy,NOISeq,nucleoSim,nucleR,omicplotR,oncomix,ORFik,Organism.dplyr,Oscope,OTUbase,OUTRIDER,panelcn.mops,PathoStat,pathview,PathwaySplice,PCAN,PGA,philr,phyloseq,PICS,PING,podkat,polyester,Polyfit,PowerExplorer,prebs,primirTSS,PROPER,psichomics,PureCN,QDNAseq,qrqc,qsea,QSutils,quantro,QuasR,r3Cseq,R453Plus1Toolbox,ramwas,RareVariantVis,Rariant,Rbowtie,Rbowtie2,Rcade,recount,recoup,REDseq,regionReport,regsplice,REMP,ReQON,restfulSE,rGADEM,rGREAT,Rhtslib,RiboProfiling,riboSeqR,RIPSeeker,RJMCMCNucleosomes,RNAprobR,rnaSeqMap,RNASeqR,RnaSeqSampleSize,RnBeads,roar,Rqc,rqt,rRDP,Rsamtools,rSFFreader,Rsubread,RSVSim,RTCGAToolbox,runibic,samExploreR,sangerseqR,savR,Scale4C,scater,scfind,scmap,scone,scPipe,scran,scruff,segmentSeq,SeqArray,seqbias,seqCAT,seqCNA,SeqGSEA,seqplots,seqsetvis,seqTools,SeqVarTools,ShortRead,SIAMCAT,SICtools,SIMLR,sincell,slalom,slingshot,SMITE,soGGi,SomaticSignatures,splatter,SplicingGraphs,splots,SRAdb,ssviz,STAN,subSeq,SummarizedExperiment,Sushi,sva,systemPipeR,TarSeqQC,TCC,TCGAbiolinks,TCGAbiolinksGUI,TCseq,TEQC,TFEA.ChIP,TimeSeriesExperiment,TissueEnrich,TitanCNA,tracktables,transcriptR,TransView,traseR,triform,TSRchitect,TSSi,TVTB,tweeDEseq,Ularcirc,VariantAnnotation,VariantFiltering,VariantTools,wavClusteR,wiggleplotr,Wrench,XBSeq,YAPSA,yarn,zinbwave" id="Sequencing" class="jstree-closed"><ins class="jstree-icon">&nbsp;</ins><a href="https://www.bioconductor.org/packages/release/BiocViews.html#"><ins class="jstree-icon">&nbsp;</ins>Sequencing (552)</a><ul><li packagelist="ATACseqQC,BiFET,chromstaR,cicero,compartmap,esATAC" id="ATACSeq" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="https://www.bioconductor.org/packages/release/BiocViews.html#"><ins class="jstree-icon">&nbsp;</ins>ATACSeq (6)</a></li><li packagelist="ALDEx2,BaalChIP,BayesPeak,ChIC,ChIPanalyser,ChIPComp,chipenrich,ChIPexoQual,ChIPpeakAnno,ChIPQC,ChIPseeker,chipseq,ChIPseqR,ChIPSeqSpike,ChIPsim,ChIPXpress,chroGPS,chromstaR,cobindR,COCOA,coMET,consensusSeekeR,CoverageView,CSAR,csaw,CSSP,DBChIP,DChIPRep,debrowser,derfinder,DESeq,DESeq2,DiffBind,DSS,EDDA,edgeR,epigenomix,FunChIP,gcapc,geneXtendeR,GenoGAM,GreyListChIP,iASeq,Imetagene,InTAD,iSeq,karyoploteR,LedPred,Linnorm,LOLA,MEDIPS,metagene,MIRA,MMDiff2,motifbreakR,motifStack,MotIV,NarrowPeaks,normr,nucleR,PICS,PowerExplorer,qsea,QuasR,Rcade,recoup,regioneR,rGADEM,RJMCMCNucleosomes,Rsubread,samExploreR,seqplots,seqsetvis,sevenC,similaRpeak,soGGi,STAN,SVM2CRM,systemPipeR,TCseq,TFEA.ChIP,TFHAZ,TransView,triform,vulcan,wiggleplotr" id="ChIPSeq" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="https://www.bioconductor.org/packages/release/BiocViews.html#"><ins class="jstree-icon">&nbsp;</ins>ChIPSeq (86)</a></li><li class="jstree-leaf" style="display: none;"><ins class="jstree-icon">&nbsp;</ins><a href="https://www.bioconductor.org/packages/release/BiocViews.html#"><ins class="jstree-icon">&nbsp;</ins>DenovoGenome</a></li><li class="jstree-leaf" style="display: none;"><ins class="jstree-icon">&nbsp;</ins><a href="https://www.bioconductor.org/packages/release/BiocViews.html#"><ins class="jstree-icon">&nbsp;</ins>DenovoTranscriptome</a></li><li packagelist="ACE,ALDEx2,ATACseqQC,breakpointR,Clomial,CNAnorm,coMET,DEComplexDisease,esATAC,GeneBreak,GenVisR,karyoploteR,maftools,NADfinder,NGScopy,omicplotR,QDNAseq,QSutils,regioneR,samExploreR,semisup,TCGAbiolinksGUI,TitanCNA,TVTB,uSORT,YAPSA" id="DNASeq" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="https://www.bioconductor.org/packages/release/BiocViews.html#"><ins class="jstree-icon">&nbsp;</ins>DNASeq (26)</a></li><li packagelist="BiFET,esATAC" id="DNaseSeq" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="https://www.bioconductor.org/packages/release/BiocViews.html#"><ins class="jstree-icon">&nbsp;</ins>DNaseSeq (2)</a></li><li packagelist="Clomial,CODEX,coMET,CopywriteR,globalSeq,iCNV,NGScopy,RVS,TitanCNA,Uniquorn" id="ExomeSeq" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="https://www.bioconductor.org/packages/release/BiocViews.html#"><ins class="jstree-icon">&nbsp;</ins>ExomeSeq (10)</a></li><li packagelist="bnbc,Chicago,diffHic,GenomicInteractions,GOTHiC,HiCBricks,HiCcompare,hicrep,HiTC,InTAD,InteractionSet,MACPET,multiHiCcompare,Pi,R3CPET,sevenC,Sushi" id="HiC" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="https://www.bioconductor.org/packages/release/BiocViews.html#"><ins class="jstree-icon">&nbsp;</ins>HiC (17)</a></li><li packagelist="BEAT,BiSeq,COCOA,COHCAP,coMET,compartmap,GEM,Guitar,karyoploteR,LOLA,methInheritSim,methylInheritance,methylKit,methylPipe,MethylSeekR,methyvim,MIRA,MPFE,msgbsR,QuasR,regioneR,Repitools,RnBeads,systemPipeR,TransView" id="MethylSeq" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="https://www.bioconductor.org/packages/release/BiocViews.html#"><ins class="jstree-icon">&nbsp;</ins>MethylSeq (25)</a></li><li class="jstree-leaf" style="display: none;"><ins class="jstree-icon">&nbsp;</ins><a href="https://www.bioconductor.org/packages/release/BiocViews.html#"><ins class="jstree-icon">&nbsp;</ins>MicrobialStrain</a></li><li packagelist="ALDEx2,BDMMAcorrect,biomformat,countsimQC,dada2,DECIPHER,decontam,DESeq2,DirichletMultinomial,metagenomeFeatures,metagenomeSeq,microbiome,omicplotR,PathoStat,philr,phyloseq,rRDP,SIAMCAT,sparseDOSSA,TimeSeriesExperiment" id="Microbiome" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="https://www.bioconductor.org/packages/release/BiocViews.html#"><ins class="jstree-icon">&nbsp;</ins>Microbiome (20)</a></li><li packagelist="anamiR,globalSeq,isomiRs,miRBaseConverter,miRLAB,miRNAmeConverter,miRNApath,miRNAtap,rcellminer,Roleswitch,SpidermiR,TargetScore" id="miRNA" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="https://www.bioconductor.org/packages/release/BiocViews.html#"><ins class="jstree-icon">&nbsp;</ins>miRNA (12)</a></li><li class="jstree-leaf" style="display: none;"><ins class="jstree-icon">&nbsp;</ins><a href="https://www.bioconductor.org/packages/release/BiocViews.html#"><ins class="jstree-icon">&nbsp;</ins>MNaseSeq</a></li><li packagelist="gCrisprTools,pogos" id="PooledScreens" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="https://www.bioconductor.org/packages/release/BiocViews.html#"><ins class="jstree-icon">&nbsp;</ins>PooledScreens (2)</a></li><li packagelist="coMET,ORFik,RiboProfiling,riboSeqR,systemPipeR" id="RiboSeq" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="https://www.bioconductor.org/packages/release/BiocViews.html#"><ins class="jstree-icon">&nbsp;</ins>RiboSeq (5)</a></li><li packagelist="BiFET,normr,RIPSeeker,wavClusteR" id="RIPSeq" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="https://www.bioconductor.org/packages/release/BiocViews.html#"><ins class="jstree-icon">&nbsp;</ins>RIPSeq (4)</a></li><li packagelist="AIMS,ALDEx2,alpine,ampliQueso,Anaquin,anota2seq,apeglm,ArrayExpressHTS,ASpli,attract,bacon,BADER,ballgown,Basic4Cseq,BASiCS,BatchQC,bayNorm,BioNetStat,biotmle,BitSeq,BPRMeth,BUMHMM,CancerInSilico,caOmicsV,casper,CausalR,cellity,cellTree,CEMiTool,clusterExperiment,CoGAPS,coMET,compcodeR,CONFESS,consensus,coseq,CountClust,countsimQC,covEB,CoverageView,cqn,cTRAP,ctsGE,customProDB,DaMiRseq,dcGSA,debrowser,DEFormats,DEGreport,DEGseq,derfinder,derfinderHelper,derfinderPlot,DESeq,DESeq2,DEsingle,DEsubs,DEXSeq,dexus,diffcoexp,discordant,doppelgangR,DRIMSeq,DropletUtils,DSS,dupRadar,easyRNASeq,EBSeq,EBSeqHMM,EDASeq,EDDA,edgeR,eegc,EGSEA,ELBOW,EnhancedVolcano,EnrichmentBrowser,erccdashboard,ERSSA,EventPointer,ExCluster,flipflop,gage,gCrisprTools,GDCRNATools,geecc,GeneStructureTools,GenomicAlignments,Glimma,globalSeq,GOexpress,goseq,goSTAG,GRridge,GSEABenchmarkeR,Guitar,HDF5Array,HTSFilter,iASeq,iasva,icetea,ideal,IHW,IMAS,InPAS,INSPEcT,InTAD,IntEREst,IsoformSwitchAnalyzeR,ISoLDE,isomiRs,IVAS,JunctionSeq,kissDE,limma,Linnorm,M3C,M3Drop,manta,MAST,MCbiclust,metaSeq,metaseqR,mfa,MGFR,MIGSA,MiRaGE,MLSeq,monocle,NBSplice,netSmooth,NOISeq,omicplotR,OPWeight,ORFik,Oscope,OUTRIDER,PathoStat,pathview,PathwaySplice,pbcmc,pcaExplorer,PGA,phantasus,phenopath,Pigengene,Polyfit,PowerExplorer,prebs,primirTSS,Prize,PROPER,psichomics,QuasR,qusage,recount,recoup,regionReport,regsplice,ReportingTools,rgsepd,RNAdecay,rnaseqcomp,rnaSeqMap,RNASeqPower,RNASeqR,RnaSeqSampleSize,ROTS,Rsubread,RTCGA,RUVSeq,samExploreR,sampleClassifier,sapFinder,SC3,SCAN.UPC,scater,scDD,scde,scFeatureFilter,scfind,scmap,SCnorm,scone,scPipe,scran,scruff,SeqGSEA,seqplots,SGSeq,SigFuge,sincell,slalom,slingshot,SMITE,splatter,SplicingGraphs,SPLINTER,sSeq,SSPA,ssviz,STAN,strandCheckR,STROMA4,subSeq,sva,SVAPLSseq,switchde,systemPipeR,TCC,TCseq,TFEA.ChIP,TimeSeriesExperiment,ToPASeq,transcriptogramer,transcriptR,TransView,Trendy,TSSi,tweeDEseq,tximeta,tximport,uSORT,variancePartition,vidger,wavClusteR,wiggleplotr,XBSeq,XINA,zFPKM,zinbwave" id="RNASeq" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="https://www.bioconductor.org/packages/release/BiocViews.html#"><ins class="jstree-icon">&nbsp;</ins>RNASeq (231)</a></li><li packagelist="FastqCleaner" id="SangerSeq" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="https://www.bioconductor.org/packages/release/BiocViews.html#"><ins class="jstree-icon">&nbsp;</ins>SangerSeq (1)</a></li><li packagelist="srnadiff" id="SmallRNA" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="https://www.bioconductor.org/packages/release/BiocViews.html#"><ins class="jstree-icon">&nbsp;</ins>SmallRNA (1)</a></li><li packagelist="Clomial,CopywriteR,LymphoSeq,MDTS,NGScopy,TarSeqQC" id="TargetedResequencing" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="https://www.bioconductor.org/packages/release/BiocViews.html#"><ins class="jstree-icon">&nbsp;</ins>TargetedResequencing (6)</a></li><li packagelist="ACE,AneuFinder,BadRegionFinder,CNAnorm,contiBAIT,DChIPRep,DECIPHER,DEComplexDisease,DMRScan,dmrseq,GeneBreak,GenoGAM,globalSeq,iCNV,LowMACA,MEAL,methInheritSim,methylInheritance,NGScopy,podkat,RareVariantVis,rGREAT,RVS,samExploreR,sscu,TitanCNA,TVTB,Uniquorn" id="WholeGenome" class="jstree-last jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="https://www.bioconductor.org/packages/release/BiocViews.html#"><ins class="jstree-icon">&nbsp;</ins>WholeGenome (28)</a></li></ul></li><li packagelist="AneuFinder,AUCell,BASiCS,bayNorm,BEARscc,BPRMeth,breakpointR,CancerInSilico,CATALYST,ccfindR,celaref,CellTrails,chromVAR,cicero,clusterExperiment,condcomp,countsimQC,cydar,DEsingle,diffcyt,DropletUtils,FCBF,HDF5Array,IrisSpatialFeatures,iSEE,LineagePulse,Linnorm,LoomExperiment,MAST,MetaNeighbor,mfa,netSmooth,OMICsPCA,phenopath,restfulSE,RGMQL,SC3,scater,scDD,scFeatureFilter,scfind,scmap,scmeth,SCnorm,scone,Sconify,scPipe,scran,scruff,SIMD,SIMLR,SingleCellExperiment,singleCellTK,slalom,slingshot,splatter,switchde,tenXplore,TRONCO,zinbwave" id="SingleCell" class="jstree-last jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="https://www.bioconductor.org/packages/release/BiocViews.html#"><ins class="jstree-icon">&nbsp;</ins>SingleCell (60)</a></li></ul></li><li packagelist="anamiR,AUCell,ChIPanalyser,DRIMSeq,geecc,GUIDEseq,iGC,metaseqR,MetID,nondetects,omicRexposome,pgca,phosphonormalizer,plyranges,rDGIdb,RNAdecay,scruff,TCGAutils,tRNAscanImport" id="WorkflowStep" class="jstree-last jstree-closed"><ins class="jstree-icon">&nbsp;</ins><a href="https://www.bioconductor.org/packages/release/BiocViews.html#"><ins class="jstree-icon">&nbsp;</ins>WorkflowStep (884)</a><ul><li packagelist="amplican,ASpli,BadRegionFinder,bamsignals,Basic4Cseq,BBCAnalyzer,bcSeq,Biostrings,chimeraviz,ChIPanalyser,ChIPexoQual,DECIPHER,diffHic,DiffLogo,DominoEffect,esATAC,GenomicAlignments,ggtree,gmapR,GreyListChIP,Guitar,Imetagene,IMMAN,IntEREst,Logolas,LowMACA,LymphoSeq,metagene,MethTargetedNGS,motifStack,msa,muscle,normr,nucleoSim,odseq,ORFik,OUTRIDER,QSutils,QuasR,R4RNA,Rbowtie,Rbowtie2,REMP,RiboProfiling,RNASeqR,Rsamtools,Rsubread,samExploreR,scruff,segmentSeq,seqcombo,SICtools,singleCellTK,strandCheckR,systemPipeR,TarSeqQC,TFBSTools,traseR,treeio,TVTB" id="Alignment" class="jstree-closed"><ins class="jstree-icon">&nbsp;</ins><a href="https://www.bioconductor.org/packages/release/BiocViews.html#"><ins class="jstree-icon">&nbsp;</ins>Alignment (60)</a><ul><li packagelist="bcSeq,ggtree,LowMACA,LymphoSeq,msa,muscle,odseq,R4RNA,treeio" id="MultipleSequenceAlignment" class="jstree-last jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="https://www.bioconductor.org/packages/release/BiocViews.html#"><ins class="jstree-icon">&nbsp;</ins>MultipleSequenceAlignment (9)</a></li></ul></li><li packagelist="AffyExpress,altcdfenvs,annaffy,annmap,annotate,AnnotationDbi,AnnotationFilter,AnnotationForge,annotationTools,annotatr,ArrayTools,artMS,biomaRt,BridgeDbR,BSgenome,CAGEfightR,Category,categoryCompare,cellbaseR,ChIPpeakAnno,ChIPseeker,chromPlot,clusterProfiler,crossmeta,csaw,DeepBlueR,DelayedArray,DMRforPairs,DOSE,ecolitk,enrichplot,ensemblVEP,ENVISIONQuery,FGNet,FunciSNP,gaggle,garfield,gCMAP,geneXtendeR,genomation,GenomeInfoDb,genomes,GenomicFeatures,GenomicRanges,GenomicScores,ggtree,GOexpress,GOpro,goProfiles,GOSemSim,GOstats,HDF5Array,HelloRanges,hiAnnotator,iBBiG,IdMappingAnalysis,IdMappingRetrieval,intansv,IsoformSwitchAnalyzeR,KEGGREST,matchBox,mdgsa,MeSHDbi,meshes,meshr,metagenomeFeatures,miRNApath,mygene,myvariant,occugene,OmaDB,OmicCircos,Onassis,Organism.dplyr,OrganismDbi,PCAN,pdInfoBuilder,podkat,RBioinf,rDGIdb,ReactomePA,rfPred,Risa,RNAither,rnaSeqMap,rols,rtracklayer,SemDist,sevenC,simpleaffy,SplicingGraphs,SummarizedExperiment,SWATH2stats,treeio,tximeta,Ularcirc,UniProt.ws,VariantAnnotation,VariantFiltering,xmapbridge,yarn" id="Annotation" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="https://www.bioconductor.org/packages/release/BiocViews.html#"><ins class="jstree-icon">&nbsp;</ins>Annotation (101)</a></li><li packagelist="affylmGUI,alpine,anota2seq,bacon,BatchQC,BDMMAcorrect,BEclear,BUScorrect,crossmeta,edgeR,ENmix,erccdashboard,gcapc,genotypeeval,Harman,iasva,IsoformSwitchAnalyzeR,limma,limmaGUI,Linnorm,MAGeCKFlute,MethylAid,pvca,ramwas,RnBeads,scran,sights,statTarget,sva,SVAPLSseq,variancePartition" id="BatchEffect" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="https://www.bioconductor.org/packages/release/BiocViews.html#"><ins class="jstree-icon">&nbsp;</ins>BatchEffect (31)</a></li><li packagelist="ASAFE,countsimQC,EDDA,gCrisprTools,kissDE,OSAT,regsplice,rexposome,RNASeqR,RnaSeqSampleSize,SWATH2stats,XBSeq" id="ExperimentalDesign" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="https://www.bioconductor.org/packages/release/BiocViews.html#"><ins class="jstree-icon">&nbsp;</ins>ExperimentalDesign (12)</a></li><li packagelist="CAGEfightR,igvR" id="GenomeBrowsers" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="https://www.bioconductor.org/packages/release/BiocViews.html#"><ins class="jstree-icon">&nbsp;</ins>GenomeBrowsers (2)</a></li><li packagelist="abseqR,adaptest,affylmGUI,bacon,baySeq,BGmix,biobroom,bioCancer,bumphunter,categoryCompare,CellScore,ChIPComp,ChIPseeker,chromstaR,chromswitch,clippda,Clomial,clusterProfiler,clusterSeq,cnvGSA,cobindR,CoGAPS,CompGO,consensusDE,consensusSeekeR,csaw,cydar,DEqMS,diffHic,DiffLogo,dks,DMRcate,dmrseq,DOSE,DRIMSeq,EBSeq,EBSeqHMM,edge,edgeR,EGSEA,erccdashboard,ERSSA,fabia,fdrame,flowMap,gaga,gage,gCrisprTools,GeneOverlap,GEWIST,globalSeq,GOexpress,GOpro,goProfiles,GOstats,Harman,HTqPCR,HTSanalyzeR,IdMappingAnalysis,IdMappingRetrieval,iGC,IHW,Imetagene,InterMineR,IsoformSwitchAnalyzeR,ISoLDE,IWTomics,LBE,limma,limmaGUI,made4,matchBox,MBttest,mCSEA,meshes,metagene,metagenomeSeq,MetaNeighbor,MineICA,mixOmics,MmPalateMiRNA,monocle,msa,Mulcom,MultiMed,multtest,NormalyzerDE,normr,OCplus,omicade4,omicRexposome,OMICsPCA,onlineFDR,OPWeight,OrderedList,parody,PathNet,pbcmc,phenoTest,phyloseq,plotGrouper,PowerExplorer,Prize,PROMISE,prot2D,psichomics,quantro,QUBIC,R4RNA,rain,RBioinf,ReactomePA,rexposome,SAGx,scDD,Sconify,segmentSeq,semisup,SIAMCAT,siggenes,sights,sigPathway,similaRpeak,snm,SpeCond,ssviz,staRank,statTarget,sva,swfdr,TransView,TTMap,TVTB,twilight,unifiedWMWqPCR" id="MultipleComparison" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="https://www.bioconductor.org/packages/release/BiocViews.html#"><ins class="jstree-icon">&nbsp;</ins>MultipleComparison (135)</a></li><li packagelist="ACME,affylmGUI,alpine,anamiR,anota2seq,artMS,AUCell,BASiCS,bayNorm,blima,bnbc,CAGEfightR,CAGEr,Cardinal,CATALYST,cellity,ChAMP,ChIPSeqSpike,CNAnorm,CNVPanelizer,CODEX,coexnet,conumee,csaw,DAPAR,DEqMS,DESeq2,DEsubs,diffHic,Doscheda,EDDA,edgeR,ENmix,funtooNorm,gCrisprTools,Harman,HiCcompare,HTSFilter,IntEREst,limma,limmaGUI,Linnorm,MAGeCKFlute,metagenomeSeq,metaseqR,minfi,missMethyl,MSstats,multiHiCcompare,NanoStringDiff,NanoStringQCPro,netSmooth,normalize450K,NormalyzerDE,normr,pbcmc,phantasus,phosphonormalizer,Pigengene,Prostar,ProteoMM,qPLEXanalyzer,qsea,quantro,ramwas,REMP,RNAdecay,RNAprobR,RNASeqR,Rnits,RUVcorr,RUVnormalize,scater,SCBN,SCnorm,scone,scran,sigFeature,sights,slalom,statTarget,sva,SVAPLSseq,TimeSeriesExperiment,TurboNorm,variancePartition,Wrench,yarn" id="Normalization" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="https://www.bioconductor.org/packages/release/BiocViews.html#"><ins class="jstree-icon">&nbsp;</ins>Normalization (88)</a></li><li packagelist="anamiR,annaffy,annotate,ASSIGN,attract,bioCancer,BioCor,BioNetStat,BLMA,Category,categoryCompare,CEMiTool,CHRONOS,clusterProfiler,cogena,coRdon,cpvSNP,CTDquerier,cTRAP,DART,DEsubs,DOSE,edgeR,EGSEA,EmpiricalBrownsMethod,EnrichmentBrowser,enrichplot,epiNEM,esetVis,fastLiquidAssociation,FELLA,FGNet,fgsea,gage,gCMAP,GeneAccord,gep2pep,GlobalAncova,globaltest,GOexpress,GOFunction,goProfiles,GOSemSim,GOSim,GOstats,graphite,GRridge,GSEABenchmarkeR,gsean,GSReg,GSVA,hipathia,InterMineR,joda,KEGGgraph,KEGGlincs,keggorthology,KEGGprofile,KEGGREST,LiquidAssociation,MAGeCKFlute,mdgsa,MetaboSignal,methylGSA,MethylMix,mgsa,mirIntegrator,miRNApath,MmPalateMiRNA,MoonlightR,NCIgraph,ndexr,nem,NetPathMiner,npGSEA,occugene,oppar,Path2PPI,PathNet,pathRender,pathVar,pathview,PathwaySplice,paxtoolsr,PCAN,Pi,piano,PPInfer,qpgraph,RandomWalkRestartMH,ReactomePA,rGREAT,RNASeqR,rsbml,safe,SAGx,SBMLR,sigaR,signet,StarBioTrek,TCGAbiolinks,TCGAbiolinksGUI,TimeSeriesExperiment,ToPASeq" id="Pathways" class="jstree-closed"><ins class="jstree-icon">&nbsp;</ins><a href="https://www.bioconductor.org/packages/release/BiocViews.html#"><ins class="jstree-icon">&nbsp;</ins>Pathways (150)</a><ul><li packagelist="graphite,loci2path,RandomWalkRestartMH,UniProt.ws" id="BioCarta" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="https://www.bioconductor.org/packages/release/BiocViews.html#"><ins class="jstree-icon">&nbsp;</ins>BioCarta (4)</a></li><li packagelist="annaffy,annotate,BgeeDB,Category,categoryCompare,cellTree,ClusterJudge,clusterProfiler,cogena,CompGO,CTDquerier,DAPAR,diffloop,EGSEA,enrichplot,ExpressionView,FELLA,FGNet,FoldGO,gage,GDCRNATools,geecc,geneXtendeR,globaltest,GOexpress,GOfuncR,GOFunction,GOpro,goProfiles,GOSemSim,goseq,GOSim,goSTAG,GOstats,GOsummaries,goTools,GRridge,GSEABase,hipathia,interactiveDisplay,interactiveDisplayBase,InterMineR,mdgsa,MetaNeighbor,mgsa,MmPalateMiRNA,OmaDB,ontoProc,PathwaySplice,pcaGoPromoter,Prostar,RamiGO,RCAS,Rcpi,ReportingTools,rGREAT,RNAither,RNASeqR,rols,SemDist,SEPA,singscore,sRAP,tRanslatome,UniProt.ws" id="GO" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="https://www.bioconductor.org/packages/release/BiocViews.html#"><ins class="jstree-icon">&nbsp;</ins>GO (65)</a></li><li packagelist="attract,BPRMeth,CHRONOS,clusterProfiler,cogena,coRdon,CTDquerier,DEsubs,EGSEA,enrichplot,ExpressionView,FELLA,GDCRNATools,graphite,GRridge,GSEABase,InterMineR,KEGGgraph,KEGGlincs,keggorthology,KEGGprofile,KEGGREST,MetaboSignal,MetID,MIGSA,mirIntegrator,MLP,PathNet,pathprint,paxtoolsr,pbcmc,pcot2,RandomWalkRestartMH,RNASeqR,SAGx,signet,singscore,StarBioTrek,UniProt.ws" id="KEGG" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="https://www.bioconductor.org/packages/release/BiocViews.html#"><ins class="jstree-icon">&nbsp;</ins>KEGG (39)</a></li><li class="jstree-leaf" style="display: none;"><ins class="jstree-icon">&nbsp;</ins><a href="https://www.bioconductor.org/packages/release/BiocViews.html#"><ins class="jstree-icon">&nbsp;</ins>NCINatureCurated</a></li><li packagelist="attract,bioCancer,clusterProfiler,graphite,InterMineR,MetaboSignal,MLP,pathprint,paxtoolsr,RandomWalkRestartMH,ReactomePA,slalom" id="Reactome" class="jstree-last jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="https://www.bioconductor.org/packages/release/BiocViews.html#"><ins class="jstree-icon">&nbsp;</ins>Reactome (12)</a></li></ul></li><li packagelist="ABarray,affy,affycomp,AffyExpress,affyILM,affylmGUI,affyPara,affypdnn,affyPLM,AffyRNADegradation,AgiMicroRna,altcdfenvs,Anaquin,aroma.light,ArrayTools,ARRmNormalization,ballgown,BatchQC,beadarray,beadarraySNP,BEclear,bigmelon,biovizBase,blima,bnbc,BUS,CAGEfightR,CAGEr,CALIB,CATALYST,cellHTS2,cellity,CGHcall,CGHnormaliter,clippda,codelink,conumee,CopywriteR,covEB,cqn,crlmm,crossmeta,cytolib,DAPAR,DeepBlueR,DEGseq,DEqMS,DEsingle,diffHic,diffloop,DNABarcodes,Doscheda,DupChecker,dyebias,easyRNASeq,EDASeq,ENmix,esATAC,ExiMiR,farms,flowWorkspace,focalCall,FourCSeq,frma,frmaTools,funtooNorm,gCrisprTools,gcrma,genArise,gespeR,GOTHiC,gprege,GreyListChIP,Harman,Harshlight,HELP,hiReadsProcessor,HMMcopy,HTqPCR,HTSFilter,iasva,iBMQ,iCheck,imageHTS,IsoCorrectoR,limma,limmaGUI,LMGene,lumi,LVSmiRNA,maCorrPlot,maigesPack,makecdfenv,MANOR,marray,MaxContrastProjection,MBCB,MEDIPS,MetaCyto,metaseqR,methylumi,Mfuzz,MiChip,minfi,miRcomp,miRNAmeConverter,mitoODE,MmPalateMiRNA,multiscan,NanoStringQCPro,netprioR,netSmooth,nnNorm,nondetects,normalize450K,normr,NTW,oligo,OLIN,OLINgui,openCyto,PepsNMR,pepStat,piano,prebs,primirTSS,proFIA,Prostar,puma,QDNAseq,qpcrNorm,qPLEXanalyzer,qrqc,qsea,quantro,QuasR,r3Cseq,R4RNA,rama,ramwas,RBioinf,Rbowtie2,rCGH,recoup,REDseq,RefPlus,REMP,ReQON,Ringo,rMAT,RNAinteract,RNAither,RnBeads,RPA,Rsubread,RTCGA,RUVSeq,SAGx,samExploreR,SCAN.UPC,scater,scFeatureFilter,scmap,scmeth,scone,scPipe,scruff,scsR,sesame,shinyMethyl,SIAMCAT,sights,simpleaffy,skewr,snapCGH,snm,spotSegmentation,sRAP,srnadiff,Starr,statTarget,stepNorm,sva,SVM2CRM,SWATH2stats,synlet,TargetSearch,TCGAutils,tilingArray,tRNAscanImport,TSSi,TurboNorm,tximeta,variancePartition,vsn,wateRmelon,xps,yarn" id="Preprocessing" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="https://www.bioconductor.org/packages/release/BiocViews.html#"><ins class="jstree-icon">&nbsp;</ins>Preprocessing (198)</a></li><li packagelist="abseqR,AffyExpress,affylmGUI,affyPLM,affyQCReport,AffyRNADegradation,alpine,altcdfenvs,arrayMvout,arrayQuality,arrayQualityMetrics,ArrayTools,artMS,AssessORF,ATACseqQC,basecallQC,Basic4Cseq,BatchQC,beadarray,bigmelon,BioQC,cellity,ChIC,ChIPexoQual,ChIPQC,chipseq,clstutils,CODEX,CONFESS,consensus,contiBAIT,conumee,countsimQC,CSSP,cummeRbund,DAPAR,DECIPHER,dexus,diffloop,dks,DMRcate,doppelgangR,Doscheda,dupRadar,dyebias,EDASeq,edgeR,ENmix,erccdashboard,ERSSA,esATAC,farms,FastqCleaner,ffpe,flowAI,flowClean,flowQB,gCrisprTools,GENESIS,Guitar,GWASTools,Harshlight,HELP,hicrep,HTqPCR,htSeqTools,iasva,ibh,iCheck,ideal,interactiveDisplay,interactiveDisplayBase,IONiseR,isobar,limma,limmaGUI,lumi,MAGeCKFlute,MANOR,manta,massiR,mdp,mdqc,MEDIPS,metaseqR,MethylAid,methylumi,minfi,miRcomp,MmPalateMiRNA,monocle,MSnbase,MSstats,MSstatsQC,MSstatsQCgui,MWASTools,NanoStringQCPro,nucleR,OLIN,OLINgui,omicsPrint,OSAT,pbcmc,pcaExplorer,piano,pRoloc,proteoQC,pvac,qcmetrics,QDNAseq,qPLEXanalyzer,qrqc,qsea,QuasR,R453Plus1Toolbox,rama,ramwas,RBioinf,recoup,REMP,ReQON,RiboProfiling,Ringo,RITAN,RNAinteract,RNAither,rnaseqcomp,RNASeqR,RnBeads,RPA,Rqc,Rsamtools,Rsubread,Scale4C,scater,scmeth,scone,scPipe,scruff,segmentSeq,seqTools,sesame,shinyMethyl,ShortRead,SICtools,sights,simpleaffy,skewr,SNAGEE,snm,spikeLI,spotSegmentation,sRAP,Starr,statTarget,strandCheckR,synapter,synlet,systemPipeR,TarSeqQC,TEQC,tRanslatome,traseR,unifiedWMWqPCR,variancePartition,wateRmelon,yaqcaffy,yarn" id="QualityControl" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="https://www.bioconductor.org/packages/release/BiocViews.html#"><ins class="jstree-icon">&nbsp;</ins>QualityControl (168)</a></li><li packagelist="abseqR,affycoretools,AffyExpress,ampliQueso,annaffy,annmap,arrayQualityMetrics,ArrayTools,BiocWorkflowTools,CellScore,ChIPQC,countsimQC,DEGreport,DMRforPairs,DynDoc,EnrichmentBrowser,Glimma,GSEABenchmarkeR,Harshlight,ideal,metaseqR,MmPalateMiRNA,NanoStringQCPro,pcaExplorer,PGA,proteoQC,qcmetrics,R453Plus1Toolbox,regionReport,rnaSeqMap,sapFinder,simpleaffy,SpeCond,tracktables,xmapbridge,yaqcaffy" id="ReportWriting" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="https://www.bioconductor.org/packages/release/BiocViews.html#"><ins class="jstree-icon">&nbsp;</ins>ReportWriting (36)</a></li><li packagelist="abseqR,ACE,AffyExpress,alpine,ampliQueso,AnalysisPageServer,annmap,annotatr,aroma.light,arrayQuality,ArrayTools,AssessORF,Basic4Cseq,BatchQC,BiGGR,bioassayR,bioCancer,biocGraph,BioMVCClass,biomvRCNS,biosvd,biovizBase,BrowserViz,CAGEfightR,CAGEr,cancerclass,CancerSubtypes,caOmicsV,CATALYST,cellGrowth,cellHTS2,cellity,cellscape,CellScore,cellTree,CGHcall,CGHregions,ChemmineOB,ChemmineR,ChIPexoQual,ChIPseeker,chroGPS,ChromHeatMap,chromPlot,clstutils,clustComp,clusterProfiler,CNEr,cogena,coMET,compEpiTools,CompGO,ComplexHeatmap,condcomp,CONFESS,copa,copynumber,CopywriteR,coRdon,CoRegNet,CountClust,countsimQC,CoverageView,CrispRVariants,ctc,cummeRbund,dagLogo,DECIPHER,DEGreport,deltaGseg,derfinderPlot,destiny,DEXSeq,DiffLogo,diffloop,Director,DMRforPairs,DOSE,drawProteins,dSimer,EBImage,ecolitk,eiR,eisa,EnrichedHeatmap,EnrichmentBrowser,enrichplot,epivizr,epivizrChart,epivizrData,epivizrServer,epivizrStandalone,esetVis,explorase,ExpressionView,fabia,facopy,FGNet,flowcatchR,flowCHIC,flowClust,flowCyBar,flowFP,flowPloidy,flowPlots,FlowSOM,flowTime,flowViz,fmcsR,focalCall,gaggle,gCMAPWeb,gCrisprTools,GDCRNATools,GeneAnswers,GeneBreak,genefu,GeneOverlap,geneplotter,GeneRegionScan,geneXtendeR,genomation,GenomeGraphs,gespeR,ggbio,ggcyto,ggtree,Glimma,GOexpress,goSTAG,GOsummaries,goTools,GRmetrics,GSCA,GSEABenchmarkeR,gtrellis,Gviz,hapFabia,heatmaps,Heatplus,HELP,HilbertCurve,HilbertVis,HilbertVisGUI,HMMcopy,HPAanalyze,HTqPCR,hyperdraw,ideal,IdeoViz,idiogram,igvR,imageHTS,interactiveDisplay,interactiveDisplayBase,InterMineR,iSEE,IsoformSwitchAnalyzeR,ivygapSE,karyoploteR,KCsmart,KEGGgraph,keggorthology,kimod,lmdme,Logolas,macat,maCorrPlot,maftools,MAGeCKFlute,manta,mapscape,maser,MassArray,matchBox,MaxContrastProjection,mcaGUI,MEDIPS,MergeMaid,metagenomeSeq,metavizr,MetCirc,methyAnalysis,MethylAid,Mfuzz,MIGSA,mimager,MineICA,missRows,monocle,motifbreakR,motifStack,MSGFgui,multiOmicsViz,MVCClass,NarrowPeaks,netbiov,NetSAM,NOISeq,NormalyzerDE,NuPoP,OLIN,OLINgui,OmicCircos,omicplotR,omicRexposome,OMICsPCA,oposSOM,OSAT,PANR,PanVizGenerator,PathoStat,pathRender,pathview,PathwaySplice,Pbase,pcaExplorer,pcaGoPromoter,PepsNMR,phantasus,piano,PICS,PING,prada,proBAMr,pRolocGUI,proteoQC,psichomics,Pviz,qcmetrics,qrqc,qsea,quantsmooth,QUBIC,R453Plus1Toolbox,R4RNA,RamiGO,ramwas,Rariant,RCASPAR,rcellminer,Rchemcpp,RCy3,RCyjs,RDAVIDWebService,RDRToolbox,ReactomePA,reb,RedeR,regionReport,ReportingTools,RGraph2js,Rgraphviz,riboSeqR,RmiR,RNAinteract,RNAither,rnaseqcomp,rnaSeqMap,RNASeqR,RTCA,rtracklayer,rWikiPathways,sangerseqR,sapFinder,SC3,Scale4C,scater,scDD,scmap,Sconify,scPipe,scran,scruff,SEPA,seqcombo,seqPattern,seqplots,seqsetvis,SigFuge,sights,signeR,SIM,simpleaffy,sincell,slalom,slingshot,SNPchip,SomaticSignatures,sparsenetgls,SplicingGraphs,SPLINTER,splots,SQUADD,sRAP,ssviz,statTarget,supraHex,survcomp,Sushi,SwimR,synlet,TarSeqQC,TCseq,tilingArray,timescape,TimeSeriesExperiment,TnT,ToPASeq,topGO,trackViewer,transcriptogramer,TransView,tRNA,tRNAdbImport,tRNAscanImport,TSCAN,TVTB,Ularcirc,unifiedWMWqPCR,vidger,wiggleplotr,xmapbridge,YAPSA,yarn" id="Visualization" class="jstree-last jstree-closed"><ins class="jstree-icon">&nbsp;</ins><a href="https://www.bioconductor.org/packages/release/BiocViews.html#"><ins class="jstree-icon">&nbsp;</ins>Visualization (419)</a><ul><li packagelist="AMOUNTAIN,ANF,BayesKnockdown,BiGGR,bioCancer,BioCor,BioNet,BioNetStat,BiRewire,birte,caOmicsV,CausalR,CEMiTool,CNORfuzzy,coexnet,CoRegNet,COSNet,CTDquerier,DEGraph,DeMAND,DEsubs,diffuStats,DriverNet,dSimer,EGAD,EGSEA,ELMER,EnrichmentBrowser,epiNEM,FELLA,FGNet,GDCRNATools,GENIE3,glmSparseNet,GOSemSim,GraphAlignment,GraphAT,graphite,GRENITS,GSEABenchmarkeR,gsean,IMMAN,interactiveDisplay,interactiveDisplayBase,KEGGlincs,KinSwingR,levi,Linnorm,LiquidAssociation,martini,MetaboSignal,MethylMix,MetID,MetNet,minet,mirIntegrator,miRLAB,MODA,MoonlightR,ndexr,netbenchmark,netbiov,NetPathMiner,netprioR,netReg,netresponse,NetSAM,netSmooth,networkBMA,Onassis,pandaR,Path2PPI,pathifier,PathNet,PathwaySplice,paxtoolsr,PCAN,pcot2,PCpheno,perturbatr,Pi,Pigengene,PPInfer,ppiStats,QUBIC,R3CPET,RandomWalkRestartMH,RBGL,RCy3,RedeR,Rgin,RGMQL,RGraph2js,RITAN,rsbml,rTRM,rTRMui,rWikiPathways,SANTA,SBMLR,SEPIRA,SLGI,SMITE,SPEM,SpidermiR,SQUADD,StarBioTrek,STRINGdb,TCGAbiolinks,TCGAbiolinksGUI,transcriptogramer,TRONCO,TxRegInfra,vtpnet,XINA" id="Network" class="jstree-last jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="https://www.bioconductor.org/packages/release/BiocViews.html#"><ins class="jstree-icon">&nbsp;</ins>Network (115)</a></li></ul></li></ul></li></ul></li><li packagelist="adme16cod.db,ag.db,agcdf,agprobe,AHCytoBands,AHEnsDbs,alternativeSplicingEvents.hg19,alternativeSplicingEvents.hg38,anopheles.db0,arabidopsis.db0,ath1121501.db,ath1121501cdf,ath1121501probe,barley1cdf,barley1probe,bovine.db,bovine.db0,bovinecdf,bovineprobe,BSgenome.Alyrata.JGI.v1,BSgenome.Amellifera.BeeBase.assembly4,BSgenome.Amellifera.UCSC.apiMel2,BSgenome.Amellifera.UCSC.apiMel2.masked,BSgenome.Aofficinalis.NCBI.V1,BSgenome.Athaliana.TAIR.04232008,BSgenome.Athaliana.TAIR.TAIR9,BSgenome.Btaurus.UCSC.bosTau3,BSgenome.Btaurus.UCSC.bosTau3.masked,BSgenome.Btaurus.UCSC.bosTau4,BSgenome.Btaurus.UCSC.bosTau4.masked,BSgenome.Btaurus.UCSC.bosTau6,BSgenome.Btaurus.UCSC.bosTau6.masked,BSgenome.Btaurus.UCSC.bosTau8,BSgenome.Carietinum.NCBI.v1,BSgenome.Celegans.UCSC.ce10,BSgenome.Celegans.UCSC.ce11,BSgenome.Celegans.UCSC.ce2,BSgenome.Celegans.UCSC.ce6,BSgenome.Cfamiliaris.UCSC.canFam2,BSgenome.Cfamiliaris.UCSC.canFam2.masked,BSgenome.Cfamiliaris.UCSC.canFam3,BSgenome.Cfamiliaris.UCSC.canFam3.masked,BSgenome.Dmelanogaster.UCSC.dm2,BSgenome.Dmelanogaster.UCSC.dm2.masked,BSgenome.Dmelanogaster.UCSC.dm3,BSgenome.Dmelanogaster.UCSC.dm3.masked,BSgenome.Dmelanogaster.UCSC.dm6,BSgenome.Drerio.UCSC.danRer10,BSgenome.Drerio.UCSC.danRer5,BSgenome.Drerio.UCSC.danRer5.masked,BSgenome.Drerio.UCSC.danRer6,BSgenome.Drerio.UCSC.danRer6.masked,BSgenome.Drerio.UCSC.danRer7,BSgenome.Drerio.UCSC.danRer7.masked,BSgenome.Ecoli.NCBI.20080805,BSgenome.Gaculeatus.UCSC.gasAcu1,BSgenome.Gaculeatus.UCSC.gasAcu1.masked,BSgenome.Ggallus.UCSC.galGal3,BSgenome.Ggallus.UCSC.galGal3.masked,BSgenome.Ggallus.UCSC.galGal4,BSgenome.Ggallus.UCSC.galGal4.masked,BSgenome.Ggallus.UCSC.galGal5,BSgenome.Hsapiens.1000genomes.hs37d5,BSgenome.Hsapiens.NCBI.GRCh38,BSgenome.Hsapiens.UCSC.hg17,BSgenome.Hsapiens.UCSC.hg17.masked,BSgenome.Hsapiens.UCSC.hg18,BSgenome.Hsapiens.UCSC.hg18.masked,BSgenome.Hsapiens.UCSC.hg19,BSgenome.Hsapiens.UCSC.hg19.masked,BSgenome.Hsapiens.UCSC.hg38,BSgenome.Hsapiens.UCSC.hg38.masked,BSgenome.Mfascicularis.NCBI.5.0,BSgenome.Mfuro.UCSC.musFur1,BSgenome.Mmulatta.UCSC.rheMac2,BSgenome.Mmulatta.UCSC.rheMac2.masked,BSgenome.Mmulatta.UCSC.rheMac3,BSgenome.Mmulatta.UCSC.rheMac3.masked,BSgenome.Mmulatta.UCSC.rheMac8,BSgenome.Mmusculus.UCSC.mm10,BSgenome.Mmusculus.UCSC.mm10.masked,BSgenome.Mmusculus.UCSC.mm8,BSgenome.Mmusculus.UCSC.mm8.masked,BSgenome.Mmusculus.UCSC.mm9,BSgenome.Mmusculus.UCSC.mm9.masked,BSgenome.Osativa.MSU.MSU7,BSgenome.Ptroglodytes.UCSC.panTro2,BSgenome.Ptroglodytes.UCSC.panTro2.masked,BSgenome.Ptroglodytes.UCSC.panTro3,BSgenome.Ptroglodytes.UCSC.panTro3.masked,BSgenome.Ptroglodytes.UCSC.panTro5,BSgenome.Ptroglodytes.UCSC.panTro6,BSgenome.Rnorvegicus.UCSC.rn4,BSgenome.Rnorvegicus.UCSC.rn4.masked,BSgenome.Rnorvegicus.UCSC.rn5,BSgenome.Rnorvegicus.UCSC.rn5.masked,BSgenome.Rnorvegicus.UCSC.rn6,BSgenome.Scerevisiae.UCSC.sacCer1,BSgenome.Scerevisiae.UCSC.sacCer2,BSgenome.Scerevisiae.UCSC.sacCer3,BSgenome.Sscrofa.UCSC.susScr11,BSgenome.Sscrofa.UCSC.susScr3,BSgenome.Sscrofa.UCSC.susScr3.masked,BSgenome.Tgondii.ToxoDB.7.0,BSgenome.Tguttata.UCSC.taeGut1,BSgenome.Tguttata.UCSC.taeGut1.masked,BSgenome.Tguttata.UCSC.taeGut2,BSgenome.Vvinifera.URGI.IGGP12Xv0,BSgenome.Vvinifera.URGI.IGGP12Xv2,BSgenome.Vvinifera.URGI.IGGP8X,bsubtiliscdf,bsubtilisprobe,canine.db,canine.db0,canine2.db,canine2cdf,canine2probe,caninecdf,canineprobe,celegans.db,celeganscdf,celegansprobe,ChemmineDrugs,chicken.db,chicken.db0,chickencdf,chickenprobe,chimp.db0,citruscdf,citrusprobe,clariomdhumanprobeset.db,clariomdhumantranscriptcluster.db,clariomshumanhttranscriptcluster.db,clariomshumantranscriptcluster.db,clariomsmousehttranscriptcluster.db,clariomsmousetranscriptcluster.db,clariomsrathttranscriptcluster.db,clariomsrattranscriptcluster.db,cMAP,cottoncdf,cottonprobe,cyp450cdf,DO.db,drosgenome1.db,drosgenome1cdf,drosgenome1probe,drosophila2.db,drosophila2cdf,drosophila2probe,ecoli2.db,ecoli2cdf,ecoli2probe,ecoliasv2cdf,ecoliasv2probe,ecolicdf,ecoliK12.db0,ecoliprobe,ecoliSakai.db0,EnsDb.Hsapiens.v75,EnsDb.Hsapiens.v79,EnsDb.Hsapiens.v86,EnsDb.Mmusculus.v75,EnsDb.Mmusculus.v79,EnsDb.Rnorvegicus.v75,EnsDb.Rnorvegicus.v79,EuPathDB,FDb.FANTOM4.promoters.hg19,FDb.InfiniumMethylation.hg18,FDb.InfiniumMethylation.hg19,FDb.UCSC.snp135common.hg19,FDb.UCSC.snp137common.hg19,FDb.UCSC.tRNAs,fitCons.UCSC.hg19,fly.db0,geneplast.data.string.v91,GenomeInfoDbData,genomewidesnp5Crlmm,genomewidesnp6Crlmm,GGHumanMethCancerPanelv1.db,GO.db,gp53cdf,grasp2db,greengenes13.5MgDb,h10kcod.db,h20kcod.db,hapmap370k,hcg110.db,hcg110cdf,hcg110probe,hgfocus.db,hgfocuscdf,hgfocusprobe,hgu133a.db,hgu133a2.db,hgu133a2cdf,hgu133a2frmavecs,hgu133a2probe,hgu133acdf,hgu133afrmavecs,hgu133aprobe,hgu133atagcdf,hgu133atagprobe,hgu133b.db,hgu133bcdf,hgu133bprobe,hgu133plus2.db,hgu133plus2cdf,hgu133plus2frmavecs,hgu133plus2probe,hgu219.db,hgu219cdf,hgu219probe,hgu95a.db,hgu95acdf,hgu95aprobe,hgu95av2,hgu95av2.db,hgu95av2cdf,hgu95av2probe,hgu95b.db,hgu95bcdf,hgu95bprobe,hgu95c.db,hgu95ccdf,hgu95cprobe,hgu95d.db,hgu95dcdf,hgu95dprobe,hgu95e.db,hgu95ecdf,hgu95eprobe,hguatlas13k.db,hgubeta7.db,hguDKFZ31.db,hgug4100a.db,hgug4101a.db,hgug4110b.db,hgug4111a.db,hgug4112a.db,hgug4845a.db,hguqiagenv3.db,hi16cod.db,hivprtplus2cdf,hom.At.inp.db,hom.Ce.inp.db,hom.Dm.inp.db,hom.Dr.inp.db,hom.Hs.inp.db,hom.Mm.inp.db,hom.Rn.inp.db,hom.Sc.inp.db,Homo.sapiens,hs25kresogen.db,Hs6UG171.db,HsAgilentDesign026652.db,Hspec,hspeccdf,hta20probeset.db,hta20transcriptcluster.db,hthgu133a.db,hthgu133acdf,hthgu133afrmavecs,hthgu133aprobe,hthgu133b.db,hthgu133bcdf,hthgu133bprobe,hthgu133pluspmcdf,hthgu133pluspmprobe,htmg430acdf,htmg430aprobe,htmg430bcdf,htmg430bprobe,htmg430pmcdf,htmg430pmprobe,htrat230pmcdf,htrat230pmprobe,htratfocuscdf,htratfocusprobe,hu35ksuba.db,hu35ksubacdf,hu35ksubaprobe,hu35ksubb.db,hu35ksubbcdf,hu35ksubbprobe,hu35ksubc.db,hu35ksubccdf,hu35ksubcprobe,hu35ksubd.db,hu35ksubdcdf,hu35ksubdprobe,hu6800.db,hu6800cdf,hu6800probe,hu6800subacdf,hu6800subbcdf,hu6800subccdf,hu6800subdcdf,huex.1.0.st.v2frmavecs,huex10stprobeset.db,huex10sttranscriptcluster.db,HuExExonProbesetLocation,HuExExonProbesetLocationHg18,HuExExonProbesetLocationHg19,hugene.1.0.st.v1frmavecs,hugene10stprobeset.db,hugene10sttranscriptcluster.db,hugene10stv1cdf,hugene10stv1probe,hugene11stprobeset.db,hugene11sttranscriptcluster.db,hugene20stprobeset.db,hugene20sttranscriptcluster.db,hugene21stprobeset.db,hugene21sttranscriptcluster.db,human.db0,human1mduov3bCrlmm,human1mv1cCrlmm,human370quadv3cCrlmm,human370v1cCrlmm,human550v3bCrlmm,human610quadv1bCrlmm,human650v3aCrlmm,human660quadv1aCrlmm,humanCHRLOC,humancytosnp12v2p1hCrlmm,humanomni1quadv1bCrlmm,humanomni25quadv1bCrlmm,humanomni5quadv1bCrlmm,humanomniexpress12v1bCrlmm,HuO22.db,hwgcod.db,IlluminaHumanMethylation27k.db,IlluminaHumanMethylation27kanno.ilmn12.hg19,IlluminaHumanMethylation27kmanifest,IlluminaHumanMethylation450kanno.ilmn12.hg19,IlluminaHumanMethylation450kmanifest,IlluminaHumanMethylation450kprobe,IlluminaHumanMethylationEPICanno.ilm10b2.hg19,IlluminaHumanMethylationEPICanno.ilm10b3.hg19,IlluminaHumanMethylationEPICanno.ilm10b4.hg19,IlluminaHumanMethylationEPICmanifest,illuminaHumanv1.db,illuminaHumanv2.db,illuminaHumanv2BeadID.db,illuminaHumanv3.db,illuminaHumanv4.db,illuminaHumanWGDASLv3.db,illuminaHumanWGDASLv4.db,illuminaMousev1.db,illuminaMousev1p1.db,illuminaMousev2.db,illuminaRatv1.db,indac.db,JASPAR2018,JazaeriMetaData.db,KEGG.db,LAPOINTE.db,LowMACAAnnotation,LRBase.Ath.eg.db,LRBase.Bta.eg.db,LRBase.Cel.eg.db,LRBase.Dme.eg.db,LRBase.Dre.eg.db,LRBase.Gga.eg.db,LRBase.Hsa.eg.db,LRBase.Mmu.eg.db,LRBase.Pab.eg.db,LRBase.Rno.eg.db,LRBase.Ssc.eg.db,LRBase.Xtr.eg.db,lumiHumanAll.db,lumiHumanIDMapping,lumiMouseAll.db,lumiMouseIDMapping,lumiRatAll.db,lumiRatIDMapping,LymphoSeqDB,m10kcod.db,m20kcod.db,MafDb.1Kgenomes.phase1.GRCh38,MafDb.1Kgenomes.phase1.hs37d5,MafDb.1Kgenomes.phase3.GRCh38,MafDb.1Kgenomes.phase3.hs37d5,MafDb.ESP6500SI.V2.SSA137.GRCh38,MafDb.ESP6500SI.V2.SSA137.hs37d5,MafDb.ExAC.r1.0.GRCh38,MafDb.ExAC.r1.0.hs37d5,MafDb.ExAC.r1.0.nonTCGA.GRCh38,MafDb.ExAC.r1.0.nonTCGA.hs37d5,MafDb.gnomAD.r2.0.1.GRCh38,MafDb.gnomAD.r2.0.1.hs37d5,MafDb.gnomAD.r2.1.hs37d5,MafDb.gnomADex.r2.0.1.GRCh38,MafDb.gnomADex.r2.0.1.hs37d5,MafDb.gnomADex.r2.1.hs37d5,MafDb.TOPMed.freeze5.hg19,MafDb.TOPMed.freeze5.hg38,maizecdf,maizeprobe,malaria.db0,medicagocdf,medicagoprobe,MeSH.Aca.eg.db,MeSH.Aga.PEST.eg.db,MeSH.Ame.eg.db,MeSH.Aml.eg.db,MeSH.Ana.eg.db,MeSH.Ani.FGSC.eg.db,MeSH.AOR.db,MeSH.Ath.eg.db,MeSH.Bfl.eg.db,MeSH.Bsu.168.eg.db,MeSH.Bta.eg.db,MeSH.Cal.SC5314.eg.db,MeSH.Cbr.eg.db,MeSH.Cel.eg.db,MeSH.Cfa.eg.db,MeSH.Cin.eg.db,MeSH.Cja.eg.db,MeSH.Cpo.eg.db,MeSH.Cre.eg.db,MeSH.Dan.eg.db,MeSH.db,MeSH.Dda.3937.eg.db,MeSH.Ddi.AX4.eg.db,MeSH.Der.eg.db,MeSH.Dgr.eg.db,MeSH.Dme.eg.db,MeSH.Dmo.eg.db,MeSH.Dpe.eg.db,MeSH.Dre.eg.db,MeSH.Dse.eg.db,MeSH.Dsi.eg.db,MeSH.Dvi.eg.db,MeSH.Dya.eg.db,MeSH.Eco.55989.eg.db,MeSH.Eco.ED1a.eg.db,MeSH.Eco.IAI39.eg.db,MeSH.Eco.K12.MG1655.eg.db,MeSH.Eco.O157.H7.Sakai.eg.db,MeSH.Eco.UMN026.eg.db,MeSH.Eqc.eg.db,MeSH.Gga.eg.db,MeSH.Gma.eg.db,MeSH.Hsa.eg.db,MeSH.Laf.eg.db,MeSH.Lma.eg.db,MeSH.Mdo.eg.db,MeSH.Mes.eg.db,MeSH.Mga.eg.db,MeSH.Miy.eg.db,MeSH.Mml.eg.db,MeSH.Mmu.eg.db,MeSH.Mtr.eg.db,MeSH.Nle.eg.db,MeSH.Oan.eg.db,MeSH.Ocu.eg.db,MeSH.Oni.eg.db,MeSH.Osa.eg.db,MeSH.Pab.eg.db,MeSH.Pae.PAO1.eg.db,MeSH.PCR.db,MeSH.Pfa.3D7.eg.db,MeSH.Pto.eg.db,MeSH.Ptr.eg.db,MeSH.Rno.eg.db,MeSH.Sce.S288c.eg.db,MeSH.Sco.A32.eg.db,MeSH.Sil.eg.db,MeSH.Spu.eg.db,MeSH.Ssc.eg.db,MeSH.Syn.eg.db,MeSH.Tbr.9274.eg.db,MeSH.Tgo.ME49.eg.db,MeSH.Tgu.eg.db,MeSH.Vvi.eg.db,MeSH.Xla.eg.db,MeSH.Xtr.eg.db,MeSH.Zma.eg.db,mgu74a.db,mgu74acdf,mgu74aprobe,mgu74av2.db,mgu74av2cdf,mgu74av2probe,mgu74b.db,mgu74bcdf,mgu74bprobe,mgu74bv2.db,mgu74bv2cdf,mgu74bv2probe,mgu74c.db,mgu74ccdf,mgu74cprobe,mgu74cv2.db,mgu74cv2cdf,mgu74cv2probe,mguatlas5k.db,mgug4104a.db,mgug4120a.db,mgug4121a.db,mgug4122a.db,mi16cod.db,mirbase.db,miRBaseVersions.db,mirna102xgaincdf,mirna10cdf,mirna10probe,mirna20cdf,miRNAtap.db,mm24kresogen.db,MmAgilentDesign026655.db,moe430a.db,moe430acdf,moe430aprobe,moe430b.db,moe430bcdf,moe430bprobe,moex10stprobeset.db,moex10sttranscriptcluster.db,MoExExonProbesetLocation,mogene.1.0.st.v1frmavecs,mogene10stprobeset.db,mogene10sttranscriptcluster.db,mogene10stv1cdf,mogene10stv1probe,mogene11stprobeset.db,mogene11sttranscriptcluster.db,mogene20stprobeset.db,mogene20sttranscriptcluster.db,mogene21stprobeset.db,mogene21sttranscriptcluster.db,mouse.db0,mouse4302.db,mouse4302cdf,mouse4302frmavecs,mouse4302probe,mouse430a2.db,mouse430a2cdf,mouse430a2frmavecs,mouse430a2probe,mouseCHRLOC,mpedbarray.db,mta10probeset.db,mta10transcriptcluster.db,mu11ksuba.db,mu11ksubacdf,mu11ksubaprobe,mu11ksubb.db,mu11ksubbcdf,mu11ksubbprobe,Mu15v1.db,mu19ksuba.db,mu19ksubacdf,mu19ksubb.db,mu19ksubbcdf,mu19ksubc.db,mu19ksubccdf,Mu22v3.db,mu6500subacdf,mu6500subbcdf,mu6500subccdf,mu6500subdcdf,Mus.musculus,mwgcod.db,Norway981.db,nugohs1a520180.db,nugohs1a520180cdf,nugohs1a520180probe,nugomm1a520177.db,nugomm1a520177cdf,nugomm1a520177probe,oligoData,OperonHumanV3.db,org.Ag.eg.db,org.At.tair.db,org.Bt.eg.db,org.Ce.eg.db,org.Cf.eg.db,org.Dm.eg.db,org.Dr.eg.db,org.EcK12.eg.db,org.EcSakai.eg.db,org.Gg.eg.db,org.Hs.eg.db,org.Mm.eg.db,org.Mmu.eg.db,org.Pf.plasmo.db,org.Pt.eg.db,org.Rn.eg.db,org.Sc.sgd.db,org.Ss.eg.db,org.Xl.eg.db,paeg1acdf,paeg1aprobe,PANTHER.db,PartheenMetaData.db,pd.081229.hg18.promoter.medip.hx1,pd.2006.07.18.hg18.refseq.promoter,pd.2006.07.18.mm8.refseq.promoter,pd.2006.10.31.rn34.refseq.promoter,pd.ag,pd.aragene.1.0.st,pd.aragene.1.1.st,pd.ath1.121501,pd.barley1,pd.bovgene.1.0.st,pd.bovgene.1.1.st,pd.bovine,pd.bsubtilis,pd.cangene.1.0.st,pd.cangene.1.1.st,pd.canine,pd.canine.2,pd.celegans,pd.charm.hg18.example,pd.chicken,pd.chigene.1.0.st,pd.chigene.1.1.st,pd.chogene.2.0.st,pd.chogene.2.1.st,pd.citrus,pd.clariom.d.human,pd.clariom.s.human,pd.clariom.s.human.ht,pd.clariom.s.mouse,pd.clariom.s.mouse.ht,pd.clariom.s.rat,pd.clariom.s.rat.ht,pd.cotton,pd.cyngene.1.0.st,pd.cyngene.1.1.st,pd.cyrgene.1.0.st,pd.cyrgene.1.1.st,pd.cytogenetics.array,pd.drogene.1.0.st,pd.drogene.1.1.st,pd.drosgenome1,pd.drosophila.2,pd.e.coli.2,pd.ecoli,pd.ecoli.asv2,pd.elegene.1.0.st,pd.elegene.1.1.st,pd.equgene.1.0.st,pd.equgene.1.1.st,pd.feinberg.hg18.me.hx1,pd.feinberg.mm8.me.hx1,pd.felgene.1.0.st,pd.felgene.1.1.st,pd.fingene.1.0.st,pd.fingene.1.1.st,pd.genomewidesnp.5,pd.genomewidesnp.6,pd.guigene.1.0.st,pd.guigene.1.1.st,pd.hc.g110,pd.hg.focus,pd.hg.u133.plus.2,pd.hg.u133a,pd.hg.u133a.2,pd.hg.u133a.tag,pd.hg.u133b,pd.hg.u219,pd.hg.u95a,pd.hg.u95av2,pd.hg.u95b,pd.hg.u95c,pd.hg.u95d,pd.hg.u95e,pd.hg18.60mer.expr,pd.ht.hg.u133.plus.pm,pd.ht.hg.u133a,pd.ht.mg.430a,pd.hta.2.0,pd.hu6800,pd.huex.1.0.st.v2,pd.hugene.1.0.st.v1,pd.hugene.1.1.st.v1,pd.hugene.2.0.st,pd.hugene.2.1.st,pd.maize,pd.mapping250k.nsp,pd.mapping250k.sty,pd.mapping50k.hind240,pd.mapping50k.xba240,pd.margene.1.0.st,pd.margene.1.1.st,pd.medgene.1.0.st,pd.medgene.1.1.st,pd.medicago,pd.mg.u74a,pd.mg.u74av2,pd.mg.u74b,pd.mg.u74bv2,pd.mg.u74c,pd.mg.u74cv2,pd.mirna.1.0,pd.mirna.2.0,pd.mirna.3.0,pd.mirna.3.1,pd.mirna.4.0,pd.moe430a,pd.moe430b,pd.moex.1.0.st.v1,pd.mogene.1.0.st.v1,pd.mogene.1.1.st.v1,pd.mogene.2.0.st,pd.mogene.2.1.st,pd.mouse430.2,pd.mouse430a.2,pd.mta.1.0,pd.mu11ksuba,pd.mu11ksubb,pd.nugo.hs1a520180,pd.nugo.mm1a520177,pd.ovigene.1.0.st,pd.ovigene.1.1.st,pd.pae.g1a,pd.plasmodium.anopheles,pd.poplar,pd.porcine,pd.porgene.1.0.st,pd.porgene.1.1.st,pd.rabgene.1.0.st,pd.rabgene.1.1.st,pd.rae230a,pd.rae230b,pd.raex.1.0.st.v1,pd.ragene.1.0.st.v1,pd.ragene.1.1.st.v1,pd.ragene.2.0.st,pd.ragene.2.1.st,pd.rat230.2,pd.rcngene.1.0.st,pd.rcngene.1.1.st,pd.rg.u34a,pd.rg.u34b,pd.rg.u34c,pd.rhegene.1.0.st,pd.rhegene.1.1.st,pd.rhesus,pd.rice,pd.rjpgene.1.0.st,pd.rjpgene.1.1.st,pd.rn.u34,pd.rta.1.0,pd.rusgene.1.0.st,pd.rusgene.1.1.st,pd.s.aureus,pd.soybean,pd.soygene.1.0.st,pd.soygene.1.1.st,pd.sugar.cane,pd.tomato,pd.u133.x3p,pd.vitis.vinifera,pd.wheat,pd.x.laevis.2,pd.x.tropicalis,pd.xenopus.laevis,pd.yeast.2,pd.yg.s98,pd.zebgene.1.0.st,pd.zebgene.1.1.st,pd.zebrafish,pedbarrayv10.db,pedbarrayv9.db,PFAM.db,phastCons100way.UCSC.hg19,phastCons100way.UCSC.hg38,phastCons7way.UCSC.hg38,pig.db0,plasmodiumanophelescdf,plasmodiumanophelesprobe,POCRCannotation.db,PolyPhen.Hsapiens.dbSNP131,poplarcdf,poplarprobe,porcine.db,porcinecdf,porcineprobe,primeviewcdf,primeviewprobe,r10kcod.db,rae230a.db,rae230acdf,rae230aprobe,rae230b.db,rae230bcdf,rae230bprobe,raex10stprobeset.db,raex10sttranscriptcluster.db,RaExExonProbesetLocation,ragene10stprobeset.db,ragene10sttranscriptcluster.db,ragene10stv1cdf,ragene10stv1probe,ragene11stprobeset.db,ragene11sttranscriptcluster.db,ragene20stprobeset.db,ragene20sttranscriptcluster.db,ragene21stprobeset.db,ragene21sttranscriptcluster.db,rat.db0,rat2302.db,rat2302cdf,rat2302probe,ratCHRLOC,rattoxfxcdf,rattoxfxprobe,Rattus.norvegicus,reactome.db,rgu34a.db,rgu34acdf,rgu34aprobe,rgu34b.db,rgu34bcdf,rgu34bprobe,rgu34c.db,rgu34ccdf,rgu34cprobe,rguatlas4k.db,rgug4105a.db,rgug4130a.db,rgug4131a.db,rhesus.db0,rhesuscdf,rhesusprobe,ri16cod.db,ribosomaldatabaseproject11.5MgDb,ricecdf,riceprobe,RmiR.Hs.miRNA,RmiR.hsa,RnAgilentDesign028282.db,rnu34.db,rnu34cdf,rnu34probe,Roberts2005Annotation.db,rta10probeset.db,rta10transcriptcluster.db,rtu34.db,rtu34cdf,rtu34probe,rwgcod.db,saureuscdf,saureusprobe,SHDZ.db,SIFT.Hsapiens.dbSNP132,SIFT.Hsapiens.dbSNP137,silva128.1MgDb,SNPlocs.Hsapiens.dbSNP.20101109,SNPlocs.Hsapiens.dbSNP.20120608,SNPlocs.Hsapiens.dbSNP141.GRCh38,SNPlocs.Hsapiens.dbSNP142.GRCh37,SNPlocs.Hsapiens.dbSNP144.GRCh37,SNPlocs.Hsapiens.dbSNP144.GRCh38,SNPlocs.Hsapiens.dbSNP149.GRCh38,SNPlocs.Hsapiens.dbSNP150.GRCh38,SNPlocs.Hsapiens.dbSNP151.GRCh38,soybeancdf,soybeanprobe,sugarcanecdf,sugarcaneprobe,targetscan.Hs.eg.db,targetscan.Mm.eg.db,test1cdf,test2cdf,test3cdf,test3probe,tomatocdf,tomatoprobe,TxDb.Athaliana.BioMart.plantsmart22,TxDb.Athaliana.BioMart.plantsmart25,TxDb.Athaliana.BioMart.plantsmart28,TxDb.Btaurus.UCSC.bosTau8.refGene,TxDb.Celegans.UCSC.ce11.refGene,TxDb.Celegans.UCSC.ce6.ensGene,TxDb.Cfamiliaris.UCSC.canFam3.refGene,TxDb.Dmelanogaster.UCSC.dm3.ensGene,TxDb.Dmelanogaster.UCSC.dm6.ensGene,TxDb.Drerio.UCSC.danRer10.refGene,TxDb.Ggallus.UCSC.galGal4.refGene,TxDb.Ggallus.UCSC.galGal5.refGene,TxDb.Hsapiens.BioMart.igis,TxDb.Hsapiens.UCSC.hg18.knownGene,TxDb.Hsapiens.UCSC.hg19.knownGene,TxDb.Hsapiens.UCSC.hg19.lincRNAsTranscripts,TxDb.Hsapiens.UCSC.hg38.knownGene,TxDb.Mmulatta.UCSC.rheMac3.refGene,TxDb.Mmulatta.UCSC.rheMac8.refGene,TxDb.Mmusculus.UCSC.mm10.ensGene,TxDb.Mmusculus.UCSC.mm10.knownGene,TxDb.Mmusculus.UCSC.mm9.knownGene,TxDb.Ptroglodytes.UCSC.panTro4.refGene,TxDb.Ptroglodytes.UCSC.panTro5.refGene,TxDb.Rnorvegicus.BioMart.igis,TxDb.Rnorvegicus.UCSC.rn4.ensGene,TxDb.Rnorvegicus.UCSC.rn5.refGene,TxDb.Rnorvegicus.UCSC.rn6.refGene,TxDb.Scerevisiae.UCSC.sacCer2.sgdGene,TxDb.Scerevisiae.UCSC.sacCer3.sgdGene,TxDb.Sscrofa.UCSC.susScr11.refGene,TxDb.Sscrofa.UCSC.susScr3.refGene,u133aaofav2cdf,u133x3p.db,u133x3pcdf,u133x3pprobe,vitisviniferacdf,vitisviniferaprobe,wheatcdf,wheatprobe,worm.db0,xenopus.db0,xenopuslaeviscdf,xenopuslaevisprobe,xlaevis.db,xlaevis2cdf,xlaevis2probe,XtraSNPlocs.Hsapiens.dbSNP141.GRCh38,XtraSNPlocs.Hsapiens.dbSNP144.GRCh37,XtraSNPlocs.Hsapiens.dbSNP144.GRCh38,xtropicaliscdf,xtropicalisprobe,ye6100subacdf,ye6100subbcdf,ye6100subccdf,ye6100subdcdf,yeast.db0,yeast2.db,yeast2cdf,yeast2probe,ygs98.db,ygs98cdf,ygs98frmavecs,ygs98probe,zebrafish.db,zebrafish.db0,zebrafishcdf,zebrafishprobe" id="AnnotationData" class="jstree-closed"><ins class="jstree-icon">&nbsp;</ins><a href="https://www.bioconductor.org/packages/release/BiocViews.html#"><ins class="jstree-icon">&nbsp;</ins>AnnotationData (941)</a><ul><li packagelist="adme16cod.db,ag.db,agcdf,agprobe,ath1121501.db,ath1121501cdf,ath1121501probe,barley1cdf,barley1probe,bovine.db,bovinecdf,bovineprobe,bsubtiliscdf,bsubtilisprobe,canine.db,canine2.db,canine2cdf,canine2probe,caninecdf,canineprobe,celegans.db,celeganscdf,celegansprobe,chicken.db,chickencdf,chickenprobe,citruscdf,citrusprobe,clariomdhumanprobeset.db,clariomdhumantranscriptcluster.db,clariomshumanhttranscriptcluster.db,clariomshumantranscriptcluster.db,clariomsmousehttranscriptcluster.db,clariomsmousetranscriptcluster.db,clariomsrathttranscriptcluster.db,clariomsrattranscriptcluster.db,cottoncdf,cottonprobe,cyp450cdf,drosgenome1.db,drosgenome1cdf,drosgenome1probe,drosophila2.db,drosophila2cdf,drosophila2probe,ecoli2.db,ecoli2cdf,ecoli2probe,ecoliasv2cdf,ecoliasv2probe,ecolicdf,ecoliprobe,GGHumanMethCancerPanelv1.db,gp53cdf,h10kcod.db,h20kcod.db,hcg110.db,hcg110cdf,hcg110probe,hgfocus.db,hgfocuscdf,hgfocusprobe,hgu133a.db,hgu133a2.db,hgu133a2cdf,hgu133a2frmavecs,hgu133a2probe,hgu133acdf,hgu133afrmavecs,hgu133aprobe,hgu133atagcdf,hgu133atagprobe,hgu133b.db,hgu133bcdf,hgu133bprobe,hgu133plus2.db,hgu133plus2cdf,hgu133plus2frmavecs,hgu133plus2probe,hgu219.db,hgu219cdf,hgu219probe,hgu95a.db,hgu95acdf,hgu95aprobe,hgu95av2,hgu95av2.db,hgu95av2cdf,hgu95av2probe,hgu95b.db,hgu95bcdf,hgu95bprobe,hgu95c.db,hgu95ccdf,hgu95cprobe,hgu95d.db,hgu95dcdf,hgu95dprobe,hgu95e.db,hgu95ecdf,hgu95eprobe,hgug4100a.db,hgug4101a.db,hgug4110b.db,hgug4111a.db,hgug4112a.db,hguqiagenv3.db,hi16cod.db,hivprtplus2cdf,hs25kresogen.db,HsAgilentDesign026652.db,Hspec,hspeccdf,hta20probeset.db,hta20transcriptcluster.db,hthgu133a.db,hthgu133acdf,hthgu133afrmavecs,hthgu133aprobe,hthgu133b.db,hthgu133bcdf,hthgu133bprobe,hthgu133pluspmcdf,hthgu133pluspmprobe,htmg430acdf,htmg430aprobe,htmg430bcdf,htmg430bprobe,htmg430pmcdf,htmg430pmprobe,htrat230pmcdf,htrat230pmprobe,htratfocuscdf,htratfocusprobe,hu35ksuba.db,hu35ksubacdf,hu35ksubaprobe,hu35ksubb.db,hu35ksubbcdf,hu35ksubbprobe,hu35ksubc.db,hu35ksubccdf,hu35ksubcprobe,hu35ksubd.db,hu35ksubdcdf,hu35ksubdprobe,hu6800.db,hu6800cdf,hu6800probe,hu6800subacdf,hu6800subbcdf,hu6800subccdf,hu6800subdcdf,huex.1.0.st.v2frmavecs,huex10stprobeset.db,huex10sttranscriptcluster.db,HuExExonProbesetLocation,HuExExonProbesetLocationHg18,HuExExonProbesetLocationHg19,hugene.1.0.st.v1frmavecs,hugene10stprobeset.db,hugene10sttranscriptcluster.db,hugene10stv1cdf,hugene10stv1probe,hugene11stprobeset.db,hugene11sttranscriptcluster.db,hugene20stprobeset.db,hugene20sttranscriptcluster.db,hugene21stprobeset.db,hugene21sttranscriptcluster.db,hwgcod.db,IlluminaHumanMethylation27k.db,IlluminaHumanMethylation27kanno.ilmn12.hg19,IlluminaHumanMethylation27kmanifest,IlluminaHumanMethylation450kanno.ilmn12.hg19,IlluminaHumanMethylation450kmanifest,IlluminaHumanMethylationEPICanno.ilm10b2.hg19,IlluminaHumanMethylationEPICanno.ilm10b3.hg19,IlluminaHumanMethylationEPICanno.ilm10b4.hg19,IlluminaHumanMethylationEPICmanifest,illuminaHumanv1.db,illuminaHumanv2.db,illuminaHumanv2BeadID.db,illuminaHumanv3.db,illuminaHumanv4.db,illuminaHumanWGDASLv3.db,illuminaHumanWGDASLv4.db,illuminaMousev1.db,illuminaMousev1p1.db,illuminaMousev2.db,illuminaRatv1.db,indac.db,lumiHumanAll.db,lumiMouseAll.db,lumiRatAll.db,m10kcod.db,m20kcod.db,maizecdf,maizeprobe,medicagocdf,medicagoprobe,mgu74a.db,mgu74acdf,mgu74aprobe,mgu74av2.db,mgu74av2cdf,mgu74av2probe,mgu74b.db,mgu74bcdf,mgu74bprobe,mgu74bv2.db,mgu74bv2cdf,mgu74bv2probe,mgu74c.db,mgu74ccdf,mgu74cprobe,mgu74cv2.db,mgu74cv2cdf,mgu74cv2probe,mgug4104a.db,mgug4120a.db,mgug4121a.db,mgug4122a.db,mi16cod.db,mirna102xgaincdf,mirna10cdf,mirna10probe,mirna20cdf,mm24kresogen.db,MmAgilentDesign026655.db,moe430a.db,moe430acdf,moe430aprobe,moe430b.db,moe430bcdf,moe430bprobe,moex10stprobeset.db,moex10sttranscriptcluster.db,MoExExonProbesetLocation,mogene.1.0.st.v1frmavecs,mogene10stprobeset.db,mogene10sttranscriptcluster.db,mogene10stv1cdf,mogene10stv1probe,mogene11stprobeset.db,mogene11sttranscriptcluster.db,mogene20stprobeset.db,mogene20sttranscriptcluster.db,mogene21stprobeset.db,mogene21sttranscriptcluster.db,mouse4302.db,mouse4302cdf,mouse4302frmavecs,mouse4302probe,mouse430a2.db,mouse430a2cdf,mouse430a2frmavecs,mouse430a2probe,mta10probeset.db,mta10transcriptcluster.db,mu11ksuba.db,mu11ksubacdf,mu11ksubaprobe,mu11ksubb.db,mu11ksubbcdf,mu11ksubbprobe,mu19ksuba.db,mu19ksubacdf,mu19ksubb.db,mu19ksubbcdf,mu19ksubc.db,mu19ksubccdf,mu6500subacdf,mu6500subbcdf,mu6500subccdf,mu6500subdcdf,mwgcod.db,nugohs1a520180.db,nugohs1a520180cdf,nugohs1a520180probe,nugomm1a520177.db,nugomm1a520177cdf,nugomm1a520177probe,paeg1acdf,paeg1aprobe,plasmodiumanophelescdf,plasmodiumanophelesprobe,poplarcdf,poplarprobe,porcine.db,porcinecdf,porcineprobe,primeviewcdf,primeviewprobe,r10kcod.db,rae230a.db,rae230acdf,rae230aprobe,rae230b.db,rae230bcdf,rae230bprobe,raex10stprobeset.db,raex10sttranscriptcluster.db,RaExExonProbesetLocation,ragene10stprobeset.db,ragene10sttranscriptcluster.db,ragene10stv1cdf,ragene10stv1probe,ragene11stprobeset.db,ragene11sttranscriptcluster.db,ragene20stprobeset.db,ragene20sttranscriptcluster.db,ragene21stprobeset.db,ragene21sttranscriptcluster.db,rat2302.db,rat2302cdf,rat2302probe,rattoxfxcdf,rattoxfxprobe,rgu34a.db,rgu34acdf,rgu34aprobe,rgu34b.db,rgu34bcdf,rgu34bprobe,rgu34c.db,rgu34ccdf,rgu34cprobe,rgug4105a.db,rgug4130a.db,rgug4131a.db,rhesuscdf,rhesusprobe,ri16cod.db,ricecdf,riceprobe,RnAgilentDesign028282.db,rnu34.db,rnu34cdf,rnu34probe,rta10probeset.db,rta10transcriptcluster.db,rtu34.db,rtu34cdf,rtu34probe,rwgcod.db,saureuscdf,saureusprobe,soybeancdf,soybeanprobe,sugarcanecdf,sugarcaneprobe,test1cdf,test2cdf,test3cdf,test3probe,tomatocdf,tomatoprobe,u133aaofav2cdf,u133x3p.db,u133x3pcdf,u133x3pprobe,vitisviniferacdf,vitisviniferaprobe,wheatcdf,wheatprobe,xenopuslaeviscdf,xenopuslaevisprobe,xlaevis.db,xlaevis2cdf,xlaevis2probe,xtropicaliscdf,xtropicalisprobe,ye6100subacdf,ye6100subbcdf,ye6100subccdf,ye6100subdcdf,yeast2.db,yeast2cdf,yeast2probe,ygs98.db,ygs98cdf,ygs98frmavecs,ygs98probe,zebrafish.db,zebrafishcdf,zebrafishprobe" id="ChipManufacturer" class="jstree-closed"><ins class="jstree-icon">&nbsp;</ins><a href="https://www.bioconductor.org/packages/release/BiocViews.html#"><ins class="jstree-icon">&nbsp;</ins>ChipManufacturer (387)</a><ul><li packagelist="ag.db,agcdf,agprobe,ath1121501.db,ath1121501cdf,ath1121501probe,barley1cdf,barley1probe,bovine.db,bovinecdf,bovineprobe,bsubtiliscdf,bsubtilisprobe,canine.db,canine2.db,canine2cdf,canine2probe,caninecdf,canineprobe,celegans.db,celeganscdf,celegansprobe,chicken.db,chickencdf,chickenprobe,citruscdf,citrusprobe,clariomdhumanprobeset.db,clariomdhumantranscriptcluster.db,clariomshumanhttranscriptcluster.db,clariomshumantranscriptcluster.db,clariomsmousehttranscriptcluster.db,clariomsmousetranscriptcluster.db,clariomsrathttranscriptcluster.db,clariomsrattranscriptcluster.db,cottoncdf,cottonprobe,cyp450cdf,drosgenome1.db,drosgenome1cdf,drosgenome1probe,drosophila2.db,drosophila2cdf,drosophila2probe,ecoli2.db,ecoli2cdf,ecoli2probe,ecoliasv2cdf,ecoliasv2probe,ecolicdf,ecoliprobe,gp53cdf,hcg110.db,hcg110cdf,hcg110probe,hgfocus.db,hgfocuscdf,hgfocusprobe,hgu133a.db,hgu133a2.db,hgu133a2cdf,hgu133a2frmavecs,hgu133a2probe,hgu133acdf,hgu133afrmavecs,hgu133aprobe,hgu133atagcdf,hgu133atagprobe,hgu133b.db,hgu133bcdf,hgu133bprobe,hgu133plus2.db,hgu133plus2cdf,hgu133plus2frmavecs,hgu133plus2probe,hgu219.db,hgu219cdf,hgu219probe,hgu95a.db,hgu95acdf,hgu95aprobe,hgu95av2,hgu95av2.db,hgu95av2cdf,hgu95av2probe,hgu95b.db,hgu95bcdf,hgu95bprobe,hgu95c.db,hgu95ccdf,hgu95cprobe,hgu95d.db,hgu95dcdf,hgu95dprobe,hgu95e.db,hgu95ecdf,hgu95eprobe,hivprtplus2cdf,Hspec,hspeccdf,hta20probeset.db,hta20transcriptcluster.db,hthgu133a.db,hthgu133acdf,hthgu133afrmavecs,hthgu133aprobe,hthgu133b.db,hthgu133bcdf,hthgu133bprobe,hthgu133pluspmcdf,hthgu133pluspmprobe,htmg430acdf,htmg430aprobe,htmg430bcdf,htmg430bprobe,htmg430pmcdf,htmg430pmprobe,htrat230pmcdf,htrat230pmprobe,htratfocuscdf,htratfocusprobe,hu35ksuba.db,hu35ksubacdf,hu35ksubaprobe,hu35ksubb.db,hu35ksubbcdf,hu35ksubbprobe,hu35ksubc.db,hu35ksubccdf,hu35ksubcprobe,hu35ksubd.db,hu35ksubdcdf,hu35ksubdprobe,hu6800.db,hu6800cdf,hu6800probe,hu6800subacdf,hu6800subbcdf,hu6800subccdf,hu6800subdcdf,huex.1.0.st.v2frmavecs,huex10stprobeset.db,huex10sttranscriptcluster.db,HuExExonProbesetLocation,HuExExonProbesetLocationHg18,HuExExonProbesetLocationHg19,hugene.1.0.st.v1frmavecs,hugene10stprobeset.db,hugene10sttranscriptcluster.db,hugene10stv1cdf,hugene10stv1probe,hugene11stprobeset.db,hugene11sttranscriptcluster.db,hugene20stprobeset.db,hugene20sttranscriptcluster.db,hugene21stprobeset.db,hugene21sttranscriptcluster.db,maizecdf,maizeprobe,medicagocdf,medicagoprobe,mgu74a.db,mgu74acdf,mgu74aprobe,mgu74av2.db,mgu74av2cdf,mgu74av2probe,mgu74b.db,mgu74bcdf,mgu74bprobe,mgu74bv2.db,mgu74bv2cdf,mgu74bv2probe,mgu74c.db,mgu74ccdf,mgu74cprobe,mgu74cv2.db,mgu74cv2cdf,mgu74cv2probe,mirna102xgaincdf,mirna10cdf,mirna10probe,mirna20cdf,moe430a.db,moe430acdf,moe430aprobe,moe430b.db,moe430bcdf,moe430bprobe,moex10stprobeset.db,moex10sttranscriptcluster.db,MoExExonProbesetLocation,mogene.1.0.st.v1frmavecs,mogene10stprobeset.db,mogene10sttranscriptcluster.db,mogene10stv1cdf,mogene10stv1probe,mogene11stprobeset.db,mogene11sttranscriptcluster.db,mogene20stprobeset.db,mogene20sttranscriptcluster.db,mogene21stprobeset.db,mogene21sttranscriptcluster.db,mouse4302.db,mouse4302cdf,mouse4302frmavecs,mouse4302probe,mouse430a2.db,mouse430a2cdf,mouse430a2frmavecs,mouse430a2probe,mta10probeset.db,mta10transcriptcluster.db,mu11ksuba.db,mu11ksubacdf,mu11ksubaprobe,mu11ksubb.db,mu11ksubbcdf,mu11ksubbprobe,mu19ksuba.db,mu19ksubacdf,mu19ksubb.db,mu19ksubbcdf,mu19ksubc.db,mu19ksubccdf,mu6500subacdf,mu6500subbcdf,mu6500subccdf,mu6500subdcdf,nugohs1a520180.db,nugohs1a520180cdf,nugohs1a520180probe,nugomm1a520177.db,nugomm1a520177cdf,nugomm1a520177probe,paeg1acdf,paeg1aprobe,plasmodiumanophelescdf,plasmodiumanophelesprobe,poplarcdf,poplarprobe,porcine.db,porcinecdf,porcineprobe,primeviewcdf,primeviewprobe,rae230a.db,rae230acdf,rae230aprobe,rae230b.db,rae230bcdf,rae230bprobe,raex10stprobeset.db,raex10sttranscriptcluster.db,RaExExonProbesetLocation,ragene10stprobeset.db,ragene10sttranscriptcluster.db,ragene10stv1cdf,ragene10stv1probe,ragene11stprobeset.db,ragene11sttranscriptcluster.db,ragene20stprobeset.db,ragene20sttranscriptcluster.db,ragene21stprobeset.db,ragene21sttranscriptcluster.db,rat2302.db,rat2302cdf,rat2302probe,rattoxfxcdf,rattoxfxprobe,rgu34a.db,rgu34acdf,rgu34aprobe,rgu34b.db,rgu34bcdf,rgu34bprobe,rgu34c.db,rgu34ccdf,rgu34cprobe,rhesuscdf,rhesusprobe,ricecdf,riceprobe,rnu34.db,rnu34cdf,rnu34probe,rta10probeset.db,rta10transcriptcluster.db,rtu34.db,rtu34cdf,rtu34probe,saureuscdf,saureusprobe,soybeancdf,soybeanprobe,sugarcanecdf,sugarcaneprobe,test1cdf,test2cdf,test3cdf,test3probe,tomatocdf,tomatoprobe,u133aaofav2cdf,u133x3p.db,u133x3pcdf,u133x3pprobe,vitisviniferacdf,vitisviniferaprobe,wheatcdf,wheatprobe,xenopuslaeviscdf,xenopuslaevisprobe,xlaevis.db,xlaevis2cdf,xlaevis2probe,xtropicaliscdf,xtropicalisprobe,ye6100subacdf,ye6100subbcdf,ye6100subccdf,ye6100subdcdf,yeast2.db,yeast2cdf,yeast2probe,ygs98.db,ygs98cdf,ygs98frmavecs,ygs98probe,zebrafish.db,zebrafishcdf,zebrafishprobe" id="AffymetrixChip" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="https://www.bioconductor.org/packages/release/BiocViews.html#"><ins class="jstree-icon">&nbsp;</ins>AffymetrixChip (332)</a></li><li packagelist="hgug4100a.db,hgug4101a.db,hgug4110b.db,hgug4111a.db,hgug4112a.db,HsAgilentDesign026652.db,mgug4104a.db,mgug4120a.db,mgug4121a.db,mgug4122a.db,MmAgilentDesign026655.db,rgug4105a.db,rgug4130a.db,rgug4131a.db,RnAgilentDesign028282.db" id="AgilentChip" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="https://www.bioconductor.org/packages/release/BiocViews.html#"><ins class="jstree-icon">&nbsp;</ins>AgilentChip (15)</a></li><li class="jstree-leaf" style="display: none;"><ins class="jstree-icon">&nbsp;</ins><a href="https://www.bioconductor.org/packages/release/BiocViews.html#"><ins class="jstree-icon">&nbsp;</ins>ClonetechChip</a></li><li packagelist="adme16cod.db,h10kcod.db,h20kcod.db,hi16cod.db,hwgcod.db,m10kcod.db,m20kcod.db,mi16cod.db,mwgcod.db,r10kcod.db,ri16cod.db,rwgcod.db" id="CodelinkChip" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="https://www.bioconductor.org/packages/release/BiocViews.html#"><ins class="jstree-icon">&nbsp;</ins>CodelinkChip (12)</a></li><li class="jstree-leaf" style="display: none;"><ins class="jstree-icon">&nbsp;</ins><a href="https://www.bioconductor.org/packages/release/BiocViews.html#"><ins class="jstree-icon">&nbsp;</ins>GEChip</a></li><li packagelist="GGHumanMethCancerPanelv1.db,IlluminaHumanMethylation27k.db,IlluminaHumanMethylation27kanno.ilmn12.hg19,IlluminaHumanMethylation27kmanifest,IlluminaHumanMethylation450kanno.ilmn12.hg19,IlluminaHumanMethylation450kmanifest,IlluminaHumanMethylationEPICanno.ilm10b2.hg19,IlluminaHumanMethylationEPICanno.ilm10b3.hg19,IlluminaHumanMethylationEPICanno.ilm10b4.hg19,IlluminaHumanMethylationEPICmanifest,illuminaHumanv1.db,illuminaHumanv2.db,illuminaHumanv2BeadID.db,illuminaHumanv3.db,illuminaHumanv4.db,illuminaHumanWGDASLv3.db,illuminaHumanWGDASLv4.db,illuminaMousev1.db,illuminaMousev1p1.db,illuminaMousev2.db,illuminaRatv1.db,lumiHumanAll.db,lumiMouseAll.db,lumiRatAll.db" id="IlluminaChip" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="https://www.bioconductor.org/packages/release/BiocViews.html#"><ins class="jstree-icon">&nbsp;</ins>IlluminaChip (24)</a></li><li packagelist="indac.db" id="INDACChip" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="https://www.bioconductor.org/packages/release/BiocViews.html#"><ins class="jstree-icon">&nbsp;</ins>INDACChip (1)</a></li><li packagelist="hguqiagenv3.db" id="QiagenChip" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="https://www.bioconductor.org/packages/release/BiocViews.html#"><ins class="jstree-icon">&nbsp;</ins>QiagenChip (1)</a></li><li packagelist="hs25kresogen.db,mm24kresogen.db" id="RNG_MRCChip" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="https://www.bioconductor.org/packages/release/BiocViews.html#"><ins class="jstree-icon">&nbsp;</ins>RNG_MRCChip (2)</a></li><li class="jstree-leaf" style="display: none;"><ins class="jstree-icon">&nbsp;</ins><a href="https://www.bioconductor.org/packages/release/BiocViews.html#"><ins class="jstree-icon">&nbsp;</ins>RocheChip</a></li><li class="jstree-last jstree-leaf" style="display: none;"><ins class="jstree-icon">&nbsp;</ins><a href="https://www.bioconductor.org/packages/release/BiocViews.html#"><ins class="jstree-icon">&nbsp;</ins>UniversityHealthNetwork</a></li></ul></li><li packagelist="adme16cod.db,ag.db,agcdf,agprobe,ath1121501.db,ath1121501cdf,ath1121501probe,celegans.db,celeganscdf,celegansprobe,drosgenome1.db,drosgenome1cdf,drosgenome1probe,drosophila2.db,drosophila2cdf,drosophila2probe,h10kcod.db,h20kcod.db,hcg110.db,hcg110cdf,hcg110probe,hgfocus.db,hgfocuscdf,hgfocusprobe,hgu133a.db,hgu133a2.db,hgu133a2cdf,hgu133a2frmavecs,hgu133a2probe,hgu133acdf,hgu133afrmavecs,hgu133aprobe,hgu133b.db,hgu133bcdf,hgu133bprobe,hgu133plus2.db,hgu133plus2cdf,hgu133plus2frmavecs,hgu133plus2probe,hgu95a.db,hgu95acdf,hgu95aprobe,hgu95av2,hgu95av2.db,hgu95av2cdf,hgu95av2probe,hgu95b.db,hgu95bcdf,hgu95bprobe,hgu95c.db,hgu95ccdf,hgu95cprobe,hgu95d.db,hgu95dcdf,hgu95dprobe,hgu95e.db,hgu95ecdf,hgu95eprobe,hguatlas13k.db,hgug4100a.db,hgug4101a.db,hgug4110b.db,hgug4111a.db,hgug4112a.db,hguqiagenv3.db,hi16cod.db,hs25kresogen.db,hu35ksuba.db,hu35ksubacdf,hu35ksubaprobe,hu35ksubb.db,hu35ksubbcdf,hu35ksubbprobe,hu35ksubc.db,hu35ksubccdf,hu35ksubcprobe,hu35ksubd.db,hu35ksubdcdf,hu35ksubdprobe,hu6800.db,hu6800cdf,hu6800probe,HuO22.db,hwgcod.db,illuminaHumanv1.db,illuminaHumanv2.db,illuminaMousev1.db,illuminaMousev1p1.db,illuminaRatv1.db,indac.db,m10kcod.db,m20kcod.db,mgu74a.db,mgu74acdf,mgu74aprobe,mgu74av2.db,mgu74av2cdf,mgu74av2probe,mgu74b.db,mgu74bcdf,mgu74bprobe,mgu74bv2.db,mgu74bv2cdf,mgu74bv2probe,mgu74c.db,mgu74ccdf,mgu74cprobe,mgu74cv2.db,mgu74cv2cdf,mgu74cv2probe,mguatlas5k.db,mgug4121a.db,mgug4122a.db,mi16cod.db,mm24kresogen.db,moe430a.db,moe430acdf,moe430aprobe,moe430b.db,moe430bcdf,moe430bprobe,mouse4302.db,mouse4302cdf,mouse4302frmavecs,mouse4302probe,mouse430a2.db,mouse430a2cdf,mouse430a2frmavecs,mouse430a2probe,mpedbarray.db,mu11ksuba.db,mu11ksubacdf,mu11ksubaprobe,mu11ksubb.db,mu11ksubbcdf,mu11ksubbprobe,Mu15v1.db,mu19ksuba.db,mu19ksubacdf,mu19ksubb.db,mu19ksubbcdf,mu19ksubc.db,mu19ksubccdf,Mu22v3.db,mwgcod.db,Norway981.db,OperonHumanV3.db,PartheenMetaData.db,pedbarrayv10.db,pedbarrayv9.db,r10kcod.db,rae230a.db,rae230acdf,rae230aprobe,rae230b.db,rae230bcdf,rae230bprobe,rat2302.db,rat2302cdf,rat2302probe,rgu34a.db,rgu34acdf,rgu34aprobe,rgu34b.db,rgu34bcdf,rgu34bprobe,rgu34c.db,rgu34ccdf,rgu34cprobe,rgug4130a.db,ri16cod.db,rnu34.db,rnu34cdf,rnu34probe,Roberts2005Annotation.db,rtu34.db,rtu34cdf,rtu34probe,rwgcod.db,SHDZ.db,u133x3p.db,u133x3pcdf,u133x3pprobe,xenopuslaeviscdf,xenopuslaevisprobe,yeast2.db,yeast2cdf,yeast2probe,ygs98.db,ygs98cdf,ygs98frmavecs,ygs98probe,zebrafish.db,zebrafishcdf,zebrafishprobe" id="ChipName" class="jstree-closed"><ins class="jstree-icon">&nbsp;</ins><a href="https://www.bioconductor.org/packages/release/BiocViews.html#"><ins class="jstree-icon">&nbsp;</ins>ChipName (195)</a><ul><li packagelist="adme16cod.db" id="adme16cod" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="https://www.bioconductor.org/packages/release/BiocViews.html#"><ins class="jstree-icon">&nbsp;</ins>adme16cod (1)</a></li><li packagelist="ag.db,agcdf,agprobe" id="ag" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="https://www.bioconductor.org/packages/release/BiocViews.html#"><ins class="jstree-icon">&nbsp;</ins>ag (3)</a></li><li packagelist="ath1121501.db,ath1121501cdf,ath1121501probe" id="ath1121501" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="https://www.bioconductor.org/packages/release/BiocViews.html#"><ins class="jstree-icon">&nbsp;</ins>ath1121501 (3)</a></li><li packagelist="celegans.db,celeganscdf,celegansprobe" id="celegans" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="https://www.bioconductor.org/packages/release/BiocViews.html#"><ins class="jstree-icon">&nbsp;</ins>celegans (3)</a></li><li packagelist="drosgenome1.db,drosgenome1cdf,drosgenome1probe" id="drosgenome1" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="https://www.bioconductor.org/packages/release/BiocViews.html#"><ins class="jstree-icon">&nbsp;</ins>drosgenome1 (3)</a></li><li packagelist="drosophila2.db,drosophila2cdf,drosophila2probe" id="drosophila2" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="https://www.bioconductor.org/packages/release/BiocViews.html#"><ins class="jstree-icon">&nbsp;</ins>drosophila2 (3)</a></li><li packagelist="h10kcod.db" id="h10kcod" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="https://www.bioconductor.org/packages/release/BiocViews.html#"><ins class="jstree-icon">&nbsp;</ins>h10kcod (1)</a></li><li packagelist="h20kcod.db" id="h20kcod" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="https://www.bioconductor.org/packages/release/BiocViews.html#"><ins class="jstree-icon">&nbsp;</ins>h20kcod (1)</a></li><li packagelist="hcg110.db,hcg110cdf,hcg110probe" id="hcg110" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="https://www.bioconductor.org/packages/release/BiocViews.html#"><ins class="jstree-icon">&nbsp;</ins>hcg110 (3)</a></li><li class="jstree-leaf" style="display: none;"><ins class="jstree-icon">&nbsp;</ins><a href="https://www.bioconductor.org/packages/release/BiocViews.html#"><ins class="jstree-icon">&nbsp;</ins>hcgi12k</a></li><li class="jstree-leaf" style="display: none;"><ins class="jstree-icon">&nbsp;</ins><a href="https://www.bioconductor.org/packages/release/BiocViews.html#"><ins class="jstree-icon">&nbsp;</ins>hcgi8k</a></li><li packagelist="hgfocus.db,hgfocuscdf,hgfocusprobe" id="hgfocus" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="https://www.bioconductor.org/packages/release/BiocViews.html#"><ins class="jstree-icon">&nbsp;</ins>hgfocus (3)</a></li><li packagelist="hgu133a.db,hgu133acdf,hgu133afrmavecs,hgu133aprobe" id="hgu133a" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="https://www.bioconductor.org/packages/release/BiocViews.html#"><ins class="jstree-icon">&nbsp;</ins>hgu133a (4)</a></li><li packagelist="hgu133a2.db,hgu133a2cdf,hgu133a2frmavecs,hgu133a2probe" id="hgu133a2" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="https://www.bioconductor.org/packages/release/BiocViews.html#"><ins class="jstree-icon">&nbsp;</ins>hgu133a2 (4)</a></li><li packagelist="hgu133b.db,hgu133bcdf,hgu133bprobe" id="hgu133b" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="https://www.bioconductor.org/packages/release/BiocViews.html#"><ins class="jstree-icon">&nbsp;</ins>hgu133b (3)</a></li><li packagelist="hgu133plus2.db,hgu133plus2cdf,hgu133plus2frmavecs,hgu133plus2probe" id="hgu133plus2" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="https://www.bioconductor.org/packages/release/BiocViews.html#"><ins class="jstree-icon">&nbsp;</ins>hgu133plus2 (4)</a></li><li packagelist="hgu95a.db,hgu95acdf,hgu95aprobe" id="hgu95a" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="https://www.bioconductor.org/packages/release/BiocViews.html#"><ins class="jstree-icon">&nbsp;</ins>hgu95a (3)</a></li><li packagelist="hgu95av2,hgu95av2.db,hgu95av2cdf,hgu95av2probe" id="hgu95av2" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="https://www.bioconductor.org/packages/release/BiocViews.html#"><ins class="jstree-icon">&nbsp;</ins>hgu95av2 (4)</a></li><li packagelist="hgu95b.db,hgu95bcdf,hgu95bprobe" id="hgu95b" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="https://www.bioconductor.org/packages/release/BiocViews.html#"><ins class="jstree-icon">&nbsp;</ins>hgu95b (3)</a></li><li packagelist="hgu95c.db,hgu95ccdf,hgu95cprobe" id="hgu95c" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="https://www.bioconductor.org/packages/release/BiocViews.html#"><ins class="jstree-icon">&nbsp;</ins>hgu95c (3)</a></li><li packagelist="hgu95d.db,hgu95dcdf,hgu95dprobe" id="hgu95d" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="https://www.bioconductor.org/packages/release/BiocViews.html#"><ins class="jstree-icon">&nbsp;</ins>hgu95d (3)</a></li><li packagelist="hgu95e.db,hgu95ecdf,hgu95eprobe" id="hgu95e" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="https://www.bioconductor.org/packages/release/BiocViews.html#"><ins class="jstree-icon">&nbsp;</ins>hgu95e (3)</a></li><li packagelist="hguatlas13k.db" id="hguatlas13k" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="https://www.bioconductor.org/packages/release/BiocViews.html#"><ins class="jstree-icon">&nbsp;</ins>hguatlas13k (1)</a></li><li packagelist="hgug4100a.db" id="hgug4100a" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="https://www.bioconductor.org/packages/release/BiocViews.html#"><ins class="jstree-icon">&nbsp;</ins>hgug4100a (1)</a></li><li packagelist="hgug4101a.db" id="hgug4101a" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="https://www.bioconductor.org/packages/release/BiocViews.html#"><ins class="jstree-icon">&nbsp;</ins>hgug4101a (1)</a></li><li packagelist="hgug4110b.db" id="hgug4110b" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="https://www.bioconductor.org/packages/release/BiocViews.html#"><ins class="jstree-icon">&nbsp;</ins>hgug4110b (1)</a></li><li packagelist="hgug4111a.db" id="hgug4111a" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="https://www.bioconductor.org/packages/release/BiocViews.html#"><ins class="jstree-icon">&nbsp;</ins>hgug4111a (1)</a></li><li packagelist="hgug4112a.db" id="hgug4112a" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="https://www.bioconductor.org/packages/release/BiocViews.html#"><ins class="jstree-icon">&nbsp;</ins>hgug4112a (1)</a></li><li packagelist="hguqiagenv3.db" id="hguqiagenv3" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="https://www.bioconductor.org/packages/release/BiocViews.html#"><ins class="jstree-icon">&nbsp;</ins>hguqiagenv3 (1)</a></li><li packagelist="hi16cod.db" id="hi16cod" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="https://www.bioconductor.org/packages/release/BiocViews.html#"><ins class="jstree-icon">&nbsp;</ins>hi16cod (1)</a></li><li packagelist="hs25kresogen.db" id="hs25kresogen" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="https://www.bioconductor.org/packages/release/BiocViews.html#"><ins class="jstree-icon">&nbsp;</ins>hs25kresogen (1)</a></li><li packagelist="hu35ksuba.db,hu35ksubacdf,hu35ksubaprobe" id="hu35ksuba" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="https://www.bioconductor.org/packages/release/BiocViews.html#"><ins class="jstree-icon">&nbsp;</ins>hu35ksuba (3)</a></li><li packagelist="hu35ksubb.db,hu35ksubbcdf,hu35ksubbprobe" id="hu35ksubb" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="https://www.bioconductor.org/packages/release/BiocViews.html#"><ins class="jstree-icon">&nbsp;</ins>hu35ksubb (3)</a></li><li packagelist="hu35ksubc.db,hu35ksubccdf,hu35ksubcprobe" id="hu35ksubc" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="https://www.bioconductor.org/packages/release/BiocViews.html#"><ins class="jstree-icon">&nbsp;</ins>hu35ksubc (3)</a></li><li packagelist="hu35ksubd.db,hu35ksubdcdf,hu35ksubdprobe" id="hu35ksubd" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="https://www.bioconductor.org/packages/release/BiocViews.html#"><ins class="jstree-icon">&nbsp;</ins>hu35ksubd (3)</a></li><li packagelist="hu6800.db,hu6800cdf,hu6800probe" id="hu6800" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="https://www.bioconductor.org/packages/release/BiocViews.html#"><ins class="jstree-icon">&nbsp;</ins>hu6800 (3)</a></li><li packagelist="HuO22.db" id="HuO22" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="https://www.bioconductor.org/packages/release/BiocViews.html#"><ins class="jstree-icon">&nbsp;</ins>HuO22 (1)</a></li><li packagelist="hwgcod.db" id="hwgcod" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="https://www.bioconductor.org/packages/release/BiocViews.html#"><ins class="jstree-icon">&nbsp;</ins>hwgcod (1)</a></li><li packagelist="illuminaHumanv1.db" id="illuminaHumanv1" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="https://www.bioconductor.org/packages/release/BiocViews.html#"><ins class="jstree-icon">&nbsp;</ins>illuminaHumanv1 (1)</a></li><li packagelist="illuminaHumanv2.db" id="illuminaHumanv2" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="https://www.bioconductor.org/packages/release/BiocViews.html#"><ins class="jstree-icon">&nbsp;</ins>illuminaHumanv2 (1)</a></li><li packagelist="illuminaMousev1.db" id="illuminaMousev1" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="https://www.bioconductor.org/packages/release/BiocViews.html#"><ins class="jstree-icon">&nbsp;</ins>illuminaMousev1 (1)</a></li><li packagelist="illuminaMousev1p1.db" id="illuminaMousev1p1" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="https://www.bioconductor.org/packages/release/BiocViews.html#"><ins class="jstree-icon">&nbsp;</ins>illuminaMousev1p1 (1)</a></li><li packagelist="illuminaRatv1.db" id="illuminaRatv1" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="https://www.bioconductor.org/packages/release/BiocViews.html#"><ins class="jstree-icon">&nbsp;</ins>illuminaRatv1 (1)</a></li><li packagelist="indac.db" id="indac" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="https://www.bioconductor.org/packages/release/BiocViews.html#"><ins class="jstree-icon">&nbsp;</ins>indac (1)</a></li><li class="jstree-leaf" style="display: none;"><ins class="jstree-icon">&nbsp;</ins><a href="https://www.bioconductor.org/packages/release/BiocViews.html#"><ins class="jstree-icon">&nbsp;</ins>JazaerimetaData</a></li><li class="jstree-leaf" style="display: none;"><ins class="jstree-icon">&nbsp;</ins><a href="https://www.bioconductor.org/packages/release/BiocViews.html#"><ins class="jstree-icon">&nbsp;</ins>lumiHumanV1</a></li><li class="jstree-leaf" style="display: none;"><ins class="jstree-icon">&nbsp;</ins><a href="https://www.bioconductor.org/packages/release/BiocViews.html#"><ins class="jstree-icon">&nbsp;</ins>lumiHumanV2</a></li><li class="jstree-leaf" style="display: none;"><ins class="jstree-icon">&nbsp;</ins><a href="https://www.bioconductor.org/packages/release/BiocViews.html#"><ins class="jstree-icon">&nbsp;</ins>lumiMouseV1</a></li><li class="jstree-leaf" style="display: none;"><ins class="jstree-icon">&nbsp;</ins><a href="https://www.bioconductor.org/packages/release/BiocViews.html#"><ins class="jstree-icon">&nbsp;</ins>lumiRatV1</a></li><li packagelist="m10kcod.db" id="m10kcod" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="https://www.bioconductor.org/packages/release/BiocViews.html#"><ins class="jstree-icon">&nbsp;</ins>m10kcod (1)</a></li><li packagelist="m20kcod.db" id="m20kcod" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="https://www.bioconductor.org/packages/release/BiocViews.html#"><ins class="jstree-icon">&nbsp;</ins>m20kcod (1)</a></li><li packagelist="mgu74a.db,mgu74acdf,mgu74aprobe" id="mgu74a" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="https://www.bioconductor.org/packages/release/BiocViews.html#"><ins class="jstree-icon">&nbsp;</ins>mgu74a (3)</a></li><li packagelist="mgu74av2.db,mgu74av2cdf,mgu74av2probe" id="mgu74av2" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="https://www.bioconductor.org/packages/release/BiocViews.html#"><ins class="jstree-icon">&nbsp;</ins>mgu74av2 (3)</a></li><li packagelist="mgu74b.db,mgu74bcdf,mgu74bprobe" id="mgu74b" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="https://www.bioconductor.org/packages/release/BiocViews.html#"><ins class="jstree-icon">&nbsp;</ins>mgu74b (3)</a></li><li packagelist="mgu74bv2.db,mgu74bv2cdf,mgu74bv2probe" id="mgu74bv2" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="https://www.bioconductor.org/packages/release/BiocViews.html#"><ins class="jstree-icon">&nbsp;</ins>mgu74bv2 (3)</a></li><li packagelist="mgu74c.db,mgu74ccdf,mgu74cprobe" id="mgu74c" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="https://www.bioconductor.org/packages/release/BiocViews.html#"><ins class="jstree-icon">&nbsp;</ins>mgu74c (3)</a></li><li packagelist="mgu74cv2.db,mgu74cv2cdf,mgu74cv2probe" id="mgu74cv2" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="https://www.bioconductor.org/packages/release/BiocViews.html#"><ins class="jstree-icon">&nbsp;</ins>mgu74cv2 (3)</a></li><li packagelist="mguatlas5k.db" id="mguatlas5k" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="https://www.bioconductor.org/packages/release/BiocViews.html#"><ins class="jstree-icon">&nbsp;</ins>mguatlas5k (1)</a></li><li packagelist="mgug4121a.db" id="mgug4121a" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="https://www.bioconductor.org/packages/release/BiocViews.html#"><ins class="jstree-icon">&nbsp;</ins>mgug4121a (1)</a></li><li packagelist="mgug4122a.db" id="mgug4122a" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="https://www.bioconductor.org/packages/release/BiocViews.html#"><ins class="jstree-icon">&nbsp;</ins>mgug4122a (1)</a></li><li packagelist="mi16cod.db" id="mi16cod" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="https://www.bioconductor.org/packages/release/BiocViews.html#"><ins class="jstree-icon">&nbsp;</ins>mi16cod (1)</a></li><li packagelist="mm24kresogen.db" id="mm24kresogen" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="https://www.bioconductor.org/packages/release/BiocViews.html#"><ins class="jstree-icon">&nbsp;</ins>mm24kresogen (1)</a></li><li packagelist="moe430a.db,moe430acdf,moe430aprobe" id="moe430a" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="https://www.bioconductor.org/packages/release/BiocViews.html#"><ins class="jstree-icon">&nbsp;</ins>moe430a (3)</a></li><li packagelist="moe430b.db,moe430bcdf,moe430bprobe" id="moe430b" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="https://www.bioconductor.org/packages/release/BiocViews.html#"><ins class="jstree-icon">&nbsp;</ins>moe430b (3)</a></li><li packagelist="mouse4302.db,mouse4302cdf,mouse4302frmavecs,mouse4302probe" id="mouse4302" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="https://www.bioconductor.org/packages/release/BiocViews.html#"><ins class="jstree-icon">&nbsp;</ins>mouse4302 (4)</a></li><li packagelist="mouse430a2.db,mouse430a2cdf,mouse430a2frmavecs,mouse430a2probe" id="mouse430a2" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="https://www.bioconductor.org/packages/release/BiocViews.html#"><ins class="jstree-icon">&nbsp;</ins>mouse430a2 (4)</a></li><li packagelist="mpedbarray.db" id="mpedbarray" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="https://www.bioconductor.org/packages/release/BiocViews.html#"><ins class="jstree-icon">&nbsp;</ins>mpedbarray (1)</a></li><li packagelist="mu11ksuba.db,mu11ksubacdf,mu11ksubaprobe" id="mu11ksuba" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="https://www.bioconductor.org/packages/release/BiocViews.html#"><ins class="jstree-icon">&nbsp;</ins>mu11ksuba (3)</a></li><li packagelist="mu11ksubb.db,mu11ksubbcdf,mu11ksubbprobe" id="mu11ksubb" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="https://www.bioconductor.org/packages/release/BiocViews.html#"><ins class="jstree-icon">&nbsp;</ins>mu11ksubb (3)</a></li><li packagelist="Mu15v1.db" id="Mu15v1" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="https://www.bioconductor.org/packages/release/BiocViews.html#"><ins class="jstree-icon">&nbsp;</ins>Mu15v1 (1)</a></li><li packagelist="mu19ksuba.db,mu19ksubacdf" id="mu19ksuba" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="https://www.bioconductor.org/packages/release/BiocViews.html#"><ins class="jstree-icon">&nbsp;</ins>mu19ksuba (2)</a></li><li packagelist="mu19ksubb.db,mu19ksubbcdf" id="mu19ksubb" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="https://www.bioconductor.org/packages/release/BiocViews.html#"><ins class="jstree-icon">&nbsp;</ins>mu19ksubb (2)</a></li><li packagelist="mu19ksubc.db,mu19ksubccdf" id="mu19ksubc" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="https://www.bioconductor.org/packages/release/BiocViews.html#"><ins class="jstree-icon">&nbsp;</ins>mu19ksubc (2)</a></li><li packagelist="Mu22v3.db" id="Mu22v3" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="https://www.bioconductor.org/packages/release/BiocViews.html#"><ins class="jstree-icon">&nbsp;</ins>Mu22v3 (1)</a></li><li packagelist="mwgcod.db" id="mwgcod" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="https://www.bioconductor.org/packages/release/BiocViews.html#"><ins class="jstree-icon">&nbsp;</ins>mwgcod (1)</a></li><li packagelist="Norway981.db" id="Norway981" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="https://www.bioconductor.org/packages/release/BiocViews.html#"><ins class="jstree-icon">&nbsp;</ins>Norway981 (1)</a></li><li packagelist="OperonHumanV3.db" id="OperonHumanV3" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="https://www.bioconductor.org/packages/release/BiocViews.html#"><ins class="jstree-icon">&nbsp;</ins>OperonHumanV3 (1)</a></li><li packagelist="PartheenMetaData.db" id="PartheenMetaData" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="https://www.bioconductor.org/packages/release/BiocViews.html#"><ins class="jstree-icon">&nbsp;</ins>PartheenMetaData (1)</a></li><li packagelist="pedbarrayv10.db" id="pedbarrayv10" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="https://www.bioconductor.org/packages/release/BiocViews.html#"><ins class="jstree-icon">&nbsp;</ins>pedbarrayv10 (1)</a></li><li packagelist="pedbarrayv9.db" id="pedbarrayv9" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="https://www.bioconductor.org/packages/release/BiocViews.html#"><ins class="jstree-icon">&nbsp;</ins>pedbarrayv9 (1)</a></li><li packagelist="r10kcod.db" id="r10kcod" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="https://www.bioconductor.org/packages/release/BiocViews.html#"><ins class="jstree-icon">&nbsp;</ins>r10kcod (1)</a></li><li packagelist="rae230a.db,rae230acdf,rae230aprobe" id="rae230a" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="https://www.bioconductor.org/packages/release/BiocViews.html#"><ins class="jstree-icon">&nbsp;</ins>rae230a (3)</a></li><li packagelist="rae230b.db,rae230bcdf,rae230bprobe" id="rae230b" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="https://www.bioconductor.org/packages/release/BiocViews.html#"><ins class="jstree-icon">&nbsp;</ins>rae230b (3)</a></li><li packagelist="rat2302.db,rat2302cdf,rat2302probe" id="rat2302" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="https://www.bioconductor.org/packages/release/BiocViews.html#"><ins class="jstree-icon">&nbsp;</ins>rat2302 (3)</a></li><li packagelist="rgu34a.db,rgu34acdf,rgu34aprobe" id="rgu34a" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="https://www.bioconductor.org/packages/release/BiocViews.html#"><ins class="jstree-icon">&nbsp;</ins>rgu34a (3)</a></li><li packagelist="rgu34b.db,rgu34bcdf,rgu34bprobe" id="rgu34b" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="https://www.bioconductor.org/packages/release/BiocViews.html#"><ins class="jstree-icon">&nbsp;</ins>rgu34b (3)</a></li><li packagelist="rgu34c.db,rgu34ccdf,rgu34cprobe" id="rgu34c" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="https://www.bioconductor.org/packages/release/BiocViews.html#"><ins class="jstree-icon">&nbsp;</ins>rgu34c (3)</a></li><li packagelist="rgug4130a.db" id="rgug4130a" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="https://www.bioconductor.org/packages/release/BiocViews.html#"><ins class="jstree-icon">&nbsp;</ins>rgug4130a (1)</a></li><li packagelist="ri16cod.db" id="ri16cod" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="https://www.bioconductor.org/packages/release/BiocViews.html#"><ins class="jstree-icon">&nbsp;</ins>ri16cod (1)</a></li><li packagelist="rnu34.db,rnu34cdf,rnu34probe" id="rnu34" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="https://www.bioconductor.org/packages/release/BiocViews.html#"><ins class="jstree-icon">&nbsp;</ins>rnu34 (3)</a></li><li packagelist="Roberts2005Annotation.db" id="Roberts2005Annotation" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="https://www.bioconductor.org/packages/release/BiocViews.html#"><ins class="jstree-icon">&nbsp;</ins>Roberts2005Annotation (1)</a></li><li packagelist="rtu34.db,rtu34cdf,rtu34probe" id="rtu34" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="https://www.bioconductor.org/packages/release/BiocViews.html#"><ins class="jstree-icon">&nbsp;</ins>rtu34 (3)</a></li><li packagelist="rwgcod.db" id="rwgcod" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="https://www.bioconductor.org/packages/release/BiocViews.html#"><ins class="jstree-icon">&nbsp;</ins>rwgcod (1)</a></li><li packagelist="SHDZ.db" id="SHDZ" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="https://www.bioconductor.org/packages/release/BiocViews.html#"><ins class="jstree-icon">&nbsp;</ins>SHDZ (1)</a></li><li packagelist="u133x3p.db,u133x3pcdf,u133x3pprobe" id="u133x3p" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="https://www.bioconductor.org/packages/release/BiocViews.html#"><ins class="jstree-icon">&nbsp;</ins>u133x3p (3)</a></li><li packagelist="xenopuslaeviscdf,xenopuslaevisprobe" id="xenopuslaevis" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="https://www.bioconductor.org/packages/release/BiocViews.html#"><ins class="jstree-icon">&nbsp;</ins>xenopuslaevis (2)</a></li><li packagelist="yeast2.db,yeast2cdf,yeast2probe" id="yeast2" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="https://www.bioconductor.org/packages/release/BiocViews.html#"><ins class="jstree-icon">&nbsp;</ins>yeast2 (3)</a></li><li packagelist="ygs98.db,ygs98cdf,ygs98frmavecs,ygs98probe" id="ygs98" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="https://www.bioconductor.org/packages/release/BiocViews.html#"><ins class="jstree-icon">&nbsp;</ins>ygs98 (4)</a></li><li packagelist="zebrafish.db,zebrafishcdf,zebrafishprobe" id="zebrafish" class="jstree-last jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="https://www.bioconductor.org/packages/release/BiocViews.html#"><ins class="jstree-icon">&nbsp;</ins>zebrafish (3)</a></li></ul></li><li packagelist="hgubeta7.db,hguDKFZ31.db" id="CustomArray" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="https://www.bioconductor.org/packages/release/BiocViews.html#"><ins class="jstree-icon">&nbsp;</ins>CustomArray (2)</a></li><li class="jstree-closed" style="display: none;"><ins class="jstree-icon">&nbsp;</ins><a href="https://www.bioconductor.org/packages/release/BiocViews.html#"><ins class="jstree-icon">&nbsp;</ins>CustomCDF</a><ul><li class="jstree-leaf" style="display: none;"><ins class="jstree-icon">&nbsp;</ins><a href="https://www.bioconductor.org/packages/release/BiocViews.html#"><ins class="jstree-icon">&nbsp;</ins>GACustomCDF</a></li><li class="jstree-last jstree-leaf" style="display: none;"><ins class="jstree-icon">&nbsp;</ins><a href="https://www.bioconductor.org/packages/release/BiocViews.html#"><ins class="jstree-icon">&nbsp;</ins>MBNICustomCDF</a></li></ul></li><li packagelist="ChemmineDrugs,RmiR.Hs.miRNA,RmiR.hsa" id="CustomDBSchema" class="jstree-closed"><ins class="jstree-icon">&nbsp;</ins><a href="https://www.bioconductor.org/packages/release/BiocViews.html#"><ins class="jstree-icon">&nbsp;</ins>CustomDBSchema (3)</a><ul><li class="jstree-last jstree-leaf" style="display: none;"><ins class="jstree-icon">&nbsp;</ins><a href="https://www.bioconductor.org/packages/release/BiocViews.html#"><ins class="jstree-icon">&nbsp;</ins>GeneCardsCustomSchema</a></li></ul></li><li packagelist="DO.db,geneplast.data.string.v91,GO.db,humanCHRLOC,JASPAR2018,KEGG.db,LowMACAAnnotation,LRBase.Ath.eg.db,LRBase.Bta.eg.db,LRBase.Cel.eg.db,LRBase.Dme.eg.db,LRBase.Dre.eg.db,LRBase.Gga.eg.db,LRBase.Hsa.eg.db,LRBase.Mmu.eg.db,LRBase.Pab.eg.db,LRBase.Rno.eg.db,LRBase.Ssc.eg.db,LRBase.Xtr.eg.db,MeSH.AOR.db,MeSH.db,MeSH.PCR.db,mirbase.db,mouseCHRLOC,ratCHRLOC,reactome.db,targetscan.Hs.eg.db,targetscan.Mm.eg.db" id="FunctionalAnnotation" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="https://www.bioconductor.org/packages/release/BiocViews.html#"><ins class="jstree-icon">&nbsp;</ins>FunctionalAnnotation (28)</a></li><li packagelist="GenomeInfoDbData" id="Organism" class="jstree-closed"><ins class="jstree-icon">&nbsp;</ins><a href="https://www.bioconductor.org/packages/release/BiocViews.html#"><ins class="jstree-icon">&nbsp;</ins>Organism (610)</a><ul><li packagelist="anopheles.db0,org.Ag.eg.db,plasmodiumanophelescdf,plasmodiumanophelesprobe" id="Anopheles_gambiae" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="https://www.bioconductor.org/packages/release/BiocViews.html#"><ins class="jstree-icon">&nbsp;</ins>Anopheles_gambiae (4)</a></li><li packagelist="BSgenome.Amellifera.BeeBase.assembly4,BSgenome.Amellifera.UCSC.apiMel2,BSgenome.Amellifera.UCSC.apiMel2.masked" id="Apis_mellifera" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="https://www.bioconductor.org/packages/release/BiocViews.html#"><ins class="jstree-icon">&nbsp;</ins>Apis_mellifera (3)</a></li><li class="jstree-leaf" style="display: none;"><ins class="jstree-icon">&nbsp;</ins><a href="https://www.bioconductor.org/packages/release/BiocViews.html#"><ins class="jstree-icon">&nbsp;</ins>Arabidopsis_lyrata</a></li><li packagelist="ag.db,agcdf,agprobe,arabidopsis.db0,ath1121501.db,ath1121501cdf,ath1121501probe,BSgenome.Athaliana.TAIR.04232008,BSgenome.Athaliana.TAIR.TAIR9,hom.At.inp.db,org.At.tair.db,TxDb.Athaliana.BioMart.plantsmart22,TxDb.Athaliana.BioMart.plantsmart25,TxDb.Athaliana.BioMart.plantsmart28" id="Arabidopsis_thaliana" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="https://www.bioconductor.org/packages/release/BiocViews.html#"><ins class="jstree-icon">&nbsp;</ins>Arabidopsis_thaliana (14)</a></li><li packagelist="BSgenome.Aofficinalis.NCBI.V1" id="Asparagus_officinalis" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="https://www.bioconductor.org/packages/release/BiocViews.html#"><ins class="jstree-icon">&nbsp;</ins>Asparagus_officinalis (1)</a></li><li packagelist="bsubtiliscdf,bsubtilisprobe" id="Bacillus_subtilis" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="https://www.bioconductor.org/packages/release/BiocViews.html#"><ins class="jstree-icon">&nbsp;</ins>Bacillus_subtilis (2)</a></li><li packagelist="bovine.db,bovine.db0,bovinecdf,bovineprobe,BSgenome.Btaurus.UCSC.bosTau3,BSgenome.Btaurus.UCSC.bosTau3.masked,BSgenome.Btaurus.UCSC.bosTau4,BSgenome.Btaurus.UCSC.bosTau4.masked,BSgenome.Btaurus.UCSC.bosTau6,BSgenome.Btaurus.UCSC.bosTau6.masked,BSgenome.Btaurus.UCSC.bosTau8,org.Bt.eg.db,TxDb.Btaurus.UCSC.bosTau8.refGene" id="Bos_taurus" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="https://www.bioconductor.org/packages/release/BiocViews.html#"><ins class="jstree-icon">&nbsp;</ins>Bos_taurus (13)</a></li><li packagelist="BSgenome.Celegans.UCSC.ce10,BSgenome.Celegans.UCSC.ce11,BSgenome.Celegans.UCSC.ce2,BSgenome.Celegans.UCSC.ce6,celegans.db,celeganscdf,celegansprobe,hom.Ce.inp.db,org.Ce.eg.db,TxDb.Celegans.UCSC.ce11.refGene,TxDb.Celegans.UCSC.ce6.ensGene,worm.db0" id="Caenorhabditis_elegans" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="https://www.bioconductor.org/packages/release/BiocViews.html#"><ins class="jstree-icon">&nbsp;</ins>Caenorhabditis_elegans (12)</a></li><li packagelist="BSgenome.Cfamiliaris.UCSC.canFam2,BSgenome.Cfamiliaris.UCSC.canFam2.masked,BSgenome.Cfamiliaris.UCSC.canFam3,BSgenome.Cfamiliaris.UCSC.canFam3.masked,canine.db,canine.db0,canine2.db,canine2cdf,canine2probe,caninecdf,canineprobe,org.Cf.eg.db,TxDb.Cfamiliaris.UCSC.canFam3.refGene" id="Canis_familiaris" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="https://www.bioconductor.org/packages/release/BiocViews.html#"><ins class="jstree-icon">&nbsp;</ins>Canis_familiaris (13)</a></li><li packagelist="BSgenome.Carietinum.NCBI.v1" id="Cicer_arietinum" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="https://www.bioconductor.org/packages/release/BiocViews.html#"><ins class="jstree-icon">&nbsp;</ins>Cicer_arietinum (1)</a></li><li class="jstree-leaf" style="display: none;"><ins class="jstree-icon">&nbsp;</ins><a href="https://www.bioconductor.org/packages/release/BiocViews.html#"><ins class="jstree-icon">&nbsp;</ins>Ciona_intestinalis</a></li><li packagelist="BSgenome.Drerio.UCSC.danRer10,BSgenome.Drerio.UCSC.danRer5,BSgenome.Drerio.UCSC.danRer5.masked,BSgenome.Drerio.UCSC.danRer6,BSgenome.Drerio.UCSC.danRer6.masked,BSgenome.Drerio.UCSC.danRer7,BSgenome.Drerio.UCSC.danRer7.masked,hom.Dr.inp.db,org.Dr.eg.db,TxDb.Drerio.UCSC.danRer10.refGene,zebrafish.db,zebrafish.db0,zebrafishcdf,zebrafishprobe" id="Danio_rerio" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="https://www.bioconductor.org/packages/release/BiocViews.html#"><ins class="jstree-icon">&nbsp;</ins>Danio_rerio (14)</a></li><li packagelist="BSgenome.Dmelanogaster.UCSC.dm2,BSgenome.Dmelanogaster.UCSC.dm2.masked,BSgenome.Dmelanogaster.UCSC.dm3,BSgenome.Dmelanogaster.UCSC.dm3.masked,BSgenome.Dmelanogaster.UCSC.dm6,drosgenome1.db,drosgenome1cdf,drosgenome1probe,drosophila2.db,drosophila2cdf,drosophila2probe,fly.db0,hom.Dm.inp.db,indac.db,org.Dm.eg.db,TxDb.Dmelanogaster.UCSC.dm3.ensGene,TxDb.Dmelanogaster.UCSC.dm6.ensGene" id="Drosophila_melanogaster" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="https://www.bioconductor.org/packages/release/BiocViews.html#"><ins class="jstree-icon">&nbsp;</ins>Drosophila_melanogaster (17)</a></li><li class="jstree-leaf" style="display: none;"><ins class="jstree-icon">&nbsp;</ins><a href="https://www.bioconductor.org/packages/release/BiocViews.html#"><ins class="jstree-icon">&nbsp;</ins>Eremothecium_gossypii</a></li><li packagelist="BSgenome.Ecoli.NCBI.20080805,ecoli2.db,ecoli2cdf,ecoli2probe,ecoliasv2cdf,ecoliasv2probe,ecolicdf,ecoliK12.db0,ecoliprobe,ecoliSakai.db0,org.EcK12.eg.db,org.EcSakai.eg.db" id="Escherichia_coli" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="https://www.bioconductor.org/packages/release/BiocViews.html#"><ins class="jstree-icon">&nbsp;</ins>Escherichia_coli (12)</a></li><li packagelist="BSgenome.Ggallus.UCSC.galGal3,BSgenome.Ggallus.UCSC.galGal3.masked,BSgenome.Ggallus.UCSC.galGal4,BSgenome.Ggallus.UCSC.galGal4.masked,BSgenome.Ggallus.UCSC.galGal5,chicken.db,chicken.db0,chickencdf,chickenprobe,org.Gg.eg.db,TxDb.Ggallus.UCSC.galGal4.refGene,TxDb.Ggallus.UCSC.galGal5.refGene" id="Gallus_gallus" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="https://www.bioconductor.org/packages/release/BiocViews.html#"><ins class="jstree-icon">&nbsp;</ins>Gallus_gallus (12)</a></li><li packagelist="BSgenome.Gaculeatus.UCSC.gasAcu1,BSgenome.Gaculeatus.UCSC.gasAcu1.masked" id="Gasterosteus_aculeatus" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="https://www.bioconductor.org/packages/release/BiocViews.html#"><ins class="jstree-icon">&nbsp;</ins>Gasterosteus_aculeatus (2)</a></li><li class="jstree-leaf" style="display: none;"><ins class="jstree-icon">&nbsp;</ins><a href="https://www.bioconductor.org/packages/release/BiocViews.html#"><ins class="jstree-icon">&nbsp;</ins>Glycine_max</a></li><li packagelist="BSgenome.Hsapiens.1000genomes.hs37d5,BSgenome.Hsapiens.NCBI.GRCh38,BSgenome.Hsapiens.UCSC.hg17,BSgenome.Hsapiens.UCSC.hg17.masked,BSgenome.Hsapiens.UCSC.hg18,BSgenome.Hsapiens.UCSC.hg18.masked,BSgenome.Hsapiens.UCSC.hg19,BSgenome.Hsapiens.UCSC.hg19.masked,BSgenome.Hsapiens.UCSC.hg38,BSgenome.Hsapiens.UCSC.hg38.masked,clariomdhumanprobeset.db,clariomdhumantranscriptcluster.db,clariomshumanhttranscriptcluster.db,clariomshumantranscriptcluster.db,cyp450cdf,EnsDb.Hsapiens.v75,EnsDb.Hsapiens.v79,EnsDb.Hsapiens.v86,FDb.FANTOM4.promoters.hg19,FDb.InfiniumMethylation.hg18,FDb.InfiniumMethylation.hg19,FDb.UCSC.snp135common.hg19,FDb.UCSC.snp137common.hg19,FDb.UCSC.tRNAs,fitCons.UCSC.hg19,GGHumanMethCancerPanelv1.db,gp53cdf,h10kcod.db,h20kcod.db,hcg110.db,hcg110cdf,hcg110probe,hgfocus.db,hgfocuscdf,hgfocusprobe,hgu133a.db,hgu133a2.db,hgu133a2cdf,hgu133a2frmavecs,hgu133a2probe,hgu133acdf,hgu133afrmavecs,hgu133aprobe,hgu133atagcdf,hgu133atagprobe,hgu133b.db,hgu133bcdf,hgu133bprobe,hgu133plus2.db,hgu133plus2cdf,hgu133plus2frmavecs,hgu133plus2probe,hgu219.db,hgu219cdf,hgu219probe,hgu95a.db,hgu95acdf,hgu95aprobe,hgu95av2,hgu95av2.db,hgu95av2cdf,hgu95av2probe,hgu95b.db,hgu95bcdf,hgu95bprobe,hgu95c.db,hgu95ccdf,hgu95cprobe,hgu95d.db,hgu95dcdf,hgu95dprobe,hgu95e.db,hgu95ecdf,hgu95eprobe,hguatlas13k.db,hgubeta7.db,hguDKFZ31.db,hgug4100a.db,hgug4101a.db,hgug4110b.db,hgug4111a.db,hgug4112a.db,hgug4845a.db,hguqiagenv3.db,hi16cod.db,hivprtplus2cdf,hom.Hs.inp.db,Homo.sapiens,hs25kresogen.db,Hs6UG171.db,HsAgilentDesign026652.db,Hspec,hspeccdf,hta20probeset.db,hta20transcriptcluster.db,hthgu133a.db,hthgu133acdf,hthgu133afrmavecs,hthgu133aprobe,hthgu133b.db,hthgu133bcdf,hthgu133bprobe,hthgu133pluspmcdf,hthgu133pluspmprobe,hu35ksuba.db,hu35ksubacdf,hu35ksubaprobe,hu35ksubb.db,hu35ksubbcdf,hu35ksubbprobe,hu35ksubc.db,hu35ksubccdf,hu35ksubcprobe,hu35ksubd.db,hu35ksubdcdf,hu35ksubdprobe,hu6800.db,hu6800cdf,hu6800probe,hu6800subacdf,hu6800subbcdf,hu6800subccdf,hu6800subdcdf,huex.1.0.st.v2frmavecs,huex10stprobeset.db,huex10sttranscriptcluster.db,HuExExonProbesetLocation,HuExExonProbesetLocationHg18,HuExExonProbesetLocationHg19,hugene.1.0.st.v1frmavecs,hugene10stprobeset.db,hugene10sttranscriptcluster.db,hugene10stv1cdf,hugene10stv1probe,hugene11stprobeset.db,hugene11sttranscriptcluster.db,hugene20stprobeset.db,hugene20sttranscriptcluster.db,hugene21stprobeset.db,hugene21sttranscriptcluster.db,human.db0,humanCHRLOC,HuO22.db,hwgcod.db,IlluminaHumanMethylation27k.db,IlluminaHumanMethylation27kanno.ilmn12.hg19,IlluminaHumanMethylation27kmanifest,IlluminaHumanMethylation450kanno.ilmn12.hg19,IlluminaHumanMethylation450kmanifest,IlluminaHumanMethylation450kprobe,IlluminaHumanMethylationEPICanno.ilm10b2.hg19,IlluminaHumanMethylationEPICanno.ilm10b3.hg19,IlluminaHumanMethylationEPICanno.ilm10b4.hg19,IlluminaHumanMethylationEPICmanifest,illuminaHumanv1.db,illuminaHumanv2.db,illuminaHumanv2BeadID.db,illuminaHumanv3.db,illuminaHumanv4.db,illuminaHumanWGDASLv3.db,illuminaHumanWGDASLv4.db,JazaeriMetaData.db,LAPOINTE.db,lumiHumanAll.db,MafDb.1Kgenomes.phase1.GRCh38,MafDb.1Kgenomes.phase1.hs37d5,MafDb.1Kgenomes.phase3.GRCh38,MafDb.1Kgenomes.phase3.hs37d5,MafDb.ESP6500SI.V2.SSA137.GRCh38,MafDb.ESP6500SI.V2.SSA137.hs37d5,MafDb.ExAC.r1.0.GRCh38,MafDb.ExAC.r1.0.hs37d5,MafDb.ExAC.r1.0.nonTCGA.GRCh38,MafDb.ExAC.r1.0.nonTCGA.hs37d5,MafDb.gnomAD.r2.0.1.GRCh38,MafDb.gnomAD.r2.0.1.hs37d5,MafDb.gnomAD.r2.1.hs37d5,MafDb.gnomADex.r2.0.1.GRCh38,MafDb.gnomADex.r2.0.1.hs37d5,MafDb.gnomADex.r2.1.hs37d5,MafDb.TOPMed.freeze5.hg19,MafDb.TOPMed.freeze5.hg38,mirna102xgaincdf,mirna10cdf,mirna10probe,mirna20cdf,mogene.1.0.st.v1frmavecs,Norway981.db,nugohs1a520180.db,nugohs1a520180cdf,nugohs1a520180probe,OperonHumanV3.db,org.Hs.eg.db,PartheenMetaData.db,pedbarrayv10.db,pedbarrayv9.db,phastCons100way.UCSC.hg19,phastCons100way.UCSC.hg38,phastCons7way.UCSC.hg38,POCRCannotation.db,PolyPhen.Hsapiens.dbSNP131,primeviewcdf,primeviewprobe,RmiR.Hs.miRNA,RmiR.hsa,Roberts2005Annotation.db,SHDZ.db,SIFT.Hsapiens.dbSNP132,SIFT.Hsapiens.dbSNP137,SNPlocs.Hsapiens.dbSNP.20101109,SNPlocs.Hsapiens.dbSNP.20120608,SNPlocs.Hsapiens.dbSNP141.GRCh38,SNPlocs.Hsapiens.dbSNP142.GRCh37,SNPlocs.Hsapiens.dbSNP144.GRCh37,SNPlocs.Hsapiens.dbSNP144.GRCh38,SNPlocs.Hsapiens.dbSNP149.GRCh38,SNPlocs.Hsapiens.dbSNP150.GRCh38,SNPlocs.Hsapiens.dbSNP151.GRCh38,test1cdf,test2cdf,test3cdf,test3probe,TxDb.Hsapiens.BioMart.igis,TxDb.Hsapiens.UCSC.hg18.knownGene,TxDb.Hsapiens.UCSC.hg19.knownGene,TxDb.Hsapiens.UCSC.hg19.lincRNAsTranscripts,TxDb.Hsapiens.UCSC.hg38.knownGene,u133aaofav2cdf,u133x3p.db,u133x3pcdf,u133x3pprobe,XtraSNPlocs.Hsapiens.dbSNP141.GRCh38,XtraSNPlocs.Hsapiens.dbSNP144.GRCh37,XtraSNPlocs.Hsapiens.dbSNP144.GRCh38" id="Homo_sapiens" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="https://www.bioconductor.org/packages/release/BiocViews.html#"><ins class="jstree-icon">&nbsp;</ins>Homo_sapiens (234)</a></li><li packagelist="barley1cdf,barley1probe" id="Hordeum_vulgare" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="https://www.bioconductor.org/packages/release/BiocViews.html#"><ins class="jstree-icon">&nbsp;</ins>Hordeum_vulgare (2)</a></li><li class="jstree-leaf" style="display: none;"><ins class="jstree-icon">&nbsp;</ins><a href="https://www.bioconductor.org/packages/release/BiocViews.html#"><ins class="jstree-icon">&nbsp;</ins>Kluyveromyces_lactis</a></li><li packagelist="BSgenome.Mfascicularis.NCBI.5.0" id="Macaca_fascicularis" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="https://www.bioconductor.org/packages/release/BiocViews.html#"><ins class="jstree-icon">&nbsp;</ins>Macaca_fascicularis (1)</a></li><li packagelist="BSgenome.Mmulatta.UCSC.rheMac2,BSgenome.Mmulatta.UCSC.rheMac2.masked,BSgenome.Mmulatta.UCSC.rheMac3,BSgenome.Mmulatta.UCSC.rheMac3.masked,BSgenome.Mmulatta.UCSC.rheMac8,org.Mmu.eg.db,rhesuscdf,rhesusprobe,TxDb.Mmulatta.UCSC.rheMac3.refGene,TxDb.Mmulatta.UCSC.rheMac8.refGene" id="Macaca_mulatta" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="https://www.bioconductor.org/packages/release/BiocViews.html#"><ins class="jstree-icon">&nbsp;</ins>Macaca_mulatta (10)</a></li><li class="jstree-leaf" style="display: none;"><ins class="jstree-icon">&nbsp;</ins><a href="https://www.bioconductor.org/packages/release/BiocViews.html#"><ins class="jstree-icon">&nbsp;</ins>Magnaporthe_grisea</a></li><li class="jstree-leaf" style="display: none;"><ins class="jstree-icon">&nbsp;</ins><a href="https://www.bioconductor.org/packages/release/BiocViews.html#"><ins class="jstree-icon">&nbsp;</ins>Medicago_truncatula</a></li><li packagelist="BSgenome.Mmusculus.UCSC.mm10,BSgenome.Mmusculus.UCSC.mm10.masked,BSgenome.Mmusculus.UCSC.mm8,BSgenome.Mmusculus.UCSC.mm8.masked,BSgenome.Mmusculus.UCSC.mm9,BSgenome.Mmusculus.UCSC.mm9.masked,clariomsmousehttranscriptcluster.db,clariomsmousetranscriptcluster.db,EnsDb.Mmusculus.v75,EnsDb.Mmusculus.v79,hom.Mm.inp.db,htmg430acdf,htmg430aprobe,htmg430bcdf,htmg430bprobe,htmg430pmcdf,htmg430pmprobe,illuminaMousev1.db,illuminaMousev1p1.db,illuminaMousev2.db,lumiMouseAll.db,m10kcod.db,m20kcod.db,mgu74a.db,mgu74acdf,mgu74aprobe,mgu74av2.db,mgu74av2cdf,mgu74av2probe,mgu74b.db,mgu74bcdf,mgu74bprobe,mgu74bv2.db,mgu74bv2cdf,mgu74bv2probe,mgu74c.db,mgu74ccdf,mgu74cprobe,mgu74cv2.db,mgu74cv2cdf,mgu74cv2probe,mguatlas5k.db,mgug4104a.db,mgug4120a.db,mgug4121a.db,mgug4122a.db,mi16cod.db,mm24kresogen.db,MmAgilentDesign026655.db,moe430a.db,moe430acdf,moe430aprobe,moe430b.db,moe430bcdf,moe430bprobe,moex10stprobeset.db,moex10sttranscriptcluster.db,mogene10stprobeset.db,mogene10sttranscriptcluster.db,mogene10stv1cdf,mogene10stv1probe,mogene11stprobeset.db,mogene11sttranscriptcluster.db,mogene20stprobeset.db,mogene20sttranscriptcluster.db,mogene21stprobeset.db,mogene21sttranscriptcluster.db,mouse.db0,mouse4302.db,mouse4302cdf,mouse4302frmavecs,mouse4302probe,mouse430a2.db,mouse430a2cdf,mouse430a2frmavecs,mouse430a2probe,mouseCHRLOC,mpedbarray.db,mta10probeset.db,mta10transcriptcluster.db,mu11ksuba.db,mu11ksubacdf,mu11ksubaprobe,mu11ksubb.db,mu11ksubbcdf,mu11ksubbprobe,Mu15v1.db,mu19ksuba.db,mu19ksubacdf,mu19ksubb.db,mu19ksubbcdf,mu19ksubc.db,mu19ksubccdf,Mu22v3.db,mu6500subacdf,mu6500subbcdf,mu6500subccdf,mu6500subdcdf,Mus.musculus,mwgcod.db,nugomm1a520177.db,nugomm1a520177cdf,nugomm1a520177probe,org.Mm.eg.db,TxDb.Mmusculus.UCSC.mm10.ensGene,TxDb.Mmusculus.UCSC.mm10.knownGene,TxDb.Mmusculus.UCSC.mm9.knownGene" id="Mus_musculus" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="https://www.bioconductor.org/packages/release/BiocViews.html#"><ins class="jstree-icon">&nbsp;</ins>Mus_musculus (107)</a></li><li class="jstree-leaf" style="display: none;"><ins class="jstree-icon">&nbsp;</ins><a href="https://www.bioconductor.org/packages/release/BiocViews.html#"><ins class="jstree-icon">&nbsp;</ins>Neurospora_crassa</a></li><li class="jstree-leaf" style="display: none;"><ins class="jstree-icon">&nbsp;</ins><a href="https://www.bioconductor.org/packages/release/BiocViews.html#"><ins class="jstree-icon">&nbsp;</ins>Oncorhynchus_mykiss</a></li><li packagelist="BSgenome.Osativa.MSU.MSU7" id="Oryza_sativa" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="https://www.bioconductor.org/packages/release/BiocViews.html#"><ins class="jstree-icon">&nbsp;</ins>Oryza_sativa (1)</a></li><li packagelist="BSgenome.Ptroglodytes.UCSC.panTro2,BSgenome.Ptroglodytes.UCSC.panTro2.masked,BSgenome.Ptroglodytes.UCSC.panTro3,BSgenome.Ptroglodytes.UCSC.panTro3.masked,BSgenome.Ptroglodytes.UCSC.panTro5,BSgenome.Ptroglodytes.UCSC.panTro6,chimp.db0,org.Pt.eg.db,TxDb.Ptroglodytes.UCSC.panTro4.refGene,TxDb.Ptroglodytes.UCSC.panTro5.refGene" id="Pan_troglodytes" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="https://www.bioconductor.org/packages/release/BiocViews.html#"><ins class="jstree-icon">&nbsp;</ins>Pan_troglodytes (10)</a></li><li packagelist="malaria.db0,org.Pf.plasmo.db,plasmodiumanophelescdf,plasmodiumanophelesprobe" id="Plasmodium_falciparum" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="https://www.bioconductor.org/packages/release/BiocViews.html#"><ins class="jstree-icon">&nbsp;</ins>Plasmodium_falciparum (4)</a></li><li packagelist="paeg1acdf,paeg1aprobe" id="Pseudomonas_aeruginosa" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="https://www.bioconductor.org/packages/release/BiocViews.html#"><ins class="jstree-icon">&nbsp;</ins>Pseudomonas_aeruginosa (2)</a></li><li packagelist="adme16cod.db,BSgenome.Rnorvegicus.UCSC.rn4,BSgenome.Rnorvegicus.UCSC.rn4.masked,BSgenome.Rnorvegicus.UCSC.rn5,BSgenome.Rnorvegicus.UCSC.rn5.masked,BSgenome.Rnorvegicus.UCSC.rn6,clariomsrathttranscriptcluster.db,clariomsrattranscriptcluster.db,EnsDb.Rnorvegicus.v75,EnsDb.Rnorvegicus.v79,hom.Rn.inp.db,htrat230pmcdf,htrat230pmprobe,htratfocuscdf,htratfocusprobe,illuminaRatv1.db,lumiRatAll.db,org.Rn.eg.db,r10kcod.db,rae230a.db,rae230acdf,rae230aprobe,rae230b.db,rae230bcdf,rae230bprobe,raex10stprobeset.db,raex10sttranscriptcluster.db,ragene10stprobeset.db,ragene10sttranscriptcluster.db,ragene10stv1cdf,ragene10stv1probe,ragene11stprobeset.db,ragene11sttranscriptcluster.db,ragene20stprobeset.db,ragene20sttranscriptcluster.db,ragene21stprobeset.db,ragene21sttranscriptcluster.db,rat.db0,rat2302.db,rat2302cdf,rat2302probe,ratCHRLOC,rattoxfxcdf,rattoxfxprobe,Rattus.norvegicus,rgu34a.db,rgu34acdf,rgu34aprobe,rgu34b.db,rgu34bcdf,rgu34bprobe,rgu34c.db,rgu34ccdf,rgu34cprobe,rguatlas4k.db,rgug4105a.db,rgug4130a.db,rgug4131a.db,ri16cod.db,RnAgilentDesign028282.db,rnu34.db,rnu34cdf,rnu34probe,rta10probeset.db,rta10transcriptcluster.db,rtu34.db,rtu34cdf,rtu34probe,rwgcod.db,TxDb.Rnorvegicus.BioMart.igis,TxDb.Rnorvegicus.UCSC.rn4.ensGene,TxDb.Rnorvegicus.UCSC.rn5.refGene,TxDb.Rnorvegicus.UCSC.rn6.refGene" id="Rattus_norvegicus" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="https://www.bioconductor.org/packages/release/BiocViews.html#"><ins class="jstree-icon">&nbsp;</ins>Rattus_norvegicus (73)</a></li><li packagelist="BSgenome.Scerevisiae.UCSC.sacCer1,BSgenome.Scerevisiae.UCSC.sacCer2,BSgenome.Scerevisiae.UCSC.sacCer3,hom.Sc.inp.db,org.Sc.sgd.db,TxDb.Scerevisiae.UCSC.sacCer2.sgdGene,TxDb.Scerevisiae.UCSC.sacCer3.sgdGene,ye6100subacdf,ye6100subbcdf,ye6100subccdf,ye6100subdcdf,yeast.db0,yeast2.db,yeast2cdf,yeast2probe,ygs98.db,ygs98cdf,ygs98probe" id="Saccharomyces_cerevisiae" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="https://www.bioconductor.org/packages/release/BiocViews.html#"><ins class="jstree-icon">&nbsp;</ins>Saccharomyces_cerevisiae (18)</a></li><li packagelist="sugarcanecdf,sugarcaneprobe" id="Saccharum_officinarum" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="https://www.bioconductor.org/packages/release/BiocViews.html#"><ins class="jstree-icon">&nbsp;</ins>Saccharum_officinarum (2)</a></li><li class="jstree-leaf" style="display: none;"><ins class="jstree-icon">&nbsp;</ins><a href="https://www.bioconductor.org/packages/release/BiocViews.html#"><ins class="jstree-icon">&nbsp;</ins>Schizosaccharomyces_pombe</a></li><li packagelist="saureuscdf,saureusprobe" id="Staphylococcus_aureus" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="https://www.bioconductor.org/packages/release/BiocViews.html#"><ins class="jstree-icon">&nbsp;</ins>Staphylococcus_aureus (2)</a></li><li packagelist="BSgenome.Sscrofa.UCSC.susScr11,BSgenome.Sscrofa.UCSC.susScr3,BSgenome.Sscrofa.UCSC.susScr3.masked,org.Ss.eg.db,pig.db0,porcine.db,porcinecdf,porcineprobe,TxDb.Sscrofa.UCSC.susScr11.refGene,TxDb.Sscrofa.UCSC.susScr3.refGene" id="Sus_scrofa" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="https://www.bioconductor.org/packages/release/BiocViews.html#"><ins class="jstree-icon">&nbsp;</ins>Sus_scrofa (10)</a></li><li packagelist="BSgenome.Tguttata.UCSC.taeGut1,BSgenome.Tguttata.UCSC.taeGut1.masked,BSgenome.Tguttata.UCSC.taeGut2" id="Taeniopygia_guttata" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="https://www.bioconductor.org/packages/release/BiocViews.html#"><ins class="jstree-icon">&nbsp;</ins>Taeniopygia_guttata (3)</a></li><li packagelist="BSgenome.Tgondii.ToxoDB.7.0" id="Toxoplasma_gondii" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="https://www.bioconductor.org/packages/release/BiocViews.html#"><ins class="jstree-icon">&nbsp;</ins>Toxoplasma_gondii (1)</a></li><li class="jstree-leaf" style="display: none;"><ins class="jstree-icon">&nbsp;</ins><a href="https://www.bioconductor.org/packages/release/BiocViews.html#"><ins class="jstree-icon">&nbsp;</ins>Triticum_aestivum</a></li><li packagelist="BSgenome.Vvinifera.URGI.IGGP12Xv0,BSgenome.Vvinifera.URGI.IGGP12Xv2,BSgenome.Vvinifera.URGI.IGGP8X,vitisviniferacdf,vitisviniferaprobe" id="Vitis_vinifera" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="https://www.bioconductor.org/packages/release/BiocViews.html#"><ins class="jstree-icon">&nbsp;</ins>Vitis_vinifera (5)</a></li><li packagelist="org.Xl.eg.db,xenopus.db0,xlaevis.db" id="Xenopus_laevis" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="https://www.bioconductor.org/packages/release/BiocViews.html#"><ins class="jstree-icon">&nbsp;</ins>Xenopus_laevis (3)</a></li><li packagelist="xtropicalisprobe" id="Xenopus_tropicalis" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="https://www.bioconductor.org/packages/release/BiocViews.html#"><ins class="jstree-icon">&nbsp;</ins>Xenopus_tropicalis (1)</a></li><li packagelist="maizecdf,maizeprobe" id="Zea_mays" class="jstree-last jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="https://www.bioconductor.org/packages/release/BiocViews.html#"><ins class="jstree-icon">&nbsp;</ins>Zea_mays (2)</a></li></ul></li><li packagelist="adme16cod.db,ag.db,agcdf,agprobe,AHEnsDbs,anopheles.db0,arabidopsis.db0,ath1121501.db,ath1121501cdf,ath1121501probe,barley1cdf,barley1probe,bovine.db,bovine.db0,bovinecdf,bovineprobe,BSgenome.Amellifera.BeeBase.assembly4,BSgenome.Amellifera.UCSC.apiMel2,BSgenome.Amellifera.UCSC.apiMel2.masked,BSgenome.Aofficinalis.NCBI.V1,BSgenome.Athaliana.TAIR.04232008,BSgenome.Athaliana.TAIR.TAIR9,BSgenome.Btaurus.UCSC.bosTau3,BSgenome.Btaurus.UCSC.bosTau3.masked,BSgenome.Btaurus.UCSC.bosTau4,BSgenome.Btaurus.UCSC.bosTau4.masked,BSgenome.Btaurus.UCSC.bosTau6,BSgenome.Btaurus.UCSC.bosTau6.masked,BSgenome.Btaurus.UCSC.bosTau8,BSgenome.Carietinum.NCBI.v1,BSgenome.Celegans.UCSC.ce10,BSgenome.Celegans.UCSC.ce11,BSgenome.Celegans.UCSC.ce2,BSgenome.Celegans.UCSC.ce6,BSgenome.Cfamiliaris.UCSC.canFam2,BSgenome.Cfamiliaris.UCSC.canFam2.masked,BSgenome.Cfamiliaris.UCSC.canFam3,BSgenome.Cfamiliaris.UCSC.canFam3.masked,BSgenome.Dmelanogaster.UCSC.dm2,BSgenome.Dmelanogaster.UCSC.dm2.masked,BSgenome.Dmelanogaster.UCSC.dm3,BSgenome.Dmelanogaster.UCSC.dm3.masked,BSgenome.Dmelanogaster.UCSC.dm6,BSgenome.Drerio.UCSC.danRer10,BSgenome.Drerio.UCSC.danRer5,BSgenome.Drerio.UCSC.danRer5.masked,BSgenome.Drerio.UCSC.danRer6,BSgenome.Drerio.UCSC.danRer6.masked,BSgenome.Drerio.UCSC.danRer7,BSgenome.Drerio.UCSC.danRer7.masked,BSgenome.Ecoli.NCBI.20080805,BSgenome.Gaculeatus.UCSC.gasAcu1,BSgenome.Gaculeatus.UCSC.gasAcu1.masked,BSgenome.Ggallus.UCSC.galGal3,BSgenome.Ggallus.UCSC.galGal3.masked,BSgenome.Ggallus.UCSC.galGal4,BSgenome.Ggallus.UCSC.galGal4.masked,BSgenome.Ggallus.UCSC.galGal5,BSgenome.Hsapiens.1000genomes.hs37d5,BSgenome.Hsapiens.NCBI.GRCh38,BSgenome.Hsapiens.UCSC.hg17,BSgenome.Hsapiens.UCSC.hg17.masked,BSgenome.Hsapiens.UCSC.hg18,BSgenome.Hsapiens.UCSC.hg18.masked,BSgenome.Hsapiens.UCSC.hg19,BSgenome.Hsapiens.UCSC.hg19.masked,BSgenome.Hsapiens.UCSC.hg38,BSgenome.Hsapiens.UCSC.hg38.masked,BSgenome.Mfascicularis.NCBI.5.0,BSgenome.Mfuro.UCSC.musFur1,BSgenome.Mmulatta.UCSC.rheMac2,BSgenome.Mmulatta.UCSC.rheMac2.masked,BSgenome.Mmulatta.UCSC.rheMac3,BSgenome.Mmulatta.UCSC.rheMac3.masked,BSgenome.Mmulatta.UCSC.rheMac8,BSgenome.Mmusculus.UCSC.mm10,BSgenome.Mmusculus.UCSC.mm10.masked,BSgenome.Mmusculus.UCSC.mm8,BSgenome.Mmusculus.UCSC.mm8.masked,BSgenome.Mmusculus.UCSC.mm9,BSgenome.Mmusculus.UCSC.mm9.masked,BSgenome.Osativa.MSU.MSU7,BSgenome.Ptroglodytes.UCSC.panTro2,BSgenome.Ptroglodytes.UCSC.panTro2.masked,BSgenome.Ptroglodytes.UCSC.panTro3,BSgenome.Ptroglodytes.UCSC.panTro3.masked,BSgenome.Ptroglodytes.UCSC.panTro5,BSgenome.Ptroglodytes.UCSC.panTro6,BSgenome.Rnorvegicus.UCSC.rn4,BSgenome.Rnorvegicus.UCSC.rn4.masked,BSgenome.Rnorvegicus.UCSC.rn5,BSgenome.Rnorvegicus.UCSC.rn5.masked,BSgenome.Rnorvegicus.UCSC.rn6,BSgenome.Scerevisiae.UCSC.sacCer1,BSgenome.Scerevisiae.UCSC.sacCer2,BSgenome.Scerevisiae.UCSC.sacCer3,BSgenome.Sscrofa.UCSC.susScr11,BSgenome.Sscrofa.UCSC.susScr3,BSgenome.Sscrofa.UCSC.susScr3.masked,BSgenome.Tguttata.UCSC.taeGut1,BSgenome.Tguttata.UCSC.taeGut1.masked,BSgenome.Tguttata.UCSC.taeGut2,BSgenome.Vvinifera.URGI.IGGP12Xv0,BSgenome.Vvinifera.URGI.IGGP12Xv2,BSgenome.Vvinifera.URGI.IGGP8X,bsubtiliscdf,bsubtilisprobe,canine.db,canine.db0,canine2.db,canine2cdf,canine2probe,caninecdf,canineprobe,celegans.db,celeganscdf,celegansprobe,chicken.db,chicken.db0,chickencdf,chickenprobe,chimp.db0,citruscdf,citrusprobe,clariomdhumanprobeset.db,clariomdhumantranscriptcluster.db,clariomshumanhttranscriptcluster.db,clariomshumantranscriptcluster.db,clariomsmousehttranscriptcluster.db,clariomsmousetranscriptcluster.db,clariomsrathttranscriptcluster.db,clariomsrattranscriptcluster.db,cottoncdf,cottonprobe,cyp450cdf,drosgenome1.db,drosgenome1cdf,drosgenome1probe,drosophila2.db,drosophila2cdf,drosophila2probe,ecoli2.db,ecoli2cdf,ecoli2probe,ecoliasv2cdf,ecoliasv2probe,ecolicdf,ecoliK12.db0,ecoliprobe,ecoliSakai.db0,EuPathDB,EuPathDB,FDb.UCSC.tRNAs,fly.db0,gp53cdf,h10kcod.db,h20kcod.db,hcg110.db,hcg110cdf,hcg110probe,hgfocus.db,hgfocuscdf,hgfocusprobe,hgu133a.db,hgu133a2.db,hgu133a2cdf,hgu133a2frmavecs,hgu133a2probe,hgu133acdf,hgu133afrmavecs,hgu133aprobe,hgu133atagcdf,hgu133atagprobe,hgu133b.db,hgu133bcdf,hgu133bprobe,hgu133plus2.db,hgu133plus2cdf,hgu133plus2frmavecs,hgu133plus2probe,hgu219.db,hgu219cdf,hgu219probe,hgu95a.db,hgu95acdf,hgu95aprobe,hgu95av2.db,hgu95av2cdf,hgu95av2probe,hgu95b.db,hgu95bcdf,hgu95bprobe,hgu95c.db,hgu95ccdf,hgu95cprobe,hgu95d.db,hgu95dcdf,hgu95dprobe,hgu95e.db,hgu95ecdf,hgu95eprobe,hguatlas13k.db,hgubeta7.db,hguDKFZ31.db,hgug4100a.db,hgug4101a.db,hgug4110b.db,hgug4111a.db,hgug4112a.db,hguqiagenv3.db,hi16cod.db,hivprtplus2cdf,hom.At.inp.db,hom.Ce.inp.db,hom.Dm.inp.db,hom.Dr.inp.db,hom.Hs.inp.db,hom.Mm.inp.db,hom.Rn.inp.db,hom.Sc.inp.db,Homo.sapiens,Hs6UG171.db,HsAgilentDesign026652.db,Hspec,hspeccdf,hta20probeset.db,hta20transcriptcluster.db,hthgu133a.db,hthgu133acdf,hthgu133afrmavecs,hthgu133aprobe,hthgu133b.db,hthgu133bcdf,hthgu133bprobe,hthgu133pluspmcdf,hthgu133pluspmprobe,htmg430acdf,htmg430aprobe,htmg430bcdf,htmg430bprobe,htmg430pmcdf,htmg430pmprobe,htrat230pmcdf,htrat230pmprobe,htratfocuscdf,htratfocusprobe,hu35ksuba.db,hu35ksubacdf,hu35ksubaprobe,hu35ksubb.db,hu35ksubbcdf,hu35ksubbprobe,hu35ksubc.db,hu35ksubccdf,hu35ksubcprobe,hu35ksubd.db,hu35ksubdcdf,hu35ksubdprobe,hu6800.db,hu6800cdf,hu6800probe,hu6800subacdf,hu6800subbcdf,hu6800subccdf,hu6800subdcdf,huex.1.0.st.v2frmavecs,huex10stprobeset.db,huex10sttranscriptcluster.db,HuExExonProbesetLocation,hugene.1.0.st.v1frmavecs,hugene10stprobeset.db,hugene10sttranscriptcluster.db,hugene10stv1cdf,hugene10stv1probe,hugene11stprobeset.db,hugene11sttranscriptcluster.db,hugene20stprobeset.db,hugene20sttranscriptcluster.db,hugene21stprobeset.db,hugene21sttranscriptcluster.db,human.db0,HuO22.db,hwgcod.db,illuminaHumanv1.db,illuminaHumanv2.db,illuminaHumanv3.db,illuminaHumanv4.db,illuminaHumanWGDASLv3.db,illuminaHumanWGDASLv4.db,illuminaMousev1.db,illuminaMousev1p1.db,illuminaMousev2.db,illuminaRatv1.db,indac.db,JazaeriMetaData.db,LAPOINTE.db,lumiHumanAll.db,lumiMouseAll.db,lumiRatAll.db,m10kcod.db,m20kcod.db,maizecdf,maizeprobe,malaria.db0,medicagocdf,medicagoprobe,MeSH.Aca.eg.db,MeSH.Aga.PEST.eg.db,MeSH.Ame.eg.db,MeSH.Aml.eg.db,MeSH.Ana.eg.db,MeSH.Ani.FGSC.eg.db,MeSH.AOR.db,MeSH.Ath.eg.db,MeSH.Bfl.eg.db,MeSH.Bsu.168.eg.db,MeSH.Bta.eg.db,MeSH.Cal.SC5314.eg.db,MeSH.Cbr.eg.db,MeSH.Cel.eg.db,MeSH.Cfa.eg.db,MeSH.Cin.eg.db,MeSH.Cja.eg.db,MeSH.Cpo.eg.db,MeSH.Cre.eg.db,MeSH.Dan.eg.db,MeSH.db,MeSH.Dda.3937.eg.db,MeSH.Ddi.AX4.eg.db,MeSH.Der.eg.db,MeSH.Dgr.eg.db,MeSH.Dme.eg.db,MeSH.Dmo.eg.db,MeSH.Dpe.eg.db,MeSH.Dre.eg.db,MeSH.Dse.eg.db,MeSH.Dsi.eg.db,MeSH.Dvi.eg.db,MeSH.Dya.eg.db,MeSH.Eco.55989.eg.db,MeSH.Eco.ED1a.eg.db,MeSH.Eco.IAI39.eg.db,MeSH.Eco.K12.MG1655.eg.db,MeSH.Eco.O157.H7.Sakai.eg.db,MeSH.Eco.UMN026.eg.db,MeSH.Eqc.eg.db,MeSH.Gga.eg.db,MeSH.Gma.eg.db,MeSH.Hsa.eg.db,MeSH.Laf.eg.db,MeSH.Lma.eg.db,MeSH.Mdo.eg.db,MeSH.Mes.eg.db,MeSH.Mga.eg.db,MeSH.Miy.eg.db,MeSH.Mml.eg.db,MeSH.Mmu.eg.db,MeSH.Mtr.eg.db,MeSH.Nle.eg.db,MeSH.Oan.eg.db,MeSH.Ocu.eg.db,MeSH.Oni.eg.db,MeSH.Osa.eg.db,MeSH.Pab.eg.db,MeSH.Pae.PAO1.eg.db,MeSH.PCR.db,MeSH.Pfa.3D7.eg.db,MeSH.Pto.eg.db,MeSH.Ptr.eg.db,MeSH.Rno.eg.db,MeSH.Sce.S288c.eg.db,MeSH.Sco.A32.eg.db,MeSH.Sil.eg.db,MeSH.Spu.eg.db,MeSH.Ssc.eg.db,MeSH.Syn.eg.db,MeSH.Tbr.9274.eg.db,MeSH.Tgo.ME49.eg.db,MeSH.Tgu.eg.db,MeSH.Vvi.eg.db,MeSH.Xla.eg.db,MeSH.Xtr.eg.db,MeSH.Zma.eg.db,mgu74a.db,mgu74acdf,mgu74aprobe,mgu74av2.db,mgu74av2cdf,mgu74av2probe,mgu74b.db,mgu74bcdf,mgu74bprobe,mgu74bv2.db,mgu74bv2cdf,mgu74bv2probe,mgu74c.db,mgu74ccdf,mgu74cprobe,mgu74cv2.db,mgu74cv2cdf,mgu74cv2probe,mguatlas5k.db,mgug4104a.db,mgug4120a.db,mgug4121a.db,mgug4122a.db,mi16cod.db,mirna102xgaincdf,mirna10cdf,mirna10probe,mirna20cdf,MmAgilentDesign026655.db,moe430a.db,moe430acdf,moe430aprobe,moe430b.db,moe430bcdf,moe430bprobe,moex10stprobeset.db,moex10sttranscriptcluster.db,MoExExonProbesetLocation,mogene.1.0.st.v1frmavecs,mogene10stprobeset.db,mogene10sttranscriptcluster.db,mogene10stv1cdf,mogene10stv1probe,mogene11stprobeset.db,mogene11sttranscriptcluster.db,mogene20stprobeset.db,mogene20sttranscriptcluster.db,mogene21stprobeset.db,mogene21sttranscriptcluster.db,mouse.db0,mouse4302.db,mouse4302cdf,mouse4302frmavecs,mouse4302probe,mouse430a2.db,mouse430a2cdf,mouse430a2frmavecs,mouse430a2probe,mpedbarray.db,mta10probeset.db,mta10transcriptcluster.db,mu11ksuba.db,mu11ksubacdf,mu11ksubaprobe,mu11ksubb.db,mu11ksubbcdf,mu11ksubbprobe,Mu15v1.db,mu19ksuba.db,mu19ksubacdf,mu19ksubb.db,mu19ksubbcdf,mu19ksubc.db,mu19ksubccdf,Mu22v3.db,mu6500subacdf,mu6500subbcdf,mu6500subccdf,mu6500subdcdf,Mus.musculus,mwgcod.db,Norway981.db,nugohs1a520180.db,nugohs1a520180cdf,nugohs1a520180probe,nugomm1a520177.db,nugomm1a520177cdf,nugomm1a520177probe,OperonHumanV3.db,org.Ag.eg.db,org.At.tair.db,org.Bt.eg.db,org.Ce.eg.db,org.Cf.eg.db,org.Dm.eg.db,org.Dr.eg.db,org.EcK12.eg.db,org.EcSakai.eg.db,org.Gg.eg.db,org.Hs.eg.db,org.Mm.eg.db,org.Mmu.eg.db,org.Pf.plasmo.db,org.Pt.eg.db,org.Rn.eg.db,org.Sc.sgd.db,org.Ss.eg.db,org.Xl.eg.db,paeg1acdf,paeg1aprobe,PartheenMetaData.db,pedbarrayv10.db,pedbarrayv9.db,pig.db0,plasmodiumanophelescdf,plasmodiumanophelesprobe,POCRCannotation.db,poplarcdf,poplarprobe,porcine.db,porcinecdf,porcineprobe,primeviewcdf,primeviewprobe,r10kcod.db,rae230a.db,rae230acdf,rae230aprobe,rae230b.db,rae230bcdf,rae230bprobe,raex10stprobeset.db,raex10sttranscriptcluster.db,RaExExonProbesetLocation,ragene10stprobeset.db,ragene10sttranscriptcluster.db,ragene10stv1cdf,ragene10stv1probe,ragene11stprobeset.db,ragene11sttranscriptcluster.db,ragene20stprobeset.db,ragene20sttranscriptcluster.db,ragene21stprobeset.db,ragene21sttranscriptcluster.db,rat.db0,rat2302.db,rat2302cdf,rat2302probe,rattoxfxcdf,rattoxfxprobe,Rattus.norvegicus,rgu34a.db,rgu34acdf,rgu34aprobe,rgu34b.db,rgu34bcdf,rgu34bprobe,rgu34c.db,rgu34ccdf,rgu34cprobe,rguatlas4k.db,rgug4105a.db,rgug4130a.db,rgug4131a.db,rhesus.db0,rhesuscdf,rhesusprobe,ri16cod.db,ricecdf,riceprobe,RnAgilentDesign028282.db,rnu34.db,rnu34cdf,rnu34probe,Roberts2005Annotation.db,rta10probeset.db,rta10transcriptcluster.db,rtu34.db,rtu34cdf,rtu34probe,rwgcod.db,saureuscdf,saureusprobe,SHDZ.db,SNPlocs.Hsapiens.dbSNP141.GRCh38,SNPlocs.Hsapiens.dbSNP144.GRCh37,SNPlocs.Hsapiens.dbSNP144.GRCh38,SNPlocs.Hsapiens.dbSNP149.GRCh38,SNPlocs.Hsapiens.dbSNP150.GRCh38,SNPlocs.Hsapiens.dbSNP151.GRCh38,soybeancdf,soybeanprobe,sugarcanecdf,sugarcaneprobe,test1cdf,test2cdf,test3cdf,test3probe,tomatocdf,tomatoprobe,TxDb.Athaliana.BioMart.plantsmart22,TxDb.Athaliana.BioMart.plantsmart25,TxDb.Athaliana.BioMart.plantsmart28,TxDb.Btaurus.UCSC.bosTau8.refGene,TxDb.Celegans.UCSC.ce11.refGene,TxDb.Celegans.UCSC.ce6.ensGene,TxDb.Cfamiliaris.UCSC.canFam3.refGene,TxDb.Dmelanogaster.UCSC.dm3.ensGene,TxDb.Dmelanogaster.UCSC.dm6.ensGene,TxDb.Drerio.UCSC.danRer10.refGene,TxDb.Ggallus.UCSC.galGal4.refGene,TxDb.Ggallus.UCSC.galGal5.refGene,TxDb.Hsapiens.BioMart.igis,TxDb.Hsapiens.UCSC.hg18.knownGene,TxDb.Hsapiens.UCSC.hg19.knownGene,TxDb.Hsapiens.UCSC.hg19.lincRNAsTranscripts,TxDb.Hsapiens.UCSC.hg38.knownGene,TxDb.Mmulatta.UCSC.rheMac3.refGene,TxDb.Mmulatta.UCSC.rheMac8.refGene,TxDb.Mmusculus.UCSC.mm10.ensGene,TxDb.Mmusculus.UCSC.mm10.knownGene,TxDb.Mmusculus.UCSC.mm9.knownGene,TxDb.Ptroglodytes.UCSC.panTro4.refGene,TxDb.Ptroglodytes.UCSC.panTro5.refGene,TxDb.Rnorvegicus.BioMart.igis,TxDb.Rnorvegicus.UCSC.rn4.ensGene,TxDb.Rnorvegicus.UCSC.rn5.refGene,TxDb.Rnorvegicus.UCSC.rn6.refGene,TxDb.Scerevisiae.UCSC.sacCer2.sgdGene,TxDb.Scerevisiae.UCSC.sacCer3.sgdGene,TxDb.Sscrofa.UCSC.susScr11.refGene,TxDb.Sscrofa.UCSC.susScr3.refGene,u133aaofav2cdf,u133x3p.db,u133x3pcdf,u133x3pprobe,vitisviniferacdf,vitisviniferaprobe,wheatcdf,wheatprobe,worm.db0,xenopus.db0,xenopuslaeviscdf,xenopuslaevisprobe,xlaevis.db,xlaevis2cdf,xlaevis2probe,XtraSNPlocs.Hsapiens.dbSNP141.GRCh38,XtraSNPlocs.Hsapiens.dbSNP144.GRCh37,XtraSNPlocs.Hsapiens.dbSNP144.GRCh38,xtropicaliscdf,xtropicalisprobe,ye6100subacdf,ye6100subbcdf,ye6100subccdf,ye6100subdcdf,yeast.db0,yeast2.db,yeast2cdf,yeast2probe,ygs98.db,ygs98cdf,ygs98frmavecs,ygs98probe,zebrafish.db,zebrafish.db0,zebrafishcdf,zebrafishprobe" id="PackageType" class="jstree-closed"><ins class="jstree-icon">&nbsp;</ins><a href="https://www.bioconductor.org/packages/release/BiocViews.html#"><ins class="jstree-icon">&nbsp;</ins>PackageType (650)</a><ul><li packagelist="AHEnsDbs,EuPathDB" id="AnnotationHub" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="https://www.bioconductor.org/packages/release/BiocViews.html#"><ins class="jstree-icon">&nbsp;</ins>AnnotationHub (2)</a></li><li packagelist="BSgenome.Amellifera.BeeBase.assembly4,BSgenome.Amellifera.UCSC.apiMel2,BSgenome.Amellifera.UCSC.apiMel2.masked,BSgenome.Aofficinalis.NCBI.V1,BSgenome.Athaliana.TAIR.04232008,BSgenome.Athaliana.TAIR.TAIR9,BSgenome.Btaurus.UCSC.bosTau3,BSgenome.Btaurus.UCSC.bosTau3.masked,BSgenome.Btaurus.UCSC.bosTau4,BSgenome.Btaurus.UCSC.bosTau4.masked,BSgenome.Btaurus.UCSC.bosTau6,BSgenome.Btaurus.UCSC.bosTau6.masked,BSgenome.Btaurus.UCSC.bosTau8,BSgenome.Carietinum.NCBI.v1,BSgenome.Celegans.UCSC.ce10,BSgenome.Celegans.UCSC.ce11,BSgenome.Celegans.UCSC.ce2,BSgenome.Celegans.UCSC.ce6,BSgenome.Cfamiliaris.UCSC.canFam2,BSgenome.Cfamiliaris.UCSC.canFam2.masked,BSgenome.Cfamiliaris.UCSC.canFam3,BSgenome.Cfamiliaris.UCSC.canFam3.masked,BSgenome.Dmelanogaster.UCSC.dm2,BSgenome.Dmelanogaster.UCSC.dm2.masked,BSgenome.Dmelanogaster.UCSC.dm3,BSgenome.Dmelanogaster.UCSC.dm3.masked,BSgenome.Dmelanogaster.UCSC.dm6,BSgenome.Drerio.UCSC.danRer10,BSgenome.Drerio.UCSC.danRer5,BSgenome.Drerio.UCSC.danRer5.masked,BSgenome.Drerio.UCSC.danRer6,BSgenome.Drerio.UCSC.danRer6.masked,BSgenome.Drerio.UCSC.danRer7,BSgenome.Drerio.UCSC.danRer7.masked,BSgenome.Ecoli.NCBI.20080805,BSgenome.Gaculeatus.UCSC.gasAcu1,BSgenome.Gaculeatus.UCSC.gasAcu1.masked,BSgenome.Ggallus.UCSC.galGal3,BSgenome.Ggallus.UCSC.galGal3.masked,BSgenome.Ggallus.UCSC.galGal4,BSgenome.Ggallus.UCSC.galGal4.masked,BSgenome.Ggallus.UCSC.galGal5,BSgenome.Hsapiens.1000genomes.hs37d5,BSgenome.Hsapiens.NCBI.GRCh38,BSgenome.Hsapiens.UCSC.hg17,BSgenome.Hsapiens.UCSC.hg17.masked,BSgenome.Hsapiens.UCSC.hg18,BSgenome.Hsapiens.UCSC.hg18.masked,BSgenome.Hsapiens.UCSC.hg19,BSgenome.Hsapiens.UCSC.hg19.masked,BSgenome.Hsapiens.UCSC.hg38,BSgenome.Hsapiens.UCSC.hg38.masked,BSgenome.Mfascicularis.NCBI.5.0,BSgenome.Mfuro.UCSC.musFur1,BSgenome.Mmulatta.UCSC.rheMac2,BSgenome.Mmulatta.UCSC.rheMac2.masked,BSgenome.Mmulatta.UCSC.rheMac3,BSgenome.Mmulatta.UCSC.rheMac3.masked,BSgenome.Mmulatta.UCSC.rheMac8,BSgenome.Mmusculus.UCSC.mm10,BSgenome.Mmusculus.UCSC.mm10.masked,BSgenome.Mmusculus.UCSC.mm8,BSgenome.Mmusculus.UCSC.mm8.masked,BSgenome.Mmusculus.UCSC.mm9,BSgenome.Mmusculus.UCSC.mm9.masked,BSgenome.Osativa.MSU.MSU7,BSgenome.Ptroglodytes.UCSC.panTro2,BSgenome.Ptroglodytes.UCSC.panTro2.masked,BSgenome.Ptroglodytes.UCSC.panTro3,BSgenome.Ptroglodytes.UCSC.panTro3.masked,BSgenome.Ptroglodytes.UCSC.panTro5,BSgenome.Ptroglodytes.UCSC.panTro6,BSgenome.Rnorvegicus.UCSC.rn4,BSgenome.Rnorvegicus.UCSC.rn4.masked,BSgenome.Rnorvegicus.UCSC.rn5,BSgenome.Rnorvegicus.UCSC.rn5.masked,BSgenome.Rnorvegicus.UCSC.rn6,BSgenome.Scerevisiae.UCSC.sacCer1,BSgenome.Scerevisiae.UCSC.sacCer2,BSgenome.Scerevisiae.UCSC.sacCer3,BSgenome.Sscrofa.UCSC.susScr11,BSgenome.Sscrofa.UCSC.susScr3,BSgenome.Sscrofa.UCSC.susScr3.masked,BSgenome.Tguttata.UCSC.taeGut1,BSgenome.Tguttata.UCSC.taeGut1.masked,BSgenome.Tguttata.UCSC.taeGut2,BSgenome.Vvinifera.URGI.IGGP12Xv0,BSgenome.Vvinifera.URGI.IGGP12Xv2,BSgenome.Vvinifera.URGI.IGGP8X" id="BSgenome" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="https://www.bioconductor.org/packages/release/BiocViews.html#"><ins class="jstree-icon">&nbsp;</ins>BSgenome (89)</a></li><li packagelist="agcdf,ath1121501cdf,barley1cdf,bovinecdf,bsubtiliscdf,canine2cdf,caninecdf,celeganscdf,chickencdf,citruscdf,cottoncdf,cyp450cdf,drosgenome1cdf,drosophila2cdf,ecoli2cdf,ecoliasv2cdf,ecolicdf,gp53cdf,hcg110cdf,hgfocuscdf,hgu133a2cdf,hgu133acdf,hgu133atagcdf,hgu133bcdf,hgu133plus2cdf,hgu219cdf,hgu95acdf,hgu95av2cdf,hgu95bcdf,hgu95ccdf,hgu95dcdf,hgu95ecdf,hivprtplus2cdf,Hspec,hspeccdf,hthgu133acdf,hthgu133bcdf,hthgu133pluspmcdf,htmg430acdf,htmg430bcdf,htmg430pmcdf,htrat230pmcdf,htratfocuscdf,hu35ksubacdf,hu35ksubbcdf,hu35ksubccdf,hu35ksubdcdf,hu6800cdf,hu6800subacdf,hu6800subbcdf,hu6800subccdf,hu6800subdcdf,hugene10stv1cdf,maizecdf,medicagocdf,mgu74acdf,mgu74av2cdf,mgu74bcdf,mgu74bv2cdf,mgu74ccdf,mgu74cv2cdf,mirna102xgaincdf,mirna10cdf,mirna20cdf,moe430acdf,moe430bcdf,mogene10stv1cdf,mouse4302cdf,mouse430a2cdf,mu11ksubacdf,mu11ksubbcdf,mu19ksubacdf,mu19ksubbcdf,mu19ksubccdf,mu6500subacdf,mu6500subbcdf,mu6500subccdf,mu6500subdcdf,nugohs1a520180cdf,nugomm1a520177cdf,paeg1acdf,plasmodiumanophelescdf,poplarcdf,porcinecdf,primeviewcdf,rae230acdf,rae230bcdf,ragene10stv1cdf,rat2302cdf,rattoxfxcdf,rgu34acdf,rgu34bcdf,rgu34ccdf,rhesuscdf,ricecdf,rnu34cdf,rtu34cdf,saureuscdf,soybeancdf,sugarcanecdf,test1cdf,test2cdf,test3cdf,tomatocdf,u133aaofav2cdf,u133x3pcdf,vitisviniferacdf,wheatcdf,xenopuslaeviscdf,xlaevis2cdf,xtropicaliscdf,ye6100subacdf,ye6100subbcdf,ye6100subccdf,ye6100subdcdf,yeast2cdf,ygs98cdf,zebrafishcdf" id="cdf" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="https://www.bioconductor.org/packages/release/BiocViews.html#"><ins class="jstree-icon">&nbsp;</ins>cdf (118)</a></li><li packagelist="adme16cod.db,ag.db,ath1121501.db,bovine.db,canine.db,canine2.db,celegans.db,chicken.db,clariomdhumanprobeset.db,clariomdhumantranscriptcluster.db,clariomshumanhttranscriptcluster.db,clariomshumantranscriptcluster.db,clariomsmousehttranscriptcluster.db,clariomsmousetranscriptcluster.db,clariomsrathttranscriptcluster.db,clariomsrattranscriptcluster.db,drosgenome1.db,drosophila2.db,ecoli2.db,h10kcod.db,h20kcod.db,hcg110.db,hgfocus.db,hgu133a.db,hgu133a2.db,hgu133b.db,hgu133plus2.db,hgu219.db,hgu95a.db,hgu95av2.db,hgu95b.db,hgu95c.db,hgu95d.db,hgu95e.db,hguatlas13k.db,hgubeta7.db,hguDKFZ31.db,hgug4100a.db,hgug4101a.db,hgug4110b.db,hgug4111a.db,hgug4112a.db,hguqiagenv3.db,hi16cod.db,Hs6UG171.db,HsAgilentDesign026652.db,hta20probeset.db,hta20transcriptcluster.db,hthgu133a.db,hthgu133b.db,hu35ksuba.db,hu35ksubb.db,hu35ksubc.db,hu35ksubd.db,hu6800.db,huex10stprobeset.db,huex10sttranscriptcluster.db,hugene10stprobeset.db,hugene10sttranscriptcluster.db,hugene11stprobeset.db,hugene11sttranscriptcluster.db,hugene20stprobeset.db,hugene20sttranscriptcluster.db,hugene21stprobeset.db,hugene21sttranscriptcluster.db,HuO22.db,hwgcod.db,illuminaHumanv1.db,illuminaHumanv2.db,illuminaHumanv3.db,illuminaHumanv4.db,illuminaHumanWGDASLv3.db,illuminaHumanWGDASLv4.db,illuminaMousev1.db,illuminaMousev1p1.db,illuminaMousev2.db,illuminaRatv1.db,indac.db,JazaeriMetaData.db,LAPOINTE.db,lumiHumanAll.db,lumiMouseAll.db,lumiRatAll.db,m10kcod.db,m20kcod.db,mgu74a.db,mgu74av2.db,mgu74b.db,mgu74bv2.db,mgu74c.db,mgu74cv2.db,mguatlas5k.db,mgug4104a.db,mgug4120a.db,mgug4121a.db,mgug4122a.db,mi16cod.db,MmAgilentDesign026655.db,moe430a.db,moe430b.db,moex10stprobeset.db,moex10sttranscriptcluster.db,mogene10stprobeset.db,mogene10sttranscriptcluster.db,mogene11stprobeset.db,mogene11sttranscriptcluster.db,mogene20stprobeset.db,mogene20sttranscriptcluster.db,mogene21stprobeset.db,mogene21sttranscriptcluster.db,mouse4302.db,mouse430a2.db,mpedbarray.db,mta10probeset.db,mta10transcriptcluster.db,mu11ksuba.db,mu11ksubb.db,Mu15v1.db,mu19ksuba.db,mu19ksubb.db,mu19ksubc.db,Mu22v3.db,mwgcod.db,Norway981.db,nugohs1a520180.db,nugomm1a520177.db,OperonHumanV3.db,PartheenMetaData.db,pedbarrayv10.db,pedbarrayv9.db,POCRCannotation.db,porcine.db,r10kcod.db,rae230a.db,rae230b.db,raex10stprobeset.db,raex10sttranscriptcluster.db,ragene10stprobeset.db,ragene10sttranscriptcluster.db,ragene11stprobeset.db,ragene11sttranscriptcluster.db,ragene20stprobeset.db,ragene20sttranscriptcluster.db,ragene21stprobeset.db,ragene21sttranscriptcluster.db,rat2302.db,rgu34a.db,rgu34b.db,rgu34c.db,rguatlas4k.db,rgug4105a.db,rgug4130a.db,rgug4131a.db,ri16cod.db,RnAgilentDesign028282.db,rnu34.db,Roberts2005Annotation.db,rta10probeset.db,rta10transcriptcluster.db,rtu34.db,rwgcod.db,SHDZ.db,u133x3p.db,xlaevis.db,yeast2.db,ygs98.db,zebrafish.db" id="ChipDb" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="https://www.bioconductor.org/packages/release/BiocViews.html#"><ins class="jstree-icon">&nbsp;</ins>ChipDb (167)</a></li><li packagelist="anopheles.db0,arabidopsis.db0,bovine.db0,canine.db0,chicken.db0,chimp.db0,ecoliK12.db0,ecoliSakai.db0,fly.db0,human.db0,malaria.db0,mouse.db0,pig.db0,rat.db0,rhesus.db0,worm.db0,xenopus.db0,yeast.db0,zebrafish.db0" id="db0" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="https://www.bioconductor.org/packages/release/BiocViews.html#"><ins class="jstree-icon">&nbsp;</ins>db0 (19)</a></li><li packagelist="EuPathDB" id="EuPathDB" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="https://www.bioconductor.org/packages/release/BiocViews.html#"><ins class="jstree-icon">&nbsp;</ins>EuPathDB (1)</a></li><li packagelist="hgu133a2frmavecs,hgu133afrmavecs,hgu133plus2frmavecs,hthgu133afrmavecs,huex.1.0.st.v2frmavecs,hugene.1.0.st.v1frmavecs,mogene.1.0.st.v1frmavecs,mouse4302frmavecs,mouse430a2frmavecs,ygs98frmavecs" id="FRMA" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="https://www.bioconductor.org/packages/release/BiocViews.html#"><ins class="jstree-icon">&nbsp;</ins>FRMA (10)</a></li><li packagelist="hom.At.inp.db,hom.Ce.inp.db,hom.Dm.inp.db,hom.Dr.inp.db,hom.Hs.inp.db,hom.Mm.inp.db,hom.Rn.inp.db,hom.Sc.inp.db" id="InparanoidDb" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="https://www.bioconductor.org/packages/release/BiocViews.html#"><ins class="jstree-icon">&nbsp;</ins>InparanoidDb (8)</a></li><li packagelist="MeSH.Aca.eg.db,MeSH.Aga.PEST.eg.db,MeSH.Ame.eg.db,MeSH.Aml.eg.db,MeSH.Ana.eg.db,MeSH.Ani.FGSC.eg.db,MeSH.AOR.db,MeSH.Ath.eg.db,MeSH.Bfl.eg.db,MeSH.Bsu.168.eg.db,MeSH.Bta.eg.db,MeSH.Cal.SC5314.eg.db,MeSH.Cbr.eg.db,MeSH.Cel.eg.db,MeSH.Cfa.eg.db,MeSH.Cin.eg.db,MeSH.Cja.eg.db,MeSH.Cpo.eg.db,MeSH.Cre.eg.db,MeSH.Dan.eg.db,MeSH.db,MeSH.Dda.3937.eg.db,MeSH.Ddi.AX4.eg.db,MeSH.Der.eg.db,MeSH.Dgr.eg.db,MeSH.Dme.eg.db,MeSH.Dmo.eg.db,MeSH.Dpe.eg.db,MeSH.Dre.eg.db,MeSH.Dse.eg.db,MeSH.Dsi.eg.db,MeSH.Dvi.eg.db,MeSH.Dya.eg.db,MeSH.Eco.55989.eg.db,MeSH.Eco.ED1a.eg.db,MeSH.Eco.IAI39.eg.db,MeSH.Eco.K12.MG1655.eg.db,MeSH.Eco.O157.H7.Sakai.eg.db,MeSH.Eco.UMN026.eg.db,MeSH.Eqc.eg.db,MeSH.Gga.eg.db,MeSH.Gma.eg.db,MeSH.Hsa.eg.db,MeSH.Laf.eg.db,MeSH.Lma.eg.db,MeSH.Mdo.eg.db,MeSH.Mes.eg.db,MeSH.Mga.eg.db,MeSH.Miy.eg.db,MeSH.Mml.eg.db,MeSH.Mmu.eg.db,MeSH.Mtr.eg.db,MeSH.Nle.eg.db,MeSH.Oan.eg.db,MeSH.Ocu.eg.db,MeSH.Oni.eg.db,MeSH.Osa.eg.db,MeSH.Pab.eg.db,MeSH.Pae.PAO1.eg.db,MeSH.PCR.db,MeSH.Pfa.3D7.eg.db,MeSH.Pto.eg.db,MeSH.Ptr.eg.db,MeSH.Rno.eg.db,MeSH.Sce.S288c.eg.db,MeSH.Sco.A32.eg.db,MeSH.Sil.eg.db,MeSH.Spu.eg.db,MeSH.Ssc.eg.db,MeSH.Syn.eg.db,MeSH.Tbr.9274.eg.db,MeSH.Tgo.ME49.eg.db,MeSH.Tgu.eg.db,MeSH.Vvi.eg.db,MeSH.Xla.eg.db,MeSH.Xtr.eg.db,MeSH.Zma.eg.db" id="MeSHDb" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="https://www.bioconductor.org/packages/release/BiocViews.html#"><ins class="jstree-icon">&nbsp;</ins>MeSHDb (77)</a></li><li packagelist="Homo.sapiens,Mus.musculus,Rattus.norvegicus" id="OrganismDb" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="https://www.bioconductor.org/packages/release/BiocViews.html#"><ins class="jstree-icon">&nbsp;</ins>OrganismDb (3)</a></li><li packagelist="org.Ag.eg.db,org.At.tair.db,org.Bt.eg.db,org.Ce.eg.db,org.Cf.eg.db,org.Dm.eg.db,org.Dr.eg.db,org.EcK12.eg.db,org.EcSakai.eg.db,org.Gg.eg.db,org.Hs.eg.db,org.Mm.eg.db,org.Mmu.eg.db,org.Pf.plasmo.db,org.Pt.eg.db,org.Rn.eg.db,org.Sc.sgd.db,org.Ss.eg.db,org.Xl.eg.db" id="OrgDb" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="https://www.bioconductor.org/packages/release/BiocViews.html#"><ins class="jstree-icon">&nbsp;</ins>OrgDb (19)</a></li><li class="jstree-leaf" style="display: none;"><ins class="jstree-icon">&nbsp;</ins><a href="https://www.bioconductor.org/packages/release/BiocViews.html#"><ins class="jstree-icon">&nbsp;</ins>PolyPhen</a></li><li packagelist="agprobe,ath1121501probe,barley1probe,bovineprobe,bsubtilisprobe,canine2probe,canineprobe,celegansprobe,chickenprobe,citrusprobe,cottonprobe,drosgenome1probe,drosophila2probe,ecoli2probe,ecoliasv2probe,ecoliprobe,hcg110probe,hgfocusprobe,hgu133a2probe,hgu133aprobe,hgu133atagprobe,hgu133bprobe,hgu133plus2probe,hgu219probe,hgu95aprobe,hgu95av2probe,hgu95bprobe,hgu95cprobe,hgu95dprobe,hgu95eprobe,hthgu133aprobe,hthgu133bprobe,hthgu133pluspmprobe,htmg430aprobe,htmg430bprobe,htmg430pmprobe,htrat230pmprobe,htratfocusprobe,hu35ksubaprobe,hu35ksubbprobe,hu35ksubcprobe,hu35ksubdprobe,hu6800probe,HuExExonProbesetLocation,hugene10stv1probe,maizeprobe,medicagoprobe,mgu74aprobe,mgu74av2probe,mgu74bprobe,mgu74bv2probe,mgu74cprobe,mgu74cv2probe,mirna10probe,moe430aprobe,moe430bprobe,MoExExonProbesetLocation,mogene10stv1probe,mouse4302probe,mouse430a2probe,mu11ksubaprobe,mu11ksubbprobe,nugohs1a520180probe,nugomm1a520177probe,paeg1aprobe,plasmodiumanophelesprobe,poplarprobe,porcineprobe,primeviewprobe,rae230aprobe,rae230bprobe,RaExExonProbesetLocation,ragene10stv1probe,rat2302probe,rattoxfxprobe,rgu34aprobe,rgu34bprobe,rgu34cprobe,rhesusprobe,riceprobe,rnu34probe,rtu34probe,saureusprobe,soybeanprobe,sugarcaneprobe,test3probe,tomatoprobe,u133x3pprobe,vitisviniferaprobe,wheatprobe,xenopuslaevisprobe,xlaevis2probe,xtropicalisprobe,yeast2probe,ygs98probe,zebrafishprobe" id="probe" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="https://www.bioconductor.org/packages/release/BiocViews.html#"><ins class="jstree-icon">&nbsp;</ins>probe (96)</a></li><li class="jstree-leaf" style="display: none;"><ins class="jstree-icon">&nbsp;</ins><a href="https://www.bioconductor.org/packages/release/BiocViews.html#"><ins class="jstree-icon">&nbsp;</ins>SIFT</a></li><li packagelist="SNPlocs.Hsapiens.dbSNP141.GRCh38,SNPlocs.Hsapiens.dbSNP144.GRCh37,SNPlocs.Hsapiens.dbSNP144.GRCh38,SNPlocs.Hsapiens.dbSNP149.GRCh38,SNPlocs.Hsapiens.dbSNP150.GRCh38,SNPlocs.Hsapiens.dbSNP151.GRCh38" id="SNPlocs" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="https://www.bioconductor.org/packages/release/BiocViews.html#"><ins class="jstree-icon">&nbsp;</ins>SNPlocs (6)</a></li><li packagelist="FDb.UCSC.tRNAs,TxDb.Athaliana.BioMart.plantsmart22,TxDb.Athaliana.BioMart.plantsmart25,TxDb.Athaliana.BioMart.plantsmart28,TxDb.Btaurus.UCSC.bosTau8.refGene,TxDb.Celegans.UCSC.ce11.refGene,TxDb.Celegans.UCSC.ce6.ensGene,TxDb.Cfamiliaris.UCSC.canFam3.refGene,TxDb.Dmelanogaster.UCSC.dm3.ensGene,TxDb.Dmelanogaster.UCSC.dm6.ensGene,TxDb.Drerio.UCSC.danRer10.refGene,TxDb.Ggallus.UCSC.galGal4.refGene,TxDb.Ggallus.UCSC.galGal5.refGene,TxDb.Hsapiens.BioMart.igis,TxDb.Hsapiens.UCSC.hg18.knownGene,TxDb.Hsapiens.UCSC.hg19.knownGene,TxDb.Hsapiens.UCSC.hg19.lincRNAsTranscripts,TxDb.Hsapiens.UCSC.hg38.knownGene,TxDb.Mmulatta.UCSC.rheMac3.refGene,TxDb.Mmulatta.UCSC.rheMac8.refGene,TxDb.Mmusculus.UCSC.mm10.ensGene,TxDb.Mmusculus.UCSC.mm10.knownGene,TxDb.Mmusculus.UCSC.mm9.knownGene,TxDb.Ptroglodytes.UCSC.panTro4.refGene,TxDb.Ptroglodytes.UCSC.panTro5.refGene,TxDb.Rnorvegicus.BioMart.igis,TxDb.Rnorvegicus.UCSC.rn4.ensGene,TxDb.Rnorvegicus.UCSC.rn5.refGene,TxDb.Rnorvegicus.UCSC.rn6.refGene,TxDb.Scerevisiae.UCSC.sacCer2.sgdGene,TxDb.Scerevisiae.UCSC.sacCer3.sgdGene,TxDb.Sscrofa.UCSC.susScr11.refGene,TxDb.Sscrofa.UCSC.susScr3.refGene" id="TxDb" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="https://www.bioconductor.org/packages/release/BiocViews.html#"><ins class="jstree-icon">&nbsp;</ins>TxDb (33)</a></li><li packagelist="XtraSNPlocs.Hsapiens.dbSNP141.GRCh38,XtraSNPlocs.Hsapiens.dbSNP144.GRCh37,XtraSNPlocs.Hsapiens.dbSNP144.GRCh38" id="XtraSNPlocs" class="jstree-last jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="https://www.bioconductor.org/packages/release/BiocViews.html#"><ins class="jstree-icon">&nbsp;</ins>XtraSNPlocs (3)</a></li></ul></li><li packagelist="LymphoSeqDB" id="SequenceAnnotation" class="jstree-last jstree-closed"><ins class="jstree-icon">&nbsp;</ins><a href="https://www.bioconductor.org/packages/release/BiocViews.html#"><ins class="jstree-icon">&nbsp;</ins>SequenceAnnotation (1)</a><ul><li packagelist="LymphoSeqDB" id="GenomicSequence" class="jstree-last jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="https://www.bioconductor.org/packages/release/BiocViews.html#"><ins class="jstree-icon">&nbsp;</ins>GenomicSequence (1)</a></li></ul></li></ul></li><li packagelist="ABAData,affycompData,affydata,Affyhgu133A2Expr,Affyhgu133aExpr,Affyhgu133Plus2Expr,AffymetrixDataTestFiles,Affymoe4302Expr,airway,ALL,allenpvc,ALLMLL,alpineData,AmpAffyExample,AneuFinderData,antiProfilesData,aracne.networks,ARRmData,AshkenazimSonChr21,ASICSdata,AssessORFData,bcellViper,beadarrayExampleData,BeadArrayUseCases,beta7,biotmleData,bladderbatch,blimaTestingData,BloodCancerMultiOmics2017,brainImageRdata,breakpointRdata,breastCancerMAINZ,breastCancerNKI,breastCancerTRANSBIG,breastCancerUNT,breastCancerUPP,breastCancerVDX,brgedata,bronchialIL13,bsseqData,cancerdata,CardinalWorkflows,ccdata,CCl4,ccTutorial,celarefData,CellMapperData,ceu1kg,ceu1kgv,ceuhm3,cgdv17,ChAMPdata,charmData,ChIC.data,ChimpHumanBrainData,chipenrich.data,ChIPexoQualExample,ChIPXpressData,chromstaRData,CLL,CLLmethylation,cMap2data,cnvGSAdata,COHCAPanno,colonCA,CONFESSdata,ConnectivityMap,COPDSexualDimorphism.data,CopyhelpeR,CopyNeutralIMA,COSMIC.67,CRCL18,curatedBladderData,curatedBreastData,curatedCRCData,curatedMetagenomicData,curatedOvarianData,curatedTCGAData,DAPARdata,davidTiling,derfinderData,DeSousa2013,diffloopdata,diggitdata,DLBCL,DmelSGI,DMRcatedata,DonaPLLP2013,DREAM4,dressCheck,DrugVsDiseasedata,dsQTL,DuoClustering2018,DvDdata,dyebiasexamples,EatonEtAlChIPseq,ecoliLeucine,EGSEAdata,ELMER.data,estrogen,etec16s,faahKO,fabiaData,facopy.annot,facsDorit,FANTOM3and4CAGE,ffpeExampleData,fibroEset,FIs,fission,Fletcher2013a,Fletcher2013b,flowFitExampleData,flowPloidyData,flowQBData,FlowSorted.Blood.450k,FlowSorted.Blood.EPIC,FlowSorted.CordBlood.450k,FlowSorted.CordBloodNorway.450k,FlowSorted.DLPFC.450k,flowWorkspaceData,frmaExampleData,FunciSNP.data,furrowSeg,gageData,gaschYHS,gatingMLData,gcspikelite,geneLenDataBase,genomationData,GeuvadisTranscriptExpr,geuvPack,geuvStore2,GGdata,GIGSEAdata,golubEsets,grndata,GSBenchMark,GSE62944,gskb,GSVAdata,GWASdata,h5vcData,hapmap100khind,hapmap100kxba,hapmap500knsp,hapmap500ksty,hapmapsnp5,hapmapsnp6,harbChIP,HarmanData,HarmonizedTCGAData,HD2013SGI,HDCytoData,healthyFlowData,HEEBOdata,HelloRangesData,hgu133abarcodevecs,hgu133plus2barcodevecs,hgu133plus2CellScore,hgu2beta7,HiCDataHumanIMR90,HiCDataLymphoblast,Hiiragi2013,HIVcDNAvantWout03,HMP16SData,hmyriB36,HSMMSingleCell,HumanAffyData,humanStemCell,IHWpaper,Illumina450ProbeVariants.db,IlluminaDataTestFiles,iontreeData,ITALICSData,Iyer517,JASPAR2014,JASPAR2016,JctSeqData,KEGGandMetacoreDzPathwaysGEO,KEGGdzPathwaysGEO,kidpack,KOdata,leeBamViews,leukemiasEset,LiebermanAidenHiC2009,ListerEtAlBSseq,lumiBarnes,LungCancerACvsSCCGEO,LungCancerLines,lungExpression,lydata,M3DExampleData,mammaPrintData,mAPKLData,maqcExpression4plex,MAQCsubset,MAQCsubsetAFX,MAQCsubsetILM,mCSEAdata,mcsurvdata,MEDIPSData,MEEBOdata,MetaGxBreast,MetaGxOvarian,MetaGxPancreas,metaMSdata,MethylAidData,methyvimData,microRNAome,MIGSAdata,minfiData,minfiDataEPIC,minionSummaryData,miRcompData,miRNATarget,mitoODEdata,MMDiffBamSubset,mosaicsExample,mouse4302barcodevecs,MSBdata,msd16s,msdata,MSMB,msPurityData,msqc1,MSstatsBioData,mtbls2,MTseekerData,MUGAExampleData,Mulder2012,mvoutData,NCIgraphData,Neve2006,NGScopyData,OMICsPCAdata,OnassisJavaLibs,parathyroidSE,pasilla,pasillaBamSubset,PasillaTranscriptExpr,PathNetData,pathprintGEOData,pcaGoPromoter.Hs.hg19,pcaGoPromoter.Mm.mm9,pcaGoPromoter.Rn.rn4,PCHiCdata,pcxnData,pd.atdschip.tiling,pepDat,PepsNMRData,PGPC,plasFIA,ppiData,prebsdata,PREDAsampledata,ProData,pRolocdata,prostateCancerCamcap,prostateCancerGrasso,prostateCancerStockholm,prostateCancerTaylor,prostateCancerVarambally,PtH2O2lipids,pumadata,PWMEnrich.Dmelanogaster.background,PWMEnrich.Hsapiens.background,PWMEnrich.Mmusculus.background,QDNAseq.hg19,QDNAseq.mm10,qPLEXdata,QUBICdata,rcellminerData,RcisTarget.hg19.motifDBs.cisbpOnly.500bp,RegParallel,restfulSEData,RforProteomics,RGMQLlib,rheumaticConditionWOLLBOLD,RIPSeekerData,RITANdata,RMassBankData,RNAinteractMAPK,RNAseqData.HNRNPC.bam.chr14,RNASeqRData,RnaSeqSampleSizeData,RnBeads.hg19,RnBeads.hg38,RnBeads.mm10,RnBeads.mm9,RnBeads.rn5,RRBSdata,rRDPData,RTCGA.clinical,RTCGA.CNV,RTCGA.methylation,RTCGA.miRNASeq,RTCGA.mRNA,RTCGA.mutations,RTCGA.PANCAN12,RTCGA.rnaseq,RTCGA.RPPA,RUVnormalizeData,sampleClassifierData,SCLCBam,scRNAseq,seq2pathway.data,seqc,seqCNA.annot,serumStimulation,sesameData,seventyGeneData,shinyMethylData,simpIntLists,Single.mTEC.Transcriptomes,SNAData,SNAGEEdata,SNPhoodData,SomatiCAData,SomaticCancerAlterations,SpikeIn,SpikeInSubset,stemHypoxia,stjudem,SVM2CRMdata,synapterdata,systemPipeRdata,TabulaMurisData,TargetScoreData,TargetSearchData,TBX20BamSubset,TCGAbiolinksGUI.data,TCGAcrcmiRNA,TCGAcrcmRNA,TCGAMethylation450k,tcgaWGBSData.hg19,TCGAWorkflowData,TENxBrainData,TENxPBMCData,TimerQuant,tinesath1cdf,tinesath1probe,tissueTreg,tofsimsData,topdownrdata,tweeDEseqCountData,tximportData,VariantToolsData,vulcandata,waveTilingData,WES.1KG.WUGSC,XhybCasneuf,yeastCC,yeastExpData,yeastGSData,yeastNagalakshmi,yeastRNASeq,yri1kgv,yriMulti,zebrafishRNASeq" id="ExperimentData" class="jstree-closed"><ins class="jstree-icon">&nbsp;</ins><a href="https://www.bioconductor.org/packages/release/BiocViews.html#"><ins class="jstree-icon">&nbsp;</ins>ExperimentData (360)</a><ul><li packagelist="ELMER.data,TCGAbiolinksGUI.data" id="AssayDomainData" class="jstree-closed"><ins class="jstree-icon">&nbsp;</ins><a href="https://www.bioconductor.org/packages/release/BiocViews.html#"><ins class="jstree-icon">&nbsp;</ins>AssayDomainData (61)</a><ul><li packagelist="AneuFinderData,cnvGSAdata,rcellminerData,seqCNA.annot" id="CopyNumberVariationData" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="https://www.bioconductor.org/packages/release/BiocViews.html#"><ins class="jstree-icon">&nbsp;</ins>CopyNumberVariationData (4)</a></li><li packagelist="COHCAPanno,RRBSdata" id="CpGIslandData" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="https://www.bioconductor.org/packages/release/BiocViews.html#"><ins class="jstree-icon">&nbsp;</ins>CpGIslandData (2)</a></li><li packagelist="ABAData,allenpvc,alpineData,brainImageRdata,ccdata,CellMapperData,CONFESSdata,curatedBladderData,curatedBreastData,Fletcher2013a,GeuvadisTranscriptExpr,HarmanData,HDCytoData,hgu133plus2CellScore,HumanAffyData,IHWpaper,KOdata,M3DExampleData,mammaPrintData,mAPKLData,MetaGxBreast,MetaGxPancreas,miRcompData,OMICsPCAdata,PasillaTranscriptExpr,pathprintGEOData,pcxnData,prostateCancerCamcap,prostateCancerGrasso,prostateCancerStockholm,prostateCancerTaylor,prostateCancerVarambally,rcellminerData,restfulSEData,sampleClassifierData,scRNAseq,TENxBrainData,TENxPBMCData" id="ExpressionData" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="https://www.bioconductor.org/packages/release/BiocViews.html#"><ins class="jstree-icon">&nbsp;</ins>ExpressionData (38)</a></li><li packagelist="ceu1kg,ceuhm3,cgdv17,DMRcatedata,Fletcher2013b,FunciSNP.data,GeuvadisTranscriptExpr,GGdata,GWASdata,hapmapsnp5,hapmapsnp6,hmyriB36,Illumina450ProbeVariants.db,leeBamViews,pd.atdschip.tiling,pumadata,rcellminerData,VariantToolsData" id="SNPData" class="jstree-last jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="https://www.bioconductor.org/packages/release/BiocViews.html#"><ins class="jstree-icon">&nbsp;</ins>SNPData (18)</a></li></ul></li><li packagelist="CLLmethylation,RegParallel" id="DiseaseModel" class="jstree-closed"><ins class="jstree-icon">&nbsp;</ins><a href="https://www.bioconductor.org/packages/release/BiocViews.html#"><ins class="jstree-icon">&nbsp;</ins>DiseaseModel (86)</a><ul><li packagelist="ALL,ALLMLL,antiProfilesData,aracne.networks,bcellViper,bladderbatch,BloodCancerMultiOmics2017,breastCancerMAINZ,breastCancerNKI,breastCancerTRANSBIG,breastCancerUNT,breastCancerUPP,breastCancerVDX,bsseqData,cancerdata,CLL,CLLmethylation,colonCA,ConnectivityMap,COSMIC.67,CRCL18,curatedBladderData,curatedBreastData,curatedOvarianData,curatedTCGAData,DeSousa2013,DLBCL,fabiaData,Fletcher2013a,Fletcher2013b,gageData,golubEsets,GSVAdata,h5vcData,hapmap100khind,hapmap100kxba,hapmap500knsp,hapmap500ksty,hapmapsnp5,HarmonizedTCGAData,HD2013SGI,Illumina450ProbeVariants.db,kidpack,KOdata,leukemiasEset,LungCancerACvsSCCGEO,LungCancerLines,lungExpression,mammaPrintData,maqcExpression4plex,MetaGxBreast,MetaGxPancreas,microRNAome,MIGSAdata,Mulder2012,Neve2006,NGScopyData,PGPC,PREDAsampledata,ProData,prostateCancerCamcap,prostateCancerGrasso,prostateCancerStockholm,prostateCancerTaylor,prostateCancerVarambally,rcellminerData,RnaSeqSampleSizeData,RRBSdata,seventyGeneData,shinyMethylData,SomatiCAData,SomaticCancerAlterations,stemHypoxia,stjudem,TCGAcrcmiRNA,TCGAcrcmRNA,TCGAMethylation450k,tcgaWGBSData.hg19,TCGAWorkflowData,vulcandata,yeastCC" id="CancerData" class="jstree-closed"><ins class="jstree-icon">&nbsp;</ins><a href="https://www.bioconductor.org/packages/release/BiocViews.html#"><ins class="jstree-icon">&nbsp;</ins>CancerData (83)</a><ul><li packagelist="breastCancerMAINZ,breastCancerNKI,breastCancerTRANSBIG,breastCancerUNT,breastCancerUPP,breastCancerVDX,curatedBreastData,fabiaData,Fletcher2013a,Fletcher2013b,gageData,mammaPrintData,MetaGxBreast,MIGSAdata,Neve2006,ProData,seventyGeneData" id="BreastCancerData" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="https://www.bioconductor.org/packages/release/BiocViews.html#"><ins class="jstree-icon">&nbsp;</ins>BreastCancerData (17)</a></li><li packagelist="antiProfilesData,bsseqData,colonCA,DeSousa2013,HD2013SGI,PGPC" id="ColonCancerData" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="https://www.bioconductor.org/packages/release/BiocViews.html#"><ins class="jstree-icon">&nbsp;</ins>ColonCancerData (6)</a></li><li packagelist="kidpack,PREDAsampledata" id="KidneyCancerData" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="https://www.bioconductor.org/packages/release/BiocViews.html#"><ins class="jstree-icon">&nbsp;</ins>KidneyCancerData (2)</a></li><li packagelist="ALL,ALLMLL,BloodCancerMultiOmics2017,CLL,CLLmethylation,golubEsets,GSVAdata,leukemiasEset,stjudem" id="LeukemiaCancerData" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="https://www.bioconductor.org/packages/release/BiocViews.html#"><ins class="jstree-icon">&nbsp;</ins>LeukemiaCancerData (9)</a></li><li packagelist="AneuFinderData,breastCancerVDX,LungCancerACvsSCCGEO,LungCancerLines,lungExpression,NGScopyData,SCLCBam" id="LungCancerData" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="https://www.bioconductor.org/packages/release/BiocViews.html#"><ins class="jstree-icon">&nbsp;</ins>LungCancerData (7)</a></li><li packagelist="curatedBladderData,curatedOvarianData" id="OvarianCancerData" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="https://www.bioconductor.org/packages/release/BiocViews.html#"><ins class="jstree-icon">&nbsp;</ins>OvarianCancerData (2)</a></li><li packagelist="prostateCancerCamcap,prostateCancerGrasso,prostateCancerStockholm,prostateCancerTaylor,prostateCancerVarambally" id="ProstateCancerData" class="jstree-last jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="https://www.bioconductor.org/packages/release/BiocViews.html#"><ins class="jstree-icon">&nbsp;</ins>ProstateCancerData (5)</a></li></ul></li><li packagelist="COPDSexualDimorphism.data" id="COPDData" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="https://www.bioconductor.org/packages/release/BiocViews.html#"><ins class="jstree-icon">&nbsp;</ins>COPDData (1)</a></li><li packagelist="HIVcDNAvantWout03" id="HIVData" class="jstree-last jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="https://www.bioconductor.org/packages/release/BiocViews.html#"><ins class="jstree-icon">&nbsp;</ins>HIVData (1)</a></li></ul></li><li packagelist="AssessORFData,brainImageRdata,ELMER.data,PepsNMRData,QDNAseq.hg19,QDNAseq.mm10,QUBICdata,TCGAbiolinksGUI.data" id="OrganismData" class="jstree-closed"><ins class="jstree-icon">&nbsp;</ins><a href="https://www.bioconductor.org/packages/release/BiocViews.html#"><ins class="jstree-icon">&nbsp;</ins>OrganismData (123)</a><ul><li class="jstree-leaf" style="display: none;"><ins class="jstree-icon">&nbsp;</ins><a href="https://www.bioconductor.org/packages/release/BiocViews.html#"><ins class="jstree-icon">&nbsp;</ins>Anopheles_gambiae_Data</a></li><li class="jstree-leaf" style="display: none;"><ins class="jstree-icon">&nbsp;</ins><a href="https://www.bioconductor.org/packages/release/BiocViews.html#"><ins class="jstree-icon">&nbsp;</ins>Apis_mellifera_Data</a></li><li class="jstree-leaf" style="display: none;"><ins class="jstree-icon">&nbsp;</ins><a href="https://www.bioconductor.org/packages/release/BiocViews.html#"><ins class="jstree-icon">&nbsp;</ins>Arabidopsis_lyrata_Data</a></li><li packagelist="pd.atdschip.tiling,pRolocdata,simpIntLists,tinesath1cdf,tinesath1probe,waveTilingData" id="Arabidopsis_thaliana_Data" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="https://www.bioconductor.org/packages/release/BiocViews.html#"><ins class="jstree-icon">&nbsp;</ins>Arabidopsis_thaliana_Data (6)</a></li><li packagelist="AssessORFData" id="Bacillus_subtilis_Data" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="https://www.bioconductor.org/packages/release/BiocViews.html#"><ins class="jstree-icon">&nbsp;</ins>Bacillus_subtilis_Data (1)</a></li><li class="jstree-leaf" style="display: none;"><ins class="jstree-icon">&nbsp;</ins><a href="https://www.bioconductor.org/packages/release/BiocViews.html#"><ins class="jstree-icon">&nbsp;</ins>Bos_taurus_Data</a></li><li packagelist="pathprintGEOData" id="Caenorhabditis_elegans_Data" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="https://www.bioconductor.org/packages/release/BiocViews.html#"><ins class="jstree-icon">&nbsp;</ins>Caenorhabditis_elegans_Data (1)</a></li><li class="jstree-leaf" style="display: none;"><ins class="jstree-icon">&nbsp;</ins><a href="https://www.bioconductor.org/packages/release/BiocViews.html#"><ins class="jstree-icon">&nbsp;</ins>Canis_familiaris_Data</a></li><li class="jstree-leaf" style="display: none;"><ins class="jstree-icon">&nbsp;</ins><a href="https://www.bioconductor.org/packages/release/BiocViews.html#"><ins class="jstree-icon">&nbsp;</ins>Ciona_intestinalis_Data</a></li><li packagelist="pathprintGEOData,TimerQuant,zebrafishRNASeq" id="Danio_rerio_Data" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="https://www.bioconductor.org/packages/release/BiocViews.html#"><ins class="jstree-icon">&nbsp;</ins>Danio_rerio_Data (3)</a></li><li packagelist="DmelSGI,furrowSeg,pasilla,PasillaTranscriptExpr,pathprintGEOData,pRolocdata,PWMEnrich.Dmelanogaster.background,RNAinteractMAPK" id="Drosophila_melanogaster_Data" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="https://www.bioconductor.org/packages/release/BiocViews.html#"><ins class="jstree-icon">&nbsp;</ins>Drosophila_melanogaster_Data (8)</a></li><li class="jstree-leaf" style="display: none;"><ins class="jstree-icon">&nbsp;</ins><a href="https://www.bioconductor.org/packages/release/BiocViews.html#"><ins class="jstree-icon">&nbsp;</ins>Eremothecium_gossypii_Data</a></li><li packagelist="AssessORFData,QUBICdata,TargetSearchData" id="Escherichia_coli_Data" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="https://www.bioconductor.org/packages/release/BiocViews.html#"><ins class="jstree-icon">&nbsp;</ins>Escherichia_coli_Data (3)</a></li><li class="jstree-leaf" style="display: none;"><ins class="jstree-icon">&nbsp;</ins><a href="https://www.bioconductor.org/packages/release/BiocViews.html#"><ins class="jstree-icon">&nbsp;</ins>Gallus_gallus_Data</a></li><li class="jstree-leaf" style="display: none;"><ins class="jstree-icon">&nbsp;</ins><a href="https://www.bioconductor.org/packages/release/BiocViews.html#"><ins class="jstree-icon">&nbsp;</ins>Gasterosteus_aculeatus_Data</a></li><li class="jstree-leaf" style="display: none;"><ins class="jstree-icon">&nbsp;</ins><a href="https://www.bioconductor.org/packages/release/BiocViews.html#"><ins class="jstree-icon">&nbsp;</ins>Glycine_max_Data</a></li><li packagelist="ABAData,Affyhgu133A2Expr,Affyhgu133aExpr,Affyhgu133Plus2Expr,AneuFinderData,aracne.networks,ASICSdata,bcellViper,beadarrayExampleData,beta7,brainImageRdata,breakpointRdata,breastCancerVDX,brgedata,ChimpHumanBrainData,ChIPXpressData,cMap2data,COHCAPanno,CONFESSdata,CopyNeutralIMA,curatedMetagenomicData,curatedTCGAData,derfinderData,DrugVsDiseasedata,EGSEAdata,fibroEset,FIs,FlowSorted.Blood.450k,FlowSorted.Blood.EPIC,FlowSorted.CordBlood.450k,FlowSorted.CordBloodNorway.450k,FlowSorted.DLPFC.450k,frmaExampleData,GeuvadisTranscriptExpr,GIGSEAdata,GSVAdata,HarmanData,HD2013SGI,HDCytoData,hgu133abarcodevecs,hgu133plus2barcodevecs,hgu133plus2CellScore,HiCDataHumanIMR90,HiCDataLymphoblast,HMP16SData,HumanAffyData,humanStemCell,Illumina450ProbeVariants.db,KOdata,leukemiasEset,ListerEtAlBSseq,MAQCsubsetAFX,MAQCsubsetILM,mCSEAdata,mcsurvdata,MetaGxBreast,MetaGxPancreas,minfiData,minfiDataEPIC,miRcompData,miRNATarget,pathprintGEOData,pcaGoPromoter.Hs.hg19,PCHiCdata,pcxnData,PepsNMRData,pRolocdata,PWMEnrich.Hsapiens.background,QDNAseq.hg19,RcisTarget.hg19.motifDBs.cisbpOnly.500bp,restfulSEData,RNAseqData.HNRNPC.bam.chr14,stemHypoxia,tcgaWGBSData.hg19,TCGAWorkflowData,tweeDEseqCountData,vulcandata,WES.1KG.WUGSC" id="Homo_sapiens_Data" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="https://www.bioconductor.org/packages/release/BiocViews.html#"><ins class="jstree-icon">&nbsp;</ins>Homo_sapiens_Data (78)</a></li><li class="jstree-leaf" style="display: none;"><ins class="jstree-icon">&nbsp;</ins><a href="https://www.bioconductor.org/packages/release/BiocViews.html#"><ins class="jstree-icon">&nbsp;</ins>Hordeum_vulgare_Data</a></li><li class="jstree-leaf" style="display: none;"><ins class="jstree-icon">&nbsp;</ins><a href="https://www.bioconductor.org/packages/release/BiocViews.html#"><ins class="jstree-icon">&nbsp;</ins>Kluyveromyces_lactis_Data</a></li><li class="jstree-leaf" style="display: none;"><ins class="jstree-icon">&nbsp;</ins><a href="https://www.bioconductor.org/packages/release/BiocViews.html#"><ins class="jstree-icon">&nbsp;</ins>Macaca_mulatta_Data</a></li><li class="jstree-leaf" style="display: none;"><ins class="jstree-icon">&nbsp;</ins><a href="https://www.bioconductor.org/packages/release/BiocViews.html#"><ins class="jstree-icon">&nbsp;</ins>Magnaporthe_grisea_Data</a></li><li class="jstree-leaf" style="display: none;"><ins class="jstree-icon">&nbsp;</ins><a href="https://www.bioconductor.org/packages/release/BiocViews.html#"><ins class="jstree-icon">&nbsp;</ins>Medicago_truncatul_Dataa</a></li><li packagelist="Affymoe4302Expr,ccTutorial,ChIPXpressData,chromstaRData,EGSEAdata,HarmanData,M3DExampleData,MMDiffBamSubset,mouse4302barcodevecs,MUGAExampleData,pathprintGEOData,pcaGoPromoter.Mm.mm9,PCHiCdata,pRolocdata,PWMEnrich.Mmusculus.background,QDNAseq.mm10,restfulSEData,tissueTreg" id="Mus_musculus_Data" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="https://www.bioconductor.org/packages/release/BiocViews.html#"><ins class="jstree-icon">&nbsp;</ins>Mus_musculus_Data (18)</a></li><li class="jstree-leaf" style="display: none;"><ins class="jstree-icon">&nbsp;</ins><a href="https://www.bioconductor.org/packages/release/BiocViews.html#"><ins class="jstree-icon">&nbsp;</ins>Neurospora_crassa_Data</a></li><li class="jstree-leaf" style="display: none;"><ins class="jstree-icon">&nbsp;</ins><a href="https://www.bioconductor.org/packages/release/BiocViews.html#"><ins class="jstree-icon">&nbsp;</ins>Oncorhynchus_mykiss_Data</a></li><li class="jstree-leaf" style="display: none;"><ins class="jstree-icon">&nbsp;</ins><a href="https://www.bioconductor.org/packages/release/BiocViews.html#"><ins class="jstree-icon">&nbsp;</ins>Oryza_sativa_Data</a></li><li packagelist="ChimpHumanBrainData" id="Pan_troglodytes_Data" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="https://www.bioconductor.org/packages/release/BiocViews.html#"><ins class="jstree-icon">&nbsp;</ins>Pan_troglodytes_Data (1)</a></li><li class="jstree-leaf" style="display: none;"><ins class="jstree-icon">&nbsp;</ins><a href="https://www.bioconductor.org/packages/release/BiocViews.html#"><ins class="jstree-icon">&nbsp;</ins>Plasmodium_falciparum_Data</a></li><li packagelist="AssessORFData" id="Pseudomonas_aeruginosa_Data" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="https://www.bioconductor.org/packages/release/BiocViews.html#"><ins class="jstree-icon">&nbsp;</ins>Pseudomonas_aeruginosa_Data (1)</a></li><li packagelist="CCl4,EGSEAdata,JctSeqData,pathprintGEOData,pcaGoPromoter.Rn.rn4" id="Rattus_norvegicus_Data" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="https://www.bioconductor.org/packages/release/BiocViews.html#"><ins class="jstree-icon">&nbsp;</ins>Rattus_norvegicus_Data (5)</a></li><li packagelist="davidTiling,DREAM4,EatonEtAlChIPseq,fission,gaschYHS,harbChIP,leeBamViews,RNASeqRData,yeastCC,yeastExpData,yeastGSData,yeastNagalakshmi,yeastRNASeq" id="Saccharomyces_cerevisiae_Data" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="https://www.bioconductor.org/packages/release/BiocViews.html#"><ins class="jstree-icon">&nbsp;</ins>Saccharomyces_cerevisiae_Data (13)</a></li><li class="jstree-leaf" style="display: none;"><ins class="jstree-icon">&nbsp;</ins><a href="https://www.bioconductor.org/packages/release/BiocViews.html#"><ins class="jstree-icon">&nbsp;</ins>Saccharum_officinarum_Data</a></li><li class="jstree-leaf" style="display: none;"><ins class="jstree-icon">&nbsp;</ins><a href="https://www.bioconductor.org/packages/release/BiocViews.html#"><ins class="jstree-icon">&nbsp;</ins>Schizosaccharomyces_pombe_Data</a></li><li packagelist="AssessORFData" id="Staphylococcus_aureus_Data" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="https://www.bioconductor.org/packages/release/BiocViews.html#"><ins class="jstree-icon">&nbsp;</ins>Staphylococcus_aureus_Data (1)</a></li><li class="jstree-leaf" style="display: none;"><ins class="jstree-icon">&nbsp;</ins><a href="https://www.bioconductor.org/packages/release/BiocViews.html#"><ins class="jstree-icon">&nbsp;</ins>Sus_scrofa_Data</a></li><li class="jstree-leaf" style="display: none;"><ins class="jstree-icon">&nbsp;</ins><a href="https://www.bioconductor.org/packages/release/BiocViews.html#"><ins class="jstree-icon">&nbsp;</ins>Taeniopygia_guttata_Data</a></li><li class="jstree-leaf" style="display: none;"><ins class="jstree-icon">&nbsp;</ins><a href="https://www.bioconductor.org/packages/release/BiocViews.html#"><ins class="jstree-icon">&nbsp;</ins>Triticum_aestivum_Data</a></li><li class="jstree-leaf" style="display: none;"><ins class="jstree-icon">&nbsp;</ins><a href="https://www.bioconductor.org/packages/release/BiocViews.html#"><ins class="jstree-icon">&nbsp;</ins>Vitis_vinifera_Data</a></li><li class="jstree-leaf" style="display: none;"><ins class="jstree-icon">&nbsp;</ins><a href="https://www.bioconductor.org/packages/release/BiocViews.html#"><ins class="jstree-icon">&nbsp;</ins>Xenopus_laevis_Data</a></li><li class="jstree-leaf" style="display: none;"><ins class="jstree-icon">&nbsp;</ins><a href="https://www.bioconductor.org/packages/release/BiocViews.html#"><ins class="jstree-icon">&nbsp;</ins>Xenopus_tropicalis_Data</a></li><li class="jstree-last jstree-leaf" style="display: none;"><ins class="jstree-icon">&nbsp;</ins><a href="https://www.bioconductor.org/packages/release/BiocViews.html#"><ins class="jstree-icon">&nbsp;</ins>Zea_mays_Data</a></li></ul></li><li packagelist="curatedMetagenomicData,HDCytoData,HDCytoData" id="PackageTypeData" class="jstree-closed"><ins class="jstree-icon">&nbsp;</ins><a href="https://www.bioconductor.org/packages/release/BiocViews.html#"><ins class="jstree-icon">&nbsp;</ins>PackageTypeData (2)</a><ul><li packagelist="curatedMetagenomicData,HDCytoData" id="ExperimentHub" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="https://www.bioconductor.org/packages/release/BiocViews.html#"><ins class="jstree-icon">&nbsp;</ins>ExperimentHub (2)</a></li><li packagelist="HDCytoData" id="ImmunoOncologyData" class="jstree-last jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="https://www.bioconductor.org/packages/release/BiocViews.html#"><ins class="jstree-icon">&nbsp;</ins>ImmunoOncologyData (1)</a></li></ul></li><li packagelist="JctSeqData,MIGSAdata,mtbls2,OMICsPCAdata,RGMQLlib" id="RepositoryData" class="jstree-closed"><ins class="jstree-icon">&nbsp;</ins><a href="https://www.bioconductor.org/packages/release/BiocViews.html#"><ins class="jstree-icon">&nbsp;</ins>RepositoryData (85)</a><ul><li packagelist="ceu1kgv,COHCAPanno,dyebiasexamples,geuvPack,hgu133plus2CellScore,HumanAffyData,kidpack,MetaGxBreast,MetaGxPancreas,MMDiffBamSubset,PREDAsampledata,rheumaticConditionWOLLBOLD,RNAseqData.HNRNPC.bam.chr14,sampleClassifierData,yri1kgv" id="ArrayExpress" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="https://www.bioconductor.org/packages/release/BiocViews.html#"><ins class="jstree-icon">&nbsp;</ins>ArrayExpress (15)</a></li><li packagelist="ChIC.data,FunciSNP.data,genomationData,LiebermanAidenHiC2009,OMICsPCAdata" id="ENCODE" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="https://www.bioconductor.org/packages/release/BiocViews.html#"><ins class="jstree-icon">&nbsp;</ins>ENCODE (5)</a></li><li packagelist="Affyhgu133A2Expr,Affyhgu133aExpr,Affyhgu133Plus2Expr,Affymoe4302Expr,airway,blimaTestingData,breastCancerMAINZ,breastCancerTRANSBIG,breastCancerUNT,breastCancerUPP,breastCancerVDX,ChimpHumanBrainData,ChIPXpressData,cMap2data,COHCAPanno,CopyNeutralIMA,curatedBreastData,DvDdata,EatonEtAlChIPseq,ffpeExampleData,fission,gageData,hgu133plus2CellScore,HiCDataHumanIMR90,HumanAffyData,JctSeqData,KEGGandMetacoreDzPathwaysGEO,KEGGdzPathwaysGEO,leukemiasEset,LiebermanAidenHiC2009,LungCancerACvsSCCGEO,M3DExampleData,MAQCsubset,mcsurvdata,MetaGxBreast,MetaGxPancreas,PasillaTranscriptExpr,pathprintGEOData,pcxnData,prostateCancerCamcap,prostateCancerGrasso,prostateCancerStockholm,prostateCancerTaylor,prostateCancerVarambally,rheumaticConditionWOLLBOLD,RIPSeekerData,RNAseqData.HNRNPC.bam.chr14,stemHypoxia,yeastCC" id="GEO" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="https://www.bioconductor.org/packages/release/BiocViews.html#"><ins class="jstree-icon">&nbsp;</ins>GEO (49)</a></li><li packagelist="ceu1kg,ceuhm3,GGdata,GWASdata,hapmap100khind,hapmap100kxba,hapmap500knsp,hapmap500ksty,hapmapsnp5,hapmapsnp6,hmyriB36,ITALICSData,VariantToolsData" id="HapMap" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="https://www.bioconductor.org/packages/release/BiocViews.html#"><ins class="jstree-icon">&nbsp;</ins>HapMap (13)</a></li><li packagelist="dsQTL,LiebermanAidenHiC2009,MetaGxBreast,MetaGxPancreas,NCIgraphData,ProData,rcellminerData,RNAseqData.HNRNPC.bam.chr14,SomaticCancerAlterations" id="NCI" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="https://www.bioconductor.org/packages/release/BiocViews.html#"><ins class="jstree-icon">&nbsp;</ins>NCI (9)</a></li><li packagelist="FIs,PathNetData,pathprintGEOData,pcxnData" id="PathwayInteractionDatabase" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="https://www.bioconductor.org/packages/release/BiocViews.html#"><ins class="jstree-icon">&nbsp;</ins>PathwayInteractionDatabase (4)</a></li><li packagelist="ceu1kg,dsQTL,FunciSNP.data,SomaticCancerAlterations,WES.1KG.WUGSC" id="Project1000genomes" class="jstree-last jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="https://www.bioconductor.org/packages/release/BiocViews.html#"><ins class="jstree-icon">&nbsp;</ins>Project1000genomes (5)</a></li></ul></li><li packagelist="BloodCancerMultiOmics2017,curatedMetagenomicData,curatedTCGAData,davidTiling,DmelSGI,furrowSeg,HarmonizedTCGAData,Hiiragi2013,HMP16SData,IHWpaper,MIGSAdata,MSBdata,MSMB,msqc1,PtH2O2lipids,RforProteomics" id="ReproducibleResearch" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="https://www.bioconductor.org/packages/release/BiocViews.html#"><ins class="jstree-icon">&nbsp;</ins>ReproducibleResearch (16)</a></li><li packagelist="affydata,Affyhgu133A2Expr,Affyhgu133Plus2Expr,Affymoe4302Expr,antiProfilesData,aracne.networks,AssessORFData,AssessORFData,bcellViper,breakpointRdata,breastCancerVDX,bsseqData,CCl4,ChimpHumanBrainData,ChIPexoQualExample,chromstaRData,cnvGSAdata,colonCA,CONFESSdata,COPDSexualDimorphism.data,COSMIC.67,curatedBreastData,davidTiling,DmelSGI,DonaPLLP2013,DREAM4,dressCheck,dsQTL,facopy.annot,FANTOM3and4CAGE,ffpeExampleData,ffpeExampleData,fibroEset,fission,FlowSorted.Blood.450k,FlowSorted.Blood.EPIC,FlowSorted.Blood.EPIC,FlowSorted.CordBlood.450k,FlowSorted.CordBloodNorway.450k,FlowSorted.DLPFC.450k,furrowSeg,gageData,gaschYHS,geneLenDataBase,genomationData,genomationData,GeuvadisTranscriptExpr,geuvPack,GGdata,golubEsets,GSE62944,HD2013SGI,hgu133plus2CellScore,hgu2beta7,HiCDataHumanIMR90,hmyriB36,JctSeqData,leukemiasEset,leukemiasEset,LiebermanAidenHiC2009,lumiBarnes,LungCancerLines,M3DExampleData,MAQCsubsetAFX,MEDIPSData,microRNAome,mitoODEdata,MMDiffBamSubset,MMDiffBamSubset,MSstatsBioData,MTseekerData,Mulder2012,pasilla,pasillaBamSubset,PasillaTranscriptExpr,pathprintGEOData,PCHiCdata,PREDAsampledata,pRolocdata,pRolocdata,PtH2O2lipids,qPLEXdata,rheumaticConditionWOLLBOLD,RIPSeekerData,RIPSeekerData,RNAinteractMAPK,RNAseqData.HNRNPC.bam.chr14,seqCNA.annot,sesameData,shinyMethylData,SomatiCAData,SomaticCancerAlterations,stemHypoxia,stemHypoxia,TCGAMethylation450k,tcgaWGBSData.hg19,TimerQuant,tissueTreg,tweeDEseqCountData,XhybCasneuf,yeastCC,yeastExpData,yeastNagalakshmi" id="SpecimenSource" class="jstree-closed"><ins class="jstree-icon">&nbsp;</ins><a href="https://www.bioconductor.org/packages/release/BiocViews.html#"><ins class="jstree-icon">&nbsp;</ins>SpecimenSource (94)</a><ul><li packagelist="CONFESSdata,DmelSGI,HD2013SGI,microRNAome,PtH2O2lipids,RNAinteractMAPK,yeastCC" id="CellCulture" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="https://www.bioconductor.org/packages/release/BiocViews.html#"><ins class="jstree-icon">&nbsp;</ins>CellCulture (7)</a></li><li packagelist="Affyhgu133A2Expr,Affyhgu133Plus2Expr,Affymoe4302Expr,aracne.networks,AssessORFData,bcellViper,breakpointRdata,breastCancerVDX,bsseqData,CCl4,ChIPexoQualExample,cnvGSAdata,COSMIC.67,davidTiling,DREAM4,dressCheck,dsQTL,facopy.annot,ffpeExampleData,fibroEset,fission,FlowSorted.Blood.EPIC,gaschYHS,geneLenDataBase,genomationData,GeuvadisTranscriptExpr,geuvPack,GGdata,golubEsets,GSE62944,hgu133plus2CellScore,hgu2beta7,HiCDataHumanIMR90,hmyriB36,JctSeqData,leukemiasEset,LiebermanAidenHiC2009,LungCancerLines,MAQCsubsetAFX,MEDIPSData,mitoODEdata,MMDiffBamSubset,MTseekerData,pasilla,pasillaBamSubset,PasillaTranscriptExpr,pathprintGEOData,RIPSeekerData,RNAseqData.HNRNPC.bam.chr14,seqCNA.annot,sesameData,shinyMethylData,SomatiCAData,SomaticCancerAlterations,TCGAMethylation450k,tcgaWGBSData.hg19,tweeDEseqCountData,yeastExpData,yeastNagalakshmi" id="Genome" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="https://www.bioconductor.org/packages/release/BiocViews.html#"><ins class="jstree-icon">&nbsp;</ins>Genome (59)</a></li><li class="jstree-leaf" style="display: none;"><ins class="jstree-icon">&nbsp;</ins><a href="https://www.bioconductor.org/packages/release/BiocViews.html#"><ins class="jstree-icon">&nbsp;</ins>Germline</a></li><li packagelist="AssessORFData,MSstatsBioData,pRolocdata,qPLEXdata" id="Proteome" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="https://www.bioconductor.org/packages/release/BiocViews.html#"><ins class="jstree-icon">&nbsp;</ins>Proteome (4)</a></li><li class="jstree-leaf" style="display: none;"><ins class="jstree-icon">&nbsp;</ins><a href="https://www.bioconductor.org/packages/release/BiocViews.html#"><ins class="jstree-icon">&nbsp;</ins>Somatic</a></li><li packagelist="chromstaRData,gageData,genomationData,MMDiffBamSubset,Mulder2012,PCHiCdata,pRolocdata,RIPSeekerData,stemHypoxia" id="StemCell" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="https://www.bioconductor.org/packages/release/BiocViews.html#"><ins class="jstree-icon">&nbsp;</ins>StemCell (9)</a></li><li packagelist="affydata,antiProfilesData,ChimpHumanBrainData,colonCA,COPDSexualDimorphism.data,curatedBreastData,DonaPLLP2013,FANTOM3and4CAGE,ffpeExampleData,FlowSorted.Blood.450k,FlowSorted.Blood.EPIC,FlowSorted.CordBlood.450k,FlowSorted.CordBloodNorway.450k,FlowSorted.DLPFC.450k,furrowSeg,leukemiasEset,lumiBarnes,M3DExampleData,PREDAsampledata,rheumaticConditionWOLLBOLD,stemHypoxia,TimerQuant,tissueTreg,XhybCasneuf" id="Tissue" class="jstree-last jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="https://www.bioconductor.org/packages/release/BiocViews.html#"><ins class="jstree-icon">&nbsp;</ins>Tissue (24)</a></li></ul></li><li packagelist="ELMER.data,MethylAidData,OMICsPCAdata,TCGAbiolinksGUI.data" id="TechnologyData" class="jstree-last jstree-closed"><ins class="jstree-icon">&nbsp;</ins><a href="https://www.bioconductor.org/packages/release/BiocViews.html#"><ins class="jstree-icon">&nbsp;</ins>TechnologyData (230)</a><ul><li packagelist="beta7,breastCancerNKI,dyebiasexamples,PWMEnrich.Hsapiens.background" id="CGHData" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="https://www.bioconductor.org/packages/release/BiocViews.html#"><ins class="jstree-icon">&nbsp;</ins>CGHData (4)</a></li><li packagelist="flowFitExampleData,flowPloidyData,flowQBData,flowWorkspaceData,gatingMLData,HDCytoData,healthyFlowData" id="FlowCytometryData" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="https://www.bioconductor.org/packages/release/BiocViews.html#"><ins class="jstree-icon">&nbsp;</ins>FlowCytometryData (7)</a></li><li packagelist="CONFESSdata,DmelSGI,HD2013SGI,TimerQuant" id="HighThroughputImagingData" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="https://www.bioconductor.org/packages/release/BiocViews.html#"><ins class="jstree-icon">&nbsp;</ins>HighThroughputImagingData (4)</a></li><li packagelist="CardinalWorkflows,DAPARdata,faahKO,gcspikelite,iontreeData,metaMSdata,msdata,msPurityData,msqc1,MSstatsBioData,mtbls2,plasFIA,ProData,pRolocdata,PtH2O2lipids,qPLEXdata,RforProteomics,RMassBankData,synapterdata,topdownrdata" id="MassSpectrometryData" class="jstree-closed"><ins class="jstree-icon">&nbsp;</ins><a href="https://www.bioconductor.org/packages/release/BiocViews.html#"><ins class="jstree-icon">&nbsp;</ins>MassSpectrometryData (20)</a><ul><li packagelist="CardinalWorkflows" id="ImagingMassSpectrometryData" class="jstree-last jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="https://www.bioconductor.org/packages/release/BiocViews.html#"><ins class="jstree-icon">&nbsp;</ins>ImagingMassSpectrometryData (1)</a></li></ul></li><li packagelist="ABAData,affycompData,affydata,AffymetrixDataTestFiles,ALLMLL,AmpAffyExample,antiProfilesData,beadarrayExampleData,BeadArrayUseCases,beta7,bladderbatch,blimaTestingData,breastCancerMAINZ,breastCancerNKI,breastCancerTRANSBIG,breastCancerUNT,breastCancerUPP,breastCancerVDX,brgedata,bronchialIL13,cancerdata,ccdata,CCl4,ccTutorial,CellMapperData,charmData,ChimpHumanBrainData,CLL,cMap2data,COHCAPanno,colonCA,ConnectivityMap,CopyNeutralIMA,curatedBladderData,curatedBreastData,curatedOvarianData,davidTiling,DeSousa2013,DLBCL,DrugVsDiseasedata,dyebiasexamples,ecoliLeucine,estrogen,fabiaData,ffpeExampleData,fibroEset,Fletcher2013a,FlowSorted.Blood.450k,FlowSorted.Blood.EPIC,FlowSorted.CordBlood.450k,FlowSorted.CordBloodNorway.450k,FlowSorted.DLPFC.450k,frmaExampleData,gageData,geuvPack,geuvStore2,GGdata,GSBenchMark,GWASdata,HarmanData,hgu133abarcodevecs,hgu133plus2barcodevecs,hgu133plus2CellScore,Hiiragi2013,HIVcDNAvantWout03,HumanAffyData,IlluminaDataTestFiles,KEGGdzPathwaysGEO,kidpack,leukemiasEset,lumiBarnes,LungCancerACvsSCCGEO,lydata,mammaPrintData,MAQCsubset,MAQCsubsetAFX,MAQCsubsetILM,mCSEAdata,mcsurvdata,MetaGxBreast,MetaGxPancreas,MethylAidData,MIGSAdata,minfiData,minfiDataEPIC,mouse4302barcodevecs,mvoutData,pathprintGEOData,pcxnData,pd.atdschip.tiling,pepDat,PREDAsampledata,prostateCancerCamcap,prostateCancerGrasso,prostateCancerStockholm,prostateCancerTaylor,prostateCancerVarambally,pumadata,rcellminerData,rheumaticConditionWOLLBOLD,RUVnormalizeData,sampleClassifierData,serumStimulation,sesameData,seventyGeneData,SNAGEEdata,SpikeIn,SpikeInSubset,stemHypoxia,stjudem,TCGAWorkflowData,tinesath1probe,waveTilingData,XhybCasneuf,yeastCC" id="MicroarrayData" class="jstree-closed"><ins class="jstree-icon">&nbsp;</ins><a href="https://www.bioconductor.org/packages/release/BiocViews.html#"><ins class="jstree-icon">&nbsp;</ins>MicroarrayData (121)</a><ul><li packagelist="breastCancerNKI,ccTutorial,DLBCL,fibroEset,genomationData,Illumina450ProbeVariants.db,leukemiasEset,LiebermanAidenHiC2009,lumiBarnes,rheumaticConditionWOLLBOLD,stjudem,tinesath1cdf" id="ChipOnChipData" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="https://www.bioconductor.org/packages/release/BiocViews.html#"><ins class="jstree-icon">&nbsp;</ins>ChipOnChipData (12)</a></li><li class="jstree-leaf" style="display: none;"><ins class="jstree-icon">&nbsp;</ins><a href="https://www.bioconductor.org/packages/release/BiocViews.html#"><ins class="jstree-icon">&nbsp;</ins>GenotypingArrayData</a></li><li packagelist="ARRmData,brgedata,COHCAPanno,CopyNeutralIMA,FlowSorted.Blood.450k,FlowSorted.Blood.EPIC,FlowSorted.CordBlood.450k,FlowSorted.CordBloodNorway.450k,FlowSorted.DLPFC.450k,mCSEAdata,MethylAidData,minfiData,minfiDataEPIC,TCGAMethylation450k" id="MethylationArrayData" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="https://www.bioconductor.org/packages/release/BiocViews.html#"><ins class="jstree-icon">&nbsp;</ins>MethylationArrayData (14)</a></li><li class="jstree-leaf" style="display: none;"><ins class="jstree-icon">&nbsp;</ins><a href="https://www.bioconductor.org/packages/release/BiocViews.html#"><ins class="jstree-icon">&nbsp;</ins>MicroRNAArrayData</a></li><li class="jstree-leaf" style="display: none;"><ins class="jstree-icon">&nbsp;</ins><a href="https://www.bioconductor.org/packages/release/BiocViews.html#"><ins class="jstree-icon">&nbsp;</ins>mRNAArrayData</a></li><li packagelist="rheumaticConditionWOLLBOLD" id="MultiChannelData" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="https://www.bioconductor.org/packages/release/BiocViews.html#"><ins class="jstree-icon">&nbsp;</ins>MultiChannelData (1)</a></li><li packagelist="breastCancerNKI,breastCancerVDX,pathprintGEOData,pcxnData,yeastCC" id="OneChannelData" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="https://www.bioconductor.org/packages/release/BiocViews.html#"><ins class="jstree-icon">&nbsp;</ins>OneChannelData (5)</a></li><li class="jstree-leaf" style="display: none;"><ins class="jstree-icon">&nbsp;</ins><a href="https://www.bioconductor.org/packages/release/BiocViews.html#"><ins class="jstree-icon">&nbsp;</ins>ProprietaryPlatformsData</a></li><li class="jstree-leaf" style="display: none;"><ins class="jstree-icon">&nbsp;</ins><a href="https://www.bioconductor.org/packages/release/BiocViews.html#"><ins class="jstree-icon">&nbsp;</ins>ReversePhaseProteinArrayData</a></li><li packagelist="affydata,ChimpHumanBrainData,colonCA,curatedBreastData,ffpeExampleData,FlowSorted.Blood.450k,FlowSorted.Blood.EPIC,FlowSorted.CordBlood.450k,FlowSorted.CordBloodNorway.450k,FlowSorted.DLPFC.450k,leukemiasEset,PREDAsampledata,rheumaticConditionWOLLBOLD,stemHypoxia,XhybCasneuf" id="TissueMicroarrayData" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="https://www.bioconductor.org/packages/release/BiocViews.html#"><ins class="jstree-icon">&nbsp;</ins>TissueMicroarrayData (15)</a></li><li packagelist="breastCancerUNT,CopyNeutralIMA,dyebiasexamples,HIVcDNAvantWout03,mammaPrintData,pathprintGEOData" id="TwoChannelData" class="jstree-last jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="https://www.bioconductor.org/packages/release/BiocViews.html#"><ins class="jstree-icon">&nbsp;</ins>TwoChannelData (6)</a></li></ul></li><li packagelist="DmelSGI,facsDorit,HD2013SGI,RNAinteractMAPK" id="MicrotitrePlateAssayData" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="https://www.bioconductor.org/packages/release/BiocViews.html#"><ins class="jstree-icon">&nbsp;</ins>MicrotitrePlateAssayData (4)</a></li><li packagelist="curatedBreastData,Hiiragi2013,miRcompData,seqc" id="qPCRData" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="https://www.bioconductor.org/packages/release/BiocViews.html#"><ins class="jstree-icon">&nbsp;</ins>qPCRData (4)</a></li><li packagelist="dyebiasexamples" id="SAGEData" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="https://www.bioconductor.org/packages/release/BiocViews.html#"><ins class="jstree-icon">&nbsp;</ins>SAGEData (1)</a></li><li packagelist="airway,alpineData,AneuFinderData,AssessORFData,breakpointRdata,bsseqData,cgdv17,diffloopdata,DREAM4,dsQTL,EatonEtAlChIPseq,etec16s,fission,genomationData,GeuvadisTranscriptExpr,geuvPack,geuvStore2,GGdata,harbChIP,HelloRangesData,HMP16SData,JASPAR2014,leeBamViews,LiebermanAidenHiC2009,ListerEtAlBSseq,MEDIPSData,microRNAome,minionSummaryData,msd16s,NGScopyData,OMICsPCAdata,parathyroidSE,PasillaTranscriptExpr,pathprintGEOData,PCHiCdata,prebsdata,restfulSEData,RIPSeekerData,RNAseqData.HNRNPC.bam.chr14,RRBSdata,rRDPData,sampleClassifierData,SCLCBam,scRNAseq,seqc,SomatiCAData,TBX20BamSubset,tcgaWGBSData.hg19,TENxBrainData,TENxPBMCData,tinesath1probe,tissueTreg,tximportData,VariantToolsData,WES.1KG.WUGSC,yeastNagalakshmi,yeastRNASeq" id="SequencingData" class="jstree-last jstree-closed"><ins class="jstree-icon">&nbsp;</ins><a href="https://www.bioconductor.org/packages/release/BiocViews.html#"><ins class="jstree-icon">&nbsp;</ins>SequencingData (83)</a><ul><li packagelist="chromstaRData,EatonEtAlChIPseq,Fletcher2013b,genomationData,LiebermanAidenHiC2009,MMDiffBamSubset,OMICsPCAdata,vulcandata,yeastNagalakshmi" id="ChIPSeqData" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="https://www.bioconductor.org/packages/release/BiocViews.html#"><ins class="jstree-icon">&nbsp;</ins>ChIPSeqData (9)</a></li><li packagelist="breakpointRdata,dsQTL,GSE62944,LiebermanAidenHiC2009,MMDiffBamSubset,pasillaBamSubset,WES.1KG.WUGSC" id="DNASeqData" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="https://www.bioconductor.org/packages/release/BiocViews.html#"><ins class="jstree-icon">&nbsp;</ins>DNASeqData (7)</a></li><li class="jstree-leaf" style="display: none;"><ins class="jstree-icon">&nbsp;</ins><a href="https://www.bioconductor.org/packages/release/BiocViews.html#"><ins class="jstree-icon">&nbsp;</ins>ExomeSeqData</a></li><li class="jstree-leaf" style="display: none;"><ins class="jstree-icon">&nbsp;</ins><a href="https://www.bioconductor.org/packages/release/BiocViews.html#"><ins class="jstree-icon">&nbsp;</ins>MethylSeqData</a></li><li packagelist="curatedMetagenomicData,etec16s,HMP16SData,msd16s,rRDPData" id="MicrobiomeData" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="https://www.bioconductor.org/packages/release/BiocViews.html#"><ins class="jstree-icon">&nbsp;</ins>MicrobiomeData (5)</a></li><li packagelist="LiebermanAidenHiC2009,microRNAome,rcellminerData,TargetScoreData" id="miRNAData" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="https://www.bioconductor.org/packages/release/BiocViews.html#"><ins class="jstree-icon">&nbsp;</ins>miRNAData (4)</a></li><li packagelist="RIPSeekerData" id="RIPSeqData" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="https://www.bioconductor.org/packages/release/BiocViews.html#"><ins class="jstree-icon">&nbsp;</ins>RIPSeqData (1)</a></li><li packagelist="ABAData,airway,allenpvc,alpineData,curatedOvarianData,derfinderData,fission,GeuvadisTranscriptExpr,GSE62944,GSVAdata,HSMMSingleCell,IHWpaper,JctSeqData,leeBamViews,LungCancerLines,M3DExampleData,microRNAome,parathyroidSE,pasilla,pasillaBamSubset,PasillaTranscriptExpr,prebsdata,RNAseqData.HNRNPC.bam.chr14,RnaSeqSampleSizeData,sampleClassifierData,scRNAseq,seqc,TabulaMurisData,TargetScoreData,TBX20BamSubset,TENxBrainData,TENxPBMCData,tissueTreg,tweeDEseqCountData,tximportData,yeastRNASeq,zebrafishRNASeq" id="RNASeqData" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="https://www.bioconductor.org/packages/release/BiocViews.html#"><ins class="jstree-icon">&nbsp;</ins>RNASeqData (37)</a></li><li class="jstree-leaf" style="display: none;"><ins class="jstree-icon">&nbsp;</ins><a href="https://www.bioconductor.org/packages/release/BiocViews.html#"><ins class="jstree-icon">&nbsp;</ins>SangerSeqData</a></li><li packagelist="allenpvc,breakpointRdata,celarefData,DuoClustering2018,TabulaMurisData,TENxBrainData,TENxPBMCData" id="SingleCellData" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="https://www.bioconductor.org/packages/release/BiocViews.html#"><ins class="jstree-icon">&nbsp;</ins>SingleCellData (7)</a></li><li packagelist="LiebermanAidenHiC2009" id="SmallRNAData" class="jstree-last jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="https://www.bioconductor.org/packages/release/BiocViews.html#"><ins class="jstree-icon">&nbsp;</ins>SmallRNAData (1)</a></li></ul></li></ul></li></ul></li><li packagelist="annotation,arrays,BiocMetaWorkflow,chipseqDB,cytofWorkflow,EGSEA123,eQTL,ExpressionNormalizationWorkflow,generegulation,highthroughputassays,liftOver,maEndToEnd,methylationArrayAnalysis,proteomics,recountWorkflow,RNAseq123,rnaseqDTU,rnaseqGene,RnaSeqGeneEdgeRQL,sequencing,simpleSingleCell,TCGAWorkflow,variants" id="Workflow" class="jstree-last jstree-closed"><ins class="jstree-icon">&nbsp;</ins><a href="https://www.bioconductor.org/packages/release/BiocViews.html#"><ins class="jstree-icon">&nbsp;</ins>Workflow (23)</a><ul><li packagelist="annotation,variants" id="AnnotationWorkflow" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="https://www.bioconductor.org/packages/release/BiocViews.html#"><ins class="jstree-icon">&nbsp;</ins>AnnotationWorkflow (2)</a></li><li packagelist="arrays,BiocMetaWorkflow,liftOver,sequencing" id="BasicWorkflow" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="https://www.bioconductor.org/packages/release/BiocViews.html#"><ins class="jstree-icon">&nbsp;</ins>BasicWorkflow (4)</a></li><li class="jstree-leaf" style="display: none;"><ins class="jstree-icon">&nbsp;</ins><a href="https://www.bioconductor.org/packages/release/BiocViews.html#"><ins class="jstree-icon">&nbsp;</ins>DifferentialSplicingWorkflow</a></li><li packagelist="chipseqDB,generegulation,methylationArrayAnalysis" id="EpigeneticsWorkflow" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="https://www.bioconductor.org/packages/release/BiocViews.html#"><ins class="jstree-icon">&nbsp;</ins>EpigeneticsWorkflow (3)</a></li><li packagelist="EGSEA123,ExpressionNormalizationWorkflow,maEndToEnd,RNAseq123,rnaseqDTU,rnaseqGene,RnaSeqGeneEdgeRQL" id="GeneExpressionWorkflow" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="https://www.bioconductor.org/packages/release/BiocViews.html#"><ins class="jstree-icon">&nbsp;</ins>GeneExpressionWorkflow (7)</a></li><li packagelist="eQTL" id="GenomicVariantsWorkflow" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="https://www.bioconductor.org/packages/release/BiocViews.html#"><ins class="jstree-icon">&nbsp;</ins>GenomicVariantsWorkflow (1)</a></li><li packagelist="chipseqDB,cytofWorkflow,EGSEA123,ExpressionNormalizationWorkflow,highthroughputassays,proteomics,RNAseq123,rnaseqDTU,rnaseqGene,RnaSeqGeneEdgeRQL,sequencing,simpleSingleCell,variants" id="ImmunoOncologyWorkflow" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="https://www.bioconductor.org/packages/release/BiocViews.html#"><ins class="jstree-icon">&nbsp;</ins>ImmunoOncologyWorkflow (13)</a></li><li packagelist="highthroughputassays,proteomics" id="ProteomicsWorkflow" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="https://www.bioconductor.org/packages/release/BiocViews.html#"><ins class="jstree-icon">&nbsp;</ins>ProteomicsWorkflow (2)</a></li><li packagelist="recountWorkflow,TCGAWorkflow" id="ResourceQueryingWorkflow" class="jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="https://www.bioconductor.org/packages/release/BiocViews.html#"><ins class="jstree-icon">&nbsp;</ins>ResourceQueryingWorkflow (2)</a></li><li packagelist="cytofWorkflow,simpleSingleCell" id="SingleCellWorkflow" class="jstree-last jstree-leaf"><ins class="jstree-icon">&nbsp;</ins><a href="https://www.bioconductor.org/packages/release/BiocViews.html#"><ins class="jstree-icon">&nbsp;</ins>SingleCellWorkflow (2)</a></li></ul></li></ul></div>
        </div>
        <div id="packages" class="biocViewsPackages"><h3>Packages found under Software:</h3>
<p>Rank based on number of downloads: lower numbers are more frequently downloaded.</p>
<div id="biocViews_package_table_wrapper" class="dataTables_wrapper" role="grid"><div id="biocViews_package_table_length" class="dataTables_length"><label>Show <select size="1" name="biocViews_package_table_length" aria-controls="biocViews_package_table"><option value="-1" selected="selected">All</option><option value="10">10</option><option value="25">25</option><option value="50">50</option><option value="100">100</option></select> entries</label></div><div class="dataTables_filter" id="biocViews_package_table_filter"><label>Search table: <input type="text" aria-controls="biocViews_package_table"></label></div><div class="dataTables_scroll"><div class="dataTables_scrollHead" style="overflow: hidden; position: relative; border: 0px; width: 100%;"><div class="dataTables_scrollHeadInner" style="width: 595px; padding-right: 0px;"><table class="dataTable" style="margin-left: 0px; width: 595px;"><thead><tr role="row"><th class="sorting" role="columnheader" tabindex="0" aria-controls="biocViews_package_table" rowspan="1" colspan="1" aria-label="Package: activate to sort column ascending" style="width: 152px;">Package</th><th class="sorting" role="columnheader" tabindex="0" aria-controls="biocViews_package_table" rowspan="1" colspan="1" aria-label="Maintainer: activate to sort column ascending" style="width: 79px;">Maintainer</th><th class="sorting" role="columnheader" tabindex="0" aria-controls="biocViews_package_table" rowspan="1" colspan="1" aria-label="Title: activate to sort column ascending" style="width: 209px;">Title</th><th class="sorting_asc" role="columnheader" tabindex="0" aria-controls="biocViews_package_table" rowspan="1" colspan="1" aria-sort="ascending" aria-label="Rank: activate to sort column descending" style="width: 33px;">Rank</th></tr></thead></table></div></div><div class="dataTables_scrollBody" style="overflow: auto; width: 100%;"><table id="biocViews_package_table" class="dataTable" aria-describedby="biocViews_package_table_info" style="margin-left: 0px; width: 595px;"><thead><tr role="row" style="height: 0px;"><th class="sorting" role="columnheader" tabindex="0" aria-controls="biocViews_package_table" rowspan="1" colspan="1" aria-label="Package: activate to sort column ascending" style="width: 152px; padding-top: 0px; padding-bottom: 0px; border-top-width: 0px; border-bottom-width: 0px; height: 0px;"></th><th class="sorting" role="columnheader" tabindex="0" aria-controls="biocViews_package_table" rowspan="1" colspan="1" aria-label="Maintainer: activate to sort column ascending" style="width: 79px; padding-top: 0px; padding-bottom: 0px; border-top-width: 0px; border-bottom-width: 0px; height: 0px;"></th><th class="sorting" role="columnheader" tabindex="0" aria-controls="biocViews_package_table" rowspan="1" colspan="1" aria-label="Title: activate to sort column ascending" style="width: 209px; padding-top: 0px; padding-bottom: 0px; border-top-width: 0px; border-bottom-width: 0px; height: 0px;"></th><th class="sorting_asc" role="columnheader" tabindex="0" aria-controls="biocViews_package_table" rowspan="1" colspan="1" aria-sort="ascending" aria-label="Rank: activate to sort column descending" style="width: 33px; padding-top: 0px; padding-bottom: 0px; border-top-width: 0px; border-bottom-width: 0px; height: 0px;"></th></tr></thead><tbody role="alert" aria-live="polite" aria-relevant="all"><tr id="pkg_BiocInstaller" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/BiocInstaller.html">BiocInstaller</a></td>
	<td class=" ">Bioconductor Package Maintainer</td>
	<td class=" ">Install/Update Bioconductor, CRAN, and github Packages</td>
	<td class="  sorting_1">1</td>
</tr><tr id="pkg_BiocGenerics" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/BiocGenerics.html">BiocGenerics</a></td>
	<td class=" ">Bioconductor Package Maintainer</td>
	<td class=" ">S4 generic functions for Bioconductor</td>
	<td class="  sorting_1">2</td>
</tr><tr id="pkg_IRanges" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/IRanges.html">IRanges</a></td>
	<td class=" ">Bioconductor Package Maintainer</td>
	<td class=" ">Infrastructure for manipulating intervals on sequences</td>
	<td class="  sorting_1">3</td>
</tr><tr id="pkg_Biobase" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/Biobase.html">Biobase</a></td>
	<td class=" ">Bioconductor Package Maintainer</td>
	<td class=" ">Biobase: Base functions for Bioconductor</td>
	<td class="  sorting_1">4</td>
</tr><tr id="pkg_S4Vectors" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/S4Vectors.html">S4Vectors</a></td>
	<td class=" ">Bioconductor Package Maintainer</td>
	<td class=" ">S4 implementation of vector-like and list-like objects</td>
	<td class="  sorting_1">5</td>
</tr><tr id="pkg_AnnotationDbi" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/AnnotationDbi.html">AnnotationDbi</a></td>
	<td class=" ">Bioconductor Package Maintainer</td>
	<td class=" ">Annotation Database Interface</td>
	<td class="  sorting_1">6</td>
</tr><tr id="pkg_zlibbioc" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/zlibbioc.html">zlibbioc</a></td>
	<td class=" ">Bioconductor Package Maintainer</td>
	<td class=" ">An R packaged zlib-1.2.5</td>
	<td class="  sorting_1">7</td>
</tr><tr id="pkg_GenomicRanges" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/GenomicRanges.html">GenomicRanges</a></td>
	<td class=" ">Bioconductor Package Maintainer</td>
	<td class=" ">Representation and manipulation of genomic intervals and variables defined along a genome</td>
	<td class="  sorting_1">8</td>
</tr><tr id="pkg_limma" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/limma.html">limma</a></td>
	<td class=" ">Gordon Smyth</td>
	<td class=" ">Linear Models for Microarray Data</td>
	<td class="  sorting_1">9</td>
</tr><tr id="pkg_XVector" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/XVector.html">XVector</a></td>
	<td class=" ">Herv Pags</td>
	<td class=" ">Representation and manipulation of external sequences</td>
	<td class="  sorting_1">10</td>
</tr><tr id="pkg_BiocParallel" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/BiocParallel.html">BiocParallel</a></td>
	<td class=" ">Bioconductor Package Maintainer</td>
	<td class=" ">Bioconductor facilities for parallel evaluation</td>
	<td class="  sorting_1">11</td>
</tr><tr id="pkg_GenomeInfoDb" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/GenomeInfoDb.html">GenomeInfoDb</a></td>
	<td class=" ">Bioconductor Package Maintainer</td>
	<td class=" ">Utilities for manipulating chromosome and other 'seqname' identifiers</td>
	<td class="  sorting_1">12</td>
</tr><tr id="pkg_Biostrings" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/Biostrings.html">Biostrings</a></td>
	<td class=" ">H. Pags</td>
	<td class=" ">Efficient manipulation of biological strings</td>
	<td class="  sorting_1">13</td>
</tr><tr id="pkg_SummarizedExperiment" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/SummarizedExperiment.html">SummarizedExperiment</a></td>
	<td class=" ">Bioconductor Package Maintainer</td>
	<td class=" ">SummarizedExperiment container</td>
	<td class="  sorting_1">14</td>
</tr><tr id="pkg_DelayedArray" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/DelayedArray.html">DelayedArray</a></td>
	<td class=" ">Herv Pags</td>
	<td class=" ">Delayed operations on array-like objects</td>
	<td class="  sorting_1">15</td>
</tr><tr id="pkg_annotate" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/annotate.html">annotate</a></td>
	<td class=" ">Bioconductor Package Maintainer</td>
	<td class=" ">Annotation for microarrays</td>
	<td class="  sorting_1">16</td>
</tr><tr id="pkg_genefilter" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/genefilter.html">genefilter</a></td>
	<td class=" ">Bioconductor Package Maintainer</td>
	<td class=" ">genefilter: methods for filtering genes from high-throughput experiments</td>
	<td class="  sorting_1">17</td>
</tr><tr id="pkg_Rsamtools" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/Rsamtools.html">Rsamtools</a></td>
	<td class=" ">Bioconductor Package Maintainer</td>
	<td class=" ">Binary alignment (BAM), FASTA, variant call (BCF), and tabix file import</td>
	<td class="  sorting_1">18</td>
</tr><tr id="pkg_biomaRt" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/biomaRt.html">biomaRt</a></td>
	<td class=" ">Mike Smith</td>
	<td class=" ">Interface to BioMart databases (i.e. Ensembl)</td>
	<td class="  sorting_1">19</td>
</tr><tr id="pkg_GenomicAlignments" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/GenomicAlignments.html">GenomicAlignments</a></td>
	<td class=" ">Bioconductor Package Maintainer</td>
	<td class=" ">Representation and manipulation of short genomic alignments</td>
	<td class="  sorting_1">20</td>
</tr><tr id="pkg_rtracklayer" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/rtracklayer.html">rtracklayer</a></td>
	<td class=" ">Michael Lawrence</td>
	<td class=" ">R interface to genome annotation files and the UCSC genome browser</td>
	<td class="  sorting_1">21</td>
</tr><tr id="pkg_graph" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/graph.html">graph</a></td>
	<td class=" ">Bioconductor Package Maintainer</td>
	<td class=" ">graph: A package to handle graph data structures</td>
	<td class="  sorting_1">22</td>
</tr><tr id="pkg_edgeR" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/edgeR.html">edgeR</a></td>
	<td class=" ">Yunshun Chen, Aaron Lun, Mark Robinson, Davis McCarthy, Gordon Smyth</td>
	<td class=" ">Empirical Analysis of Digital Gene Expression Data in R</td>
	<td class="  sorting_1">23</td>
</tr><tr id="pkg_GenomicFeatures" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/GenomicFeatures.html">GenomicFeatures</a></td>
	<td class=" ">Bioconductor Package Maintainer</td>
	<td class=" ">Tools for making and manipulating transcript centric annotations</td>
	<td class="  sorting_1">24</td>
</tr><tr id="pkg_preprocessCore" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/preprocessCore.html">preprocessCore</a></td>
	<td class=" ">Ben Bolstad</td>
	<td class=" ">A collection of pre-processing functions</td>
	<td class="  sorting_1">25</td>
</tr><tr id="pkg_DESeq2" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/DESeq2.html">DESeq2</a></td>
	<td class=" ">Michael Love</td>
	<td class=" ">Differential gene expression analysis based on the negative binomial distribution</td>
	<td class="  sorting_1">26</td>
</tr><tr id="pkg_geneplotter" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/geneplotter.html">geneplotter</a></td>
	<td class=" ">Bioconductor Package Maintainer</td>
	<td class=" ">Graphics related functions for Bioconductor</td>
	<td class="  sorting_1">27</td>
</tr><tr id="pkg_affy" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/affy.html">affy</a></td>
	<td class=" ">Rafael A. Irizarry</td>
	<td class=" ">Methods for Affymetrix Oligonucleotide Arrays</td>
	<td class="  sorting_1">28</td>
</tr><tr id="pkg_affyio" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/affyio.html">affyio</a></td>
	<td class=" ">Ben Bolstad</td>
	<td class=" ">Tools for parsing Affymetrix data files</td>
	<td class="  sorting_1">29</td>
</tr><tr id="pkg_rhdf5" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/rhdf5.html">rhdf5</a></td>
	<td class=" ">Mike Smith</td>
	<td class=" ">HDF5 interface to R</td>
	<td class="  sorting_1">30</td>
</tr><tr id="pkg_BSgenome" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/BSgenome.html">BSgenome</a></td>
	<td class=" ">H. Pags</td>
	<td class=" ">Software infrastructure for efficient representation of full genomes and their SNPs</td>
	<td class="  sorting_1">31</td>
</tr><tr id="pkg_RBGL" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/RBGL.html">RBGL</a></td>
	<td class=" ">Bioconductor Package Maintainer</td>
	<td class=" ">An interface to the BOOST graph library</td>
	<td class="  sorting_1">32</td>
</tr><tr id="pkg_multtest" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/multtest.html">multtest</a></td>
	<td class=" ">Katherine S. Pollard</td>
	<td class=" ">Resampling-based multiple hypothesis testing</td>
	<td class="  sorting_1">33</td>
</tr><tr id="pkg_Rgraphviz" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/Rgraphviz.html">Rgraphviz</a></td>
	<td class=" ">Kasper Daniel Hansen</td>
	<td class=" ">Provides plotting capabilities for R graph objects</td>
	<td class="  sorting_1">34</td>
</tr><tr id="pkg_impute" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/impute.html">impute</a></td>
	<td class=" ">Balasubramanian Narasimhan</td>
	<td class=" ">impute: Imputation for microarray data</td>
	<td class="  sorting_1">35</td>
</tr><tr id="pkg_qvalue" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/qvalue.html">qvalue</a></td>
	<td class=" ">John D. Storey, Andrew J. Bass</td>
	<td class=" ">Q-value estimation for false discovery rate control</td>
	<td class="  sorting_1">36</td>
</tr><tr id="pkg_VariantAnnotation" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/VariantAnnotation.html">VariantAnnotation</a></td>
	<td class=" ">Valerie Obenchain</td>
	<td class=" ">Annotation of Genetic Variants</td>
	<td class="  sorting_1">37</td>
</tr><tr id="pkg_GEOquery" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/GEOquery.html">GEOquery</a></td>
	<td class=" ">Sean Davis</td>
	<td class=" ">Get data from NCBI Gene Expression Omnibus (GEO)</td>
	<td class="  sorting_1">38</td>
</tr><tr id="pkg_Rhdf5lib" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/Rhdf5lib.html">Rhdf5lib</a></td>
	<td class=" ">Mike Smith</td>
	<td class=" ">hdf5 library as an R package</td>
	<td class="  sorting_1">39</td>
</tr><tr id="pkg_ShortRead" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/ShortRead.html">ShortRead</a></td>
	<td class=" ">Bioconductor Package Maintainer</td>
	<td class=" ">FASTQ input and manipulation</td>
	<td class="  sorting_1">40</td>
</tr><tr id="pkg_BiocVersion" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/BiocVersion.html">BiocVersion</a></td>
	<td class=" ">Bioconductor Package Maintainer</td>
	<td class=" ">Set the appropriate version of Bioconductor packages</td>
	<td class="  sorting_1">41</td>
</tr><tr id="pkg_ProtGenerics" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/ProtGenerics.html">ProtGenerics</a></td>
	<td class=" ">Laurent Gatto</td>
	<td class=" ">S4 generic functions for Bioconductor proteomics infrastructure</td>
	<td class="  sorting_1">42</td>
</tr><tr id="pkg_ensembldb" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/ensembldb.html">ensembldb</a></td>
	<td class=" ">Johannes Rainer</td>
	<td class=" ">Utilities to create and use Ensembl-based annotation databases</td>
	<td class="  sorting_1">43</td>
</tr><tr id="pkg_sva" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/sva.html">sva</a></td>
	<td class=" ">Jeffrey T. Leek, John D. Storey, W. Evan Johnson</td>
	<td class=" ">Surrogate Variable Analysis</td>
	<td class="  sorting_1">44</td>
</tr><tr id="pkg_biovizBase" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/biovizBase.html">biovizBase</a></td>
	<td class=" ">Michael Lawrence</td>
	<td class=" ">Basic graphic utilities for visualization of genomic data.</td>
	<td class="  sorting_1">45</td>
</tr><tr id="pkg_AnnotationHub" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/AnnotationHub.html">AnnotationHub</a></td>
	<td class=" ">Bioconductor Package Maintainer</td>
	<td class=" ">Client to access AnnotationHub resources</td>
	<td class="  sorting_1">46</td>
</tr><tr id="pkg_GSEABase" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/GSEABase.html">GSEABase</a></td>
	<td class=" ">Bioconductor Package Maintainer</td>
	<td class=" ">Gene set enrichment data structures and methods</td>
	<td class="  sorting_1">47</td>
</tr><tr id="pkg_DESeq" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/DESeq.html">DESeq</a></td>
	<td class=" ">Simon Anders</td>
	<td class=" ">Differential gene expression analysis based on the negative binomial distribution</td>
	<td class="  sorting_1">48</td>
</tr><tr id="pkg_AnnotationFilter" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/AnnotationFilter.html">AnnotationFilter</a></td>
	<td class=" ">Bioconductor Package Maintainer</td>
	<td class=" ">Facilities for Filtering Bioconductor Annotation Resources</td>
	<td class="  sorting_1">49</td>
</tr><tr id="pkg_GOSemSim" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/GOSemSim.html">GOSemSim</a></td>
	<td class=" ">Guangchuang Yu</td>
	<td class=" ">GO-terms Semantic Similarity Measures</td>
	<td class="  sorting_1">50</td>
</tr><tr id="pkg_fgsea" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/fgsea.html">fgsea</a></td>
	<td class=" ">Alexey Sergushichev</td>
	<td class=" ">Fast Gene Set Enrichment Analysis</td>
	<td class="  sorting_1">51</td>
</tr><tr id="pkg_DOSE" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/DOSE.html">DOSE</a></td>
	<td class=" ">Guangchuang Yu</td>
	<td class=" ">Disease Ontology Semantic and Enrichment analysis</td>
	<td class="  sorting_1">52</td>
</tr><tr id="pkg_clusterProfiler" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/clusterProfiler.html">clusterProfiler</a></td>
	<td class=" ">Guangchuang Yu</td>
	<td class=" ">statistical analysis and visualization of functional profiles for genes and gene clusters</td>
	<td class="  sorting_1">53</td>
</tr><tr id="pkg_KEGGREST" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/KEGGREST.html">KEGGREST</a></td>
	<td class=" ">Bioconductor Package Maintainer</td>
	<td class=" ">Client-side REST access to KEGG</td>
	<td class="  sorting_1">54</td>
</tr><tr id="pkg_interactiveDisplayBase" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/interactiveDisplayBase.html">interactiveDisplayBase</a></td>
	<td class=" ">Shawn Balcome</td>
	<td class=" ">Base package for enabling powerful shiny web displays of Bioconductor objects</td>
	<td class="  sorting_1">55</td>
</tr><tr id="pkg_vsn" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/vsn.html">vsn</a></td>
	<td class=" ">Wolfgang Huber</td>
	<td class=" ">Variance stabilization and calibration for microarray data</td>
	<td class="  sorting_1">56</td>
</tr><tr id="pkg_ComplexHeatmap" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/ComplexHeatmap.html">ComplexHeatmap</a></td>
	<td class=" ">Zuguang Gu</td>
	<td class=" ">Making Complex Heatmaps</td>
	<td class="  sorting_1">57</td>
</tr><tr id="pkg_Gviz" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/Gviz.html">Gviz</a></td>
	<td class=" ">Robert Ivanek</td>
	<td class=" ">Plotting data and annotation information along genomic coordinates</td>
	<td class="  sorting_1">58</td>
</tr><tr id="pkg_phyloseq" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/phyloseq.html">phyloseq</a></td>
	<td class=" ">Paul J. McMurdie</td>
	<td class=" ">Handling and analysis of high-throughput microbiome census data</td>
	<td class="  sorting_1">59</td>
</tr><tr id="pkg_AnnotationForge" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/AnnotationForge.html">AnnotationForge</a></td>
	<td class=" ">Bioconductor Package Maintainer</td>
	<td class=" ">Code for Building Annotation Database Packages</td>
	<td class="  sorting_1">60</td>
</tr><tr id="pkg_Category" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/Category.html">Category</a></td>
	<td class=" ">Bioconductor Package Maintainer</td>
	<td class=" ">Category Analysis</td>
	<td class="  sorting_1">61</td>
</tr><tr id="pkg_pcaMethods" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/pcaMethods.html">pcaMethods</a></td>
	<td class=" ">Henning Redestig</td>
	<td class=" ">A collection of PCA methods</td>
	<td class="  sorting_1">62</td>
</tr><tr id="pkg_tximport" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/tximport.html">tximport</a></td>
	<td class=" ">Michael Love</td>
	<td class=" ">Import and summarize transcript-level estimates for transcript- and gene-level analysis</td>
	<td class="  sorting_1">63</td>
</tr><tr id="pkg_topGO" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/topGO.html">topGO</a></td>
	<td class=" ">Adrian Alexa</td>
	<td class=" ">Enrichment Analysis for Gene Ontology</td>
	<td class="  sorting_1">64</td>
</tr><tr id="pkg_HDF5Array" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/HDF5Array.html">HDF5Array</a></td>
	<td class=" ">Herv Pags</td>
	<td class=" ">HDF5 backend for DelayedArray objects</td>
	<td class="  sorting_1">65</td>
</tr><tr id="pkg_GOstats" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/GOstats.html">GOstats</a></td>
	<td class=" ">Bioconductor Package Maintainer</td>
	<td class=" ">Tools for manipulating GO and microarrays</td>
	<td class="  sorting_1">66</td>
</tr><tr id="pkg_KEGGgraph" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/KEGGgraph.html">KEGGgraph</a></td>
	<td class=" ">Jitao David Zhang</td>
	<td class=" ">KEGGgraph: A graph approach to KEGG PATHWAY in R and Bioconductor</td>
	<td class="  sorting_1">67</td>
</tr><tr id="pkg_pathview" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/pathview.html">pathview</a></td>
	<td class=" ">Weijun Luo</td>
	<td class=" ">a tool set for pathway based data integration and visualization</td>
	<td class="  sorting_1">68</td>
</tr><tr id="pkg_biomformat" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/biomformat.html">biomformat</a></td>
	<td class=" ">Paul J. McMurdie</td>
	<td class=" ">An interface package for the BIOM file format</td>
	<td class="  sorting_1">69</td>
</tr><tr id="pkg_EBImage" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/EBImage.html">EBImage</a></td>
	<td class=" ">Andrzej Ole</td>
	<td class=" ">Image processing and analysis toolbox for R</td>
	<td class="  sorting_1">70</td>
</tr><tr id="pkg_OrganismDbi" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/OrganismDbi.html">OrganismDbi</a></td>
	<td class=" ">Biocore Data Team</td>
	<td class=" ">Software to enable the smooth interfacing of different database packages</td>
	<td class="  sorting_1">71</td>
</tr><tr id="pkg_aroma.light" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/aroma.light.html">aroma.light</a></td>
	<td class=" ">Henrik Bengtsson</td>
	<td class=" ">Light-Weight Methods for Normalization and Visualization of Microarray Data using Only Basic R Data Types</td>
	<td class="  sorting_1">72</td>
</tr><tr id="pkg_BiocStyle" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/BiocStyle.html">BiocStyle</a></td>
	<td class=" ">Bioconductor Package Maintainer</td>
	<td class=" ">Standard styles for vignettes and other Bioconductor documents</td>
	<td class="  sorting_1">73</td>
</tr><tr id="pkg_illuminaio" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/illuminaio.html">illuminaio</a></td>
	<td class=" ">Kasper Daniel Hansen</td>
	<td class=" ">Parsing Illumina Microarray Output Files</td>
	<td class="  sorting_1">74</td>
</tr><tr id="pkg_ggbio" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/ggbio.html">ggbio</a></td>
	<td class=" ">Michael Lawrence</td>
	<td class=" ">Visualization tools for genomic data</td>
	<td class="  sorting_1">75</td>
</tr><tr id="pkg_EDASeq" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/EDASeq.html">EDASeq</a></td>
	<td class=" ">Davide Risso</td>
	<td class=" ">Exploratory Data Analysis and Normalization for RNA-Seq</td>
	<td class="  sorting_1">76</td>
</tr><tr id="pkg_siggenes" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/siggenes.html">siggenes</a></td>
	<td class=" ">Holger Schwender</td>
	<td class=" ">Multiple testing using SAM and Efron's empirical Bayes approaches</td>
	<td class="  sorting_1">77</td>
</tr><tr id="pkg_bumphunter" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/bumphunter.html">bumphunter</a></td>
	<td class=" ">Rafael A. Irizarry</td>
	<td class=" ">Bump Hunter</td>
	<td class="  sorting_1">78</td>
</tr><tr id="pkg_DNAcopy" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/DNAcopy.html">DNAcopy</a></td>
	<td class=" ">Venkatraman E. Seshan</td>
	<td class=" ">DNA copy number data analysis</td>
	<td class="  sorting_1">79</td>
</tr><tr id="pkg_oligoClasses" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/oligoClasses.html">oligoClasses</a></td>
	<td class=" ">Benilton Carvalho and Robert Scharpf</td>
	<td class=" ">Classes for high-throughput arrays supported by oligo and crlmm</td>
	<td class="  sorting_1">80</td>
</tr><tr id="pkg_enrichplot" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/enrichplot.html">enrichplot</a></td>
	<td class=" ">Guangchuang Yu</td>
	<td class=" ">Visualization of Functional Enrichment Result</td>
	<td class="  sorting_1">81</td>
</tr><tr id="pkg_gcrma" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/gcrma.html">gcrma</a></td>
	<td class=" ">Z. Wu</td>
	<td class=" ">Background Adjustment Using Sequence Information</td>
	<td class="  sorting_1">82</td>
</tr><tr id="pkg_ConsensusClusterPlus" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/ConsensusClusterPlus.html">ConsensusClusterPlus</a></td>
	<td class=" ">Matt Wilkerson</td>
	<td class=" ">ConsensusClusterPlus</td>
	<td class="  sorting_1">83</td>
</tr><tr id="pkg_minfi" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/minfi.html">minfi</a></td>
	<td class=" ">Kasper Daniel Hansen</td>
	<td class=" ">Analyze Illumina Infinium DNA methylation arrays</td>
	<td class="  sorting_1">84</td>
</tr><tr id="pkg_oligo" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/oligo.html">oligo</a></td>
	<td class=" ">Benilton Carvalho</td>
	<td class=" ">Preprocessing tools for oligonucleotide arrays</td>
	<td class="  sorting_1">85</td>
</tr><tr id="pkg_scater" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/scater.html">scater</a></td>
	<td class=" ">Davis McCarthy</td>
	<td class=" ">Single-Cell Analysis Toolkit for Gene Expression Data in R</td>
	<td class="  sorting_1">86</td>
</tr><tr id="pkg_Rsubread" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/Rsubread.html">Rsubread</a></td>
	<td class=" ">Wei Shi, Yang Liao and Gordon K Smyth</td>
	<td class=" ">Subread sequence alignment and counting for R</td>
	<td class="  sorting_1">87</td>
</tr><tr id="pkg_affxparser" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/affxparser.html">affxparser</a></td>
	<td class=" ">Kasper Daniel Hansen</td>
	<td class=" ">Affymetrix File Parsing SDK</td>
	<td class="  sorting_1">88</td>
</tr><tr id="pkg_SingleCellExperiment" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/SingleCellExperiment.html">SingleCellExperiment</a></td>
	<td class=" ">Davide Risso</td>
	<td class=" ">S4 Classes for Single Cell Data</td>
	<td class="  sorting_1">89</td>
</tr><tr id="pkg_DelayedMatrixStats" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/DelayedMatrixStats.html">DelayedMatrixStats</a></td>
	<td class=" ">Peter Hickey</td>
	<td class=" ">Functions that Apply to Rows and Columns of 'DelayedMatrix' Objects</td>
	<td class="  sorting_1">90</td>
</tr><tr id="pkg_ggtree" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/ggtree.html">ggtree</a></td>
	<td class=" ">Guangchuang Yu</td>
	<td class=" ">an R package for visualization and annotation of phylogenetic trees with their covariates and other associated data</td>
	<td class="  sorting_1">91</td>
</tr><tr id="pkg_biocViews" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/biocViews.html">biocViews</a></td>
	<td class=" ">Bioconductor Package Maintainer</td>
	<td class=" ">Categorized views of R package repositories</td>
	<td class="  sorting_1">92</td>
</tr><tr id="pkg_flowCore" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/flowCore.html">flowCore</a></td>
	<td class=" ">Mike Jiang</td>
	<td class=" ">flowCore: Basic structures for flow cytometry data</td>
	<td class="  sorting_1">93</td>
</tr><tr id="pkg_MSnbase" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/MSnbase.html">MSnbase</a></td>
	<td class=" ">Laurent Gatto</td>
	<td class=" ">Base Functions and Classes for Mass Spectrometry and Proteomics</td>
	<td class="  sorting_1">94</td>
</tr><tr id="pkg_beachmat" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/beachmat.html">beachmat</a></td>
	<td class=" ">Aaron Lun</td>
	<td class=" ">Compiling Bioconductor to Handle Each Matrix Type</td>
	<td class="  sorting_1">95</td>
</tr><tr id="pkg_mzR" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/mzR.html">mzR</a></td>
	<td class=" ">Steffen Neumann, Laurent Gatto, Qiang Kou</td>
	<td class=" ">parser for netCDF, mzXML, mzData and mzML and mzIdentML files (mass spectrometry data)</td>
	<td class="  sorting_1">96</td>
</tr><tr id="pkg_marray" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/marray.html">marray</a></td>
	<td class=" ">Yee Hwa (Jean) Yang</td>
	<td class=" ">Exploratory analysis for two-color spotted microarray data</td>
	<td class="  sorting_1">97</td>
</tr><tr id="pkg_affyPLM" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/affyPLM.html">affyPLM</a></td>
	<td class=" ">Ben Bolstad</td>
	<td class=" ">Methods for fitting probe-level models</td>
	<td class="  sorting_1">98</td>
</tr><tr id="pkg_TCGAbiolinks" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/TCGAbiolinks.html">TCGAbiolinks</a></td>
	<td class=" ">Antonio Colaprico, Tiago Chedraoui Silva</td>
	<td class=" ">TCGAbiolinks: An R/Bioconductor package for integrative analysis with GDC data</td>
	<td class="  sorting_1">99</td>
</tr><tr id="pkg_treeio" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/treeio.html">treeio</a></td>
	<td class=" ">Guangchuang Yu</td>
	<td class=" ">Base Classes and Functions for Phylogenetic Tree Input and Output</td>
	<td class="  sorting_1">100</td>
</tr><tr id="pkg_monocle" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/monocle.html">monocle</a></td>
	<td class=" ">Cole Trapnell</td>
	<td class=" ">Clustering, differential expression, and trajectory analysis for single- cell RNA-Seq</td>
	<td class="  sorting_1">101</td>
</tr><tr id="pkg_snpStats" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/snpStats.html">snpStats</a></td>
	<td class=" ">David Clayton</td>
	<td class=" ">SnpMatrix and XSnpMatrix classes and methods</td>
	<td class="  sorting_1">102</td>
</tr><tr id="pkg_mzID" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/mzID.html">mzID</a></td>
	<td class=" ">Thomas Lin Pedersen</td>
	<td class=" ">An mzIdentML parser for R</td>
	<td class="  sorting_1">103</td>
</tr><tr id="pkg_scran" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/scran.html">scran</a></td>
	<td class=" ">Aaron Lun</td>
	<td class=" ">Methods for Single-Cell RNA-Seq Data Analysis</td>
	<td class="  sorting_1">104</td>
</tr><tr id="pkg_seqLogo" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/seqLogo.html">seqLogo</a></td>
	<td class=" ">Oliver Bembom</td>
	<td class=" ">Sequence logos for DNA sequence alignments</td>
	<td class="  sorting_1">105</td>
</tr><tr id="pkg_graphite" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/graphite.html">graphite</a></td>
	<td class=" ">Gabriele Sales</td>
	<td class=" ">GRAPH Interaction from pathway Topological Environment</td>
	<td class="  sorting_1">106</td>
</tr><tr id="pkg_methylumi" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/methylumi.html">methylumi</a></td>
	<td class=" ">Sean Davis</td>
	<td class=" ">Handle Illumina methylation data</td>
	<td class="  sorting_1">107</td>
</tr><tr id="pkg_xcms" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/xcms.html">xcms</a></td>
	<td class=" ">Steffen Neumann</td>
	<td class=" ">LC/MS and GC/MS Data Analysis</td>
	<td class="  sorting_1">108</td>
</tr><tr id="pkg_goseq" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/goseq.html">goseq</a></td>
	<td class=" ">Nadia Davidson, Anthony Hawkins</td>
	<td class=" ">Gene Ontology analyser for RNA-seq and other length biased data</td>
	<td class="  sorting_1">109</td>
</tr><tr id="pkg_dada2" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/dada2.html">dada2</a></td>
	<td class=" ">Benjamin Callahan</td>
	<td class=" ">Accurate, high-resolution sample inference from amplicon sequencing data</td>
	<td class="  sorting_1">110</td>
</tr><tr id="pkg_gdsfmt" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/gdsfmt.html">gdsfmt</a></td>
	<td class=" ">Xiuwen Zheng</td>
	<td class=" ">R Interface to CoreArray Genomic Data Structure (GDS) Files</td>
	<td class="  sorting_1">110</td>
</tr><tr id="pkg_ChIPseeker" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/ChIPseeker.html">ChIPseeker</a></td>
	<td class=" ">Guangchuang Yu</td>
	<td class=" ">ChIPseeker for ChIP peak Annotation, Comparison, and Visualization</td>
	<td class="  sorting_1">112</td>
</tr><tr id="pkg_lumi" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/lumi.html">lumi</a></td>
	<td class=" ">Pan Du, Lei Huang, Gang Feng</td>
	<td class=" ">BeadArray Specific Methods for Illumina Methylation and Expression Microarrays</td>
	<td class="  sorting_1">113</td>
</tr><tr id="pkg_regioneR" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/regioneR.html">regioneR</a></td>
	<td class=" ">Bernat Gel</td>
	<td class=" ">Association analysis of genomic regions based on permutation tests</td>
	<td class="  sorting_1">114</td>
</tr><tr id="pkg_ReportingTools" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/ReportingTools.html">ReportingTools</a></td>
	<td class=" ">Jason A. Hackney, Gabriel Becker, Jessica L. Larson</td>
	<td class=" ">Tools for making reports in various formats</td>
	<td class="  sorting_1">115</td>
</tr><tr id="pkg_cummeRbund" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/cummeRbund.html">cummeRbund</a></td>
	<td class=" ">Loyal A. Goff</td>
	<td class=" ">Analysis, exploration, manipulation, and visualization of Cufflinks high-throughput sequencing data.</td>
	<td class="  sorting_1">116</td>
</tr><tr id="pkg_gage" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/gage.html">gage</a></td>
	<td class=" ">Weijun Luo</td>
	<td class=" ">Generally Applicable Gene-set Enrichment for Pathway Analysis</td>
	<td class="  sorting_1">117</td>
</tr><tr id="pkg_ballgown" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/ballgown.html">ballgown</a></td>
	<td class=" ">Jack Fu</td>
	<td class=" ">Flexible, isoform-level differential expression analysis</td>
	<td class="  sorting_1">118</td>
</tr><tr id="pkg_MassSpecWavelet" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/MassSpecWavelet.html">MassSpecWavelet</a></td>
	<td class=" ">Pan Du</td>
	<td class=" ">Mass spectrum processing by wavelet-based algorithms</td>
	<td class="  sorting_1">119</td>
</tr><tr id="pkg_ROC" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/ROC.html">ROC</a></td>
	<td class=" ">Vince Carey</td>
	<td class=" ">utilities for ROC, with uarray focus</td>
	<td class="  sorting_1">120</td>
</tr><tr id="pkg_maftools" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/maftools.html">maftools</a></td>
	<td class=" ">Anand Mayakonda</td>
	<td class=" ">Summarize, Analyze and Visualize MAF Files</td>
	<td class="  sorting_1">121</td>
</tr><tr id="pkg_ReactomePA" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/ReactomePA.html">ReactomePA</a></td>
	<td class=" ">Guangchuang Yu</td>
	<td class=" ">Reactome Pathway Analysis</td>
	<td class="  sorting_1">122</td>
</tr><tr id="pkg_destiny" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/destiny.html">destiny</a></td>
	<td class=" ">Philipp Angerer</td>
	<td class=" ">Creates diffusion maps</td>
	<td class="  sorting_1">123</td>
</tr><tr id="pkg_bsseq" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/bsseq.html">bsseq</a></td>
	<td class=" ">Kasper Daniel Hansen</td>
	<td class=" ">Analyze, manage and store bisulfite sequencing data</td>
	<td class="  sorting_1">124</td>
</tr><tr id="pkg_ChIPpeakAnno" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/ChIPpeakAnno.html">ChIPpeakAnno</a></td>
	<td class=" ">Lihua Julie Zhu, Jianhong Ou</td>
	<td class=" ">Batch annotation of the peaks identified from either ChIP-seq, ChIP-chip experiments or any experiments resulted in large number of chromosome ranges</td>
	<td class="  sorting_1">125</td>
</tr><tr id="pkg_SNPRelate" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/SNPRelate.html">SNPRelate</a></td>
	<td class=" ">Xiuwen Zheng</td>
	<td class=" ">Parallel Computing Toolset for Relatedness and Principal Component Analysis of SNP Data</td>
	<td class="  sorting_1">126</td>
</tr><tr id="pkg_Rhtslib" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/Rhtslib.html">Rhtslib</a></td>
	<td class=" ">Bioconductor Package Maintainer</td>
	<td class=" ">HTSlib high-throughput sequencing library as an R package</td>
	<td class="  sorting_1">127</td>
</tr><tr id="pkg_globaltest" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/globaltest.html">globaltest</a></td>
	<td class=" ">Jelle Goeman</td>
	<td class=" ">Testing Groups of Covariates/Features for Association with a Response Variable, with Applications to Gene Set Testing</td>
	<td class="  sorting_1">128</td>
</tr><tr id="pkg_simpleaffy" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/simpleaffy.html">simpleaffy</a></td>
	<td class=" ">Crispin Miller</td>
	<td class=" ">Very simple high level analysis of Affymetrix data</td>
	<td class="  sorting_1">128</td>
</tr><tr id="pkg_apeglm" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/apeglm.html">apeglm</a></td>
	<td class=" ">Anqi Zhu</td>
	<td class=" ">Approximate posterior estimation for GLM coefficients</td>
	<td class="  sorting_1">130</td>
</tr><tr id="pkg_systemPipeR" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/systemPipeR.html">systemPipeR</a></td>
	<td class=" ">Thomas Girke</td>
	<td class=" ">systemPipeR: NGS workflow and report generation environment</td>
	<td class="  sorting_1">131</td>
</tr><tr id="pkg_msa" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/msa.html">msa</a></td>
	<td class=" ">Ulrich Bodenhofer</td>
	<td class=" ">Multiple Sequence Alignment</td>
	<td class="  sorting_1">132</td>
</tr><tr id="pkg_Glimma" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/Glimma.html">Glimma</a></td>
	<td class=" ">Shian Su</td>
	<td class=" ">Interactive HTML graphics</td>
	<td class="  sorting_1">133</td>
</tr><tr id="pkg_GSVA" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/GSVA.html">GSVA</a></td>
	<td class=" ">Justin Guinney</td>
	<td class=" ">Gene Set Variation Analysis for microarray and RNA-seq data</td>
	<td class="  sorting_1">134</td>
</tr><tr id="pkg_minet" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/minet.html">minet</a></td>
	<td class=" ">Patrick E. Meyer</td>
	<td class=" ">Mutual Information NETworks</td>
	<td class="  sorting_1">135</td>
</tr><tr id="pkg_GenomicFiles" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/GenomicFiles.html">GenomicFiles</a></td>
	<td class=" ">Bioconductor Package Maintainer</td>
	<td class=" ">Distributed computing by file or by range</td>
	<td class="  sorting_1">136</td>
</tr><tr id="pkg_DECIPHER" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/DECIPHER.html">DECIPHER</a></td>
	<td class=" ">Erik Wright</td>
	<td class=" ">Tools for curating, analyzing, and manipulating biological sequences</td>
	<td class="  sorting_1">137</td>
</tr><tr id="pkg_dupRadar" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/dupRadar.html">dupRadar</a></td>
	<td class=" ">Sergi Sayols, Holger Klein</td>
	<td class=" ">Assessment of duplication rates in RNA-Seq datasets</td>
	<td class="  sorting_1">137</td>
</tr><tr id="pkg_RUVSeq" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/RUVSeq.html">RUVSeq</a></td>
	<td class=" ">Davide Risso</td>
	<td class=" ">Remove Unwanted Variation from RNA-Seq Data</td>
	<td class="  sorting_1">139</td>
</tr><tr id="pkg_DEXSeq" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/DEXSeq.html">DEXSeq</a></td>
	<td class=" ">Alejandro Reyes</td>
	<td class=" ">Inference of differential exon usage in RNA-Seq</td>
	<td class="  sorting_1">140</td>
</tr><tr id="pkg_MultiAssayExperiment" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/MultiAssayExperiment.html">MultiAssayExperiment</a></td>
	<td class=" ">Marcel Ramos</td>
	<td class=" ">Software for the integration of multi-omics experiments in Bioconductor</td>
	<td class="  sorting_1">141</td>
</tr><tr id="pkg_beadarray" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/beadarray.html">beadarray</a></td>
	<td class=" ">Mark Dunning</td>
	<td class=" ">Quality assessment and low-level analysis for Illumina BeadArray data</td>
	<td class="  sorting_1">142</td>
</tr><tr id="pkg_DiffBind" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/DiffBind.html">DiffBind</a></td>
	<td class=" ">Rory Stark</td>
	<td class=" ">Differential Binding Analysis of ChIP-Seq Peak Data</td>
	<td class="  sorting_1">143</td>
</tr><tr id="pkg_DynDoc" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/DynDoc.html">DynDoc</a></td>
	<td class=" ">Bioconductor Package Maintainer</td>
	<td class=" ">Dynamic document tools</td>
	<td class="  sorting_1">144</td>
</tr><tr id="pkg_flowViz" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/flowViz.html">flowViz</a></td>
	<td class=" ">Mike Jiang</td>
	<td class=" ">Visualization for flow cytometry</td>
	<td class="  sorting_1">144</td>
</tr><tr id="pkg_BeadDataPackR" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/BeadDataPackR.html">BeadDataPackR</a></td>
	<td class=" ">Mike Smith</td>
	<td class=" ">Compression of Illumina BeadArray data</td>
	<td class="  sorting_1">146</td>
</tr><tr id="pkg_Heatplus" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/Heatplus.html">Heatplus</a></td>
	<td class=" ">Alexander Ploner</td>
	<td class=" ">Heatmaps with row and/or column covariates and colored clusters</td>
	<td class="  sorting_1">147</td>
</tr><tr id="pkg_widgetTools" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/widgetTools.html">widgetTools</a></td>
	<td class=" ">Jianhua Zhang</td>
	<td class=" ">Creates an interactive tcltk widget</td>
	<td class="  sorting_1">148</td>
</tr><tr id="pkg_DSS" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/DSS.html">DSS</a></td>
	<td class=" ">Hao Wu</td>
	<td class=" ">Dispersion shrinkage for sequencing data</td>
	<td class="  sorting_1">149</td>
</tr><tr id="pkg_tkWidgets" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/tkWidgets.html">tkWidgets</a></td>
	<td class=" ">J. Zhang</td>
	<td class=" ">R based tk widgets</td>
	<td class="  sorting_1">150</td>
</tr><tr id="pkg_missMethyl" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/missMethyl.html">missMethyl</a></td>
	<td class=" ">Belinda Phipson, Jovana Maksimovic, Andrew Lonsdale</td>
	<td class=" ">Analysing Illumina HumanMethylation BeadChip Data</td>
	<td class="  sorting_1">151</td>
</tr><tr id="pkg_wateRmelon" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/wateRmelon.html">wateRmelon</a></td>
	<td class=" ">Leo</td>
	<td class=" ">Illumina 450 methylation array normalization and metrics</td>
	<td class="  sorting_1">152</td>
</tr><tr id="pkg_lpsymphony" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/lpsymphony.html">lpsymphony</a></td>
	<td class=" ">Vladislav Kim</td>
	<td class=" ">Symphony integer linear programming solver in R</td>
	<td class="  sorting_1">153</td>
</tr><tr id="pkg_flowWorkspace" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/flowWorkspace.html">flowWorkspace</a></td>
	<td class=" ">Greg Finak,Mike Jiang</td>
	<td class=" ">Infrastructure for representing and interacting with gated and ungated cytometry data sets.</td>
	<td class="  sorting_1">154</td>
</tr><tr id="pkg_NOISeq" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/NOISeq.html">NOISeq</a></td>
	<td class=" ">Sonia Tarazona</td>
	<td class=" ">Exploratory analysis and differential expression for RNA-seq data</td>
	<td class="  sorting_1">155</td>
</tr><tr id="pkg_metagenomeSeq" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/metagenomeSeq.html">metagenomeSeq</a></td>
	<td class=" ">Joseph N. Paulson</td>
	<td class=" ">Statistical analysis for sparse high-throughput sequencing</td>
	<td class="  sorting_1">156</td>
</tr><tr id="pkg_EBSeq" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/EBSeq.html">EBSeq</a></td>
	<td class=" ">Ning Leng</td>
	<td class=" ">An R package for gene and isoform differential expression analysis of RNA-seq data</td>
	<td class="  sorting_1">157</td>
</tr><tr id="pkg_SPIA" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/SPIA.html">SPIA</a></td>
	<td class=" ">Adi Laurentiu Tarca</td>
	<td class=" ">Signaling Pathway Impact Analysis (SPIA) using combined evidence of pathway over-representation and unusual signaling perturbations</td>
	<td class="  sorting_1">158</td>
</tr><tr id="pkg_RTCGAToolbox" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/RTCGAToolbox.html">RTCGAToolbox</a></td>
	<td class=" ">Marcel Ramos</td>
	<td class=" ">A new tool for exporting TCGA Firehose data</td>
	<td class="  sorting_1">159</td>
</tr><tr id="pkg_BiocFileCache" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/BiocFileCache.html">BiocFileCache</a></td>
	<td class=" ">Lori Shepherd</td>
	<td class=" ">Manage Files Across Sessions</td>
	<td class="  sorting_1">160</td>
</tr><tr id="pkg_DMRcate" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/DMRcate.html">DMRcate</a></td>
	<td class=" ">Tim Peters</td>
	<td class=" ">Methylation array and sequencing spatial analysis methods</td>
	<td class="  sorting_1">161</td>
</tr><tr id="pkg_flowUtils" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/flowUtils.html">flowUtils</a></td>
	<td class=" ">Josef Spidlen</td>
	<td class=" ">Utilities for flow cytometry</td>
	<td class="  sorting_1">162</td>
</tr><tr id="pkg_rGADEM" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/rGADEM.html">rGADEM</a></td>
	<td class=" ">Arnaud Droit</td>
	<td class=" ">de novo motif discovery</td>
	<td class="  sorting_1">163</td>
</tr><tr id="pkg_survcomp" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/survcomp.html">survcomp</a></td>
	<td class=" ">Benjamin Haibe-Kains, Markus Schroeder, Catharina Olsen</td>
	<td class=" ">Performance Assessment and Comparison for Survival Analysis</td>
	<td class="  sorting_1">163</td>
</tr><tr id="pkg_baySeq" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/baySeq.html">baySeq</a></td>
	<td class=" ">Thomas J. Hardcastle</td>
	<td class=" ">Empirical Bayesian analysis of patterns of differential expression in count data</td>
	<td class="  sorting_1">165</td>
</tr><tr id="pkg_methylKit" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/methylKit.html">methylKit</a></td>
	<td class=" ">Altuna Akalin, Alexander Gosdschan</td>
	<td class=" ">DNA methylation analysis from high-throughput bisulfite sequencing results</td>
	<td class="  sorting_1">166</td>
</tr><tr id="pkg_annaffy" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/annaffy.html">annaffy</a></td>
	<td class=" ">Colin A. Smith</td>
	<td class=" ">Annotation tools for Affymetrix biological metadata</td>
	<td class="  sorting_1">167</td>
</tr><tr id="pkg_SC3" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/SC3.html">SC3</a></td>
	<td class=" ">Vladimir Kiselev</td>
	<td class=" ">Single-Cell Consensus Clustering</td>
	<td class="  sorting_1">168</td>
</tr><tr id="pkg_IHW" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/IHW.html">IHW</a></td>
	<td class=" ">Nikos Ignatiadis</td>
	<td class=" ">Independent Hypothesis Weighting</td>
	<td class="  sorting_1">169</td>
</tr><tr id="pkg_SRAdb" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/SRAdb.html">SRAdb</a></td>
	<td class=" ">Jack Zhu</td>
	<td class=" ">A compilation of metadata from NCBI SRA and tools</td>
	<td class="  sorting_1">169</td>
</tr><tr id="pkg_FlowSOM" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/FlowSOM.html">FlowSOM</a></td>
	<td class=" ">Sofie Van Gassen</td>
	<td class=" ">Using self-organizing maps for visualization and interpretation of cytometry data</td>
	<td class="  sorting_1">171</td>
</tr><tr id="pkg_fastseg" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/fastseg.html">fastseg</a></td>
	<td class=" ">Guenter Klambauer</td>
	<td class=" ">fastseg - a fast segmentation algorithm</td>
	<td class="  sorting_1">172</td>
</tr><tr id="pkg_motifStack" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/motifStack.html">motifStack</a></td>
	<td class=" ">Jianhong Ou</td>
	<td class=" ">Plot stacked logos for single or multiple DNA, RNA and amino acid sequence</td>
	<td class="  sorting_1">173</td>
</tr><tr id="pkg_MAST" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/MAST.html">MAST</a></td>
	<td class=" ">Andrew McDavid</td>
	<td class=" ">Model-based Analysis of Single Cell Transcriptomics</td>
	<td class="  sorting_1">174</td>
</tr><tr id="pkg_RaggedExperiment" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/RaggedExperiment.html">RaggedExperiment</a></td>
	<td class=" ">Martin Morgan</td>
	<td class=" ">Representation of Sparse Experiments and Assays Across Samples</td>
	<td class="  sorting_1">175</td>
</tr><tr id="pkg_ncdfFlow" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/ncdfFlow.html">ncdfFlow</a></td>
	<td class=" ">Mike Jiang</td>
	<td class=" ">ncdfFlow: A package that provides HDF5 based storage for flow cytometry data.</td>
	<td class="  sorting_1">176</td>
</tr><tr id="pkg_arrayQualityMetrics" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/arrayQualityMetrics.html">arrayQualityMetrics</a></td>
	<td class=" ">Mike Smith</td>
	<td class=" ">Quality metrics report for microarray data sets</td>
	<td class="  sorting_1">177</td>
</tr><tr id="pkg_RTCGA" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/RTCGA.html">RTCGA</a></td>
	<td class=" ">Marcin Kosinski</td>
	<td class=" ">The Cancer Genome Atlas Data Integration</td>
	<td class="  sorting_1">177</td>
</tr><tr id="pkg_DirichletMultinomial" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/DirichletMultinomial.html">DirichletMultinomial</a></td>
	<td class=" ">Martin Morgan</td>
	<td class=" ">Dirichlet-Multinomial Mixture Model Machine Learning for Microbiome Data</td>
	<td class="  sorting_1">179</td>
</tr><tr id="pkg_affycoretools" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/affycoretools.html">affycoretools</a></td>
	<td class=" ">James W. MacDonald</td>
	<td class=" ">Functions useful for those doing repetitive analyses with Affymetrix GeneChips</td>
	<td class="  sorting_1">180</td>
</tr><tr id="pkg_ArrayExpress" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/ArrayExpress.html">ArrayExpress</a></td>
	<td class=" ">Suhaib Mohammed</td>
	<td class=" ">Access the ArrayExpress Microarray Database at EBI and build Bioconductor data structures: ExpressionSet, AffyBatch, NChannelSet</td>
	<td class="  sorting_1">181</td>
</tr><tr id="pkg_ChAMP" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/ChAMP.html">ChAMP</a></td>
	<td class=" ">Yuan Tian</td>
	<td class=" ">Chip Analysis Methylation Pipeline for Illumina HumanMethylation450 and EPIC</td>
	<td class="  sorting_1">182</td>
</tr><tr id="pkg_CAMERA" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/CAMERA.html">CAMERA</a></td>
	<td class=" ">Steffen Neumann</td>
	<td class=" ">Collection of annotation related methods for mass spectrometry data</td>
	<td class="  sorting_1">183</td>
</tr><tr id="pkg_STRINGdb" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/STRINGdb.html">STRINGdb</a></td>
	<td class=" ">Damian Szklarczyk</td>
	<td class=" ">STRINGdb (Search Tool for the Retrieval of Interacting proteins database)</td>
	<td class="  sorting_1">184</td>
</tr><tr id="pkg_MotIV" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/MotIV.html">MotIV</a></td>
	<td class=" ">Eloi Mercier, Raphael Gottardo</td>
	<td class=" ">Motif Identification and Validation</td>
	<td class="  sorting_1">185</td>
</tr><tr id="pkg_TFBSTools" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/TFBSTools.html">TFBSTools</a></td>
	<td class=" ">Ge Tan</td>
	<td class=" ">Software Package for Transcription Factor Binding Site (TFBS) Analysis</td>
	<td class="  sorting_1">185</td>
</tr><tr id="pkg_chipseq" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/chipseq.html">chipseq</a></td>
	<td class=" ">Bioconductor Package Maintainer</td>
	<td class=" ">chipseq: A package for analyzing chipseq data</td>
	<td class="  sorting_1">187</td>
</tr><tr id="pkg_ChemmineR" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/ChemmineR.html">ChemmineR</a></td>
	<td class=" ">Thomas Girke</td>
	<td class=" ">Cheminformatics Toolkit for R</td>
	<td class="  sorting_1">188</td>
</tr><tr id="pkg_CNEr" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/CNEr.html">CNEr</a></td>
	<td class=" ">Ge Tan</td>
	<td class=" ">CNE Detection and Visualization</td>
	<td class="  sorting_1">189</td>
</tr><tr id="pkg_RProtoBufLib" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/RProtoBufLib.html">RProtoBufLib</a></td>
	<td class=" ">Mike Jiang</td>
	<td class=" ">C++ headers and static libraries of Protocol buffers</td>
	<td class="  sorting_1">190</td>
</tr><tr id="pkg_cytolib" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/cytolib.html">cytolib</a></td>
	<td class=" ">Mike Jiang</td>
	<td class=" ">C++ infrastructure for representing and interacting with the gated cytometry</td>
	<td class="  sorting_1">191</td>
</tr><tr id="pkg_GenomicDataCommons" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/GenomicDataCommons.html">GenomicDataCommons</a></td>
	<td class=" ">Sean Davis</td>
	<td class=" ">NIH / NCI Genomic Data Commons Access</td>
	<td class="  sorting_1">191</td>
</tr><tr id="pkg_RDAVIDWebService" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/RDAVIDWebService.html">RDAVIDWebService</a></td>
	<td class=" ">Cristobal Fresno</td>
	<td class=" ">An R Package for retrieving data from DAVID into R objects using Web Services API.</td>
	<td class="  sorting_1">193</td>
</tr><tr id="pkg_RankProd" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/RankProd.html">RankProd</a></td>
	<td class=" ">Francesco Del Carratore</td>
	<td class=" ">Rank Product method for identifying differentially expressed genes with application in meta-analysis</td>
	<td class="  sorting_1">194</td>
</tr><tr id="pkg_BiocCheck" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/BiocCheck.html">BiocCheck</a></td>
	<td class=" ">Bioconductor Package Maintainer</td>
	<td class=" ">Bioconductor-specific package checks</td>
	<td class="  sorting_1">195</td>
</tr><tr id="pkg_FEM" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/FEM.html">FEM</a></td>
	<td class=" ">Zhen Yang</td>
	<td class=" ">Identification of Functional Epigenetic Modules</td>
	<td class="  sorting_1">196</td>
</tr><tr id="pkg_safe" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/safe.html">safe</a></td>
	<td class=" ">William T. Barry</td>
	<td class=" ">Significance Analysis of Function and Expression</td>
	<td class="  sorting_1">197</td>
</tr><tr id="pkg_ExperimentHub" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/ExperimentHub.html">ExperimentHub</a></td>
	<td class=" ">Bioconductor Package Maintainer</td>
	<td class=" ">Client to access ExperimentHub resources</td>
	<td class="  sorting_1">198</td>
</tr><tr id="pkg_MLInterfaces" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/MLInterfaces.html">MLInterfaces</a></td>
	<td class=" ">V. Carey</td>
	<td class=" ">Uniform interfaces to R machine learning procedures for data in Bioconductor containers</td>
	<td class="  sorting_1">198</td>
</tr><tr id="pkg_quantsmooth" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/quantsmooth.html">quantsmooth</a></td>
	<td class=" ">Jan Oosting</td>
	<td class=" ">Quantile smoothing and genomic visualization of array data</td>
	<td class="  sorting_1">200</td>
</tr><tr id="pkg_microbiome" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/microbiome.html">microbiome</a></td>
	<td class=" ">Leo Lahti</td>
	<td class=" ">Microbiome Analytics</td>
	<td class="  sorting_1">201</td>
</tr><tr id="pkg_supraHex" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/supraHex.html">supraHex</a></td>
	<td class=" ">Hai Fang</td>
	<td class=" ">supraHex: a supra-hexagonal map for analysing tabular omics data</td>
	<td class="  sorting_1">201</td>
</tr><tr id="pkg_made4" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/made4.html">made4</a></td>
	<td class=" ">Aedin Culhane</td>
	<td class=" ">Multivariate analysis of microarray data using ADE4</td>
	<td class="  sorting_1">203</td>
</tr><tr id="pkg_scde" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/scde.html">scde</a></td>
	<td class=" ">Jean Fan</td>
	<td class=" ">Single Cell Differential Expression</td>
	<td class="  sorting_1">203</td>
</tr><tr id="pkg_mygene" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/mygene.html">mygene</a></td>
	<td class=" ">Adam Mark, Cyrus Afrasiabi, Chunlei Wu</td>
	<td class=" ">Access MyGene.Info_ services</td>
	<td class="  sorting_1">205</td>
</tr><tr id="pkg_derfinder" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/derfinder.html">derfinder</a></td>
	<td class=" ">Leonardo Collado-Torres</td>
	<td class=" ">Annotation-agnostic differential expression analysis of RNA-seq data at base-pair resolution via the DER Finder approach</td>
	<td class="  sorting_1">206</td>
</tr><tr id="pkg_ropls" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/ropls.html">ropls</a></td>
	<td class=" ">Etienne A. Thevenot</td>
	<td class=" ">PCA, PLS(-DA) and OPLS(-DA) for multivariate analysis and feature selection of omics data</td>
	<td class="  sorting_1">206</td>
</tr><tr id="pkg_piano" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/piano.html">piano</a></td>
	<td class=" ">Leif Varemo</td>
	<td class=" ">Platform for integrative analysis of omics data</td>
	<td class="  sorting_1">208</td>
</tr><tr id="pkg_GWASTools" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/GWASTools.html">GWASTools</a></td>
	<td class=" ">Stephanie M. Gogarten</td>
	<td class=" ">Tools for Genome Wide Association Studies</td>
	<td class="  sorting_1">209</td>
</tr><tr id="pkg_ctc" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/ctc.html">ctc</a></td>
	<td class=" ">Antoine Lucas</td>
	<td class=" ">Cluster and Tree Conversion.</td>
	<td class="  sorting_1">210</td>
</tr><tr id="pkg_DEGseq" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/DEGseq.html">DEGseq</a></td>
	<td class=" ">Likun Wang</td>
	<td class=" ">Identify Differentially Expressed Genes from RNA-seq data</td>
	<td class="  sorting_1">211</td>
</tr><tr id="pkg_genomation" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/genomation.html">genomation</a></td>
	<td class=" ">Altuna Akalin, Vedran Franke, Katarzyna Wreczycka</td>
	<td class=" ">Summary, annotation and visualization of genomic data</td>
	<td class="  sorting_1">211</td>
</tr><tr id="pkg_ggcyto" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/ggcyto.html">ggcyto</a></td>
	<td class=" ">Mike Jiang</td>
	<td class=" ">Visualize Cytometry data with ggplot</td>
	<td class="  sorting_1">211</td>
</tr><tr id="pkg_derfinderHelper" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/derfinderHelper.html">derfinderHelper</a></td>
	<td class=" ">Leonardo Collado-Torres</td>
	<td class=" ">derfinder helper package</td>
	<td class="  sorting_1">214</td>
</tr><tr id="pkg_MotifDb" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/MotifDb.html">MotifDb</a></td>
	<td class=" ">Paul Shannon</td>
	<td class=" ">An Annotated Collection of Protein-DNA Binding Sequence Motifs</td>
	<td class="  sorting_1">215</td>
</tr><tr id="pkg_RCy3" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/RCy3.html">RCy3</a></td>
	<td class=" ">Alexander Pico, Tanja Muetze, Paul Shannon</td>
	<td class=" ">Functions to Access and Control Cytoscape</td>
	<td class="  sorting_1">216</td>
</tr><tr id="pkg_flowStats" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/flowStats.html">flowStats</a></td>
	<td class=" ">Greg Finak and Mike Jiang</td>
	<td class=" ">Statistical methods for the analysis of flow cytometry data</td>
	<td class="  sorting_1">217</td>
</tr><tr id="pkg_cytofkit" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/cytofkit.html">cytofkit</a></td>
	<td class=" ">Jinmiao Chen, Matthew Myint</td>
	<td class=" ">cytofkit: an integrated mass cytometry data analysis pipeline</td>
	<td class="  sorting_1">218</td>
</tr><tr id="pkg_ELMER" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/ELMER.html">ELMER</a></td>
	<td class=" ">Tiago Chedraoui Silva</td>
	<td class=" ">Inferring Regulatory Element Landscapes and Transcription Factor Networks Using Cancer Methylomes</td>
	<td class="  sorting_1">219</td>
</tr><tr id="pkg_karyoploteR" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/karyoploteR.html">karyoploteR</a></td>
	<td class=" ">Bernat Gel</td>
	<td class=" ">Plot customizable linear genomes displaying arbitrary data</td>
	<td class="  sorting_1">220</td>
</tr><tr id="pkg_flowClust" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/flowClust.html">flowClust</a></td>
	<td class=" ">Greg Finak, Mike Jiang</td>
	<td class=" ">Clustering for Flow Cytometry</td>
	<td class="  sorting_1">221</td>
</tr><tr id="pkg_Icens" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/Icens.html">Icens</a></td>
	<td class=" ">Bioconductor Package Maintainer</td>
	<td class=" ">NPMLE for Censored and Truncated Data</td>
	<td class="  sorting_1">222</td>
</tr><tr id="pkg_affyQCReport" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/affyQCReport.html">affyQCReport</a></td>
	<td class=" ">Craig Parman</td>
	<td class=" ">QC Report Generation for affyBatch objects</td>
	<td class="  sorting_1">223</td>
</tr><tr id="pkg_BioNet" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/BioNet.html">BioNet</a></td>
	<td class=" ">Marcus Dittrich</td>
	<td class=" ">Routines for the functional analysis of biological networks</td>
	<td class="  sorting_1">223</td>
</tr><tr id="pkg_seqPattern" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/seqPattern.html">seqPattern</a></td>
	<td class=" ">Vanja Haberle</td>
	<td class=" ">Visualising oligonucleotide patterns and motif occurrences across a set of sorted sequences</td>
	<td class="  sorting_1">225</td>
</tr><tr id="pkg_Ringo" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/Ringo.html">Ringo</a></td>
	<td class=" ">J. Toedling</td>
	<td class=" ">R Investigation of ChIP-chip Oligoarrays</td>
	<td class="  sorting_1">226</td>
</tr><tr id="pkg_Sushi" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/Sushi.html">Sushi</a></td>
	<td class=" ">Douglas H Phanstiel</td>
	<td class=" ">Tools for visualizing genomics data</td>
	<td class="  sorting_1">226</td>
</tr><tr id="pkg_ReadqPCR" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/ReadqPCR.html">ReadqPCR</a></td>
	<td class=" ">James Perkins</td>
	<td class=" ">Read qPCR data</td>
	<td class="  sorting_1">228</td>
</tr><tr id="pkg_Rbowtie" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/Rbowtie.html">Rbowtie</a></td>
	<td class=" ">Michael Stadler</td>
	<td class=" ">R bowtie wrapper</td>
	<td class="  sorting_1">229</td>
</tr><tr id="pkg_zinbwave" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/zinbwave.html">zinbwave</a></td>
	<td class=" ">Davide Risso</td>
	<td class=" ">Zero-Inflated Negative Binomial Model for RNA-Seq Data</td>
	<td class="  sorting_1">229</td>
</tr><tr id="pkg_QuasR" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/QuasR.html">QuasR</a></td>
	<td class=" ">Michael Stadler</td>
	<td class=" ">Quantify and Annotate Short Reads in R</td>
	<td class="  sorting_1">231</td>
</tr><tr id="pkg_Mfuzz" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/Mfuzz.html">Mfuzz</a></td>
	<td class=" ">Matthias Futschik</td>
	<td class=" ">Soft clustering of time series gene expression data</td>
	<td class="  sorting_1">232</td>
</tr><tr id="pkg_rpx" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/rpx.html">rpx</a></td>
	<td class=" ">Laurent Gatto</td>
	<td class=" ">R Interface to the ProteomeXchange Repository</td>
	<td class="  sorting_1">232</td>
</tr><tr id="pkg_MSstats" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/MSstats.html">MSstats</a></td>
	<td class=" ">Meena Choi</td>
	<td class=" ">Protein Significance Analysis in DDA, SRM and DIA for Label-free or Label-based Proteomics Experiments</td>
	<td class="  sorting_1">234</td>
</tr><tr id="pkg_NormqPCR" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/NormqPCR.html">NormqPCR</a></td>
	<td class=" ">James Perkins</td>
	<td class=" ">Functions for normalisation of RT-qPCR data</td>
	<td class="  sorting_1">235</td>
</tr><tr id="pkg_GenVisR" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/GenVisR.html">GenVisR</a></td>
	<td class=" ">Zachary Skidmore</td>
	<td class=" ">Genomic Visualizations in R</td>
	<td class="  sorting_1">236</td>
</tr><tr id="pkg_recount" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/recount.html">recount</a></td>
	<td class=" ">Leonardo Collado-Torres</td>
	<td class=" ">Explore and download data from the recount project</td>
	<td class="  sorting_1">237</td>
</tr><tr id="pkg_SCAN.UPC" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/SCAN.UPC.html">SCAN.UPC</a></td>
	<td class=" ">Stephen R. Piccolo</td>
	<td class=" ">Single-channel array normalization (SCAN) and Universal exPression Codes (UPC)</td>
	<td class="  sorting_1">237</td>
</tr><tr id="pkg_Rdisop" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/Rdisop.html">Rdisop</a></td>
	<td class=" ">Steffen Neumann</td>
	<td class=" ">Decomposition of Isotopic Patterns</td>
	<td class="  sorting_1">239</td>
</tr><tr id="pkg_SSPA" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/SSPA.html">SSPA</a></td>
	<td class=" ">Maarten van Iterson</td>
	<td class=" ">General Sample Size and Power Analysis for Microarray and Next-Generation Sequencing Data</td>
	<td class="  sorting_1">239</td>
</tr><tr id="pkg_EnrichmentBrowser" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/EnrichmentBrowser.html">EnrichmentBrowser</a></td>
	<td class=" ">Ludwig Geistlinger</td>
	<td class=" ">Seamless navigation through combined results of set-based and network-based enrichment analysis</td>
	<td class="  sorting_1">241</td>
</tr><tr id="pkg_UniProt.ws" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/UniProt.ws.html">UniProt.ws</a></td>
	<td class=" ">Marc Carlson</td>
	<td class=" ">R Interface to UniProt Web Services</td>
	<td class="  sorting_1">242</td>
</tr><tr id="pkg_maSigPro" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/maSigPro.html">maSigPro</a></td>
	<td class=" ">Maria Jose Nueda</td>
	<td class=" ">Significant Gene Expression Profile Differences in Time Course Gene Expression Data</td>
	<td class="  sorting_1">243</td>
</tr><tr id="pkg_LEA" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/LEA.html">LEA</a></td>
	<td class=" ">Eric Frichot, Olivier Francois</td>
	<td class=" ">LEA: an R package for Landscape and Ecological Association Studies</td>
	<td class="  sorting_1">244</td>
</tr><tr id="pkg_bioDist" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/bioDist.html">bioDist</a></td>
	<td class=" ">Bioconductor Package Maintainer</td>
	<td class=" ">Different distance measures</td>
	<td class="  sorting_1">245</td>
</tr><tr id="pkg_GenomicScores" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/GenomicScores.html">GenomicScores</a></td>
	<td class=" ">Robert Castelo</td>
	<td class=" ">Infrastructure to work with genomewide position-specific scores</td>
	<td class="  sorting_1">245</td>
</tr><tr id="pkg_clusterExperiment" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/clusterExperiment.html">clusterExperiment</a></td>
	<td class=" ">Elizabeth Purdom</td>
	<td class=" ">Compare Clusterings for Single-Cell Sequencing</td>
	<td class="  sorting_1">247</td>
</tr><tr id="pkg_GENIE3" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/GENIE3.html">GENIE3</a></td>
	<td class=" ">Van Anh Huynh-Thu</td>
	<td class=" ">GEne Network Inference with Ensemble of trees</td>
	<td class="  sorting_1">248</td>
</tr><tr id="pkg_genomeIntervals" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/genomeIntervals.html">genomeIntervals</a></td>
	<td class=" ">Julien Gagneur</td>
	<td class=" ">Operations on genomic intervals</td>
	<td class="  sorting_1">249</td>
</tr><tr id="pkg_GlobalAncova" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/GlobalAncova.html">GlobalAncova</a></td>
	<td class=" ">Manuela Hummel</td>
	<td class=" ">Global test for groups of variables via model comparisons</td>
	<td class="  sorting_1">249</td>
</tr><tr id="pkg_csaw" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/csaw.html">csaw</a></td>
	<td class=" ">Aaron Lun</td>
	<td class=" ">ChIP-Seq Analysis with Windows</td>
	<td class="  sorting_1">251</td>
</tr><tr id="pkg_genefu" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/genefu.html">genefu</a></td>
	<td class=" ">Benjamin Haibe-Kains, Markus Schroeder</td>
	<td class=" ">Computation of Gene Expression-Based Signatures in Breast Cancer</td>
	<td class="  sorting_1">251</td>
</tr><tr id="pkg_HilbertVis" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/HilbertVis.html">HilbertVis</a></td>
	<td class=" ">Simon Anders</td>
	<td class=" ">Hilbert curve visualization</td>
	<td class="  sorting_1">253</td>
</tr><tr id="pkg_DropletUtils" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/DropletUtils.html">DropletUtils</a></td>
	<td class=" ">Aaron Lun</td>
	<td class=" ">Utilities for Handling Single-Cell Droplet Data</td>
	<td class="  sorting_1">254</td>
</tr><tr id="pkg_exomeCopy" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/exomeCopy.html">exomeCopy</a></td>
	<td class=" ">Michael Love</td>
	<td class=" ">Copy number variant detection from exome sequencing read depth</td>
	<td class="  sorting_1">254</td>
</tr><tr id="pkg_OmicCircos" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/OmicCircos.html">OmicCircos</a></td>
	<td class=" ">Ying Hu</td>
	<td class=" ">High-quality circular visualization of omics data</td>
	<td class="  sorting_1">256</td>
</tr><tr id="pkg_copynumber" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/copynumber.html">copynumber</a></td>
	<td class=" ">Gro Nilsen</td>
	<td class=" ">Segmentation of single- and multi-track copy number data by penalized least squares regression.</td>
	<td class="  sorting_1">257</td>
</tr><tr id="pkg_interactiveDisplay" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/interactiveDisplay.html">interactiveDisplay</a></td>
	<td class=" ">Shawn Balcome</td>
	<td class=" ">Package for enabling powerful shiny web displays of Bioconductor objects</td>
	<td class="  sorting_1">257</td>
</tr><tr id="pkg_pcaExplorer" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/pcaExplorer.html">pcaExplorer</a></td>
	<td class=" ">Federico Marini</td>
	<td class=" ">Interactive Visualization of RNA-seq Data Using a Principal Components Approach</td>
	<td class="  sorting_1">259</td>
</tr><tr id="pkg_SeqArray" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/SeqArray.html">SeqArray</a></td>
	<td class=" ">Xiuwen Zheng</td>
	<td class=" ">Data Management of Large-scale Whole-genome Sequence Variant Calls</td>
	<td class="  sorting_1">260</td>
</tr><tr id="pkg_GEOmetadb" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/GEOmetadb.html">GEOmetadb</a></td>
	<td class=" ">Jack Zhu</td>
	<td class=" ">A compilation of metadata from NCBI GEO</td>
	<td class="  sorting_1">261</td>
</tr><tr id="pkg_pRoloc" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/pRoloc.html">pRoloc</a></td>
	<td class=" ">Laurent Gatto</td>
	<td class=" ">A unifying bioinformatics framework for spatial proteomics</td>
	<td class="  sorting_1">261</td>
</tr><tr id="pkg_Repitools" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/Repitools.html">Repitools</a></td>
	<td class=" ">Mark Robinson</td>
	<td class=" ">Epigenomic tools</td>
	<td class="  sorting_1">263</td>
</tr><tr id="pkg_SNPchip" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/SNPchip.html">SNPchip</a></td>
	<td class=" ">Robert Scharpf</td>
	<td class=" ">Visualizations for copy number alterations</td>
	<td class="  sorting_1">263</td>
</tr><tr id="pkg_RnBeads" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/RnBeads.html">RnBeads</a></td>
	<td class=" ">Fabian Mueller</td>
	<td class=" ">RnBeads</td>
	<td class="  sorting_1">265</td>
</tr><tr id="pkg_sangerseqR" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/sangerseqR.html">sangerseqR</a></td>
	<td class=" ">Jonathon Hill</td>
	<td class=" ">Tools for Sanger Sequencing Data in R</td>
	<td class="  sorting_1">266</td>
</tr><tr id="pkg_prada" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/prada.html">prada</a></td>
	<td class=" ">Florian Hahne</td>
	<td class=" ">Data analysis for cell-based functional assays</td>
	<td class="  sorting_1">267</td>
</tr><tr id="pkg_rols" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/rols.html">rols</a></td>
	<td class=" ">Laurent Gatto</td>
	<td class=" ">An R interface to the Ontology Lookup Service</td>
	<td class="  sorting_1">267</td>
</tr><tr id="pkg_AUCell" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/AUCell.html">AUCell</a></td>
	<td class=" ">Sara Aibar</td>
	<td class=" ">AUCell: Analysis of 'gene set' activity in single-cell RNA-seq data (e.g. identify cells with specific gene signatures)</td>
	<td class="  sorting_1">269</td>
</tr><tr id="pkg_InteractionSet" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/InteractionSet.html">InteractionSet</a></td>
	<td class=" ">Aaron Lun</td>
	<td class=" ">Base Classes for Storing Genomic Interaction Data</td>
	<td class="  sorting_1">270</td>
</tr><tr id="pkg_AIMS" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/AIMS.html">AIMS</a></td>
	<td class=" ">Eric R Paquet</td>
	<td class=" ">AIMS : Absolute Assignment of Breast Cancer Intrinsic Molecular Subtype</td>
	<td class="  sorting_1">271</td>
</tr><tr id="pkg_biocGraph" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/biocGraph.html">biocGraph</a></td>
	<td class=" ">Florian Hahne</td>
	<td class=" ">Graph examples and use cases in Bioinformatics</td>
	<td class="  sorting_1">272</td>
</tr><tr id="pkg_hopach" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/hopach.html">hopach</a></td>
	<td class=" ">Katherine S. Pollard</td>
	<td class=" ">Hierarchical Ordered Partitioning and Collapsing Hybrid (HOPACH)</td>
	<td class="  sorting_1">273</td>
</tr><tr id="pkg_hpar" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/hpar.html">hpar</a></td>
	<td class=" ">Laurent Gatto</td>
	<td class=" ">Human Protein Atlas in R</td>
	<td class="  sorting_1">274</td>
</tr><tr id="pkg_SomaticSignatures" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/SomaticSignatures.html">SomaticSignatures</a></td>
	<td class=" ">Julian Gehring</td>
	<td class=" ">Somatic Signatures</td>
	<td class="  sorting_1">274</td>
</tr><tr id="pkg_erma" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/erma.html">erma</a></td>
	<td class=" ">VJ Carey</td>
	<td class=" ">epigenomic road map adventures</td>
	<td class="  sorting_1">276</td>
</tr><tr id="pkg_GenomeGraphs" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/GenomeGraphs.html">GenomeGraphs</a></td>
	<td class=" ">Steffen Durinck</td>
	<td class=" ">Plotting genomic information from Ensembl</td>
	<td class="  sorting_1">277</td>
</tr><tr id="pkg_CNTools" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/CNTools.html">CNTools</a></td>
	<td class=" ">J. Zhang</td>
	<td class=" ">Convert segment data into a region by sample matrix to allow for other high level computational analyses.</td>
	<td class="  sorting_1">278</td>
</tr><tr id="pkg_pathifier" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/pathifier.html">pathifier</a></td>
	<td class=" ">Assif Yitzhaky</td>
	<td class=" ">Quantify deregulation of pathways in cancer</td>
	<td class="  sorting_1">278</td>
</tr><tr id="pkg_openCyto" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/openCyto.html">openCyto</a></td>
	<td class=" ">Mike Jiang</td>
	<td class=" ">Hierarchical Gating Pipeline for flow cytometry data</td>
	<td class="  sorting_1">280</td>
</tr><tr id="pkg_ATACseqQC" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/ATACseqQC.html">ATACseqQC</a></td>
	<td class=" ">Jianhong Ou</td>
	<td class=" ">ATAC-seq Quality Control</td>
	<td class="  sorting_1">281</td>
</tr><tr id="pkg_CGHbase" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/CGHbase.html">CGHbase</a></td>
	<td class=" ">Mark van de Wiel</td>
	<td class=" ">CGHbase: Base functions and classes for arrayCGH data analysis.</td>
	<td class="  sorting_1">281</td>
</tr><tr id="pkg_OrderedList" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/OrderedList.html">OrderedList</a></td>
	<td class=" ">Claudio Lottaz</td>
	<td class=" ">Similarities of Ordered Gene Lists</td>
	<td class="  sorting_1">281</td>
</tr><tr id="pkg_ChIPQC" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/ChIPQC.html">ChIPQC</a></td>
	<td class=" ">Tom Carroll, Rory Stark</td>
	<td class=" ">Quality metrics for ChIPseq data</td>
	<td class="  sorting_1">284</td>
</tr><tr id="pkg_PGSEA" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/PGSEA.html">PGSEA</a></td>
	<td class=" ">Karl Dykema</td>
	<td class=" ">Parametric Gene Set Enrichment Analysis</td>
	<td class="  sorting_1">285</td>
</tr><tr id="pkg_frma" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/frma.html">frma</a></td>
	<td class=" ">Matthew N. McCall</td>
	<td class=" ">Frozen RMA and Barcode</td>
	<td class="  sorting_1">286</td>
</tr><tr id="pkg_twilight" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/twilight.html">twilight</a></td>
	<td class=" ">Stefanie Scheid</td>
	<td class=" ">Estimation of local false discovery rate</td>
	<td class="  sorting_1">286</td>
</tr><tr id="pkg_annotatr" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/annotatr.html">annotatr</a></td>
	<td class=" ">Raymond G. Cavalcante</td>
	<td class=" ">Annotation of Genomic Regions to Genomic Annotations</td>
	<td class="  sorting_1">288</td>
</tr><tr id="pkg_logicFS" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/logicFS.html">logicFS</a></td>
	<td class=" ">Holger Schwender</td>
	<td class=" ">Identification of SNP Interactions</td>
	<td class="  sorting_1">288</td>
</tr><tr id="pkg_cellHTS2" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/cellHTS2.html">cellHTS2</a></td>
	<td class=" ">Joseph Barry</td>
	<td class=" ">Analysis of cell-based screens - revised version of cellHTS</td>
	<td class="  sorting_1">290</td>
</tr><tr id="pkg_GLAD" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/GLAD.html">GLAD</a></td>
	<td class=" ">Philippe Hupe</td>
	<td class=" ">Gain and Loss Analysis of DNA</td>
	<td class="  sorting_1">290</td>
</tr><tr id="pkg_gQTLBase" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/gQTLBase.html">gQTLBase</a></td>
	<td class=" ">VJ Carey</td>
	<td class=" ">gQTLBase: infrastructure for eQTL, mQTL and similar studies</td>
	<td class="  sorting_1">292</td>
</tr><tr id="pkg_gQTLstats" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/gQTLstats.html">gQTLstats</a></td>
	<td class=" ">VJ Carey</td>
	<td class=" ">gQTLstats: computationally efficient analysis for eQTL and allied studies</td>
	<td class="  sorting_1">292</td>
</tr><tr id="pkg_HiTC" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/HiTC.html">HiTC</a></td>
	<td class=" ">Nicolas Servant</td>
	<td class=" ">High Throughput Chromosome Conformation Capture analysis</td>
	<td class="  sorting_1">292</td>
</tr><tr id="pkg_plier" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/plier.html">plier</a></td>
	<td class=" ">Crispin Miller</td>
	<td class=" ">Implements the Affymetrix PLIER algorithm</td>
	<td class="  sorting_1">292</td>
</tr><tr id="pkg_splots" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/splots.html">splots</a></td>
	<td class=" ">Wolfgang Huber</td>
	<td class=" ">Visualization of high-throughput assays in microtitre plate or slide format</td>
	<td class="  sorting_1">292</td>
</tr><tr id="pkg_gwascat" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/gwascat.html">gwascat</a></td>
	<td class=" ">VJ Carey</td>
	<td class=" ">representing and modeling data in the EMBL-EBI GWAS catalog</td>
	<td class="  sorting_1">297</td>
</tr><tr id="pkg_bamsignals" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/bamsignals.html">bamsignals</a></td>
	<td class=" ">Alessandro Mammana</td>
	<td class=" ">Extract read count signals from bam files</td>
	<td class="  sorting_1">298</td>
</tr><tr id="pkg_KEGGprofile" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/KEGGprofile.html">KEGGprofile</a></td>
	<td class=" ">Shilin Zhao</td>
	<td class=" ">An annotation and visualization package for multi-types and multi-groups expression data in KEGG pathway</td>
	<td class="  sorting_1">298</td>
</tr><tr id="pkg_trackViewer" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/trackViewer.html">trackViewer</a></td>
	<td class=" ">Jianhong Ou</td>
	<td class=" ">A R/Bioconductor package for drawing elegant interactive tracks or lollipop plot to facilitate integrated analysis of multi-omics data</td>
	<td class="  sorting_1">300</td>
</tr><tr id="pkg_qusage" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/qusage.html">qusage</a></td>
	<td class=" ">Christopher Bolen</td>
	<td class=" ">qusage: Quantitative Set Analysis for Gene Expression</td>
	<td class="  sorting_1">301</td>
</tr><tr id="pkg_splatter" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/splatter.html">splatter</a></td>
	<td class=" ">Luke Zappia</td>
	<td class=" ">Simple Simulation of Single-cell RNA Sequencing Data</td>
	<td class="  sorting_1">301</td>
</tr><tr id="pkg_DEFormats" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/DEFormats.html">DEFormats</a></td>
	<td class=" ">Andrzej Ole</td>
	<td class=" ">Differential gene expression data formats converter</td>
	<td class="  sorting_1">303</td>
</tr><tr id="pkg_HTqPCR" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/HTqPCR.html">HTqPCR</a></td>
	<td class=" ">Heidi Dvinge</td>
	<td class=" ">Automated analysis of high-throughput qPCR data</td>
	<td class="  sorting_1">303</td>
</tr><tr id="pkg_CGHcall" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/CGHcall.html">CGHcall</a></td>
	<td class=" ">Mark van de Wiel</td>
	<td class=" ">Calling aberrations for array CGH tumor profiles.</td>
	<td class="  sorting_1">305</td>
</tr><tr id="pkg_ALDEx2" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/ALDEx2.html">ALDEx2</a></td>
	<td class=" ">Greg Gloor</td>
	<td class=" ">Analysis Of Differential Abundance Taking Sample Variation Into Account</td>
	<td class="  sorting_1">306</td>
</tr><tr id="pkg_MSGFplus" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/MSGFplus.html">MSGFplus</a></td>
	<td class=" ">Thomas Lin Pedersen</td>
	<td class=" ">An interface between R and MS-GF+</td>
	<td class="  sorting_1">306</td>
</tr><tr id="pkg_PADOG" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/PADOG.html">PADOG</a></td>
	<td class=" ">Adi Laurentiu Tarca</td>
	<td class=" ">Pathway Analysis with Down-weighting of Overlapping Genes (PADOG)</td>
	<td class="  sorting_1">308</td>
</tr><tr id="pkg_cn.mops" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/cn.mops.html">cn.mops</a></td>
	<td class=" ">Guenter Klambauer</td>
	<td class=" ">cn.mops - Mixture of Poissons for CNV detection in NGS data</td>
	<td class="  sorting_1">309</td>
</tr><tr id="pkg_MSnID" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/MSnID.html">MSnID</a></td>
	<td class=" ">Vlad Petyuk</td>
	<td class=" ">Utilities for Exploration and Assessment of Confidence of LC-MSn Proteomics Identifications</td>
	<td class="  sorting_1">309</td>
</tr><tr id="pkg_TCC" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/TCC.html">TCC</a></td>
	<td class=" ">Jianqiang Sun, Tomoaki Nishiyama</td>
	<td class=" ">TCC: Differential expression analysis for tag count data with robust normalization strategies</td>
	<td class="  sorting_1">309</td>
</tr><tr id="pkg_ChemmineOB" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/ChemmineOB.html">ChemmineOB</a></td>
	<td class=" ">Thomas Girke</td>
	<td class=" ">R interface to a subset of OpenBabel functionalities</td>
	<td class="  sorting_1">312</td>
</tr><tr id="pkg_Organism.dplyr" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/Organism.dplyr.html">Organism.dplyr</a></td>
	<td class=" ">Martin Morgan</td>
	<td class=" ">dplyr-based Access to Bioconductor Annotation Resources</td>
	<td class="  sorting_1">312</td>
</tr><tr id="pkg_biobroom" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/biobroom.html">biobroom</a></td>
	<td class=" ">John D. Storey and Andrew J. Bass</td>
	<td class=" ">Turn Bioconductor objects into tidy data frames</td>
	<td class="  sorting_1">314</td>
</tr><tr id="pkg_chopsticks" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/chopsticks.html">chopsticks</a></td>
	<td class=" ">Hin-Tak Leung</td>
	<td class=" ">The 'snp.matrix' and 'X.snp.matrix' Classes</td>
	<td class="  sorting_1">314</td>
</tr><tr id="pkg_TCGAbiolinksGUI" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/TCGAbiolinksGUI.html">TCGAbiolinksGUI</a></td>
	<td class=" ">Tiago C. Silva</td>
	<td class=" ">"TCGAbiolinksGUI: A Graphical User Interface to analyze cancer molecular and clinical data"</td>
	<td class="  sorting_1">314</td>
</tr><tr id="pkg_cleaver" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/cleaver.html">cleaver</a></td>
	<td class=" ">Sebastian Gibb</td>
	<td class=" ">Cleavage of Polypeptide Sequences</td>
	<td class="  sorting_1">317</td>
</tr><tr id="pkg_EGSEA" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/EGSEA.html">EGSEA</a></td>
	<td class=" ">Monther Alhamdoosh</td>
	<td class=" ">Ensemble of Gene Set Enrichment Analyses</td>
	<td class="  sorting_1">318</td>
</tr><tr id="pkg_flowMeans" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/flowMeans.html">flowMeans</a></td>
	<td class=" ">Nima Aghaeepour</td>
	<td class=" ">Non-parametric Flow Cytometry Data Gating</td>
	<td class="  sorting_1">318</td>
</tr><tr id="pkg_lfa" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/lfa.html">lfa</a></td>
	<td class=" ">Wei Hao, John D. Storey</td>
	<td class=" ">Logistic Factor Analysis for Categorical Data</td>
	<td class="  sorting_1">320</td>
</tr><tr id="pkg_MutationalPatterns" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/MutationalPatterns.html">MutationalPatterns</a></td>
	<td class=" ">Roel Janssen, Francis Blokzijl</td>
	<td class=" ">Comprehensive genome-wide analysis of mutational processes</td>
	<td class="  sorting_1">320</td>
</tr><tr id="pkg_SeqVarTools" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/SeqVarTools.html">SeqVarTools</a></td>
	<td class=" ">Stephanie M. Gogarten</td>
	<td class=" ">Tools for variant data</td>
	<td class="  sorting_1">322</td>
</tr><tr id="pkg_viper" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/viper.html">viper</a></td>
	<td class=" ">Mariano J Alvarez</td>
	<td class=" ">Virtual Inference of Protein-activity by Enriched Regulon analysis</td>
	<td class="  sorting_1">322</td>
</tr><tr id="pkg_convert" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/convert.html">convert</a></td>
	<td class=" ">Yee Hwa (Jean) Yang</td>
	<td class=" ">Convert Microarray Data Objects</td>
	<td class="  sorting_1">324</td>
</tr><tr id="pkg_cqn" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/cqn.html">cqn</a></td>
	<td class=" ">Kasper Daniel Hansen</td>
	<td class=" ">Conditional quantile normalization</td>
	<td class="  sorting_1">325</td>
</tr><tr id="pkg_genoset" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/genoset.html">genoset</a></td>
	<td class=" ">Peter M. Haverty</td>
	<td class=" ">A RangedSummarizedExperiment with methods for copy number analysis</td>
	<td class="  sorting_1">325</td>
</tr><tr id="pkg_DEGreport" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/DEGreport.html">DEGreport</a></td>
	<td class=" ">Lorena Pantano</td>
	<td class=" ">Report of DEG analysis</td>
	<td class="  sorting_1">327</td>
</tr><tr id="pkg_gaia" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/gaia.html">gaia</a></td>
	<td class=" ">S. Morganella</td>
	<td class=" ">GAIA: An R package for genomic analysis of significant chromosomal aberrations.</td>
	<td class="  sorting_1">327</td>
</tr><tr id="pkg_makecdfenv" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/makecdfenv.html">makecdfenv</a></td>
	<td class=" ">James W. MacDonald</td>
	<td class=" ">CDF Environment Maker</td>
	<td class="  sorting_1">327</td>
</tr><tr id="pkg_QDNAseq" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/QDNAseq.html">QDNAseq</a></td>
	<td class=" ">Daoud Sie</td>
	<td class=" ">Quantitative DNA sequencing for chromosomal aberrations</td>
	<td class="  sorting_1">327</td>
</tr><tr id="pkg_esATAC" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/esATAC.html">esATAC</a></td>
	<td class=" ">Zheng Wei</td>
	<td class=" ">An Easy-to-use Systematic pipeline for ATACseq data analysis</td>
	<td class="  sorting_1">331</td>
</tr><tr id="pkg_qpgraph" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/qpgraph.html">qpgraph</a></td>
	<td class=" ">Robert Castelo</td>
	<td class=" ">Estimation of genetic and molecular regulatory networks from high-throughput genomics data</td>
	<td class="  sorting_1">332</td>
</tr><tr id="pkg_crlmm" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/crlmm.html">crlmm</a></td>
	<td class=" ">Benilton S Carvalho, Robert Scharpf, Matt Ritchie</td>
	<td class=" ">Genotype Calling (CRLMM) and Copy Number Analysis tool for Affymetrix SNP 5.0 and 6.0 and Illumina arrays</td>
	<td class="  sorting_1">333</td>
</tr><tr id="pkg_GeneOverlap" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/GeneOverlap.html">GeneOverlap</a></td>
	<td class=" ">Li Shen, Mount Sinai</td>
	<td class=" ">Test and visualize gene overlaps</td>
	<td class="  sorting_1">334</td>
</tr><tr id="pkg_M3Drop" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/M3Drop.html">M3Drop</a></td>
	<td class=" ">Tallulah Andrews</td>
	<td class=" ">Michaelis-Menten Modelling of Dropouts in single-cell RNASeq</td>
	<td class="  sorting_1">334</td>
</tr><tr id="pkg_rTANDEM" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/rTANDEM.html">rTANDEM</a></td>
	<td class=" ">Frederic Fournier</td>
	<td class=" ">Interfaces the tandem protein identification algorithm in R</td>
	<td class="  sorting_1">336</td>
</tr><tr id="pkg_TSCAN" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/TSCAN.html">TSCAN</a></td>
	<td class=" ">Zhicheng Ji</td>
	<td class=" ">TSCAN: Tools for Single-Cell ANalysis</td>
	<td class="  sorting_1">336</td>
</tr><tr id="pkg_BiocNeighbors" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/BiocNeighbors.html">BiocNeighbors</a></td>
	<td class=" ">Aaron Lun</td>
	<td class=" ">Nearest Neighbor Detection for Bioconductor Packages</td>
	<td class="  sorting_1">338</td>
</tr><tr id="pkg_Rbowtie2" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/Rbowtie2.html">Rbowtie2</a></td>
	<td class=" ">Zheng Wei</td>
	<td class=" ">An R Wrapper for Bowtie2 and AdapterRemoval</td>
	<td class="  sorting_1">338</td>
</tr><tr id="pkg_muscle" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/muscle.html">muscle</a></td>
	<td class=" ">Alex T. Kalinka</td>
	<td class=" ">Multiple Sequence Alignment with MUSCLE</td>
	<td class="  sorting_1">340</td>
</tr><tr id="pkg_isobar" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/isobar.html">isobar</a></td>
	<td class=" ">Florian P Breitwieser</td>
	<td class=" ">Analysis and quantitation of isobarically tagged MSMS proteomics data</td>
	<td class="  sorting_1">341</td>
</tr><tr id="pkg_SeqGSEA" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/SeqGSEA.html">SeqGSEA</a></td>
	<td class=" ">Xi Wang</td>
	<td class=" ">Gene Set Enrichment Analysis (GSEA) of RNA-Seq Data: integrating differential expression and splicing</td>
	<td class="  sorting_1">341</td>
</tr><tr id="pkg_MeSHDbi" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/MeSHDbi.html">MeSHDbi</a></td>
	<td class=" ">Koki Tsuyuzaki</td>
	<td class=" ">DBI to construct MeSH-related package from sqlite file</td>
	<td class="  sorting_1">343</td>
</tr><tr id="pkg_BRAIN" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/BRAIN.html">BRAIN</a></td>
	<td class=" ">Piotr Dittwald</td>
	<td class=" ">Baffling Recursive Algorithm for Isotope distributioN calculations</td>
	<td class="  sorting_1">344</td>
</tr><tr id="pkg_aCGH" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/aCGH.html">aCGH</a></td>
	<td class=" ">Peter Dimitrov</td>
	<td class=" ">Classes and functions for Array Comparative Genomic Hybridization data.</td>
	<td class="  sorting_1">345</td>
</tr><tr id="pkg_SIMLR" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/SIMLR.html">SIMLR</a></td>
	<td class=" ">Luca De Sano</td>
	<td class=" ">Single-cell Interpretation via Multi-kernel LeaRning (SIMLR)</td>
	<td class="  sorting_1">346</td>
</tr><tr id="pkg_HTSFilter" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/HTSFilter.html">HTSFilter</a></td>
	<td class=" ">Andrea Rau</td>
	<td class=" ">Filter replicated high-throughput transcriptome sequencing data</td>
	<td class="  sorting_1">347</td>
</tr><tr id="pkg_msmsEDA" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/msmsEDA.html">msmsEDA</a></td>
	<td class=" ">Josep Gregori</td>
	<td class=" ">Exploratory Data Analysis of LC-MS/MS data by spectral counts</td>
	<td class="  sorting_1">348</td>
</tr><tr id="pkg_RDRToolbox" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/RDRToolbox.html">RDRToolbox</a></td>
	<td class=" ">Christoph Bartenhagen</td>
	<td class=" ">A package for nonlinear dimension reduction with Isomap and LLE.</td>
	<td class="  sorting_1">348</td>
</tr><tr id="pkg_easyRNASeq" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/easyRNASeq.html">easyRNASeq</a></td>
	<td class=" ">Nicolas Delhomme</td>
	<td class=" ">Count summarization and normalization for RNA-Seq data</td>
	<td class="  sorting_1">350</td>
</tr><tr id="pkg_fmcsR" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/fmcsR.html">fmcsR</a></td>
	<td class=" ">Thomas Girke</td>
	<td class=" ">Mismatch Tolerant Maximum Common Substructure Searching</td>
	<td class="  sorting_1">351</td>
</tr><tr id="pkg_scone" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/scone.html">scone</a></td>
	<td class=" ">Michael Cole</td>
	<td class=" ">Single Cell Overview of Normalized Expression data</td>
	<td class="  sorting_1">352</td>
</tr><tr id="pkg_EnrichedHeatmap" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/EnrichedHeatmap.html">EnrichedHeatmap</a></td>
	<td class=" ">Zuguang Gu</td>
	<td class=" ">Making Enriched Heatmaps</td>
	<td class="  sorting_1">353</td>
</tr><tr id="pkg_fabia" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/fabia.html">fabia</a></td>
	<td class=" ">Sepp Hochreiter</td>
	<td class=" ">FABIA: Factor Analysis for Bicluster Acquisition</td>
	<td class="  sorting_1">354</td>
</tr><tr id="pkg_GENESIS" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/GENESIS.html">GENESIS</a></td>
	<td class=" ">Stephanie M. Gogarten</td>
	<td class=" ">GENetic EStimation and Inference in Structured samples (GENESIS): Statistical methods for analyzing genetic data from samples with population structure and/or relatedness</td>
	<td class="  sorting_1">354</td>
</tr><tr id="pkg_msmsTests" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/msmsTests.html">msmsTests</a></td>
	<td class=" ">Josep Gregori i Font</td>
	<td class=" ">LC-MS/MS Differential Expression Tests</td>
	<td class="  sorting_1">356</td>
</tr><tr id="pkg_BiSeq" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/BiSeq.html">BiSeq</a></td>
	<td class=" ">Katja Hebestreit</td>
	<td class=" ">Processing and analyzing bisulfite sequencing data</td>
	<td class="  sorting_1">357</td>
</tr><tr id="pkg_DRIMSeq" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/DRIMSeq.html">DRIMSeq</a></td>
	<td class=" ">Malgorzata Nowicka</td>
	<td class=" ">Differential transcript usage and tuQTL analyses with Dirichlet-multinomial model in RNA-seq</td>
	<td class="  sorting_1">357</td>
</tr><tr id="pkg_motifmatchr" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/motifmatchr.html">motifmatchr</a></td>
	<td class=" ">Alicia Schep</td>
	<td class=" ">Fast Motif Matching in R</td>
	<td class="  sorting_1">357</td>
</tr><tr id="pkg_ldblock" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/ldblock.html">ldblock</a></td>
	<td class=" ">VJ Carey</td>
	<td class=" ">data structures for linkage disequilibrium measures in populations</td>
	<td class="  sorting_1">360</td>
</tr><tr id="pkg_BitSeq" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/BitSeq.html">BitSeq</a></td>
	<td class=" ">Antti Honkela, Panagiotis Papastamoulis</td>
	<td class=" ">Transcript expression inference and differential expression analysis for RNA-seq data</td>
	<td class="  sorting_1">361</td>
</tr><tr id="pkg_microRNA" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/microRNA.html">microRNA</a></td>
	<td class=" ">"James F. Reid"</td>
	<td class=" ">Data and functions for dealing with microRNAs</td>
	<td class="  sorting_1">361</td>
</tr><tr id="pkg_RedeR" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/RedeR.html">RedeR</a></td>
	<td class=" ">Mauro Castro</td>
	<td class=" ">Interactive visualization and manipulation of nested networks</td>
	<td class="  sorting_1">361</td>
</tr><tr id="pkg_agilp" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/agilp.html">agilp</a></td>
	<td class=" ">Benny Chain</td>
	<td class=" ">Agilent expression array processing package</td>
	<td class="  sorting_1">364</td>
</tr><tr id="pkg_HMMcopy" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/HMMcopy.html">HMMcopy</a></td>
	<td class=" ">Daniel Lai, Sohrab Shah</td>
	<td class=" ">Copy number prediction with correction for GC and mappability bias for HTS data</td>
	<td class="  sorting_1">365</td>
</tr><tr id="pkg_SCnorm" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/SCnorm.html">SCnorm</a></td>
	<td class=" ">Rhonda Bacher</td>
	<td class=" ">Normalization of single cell RNA-seq data</td>
	<td class="  sorting_1">365</td>
</tr><tr id="pkg_PROcess" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/PROcess.html">PROcess</a></td>
	<td class=" ">Xiaochun Li</td>
	<td class=" ">Ciphergen SELDI-TOF Processing</td>
	<td class="  sorting_1">367</td>
</tr><tr id="pkg_RNASeqPower" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/RNASeqPower.html">RNASeqPower</a></td>
	<td class=" ">Terry M Therneau</td>
	<td class=" ">Sample size for RNAseq studies</td>
	<td class="  sorting_1">367</td>
</tr><tr id="pkg_synapter" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/synapter.html">synapter</a></td>
	<td class=" ">Laurent Gatto and Sebastian Gibb</td>
	<td class=" ">Label-free data analysis pipeline for optimal identification and quantitation</td>
	<td class="  sorting_1">367</td>
</tr><tr id="pkg_affylmGUI" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/affylmGUI.html">affylmGUI</a></td>
	<td class=" ">Gordon Smyth</td>
	<td class=" ">GUI for limma Package with Affymetrix Microarrays</td>
	<td class="  sorting_1">370</td>
</tr><tr id="pkg_EBarrays" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/EBarrays.html">EBarrays</a></td>
	<td class=" ">Ming Yuan</td>
	<td class=" ">Unified Approach for Simultaneous Gene Clustering and Differential Expression Identification</td>
	<td class="  sorting_1">370</td>
</tr><tr id="pkg_regionReport" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/regionReport.html">regionReport</a></td>
	<td class=" ">Leonardo Collado-Torres</td>
	<td class=" ">Generate HTML or PDF reports for a set of genomic regions or DESeq2/edgeR results</td>
	<td class="  sorting_1">370</td>
</tr><tr id="pkg_a4Core" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/a4Core.html">a4Core</a></td>
	<td class=" ">Tobias Verbeke, Willem Ligtenberg</td>
	<td class=" ">Automated Affymetrix Array Analysis Core Package</td>
	<td class="  sorting_1">373</td>
</tr><tr id="pkg_affycomp" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/affycomp.html">affycomp</a></td>
	<td class=" ">Rafael A. Irizarry</td>
	<td class=" ">Graphics Toolbox for Assessment of Affymetrix Expression Measures</td>
	<td class="  sorting_1">373</td>
</tr><tr id="pkg_gpls" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/gpls.html">gpls</a></td>
	<td class=" ">Bioconductor Package Maintainer</td>
	<td class=" ">Classification using generalized partial least squares</td>
	<td class="  sorting_1">373</td>
</tr><tr id="pkg_heatmaps" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/heatmaps.html">heatmaps</a></td>
	<td class=" ">Malcolm Perry</td>
	<td class=" ">Flexible Heatmaps for Functional Genomics and Sequence Features</td>
	<td class="  sorting_1">373</td>
</tr><tr id="pkg_pdInfoBuilder" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/pdInfoBuilder.html">pdInfoBuilder</a></td>
	<td class=" ">Benilton Carvalho</td>
	<td class=" ">Platform Design Information Package Builder</td>
	<td class="  sorting_1">373</td>
</tr><tr id="pkg_qrqc" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/qrqc.html">qrqc</a></td>
	<td class=" ">Vince Buffalo</td>
	<td class=" ">Quick Read Quality Control</td>
	<td class="  sorting_1">373</td>
</tr><tr id="pkg_ROTS" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/ROTS.html">ROTS</a></td>
	<td class=" ">Fatemeh Seyednasrollah</td>
	<td class=" ">Reproducibility-Optimized Test Statistic</td>
	<td class="  sorting_1">373</td>
</tr><tr id="pkg_shinyMethyl" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/shinyMethyl.html">shinyMethyl</a></td>
	<td class=" ">Jean-Philippe Fortin</td>
	<td class=" ">Interactive visualization for Illumina methylation arrays</td>
	<td class="  sorting_1">373</td>
</tr><tr id="pkg_polyester" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/polyester.html">polyester</a></td>
	<td class=" ">Jack Fu, Jeff Leek</td>
	<td class=" ">Simulate RNA-seq reads</td>
	<td class="  sorting_1">381</td>
</tr><tr id="pkg_snm" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/snm.html">snm</a></td>
	<td class=" ">John D. Storey</td>
	<td class=" ">Supervised Normalization of Microarrays</td>
	<td class="  sorting_1">381</td>
</tr><tr id="pkg_iClusterPlus" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/iClusterPlus.html">iClusterPlus</a></td>
	<td class=" ">Qianxing Mo, Ronglai Shen</td>
	<td class=" ">Integrative clustering of multi-type genomic data</td>
	<td class="  sorting_1">383</td>
</tr><tr id="pkg_a4Preproc" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/a4Preproc.html">a4Preproc</a></td>
	<td class=" ">Tobias Verbeke, Willem Ligtenberg</td>
	<td class=" ">Automated Affymetrix Array Analysis Preprocessing Package</td>
	<td class="  sorting_1">384</td>
</tr><tr id="pkg_methyAnalysis" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/methyAnalysis.html">methyAnalysis</a></td>
	<td class=" ">Pan Du, Lei Huang, Gang Feng</td>
	<td class=" ">DNA methylation data analysis and visualization</td>
	<td class="  sorting_1">384</td>
</tr><tr id="pkg_pvca" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/pvca.html">pvca</a></td>
	<td class=" ">Jianying LI</td>
	<td class=" ">Principal Variance Component Analysis (PVCA)</td>
	<td class="  sorting_1">384</td>
</tr><tr id="pkg_RcisTarget" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/RcisTarget.html">RcisTarget</a></td>
	<td class=" ">Sara Aibar</td>
	<td class=" ">RcisTarget: Identify transcription factor binding motifs enriched on a gene list</td>
	<td class="  sorting_1">387</td>
</tr><tr id="pkg_BHC" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/BHC.html">BHC</a></td>
	<td class=" ">Rich Savage</td>
	<td class=" ">Bayesian Hierarchical Clustering</td>
	<td class="  sorting_1">388</td>
</tr><tr id="pkg_flowAI" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/flowAI.html">flowAI</a></td>
	<td class=" ">Gianni Monaco</td>
	<td class=" ">Automatic and interactive quality control for flow cytometry data</td>
	<td class="  sorting_1">388</td>
</tr><tr id="pkg_SAGx" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/SAGx.html">SAGx</a></td>
	<td class=" ">Per Broberg,</td>
	<td class=" ">Statistical Analysis of the GeneChip</td>
	<td class="  sorting_1">388</td>
</tr><tr id="pkg_HTSanalyzeR" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/HTSanalyzeR.html">HTSanalyzeR</a></td>
	<td class=" ">Xin Wang</td>
	<td class=" ">Gene set over-representation, enrichment and network analyses for high-throughput screens</td>
	<td class="  sorting_1">391</td>
</tr><tr id="pkg_rBiopaxParser" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/rBiopaxParser.html">rBiopaxParser</a></td>
	<td class=" ">Frank Kramer</td>
	<td class=" ">Parses BioPax files and represents them in R</td>
	<td class="  sorting_1">391</td>
</tr><tr id="pkg_DEP" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/DEP.html">DEP</a></td>
	<td class=" ">Arne Smits</td>
	<td class=" ">Differential Enrichment analysis of Proteomics data</td>
	<td class="  sorting_1">393</td>
</tr><tr id="pkg_goProfiles" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/goProfiles.html">goProfiles</a></td>
	<td class=" ">Alex Sanchez</td>
	<td class=" ">goProfiles: an R package for the statistical analysis of functional profiles</td>
	<td class="  sorting_1">393</td>
</tr><tr id="pkg_htSeqTools" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/htSeqTools.html">htSeqTools</a></td>
	<td class=" ">Oscar Reina</td>
	<td class=" ">Quality Control, Visualization and Processing for High-Throughput Sequencing data</td>
	<td class="  sorting_1">393</td>
</tr><tr id="pkg_seqplots" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/seqplots.html">seqplots</a></td>
	<td class=" ">Przemyslaw Stempor</td>
	<td class=" ">An interactive tool for visualizing NGS signals and sequence motif densities along genomic features using average plots and heatmaps</td>
	<td class="  sorting_1">393</td>
</tr><tr id="pkg_SGSeq" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/SGSeq.html">SGSeq</a></td>
	<td class=" ">Leonard Goldstein</td>
	<td class=" ">Splice event prediction and quantification from RNA-seq data</td>
	<td class="  sorting_1">393</td>
</tr><tr id="pkg_variancePartition" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/variancePartition.html">variancePartition</a></td>
	<td class=" ">Gabriel E. Hoffman</td>
	<td class=" ">Quantify and interpret divers of variation in multilevel gene expression experiments</td>
	<td class="  sorting_1">393</td>
</tr><tr id="pkg_GOexpress" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/GOexpress.html">GOexpress</a></td>
	<td class=" ">Kevin Rue-Albrecht</td>
	<td class=" ">Visualise microarray and RNAseq data using gene ontology annotations</td>
	<td class="  sorting_1">399</td>
</tr><tr id="pkg_IPPD" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/IPPD.html">IPPD</a></td>
	<td class=" ">Martin Slawski</td>
	<td class=" ">Isotopic peak pattern deconvolution for Protein Mass Spectrometry by template matching</td>
	<td class="  sorting_1">399</td>
</tr><tr id="pkg_scmap" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/scmap.html">scmap</a></td>
	<td class=" ">Vladimir Kiselev</td>
	<td class=" ">A tool for unsupervised projection of single cell RNA-seq data</td>
	<td class="  sorting_1">399</td>
</tr><tr id="pkg_sigPathway" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/sigPathway.html">sigPathway</a></td>
	<td class=" ">Weil Lai</td>
	<td class=" ">Pathway Analysis</td>
	<td class="  sorting_1">399</td>
</tr><tr id="pkg_flowDensity" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/flowDensity.html">flowDensity</a></td>
	<td class=" ">Mehrnoush Malek</td>
	<td class=" ">Sequential Flow Cytometry Data Gating</td>
	<td class="  sorting_1">403</td>
</tr><tr id="pkg_GSAR" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/GSAR.html">GSAR</a></td>
	<td class=" ">Yasir Rahmatallah, Galina Glazko</td>
	<td class=" ">Gene Set Analysis in R</td>
	<td class="  sorting_1">403</td>
</tr><tr id="pkg_IPO" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/IPO.html">IPO</a></td>
	<td class=" ">Thomas Riebenbauer</td>
	<td class=" ">Automated Optimization of XCMS Data Processing parameters</td>
	<td class="  sorting_1">403</td>
</tr><tr id="pkg_genbankr" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/genbankr.html">genbankr</a></td>
	<td class=" ">Gabriel Becker</td>
	<td class=" ">Parsing GenBank files into semantically useful objects</td>
	<td class="  sorting_1">406</td>
</tr><tr id="pkg_kebabs" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/kebabs.html">kebabs</a></td>
	<td class=" ">Ulrich Bodenhofer</td>
	<td class=" ">Kernel-Based Analysis Of Biological Sequences</td>
	<td class="  sorting_1">406</td>
</tr><tr id="pkg_CEMiTool" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/CEMiTool.html">CEMiTool</a></td>
	<td class=" ">Helder Nakaya</td>
	<td class=" ">Co-expression Modules identification Tool</td>
	<td class="  sorting_1">408</td>
</tr><tr id="pkg_CMA" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/CMA.html">CMA</a></td>
	<td class=" ">Christoph Bernau</td>
	<td class=" ">Synthesis of microarray-based classification</td>
	<td class="  sorting_1">408</td>
</tr><tr id="pkg_ddCt" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/ddCt.html">ddCt</a></td>
	<td class=" ">Jitao David Zhang</td>
	<td class=" ">The ddCt Algorithm for the Analysis of Quantitative Real-Time PCR (qRT-PCR)</td>
	<td class="  sorting_1">408</td>
</tr><tr id="pkg_GenomicInteractions" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/GenomicInteractions.html">GenomicInteractions</a></td>
	<td class=" ">Liz Ing-Simmons</td>
	<td class=" ">R package for handling genomic interaction data</td>
	<td class="  sorting_1">408</td>
</tr><tr id="pkg_MSGFgui" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/MSGFgui.html">MSGFgui</a></td>
	<td class=" ">Thomas Lin Pedersen</td>
	<td class=" ">A shiny GUI for MSGFplus</td>
	<td class="  sorting_1">408</td>
</tr><tr id="pkg_PharmacoGx" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/PharmacoGx.html">PharmacoGx</a></td>
	<td class=" ">Benjamin Haibe-Kains</td>
	<td class=" ">Analysis of Large-Scale Pharmacogenomic Data</td>
	<td class="  sorting_1">408</td>
</tr><tr id="pkg_GeneAnswers" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/GeneAnswers.html">GeneAnswers</a></td>
	<td class=" ">Lei Huang and Gang Feng</td>
	<td class=" ">Integrated Interpretation of Genes</td>
	<td class="  sorting_1">414</td>
</tr><tr id="pkg_FGNet" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/FGNet.html">FGNet</a></td>
	<td class=" ">Sara Aibar</td>
	<td class=" ">Functional Gene Networks derived from biological enrichment analyses</td>
	<td class="  sorting_1">415</td>
</tr><tr id="pkg_ArrayTools" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/ArrayTools.html">ArrayTools</a></td>
	<td class=" ">Arthur Li</td>
	<td class=" ">geneChip Analysis Package</td>
	<td class="  sorting_1">416</td>
</tr><tr id="pkg_GOsummaries" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/GOsummaries.html">GOsummaries</a></td>
	<td class=" ">Raivo Kolde</td>
	<td class=" ">Word cloud summaries of GO enrichment analysis</td>
	<td class="  sorting_1">416</td>
</tr><tr id="pkg_hypergraph" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/hypergraph.html">hypergraph</a></td>
	<td class=" ">Bioconductor Package Maintainer</td>
	<td class=" ">A package providing hypergraph data structures</td>
	<td class="  sorting_1">416</td>
</tr><tr id="pkg_annotationTools" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/annotationTools.html">annotationTools</a></td>
	<td class=" ">Alexandre Kuhn</td>
	<td class=" ">Annotate microarrays and perform cross-species gene expression analyses using flat file databases.</td>
	<td class="  sorting_1">419</td>
</tr><tr id="pkg_BatchQC" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/BatchQC.html">BatchQC</a></td>
	<td class=" ">Solaiappan Manimaran</td>
	<td class=" ">Batch Effects Quality Control Software</td>
	<td class="  sorting_1">419</td>
</tr><tr id="pkg_EasyqpcR" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/EasyqpcR.html">EasyqpcR</a></td>
	<td class=" ">Le Pape Sylvain</td>
	<td class=" ">EasyqpcR for low-throughput real-time quantitative PCR data analysis</td>
	<td class="  sorting_1">419</td>
</tr><tr id="pkg_ensemblVEP" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/ensemblVEP.html">ensemblVEP</a></td>
	<td class=" ">Bioconductor Package Maintainer</td>
	<td class=" ">R Interface to Ensembl Variant Effect Predictor</td>
	<td class="  sorting_1">419</td>
</tr><tr id="pkg_GeneticsPed" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/GeneticsPed.html">GeneticsPed</a></td>
	<td class=" ">David Henderson</td>
	<td class=" ">Pedigree and genetic relationship functions</td>
	<td class="  sorting_1">419</td>
</tr><tr id="pkg_GSEAlm" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/GSEAlm.html">GSEAlm</a></td>
	<td class=" ">Assaf Oron</td>
	<td class=" ">Linear Model Toolset for Gene Set Enrichment Analysis</td>
	<td class="  sorting_1">419</td>
</tr><tr id="pkg_PureCN" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/PureCN.html">PureCN</a></td>
	<td class=" ">Markus Riester</td>
	<td class=" ">Copy number calling and SNV classification using targeted short read sequencing</td>
	<td class="  sorting_1">419</td>
</tr><tr id="pkg_sSeq" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/sSeq.html">sSeq</a></td>
	<td class=" ">Danni Yu</td>
	<td class=" ">Shrinkage estimation of dispersion in Negative Binomial models for RNA-seq experiments with small sample size</td>
	<td class="  sorting_1">419</td>
</tr><tr id="pkg_ChIPseqR" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/ChIPseqR.html">ChIPseqR</a></td>
	<td class=" ">Peter Humburg</td>
	<td class=" ">Identifying Protein Binding Sites in High-Throughput Sequencing Data</td>
	<td class="  sorting_1">427</td>
</tr><tr id="pkg_IsoformSwitchAnalyzeR" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/IsoformSwitchAnalyzeR.html">IsoformSwitchAnalyzeR</a></td>
	<td class=" ">Kristoffer Vitting-Seerup</td>
	<td class=" ">An R package to Identify, Annotate and Visualize Alternative Splicing and Isoform Switches with Functional Consequences (from RNA-seq data)</td>
	<td class="  sorting_1">427</td>
</tr><tr id="pkg_LOLA" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/LOLA.html">LOLA</a></td>
	<td class=" ">Nathan Sheffield</td>
	<td class=" ">Locus overlap analysis for enrichment of genomic ranges</td>
	<td class="  sorting_1">427</td>
</tr><tr id="pkg_OCplus" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/OCplus.html">OCplus</a></td>
	<td class=" ">Alexander Ploner</td>
	<td class=" ">Operating characteristics plus sample size and local fdr for microarray experiments</td>
	<td class="  sorting_1">427</td>
</tr><tr id="pkg_puma" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/puma.html">puma</a></td>
	<td class=" ">Xuejun Liu</td>
	<td class=" ">Propagating Uncertainty in Microarray Analysis(including Affymetrix tranditional 3' arrays and exon arrays and Human Transcriptome Array 2.0)</td>
	<td class="  sorting_1">427</td>
</tr><tr id="pkg_PWMEnrich" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/PWMEnrich.html">PWMEnrich</a></td>
	<td class=" ">Robert Stojnic</td>
	<td class=" ">PWM enrichment analysis</td>
	<td class="  sorting_1">427</td>
</tr><tr id="pkg_CancerSubtypes" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/CancerSubtypes.html">CancerSubtypes</a></td>
	<td class=" ">Taosheng Xu</td>
	<td class=" ">Cancer subtypes identification, validation and visualization based on multiple genomic data sets</td>
	<td class="  sorting_1">433</td>
</tr><tr id="pkg_CSAR" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/CSAR.html">CSAR</a></td>
	<td class=" ">Jose M Muino</td>
	<td class=" ">Statistical tools for the analysis of ChIP-seq data</td>
	<td class="  sorting_1">433</td>
</tr><tr id="pkg_edge" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/edge.html">edge</a></td>
	<td class=" ">John D. Storey, Andrew J. Bass</td>
	<td class=" ">Extraction of Differential Gene Expression</td>
	<td class="  sorting_1">433</td>
</tr><tr id="pkg_GDCRNATools" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/GDCRNATools.html">GDCRNATools</a></td>
	<td class=" ">Ruidong Li, Han Qu</td>
	<td class=" ">GDCRNATools: an R/Bioconductor package for integrative analysis of lncRNA, mRNA, and miRNA data in GDC</td>
	<td class="  sorting_1">433</td>
</tr><tr id="pkg_MEDIPS" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/MEDIPS.html">MEDIPS</a></td>
	<td class=" ">Lukas Chavez</td>
	<td class=" ">DNA IP-seq data analysis</td>
	<td class="  sorting_1">433</td>
</tr><tr id="pkg_nem" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/nem.html">nem</a></td>
	<td class=" ">Holger Froehlich</td>
	<td class=" ">(Dynamic) Nested Effects Models and Deterministic Effects Propagation Networks to reconstruct phenotypic hierarchies</td>
	<td class="  sorting_1">433</td>
</tr><tr id="pkg_CODEX" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/CODEX.html">CODEX</a></td>
	<td class=" ">Yuchao Jiang</td>
	<td class=" ">A Normalization and Copy Number Variation Detection Method for Whole Exome Sequencing</td>
	<td class="  sorting_1">439</td>
</tr><tr id="pkg_girafe" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/girafe.html">girafe</a></td>
	<td class=" ">J. Toedling</td>
	<td class=" ">Genome Intervals and Read Alignments for Functional Exploration</td>
	<td class="  sorting_1">439</td>
</tr><tr id="pkg_mosaics" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/mosaics.html">mosaics</a></td>
	<td class=" ">Dongjun Chung</td>
	<td class=" ">MOSAiCS (MOdel-based one and two Sample Analysis and Inference for ChIP-Seq)</td>
	<td class="  sorting_1">439</td>
</tr><tr id="pkg_PICS" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/PICS.html">PICS</a></td>
	<td class=" ">Renan Sauteraud</td>
	<td class=" ">Probabilistic inference of ChIP-seq</td>
	<td class="  sorting_1">439</td>
</tr><tr id="pkg_RIPSeeker" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/RIPSeeker.html">RIPSeeker</a></td>
	<td class=" ">Yue Li</td>
	<td class=" ">RIPSeeker: a statistical package for identifying protein-associated transcripts from RIP-seq experiments</td>
	<td class="  sorting_1">439</td>
</tr><tr id="pkg_GGBase" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/GGBase.html">GGBase</a></td>
	<td class=" ">VJ Carey</td>
	<td class=" ">GGBase infrastructure for genetics of gene expression package GGtools</td>
	<td class="  sorting_1">444</td>
</tr><tr id="pkg_parody" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/parody.html">parody</a></td>
	<td class=" ">VJ Carey</td>
	<td class=" ">Parametric And Resistant Outlier DYtection</td>
	<td class="  sorting_1">445</td>
</tr><tr id="pkg_BicARE" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/BicARE.html">BicARE</a></td>
	<td class=" ">Pierre Gestraud</td>
	<td class=" ">Biclustering Analysis and Results Exploration</td>
	<td class="  sorting_1">446</td>
</tr><tr id="pkg_clusterStab" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/clusterStab.html">clusterStab</a></td>
	<td class=" ">James W. MacDonald</td>
	<td class=" ">Compute cluster stability scores for microarray data</td>
	<td class="  sorting_1">446</td>
</tr><tr id="pkg_debrowser" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/debrowser.html">debrowser</a></td>
	<td class=" ">Alper Kucukural</td>
	<td class=" ">Interactive Differential Expresion Analysis Browser</td>
	<td class="  sorting_1">446</td>
</tr><tr id="pkg_flowPeaks" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/flowPeaks.html">flowPeaks</a></td>
	<td class=" ">Yongchao Ge</td>
	<td class=" ">An R package for flow data clustering</td>
	<td class="  sorting_1">446</td>
</tr><tr id="pkg_rGREAT" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/rGREAT.html">rGREAT</a></td>
	<td class=" ">Zuguang Gu</td>
	<td class=" ">Client for GREAT Analysis</td>
	<td class="  sorting_1">446</td>
</tr><tr id="pkg_a4Base" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/a4Base.html">a4Base</a></td>
	<td class=" ">Tobias Verbeke, Willem Ligtenberg</td>
	<td class=" ">Automated Affymetrix Array Analysis Base Package</td>
	<td class="  sorting_1">451</td>
</tr><tr id="pkg_BayesPeak" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/BayesPeak.html">BayesPeak</a></td>
	<td class=" ">Jonathan Cairns</td>
	<td class=" ">Bayesian Analysis of ChIP-seq Data</td>
	<td class="  sorting_1">451</td>
</tr><tr id="pkg_ChIPsim" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/ChIPsim.html">ChIPsim</a></td>
	<td class=" ">Peter Humburg</td>
	<td class=" ">Simulation of ChIP-seq experiments</td>
	<td class="  sorting_1">451</td>
</tr><tr id="pkg_flowCL" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/flowCL.html">flowCL</a></td>
	<td class=" ">Justin Meskas</td>
	<td class=" ">Semantic labelling of flow cytometric cell populations</td>
	<td class="  sorting_1">451</td>
</tr><tr id="pkg_GOSim" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/GOSim.html">GOSim</a></td>
	<td class=" ">Holger Froehlich</td>
	<td class=" ">Computation of functional similarities between GO terms and gene products; GO enrichment analysis</td>
	<td class="  sorting_1">455</td>
</tr><tr id="pkg_gtrellis" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/gtrellis.html">gtrellis</a></td>
	<td class=" ">Zuguang Gu</td>
	<td class=" ">Genome Level Trellis Layout</td>
	<td class="  sorting_1">455</td>
</tr><tr id="pkg_JunctionSeq" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/JunctionSeq.html">JunctionSeq</a></td>
	<td class=" ">Stephen Hartley</td>
	<td class=" ">JunctionSeq: A Utility for Detection of Differential Exon and Splice-Junction Usage in RNA-Seq data</td>
	<td class="  sorting_1">455</td>
</tr><tr id="pkg_multiClust" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/multiClust.html">multiClust</a></td>
	<td class=" ">Nathan Lawlor</td>
	<td class=" ">multiClust: An R-package for Identifying Biologically Relevant Clusters in Cancer Transcriptome Profiles</td>
	<td class="  sorting_1">455</td>
</tr><tr id="pkg_DeconRNASeq" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/DeconRNASeq.html">DeconRNASeq</a></td>
	<td class=" ">Ting Gong</td>
	<td class=" ">Deconvolution of Heterogeneous Tissue Samples for mRNA-Seq data</td>
	<td class="  sorting_1">459</td>
</tr><tr id="pkg_paxtoolsr" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/paxtoolsr.html">paxtoolsr</a></td>
	<td class=" ">Augustin Luna</td>
	<td class=" ">PaxtoolsR: Access Pathways from Multiple Databases through BioPAX and Pathway Commons</td>
	<td class="  sorting_1">459</td>
</tr><tr id="pkg_pcaGoPromoter" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/pcaGoPromoter.html">pcaGoPromoter</a></td>
	<td class=" ">Morten Hansen</td>
	<td class=" ">pcaGoPromoter is used to analyze DNA micro array data</td>
	<td class="  sorting_1">459</td>
</tr><tr id="pkg_r3Cseq" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/r3Cseq.html">r3Cseq</a></td>
	<td class=" ">Supat Thongjuea</td>
	<td class=" ">Analysis of Chromosome Conformation Capture and Next-generation Sequencing (3C-seq)</td>
	<td class="  sorting_1">459</td>
</tr><tr id="pkg_rsbml" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/rsbml.html">rsbml</a></td>
	<td class=" ">Michael Lawrence</td>
	<td class=" ">R support for SBML, using libsbml</td>
	<td class="  sorting_1">459</td>
</tr><tr id="pkg_AgiMicroRna" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/AgiMicroRna.html">AgiMicroRna</a></td>
	<td class=" ">Pedro Lopez-Romero</td>
	<td class=" ">Processing and Differential Expression Analysis of Agilent microRNA chips</td>
	<td class="  sorting_1">464</td>
</tr><tr id="pkg_Cardinal" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/Cardinal.html">Cardinal</a></td>
	<td class=" ">Kylie A. Bemis</td>
	<td class=" ">A mass spectrometry imaging toolbox for statistical analysis</td>
	<td class="  sorting_1">464</td>
</tr><tr id="pkg_clipper" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/clipper.html">clipper</a></td>
	<td class=" ">Paolo Martini</td>
	<td class=" ">Gene Set Analysis Exploiting Pathway Topology</td>
	<td class="  sorting_1">464</td>
</tr><tr id="pkg_flowClean" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/flowClean.html">flowClean</a></td>
	<td class=" ">Kipper Fletez-Brant</td>
	<td class=" ">flowClean</td>
	<td class="  sorting_1">464</td>
</tr><tr id="pkg_miRNAtap" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/miRNAtap.html">miRNAtap</a></td>
	<td class=" ">Maciej Pajak</td>
	<td class=" ">miRNAtap: microRNA Targets - Aggregated Predictions</td>
	<td class="  sorting_1">464</td>
</tr><tr id="pkg_mixOmics" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/mixOmics.html">mixOmics</a></td>
	<td class=" ">Kim-Anh Le Cao</td>
	<td class=" ">Omics Data Integration Project</td>
	<td class="  sorting_1">464</td>
</tr><tr id="pkg_MLSeq" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/MLSeq.html">MLSeq</a></td>
	<td class=" ">Gokmen Zararsiz</td>
	<td class=" ">Machine Learning Interface for RNA-Seq Data</td>
	<td class="  sorting_1">464</td>
</tr><tr id="pkg_CellNOptR" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/CellNOptR.html">CellNOptR</a></td>
	<td class=" ">A.Gabor</td>
	<td class=" ">Training of boolean logic models of signalling networks using prior knowledge networks and perturbation data</td>
	<td class="  sorting_1">471</td>
</tr><tr id="pkg_AnnotationFuncs" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/AnnotationFuncs.html">AnnotationFuncs</a></td>
	<td class=" ">Stefan McKinnon Edwards</td>
	<td class=" ">Annotation translation functions</td>
	<td class="  sorting_1">472</td>
</tr><tr id="pkg_CRISPRseek" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/CRISPRseek.html">CRISPRseek</a></td>
	<td class=" ">Lihua Julie Zhu</td>
	<td class=" ">Design of target-specific guide RNAs in CRISPR-Cas9, genome-editing systems</td>
	<td class="  sorting_1">472</td>
</tr><tr id="pkg_VariantTools" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/VariantTools.html">VariantTools</a></td>
	<td class=" ">Michael Lawrence</td>
	<td class=" ">Tools for Exploratory Analysis of Variant Calls</td>
	<td class="  sorting_1">472</td>
</tr><tr id="pkg_BASiCS" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/BASiCS.html">BASiCS</a></td>
	<td class=" ">Catalina A. Vallejos, Nils Eling</td>
	<td class=" ">Bayesian Analysis of Single-Cell Sequencing data</td>
	<td class="  sorting_1">475</td>
</tr><tr id="pkg_GGtools" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/GGtools.html">GGtools</a></td>
	<td class=" ">VJ Carey</td>
	<td class=" ">software and data for analyses in genetics of gene expression</td>
	<td class="  sorting_1">475</td>
</tr><tr id="pkg_iBBiG" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/iBBiG.html">iBBiG</a></td>
	<td class=" ">Aedin Culhane</td>
	<td class=" ">Iterative Binary Biclustering of Genesets</td>
	<td class="  sorting_1">475</td>
</tr><tr id="pkg_limmaGUI" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/limmaGUI.html">limmaGUI</a></td>
	<td class=" ">Gordon Smyth</td>
	<td class=" ">GUI for limma Package With Two Color Microarrays</td>
	<td class="  sorting_1">475</td>
</tr><tr id="pkg_Linnorm" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/Linnorm.html">Linnorm</a></td>
	<td class=" ">Ken Shun Hang Yip</td>
	<td class=" ">Linear model and normality based transformation method (Linnorm)</td>
	<td class="  sorting_1">475</td>
</tr><tr id="pkg_MethylMix" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/MethylMix.html">MethylMix</a></td>
	<td class=" ">Olivier Gevaert</td>
	<td class=" ">MethylMix: Identifying methylation driven cancer genes</td>
	<td class="  sorting_1">475</td>
</tr><tr id="pkg_scDD" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/scDD.html">scDD</a></td>
	<td class=" ">Keegan Korthauer</td>
	<td class=" ">Mixture modeling of single-cell RNA-seq data to identify genes with differential distributions</td>
	<td class="  sorting_1">475</td>
</tr><tr id="pkg_SplicingGraphs" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/SplicingGraphs.html">SplicingGraphs</a></td>
	<td class=" ">H. Pags</td>
	<td class=" ">Create, manipulate, visualize splicing graphs, and assign RNA-seq reads to them</td>
	<td class="  sorting_1">475</td>
</tr><tr id="pkg_arrayQuality" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/arrayQuality.html">arrayQuality</a></td>
	<td class=" ">Agnes Paquet</td>
	<td class=" ">Assessing array quality on spotted arrays</td>
	<td class="  sorting_1">483</td>
</tr><tr id="pkg_AnnotationHubData" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/AnnotationHubData.html">AnnotationHubData</a></td>
	<td class=" ">Bioconductor Package Maintainer</td>
	<td class=" ">Transform public data resources into Bioconductor Data Structures</td>
	<td class="  sorting_1">484</td>
</tr><tr id="pkg_CAGEr" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/CAGEr.html">CAGEr</a></td>
	<td class=" ">Vanja Haberle, Charles Plessy</td>
	<td class=" ">Analysis of CAGE (Cap Analysis of Gene Expression) sequencing data for precise mapping of transcription start sites and promoterome mining</td>
	<td class="  sorting_1">484</td>
</tr><tr id="pkg_Mulcom" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/Mulcom.html">Mulcom</a></td>
	<td class=" ">Claudio Isella</td>
	<td class=" ">Calculates Mulcom test</td>
	<td class="  sorting_1">484</td>
</tr><tr id="pkg_Rcpi" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/Rcpi.html">Rcpi</a></td>
	<td class=" ">Nan Xiao</td>
	<td class=" ">Molecular Informatics Toolkit for Compound-Protein Interaction in Drug Discovery</td>
	<td class="  sorting_1">484</td>
</tr><tr id="pkg_RTN" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/RTN.html">RTN</a></td>
	<td class=" ">Mauro Castro</td>
	<td class=" ">Reconstruction of transcriptional networks and analysis of master regulators</td>
	<td class="  sorting_1">484</td>
</tr><tr id="pkg_customProDB" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/customProDB.html">customProDB</a></td>
	<td class=" ">Xiaojing Wang Bo Wen</td>
	<td class=" ">Generate customized protein database from NGS data, with a focus on RNA-Seq data, for proteomics search</td>
	<td class="  sorting_1">489</td>
</tr><tr id="pkg_DAPAR" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/DAPAR.html">DAPAR</a></td>
	<td class=" ">Samuel Wieczorek</td>
	<td class=" ">Tools for the Differential Analysis of Proteins Abundance with R</td>
	<td class="  sorting_1">489</td>
</tr><tr id="pkg_IdeoViz" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/IdeoViz.html">IdeoViz</a></td>
	<td class=" ">Shraddha Pai</td>
	<td class=" ">Plots data (continuous/discrete) along chromosomal ideogram</td>
	<td class="  sorting_1">489</td>
</tr><tr id="pkg_derfinderPlot" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/derfinderPlot.html">derfinderPlot</a></td>
	<td class=" ">Leonardo Collado-Torres</td>
	<td class=" ">Plotting functions for derfinder</td>
	<td class="  sorting_1">492</td>
</tr><tr id="pkg_exomePeak" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/exomePeak.html">exomePeak</a></td>
	<td class=" ">Lin Zhang, Lian Liu, Jia Meng</td>
	<td class=" ">exome-based anlaysis of MeRIP-Seq data: peak calling and differential analysis</td>
	<td class="  sorting_1">492</td>
</tr><tr id="pkg_NanoStringQCPro" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/NanoStringQCPro.html">NanoStringQCPro</a></td>
	<td class=" ">Robert Ziman</td>
	<td class=" ">Quality metrics and data processing methods for NanoString mRNA gene expression data</td>
	<td class="  sorting_1">492</td>
</tr><tr id="pkg_segmentSeq" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/segmentSeq.html">segmentSeq</a></td>
	<td class=" ">Thomas J. Hardcastle</td>
	<td class=" ">Methods for identifying small RNA loci from high-throughput sequencing data</td>
	<td class="  sorting_1">492</td>
</tr><tr id="pkg_synergyfinder" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/synergyfinder.html">synergyfinder</a></td>
	<td class=" ">Liye He</td>
	<td class=" ">Calculate and Visualize Synergy Scores for Drug Combinations</td>
	<td class="  sorting_1">492</td>
</tr><tr id="pkg_BgeeDB" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/BgeeDB.html">BgeeDB</a></td>
	<td class=" ">Julien Wollbrett, Julien Roux, Andrea Komljenovic, Frederic Bastian</td>
	<td class=" ">Annotation and gene expression data retrieval from Bgee database</td>
	<td class="  sorting_1">497</td>
</tr><tr id="pkg_cghMCR" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/cghMCR.html">cghMCR</a></td>
	<td class=" ">J. Zhang</td>
	<td class=" ">Find chromosome regions showing common gains/losses</td>
	<td class="  sorting_1">497</td>
</tr><tr id="pkg_DBChIP" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/DBChIP.html">DBChIP</a></td>
	<td class=" ">Kun Liang</td>
	<td class=" ">Differential Binding of Transcription Factor with ChIP-seq</td>
	<td class="  sorting_1">497</td>
</tr><tr id="pkg_deepSNV" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/deepSNV.html">deepSNV</a></td>
	<td class=" ">Moritz Gerstung</td>
	<td class=" ">Detection of subclonal SNVs in deep sequencing data.</td>
	<td class="  sorting_1">497</td>
</tr><tr id="pkg_HilbertCurve" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/HilbertCurve.html">HilbertCurve</a></td>
	<td class=" ">Zuguang Gu</td>
	<td class=" ">Making 2D Hilbert Curve</td>
	<td class="  sorting_1">497</td>
</tr><tr id="pkg_phenoTest" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/phenoTest.html">phenoTest</a></td>
	<td class=" ">Evarist Planet</td>
	<td class=" ">Tools to test association between gene expression and phenotype in a way that is efficient, structured, fast and scalable. We also provide tools to do GSEA (Gene set enrichment analysis) and copy number variation.</td>
	<td class="  sorting_1">497</td>
</tr><tr id="pkg_Rqc" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/Rqc.html">Rqc</a></td>
	<td class=" ">Welliton Souza</td>
	<td class=" ">Quality Control Tool for High-Throughput Sequencing Data</td>
	<td class="  sorting_1">497</td>
</tr><tr id="pkg_affypdnn" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/affypdnn.html">affypdnn</a></td>
	<td class=" ">Laurent Gautier</td>
	<td class=" ">Probe Dependent Nearest Neighbours (PDNN) for the affy package</td>
	<td class="  sorting_1">504</td>
</tr><tr id="pkg_CATALYST" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/CATALYST.html">CATALYST</a></td>
	<td class=" ">Helena L. Crowell</td>
	<td class=" ">Cytometry dATa anALYSis Tools</td>
	<td class="  sorting_1">504</td>
</tr><tr id="pkg_chimeraviz" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/chimeraviz.html">chimeraviz</a></td>
	<td class=" ">Stian Lgstad</td>
	<td class=" ">Visualization tools for gene fusions</td>
	<td class="  sorting_1">504</td>
</tr><tr id="pkg_DNABarcodes" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/DNABarcodes.html">DNABarcodes</a></td>
	<td class=" ">Tilo Buschmann</td>
	<td class=" ">A tool for creating and analysing DNA barcodes used in Next Generation Sequencing multiplexing experiments</td>
	<td class="  sorting_1">504</td>
</tr><tr id="pkg_flowMerge" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/flowMerge.html">flowMerge</a></td>
	<td class=" ">Greg Finak</td>
	<td class=" ">Cluster Merging for Flow Cytometry Data</td>
	<td class="  sorting_1">504</td>
</tr><tr id="pkg_vbmp" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/vbmp.html">vbmp</a></td>
	<td class=" ">Nicola Lama</td>
	<td class=" ">Variational Bayesian Multinomial Probit Regression</td>
	<td class="  sorting_1">509</td>
</tr><tr id="pkg_conumee" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/conumee.html">conumee</a></td>
	<td class=" ">Volker Hovestadt</td>
	<td class=" ">Enhanced copy-number variation analysis using Illumina DNA methylation arrays</td>
	<td class="  sorting_1">510</td>
</tr><tr id="pkg_CopywriteR" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/CopywriteR.html">CopywriteR</a></td>
	<td class=" ">Oscar Krijgsman</td>
	<td class=" ">Copy number information from targeted sequencing using off-target reads</td>
	<td class="  sorting_1">510</td>
</tr><tr id="pkg_ENmix" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/ENmix.html">ENmix</a></td>
	<td class=" ">Zongli Xu</td>
	<td class=" ">Data preprocessing and quality control for Illumina HumanMethylation450 and MethylationEPIC BeadChip</td>
	<td class="  sorting_1">510</td>
</tr><tr id="pkg_NanoStringDiff" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/NanoStringDiff.html">NanoStringDiff</a></td>
	<td class=" ">tingting zhai,hong wang</td>
	<td class=" ">Differential Expression Analysis of NanoString nCounter Data</td>
	<td class="  sorting_1">510</td>
</tr><tr id="pkg_pcot2" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/pcot2.html">pcot2</a></td>
	<td class=" ">Sarah Song</td>
	<td class=" ">Principal Coordinates and Hotelling's T-Square method</td>
	<td class="  sorting_1">510</td>
</tr><tr id="pkg_REDseq" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/REDseq.html">REDseq</a></td>
	<td class=" ">Lihua Julie Zhu</td>
	<td class=" ">Analysis of high-throughput sequencing data processed by restriction enzyme digestion</td>
	<td class="  sorting_1">510</td>
</tr><tr id="pkg_TCGAutils" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/TCGAutils.html">TCGAutils</a></td>
	<td class=" ">Marcel Ramos</td>
	<td class=" ">TCGA utility functions for data management</td>
	<td class="  sorting_1">510</td>
</tr><tr id="pkg_ACME" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/ACME.html">ACME</a></td>
	<td class=" ">Sean Davis</td>
	<td class=" ">Algorithms for Calculating Microarray Enrichment (ACME)</td>
	<td class="  sorting_1">517</td>
</tr><tr id="pkg_BiRewire" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/BiRewire.html">BiRewire</a></td>
	<td class=" ">Andrea Gobbi</td>
	<td class=" ">High-performing routines for the randomization of a bipartite graph (or a binary event matrix), undirected and directed signed graph preserving degree distribution (or marginal totals)</td>
	<td class="  sorting_1">517</td>
</tr><tr id="pkg_cogena" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/cogena.html">cogena</a></td>
	<td class=" ">Zhilong Jia</td>
	<td class=" ">co-expressed gene-set enrichment analysis</td>
	<td class="  sorting_1">517</td>
</tr><tr id="pkg_LBE" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/LBE.html">LBE</a></td>
	<td class=" ">Cyril Dalmasso</td>
	<td class=" ">Estimation of the false discovery rate.</td>
	<td class="  sorting_1">517</td>
</tr><tr id="pkg_SpeCond" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/SpeCond.html">SpeCond</a></td>
	<td class=" ">Florence Cavalli</td>
	<td class=" ">Condition specific detection from expression data</td>
	<td class="  sorting_1">517</td>
</tr><tr id="pkg_DEDS" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/DEDS.html">DEDS</a></td>
	<td class=" ">Yuanyuan Xiao</td>
	<td class=" ">Differential Expression via Distance Summary for Microarray Data</td>
	<td class="  sorting_1">522</td>
</tr><tr id="pkg_diffloop" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/diffloop.html">diffloop</a></td>
	<td class=" ">Caleb Lareau</td>
	<td class=" ">Identifying differential DNA loops from chromatin topology data</td>
	<td class="  sorting_1">522</td>
</tr><tr id="pkg_Logolas" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/Logolas.html">Logolas</a></td>
	<td class=" ">Kushal Dey</td>
	<td class=" ">EDLogo Plots Featuring String Logos and Adaptive Scaling of Position-Weight Matrices</td>
	<td class="  sorting_1">522</td>
</tr><tr id="pkg_MergeMaid" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/MergeMaid.html">MergeMaid</a></td>
	<td class=" ">Xiaogang Zhong</td>
	<td class=" ">Merge Maid</td>
	<td class="  sorting_1">522</td>
</tr><tr id="pkg_rqubic" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/rqubic.html">rqubic</a></td>
	<td class=" ">Jitao David Zhang</td>
	<td class=" ">Qualitative biclustering algorithm for expression data analysis in R</td>
	<td class="  sorting_1">522</td>
</tr><tr id="pkg_diffHic" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/diffHic.html">diffHic</a></td>
	<td class=" ">Aaron Lun</td>
	<td class=" ">Differential Analyis of Hi-C Data</td>
	<td class="  sorting_1">527</td>
</tr><tr id="pkg_geNetClassifier" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/geNetClassifier.html">geNetClassifier</a></td>
	<td class=" ">Sara Aibar</td>
	<td class=" ">Classify diseases and build associated gene networks using gene expression profiles</td>
	<td class="  sorting_1">527</td>
</tr><tr id="pkg_MAIT" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/MAIT.html">MAIT</a></td>
	<td class=" ">Pol Sola-Santos</td>
	<td class=" ">Statistical Analysis of Metabolomic Data</td>
	<td class="  sorting_1">527</td>
</tr><tr id="pkg_motifRG" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/motifRG.html">motifRG</a></td>
	<td class=" ">Zizhen Yao</td>
	<td class=" ">A package for discriminative motif discovery, designed for high throughput sequencing dataset</td>
	<td class="  sorting_1">527</td>
</tr><tr id="pkg_SamSPECTRAL" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/SamSPECTRAL.html">SamSPECTRAL</a></td>
	<td class=" ">Habil</td>
	<td class=" ">Identifies cell population in flow cytometry data.</td>
	<td class="  sorting_1">527</td>
</tr><tr id="pkg_TitanCNA" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/TitanCNA.html">TitanCNA</a></td>
	<td class=" ">Gavin Ha, Sohrab P Shah</td>
	<td class=" ">Subclonal copy number and LOH prediction from whole genome sequencing of tumours</td>
	<td class="  sorting_1">527</td>
</tr><tr id="pkg_ASpli" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/ASpli.html">ASpli</a></td>
	<td class=" ">Estefania Mancini</td>
	<td class=" ">Analysis of alternative splicing using RNA-Seq</td>
	<td class="  sorting_1">533</td>
</tr><tr id="pkg_chromVAR" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/chromVAR.html">chromVAR</a></td>
	<td class=" ">Alicia Schep</td>
	<td class=" ">Chromatin Variation Across Regions</td>
	<td class="  sorting_1">533</td>
</tr><tr id="pkg_goTools" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/goTools.html">goTools</a></td>
	<td class=" ">Agnes Paquet</td>
	<td class=" ">Functions for Gene Ontology database</td>
	<td class="  sorting_1">533</td>
</tr><tr id="pkg_GRmetrics" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/GRmetrics.html">GRmetrics</a></td>
	<td class=" ">Nicholas Clark, Mario Medvedovic</td>
	<td class=" ">Calculate growth-rate inhibition (GR) metrics</td>
	<td class="  sorting_1">533</td>
</tr><tr id="pkg_matter" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/matter.html">matter</a></td>
	<td class=" ">Kylie A. Bemis</td>
	<td class=" ">A framework for rapid prototyping with binary data on disk</td>
	<td class="  sorting_1">533</td>
</tr><tr id="pkg_seqTools" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/seqTools.html">seqTools</a></td>
	<td class=" ">Wolfgang Kaisers</td>
	<td class=" ">Analysis of nucleotide, sequence and quality content on fastq files</td>
	<td class="  sorting_1">533</td>
</tr><tr id="pkg_tilingArray" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/tilingArray.html">tilingArray</a></td>
	<td class=" ">Zhenyu Xu</td>
	<td class=" ">Transcript mapping with high-density oligonucleotide tiling arrays</td>
	<td class="  sorting_1">533</td>
</tr><tr id="pkg_a4" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/a4.html">a4</a></td>
	<td class=" ">Tobias Verbeke, Willem Ligtenberg</td>
	<td class=" ">Automated Affymetrix Array Analysis Umbrella Package</td>
	<td class="  sorting_1">540</td>
</tr><tr id="pkg_chromPlot" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/chromPlot.html">chromPlot</a></td>
	<td class=" ">Karen Y. Orostica</td>
	<td class=" ">Global visualization tool of genomic data</td>
	<td class="  sorting_1">540</td>
</tr><tr id="pkg_quantro" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/quantro.html">quantro</a></td>
	<td class=" ">Stephanie Hicks</td>
	<td class=" ">A test for when to use quantile normalization</td>
	<td class="  sorting_1">540</td>
</tr><tr id="pkg_statTarget" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/statTarget.html">statTarget</a></td>
	<td class=" ">Hemi Luan</td>
	<td class=" ">Statistical Analysis of Molecular Profiles</td>
	<td class="  sorting_1">540</td>
</tr><tr id="pkg_coMET" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/coMET.html">coMET</a></td>
	<td class=" ">Tiphaine Martin</td>
	<td class=" ">coMET: visualisation of regional epigenome-wide association scan (EWAS) results and DNA co-methylation patterns</td>
	<td class="  sorting_1">544</td>
</tr><tr id="pkg_CRImage" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/CRImage.html">CRImage</a></td>
	<td class=" ">Henrik Failmezger, Yinyin Yuan</td>
	<td class=" ">CRImage a package to classify cells and calculate tumour cellularity</td>
	<td class="  sorting_1">544</td>
</tr><tr id="pkg_meshr" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/meshr.html">meshr</a></td>
	<td class=" ">Koki Tsuyuzaki</td>
	<td class=" ">Tools for conducting enrichment analysis of MeSH</td>
	<td class="  sorting_1">544</td>
</tr><tr id="pkg_methylPipe" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/methylPipe.html">methylPipe</a></td>
	<td class=" ">Kamal Kishore</td>
	<td class=" ">Base resolution DNA methylation data analysis</td>
	<td class="  sorting_1">544</td>
</tr><tr id="pkg_QUBIC" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/QUBIC.html">QUBIC</a></td>
	<td class=" ">Yu Zhang</td>
	<td class=" ">An R package for qualitative biclustering in support of gene co-expression analyses</td>
	<td class="  sorting_1">544</td>
</tr><tr id="pkg_rbsurv" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/rbsurv.html">rbsurv</a></td>
	<td class=" ">Soo-heang Eo</td>
	<td class=" ">Robust likelihood-based survival modeling with microarray data</td>
	<td class="  sorting_1">544</td>
</tr><tr id="pkg_RGSEA" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/RGSEA.html">RGSEA</a></td>
	<td class=" ">Chengcheng Ma</td>
	<td class=" ">Random Gene Set Enrichment Analysis</td>
	<td class="  sorting_1">544</td>
</tr><tr id="pkg_RPA" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/RPA.html">RPA</a></td>
	<td class=" ">Leo Lahti</td>
	<td class=" ">RPA: Robust Probabilistic Averaging for probe-level analysis</td>
	<td class="  sorting_1">544</td>
</tr><tr id="pkg_timecourse" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/timecourse.html">timecourse</a></td>
	<td class=" ">Yu Chuan Tai</td>
	<td class=" ">Statistical Analysis for Developmental Microarray Time Course Data</td>
	<td class="  sorting_1">544</td>
</tr><tr id="pkg_chimera" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/chimera.html">chimera</a></td>
	<td class=" ">Raffaele A Calogero</td>
	<td class=" ">A package for secondary analysis of fusion products</td>
	<td class="  sorting_1">553</td>
</tr><tr id="pkg_chipenrich" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/chipenrich.html">chipenrich</a></td>
	<td class=" ">Raymond G. Cavalcante</td>
	<td class=" ">Gene Set Enrichment For ChIP-seq Peak Data</td>
	<td class="  sorting_1">553</td>
</tr><tr id="pkg_epivizr" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/epivizr.html">epivizr</a></td>
	<td class=" ">Hector Corrada Bravo</td>
	<td class=" ">R Interface to epiviz web app</td>
	<td class="  sorting_1">553</td>
</tr><tr id="pkg_GeneMeta" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/GeneMeta.html">GeneMeta</a></td>
	<td class=" ">Bioconductor Package Maintainer</td>
	<td class=" ">MetaAnalysis for High Throughput Experiments</td>
	<td class="  sorting_1">553</td>
</tr><tr id="pkg_TRONCO" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/TRONCO.html">TRONCO</a></td>
	<td class=" ">Luca De Sano</td>
	<td class=" ">TRONCO, an R package for TRanslational ONCOlogy</td>
	<td class="  sorting_1">553</td>
</tr><tr id="pkg_a4Reporting" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/a4Reporting.html">a4Reporting</a></td>
	<td class=" ">Tobias Verbeke, Willem Ligtenberg</td>
	<td class=" ">Automated Affymetrix Array Analysis Reporting Package</td>
	<td class="  sorting_1">558</td>
</tr><tr id="pkg_epivizrData" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/epivizrData.html">epivizrData</a></td>
	<td class=" ">Hector Corrada Bravo</td>
	<td class=" ">Data Management API for epiviz interactive visualization app</td>
	<td class="  sorting_1">558</td>
</tr><tr id="pkg_GeneSelector" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/GeneSelector.html">GeneSelector</a></td>
	<td class=" ">Martin Slawski</td>
	<td class=" ">Stability and Aggregation of ranked gene lists</td>
	<td class="  sorting_1">558</td>
</tr><tr id="pkg_a4Classif" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/a4Classif.html">a4Classif</a></td>
	<td class=" ">Tobias Verbeke, Willem Ligtenberg</td>
	<td class=" ">Automated Affymetrix Array Analysis Classification Package</td>
	<td class="  sorting_1">561</td>
</tr><tr id="pkg_BaseSpaceR" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/BaseSpaceR.html">BaseSpaceR</a></td>
	<td class=" ">Jared O'Connell</td>
	<td class=" ">R SDK for BaseSpace RESTful API</td>
	<td class="  sorting_1">561</td>
</tr><tr id="pkg_cellTree" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/cellTree.html">cellTree</a></td>
	<td class=" ">David duVerle</td>
	<td class=" ">Inference and visualisation of Single-Cell RNA-seq data as a hierarchical tree structure</td>
	<td class="  sorting_1">561</td>
</tr><tr id="pkg_netbiov" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/netbiov.html">netbiov</a></td>
	<td class=" ">Shailesh tripathi</td>
	<td class=" ">A package for visualizing complex biological network</td>
	<td class="  sorting_1">561</td>
</tr><tr id="pkg_Prostar" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/Prostar.html">Prostar</a></td>
	<td class=" ">Samuel Wieczorek</td>
	<td class=" ">Provides a GUI for DAPAR</td>
	<td class="  sorting_1">561</td>
</tr><tr id="pkg_BCRANK" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/BCRANK.html">BCRANK</a></td>
	<td class=" ">Adam Ameur</td>
	<td class=" ">Predicting binding site consensus from ranked DNA sequences</td>
	<td class="  sorting_1">566</td>
</tr><tr id="pkg_CountClust" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/CountClust.html">CountClust</a></td>
	<td class=" ">Kushal Dey</td>
	<td class=" ">Clustering and Visualizing RNA-Seq Expression Data using Grade of Membership Models</td>
	<td class="  sorting_1">566</td>
</tr><tr id="pkg_ssize" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/ssize.html">ssize</a></td>
	<td class=" ">Gregory R. Warnes</td>
	<td class=" ">Estimate Microarray Sample Size</td>
	<td class="  sorting_1">566</td>
</tr><tr id="pkg_erccdashboard" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/erccdashboard.html">erccdashboard</a></td>
	<td class=" ">Sarah Munro</td>
	<td class=" ">Assess Differential Gene Expression Experiments with ERCC Controls</td>
	<td class="  sorting_1">569</td>
</tr><tr id="pkg_flowFP" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/flowFP.html">flowFP</a></td>
	<td class=" ">Herb Holyst</td>
	<td class=" ">Fingerprinting for Flow Cytometry</td>
	<td class="  sorting_1">569</td>
</tr><tr id="pkg_maanova" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/maanova.html">maanova</a></td>
	<td class=" ">Keith Sheppard</td>
	<td class=" ">Tools for analyzing Micro Array experiments</td>
	<td class="  sorting_1">569</td>
</tr><tr id="pkg_metaMS" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/metaMS.html">metaMS</a></td>
	<td class=" ">Ron Wehrens</td>
	<td class=" ">MS-based metabolomics annotation pipeline</td>
	<td class="  sorting_1">569</td>
</tr><tr id="pkg_VanillaICE" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/VanillaICE.html">VanillaICE</a></td>
	<td class=" ">Robert Scharpf</td>
	<td class=" ">A Hidden Markov Model for high throughput genotyping arrays</td>
	<td class="  sorting_1">569</td>
</tr><tr id="pkg_VariantFiltering" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/VariantFiltering.html">VariantFiltering</a></td>
	<td class=" ">Robert Castelo</td>
	<td class=" ">Filtering of coding and non-coding genetic variants</td>
	<td class="  sorting_1">569</td>
</tr><tr id="pkg_ABSSeq" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/ABSSeq.html">ABSSeq</a></td>
	<td class=" ">Wentao Yang</td>
	<td class=" ">ABSSeq: a new RNA-Seq analysis method based on modelling absolute expression differences</td>
	<td class="  sorting_1">575</td>
</tr><tr id="pkg_CNORode" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/CNORode.html">CNORode</a></td>
	<td class=" ">David Henriques</td>
	<td class=" ">ODE add-on to CellNOptR</td>
	<td class="  sorting_1">575</td>
</tr><tr id="pkg_CrispRVariants" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/CrispRVariants.html">CrispRVariants</a></td>
	<td class=" ">Helen Lindsay</td>
	<td class=" ">Tools for counting and visualising mutations in a target location</td>
	<td class="  sorting_1">575</td>
</tr><tr id="pkg_EBSeqHMM" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/EBSeqHMM.html">EBSeqHMM</a></td>
	<td class=" ">Ning Leng</td>
	<td class=" ">Bayesian analysis for identifying gene or isoform expression changes in ordered RNA-seq experiments</td>
	<td class="  sorting_1">575</td>
</tr><tr id="pkg_keggorthology" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/keggorthology.html">keggorthology</a></td>
	<td class=" ">VJ Carey</td>
	<td class=" ">graph support for KO, KEGG Orthology</td>
	<td class="  sorting_1">575</td>
</tr><tr id="pkg_SpidermiR" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/SpidermiR.html">SpidermiR</a></td>
	<td class=" ">Claudia Cava</td>
	<td class=" ">SpidermiR: An R/Bioconductor package for integrative network analysis with miRNA data</td>
	<td class="  sorting_1">575</td>
</tr><tr id="pkg_weaver" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/weaver.html">weaver</a></td>
	<td class=" ">Seth Falcon</td>
	<td class=" ">Tools and extensions for processing Sweave documents</td>
	<td class="  sorting_1">575</td>
</tr><tr id="pkg_ABarray" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/ABarray.html">ABarray</a></td>
	<td class=" ">Yongming Andrew Sun</td>
	<td class=" ">Microarray QA and statistical data analysis for Applied Biosystems Genome Survey Microrarray (AB1700) gene expression data.</td>
	<td class="  sorting_1">582</td>
</tr><tr id="pkg_bioassayR" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/bioassayR.html">bioassayR</a></td>
	<td class=" ">Tyler Backman</td>
	<td class=" ">Cross-target analysis of small molecule bioactivity</td>
	<td class="  sorting_1">582</td>
</tr><tr id="pkg_epivizrServer" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/epivizrServer.html">epivizrServer</a></td>
	<td class=" ">Hector Corrada Bravo</td>
	<td class=" ">WebSocket server infrastructure for epivizr apps and packages</td>
	<td class="  sorting_1">582</td>
</tr><tr id="pkg_FindMyFriends" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/FindMyFriends.html">FindMyFriends</a></td>
	<td class=" ">Thomas Lin Pedersen</td>
	<td class=" ">Microbial Comparative Genomics in R</td>
	<td class="  sorting_1">582</td>
</tr><tr id="pkg_hyperdraw" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/hyperdraw.html">hyperdraw</a></td>
	<td class=" ">Paul Murrell</td>
	<td class=" ">Visualizing Hypergaphs</td>
	<td class="  sorting_1">582</td>
</tr><tr id="pkg_multiMiR" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/multiMiR.html">multiMiR</a></td>
	<td class=" ">Matt Mulvahill</td>
	<td class=" ">Integration of multiple microRNA-target databases with their disease and drug associations</td>
	<td class="  sorting_1">582</td>
</tr><tr id="pkg_RiboProfiling" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/RiboProfiling.html">RiboProfiling</a></td>
	<td class=" ">A. Popa</td>
	<td class=" ">Ribosome Profiling Data Analysis: from BAM to Data Representation and Interpretation</td>
	<td class="  sorting_1">582</td>
</tr><tr id="pkg_RnaSeqSampleSize" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/RnaSeqSampleSize.html">RnaSeqSampleSize</a></td>
	<td class=" ">Shilin Zhao</td>
	<td class=" ">RnaSeqSampleSize</td>
	<td class="  sorting_1">582</td>
</tr><tr id="pkg_signeR" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/signeR.html">signeR</a></td>
	<td class=" ">Renan Valieris</td>
	<td class=" ">Empirical Bayesian approach to mutational signature discovery</td>
	<td class="  sorting_1">582</td>
</tr><tr id="pkg_apComplex" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/apComplex.html">apComplex</a></td>
	<td class=" ">Denise Scholtens</td>
	<td class=" ">Estimate protein complex membership using AP-MS protein data</td>
	<td class="  sorting_1">591</td>
</tr><tr id="pkg_DOQTL" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/DOQTL.html">DOQTL</a></td>
	<td class=" ">Daniel Gatti</td>
	<td class=" ">Genotyping and QTL Mapping in DO Mice</td>
	<td class="  sorting_1">591</td>
</tr><tr id="pkg_MultiDataSet" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/MultiDataSet.html">MultiDataSet</a></td>
	<td class=" ">Carlos Ruiz-Arenas</td>
	<td class=" ">Implementation of MultiDataSet and ResultSet</td>
	<td class="  sorting_1">591</td>
</tr><tr id="pkg_psichomics" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/psichomics.html">psichomics</a></td>
	<td class=" ">Nuno Saraiva-Agostinho</td>
	<td class=" ">Graphical Interface for Alternative Splicing Quantification, Analysis and Visualisation</td>
	<td class="  sorting_1">591</td>
</tr><tr id="pkg_RpsiXML" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/RpsiXML.html">RpsiXML</a></td>
	<td class=" ">Jitao David Zhang</td>
	<td class=" ">R interface to PSI-MI 2.5 files</td>
	<td class="  sorting_1">591</td>
</tr><tr id="pkg_sevenbridges" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/sevenbridges.html">sevenbridges</a></td>
	<td class=" ">Nan Xiao</td>
	<td class=" ">Seven Bridges Platform API Client and Common Workflow Language Tool Builder in R</td>
	<td class="  sorting_1">591</td>
</tr><tr id="pkg_tweeDEseq" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/tweeDEseq.html">tweeDEseq</a></td>
	<td class=" ">Juan R Gonzalez</td>
	<td class=" ">RNA-seq data analysis using the Poisson-Tweedie family of distributions</td>
	<td class="  sorting_1">591</td>
</tr><tr id="pkg_AneuFinder" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/AneuFinder.html">AneuFinder</a></td>
	<td class=" ">Aaron Taudt</td>
	<td class=" ">Analysis of Copy Number Variation in Single-Cell-Sequencing Data</td>
	<td class="  sorting_1">598</td>
</tr><tr id="pkg_CoRegNet" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/CoRegNet.html">CoRegNet</a></td>
	<td class=" ">Remy Nicolle</td>
	<td class=" ">CoRegNet : reconstruction and integrated analysis of co-regulatory networks</td>
	<td class="  sorting_1">598</td>
</tr><tr id="pkg_CoverageView" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/CoverageView.html">CoverageView</a></td>
	<td class=" ">Ernesto Lowy</td>
	<td class=" ">Coverage visualization package for R</td>
	<td class="  sorting_1">598</td>
</tr><tr id="pkg_DMRcaller" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/DMRcaller.html">DMRcaller</a></td>
	<td class=" ">Nicolae Radu Zabet</td>
	<td class=" ">Differentially Methylated Regions caller</td>
	<td class="  sorting_1">598</td>
</tr><tr id="pkg_DNAshapeR" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/DNAshapeR.html">DNAshapeR</a></td>
	<td class=" ">Tsu-Pei Chiu</td>
	<td class=" ">High-throughput prediction of DNA shape features</td>
	<td class="  sorting_1">598</td>
</tr><tr id="pkg_ENCODExplorer" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/ENCODExplorer.html">ENCODExplorer</a></td>
	<td class=" ">Charles Joly Beauparlant</td>
	<td class=" ">A compilation of ENCODE metadata</td>
	<td class="  sorting_1">598</td>
</tr><tr id="pkg_GOTHiC" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/GOTHiC.html">GOTHiC</a></td>
	<td class=" ">Borbala Mifsud</td>
	<td class=" ">Binomial test for Hi-C data analysis</td>
	<td class="  sorting_1">598</td>
</tr><tr id="pkg_HIBAG" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/HIBAG.html">HIBAG</a></td>
	<td class=" ">Xiuwen Zheng</td>
	<td class=" ">HLA Genotype Imputation with Attribute Bagging</td>
	<td class="  sorting_1">598</td>
</tr><tr id="pkg_metaArray" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/metaArray.html">metaArray</a></td>
	<td class=" ">Hyungwon Choi</td>
	<td class=" ">Integration of Microarray Data for Meta-analysis</td>
	<td class="  sorting_1">598</td>
</tr><tr id="pkg_metaseqR" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/metaseqR.html">metaseqR</a></td>
	<td class=" ">Panagiotis Moulos</td>
	<td class=" ">An R package for the analysis and result reporting of RNA-Seq data by combining multiple statistical algorithms.</td>
	<td class="  sorting_1">598</td>
</tr><tr id="pkg_philr" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/philr.html">philr</a></td>
	<td class=" ">Justin Silverman</td>
	<td class=" ">Phylogenetic partitioning based ILR transform for metagenomics data</td>
	<td class="  sorting_1">598</td>
</tr><tr id="pkg_sincell" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/sincell.html">sincell</a></td>
	<td class=" ">Miguel Julia, Antonio Rausell</td>
	<td class=" ">R package for the statistical assessment of cell state hierarchies from single-cell RNA-seq data</td>
	<td class="  sorting_1">598</td>
</tr><tr id="pkg_SWATH2stats" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/SWATH2stats.html">SWATH2stats</a></td>
	<td class=" ">Peter Blattmann</td>
	<td class=" ">Transform and Filter SWATH Data for Statistical Packages</td>
	<td class="  sorting_1">598</td>
</tr><tr id="pkg_TEQC" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/TEQC.html">TEQC</a></td>
	<td class=" ">Manuela Hummel</td>
	<td class=" ">Quality control for target capture experiments</td>
	<td class="  sorting_1">598</td>
</tr><tr id="pkg_yaqcaffy" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/yaqcaffy.html">yaqcaffy</a></td>
	<td class=" ">Laurent Gatto</td>
	<td class=" ">Affymetrix expression data quality control and reproducibility analysis</td>
	<td class="  sorting_1">598</td>
</tr><tr id="pkg_BiocWorkflowTools" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/BiocWorkflowTools.html">BiocWorkflowTools</a></td>
	<td class=" ">Mike Smith</td>
	<td class=" ">Tools to aid the development of Bioconductor Workflow packages</td>
	<td class="  sorting_1">613</td>
</tr><tr id="pkg_BridgeDbR" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/BridgeDbR.html">BridgeDbR</a></td>
	<td class=" ">Egon Willighagen</td>
	<td class=" ">Code for using BridgeDb identifier mapping framework from within R</td>
	<td class="  sorting_1">613</td>
</tr><tr id="pkg_ClassifyR" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/ClassifyR.html">ClassifyR</a></td>
	<td class=" ">Dario Strbenac</td>
	<td class=" ">A framework for cross-validated classification problems, with applications to differential variability and differential distribution testing</td>
	<td class="  sorting_1">613</td>
</tr><tr id="pkg_EmpiricalBrownsMethod" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/EmpiricalBrownsMethod.html">EmpiricalBrownsMethod</a></td>
	<td class=" ">David Gibbs</td>
	<td class=" ">Uses Brown's method to combine p-values from dependent tests</td>
	<td class="  sorting_1">613</td>
</tr><tr id="pkg_flowType" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/flowType.html">flowType</a></td>
	<td class=" ">Nima Aghaeepour</td>
	<td class=" ">Phenotyping Flow Cytometry Assays</td>
	<td class="  sorting_1">613</td>
</tr><tr id="pkg_GeneticsDesign" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/GeneticsDesign.html">GeneticsDesign</a></td>
	<td class=" ">The R Genetics Project</td>
	<td class=" ">Functions for designing genetics studies</td>
	<td class="  sorting_1">613</td>
</tr><tr id="pkg_Harman" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/Harman.html">Harman</a></td>
	<td class=" ">Jason Ross</td>
	<td class=" ">The removal of batch effects from datasets using a PCA and constrained optimisation based technique</td>
	<td class="  sorting_1">613</td>
</tr><tr id="pkg_iCOBRA" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/iCOBRA.html">iCOBRA</a></td>
	<td class=" ">Charlotte Soneson</td>
	<td class=" ">Comparison and Visualization of Ranking and Assignment Methods</td>
	<td class="  sorting_1">613</td>
</tr><tr id="pkg_metagene" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/metagene.html">metagene</a></td>
	<td class=" ">Charles Joly Beauparlant</td>
	<td class=" ">A package to produce metagene plots</td>
	<td class="  sorting_1">613</td>
</tr><tr id="pkg_mgsa" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/mgsa.html">mgsa</a></td>
	<td class=" ">Sebastian Bauer</td>
	<td class=" ">Model-based gene set analysis</td>
	<td class="  sorting_1">613</td>
</tr><tr id="pkg_MLP" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/MLP.html">MLP</a></td>
	<td class=" ">Tobias Verbeke</td>
	<td class=" ">MLP</td>
	<td class="  sorting_1">613</td>
</tr><tr id="pkg_rCGH" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/rCGH.html">rCGH</a></td>
	<td class=" ">Frederic Commo</td>
	<td class=" ">Comprehensive Pipeline for Analyzing and Visualizing Array-Based CGH Data</td>
	<td class="  sorting_1">613</td>
</tr><tr id="pkg_RMassBank" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/RMassBank.html">RMassBank</a></td>
	<td class=" ">RMassBank at Eawag</td>
	<td class=" ">Workflow to process tandem MS files and build MassBank records</td>
	<td class="  sorting_1">613</td>
</tr><tr id="pkg_TCseq" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/TCseq.html">TCseq</a></td>
	<td class=" ">Mengjun Wu</td>
	<td class=" ">Time course sequencing data analysis</td>
	<td class="  sorting_1">613</td>
</tr><tr id="pkg_BioSeqClass" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/BioSeqClass.html">BioSeqClass</a></td>
	<td class=" ">Li Hong</td>
	<td class=" ">Classification for Biological Sequences</td>
	<td class="  sorting_1">627</td>
</tr><tr id="pkg_CGEN" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/CGEN.html">CGEN</a></td>
	<td class=" ">William Wheeler</td>
	<td class=" ">An R package for analysis of case-control studies in genetic epidemiology</td>
	<td class="  sorting_1">627</td>
</tr><tr id="pkg_COMPASS" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/COMPASS.html">COMPASS</a></td>
	<td class=" ">Greg Finak</td>
	<td class=" ">Combinatorial Polyfunctionality Analysis of Single Cells</td>
	<td class="  sorting_1">627</td>
</tr><tr id="pkg_NetPathMiner" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/NetPathMiner.html">NetPathMiner</a></td>
	<td class=" ">Ahmed Mohamed</td>
	<td class=" ">NetPathMiner for Biological Network Construction, Path Mining and Visualization</td>
	<td class="  sorting_1">627</td>
</tr><tr id="pkg_omicade4" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/omicade4.html">omicade4</a></td>
	<td class=" ">Chen Meng</td>
	<td class=" ">Multiple co-inertia analysis of omics datasets</td>
	<td class="  sorting_1">627</td>
</tr><tr id="pkg_oposSOM" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/oposSOM.html">oposSOM</a></td>
	<td class=" ">Henry Loeffler-Wirth</td>
	<td class=" ">Comprehensive analysis of transciptome data</td>
	<td class="  sorting_1">627</td>
</tr><tr id="pkg_AffyCompatible" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/AffyCompatible.html">AffyCompatible</a></td>
	<td class=" ">Martin Morgan</td>
	<td class=" ">Affymetrix GeneChip software compatibility</td>
	<td class="  sorting_1">633</td>
</tr><tr id="pkg_altcdfenvs" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/altcdfenvs.html">altcdfenvs</a></td>
	<td class=" ">Laurent Gautier</td>
	<td class=" ">alternative CDF environments (aka probeset mappings)</td>
	<td class="  sorting_1">633</td>
</tr><tr id="pkg_beadarraySNP" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/beadarraySNP.html">beadarraySNP</a></td>
	<td class=" ">Jan Oosting</td>
	<td class=" ">Normalization and reporting of Illumina SNP bead arrays</td>
	<td class="  sorting_1">633</td>
</tr><tr id="pkg_COHCAP" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/COHCAP.html">COHCAP</a></td>
	<td class=" ">Charles Warden</td>
	<td class=" ">CpG Island Analysis Pipeline for Illumina Methylation Array and Targeted BS-Seq Data</td>
	<td class="  sorting_1">633</td>
</tr><tr id="pkg_compcodeR" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/compcodeR.html">compcodeR</a></td>
	<td class=" ">Charlotte Soneson</td>
	<td class=" ">RNAseq data simulation, differential expression analysis and performance comparison of differential expression methods</td>
	<td class="  sorting_1">633</td>
</tr><tr id="pkg_eisa" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/eisa.html">eisa</a></td>
	<td class=" ">Gabor Csardi</td>
	<td class=" ">Expression data analysis via the Iterative Signature Algorithm</td>
	<td class="  sorting_1">633</td>
</tr><tr id="pkg_PSICQUIC" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/PSICQUIC.html">PSICQUIC</a></td>
	<td class=" ">Paul Shannon</td>
	<td class=" ">Proteomics Standard Initiative Common QUery InterfaCe</td>
	<td class="  sorting_1">633</td>
</tr><tr id="pkg_ABAEnrichment" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/ABAEnrichment.html">ABAEnrichment</a></td>
	<td class=" ">Steffi Grote</td>
	<td class=" ">Gene expression enrichment in human brain regions</td>
	<td class="  sorting_1">640</td>
</tr><tr id="pkg_AllelicImbalance" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/AllelicImbalance.html">AllelicImbalance</a></td>
	<td class=" ">Jesper R Gadin</td>
	<td class=" ">Investigates Allele Specific Expression</td>
	<td class="  sorting_1">640</td>
</tr><tr id="pkg_CNVtools" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/CNVtools.html">CNVtools</a></td>
	<td class=" ">Chris Barnes</td>
	<td class=" ">A package to test genetic association with CNV data</td>
	<td class="  sorting_1">640</td>
</tr><tr id="pkg_codelink" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/codelink.html">codelink</a></td>
	<td class=" ">Diego Diez</td>
	<td class=" ">Manipulation of Codelink microarray data</td>
	<td class="  sorting_1">640</td>
</tr><tr id="pkg_IONiseR" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/IONiseR.html">IONiseR</a></td>
	<td class=" ">Mike Smith</td>
	<td class=" ">Quality Assessment Tools for Oxford Nanopore MinION data</td>
	<td class="  sorting_1">640</td>
</tr><tr id="pkg_myvariant" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/myvariant.html">myvariant</a></td>
	<td class=" ">Adam Mark, Chunlei Wu</td>
	<td class=" ">Accesses MyVariant.info variant query and annotation services</td>
	<td class="  sorting_1">640</td>
</tr><tr id="pkg_readat" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/readat.html">readat</a></td>
	<td class=" ">Richard Cotton</td>
	<td class=" ">Functionality to Read and Manipulate SomaLogic ADAT files</td>
	<td class="  sorting_1">640</td>
</tr><tr id="pkg_Roleswitch" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/Roleswitch.html">Roleswitch</a></td>
	<td class=" ">Yue Li</td>
	<td class=" ">Infer miRNA-mRNA interactions using paired expression data from a single sample</td>
	<td class="  sorting_1">640</td>
</tr><tr id="pkg_scPipe" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/scPipe.html">scPipe</a></td>
	<td class=" ">Luyi Tian</td>
	<td class=" ">pipeline for single cell RNA-seq data analysis</td>
	<td class="  sorting_1">640</td>
</tr><tr id="pkg_TPP" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/TPP.html">TPP</a></td>
	<td class=" ">Dorothee Childs</td>
	<td class=" ">Analyze thermal proteome profiling (TPP) experiments</td>
	<td class="  sorting_1">640</td>
</tr><tr id="pkg_trio" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/trio.html">trio</a></td>
	<td class=" ">Holger Schwender</td>
	<td class=" ">Testing of SNPs and SNP Interactions in Case-Parent Trio Studies</td>
	<td class="  sorting_1">640</td>
</tr><tr id="pkg_acde" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/acde.html">acde</a></td>
	<td class=" ">Juan Pablo Acosta</td>
	<td class=" ">Artificial Components Detection of Differentially Expressed Genes</td>
	<td class="  sorting_1">651</td>
</tr><tr id="pkg_AffyExpress" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/AffyExpress.html">AffyExpress</a></td>
	<td class=" ">Xuejun Arthur Li</td>
	<td class=" ">Affymetrix Quality Assessment and Analysis Tool</td>
	<td class="  sorting_1">651</td>
</tr><tr id="pkg_Chicago" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/Chicago.html">Chicago</a></td>
	<td class=" ">Mikhail Spivakov</td>
	<td class=" ">CHiCAGO: Capture Hi-C Analysis of Genomic Organization</td>
	<td class="  sorting_1">651</td>
</tr><tr id="pkg_CNAnorm" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/CNAnorm.html">CNAnorm</a></td>
	<td class=" ">Stefano Berri</td>
	<td class=" ">A normalization method for Copy Number Aberration in cancer samples</td>
	<td class="  sorting_1">651</td>
</tr><tr id="pkg_CoGAPS" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/CoGAPS.html">CoGAPS</a></td>
	<td class=" ">Elana J. Fertig, Thomas D. Sherman</td>
	<td class=" ">Coordinated Gene Activity in Pattern Sets</td>
	<td class="  sorting_1">651</td>
</tr><tr id="pkg_CytoML" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/CytoML.html">CytoML</a></td>
	<td class=" ">Mike Jiang</td>
	<td class=" ">A GatingML Interface for Cross Platform Cytometry Data Sharing</td>
	<td class="  sorting_1">651</td>
</tr><tr id="pkg_FitHiC" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/FitHiC.html">FitHiC</a></td>
	<td class=" ">Ruyu Tan</td>
	<td class=" ">Confidence estimation for intra-chromosomal contact maps</td>
	<td class="  sorting_1">651</td>
</tr><tr id="pkg_flipflop" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/flipflop.html">flipflop</a></td>
	<td class=" ">Elsa Bernard</td>
	<td class=" ">Fast lasso-based isoform prediction as a flow problem</td>
	<td class="  sorting_1">651</td>
</tr><tr id="pkg_flowTrans" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/flowTrans.html">flowTrans</a></td>
	<td class=" ">Greg Finak</td>
	<td class=" ">Parameter Optimization for Flow Cytometry Data Transformation</td>
	<td class="  sorting_1">651</td>
</tr><tr id="pkg_HelloRanges" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/HelloRanges.html">HelloRanges</a></td>
	<td class=" ">Michael Lawrence</td>
	<td class=" ">Introduce *Ranges to bedtools users</td>
	<td class="  sorting_1">651</td>
</tr><tr id="pkg_motifbreakR" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/motifbreakR.html">motifbreakR</a></td>
	<td class=" ">Simon Gert Coetzee</td>
	<td class=" ">A Package For Predicting The Disruptiveness Of Single Nucleotide Polymorphisms On Transcription Factor Binding Sites</td>
	<td class="  sorting_1">651</td>
</tr><tr id="pkg_OmicsMarkeR" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/OmicsMarkeR.html">OmicsMarkeR</a></td>
	<td class=" ">Charles E. Determan Jr.</td>
	<td class=" ">Classification and Feature Selection for 'Omics' Datasets</td>
	<td class="  sorting_1">651</td>
</tr><tr id="pkg_OutlierD" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/OutlierD.html">OutlierD</a></td>
	<td class=" ">Sukwoo Kim</td>
	<td class=" ">Outlier detection using quantile regression on the M-A scatterplots of high-throughput data</td>
	<td class="  sorting_1">651</td>
</tr><tr id="pkg_pkgDepTools" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/pkgDepTools.html">pkgDepTools</a></td>
	<td class=" ">Bioconductor Package Maintainer</td>
	<td class=" ">Package Dependency Tools</td>
	<td class="  sorting_1">651</td>
</tr><tr id="pkg_RbcBook1" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/RbcBook1.html">RbcBook1</a></td>
	<td class=" ">Vince Carey</td>
	<td class=" ">Support for Springer monograph on Bioconductor</td>
	<td class="  sorting_1">651</td>
</tr><tr id="pkg_alpine" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/alpine.html">alpine</a></td>
	<td class=" ">Michael Love</td>
	<td class=" ">alpine</td>
	<td class="  sorting_1">666</td>
</tr><tr id="pkg_ASSET" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/ASSET.html">ASSET</a></td>
	<td class=" ">Samsiddhi Bhattacharjee</td>
	<td class=" ">An R package for subset-based association analysis of heterogeneous traits and subtypes</td>
	<td class="  sorting_1">666</td>
</tr><tr id="pkg_bacon" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/bacon.html">bacon</a></td>
	<td class=" ">Maarten van Iterson</td>
	<td class=" ">Controlling bias and inflation in association studies using the empirical null distribution</td>
	<td class="  sorting_1">666</td>
</tr><tr id="pkg_BrowserViz" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/BrowserViz.html">BrowserViz</a></td>
	<td class=" ">Paul Shannon</td>
	<td class=" ">BrowserViz: interactive R/browser graphics using websockets and JSON</td>
	<td class="  sorting_1">666</td>
</tr><tr id="pkg_ChromHeatMap" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/ChromHeatMap.html">ChromHeatMap</a></td>
	<td class=" ">Tim F. Rayner</td>
	<td class=" ">Heat map plotting by genome coordinate</td>
	<td class="  sorting_1">666</td>
</tr><tr id="pkg_chromstaR" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/chromstaR.html">chromstaR</a></td>
	<td class=" ">Aaron Taudt</td>
	<td class=" ">Combinatorial and Differential Chromatin State Analysis for ChIP-Seq Data</td>
	<td class="  sorting_1">666</td>
</tr><tr id="pkg_compEpiTools" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/compEpiTools.html">compEpiTools</a></td>
	<td class=" ">Kamal Kishore</td>
	<td class=" ">Tools for computational epigenomics</td>
	<td class="  sorting_1">666</td>
</tr><tr id="pkg_CompGO" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/CompGO.html">CompGO</a></td>
	<td class=" ">Ashley J. Waardenberg</td>
	<td class=" ">An R pipeline for .bed file annotation, comparing GO term enrichment between gene sets and data visualisation</td>
	<td class="  sorting_1">666</td>
</tr><tr id="pkg_DriverNet" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/DriverNet.html">DriverNet</a></td>
	<td class=" ">Jiarui Ding</td>
	<td class=" ">Drivernet: uncovering somatic driver mutations modulating transcriptional networks in cancer</td>
	<td class="  sorting_1">666</td>
</tr><tr id="pkg_farms" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/farms.html">farms</a></td>
	<td class=" ">Djork-Arne Clevert</td>
	<td class=" ">FARMS - Factor Analysis for Robust Microarray Summarization</td>
	<td class="  sorting_1">666</td>
</tr><tr id="pkg_FourCSeq" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/FourCSeq.html">FourCSeq</a></td>
	<td class=" ">Felix A. Klein</td>
	<td class=" ">Package analyse 4C sequencing data</td>
	<td class="  sorting_1">666</td>
</tr><tr id="pkg_gespeR" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/gespeR.html">gespeR</a></td>
	<td class=" ">Fabian Schmich</td>
	<td class=" ">Gene-Specific Phenotype EstimatoR</td>
	<td class="  sorting_1">666</td>
</tr><tr id="pkg_GOFunction" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/GOFunction.html">GOFunction</a></td>
	<td class=" ">Jing Wang</td>
	<td class=" ">GO-function: deriving biologcially relevant functions from statistically significant functions</td>
	<td class="  sorting_1">666</td>
</tr><tr id="pkg_groHMM" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/groHMM.html">groHMM</a></td>
	<td class=" ">Anusha Nagari, Tulip Nandu, W. Lee Kraus</td>
	<td class=" ">GRO-seq Analysis Pipeline</td>
	<td class="  sorting_1">666</td>
</tr><tr id="pkg_MEIGOR" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/MEIGOR.html">MEIGOR</a></td>
	<td class=" ">Jose Egea</td>
	<td class=" ">MEIGO - MEtaheuristics for bIoinformatics Global Optimization</td>
	<td class="  sorting_1">666</td>
</tr><tr id="pkg_pandaR" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/pandaR.html">pandaR</a></td>
	<td class=" ">Joseph N. Paulson, Dan Schlauch</td>
	<td class=" ">PANDA Algorithm</td>
	<td class="  sorting_1">666</td>
</tr><tr id="pkg_RamiGO" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/RamiGO.html">RamiGO</a></td>
	<td class=" ">Markus Schroeder</td>
	<td class=" ">AmiGO visualize R interface</td>
	<td class="  sorting_1">666</td>
</tr><tr id="pkg_ROntoTools" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/ROntoTools.html">ROntoTools</a></td>
	<td class=" ">Calin Voichita</td>
	<td class=" ">R Onto-Tools suite</td>
	<td class="  sorting_1">666</td>
</tr><tr id="pkg_RRHO" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/RRHO.html">RRHO</a></td>
	<td class=" ">Jonathan Rosenblatt</td>
	<td class=" ">Inference on agreement between ordered lists</td>
	<td class="  sorting_1">666</td>
</tr><tr id="pkg_affyPara" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/affyPara.html">affyPara</a></td>
	<td class=" ">Markus Schmidberger</td>
	<td class=" ">Parallelized preprocessing methods for Affymetrix Oligonucleotide Arrays</td>
	<td class="  sorting_1">685</td>
</tr><tr id="pkg_anota" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/anota.html">anota</a></td>
	<td class=" ">Ola Larsson</td>
	<td class=" ">ANalysis Of Translational Activity (ANOTA).</td>
	<td class="  sorting_1">685</td>
</tr><tr id="pkg_Basic4Cseq" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/Basic4Cseq.html">Basic4Cseq</a></td>
	<td class=" ">Carolin Walter</td>
	<td class=" ">Basic4Cseq: an R/Bioconductor package for analyzing 4C-seq data</td>
	<td class="  sorting_1">685</td>
</tr><tr id="pkg_ccmap" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/ccmap.html">ccmap</a></td>
	<td class=" ">Alex Pickering</td>
	<td class=" ">Combination Connectivity Mapping</td>
	<td class="  sorting_1">685</td>
</tr><tr id="pkg_CGHregions" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/CGHregions.html">CGHregions</a></td>
	<td class=" ">Sjoerd Vosse</td>
	<td class=" ">Dimension Reduction for Array CGH Data with Minimal Information Loss.</td>
	<td class="  sorting_1">685</td>
</tr><tr id="pkg_CNVPanelizer" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/CNVPanelizer.html">CNVPanelizer</a></td>
	<td class=" ">Thomas Wolf</td>
	<td class=" ">Reliable CNV detection in targeted sequencing applications</td>
	<td class="  sorting_1">685</td>
</tr><tr id="pkg_ExpressionAtlas" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/ExpressionAtlas.html">ExpressionAtlas</a></td>
	<td class=" ">Suhaib Mohammed</td>
	<td class=" ">Download datasets from EMBL-EBI Expression Atlas</td>
	<td class="  sorting_1">685</td>
</tr><tr id="pkg_genomes" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/genomes.html">genomes</a></td>
	<td class=" ">Chris Stubben</td>
	<td class=" ">Genome sequencing project metadata</td>
	<td class="  sorting_1">685</td>
</tr><tr id="pkg_hiAnnotator" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/hiAnnotator.html">hiAnnotator</a></td>
	<td class=" ">Nirav V Malani</td>
	<td class=" ">Functions for annotating GRanges objects</td>
	<td class="  sorting_1">685</td>
</tr><tr id="pkg_ImpulseDE2" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/ImpulseDE2.html">ImpulseDE2</a></td>
	<td class=" ">David S Fischer</td>
	<td class=" ">Differential expression analysis of longitudinal count data sets</td>
	<td class="  sorting_1">685</td>
</tr><tr id="pkg_LMGene" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/LMGene.html">LMGene</a></td>
	<td class=" ">Blythe Durbin-Johnson</td>
	<td class=" ">LMGene Software for Data Transformation and Identification of Differentially Expressed Genes in Gene Expression Arrays</td>
	<td class="  sorting_1">685</td>
</tr><tr id="pkg_nucleR" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/nucleR.html">nucleR</a></td>
	<td class=" ">Ricard Illa</td>
	<td class=" ">Nucleosome positioning package for R</td>
	<td class="  sorting_1">685</td>
</tr><tr id="pkg_PROPER" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/PROPER.html">PROPER</a></td>
	<td class=" ">Hao Wu</td>
	<td class=" ">PROspective Power Evaluation for RNAseq</td>
	<td class="  sorting_1">685</td>
</tr><tr id="pkg_scfind" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/scfind.html">scfind</a></td>
	<td class=" ">Vladimir Kiselev</td>
	<td class=" ">A search tool for single cell RNA-seq data by gene lists</td>
	<td class="  sorting_1">685</td>
</tr><tr id="pkg_alsace" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/alsace.html">alsace</a></td>
	<td class=" ">Ron Wehrens</td>
	<td class=" ">ALS for the Automatic Chemical Exploration of mixtures</td>
	<td class="  sorting_1">699</td>
</tr><tr id="pkg_BiGGR" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/BiGGR.html">BiGGR</a></td>
	<td class=" ">Anand K. Gavai, Hannes Hettling</td>
	<td class=" ">Constraint based modeling in R using metabolic reconstruction databases</td>
	<td class="  sorting_1">699</td>
</tr><tr id="pkg_CALIB" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/CALIB.html">CALIB</a></td>
	<td class=" ">Hui Zhao</td>
	<td class=" ">Calibration model for estimating absolute expression levels from microarray data</td>
	<td class="  sorting_1">699</td>
</tr><tr id="pkg_crossmeta" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/crossmeta.html">crossmeta</a></td>
	<td class=" ">Alex Pickering</td>
	<td class=" ">Cross Platform Meta-Analysis of Microarray Data</td>
	<td class="  sorting_1">699</td>
</tr><tr id="pkg_Guitar" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/Guitar.html">Guitar</a></td>
	<td class=" ">Jia Meng</td>
	<td class=" ">Guitar</td>
	<td class="  sorting_1">699</td>
</tr><tr id="pkg_LPE" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/LPE.html">LPE</a></td>
	<td class=" ">Nitin Jain</td>
	<td class=" ">Methods for analyzing microarray data using Local Pooled Error (LPE) method</td>
	<td class="  sorting_1">699</td>
</tr><tr id="pkg_meshes" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/meshes.html">meshes</a></td>
	<td class=" ">Guangchuang Yu</td>
	<td class=" ">MeSH Enrichment and Semantic analyses</td>
	<td class="  sorting_1">699</td>
</tr><tr id="pkg_MethylSeekR" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/MethylSeekR.html">MethylSeekR</a></td>
	<td class=" ">Lukas Burger</td>
	<td class=" ">Segmentation of Bis-seq data</td>
	<td class="  sorting_1">699</td>
</tr><tr id="pkg_PAA" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/PAA.html">PAA</a></td>
	<td class=" ">Michael Turewicz, Martin Eisenacher</td>
	<td class=" ">PAA (Protein Array Analyzer)</td>
	<td class="  sorting_1">699</td>
</tr><tr id="pkg_panp" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/panp.html">panp</a></td>
	<td class=" ">Peter Warren</td>
	<td class=" ">Presence-Absence Calls from Negative Strand Matching Probesets</td>
	<td class="  sorting_1">699</td>
</tr><tr id="pkg_ccrepe" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/ccrepe.html">ccrepe</a></td>
	<td class=" ">Emma Schwager,Craig Bielski, George Weingart</td>
	<td class=" ">ccrepe_and_nc.score</td>
	<td class="  sorting_1">709</td>
</tr><tr id="pkg_frmaTools" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/frmaTools.html">frmaTools</a></td>
	<td class=" ">Matthew N. McCall</td>
	<td class=" ">Frozen RMA Tools</td>
	<td class="  sorting_1">709</td>
</tr><tr id="pkg_rain" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/rain.html">rain</a></td>
	<td class=" ">Paul F. Thaben</td>
	<td class=" ">Rhythmicity Analysis Incorporating Non-parametric Methods</td>
	<td class="  sorting_1">709</td>
</tr><tr id="pkg_Rcade" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/Rcade.html">Rcade</a></td>
	<td class=" ">Jonathan Cairns</td>
	<td class=" ">R-based analysis of ChIP-seq And Differential Expression - a tool for integrating a count-based ChIP-seq analysis with differential expression summary data</td>
	<td class="  sorting_1">709</td>
</tr><tr id="pkg_rhdf5client" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/rhdf5client.html">rhdf5client</a></td>
	<td class=" ">Samuela Pollack</td>
	<td class=" ">Access HDF5 content from h5serv</td>
	<td class="  sorting_1">709</td>
</tr><tr id="pkg_riboSeqR" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/riboSeqR.html">riboSeqR</a></td>
	<td class=" ">Thomas J. Hardcastle</td>
	<td class=" ">Analysis of sequencing data from ribosome profiling experiments</td>
	<td class="  sorting_1">709</td>
</tr><tr id="pkg_savR" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/savR.html">savR</a></td>
	<td class=" ">R. Brent Calder</td>
	<td class=" ">Parse and analyze Illumina SAV files</td>
	<td class="  sorting_1">709</td>
</tr><tr id="pkg_snapCGH" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/snapCGH.html">snapCGH</a></td>
	<td class=" ">John Marioni</td>
	<td class=" ">Segmentation, normalisation and processing of aCGH data.</td>
	<td class="  sorting_1">709</td>
</tr><tr id="pkg_subSeq" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/subSeq.html">subSeq</a></td>
	<td class=" ">Andrew J. Bass, John D. Storey</td>
	<td class=" ">Subsampling of high-throughput sequencing count data</td>
	<td class="  sorting_1">709</td>
</tr><tr id="pkg_ADaCGH2" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/ADaCGH2.html">ADaCGH2</a></td>
	<td class=" ">Ramon Diaz-Uriarte</td>
	<td class=" ">Analysis of big data from aCGH experiments using parallel computing and ff objects</td>
	<td class="  sorting_1">718</td>
</tr><tr id="pkg_adSplit" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/adSplit.html">adSplit</a></td>
	<td class=" ">Claudio Lottaz</td>
	<td class=" ">Annotation-Driven Clustering</td>
	<td class="  sorting_1">718</td>
</tr><tr id="pkg_ArrayTV" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/ArrayTV.html">ArrayTV</a></td>
	<td class=" ">Eitan Halper-Stromberg</td>
	<td class=" ">Implementation of wave correction for arrays</td>
	<td class="  sorting_1">718</td>
</tr><tr id="pkg_BufferedMatrix" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/BufferedMatrix.html">BufferedMatrix</a></td>
	<td class=" ">Ben Bolstad</td>
	<td class=" ">A matrix data storage object held in temporary files</td>
	<td class="  sorting_1">718</td>
</tr><tr id="pkg_CancerMutationAnalysis" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/CancerMutationAnalysis.html">CancerMutationAnalysis</a></td>
	<td class=" ">Simina M. Boca</td>
	<td class=" ">Cancer mutation analysis</td>
	<td class="  sorting_1">718</td>
</tr><tr id="pkg_charm" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/charm.html">charm</a></td>
	<td class=" ">Peter Murakami</td>
	<td class=" ">Analysis of DNA methylation data from CHARM microarrays</td>
	<td class="  sorting_1">718</td>
</tr><tr id="pkg_DeepBlueR" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/DeepBlueR.html">DeepBlueR</a></td>
	<td class=" ">Felipe Albrecht, Markus List</td>
	<td class=" ">DeepBlueR</td>
	<td class="  sorting_1">718</td>
</tr><tr id="pkg_EpiDISH" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/EpiDISH.html">EpiDISH</a></td>
	<td class=" ">Shijie Charles Zheng</td>
	<td class=" ">Epigenetic Dissection of Intra-Sample-Heterogeneity</td>
	<td class="  sorting_1">718</td>
</tr><tr id="pkg_ffpe" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/ffpe.html">ffpe</a></td>
	<td class=" ">Levi Waldron</td>
	<td class=" ">Quality assessment and control for FFPE microarray expression data</td>
	<td class="  sorting_1">718</td>
</tr><tr id="pkg_Genominator" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/Genominator.html">Genominator</a></td>
	<td class=" ">James Bullard</td>
	<td class=" ">Analyze, manage and store genomic data</td>
	<td class="  sorting_1">718</td>
</tr><tr id="pkg_iterativeBMA" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/iterativeBMA.html">iterativeBMA</a></td>
	<td class=" ">Ka Yee Yeung</td>
	<td class=" ">The Iterative Bayesian Model Averaging (BMA) algorithm</td>
	<td class="  sorting_1">718</td>
</tr><tr id="pkg_mdgsa" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/mdgsa.html">mdgsa</a></td>
	<td class=" ">David Montaner</td>
	<td class=" ">Multi Dimensional Gene Set Analysis.</td>
	<td class="  sorting_1">718</td>
</tr><tr id="pkg_pRolocGUI" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/pRolocGUI.html">pRolocGUI</a></td>
	<td class=" ">Laurent Gatto, Lisa M Breckels</td>
	<td class=" ">Interactive visualisation of spatial proteomics data</td>
	<td class="  sorting_1">718</td>
</tr><tr id="pkg_RUVnormalize" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/RUVnormalize.html">RUVnormalize</a></td>
	<td class=" ">Laurent Jacob</td>
	<td class=" ">RUV for normalization of expression array data</td>
	<td class="  sorting_1">718</td>
</tr><tr id="pkg_affyContam" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/affyContam.html">affyContam</a></td>
	<td class=" ">V. Carey</td>
	<td class=" ">structured corruption of affymetrix cel file data</td>
	<td class="  sorting_1">732</td>
</tr><tr id="pkg_biosigner" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/biosigner.html">biosigner</a></td>
	<td class=" ">Philippe Rinaudo, Etienne Thevenot</td>
	<td class=" ">Signature discovery from omics data</td>
	<td class="  sorting_1">732</td>
</tr><tr id="pkg_coseq" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/coseq.html">coseq</a></td>
	<td class=" ">Andrea Rau</td>
	<td class=" ">Co-Expression Analysis of Sequencing Data</td>
	<td class="  sorting_1">732</td>
</tr><tr id="pkg_GeneNetworkBuilder" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/GeneNetworkBuilder.html">GeneNetworkBuilder</a></td>
	<td class=" ">Jianhong Ou</td>
	<td class=" ">Build Regulatory Network from ChIP-chip/ChIP-seq and Expression Data</td>
	<td class="  sorting_1">732</td>
</tr><tr id="pkg_Metab" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/Metab.html">Metab</a></td>
	<td class=" ">Raphael Aggio</td>
	<td class=" ">Metab: An R Package for a High-Throughput Analysis of Metabolomics Data Generated by GC-MS.</td>
	<td class="  sorting_1">732</td>
</tr><tr id="pkg_MWASTools" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/MWASTools.html">MWASTools</a></td>
	<td class=" ">Andrea Rodriguez-Martinez, Rafael Ayala</td>
	<td class=" ">MWASTools: an integrated pipeline to perform metabolome-wide association studies</td>
	<td class="  sorting_1">732</td>
</tr><tr id="pkg_OSAT" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/OSAT.html">OSAT</a></td>
	<td class=" ">Li Yan</td>
	<td class=" ">OSAT: Optimal Sample Assignment Tool</td>
	<td class="  sorting_1">732</td>
</tr><tr id="pkg_PathoStat" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/PathoStat.html">PathoStat</a></td>
	<td class=" ">Solaiappan Manimaran, Yue Zhao</td>
	<td class=" ">PathoStat Statistical Microbiome Analysis Package</td>
	<td class="  sorting_1">732</td>
</tr><tr id="pkg_rcellminer" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/rcellminer.html">rcellminer</a></td>
	<td class=" ">Augustin Luna, Vinodh Rajapakse, Fathi Elloumi</td>
	<td class=" ">rcellminer: Molecular Profiles and Drug Response for the NCI-60 Cell Lines</td>
	<td class="  sorting_1">732</td>
</tr><tr id="pkg_roar" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/roar.html">roar</a></td>
	<td class=" ">Elena Grassi</td>
	<td class=" ">Identify differential APA usage from RNA-seq alignments</td>
	<td class="  sorting_1">732</td>
</tr><tr id="pkg_SBMLR" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/SBMLR.html">SBMLR</a></td>
	<td class=" ">Tomas Radivoyevitch</td>
	<td class=" ">SBML-R Interface and Analysis Tools</td>
	<td class="  sorting_1">732</td>
</tr><tr id="pkg_BioQC" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/BioQC.html">BioQC</a></td>
	<td class=" ">Jitao David Zhang</td>
	<td class=" ">Detect tissue heterogeneity in expression profiles with gene sets</td>
	<td class="  sorting_1">743</td>
</tr><tr id="pkg_ChIPComp" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/ChIPComp.html">ChIPComp</a></td>
	<td class=" ">Li Chen</td>
	<td class=" ">Quantitative comparison of multiple ChIP-seq datasets</td>
	<td class="  sorting_1">743</td>
</tr><tr id="pkg_epigenomix" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/epigenomix.html">epigenomix</a></td>
	<td class=" ">Hans-Ulrich Klein</td>
	<td class=" ">Epigenetic and gene transcription data normalization and integration with mixture models</td>
	<td class="  sorting_1">743</td>
</tr><tr id="pkg_miRLAB" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/miRLAB.html">miRLAB</a></td>
	<td class=" ">Thuc Duy Le</td>
	<td class=" ">Dry lab for exploring miRNA-mRNA relationships</td>
	<td class="  sorting_1">743</td>
</tr><tr id="pkg_plgem" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/plgem.html">plgem</a></td>
	<td class=" ">Norman Pavelka</td>
	<td class=" ">Detect differential expression in microarray and proteomics datasets with the Power Law Global Error Model (PLGEM)</td>
	<td class="  sorting_1">743</td>
</tr><tr id="pkg_ScISI" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/ScISI.html">ScISI</a></td>
	<td class=" ">Tony Chiang</td>
	<td class=" ">In Silico Interactome</td>
	<td class="  sorting_1">743</td>
</tr><tr id="pkg_SLqPCR" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/SLqPCR.html">SLqPCR</a></td>
	<td class=" ">Matthias Kohl</td>
	<td class=" ">Functions for analysis of real-time quantitative PCR data at SIRS-Lab GmbH</td>
	<td class="  sorting_1">743</td>
</tr><tr id="pkg_TargetScore" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/TargetScore.html">TargetScore</a></td>
	<td class=" ">Yue Li</td>
	<td class=" ">TargetScore: Infer microRNA targets using microRNA-overexpression data and sequence information</td>
	<td class="  sorting_1">743</td>
</tr><tr id="pkg_anamiR" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/anamiR.html">anamiR</a></td>
	<td class=" ">Ti-Tai Wang</td>
	<td class=" ">An integrated analysis package of miRNA and mRNA expression data</td>
	<td class="  sorting_1">751</td>
</tr><tr id="pkg_bioCancer" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/bioCancer.html">bioCancer</a></td>
	<td class=" ">Karim Mezhoud</td>
	<td class=" ">Interactive Multi-Omics Cancers Data Visualization and Analysis</td>
	<td class="  sorting_1">751</td>
</tr><tr id="pkg_DART" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/DART.html">DART</a></td>
	<td class=" ">Charles Shijie Zheng</td>
	<td class=" ">Denoising Algorithm based on Relevance network Topology</td>
	<td class="  sorting_1">751</td>
</tr><tr id="pkg_FunciSNP" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/FunciSNP.html">FunciSNP</a></td>
	<td class=" ">Simon G. Coetzee</td>
	<td class=" ">Integrating Functional Non-coding Datasets with Genetic Association Studies to Identify Candidate Regulatory SNPs</td>
	<td class="  sorting_1">751</td>
</tr><tr id="pkg_iPAC" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/iPAC.html">iPAC</a></td>
	<td class=" ">Gregory Ryslik</td>
	<td class=" ">Identification of Protein Amino acid Clustering</td>
	<td class="  sorting_1">751</td>
</tr><tr id="pkg_miRNApath" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/miRNApath.html">miRNApath</a></td>
	<td class=" ">James M. Ward</td>
	<td class=" ">miRNApath: Pathway Enrichment for miRNA Expression Data</td>
	<td class="  sorting_1">751</td>
</tr><tr id="pkg_OncoSimulR" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/OncoSimulR.html">OncoSimulR</a></td>
	<td class=" ">Ramon Diaz-Uriarte</td>
	<td class=" ">Forward Genetic Simulation of Cancer Progression with Epistasis</td>
	<td class="  sorting_1">751</td>
</tr><tr id="pkg_Pviz" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/Pviz.html">Pviz</a></td>
	<td class=" ">Renan Sauteraud</td>
	<td class=" ">Peptide Annotation and Data Visualization using Gviz</td>
	<td class="  sorting_1">751</td>
</tr><tr id="pkg_sizepower" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/sizepower.html">sizepower</a></td>
	<td class=" ">Weiliang Qiu</td>
	<td class=" ">Sample Size and Power Calculation in Micorarray Studies</td>
	<td class="  sorting_1">751</td>
</tr><tr id="pkg_soGGi" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/soGGi.html">soGGi</a></td>
	<td class=" ">Tom Carroll</td>
	<td class=" ">Visualise ChIP-seq, MNase-seq and motif occurrence as aggregate plots Summarised Over Grouped Genomic Intervals</td>
	<td class="  sorting_1">751</td>
</tr><tr id="pkg_YAPSA" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/YAPSA.html">YAPSA</a></td>
	<td class=" ">Daniel Huebschmann</td>
	<td class=" ">Yet Another Package for Signature Analysis</td>
	<td class="  sorting_1">751</td>
</tr><tr id="pkg_AMOUNTAIN" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/AMOUNTAIN.html">AMOUNTAIN</a></td>
	<td class=" ">Dong Li</td>
	<td class=" ">Active modules for multilayer weighted gene co-expression networks: a continuous optimization approach</td>
	<td class="  sorting_1">762</td>
</tr><tr id="pkg_BiocCaseStudies" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/BiocCaseStudies.html">BiocCaseStudies</a></td>
	<td class=" ">Bioconductor Package Maintainer</td>
	<td class=" ">BiocCaseStudies: Support for the Case Studies Monograph</td>
	<td class="  sorting_1">762</td>
</tr><tr id="pkg_canceR" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/canceR.html">canceR</a></td>
	<td class=" ">Karim Mezhoud</td>
	<td class=" ">A Graphical User Interface for accessing and modeling the Cancer Genomics Data of MSKCC</td>
	<td class="  sorting_1">762</td>
</tr><tr id="pkg_CausalR" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/CausalR.html">CausalR</a></td>
	<td class=" ">Glyn Bradley, Steven Barrett</td>
	<td class=" ">Causal network analysis methods</td>
	<td class="  sorting_1">762</td>
</tr><tr id="pkg_cellity" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/cellity.html">cellity</a></td>
	<td class=" ">Tomislav Ilicic</td>
	<td class=" ">Quality Control for Single-Cell RNA-seq Data</td>
	<td class="  sorting_1">762</td>
</tr><tr id="pkg_clippda" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/clippda.html">clippda</a></td>
	<td class=" ">Stephen Nyangoma</td>
	<td class=" ">A package for the clinical proteomic profiling data analysis</td>
	<td class="  sorting_1">762</td>
</tr><tr id="pkg_ctsGE" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/ctsGE.html">ctsGE</a></td>
	<td class=" ">Michal Sharabi-Schwager</td>
	<td class=" ">Clustering of Time Series Gene Expression data</td>
	<td class="  sorting_1">762</td>
</tr><tr id="pkg_cydar" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/cydar.html">cydar</a></td>
	<td class=" ">Aaron Lun</td>
	<td class=" ">Using Mass Cytometry for Differential Abundance Analyses</td>
	<td class="  sorting_1">762</td>
</tr><tr id="pkg_esetVis" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/esetVis.html">esetVis</a></td>
	<td class=" ">Laure Cougnaud</td>
	<td class=" ">Visualizations of expressionSet Bioconductor object</td>
	<td class="  sorting_1">762</td>
</tr><tr id="pkg_FamAgg" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/FamAgg.html">FamAgg</a></td>
	<td class=" ">Johannes Rainer</td>
	<td class=" ">Pedigree Analysis and Familial Aggregation</td>
	<td class="  sorting_1">762</td>
</tr><tr id="pkg_GeneExpressionSignature" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/GeneExpressionSignature.html">GeneExpressionSignature</a></td>
	<td class=" ">Yang Cao, Fei Li,Lu Han</td>
	<td class=" ">Gene Expression Signature based Similarity Metric</td>
	<td class="  sorting_1">762</td>
</tr><tr id="pkg_HiCcompare" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/HiCcompare.html">HiCcompare</a></td>
	<td class=" ">John Stansfield</td>
	<td class=" ">HiCcompare: Joint normalization and comparative analysis of multiple Hi-C datasets</td>
	<td class="  sorting_1">762</td>
</tr><tr id="pkg_HybridMTest" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/HybridMTest.html">HybridMTest</a></td>
	<td class=" ">Demba Fofana</td>
	<td class=" ">Hybrid Multiple Testing</td>
	<td class="  sorting_1">762</td>
</tr><tr id="pkg_intansv" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/intansv.html">intansv</a></td>
	<td class=" ">Wen Yao</td>
	<td class=" ">Integrative analysis of structural variations</td>
	<td class="  sorting_1">762</td>
</tr><tr id="pkg_PathNet" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/PathNet.html">PathNet</a></td>
	<td class=" ">Jason B. Smith</td>
	<td class=" ">An R package for pathway analysis using topological information</td>
	<td class="  sorting_1">762</td>
</tr><tr id="pkg_Pbase" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/Pbase.html">Pbase</a></td>
	<td class=" ">Sebastian Gibb, Laurent Gatto</td>
	<td class=" ">Manipulating and exploring protein and proteomics data</td>
	<td class="  sorting_1">762</td>
</tr><tr id="pkg_SNPediaR" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/SNPediaR.html">SNPediaR</a></td>
	<td class=" ">David Montaner</td>
	<td class=" ">Query data from SNPedia</td>
	<td class="  sorting_1">762</td>
</tr><tr id="pkg_tracktables" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/tracktables.html">tracktables</a></td>
	<td class=" ">Tom Carroll</td>
	<td class=" ">Build IGV tracks and HTML reports</td>
	<td class="  sorting_1">762</td>
</tr><tr id="pkg_affyILM" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/affyILM.html">affyILM</a></td>
	<td class=" ">Myriam Kroll and Fabrice Berger</td>
	<td class=" ">Linear Model of background subtraction and the Langmuir isotherm</td>
	<td class="  sorting_1">780</td>
</tr><tr id="pkg_AffyRNADegradation" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/AffyRNADegradation.html">AffyRNADegradation</a></td>
	<td class=" ">Mario Fasold</td>
	<td class=" ">Analyze and correct probe positional bias in microarray data due to RNA degradation</td>
	<td class="  sorting_1">780</td>
</tr><tr id="pkg_clst" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/clst.html">clst</a></td>
	<td class=" ">Noah Hoffman</td>
	<td class=" ">Classification by local similarity threshold</td>
	<td class="  sorting_1">780</td>
</tr><tr id="pkg_CoCiteStats" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/CoCiteStats.html">CoCiteStats</a></td>
	<td class=" ">Bioconductor Package Maintainer</td>
	<td class=" ">Different test statistics based on co-citation.</td>
	<td class="  sorting_1">780</td>
</tr><tr id="pkg_cosmiq" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/cosmiq.html">cosmiq</a></td>
	<td class=" ">David Fischer, Christian Panse</td>
	<td class=" ">cosmiq - COmbining Single Masses Into Quantities</td>
	<td class="  sorting_1">780</td>
</tr><tr id="pkg_DEsubs" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/DEsubs.html">DEsubs</a></td>
	<td class=" ">Aristidis G. Vrahatis, Panos Balomenos</td>
	<td class=" ">DEsubs: an R package for flexible identification of differentially expressed subpathways using RNA-seq expression experiments</td>
	<td class="  sorting_1">780</td>
</tr><tr id="pkg_gmapR" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/gmapR.html">gmapR</a></td>
	<td class=" ">Michael Lawrence</td>
	<td class=" ">An R interface to the GMAP/GSNAP/GSTRUCT suite</td>
	<td class="  sorting_1">780</td>
</tr><tr id="pkg_maigesPack" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/maigesPack.html">maigesPack</a></td>
	<td class=" ">Gustavo H. Esteves</td>
	<td class=" ">Functions to handle cDNA microarray data, including several methods of data analysis</td>
	<td class="  sorting_1">780</td>
</tr><tr id="pkg_MineICA" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/MineICA.html">MineICA</a></td>
	<td class=" ">Anne Biton</td>
	<td class=" ">Analysis of an ICA decomposition obtained on genomics data</td>
	<td class="  sorting_1">780</td>
</tr><tr id="pkg_ppiStats" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/ppiStats.html">ppiStats</a></td>
	<td class=" ">Bioconductor Package Maintainer</td>
	<td class=" ">Protein-Protein Interaction Statistical Package</td>
	<td class="  sorting_1">780</td>
</tr><tr id="pkg_seqbias" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/seqbias.html">seqbias</a></td>
	<td class=" ">Daniel Jones</td>
	<td class=" ">Estimation of per-position bias in high-throughput sequencing data</td>
	<td class="  sorting_1">780</td>
</tr><tr id="pkg_sigaR" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/sigaR.html">sigaR</a></td>
	<td class=" ">Wessel N. van Wieringen</td>
	<td class=" ">Statistics for Integrative Genomics Analyses in R</td>
	<td class="  sorting_1">780</td>
</tr><tr id="pkg_SPEM" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/SPEM.html">SPEM</a></td>
	<td class=" ">Xinyi YANG</td>
	<td class=" ">S-system parameter estimation method</td>
	<td class="  sorting_1">780</td>
</tr><tr id="pkg_tspair" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/tspair.html">tspair</a></td>
	<td class=" ">Jeffrey T. Leek</td>
	<td class=" ">Top Scoring Pairs for Microarray Classification</td>
	<td class="  sorting_1">780</td>
</tr><tr id="pkg_AGDEX" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/AGDEX.html">AGDEX</a></td>
	<td class=" ">Cuilan lani Gao</td>
	<td class=" ">Agreement of Differential Expression Analysis</td>
	<td class="  sorting_1">794</td>
</tr><tr id="pkg_ASGSCA" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/ASGSCA.html">ASGSCA</a></td>
	<td class=" ">Hela Romdhani</td>
	<td class=" ">Association Studies for multiple SNPs and multiple traits using Generalized Structured Equation Models</td>
	<td class="  sorting_1">794</td>
</tr><tr id="pkg_BEclear" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/BEclear.html">BEclear</a></td>
	<td class=" ">David Rasp</td>
	<td class=" ">Correct for batch effects in DNA methylation data</td>
	<td class="  sorting_1">794</td>
</tr><tr id="pkg_birta" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/birta.html">birta</a></td>
	<td class=" ">Benedikt Zacher, Holger Froehlich</td>
	<td class=" ">Bayesian Inference of Regulation of Transcriptional Activity</td>
	<td class="  sorting_1">794</td>
</tr><tr id="pkg_BufferedMatrixMethods" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/BufferedMatrixMethods.html">BufferedMatrixMethods</a></td>
	<td class=" ">Ben Bolstad</td>
	<td class=" ">Microarray Data related methods that utlize BufferedMatrix objects</td>
	<td class="  sorting_1">794</td>
</tr><tr id="pkg_cancerclass" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/cancerclass.html">cancerclass</a></td>
	<td class=" ">Daniel Kosztyla</td>
	<td class=" ">Development and validation of diagnostic tests from high-dimensional molecular data</td>
	<td class="  sorting_1">794</td>
</tr><tr id="pkg_cn.farms" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/cn.farms.html">cn.farms</a></td>
	<td class=" ">Andreas Mitterecker</td>
	<td class=" ">cn.FARMS - factor analysis for copy number estimation</td>
	<td class="  sorting_1">794</td>
</tr><tr id="pkg_CNORdt" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/CNORdt.html">CNORdt</a></td>
	<td class=" ">A. MacNamara</td>
	<td class=" ">Add-on to CellNOptR: Discretized time treatments</td>
	<td class="  sorting_1">794</td>
</tr><tr id="pkg_copa" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/copa.html">copa</a></td>
	<td class=" ">James W. MacDonald</td>
	<td class=" ">Functions to perform cancer outlier profile analysis.</td>
	<td class="  sorting_1">794</td>
</tr><tr id="pkg_cycle" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/cycle.html">cycle</a></td>
	<td class=" ">Matthias Futschik</td>
	<td class=" ">Significance of periodic expression pattern in time-series data</td>
	<td class="  sorting_1">794</td>
</tr><tr id="pkg_FlowRepositoryR" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/FlowRepositoryR.html">FlowRepositoryR</a></td>
	<td class=" ">Josef Spidlen</td>
	<td class=" ">FlowRepository R Interface</td>
	<td class="  sorting_1">794</td>
</tr><tr id="pkg_gaga" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/gaga.html">gaga</a></td>
	<td class=" ">David Rossell</td>
	<td class=" ">GaGa hierarchical model for high-throughput data analysis</td>
	<td class="  sorting_1">794</td>
</tr><tr id="pkg_ImmuneSpaceR" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/ImmuneSpaceR.html">ImmuneSpaceR</a></td>
	<td class=" ">ImmuneSpace Package Maintainer</td>
	<td class=" ">A Thin Wrapper around the ImmuneSpace Database</td>
	<td class="  sorting_1">794</td>
</tr><tr id="pkg_PREDA" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/PREDA.html">PREDA</a></td>
	<td class=" ">Francesco Ferrari</td>
	<td class=" ">Position Related Data Analysis</td>
	<td class="  sorting_1">794</td>
</tr><tr id="pkg_qpcrNorm" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/qpcrNorm.html">qpcrNorm</a></td>
	<td class=" ">Jessica Mar</td>
	<td class=" ">Data-driven normalization strategies for high-throughput qPCR data.</td>
	<td class="  sorting_1">794</td>
</tr><tr id="pkg_Rchemcpp" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/Rchemcpp.html">Rchemcpp</a></td>
	<td class=" ">Guenter Klambauer</td>
	<td class=" ">Similarity measures for chemical compounds</td>
	<td class="  sorting_1">794</td>
</tr><tr id="pkg_restfulSE" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/restfulSE.html">restfulSE</a></td>
	<td class=" ">Shweta Gopaulakrishnan</td>
	<td class=" ">Access matrix-like HDF5 server content or BigQuery content through a SummarizedExperiment interface</td>
	<td class="  sorting_1">794</td>
</tr><tr id="pkg_rTRM" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/rTRM.html">rTRM</a></td>
	<td class=" ">Diego Diez</td>
	<td class=" ">Identification of transcriptional regulatory modules from PPI networks</td>
	<td class="  sorting_1">794</td>
</tr><tr id="pkg_antiProfiles" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/antiProfiles.html">antiProfiles</a></td>
	<td class=" ">Hector Corrada Bravo</td>
	<td class=" ">Implementation of gene expression anti-profiles</td>
	<td class="  sorting_1">812</td>
</tr><tr id="pkg_ASSIGN" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/ASSIGN.html">ASSIGN</a></td>
	<td class=" ">Ying Shen, W. Evan Johnson, David Jenkins, Mumtehena Rahman</td>
	<td class=" ">Adaptive Signature Selection and InteGratioN (ASSIGN)</td>
	<td class="  sorting_1">812</td>
</tr><tr id="pkg_BADER" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/BADER.html">BADER</a></td>
	<td class=" ">Andreas Neudecker</td>
	<td class=" ">Bayesian Analysis of Differential Expression in RNA Sequencing Data</td>
	<td class="  sorting_1">812</td>
</tr><tr id="pkg_cellGrowth" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/cellGrowth.html">cellGrowth</a></td>
	<td class=" ">Julien Gagneur</td>
	<td class=" ">Fitting cell population growth models</td>
	<td class="  sorting_1">812</td>
</tr><tr id="pkg_cellscape" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/cellscape.html">cellscape</a></td>
	<td class=" ">Maia Smith</td>
	<td class=" ">Explores single cell copy number profiles in the context of a single cell tree</td>
	<td class="  sorting_1">812</td>
</tr><tr id="pkg_cisPath" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/cisPath.html">cisPath</a></td>
	<td class=" ">Likun Wang</td>
	<td class=" ">Visualization and management of the protein-protein interaction networks.</td>
	<td class="  sorting_1">812</td>
</tr><tr id="pkg_cleanUpdTSeq" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/cleanUpdTSeq.html">cleanUpdTSeq</a></td>
	<td class=" ">Sarah Sheppard; Jianhong Ou; Lihua Julie Zhu</td>
	<td class=" ">This package classifies putative polyadenylation sites as true or false/internally oligodT primed</td>
	<td class="  sorting_1">812</td>
</tr><tr id="pkg_CNORfuzzy" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/CNORfuzzy.html">CNORfuzzy</a></td>
	<td class=" ">T. Cokelaer</td>
	<td class=" ">Addon to CellNOptR: Fuzzy Logic</td>
	<td class="  sorting_1">812</td>
</tr><tr id="pkg_CNVrd2" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/CNVrd2.html">CNVrd2</a></td>
	<td class=" ">Hoang Tan Nguyen</td>
	<td class=" ">CNVrd2: a read depth-based method to detect and genotype complex common copy number variants from next generation sequencing data.</td>
	<td class="  sorting_1">812</td>
</tr><tr id="pkg_CorMut" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/CorMut.html">CorMut</a></td>
	<td class=" ">Zhenpeng Li</td>
	<td class=" ">Detect the correlated mutations based on selection pressure</td>
	<td class="  sorting_1">812</td>
</tr><tr id="pkg_DaMiRseq" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/DaMiRseq.html">DaMiRseq</a></td>
	<td class=" ">Mattia Chiesa</td>
	<td class=" ">Data Mining for RNA-seq data: normalization, feature selection and classification</td>
	<td class="  sorting_1">812</td>
</tr><tr id="pkg_DeMAND" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/DeMAND.html">DeMAND</a></td>
	<td class=" ">Jung Hoon Woo, Mariano Alvarez</td>
	<td class=" ">DeMAND</td>
	<td class="  sorting_1">812</td>
</tr><tr id="pkg_dexus" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/dexus.html">dexus</a></td>
	<td class=" ">Guenter Klambauer</td>
	<td class=" ">DEXUS - Identifying Differential Expression in RNA-Seq Studies with Unknown Conditions or without Replicates</td>
	<td class="  sorting_1">812</td>
</tr><tr id="pkg_EBcoexpress" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/EBcoexpress.html">EBcoexpress</a></td>
	<td class=" ">John A. Dawson</td>
	<td class=" ">EBcoexpress for Differential Co-Expression Analysis</td>
	<td class="  sorting_1">812</td>
</tr><tr id="pkg_EGAD" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/EGAD.html">EGAD</a></td>
	<td class=" ">Sara Ballouz</td>
	<td class=" ">Extending guilt by association by degree</td>
	<td class="  sorting_1">812</td>
</tr><tr id="pkg_factDesign" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/factDesign.html">factDesign</a></td>
	<td class=" ">Denise Scholtens</td>
	<td class=" ">Factorial designed microarray experiment analysis</td>
	<td class="  sorting_1">812</td>
</tr><tr id="pkg_genArise" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/genArise.html">genArise</a></td>
	<td class=" ">IFC Development Team</td>
	<td class=" ">Microarray Analysis tool</td>
	<td class="  sorting_1">812</td>
</tr><tr id="pkg_GeneSelectMMD" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/GeneSelectMMD.html">GeneSelectMMD</a></td>
	<td class=" ">Weiliang Qiu</td>
	<td class=" ">Gene selection based on the marginal distributions of gene profiles that characterized by a mixture of three-component multivariate distributions</td>
	<td class="  sorting_1">812</td>
</tr><tr id="pkg_GUIDEseq" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/GUIDEseq.html">GUIDEseq</a></td>
	<td class=" ">Lihua Julie Zhu</td>
	<td class=" ">GUIDE-seq analysis pipeline</td>
	<td class="  sorting_1">812</td>
</tr><tr id="pkg_hicrep" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/hicrep.html">hicrep</a></td>
	<td class=" ">Tao Yang</td>
	<td class=" ">Measuring the reproducibility of Hi-C data</td>
	<td class="  sorting_1">812</td>
</tr><tr id="pkg_lmdme" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/lmdme.html">lmdme</a></td>
	<td class=" ">Cristobal Fresno</td>
	<td class=" ">Linear Model decomposition for Designed Multivariate Experiments</td>
	<td class="  sorting_1">812</td>
</tr><tr id="pkg_metaSeq" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/metaSeq.html">metaSeq</a></td>
	<td class=" ">Koki Tsuyuzaki</td>
	<td class=" ">Meta-analysis of RNA-Seq count data in multiple studies</td>
	<td class="  sorting_1">812</td>
</tr><tr id="pkg_MethylAid" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/MethylAid.html">MethylAid</a></td>
	<td class=" ">M. van Iterson</td>
	<td class=" ">Visual and interactive quality control of large Illumina DNA Methylation array data sets</td>
	<td class="  sorting_1">812</td>
</tr><tr id="pkg_nondetects" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/nondetects.html">nondetects</a></td>
	<td class=" ">Valeriia Sherina</td>
	<td class=" ">Non-detects in qPCR data</td>
	<td class="  sorting_1">812</td>
</tr><tr id="pkg_qcmetrics" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/qcmetrics.html">qcmetrics</a></td>
	<td class=" ">Laurent Gatto</td>
	<td class=" ">A Framework for Quality Control</td>
	<td class="  sorting_1">812</td>
</tr><tr id="pkg_RBioinf" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/RBioinf.html">RBioinf</a></td>
	<td class=" ">Robert Gentleman</td>
	<td class=" ">RBioinf</td>
	<td class="  sorting_1">812</td>
</tr><tr id="pkg_RCyjs" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/RCyjs.html">RCyjs</a></td>
	<td class=" ">Paul Shannon</td>
	<td class=" ">Display and manipulate graphs in cytoscape.js</td>
	<td class="  sorting_1">812</td>
</tr><tr id="pkg_STATegRa" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/STATegRa.html">STATegRa</a></td>
	<td class=" ">David Gomez-Cabrero, Nria Planell</td>
	<td class=" ">Classes and methods for multi-omics data integration</td>
	<td class="  sorting_1">812</td>
</tr><tr id="pkg_switchBox" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/switchBox.html">switchBox</a></td>
	<td class=" ">Bahman Afsari, Luigi Marchionni, Wikum Dinalankara</td>
	<td class=" ">Utilities to train and validate classifiers based on pair switching using the K-Top-Scoring-Pair (KTSP) algorithm</td>
	<td class="  sorting_1">812</td>
</tr><tr id="pkg_ARRmNormalization" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/ARRmNormalization.html">ARRmNormalization</a></td>
	<td class=" ">Jean-Philippe Fortin</td>
	<td class=" ">Adaptive Robust Regression normalization for Illumina methylation data</td>
	<td class="  sorting_1">841</td>
</tr><tr id="pkg_ASEB" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/ASEB.html">ASEB</a></td>
	<td class=" ">Likun Wang</td>
	<td class=" ">Predict Acetylated Lysine Sites</td>
	<td class="  sorting_1">841</td>
</tr><tr id="pkg_bgx" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/bgx.html">bgx</a></td>
	<td class=" ">Ernest Turro</td>
	<td class=" ">Bayesian Gene eXpression</td>
	<td class="  sorting_1">841</td>
</tr><tr id="pkg_BubbleTree" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/BubbleTree.html">BubbleTree</a></td>
	<td class=" ">Todd Creasy, Wei Zhu</td>
	<td class=" ">BubbleTree: an intuitive visualization to elucidate tumoral aneuploidy and clonality in somatic mosaicism using next generation sequencing data</td>
	<td class="  sorting_1">841</td>
</tr><tr id="pkg_chromDraw" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/chromDraw.html">chromDraw</a></td>
	<td class=" ">Jan Janecka</td>
	<td class=" ">chromDraw is a R package for drawing the schemes of karyotypes in the linear and circular fashion.</td>
	<td class="  sorting_1">841</td>
</tr><tr id="pkg_EDDA" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/EDDA.html">EDDA</a></td>
	<td class=" ">Chia Kuan Hui Burton, Niranjan Nagarajan</td>
	<td class=" ">Experimental Design in Differential Abundance analysis</td>
	<td class="  sorting_1">841</td>
</tr><tr id="pkg_epivizrStandalone" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/epivizrStandalone.html">epivizrStandalone</a></td>
	<td class=" ">Hector Corrada Bravo</td>
	<td class=" ">Run Epiviz Interactive Genomic Data Visualization App within R</td>
	<td class="  sorting_1">841</td>
</tr><tr id="pkg_ExiMiR" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/ExiMiR.html">ExiMiR</a></td>
	<td class=" ">Sylvain Gubian</td>
	<td class=" ">R functions for the normalization of Exiqon miRNA array data</td>
	<td class="  sorting_1">841</td>
</tr><tr id="pkg_flagme" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/flagme.html">flagme</a></td>
	<td class=" ">Mark Robinson, Riccardo Romoli</td>
	<td class=" ">Analysis of Metabolomics GC/MS Data</td>
	<td class="  sorting_1">841</td>
</tr><tr id="pkg_GraphAlignment" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/GraphAlignment.html">GraphAlignment</a></td>
	<td class=" ">Joern P. Meier</td>
	<td class=" ">GraphAlignment</td>
	<td class="  sorting_1">841</td>
</tr><tr id="pkg_h5vc" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/h5vc.html">h5vc</a></td>
	<td class=" ">Paul Theodor Pyl</td>
	<td class=" ">Managing alignment tallies using a hdf5 backend</td>
	<td class="  sorting_1">841</td>
</tr><tr id="pkg_imageHTS" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/imageHTS.html">imageHTS</a></td>
	<td class=" ">Joseph Barry</td>
	<td class=" ">Analysis of high-throughput microscopy-based screens</td>
	<td class="  sorting_1">841</td>
</tr><tr id="pkg_InterMineR" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/InterMineR.html">InterMineR</a></td>
	<td class=" ">InterMine Team</td>
	<td class=" ">R Interface with InterMine-Powered Databases</td>
	<td class="  sorting_1">841</td>
</tr><tr id="pkg_LymphoSeq" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/LymphoSeq.html">LymphoSeq</a></td>
	<td class=" ">David Coffey</td>
	<td class=" ">Analyze high-throughput sequencing of T and B cell receptors</td>
	<td class="  sorting_1">841</td>
</tr><tr id="pkg_MoonlightR" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/MoonlightR.html">MoonlightR</a></td>
	<td class=" ">Antonio Colaprico, Catharina Olsen</td>
	<td class=" ">Identify oncogenes and tumor suppressor genes from omics data</td>
	<td class="  sorting_1">841</td>
</tr><tr id="pkg_netbenchmark" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/netbenchmark.html">netbenchmark</a></td>
	<td class=" ">Pau Bellot</td>
	<td class=" ">Benchmarking of several gene network inference methods</td>
	<td class="  sorting_1">841</td>
</tr><tr id="pkg_netresponse" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/netresponse.html">netresponse</a></td>
	<td class=" ">Leo Lahti</td>
	<td class=" ">Functional Network Analysis</td>
	<td class="  sorting_1">841</td>
</tr><tr id="pkg_networkBMA" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/networkBMA.html">networkBMA</a></td>
	<td class=" ">Ka Yee Yeung</td>
	<td class=" ">Regression-based network inference using Bayesian Model Averaging</td>
	<td class="  sorting_1">841</td>
</tr><tr id="pkg_Path2PPI" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/Path2PPI.html">Path2PPI</a></td>
	<td class=" ">Oliver Philipp</td>
	<td class=" ">Prediction of pathway-related protein-protein interaction networks</td>
	<td class="  sorting_1">841</td>
</tr><tr id="pkg_plateCore" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/plateCore.html">plateCore</a></td>
	<td class=" ">Errol Strain</td>
	<td class=" ">Statistical tools and data structures for plate-based flow cytometry</td>
	<td class="  sorting_1">841</td>
</tr><tr id="pkg_psygenet2r" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/psygenet2r.html">psygenet2r</a></td>
	<td class=" ">Alba Gutierrez-Sacristan</td>
	<td class=" ">psygenet2r - An R package for querying PsyGeNET and to perform comorbidity studies in psychiatric disorders</td>
	<td class="  sorting_1">841</td>
</tr><tr id="pkg_QUALIFIER" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/QUALIFIER.html">QUALIFIER</a></td>
	<td class=" ">Mike Jiang</td>
	<td class=" ">Quality Control of Gated Flow Cytometry Experiments</td>
	<td class="  sorting_1">841</td>
</tr><tr id="pkg_Risa" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/Risa.html">Risa</a></td>
	<td class=" ">Alejandra Gonzalez-Beltran</td>
	<td class=" ">Converting experimental metadata from ISA-tab into Bioconductor data structures</td>
	<td class="  sorting_1">841</td>
</tr><tr id="pkg_RUVcorr" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/RUVcorr.html">RUVcorr</a></td>
	<td class=" ">Saskia Freytag</td>
	<td class=" ">Removal of unwanted variation for gene-gene correlations and related analysis</td>
	<td class="  sorting_1">841</td>
</tr><tr id="pkg_seq2pathway" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/seq2pathway.html">seq2pathway</a></td>
	<td class=" ">Xinan Yang with contribution from Lorenzo Pesce and Ana Marija Sokovic</td>
	<td class=" ">a novel tool for functional gene-set (or termed as pathway) analysis of next-generation sequencing data</td>
	<td class="  sorting_1">841</td>
</tr><tr id="pkg_XBSeq" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/XBSeq.html">XBSeq</a></td>
	<td class=" ">Yuanhang Liu</td>
	<td class=" ">Test for differential expression for RNA-seq data</td>
	<td class="  sorting_1">841</td>
</tr><tr id="pkg_xps" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/xps.html">xps</a></td>
	<td class=" ">Christian Stratowa</td>
	<td class=" ">Processing and Analysis of Affymetrix Oligonucleotide Arrays including Exon Arrays, Whole Genome Arrays and Plate Arrays</td>
	<td class="  sorting_1">841</td>
</tr><tr id="pkg_BAC" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/BAC.html">BAC</a></td>
	<td class=" ">Raphael Gottardo</td>
	<td class=" ">Bayesian Analysis of Chip-chip experiment</td>
	<td class="  sorting_1">868</td>
</tr><tr id="pkg_blima" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/blima.html">blima</a></td>
	<td class=" ">Vojtch Kulvait</td>
	<td class=" ">Tools for the preprocessing and analysis of the Illumina microarrays on the detector (bead) level</td>
	<td class="  sorting_1">868</td>
</tr><tr id="pkg_categoryCompare" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/categoryCompare.html">categoryCompare</a></td>
	<td class=" ">Robert M. Flight</td>
	<td class=" ">Meta-analysis of high-throughput experiments using feature annotations</td>
	<td class="  sorting_1">868</td>
</tr><tr id="pkg_Clonality" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/Clonality.html">Clonality</a></td>
	<td class=" ">Irina Ostrovnaya</td>
	<td class=" ">Clonality testing</td>
	<td class="  sorting_1">868</td>
</tr><tr id="pkg_ClusterSignificance" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/ClusterSignificance.html">ClusterSignificance</a></td>
	<td class=" ">Jason T Serviss</td>
	<td class=" ">The ClusterSignificance package provides tools to assess if class clusters in dimensionality reduced data representations have a separation different from permuted data</td>
	<td class="  sorting_1">868</td>
</tr><tr id="pkg_CNORfeeder" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/CNORfeeder.html">CNORfeeder</a></td>
	<td class=" ">F.Eduati</td>
	<td class=" ">Integration of CellNOptR to add missing links</td>
	<td class="  sorting_1">868</td>
</tr><tr id="pkg_dyebias" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/dyebias.html">dyebias</a></td>
	<td class=" ">Philip Lijnzaad</td>
	<td class=" ">The GASSCO method for correcting for slide-dependent gene-specific dye bias</td>
	<td class="  sorting_1">868</td>
</tr><tr id="pkg_FRGEpistasis" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/FRGEpistasis.html">FRGEpistasis</a></td>
	<td class=" ">Futao Zhang</td>
	<td class=" ">Epistasis Analysis for Quantitative Traits by Functional Regression Model</td>
	<td class="  sorting_1">868</td>
</tr><tr id="pkg_GoogleGenomics" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/GoogleGenomics.html">GoogleGenomics</a></td>
	<td class=" ">Siddhartha Bagaria</td>
	<td class=" ">R Client for Google Genomics API</td>
	<td class="  sorting_1">868</td>
</tr><tr id="pkg_hapFabia" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/hapFabia.html">hapFabia</a></td>
	<td class=" ">Sepp Hochreiter</td>
	<td class=" ">hapFabia: Identification of very short segments of identity by descent (IBD) characterized by rare variants in large sequencing data</td>
	<td class="  sorting_1">868</td>
</tr><tr id="pkg_ideal" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/ideal.html">ideal</a></td>
	<td class=" ">Federico Marini</td>
	<td class=" ">Interactive Differential Expression AnaLysis</td>
	<td class="  sorting_1">868</td>
</tr><tr id="pkg_metabomxtr" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/metabomxtr.html">metabomxtr</a></td>
	<td class=" ">Michael Nodzenski</td>
	<td class=" ">A package to run mixture models for truncated metabolomics data with normal or lognormal distributions</td>
	<td class="  sorting_1">868</td>
</tr><tr id="pkg_NarrowPeaks" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/NarrowPeaks.html">NarrowPeaks</a></td>
	<td class=" ">Pedro Madrigal</td>
	<td class=" ">Shape-based Analysis of Variation in ChIP-seq using Functional PCA</td>
	<td class="  sorting_1">868</td>
</tr><tr id="pkg_npGSEA" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/npGSEA.html">npGSEA</a></td>
	<td class=" ">Jessica Larson</td>
	<td class=" ">Permutation approximation methods for gene set enrichment analysis (non-permutation GSEA)</td>
	<td class="  sorting_1">868</td>
</tr><tr id="pkg_PAPi" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/PAPi.html">PAPi</a></td>
	<td class=" ">Raphael Aggio</td>
	<td class=" ">Predict metabolic pathway activity based on metabolomics data</td>
	<td class="  sorting_1">868</td>
</tr><tr id="pkg_PECA" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/PECA.html">PECA</a></td>
	<td class=" ">Tomi Suomi</td>
	<td class=" ">Probe-level Expression Change Averaging</td>
	<td class="  sorting_1">868</td>
</tr><tr id="pkg_proBAMr" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/proBAMr.html">proBAMr</a></td>
	<td class=" ">Xiaojing Wang</td>
	<td class=" ">Generating SAM file for PSMs in shotgun proteomics data</td>
	<td class="  sorting_1">868</td>
</tr><tr id="pkg_RchyOptimyx" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/RchyOptimyx.html">RchyOptimyx</a></td>
	<td class=" ">Adrin Jalali, Nima Aghaeepour</td>
	<td class=" ">Optimyzed Cellular Hierarchies for Flow Cytometry</td>
	<td class="  sorting_1">868</td>
</tr><tr id="pkg_RGalaxy" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/RGalaxy.html">RGalaxy</a></td>
	<td class=" ">Bioconductor Package Maintainer</td>
	<td class=" ">Make an R function available in the Galaxy web platform</td>
	<td class="  sorting_1">868</td>
</tr><tr id="pkg_Anaquin" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/Anaquin.html">Anaquin</a></td>
	<td class=" ">Ted Wong</td>
	<td class=" ">Statistical analysis of sequins</td>
	<td class="  sorting_1">887</td>
</tr><tr id="pkg_attract" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/attract.html">attract</a></td>
	<td class=" ">Samuel Zimmerman</td>
	<td class=" ">Methods to Find the Gene Expression Modules that Represent the Drivers of Kauffman's Attractor Landscape</td>
	<td class="  sorting_1">887</td>
</tr><tr id="pkg_BioCor" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/BioCor.html">BioCor</a></td>
	<td class=" ">Llus Revilla Sancho</td>
	<td class=" ">Functional similarities</td>
	<td class="  sorting_1">887</td>
</tr><tr id="pkg_biosvd" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/biosvd.html">biosvd</a></td>
	<td class=" ">Anneleen Daemen, Matthew Brauer</td>
	<td class=" ">Package for high-throughput data processing, outlier detection, noise removal and dynamic modeling</td>
	<td class="  sorting_1">887</td>
</tr><tr id="pkg_CFAssay" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/CFAssay.html">CFAssay</a></td>
	<td class=" ">Herbert Braselmann</td>
	<td class=" ">Statistical analysis for the Colony Formation Assay</td>
	<td class="  sorting_1">887</td>
</tr><tr id="pkg_CHRONOS" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/CHRONOS.html">CHRONOS</a></td>
	<td class=" ">Panos Balomenos</td>
	<td class=" ">CHRONOS: A time-varying method for microRNA-mediated sub-pathway enrichment analysis</td>
	<td class="  sorting_1">887</td>
</tr><tr id="pkg_clonotypeR" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/clonotypeR.html">clonotypeR</a></td>
	<td class=" ">Charles Plessy</td>
	<td class=" ">High throughput analysis of T cell antigen receptor sequences</td>
	<td class="  sorting_1">887</td>
</tr><tr id="pkg_DiffLogo" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/DiffLogo.html">DiffLogo</a></td>
	<td class=" ">Hendrik Treutler</td>
	<td class=" ">DiffLogo: A comparative visualisation of biooligomer motifs</td>
	<td class="  sorting_1">887</td>
</tr><tr id="pkg_diggit" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/diggit.html">diggit</a></td>
	<td class=" ">Mariano J Alvarez</td>
	<td class=" ">Inference of Genetic Variants Driving Cellular Phenotypes</td>
	<td class="  sorting_1">887</td>
</tr><tr id="pkg_DMRforPairs" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/DMRforPairs.html">DMRforPairs</a></td>
	<td class=" ">Martin Rijlaarsdam</td>
	<td class=" ">DMRforPairs: identifying Differentially Methylated Regions between unique samples using array based methylation profiles</td>
	<td class="  sorting_1">887</td>
</tr><tr id="pkg_ecolitk" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/ecolitk.html">ecolitk</a></td>
	<td class=" ">Laurent Gautier</td>
	<td class=" ">Meta-data and tools for E. coli</td>
	<td class="  sorting_1">887</td>
</tr><tr id="pkg_flowQ" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/flowQ.html">flowQ</a></td>
	<td class=" ">Mike Jiang</td>
	<td class=" ">Quality control for flow cytometry</td>
	<td class="  sorting_1">887</td>
</tr><tr id="pkg_gaggle" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/gaggle.html">gaggle</a></td>
	<td class=" ">Christopher Bare</td>
	<td class=" ">Broadcast data between R and Gaggle</td>
	<td class="  sorting_1">887</td>
</tr><tr id="pkg_geneRecommender" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/geneRecommender.html">geneRecommender</a></td>
	<td class=" ">Greg Hather</td>
	<td class=" ">A gene recommender algorithm to identify genes coexpressed with a query set of genes</td>
	<td class="  sorting_1">887</td>
</tr><tr id="pkg_GraphPAC" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/GraphPAC.html">GraphPAC</a></td>
	<td class=" ">Gregory Ryslik</td>
	<td class=" ">Identification of Mutational Clusters in Proteins via a Graph Theoretical Approach.</td>
	<td class="  sorting_1">887</td>
</tr><tr id="pkg_GreyListChIP" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/GreyListChIP.html">GreyListChIP</a></td>
	<td class=" ">Gordon Brown</td>
	<td class=" ">Grey Lists -- Mask Artefact Regions Based on ChIP Inputs</td>
	<td class="  sorting_1">887</td>
</tr><tr id="pkg_MBASED" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/MBASED.html">MBASED</a></td>
	<td class=" ">Oleg Mayba</td>
	<td class=" ">Package containing functions for ASE analysis using Meta-analysis Based Allele-Specific Expression Detection</td>
	<td class="  sorting_1">887</td>
</tr><tr id="pkg_methVisual" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/methVisual.html">methVisual</a></td>
	<td class=" ">Arie Zackay</td>
	<td class=" ">Methods for visualization and statistics on DNA methylation data</td>
	<td class="  sorting_1">887</td>
</tr><tr id="pkg_mogsa" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/mogsa.html">mogsa</a></td>
	<td class=" ">Chen Meng</td>
	<td class=" ">Multiple omics data integrative clustering and gene set analysis</td>
	<td class="  sorting_1">887</td>
</tr><tr id="pkg_OncoScore" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/OncoScore.html">OncoScore</a></td>
	<td class=" ">Luca De Sano</td>
	<td class=" ">A tool to identify potentially oncogenic genes</td>
	<td class="  sorting_1">887</td>
</tr><tr id="pkg_openPrimeR" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/openPrimeR.html">openPrimeR</a></td>
	<td class=" ">Matthias Dring</td>
	<td class=" ">Multiplex PCR Primer Design and Analysis</td>
	<td class="  sorting_1">887</td>
</tr><tr id="pkg_PGA" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/PGA.html">PGA</a></td>
	<td class=" ">Bo Wen, Shaohang Xu</td>
	<td class=" ">An package for identification of novel peptides by customized database derived from RNA-Seq</td>
	<td class="  sorting_1">887</td>
</tr><tr id="pkg_proteoQC" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/proteoQC.html">proteoQC</a></td>
	<td class=" ">Bo Wen</td>
	<td class=" ">An R package for proteomics data quality control</td>
	<td class="  sorting_1">887</td>
</tr><tr id="pkg_R453Plus1Toolbox" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/R453Plus1Toolbox.html">R453Plus1Toolbox</a></td>
	<td class=" ">Hans-Ulrich Klein</td>
	<td class=" ">A package for importing and analyzing data from Roche's Genome Sequencer System</td>
	<td class="  sorting_1">887</td>
</tr><tr id="pkg_REMP" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/REMP.html">REMP</a></td>
	<td class=" ">Yinan Zheng</td>
	<td class=" ">Repetitive Element Methylation Prediction</td>
	<td class="  sorting_1">887</td>
</tr><tr id="pkg_rnaSeqMap" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/rnaSeqMap.html">rnaSeqMap</a></td>
	<td class=" ">Michal Okoniewski</td>
	<td class=" ">rnaSeq secondary analyses</td>
	<td class="  sorting_1">887</td>
</tr><tr id="pkg_RSVSim" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/RSVSim.html">RSVSim</a></td>
	<td class=" ">Christoph Bartenhagen</td>
	<td class=" ">RSVSim: an R/Bioconductor package for the simulation of structural variations</td>
	<td class="  sorting_1">887</td>
</tr><tr id="pkg_RTCA" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/RTCA.html">RTCA</a></td>
	<td class=" ">Jitao David Zhang</td>
	<td class=" ">Open-source toolkit to analyse data from xCELLigence System (RTCA)</td>
	<td class="  sorting_1">887</td>
</tr><tr id="pkg_SANTA" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/SANTA.html">SANTA</a></td>
	<td class=" ">Alex J. Cornish</td>
	<td class=" ">Spatial Analysis of Network Associations</td>
	<td class="  sorting_1">887</td>
</tr><tr id="pkg_seqCNA" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/seqCNA.html">seqCNA</a></td>
	<td class=" ">David Mosen-Ansorena</td>
	<td class=" ">Copy number analysis of high-throughput sequencing cancer data</td>
	<td class="  sorting_1">887</td>
</tr><tr id="pkg_specL" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/specL.html">specL</a></td>
	<td class=" ">Christian Panse, Witold E. Wolski</td>
	<td class=" ">specL - Prepare Peptide Spectrum Matches for Use in Targeted Proteomics</td>
	<td class="  sorting_1">887</td>
</tr><tr id="pkg_yarn" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/yarn.html">yarn</a></td>
	<td class=" ">Joseph N Paulson</td>
	<td class=" ">YARN: Robust Multi-Condition RNA-Seq Preprocessing and Normalization</td>
	<td class="  sorting_1">887</td>
</tr><tr id="pkg_amplican" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/amplican.html">amplican</a></td>
	<td class=" ">Eivind Valen</td>
	<td class=" ">Automated analysis of CRISPR experiments</td>
	<td class="  sorting_1">919</td>
</tr><tr id="pkg_arrayMvout" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/arrayMvout.html">arrayMvout</a></td>
	<td class=" ">V. Carey</td>
	<td class=" ">multivariate outlier detection for expression array QA</td>
	<td class="  sorting_1">919</td>
</tr><tr id="pkg_BEAT" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/BEAT.html">BEAT</a></td>
	<td class=" ">Kemal Akman</td>
	<td class=" ">BEAT - BS-Seq Epimutation Analysis Toolkit</td>
	<td class="  sorting_1">919</td>
</tr><tr id="pkg_BrainStars" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/BrainStars.html">BrainStars</a></td>
	<td class=" ">Itoshi NIKAIDO</td>
	<td class=" ">query gene expression data and plots from BrainStars (B*)</td>
	<td class="  sorting_1">919</td>
</tr><tr id="pkg_bridge" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/bridge.html">bridge</a></td>
	<td class=" ">Raphael Gottardo</td>
	<td class=" ">Bayesian Robust Inference for Differential Gene Expression</td>
	<td class="  sorting_1">919</td>
</tr><tr id="pkg_casper" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/casper.html">casper</a></td>
	<td class=" ">David Rossell</td>
	<td class=" ">Characterization of Alternative Splicing based on Paired-End Reads</td>
	<td class="  sorting_1">919</td>
</tr><tr id="pkg_chroGPS" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/chroGPS.html">chroGPS</a></td>
	<td class=" ">Oscar Reina</td>
	<td class=" ">chroGPS2: Generation, visualization and differential analysis of epigenome maps</td>
	<td class="  sorting_1">919</td>
</tr><tr id="pkg_cnvGSA" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/cnvGSA.html">cnvGSA</a></td>
	<td class=" ">Joseph Lugo</td>
	<td class=" ">Gene Set Analysis of (Rare) Copy Number Variants</td>
	<td class="  sorting_1">919</td>
</tr><tr id="pkg_consensusSeekeR" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/consensusSeekeR.html">consensusSeekeR</a></td>
	<td class=" ">Astrid Deschenes</td>
	<td class=" ">Detection of consensus regions inside a group of experiences using genomic positions and genomic ranges</td>
	<td class="  sorting_1">919</td>
</tr><tr id="pkg_cpvSNP" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/cpvSNP.html">cpvSNP</a></td>
	<td class=" ">Caitlin McHugh</td>
	<td class=" ">Gene set analysis methods for SNP association p-values that lie in genes in given gene sets</td>
	<td class="  sorting_1">919</td>
</tr><tr id="pkg_CSSP" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/CSSP.html">CSSP</a></td>
	<td class=" ">Chandler Zuo</td>
	<td class=" ">ChIP-Seq Statistical Power</td>
	<td class="  sorting_1">919</td>
</tr><tr id="pkg_DFP" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/DFP.html">DFP</a></td>
	<td class=" ">Rodrigo Alvarez-Glez</td>
	<td class=" ">Gene Selection</td>
	<td class="  sorting_1">919</td>
</tr><tr id="pkg_fdrame" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/fdrame.html">fdrame</a></td>
	<td class=" ">Effi Kenigsberg</td>
	<td class=" ">FDR adjustments of Microarray Experiments (FDR-AME)</td>
	<td class="  sorting_1">919</td>
</tr><tr id="pkg_flowPloidy" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/flowPloidy.html">flowPloidy</a></td>
	<td class=" ">Tyler Smith</td>
	<td class=" ">Analyze flow cytometer data to determine sample ploidy</td>
	<td class="  sorting_1">919</td>
</tr><tr id="pkg_GOpro" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/GOpro.html">GOpro</a></td>
	<td class=" ">Lidia Chrabaszcz</td>
	<td class=" ">Find the most characteristic gene ontology terms for groups of human genes</td>
	<td class="  sorting_1">919</td>
</tr><tr id="pkg_immunoClust" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/immunoClust.html">immunoClust</a></td>
	<td class=" ">Till Soerensen</td>
	<td class=" ">immunoClust - Automated Pipeline for Population Detection in Flow Cytometry</td>
	<td class="  sorting_1">919</td>
</tr><tr id="pkg_INSPEcT" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/INSPEcT.html">INSPEcT</a></td>
	<td class=" ">Stefano de Pretis</td>
	<td class=" ">Analysis of 4sU-seq and RNA-seq time-course data</td>
	<td class="  sorting_1">919</td>
</tr><tr id="pkg_IsoGeneGUI" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/IsoGeneGUI.html">IsoGeneGUI</a></td>
	<td class=" ">Setia Pramana</td>
	<td class=" ">A graphical user interface to conduct a dose-response analysis of microarray data</td>
	<td class="  sorting_1">919</td>
</tr><tr id="pkg_LOBSTAHS" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/LOBSTAHS.html">LOBSTAHS</a></td>
	<td class=" ">James Collins</td>
	<td class=" ">Lipid and Oxylipin Biomarker Screening through Adduct Hierarchy Sequences</td>
	<td class="  sorting_1">919</td>
</tr><tr id="pkg_MBCB" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/MBCB.html">MBCB</a></td>
	<td class=" ">Jeff Allen</td>
	<td class=" ">MBCB (Model-based Background Correction for Beadarray)</td>
	<td class="  sorting_1">919</td>
</tr><tr id="pkg_metagenomeFeatures" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/metagenomeFeatures.html">metagenomeFeatures</a></td>
	<td class=" ">Nathan D. Olson</td>
	<td class=" ">Exploration of marker-gene sequence taxonomic annotations</td>
	<td class="  sorting_1">919</td>
</tr><tr id="pkg_NCIgraph" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/NCIgraph.html">NCIgraph</a></td>
	<td class=" ">Laurent Jacob</td>
	<td class=" ">Pathways from the NCI Pathways Database</td>
	<td class="  sorting_1">919</td>
</tr><tr id="pkg_Oscope" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/Oscope.html">Oscope</a></td>
	<td class=" ">Ning Leng</td>
	<td class=" ">Oscope - A statistical pipeline for identifying oscillatory genes in unsynchronized single cell RNA-seq</td>
	<td class="  sorting_1">919</td>
</tr><tr id="pkg_PhenStat" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/PhenStat.html">PhenStat</a></td>
	<td class=" ">Hamed Haselimashhadi</td>
	<td class=" ">Statistical analysis of phenotypic data</td>
	<td class="  sorting_1">919</td>
</tr><tr id="pkg_pqsfinder" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/pqsfinder.html">pqsfinder</a></td>
	<td class=" ">Jiri Hon</td>
	<td class=" ">Identification of potential quadruplex forming sequences</td>
	<td class="  sorting_1">919</td>
</tr><tr id="pkg_rDGIdb" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/rDGIdb.html">rDGIdb</a></td>
	<td class=" ">Thomas Thurnherr</td>
	<td class=" ">R Wrapper for DGIdb</td>
	<td class="  sorting_1">919</td>
</tr><tr id="pkg_runibic" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/runibic.html">runibic</a></td>
	<td class=" ">Patryk Orzechowski</td>
	<td class=" ">runibic: row-based biclustering algorithm for analysis of gene expression data in R</td>
	<td class="  sorting_1">919</td>
</tr><tr id="pkg_shinyTANDEM" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/shinyTANDEM.html">shinyTANDEM</a></td>
	<td class=" ">Frederic Fournier</td>
	<td class=" ">Provides a GUI for rTANDEM</td>
	<td class="  sorting_1">919</td>
</tr><tr id="pkg_STAN" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/STAN.html">STAN</a></td>
	<td class=" ">Rafael Campos-Martin</td>
	<td class=" ">The Genomic STate ANnotation Package</td>
	<td class="  sorting_1">919</td>
</tr><tr id="pkg_TarSeqQC" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/TarSeqQC.html">TarSeqQC</a></td>
	<td class=" ">Gabriela Merino</td>
	<td class=" ">TARgeted SEQuencing Experiment Quality Control</td>
	<td class="  sorting_1">919</td>
</tr><tr id="pkg_TDARACNE" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/TDARACNE.html">TDARACNE</a></td>
	<td class=" ">Zoppoli Pietro</td>
	<td class=" ">Network reverse engineering from time course data.</td>
	<td class="  sorting_1">919</td>
</tr><tr id="pkg_triplex" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/triplex.html">triplex</a></td>
	<td class=" ">Jiri Hon</td>
	<td class=" ">Search and visualize intramolecular triplex-forming sequences in DNA</td>
	<td class="  sorting_1">919</td>
</tr><tr id="pkg_wavClusteR" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/wavClusteR.html">wavClusteR</a></td>
	<td class=" ">Federico Comoglio</td>
	<td class=" ">Sensitive and highly resolved identification of RNA-protein interaction sites in PAR-CLIP data</td>
	<td class="  sorting_1">919</td>
</tr><tr id="pkg_ampliQueso" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/ampliQueso.html">ampliQueso</a></td>
	<td class=" ">Michal Okoniewski</td>
	<td class=" ">Analysis of amplicon enrichment panels</td>
	<td class="  sorting_1">952</td>
</tr><tr id="pkg_AnalysisPageServer" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/AnalysisPageServer.html">AnalysisPageServer</a></td>
	<td class=" ">Brad Friedman</td>
	<td class=" ">A framework for sharing interactive data and plots from R through the web</td>
	<td class="  sorting_1">952</td>
</tr><tr id="pkg_BAGS" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/BAGS.html">BAGS</a></td>
	<td class=" ">Alejandro Quiroz-Zarate</td>
	<td class=" ">A Bayesian Approach for Geneset Selection</td>
	<td class="  sorting_1">952</td>
</tr><tr id="pkg_basecallQC" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/basecallQC.html">basecallQC</a></td>
	<td class=" ">Thomas Carroll</td>
	<td class=" ">Working with Illumina Basecalling and Demultiplexing input and output files</td>
	<td class="  sorting_1">952</td>
</tr><tr id="pkg_BasicSTARRseq" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/BasicSTARRseq.html">BasicSTARRseq</a></td>
	<td class=" ">Annika Buerger</td>
	<td class=" ">Basic peak calling on STARR-seq data</td>
	<td class="  sorting_1">952</td>
</tr><tr id="pkg_bgafun" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/bgafun.html">bgafun</a></td>
	<td class=" ">Iain Wallace</td>
	<td class=" ">BGAfun A method to identify specifity determining residues in protein families</td>
	<td class="  sorting_1">952</td>
</tr><tr id="pkg_bigmelon" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/bigmelon.html">bigmelon</a></td>
	<td class=" ">Tyler J. Gorrie-Stone</td>
	<td class=" ">Illumina methylation array analysis for large experiments</td>
	<td class="  sorting_1">952</td>
</tr><tr id="pkg_BUS" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/BUS.html">BUS</a></td>
	<td class=" ">Yuanhua Liu</td>
	<td class=" ">Gene network reconstruction</td>
	<td class="  sorting_1">952</td>
</tr><tr id="pkg_CGHnormaliter" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/CGHnormaliter.html">CGHnormaliter</a></td>
	<td class=" ">Bart P.P. van Houte</td>
	<td class=" ">Normalization of array CGH data with imbalanced aberrations.</td>
	<td class="  sorting_1">952</td>
</tr><tr id="pkg_Clomial" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/Clomial.html">Clomial</a></td>
	<td class=" ">Habil Zare</td>
	<td class=" ">Infers clonal composition of a tumor</td>
	<td class="  sorting_1">952</td>
</tr><tr id="pkg_clstutils" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/clstutils.html">clstutils</a></td>
	<td class=" ">Noah Hoffman</td>
	<td class=" ">Tools for performing taxonomic assignment.</td>
	<td class="  sorting_1">952</td>
</tr><tr id="pkg_cobindR" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/cobindR.html">cobindR</a></td>
	<td class=" ">Manuela Benary</td>
	<td class=" ">Finding Co-occuring motifs of transcription factor binding sites</td>
	<td class="  sorting_1">952</td>
</tr><tr id="pkg_Cormotif" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/Cormotif.html">Cormotif</a></td>
	<td class=" ">Yingying Wei</td>
	<td class=" ">Correlation Motif Fit</td>
	<td class="  sorting_1">952</td>
</tr><tr id="pkg_CVE" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/CVE.html">CVE</a></td>
	<td class=" ">Andreas Mock</td>
	<td class=" ">Cancer Variant Explorer</td>
	<td class="  sorting_1">952</td>
</tr><tr id="pkg_DChIPRep" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/DChIPRep.html">DChIPRep</a></td>
	<td class=" ">Bernd Klaus</td>
	<td class=" ">DChIPRep - Analysis of chromatin modification ChIP-Seq data with replication</td>
	<td class="  sorting_1">952</td>
</tr><tr id="pkg_DEGraph" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/DEGraph.html">DEGraph</a></td>
	<td class=" ">Laurent Jacob</td>
	<td class=" ">Two-sample tests on a graph</td>
	<td class="  sorting_1">952</td>
</tr><tr id="pkg_EventPointer" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/EventPointer.html">EventPointer</a></td>
	<td class=" ">Juan Pablo Romero</td>
	<td class=" ">An effective identification of alternative splicing events using junction arrays and RNA-Seq data</td>
	<td class="  sorting_1">952</td>
</tr><tr id="pkg_ExperimentHubData" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/ExperimentHubData.html">ExperimentHubData</a></td>
	<td class=" ">Bioconductor Package Maintainer</td>
	<td class=" ">Add resources to ExperimentHub</td>
	<td class="  sorting_1">952</td>
</tr><tr id="pkg_flowCyBar" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/flowCyBar.html">flowCyBar</a></td>
	<td class=" ">Joachim Schumann</td>
	<td class=" ">Analyze flow cytometric data using gate information</td>
	<td class="  sorting_1">952</td>
</tr><tr id="pkg_flowMap" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/flowMap.html">flowMap</a></td>
	<td class=" ">Chiaowen Joyce Hsiao</td>
	<td class=" ">Mapping cell populations in flow cytometry data for cross-sample comparisons using the Friedman-Rafsky Test</td>
	<td class="  sorting_1">952</td>
</tr><tr id="pkg_flowMatch" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/flowMatch.html">flowMatch</a></td>
	<td class=" ">Ariful Azad</td>
	<td class=" ">Matching and meta-clustering in flow cytometry</td>
	<td class="  sorting_1">952</td>
</tr><tr id="pkg_flowVS" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/flowVS.html">flowVS</a></td>
	<td class=" ">Ariful Azad</td>
	<td class=" ">Variance stabilization in flow cytometry (and microarrays)</td>
	<td class="  sorting_1">952</td>
</tr><tr id="pkg_genoCN" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/genoCN.html">genoCN</a></td>
	<td class=" ">Wei Sun</td>
	<td class=" ">genotyping and copy number study tools</td>
	<td class="  sorting_1">952</td>
</tr><tr id="pkg_GenoGAM" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/GenoGAM.html">GenoGAM</a></td>
	<td class=" ">Georg Stricker</td>
	<td class=" ">A GAM based framework for analysis of ChIP-Seq data</td>
	<td class="  sorting_1">952</td>
</tr><tr id="pkg_GEOsubmission" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/GEOsubmission.html">GEOsubmission</a></td>
	<td class=" ">Alexandre Kuhn</td>
	<td class=" ">Prepares microarray data for submission to GEO</td>
	<td class="  sorting_1">952</td>
</tr><tr id="pkg_GraphAT" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/GraphAT.html">GraphAT</a></td>
	<td class=" ">Thomas LaFramboise</td>
	<td class=" ">Graph Theoretic Association Tests</td>
	<td class="  sorting_1">952</td>
</tr><tr id="pkg_iterativeBMAsurv" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/iterativeBMAsurv.html">iterativeBMAsurv</a></td>
	<td class=" ">Ka Yee Yeung</td>
	<td class=" ">The Iterative Bayesian Model Averaging (BMA) Algorithm For Survival Analysis</td>
	<td class="  sorting_1">952</td>
</tr><tr id="pkg_KEGGlincs" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/KEGGlincs.html">KEGGlincs</a></td>
	<td class=" ">Shana White, Mario Medvedovic</td>
	<td class=" ">Visualize all edges within a KEGG pathway and overlay LINCS data [option]</td>
	<td class="  sorting_1">952</td>
</tr><tr id="pkg_LVSmiRNA" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/LVSmiRNA.html">LVSmiRNA</a></td>
	<td class=" ">Stefano Calza</td>
	<td class=" ">LVS normalization for Agilent miRNA data</td>
	<td class="  sorting_1">952</td>
</tr><tr id="pkg_manta" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/manta.html">manta</a></td>
	<td class=" ">Chris Berthiaume, Adrian Marchetti</td>
	<td class=" ">Microbial Assemblage Normalized Transcript Analysis</td>
	<td class="  sorting_1">952</td>
</tr><tr id="pkg_MCRestimate" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/MCRestimate.html">MCRestimate</a></td>
	<td class=" ">Marc Johannes</td>
	<td class=" ">Misclassification error estimation with cross-validation</td>
	<td class="  sorting_1">952</td>
</tr><tr id="pkg_Mergeomics" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/Mergeomics.html">Mergeomics</a></td>
	<td class=" ">Zeyneb Kurt</td>
	<td class=" ">Integrative network analysis of omics data</td>
	<td class="  sorting_1">952</td>
</tr><tr id="pkg_MIMOSA" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/MIMOSA.html">MIMOSA</a></td>
	<td class=" ">Greg Finak</td>
	<td class=" ">Mixture Models for Single-Cell Assays</td>
	<td class="  sorting_1">952</td>
</tr><tr id="pkg_mQTL.NMR" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/mQTL.NMR.html">mQTL.NMR</a></td>
	<td class=" ">Lyamine Hedjazi</td>
	<td class=" ">Metabolomic Quantitative Trait Locus Mapping for 1H NMR data</td>
	<td class="  sorting_1">952</td>
</tr><tr id="pkg_nethet" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/nethet.html">nethet</a></td>
	<td class=" ">Nicolas Staedler, Frank Dondelinger</td>
	<td class=" ">A bioconductor package for high-dimensional exploration of biological network heterogeneity</td>
	<td class="  sorting_1">952</td>
</tr><tr id="pkg_normr" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/normr.html">normr</a></td>
	<td class=" ">Johannes Helmuth</td>
	<td class=" ">Normalization and difference calling in ChIP-seq data</td>
	<td class="  sorting_1">952</td>
</tr><tr id="pkg_OLIN" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/OLIN.html">OLIN</a></td>
	<td class=" ">Matthias Futschik</td>
	<td class=" ">Optimized local intensity-dependent normalisation of two-color microarrays</td>
	<td class="  sorting_1">952</td>
</tr><tr id="pkg_pathRender" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/pathRender.html">pathRender</a></td>
	<td class=" ">Vince Carey</td>
	<td class=" ">Render molecular pathways</td>
	<td class="  sorting_1">952</td>
</tr><tr id="pkg_Pigengene" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/Pigengene.html">Pigengene</a></td>
	<td class=" ">Habil Zare</td>
	<td class=" ">Infers biological signatures from gene expression data</td>
	<td class="  sorting_1">952</td>
</tr><tr id="pkg_Rariant" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/Rariant.html">Rariant</a></td>
	<td class=" ">Julian Gehring</td>
	<td class=" ">Identification and Assessment of Single Nucleotide Variants through Shifts in Non-Consensus Base Call Frequencies</td>
	<td class="  sorting_1">952</td>
</tr><tr id="pkg_RCAS" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/RCAS.html">RCAS</a></td>
	<td class=" ">Bora Uyar</td>
	<td class=" ">RNA Centric Annotation System</td>
	<td class="  sorting_1">952</td>
</tr><tr id="pkg_SIMAT" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/SIMAT.html">SIMAT</a></td>
	<td class=" ">M. R. Nezami Ranjbar</td>
	<td class=" ">GC-SIM-MS data processing and alaysis tool</td>
	<td class="  sorting_1">952</td>
</tr><tr id="pkg_SMITE" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/SMITE.html">SMITE</a></td>
	<td class=" ">Neil Ari Wijetunga, Andrew Damon Johnston</td>
	<td class=" ">Significance-based Modules Integrating the Transcriptome and Epigenome</td>
	<td class="  sorting_1">952</td>
</tr><tr id="pkg_tigre" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/tigre.html">tigre</a></td>
	<td class=" ">Antti Honkela</td>
	<td class=" ">Transcription factor Inference through Gaussian process Reconstruction of Expression</td>
	<td class="  sorting_1">952</td>
</tr><tr id="pkg_BioMVCClass" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/BioMVCClass.html">BioMVCClass</a></td>
	<td class=" ">Elizabeth Whalen</td>
	<td class=" ">Model-View-Controller (MVC) Classes That Use Biobase</td>
	<td class="  sorting_1">996</td>
</tr><tr id="pkg_biomvRCNS" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/biomvRCNS.html">biomvRCNS</a></td>
	<td class=" ">Yang Du</td>
	<td class=" ">Copy Number study and Segmentation for multivariate biological data</td>
	<td class="  sorting_1">996</td>
</tr><tr id="pkg_biotmle" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/biotmle.html">biotmle</a></td>
	<td class=" ">Nima Hejazi</td>
	<td class=" ">Targeted Learning with Moderated Statistics for Biomarker Discovery</td>
	<td class="  sorting_1">996</td>
</tr><tr id="pkg_BPRMeth" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/BPRMeth.html">BPRMeth</a></td>
	<td class=" ">Chantriolnt-Andreas Kapourani</td>
	<td class=" ">Model higher-order methylation profiles</td>
	<td class="  sorting_1">996</td>
</tr><tr id="pkg_CORREP" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/CORREP.html">CORREP</a></td>
	<td class=" ">Dongxiao Zhu</td>
	<td class=" ">Multivariate Correlation Estimator and Statistical Inference Procedures.</td>
	<td class="  sorting_1">996</td>
</tr><tr id="pkg_COSNet" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/COSNet.html">COSNet</a></td>
	<td class=" ">Marco Frasca</td>
	<td class=" ">Cost Sensitive Network for node label prediction on graphs with highly unbalanced labelings</td>
	<td class="  sorting_1">996</td>
</tr><tr id="pkg_EMDomics" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/EMDomics.html">EMDomics</a></td>
	<td class=" ">Sadhika Malladi and Daniel Schmolze</td>
	<td class=" ">Earth Mover's Distance for Differential Analysis of Genomics Data</td>
	<td class="  sorting_1">996</td>
</tr><tr id="pkg_ENVISIONQuery" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/ENVISIONQuery.html">ENVISIONQuery</a></td>
	<td class=" ">Alex Lisovich, Roger Day</td>
	<td class=" ">Retrieval from the ENVISION bioinformatics data portal into R</td>
	<td class="  sorting_1">996</td>
</tr><tr id="pkg_ExpressionView" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/ExpressionView.html">ExpressionView</a></td>
	<td class=" ">Gabor Csardi</td>
	<td class=" ">Visualize biclusters identified in gene expression data</td>
	<td class="  sorting_1">996</td>
</tr><tr id="pkg_flowcatchR" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/flowcatchR.html">flowcatchR</a></td>
	<td class=" ">Federico Marini</td>
	<td class=" ">Tools to analyze in vivo microscopy imaging data focused on tracking flowing blood cells</td>
	<td class="  sorting_1">996</td>
</tr><tr id="pkg_GenomicTuples" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/GenomicTuples.html">GenomicTuples</a></td>
	<td class=" ">Peter Hickey</td>
	<td class=" ">Representation and Manipulation of Genomic Tuples</td>
	<td class="  sorting_1">996</td>
</tr><tr id="pkg_GRENITS" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/GRENITS.html">GRENITS</a></td>
	<td class=" ">Edward Morrissey</td>
	<td class=" ">Gene Regulatory Network Inference Using Time Series</td>
	<td class="  sorting_1">996</td>
</tr><tr id="pkg_Harshlight" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/Harshlight.html">Harshlight</a></td>
	<td class=" ">Maurizio Pellegrino</td>
	<td class=" ">A "corrective make-up" program for microarray chips</td>
	<td class="  sorting_1">996</td>
</tr><tr id="pkg_HELP" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/HELP.html">HELP</a></td>
	<td class=" ">Reid F. Thompson</td>
	<td class=" ">Tools for HELP data analysis</td>
	<td class="  sorting_1">996</td>
</tr><tr id="pkg_HEM" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/HEM.html">HEM</a></td>
	<td class=" ">HyungJun Cho</td>
	<td class=" ">Heterogeneous error model for identification of differentially expressed genes under multiple conditions</td>
	<td class="  sorting_1">996</td>
</tr><tr id="pkg_les" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/les.html">les</a></td>
	<td class=" ">Julian Gehring</td>
	<td class=" ">Identifying Differential Effects in Tiling Microarray Data</td>
	<td class="  sorting_1">996</td>
</tr><tr id="pkg_LiquidAssociation" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/LiquidAssociation.html">LiquidAssociation</a></td>
	<td class=" ">Yen-Yi Ho</td>
	<td class=" ">LiquidAssociation</td>
	<td class="  sorting_1">996</td>
</tr><tr id="pkg_MetaboSignal" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/MetaboSignal.html">MetaboSignal</a></td>
	<td class=" ">Andrea Rodriguez-Martinez, Rafael Ayala</td>
	<td class=" ">MetaboSignal: a network-based approach to overlay and explore metabolic and signaling KEGG pathways</td>
	<td class="  sorting_1">996</td>
</tr><tr id="pkg_motifcounter" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/motifcounter.html">motifcounter</a></td>
	<td class=" ">Wolfgang Kopp</td>
	<td class=" ">R package for analysing TFBSs in DNA sequences</td>
	<td class="  sorting_1">996</td>
</tr><tr id="pkg_msPurity" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/msPurity.html">msPurity</a></td>
	<td class=" ">Thomas N. Lawson</td>
	<td class=" ">Automated Evaluation of Precursor Ion Purity for Mass Spectrometry Based Fragmentation in Metabolomics</td>
	<td class="  sorting_1">996</td>
</tr><tr id="pkg_NuPoP" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/NuPoP.html">NuPoP</a></td>
	<td class=" ">Ji-Ping Wang</td>
	<td class=" ">An R package for nucleosome positioning prediction</td>
	<td class="  sorting_1">996</td>
</tr><tr id="pkg_pepStat" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/pepStat.html">pepStat</a></td>
	<td class=" ">Gregory C Imholte</td>
	<td class=" ">Statistical analysis of peptide microarrays</td>
	<td class="  sorting_1">996</td>
</tr><tr id="pkg_pepXMLTab" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/pepXMLTab.html">pepXMLTab</a></td>
	<td class=" ">Xiaojing Wang</td>
	<td class=" ">Parsing pepXML files and filter based on peptide FDR.</td>
	<td class="  sorting_1">996</td>
</tr><tr id="pkg_PLPE" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/PLPE.html">PLPE</a></td>
	<td class=" ">Soo-heang Eo</td>
	<td class=" ">Local Pooled Error Test for Differential Expression with Paired High-throughput Data</td>
	<td class="  sorting_1">996</td>
</tr><tr id="pkg_procoil" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/procoil.html">procoil</a></td>
	<td class=" ">Ulrich Bodenhofer</td>
	<td class=" ">Prediction of Oligomerization of Coiled Coil Proteins</td>
	<td class="  sorting_1">996</td>
</tr><tr id="pkg_pwOmics" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/pwOmics.html">pwOmics</a></td>
	<td class=" ">Maren Sitte</td>
	<td class=" ">Pathway-based data integration of omics data</td>
	<td class="  sorting_1">996</td>
</tr><tr id="pkg_rama" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/rama.html">rama</a></td>
	<td class=" ">Raphael Gottardo</td>
	<td class=" ">Robust Analysis of MicroArrays</td>
	<td class="  sorting_1">996</td>
</tr><tr id="pkg_RefPlus" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/RefPlus.html">RefPlus</a></td>
	<td class=" ">Kai-Ming Chang</td>
	<td class=" ">A function set for the Extrapolation Strategy (RMA+) and Extrapolation Averaging (RMA++) methods.</td>
	<td class="  sorting_1">996</td>
</tr><tr id="pkg_rfPred" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/rfPred.html">rfPred</a></td>
	<td class=" ">Hugo Varet</td>
	<td class=" ">Assign rfPred functional prediction scores to a missense variants list</td>
	<td class="  sorting_1">996</td>
</tr><tr id="pkg_SELEX" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/SELEX.html">SELEX</a></td>
	<td class=" ">Harmen Bussemaker</td>
	<td class=" ">Functions for analyzing SELEX-seq data</td>
	<td class="  sorting_1">996</td>
</tr><tr id="pkg_stageR" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/stageR.html">stageR</a></td>
	<td class=" ">Koen Van den Berge</td>
	<td class=" ">stageR: stage-wise analysis of high throughput gene expression data in R</td>
	<td class="  sorting_1">996</td>
</tr><tr id="pkg_TargetSearch" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/TargetSearch.html">TargetSearch</a></td>
	<td class=" ">Alvaro Cuadros-Inostroza</td>
	<td class=" ">A package for the analysis of GC-MS metabolite profiling data</td>
	<td class="  sorting_1">996</td>
</tr><tr id="pkg_yamss" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/yamss.html">yamss</a></td>
	<td class=" ">Leslie Myint</td>
	<td class=" ">Tools for high-throughput metabolomics</td>
	<td class="  sorting_1">996</td>
</tr><tr id="pkg_zFPKM" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/zFPKM.html">zFPKM</a></td>
	<td class=" ">Ron Ammar</td>
	<td class=" ">A suite of functions to facilitate zFPKM transformations</td>
	<td class="  sorting_1">996</td>
</tr><tr id="pkg_BaalChIP" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/BaalChIP.html">BaalChIP</a></td>
	<td class=" ">Ines de Santiago</td>
	<td class=" ">BaalChIP: Bayesian analysis of allele-specific transcription factor binding in cancer genomes</td>
	<td class="  sorting_1">1030</td>
</tr><tr id="pkg_BadRegionFinder" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/BadRegionFinder.html">BadRegionFinder</a></td>
	<td class=" ">Sarah Sandmann</td>
	<td class=" ">BadRegionFinder: an R/Bioconductor package for identifying regions with bad coverage</td>
	<td class="  sorting_1">1030</td>
</tr><tr id="pkg_BBCAnalyzer" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/BBCAnalyzer.html">BBCAnalyzer</a></td>
	<td class=" ">Sarah Sandmann</td>
	<td class=" ">BBCAnalyzer: an R/Bioconductor package for visualizing base counts</td>
	<td class="  sorting_1">1030</td>
</tr><tr id="pkg_birte" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/birte.html">birte</a></td>
	<td class=" ">Holger Froehlich</td>
	<td class=" ">Bayesian Inference of Regulatory Influence on Expression (biRte)</td>
	<td class="  sorting_1">1030</td>
</tr><tr id="pkg_BLMA" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/BLMA.html">BLMA</a></td>
	<td class=" ">Tin Nguyen</td>
	<td class=" ">BLMA: A package for bi-level meta-analysis</td>
	<td class="  sorting_1">1030</td>
</tr><tr id="pkg_CexoR" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/CexoR.html">CexoR</a></td>
	<td class=" ">Pedro Madrigal</td>
	<td class=" ">An R package to uncover high-resolution protein-DNA interactions in ChIP-exo replicates</td>
	<td class="  sorting_1">1030</td>
</tr><tr id="pkg_clustComp" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/clustComp.html">clustComp</a></td>
	<td class=" ">Aurora Torrente</td>
	<td class=" ">Clustering Comparison Package</td>
	<td class="  sorting_1">1030</td>
</tr><tr id="pkg_coGPS" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/coGPS.html">coGPS</a></td>
	<td class=" ">Yingying Wei</td>
	<td class=" ">cancer outlier Gene Profile Sets</td>
	<td class="  sorting_1">1030</td>
</tr><tr id="pkg_dagLogo" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/dagLogo.html">dagLogo</a></td>
	<td class=" ">Jianhong Ou</td>
	<td class=" ">dagLogo</td>
	<td class="  sorting_1">1030</td>
</tr><tr id="pkg_daMA" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/daMA.html">daMA</a></td>
	<td class=" ">Jobst Landgrebe</td>
	<td class=" ">Efficient design and analysis of factorial two-colour microarray data</td>
	<td class="  sorting_1">1030</td>
</tr><tr id="pkg_dSimer" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/dSimer.html">dSimer</a></td>
	<td class=" ">Peng Ni</td>
	<td class=" ">Integration of Disease Similarity Methods</td>
	<td class="  sorting_1">1030</td>
</tr><tr id="pkg_DTA" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/DTA.html">DTA</a></td>
	<td class=" ">Bjoern Schwalb</td>
	<td class=" ">Dynamic Transcriptome Analysis</td>
	<td class="  sorting_1">1030</td>
</tr><tr id="pkg_ELBOW" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/ELBOW.html">ELBOW</a></td>
	<td class=" ">Graham Alvare, Xiangli Zhang</td>
	<td class=" ">ELBOW - Evaluating foLd change By the lOgit Way</td>
	<td class="  sorting_1">1030</td>
</tr><tr id="pkg_FISHalyseR" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/FISHalyseR.html">FISHalyseR</a></td>
	<td class=" ">Karesh Arunakirinathan, Andreas Heindl</td>
	<td class=" ">FISHalyseR a package for automated FISH quantification</td>
	<td class="  sorting_1">1030</td>
</tr><tr id="pkg_flowFit" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/flowFit.html">flowFit</a></td>
	<td class=" ">Davide Rambaldi</td>
	<td class=" ">Estimate proliferation in cell-tracking dye studies</td>
	<td class="  sorting_1">1030</td>
</tr><tr id="pkg_flowPlots" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/flowPlots.html">flowPlots</a></td>
	<td class=" ">N. Hawkins</td>
	<td class=" ">flowPlots: analysis plots and data class for gated flow cytometry data</td>
	<td class="  sorting_1">1030</td>
</tr><tr id="pkg_flowQB" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/flowQB.html">flowQB</a></td>
	<td class=" ">Josef Spidlen</td>
	<td class=" ">Automated Quadratic Characterization of Flow Cytometer Instrument Sensitivity: Q, B and CV instrinsic calculations</td>
	<td class="  sorting_1">1030</td>
</tr><tr id="pkg_focalCall" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/focalCall.html">focalCall</a></td>
	<td class=" ">Oscar Krijgsman</td>
	<td class=" ">Detection of focal aberrations in DNA copy number data</td>
	<td class="  sorting_1">1030</td>
</tr><tr id="pkg_GeneRegionScan" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/GeneRegionScan.html">GeneRegionScan</a></td>
	<td class=" ">Lasse Folkersen</td>
	<td class=" ">GeneRegionScan</td>
	<td class="  sorting_1">1030</td>
</tr><tr id="pkg_GSReg" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/GSReg.html">GSReg</a></td>
	<td class=" ">Bahman Afsari, Elana J. Fertig</td>
	<td class=" ">Gene Set Regulation (GS-Reg)</td>
	<td class="  sorting_1">1030</td>
</tr><tr id="pkg_iCARE" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/iCARE.html">iCARE</a></td>
	<td class=" ">Bill Wheeler</td>
	<td class=" ">A Tool for Individualized Coherent Absolute Risk Estimation (iCARE)</td>
	<td class="  sorting_1">1030</td>
</tr><tr id="pkg_iCheck" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/iCheck.html">iCheck</a></td>
	<td class=" ">Weiliang Qiu</td>
	<td class=" ">QC Pipeline and Data Analysis Tools for High-Dimensional Illumina mRNA Expression Data</td>
	<td class="  sorting_1">1030</td>
</tr><tr id="pkg_iSeq" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/iSeq.html">iSeq</a></td>
	<td class=" ">Qianxing Mo</td>
	<td class=" ">Bayesian Hierarchical Modeling of ChIP-seq Data Through Hidden Ising Models</td>
	<td class="  sorting_1">1030</td>
</tr><tr id="pkg_ITALICS" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/ITALICS.html">ITALICS</a></td>
	<td class=" ">Guillem Rigaill</td>
	<td class=" ">ITALICS</td>
	<td class="  sorting_1">1030</td>
</tr><tr id="pkg_maCorrPlot" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/maCorrPlot.html">maCorrPlot</a></td>
	<td class=" ">Alexander Ploner</td>
	<td class=" ">Visualize artificial correlation in microarray data</td>
	<td class="  sorting_1">1030</td>
</tr><tr id="pkg_maPredictDSC" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/maPredictDSC.html">maPredictDSC</a></td>
	<td class=" ">Adi Laurentiu Tarca</td>
	<td class=" ">Phenotype prediction using microarray data: approach of the best overall team in the IMPROVER Diagnostic Signature Challenge</td>
	<td class="  sorting_1">1030</td>
</tr><tr id="pkg_MassArray" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/MassArray.html">MassArray</a></td>
	<td class=" ">Reid F. Thompson</td>
	<td class=" ">Analytical Tools for MassArray Data</td>
	<td class="  sorting_1">1030</td>
</tr><tr id="pkg_mdqc" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/mdqc.html">mdqc</a></td>
	<td class=" ">Gabriela Cohen-Freue</td>
	<td class=" ">Mahalanobis Distance Quality Control for microarrays</td>
	<td class="  sorting_1">1030</td>
</tr><tr id="pkg_metaCCA" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/metaCCA.html">metaCCA</a></td>
	<td class=" ">Anna Cichonska</td>
	<td class=" ">Summary Statistics-Based Multivariate Meta-Analysis of Genome-Wide Association Studies Using Canonical Correlation Analysis</td>
	<td class="  sorting_1">1030</td>
</tr><tr id="pkg_MiRaGE" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/MiRaGE.html">MiRaGE</a></td>
	<td class=" ">Y-h. Taguchi</td>
	<td class=" ">MiRNA Ranking by Gene Expression</td>
	<td class="  sorting_1">1030</td>
</tr><tr id="pkg_mirIntegrator" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/mirIntegrator.html">mirIntegrator</a></td>
	<td class=" ">Diana Diaz</td>
	<td class=" ">Integrating microRNA expression into signaling pathways for pathway analysis</td>
	<td class="  sorting_1">1030</td>
</tr><tr id="pkg_Mirsynergy" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/Mirsynergy.html">Mirsynergy</a></td>
	<td class=" ">Yue Li</td>
	<td class=" ">Mirsynergy</td>
	<td class="  sorting_1">1030</td>
</tr><tr id="pkg_MVCClass" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/MVCClass.html">MVCClass</a></td>
	<td class=" ">Elizabeth Whalen</td>
	<td class=" ">Model-View-Controller (MVC) Classes</td>
	<td class="  sorting_1">1030</td>
</tr><tr id="pkg_OTUbase" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/OTUbase.html">OTUbase</a></td>
	<td class=" ">Daniel Beck</td>
	<td class=" ">Provides structure and functions for the analysis of OTU data</td>
	<td class="  sorting_1">1030</td>
</tr><tr id="pkg_Pi" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/Pi.html">Pi</a></td>
	<td class=" ">Hai Fang</td>
	<td class=" ">Leveraging Genetic Evidence to Prioritise Drug Targets at the Gene and Pathway Level</td>
	<td class="  sorting_1">1030</td>
</tr><tr id="pkg_pint" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/pint.html">pint</a></td>
	<td class=" ">Olli-Pekka Huovilainen</td>
	<td class=" ">Pairwise INTegration of functional genomics data</td>
	<td class="  sorting_1">1030</td>
</tr><tr id="pkg_R4RNA" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/R4RNA.html">R4RNA</a></td>
	<td class=" ">Daniel Lai</td>
	<td class=" ">An R package for RNA visualization and analysis</td>
	<td class="  sorting_1">1030</td>
</tr><tr id="pkg_randPack" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/randPack.html">randPack</a></td>
	<td class=" ">Robert Gentleman</td>
	<td class=" ">Randomization routines for Clinical Trials</td>
	<td class="  sorting_1">1030</td>
</tr><tr id="pkg_RNAither" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/RNAither.html">RNAither</a></td>
	<td class=" ">Lars Kaderali</td>
	<td class=" ">Statistical analysis of high-throughput RNAi screens</td>
	<td class="  sorting_1">1030</td>
</tr><tr id="pkg_Rtreemix" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/Rtreemix.html">Rtreemix</a></td>
	<td class=" ">Jasmina Bogojeska</td>
	<td class=" ">Rtreemix: Mutagenetic trees mixture models.</td>
	<td class="  sorting_1">1030</td>
</tr><tr id="pkg_switchde" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/switchde.html">switchde</a></td>
	<td class=" ">Kieran Campbell</td>
	<td class=" ">Switch-like differential expression across single-cell trajectories</td>
	<td class="  sorting_1">1030</td>
</tr><tr id="pkg_TSRchitect" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/TSRchitect.html">TSRchitect</a></td>
	<td class=" ">R. Taylor Raborn</td>
	<td class=" ">Promoter identification from large-scale TSS profiling data</td>
	<td class="  sorting_1">1030</td>
</tr><tr id="pkg_webbioc" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/webbioc.html">webbioc</a></td>
	<td class=" ">Colin A. Smith</td>
	<td class=" ">Bioconductor Web Interface</td>
	<td class="  sorting_1">1030</td>
</tr><tr id="pkg_CAFE" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/CAFE.html">CAFE</a></td>
	<td class=" ">Sander Bollen</td>
	<td class=" ">Chromosmal Aberrations Finder in Expression data</td>
	<td class="  sorting_1">1073</td>
</tr><tr id="pkg_caOmicsV" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/caOmicsV.html">caOmicsV</a></td>
	<td class=" ">Henry Zhang</td>
	<td class=" ">Visualization of multi-dimentional cancer genomics data</td>
	<td class="  sorting_1">1073</td>
</tr><tr id="pkg_ChIPexoQual" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/ChIPexoQual.html">ChIPexoQual</a></td>
	<td class=" ">Rene Welch</td>
	<td class=" ">ChIPexoQual</td>
	<td class="  sorting_1">1073</td>
</tr><tr id="pkg_CNPBayes" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/CNPBayes.html">CNPBayes</a></td>
	<td class=" ">Jacob Carey</td>
	<td class=" ">Bayesian mixture models for copy number polymorphisms</td>
	<td class="  sorting_1">1073</td>
</tr><tr id="pkg_coexnet" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/coexnet.html">coexnet</a></td>
	<td class=" ">Juan David Henao</td>
	<td class=" ">coexnet: An R package to build CO-EXpression NETworks from Microarray Data</td>
	<td class="  sorting_1">1073</td>
</tr><tr id="pkg_crisprseekplus" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/crisprseekplus.html">crisprseekplus</a></td>
	<td class=" ">Alper Kucukural</td>
	<td class=" ">crisprseekplus</td>
	<td class="  sorting_1">1073</td>
</tr><tr id="pkg_discordant" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/discordant.html">discordant</a></td>
	<td class=" ">Charlotte Siska</td>
	<td class=" ">The Discordant Method: A Novel Approach for Differential Correlation</td>
	<td class="  sorting_1">1073</td>
</tr><tr id="pkg_dualKS" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/dualKS.html">dualKS</a></td>
	<td class=" ">Eric J. Kort, Yarong Yang</td>
	<td class=" ">Dual KS Discriminant Analysis and Classification</td>
	<td class="  sorting_1">1073</td>
</tr><tr id="pkg_fastLiquidAssociation" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/fastLiquidAssociation.html">fastLiquidAssociation</a></td>
	<td class=" ">Tina Gunderson</td>
	<td class=" ">functions for genome-wide application of Liquid Association</td>
	<td class="  sorting_1">1073</td>
</tr><tr id="pkg_flowBin" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/flowBin.html">flowBin</a></td>
	<td class=" ">Kieran O'Neill</td>
	<td class=" ">Combining multitube flow cytometry data by binning</td>
	<td class="  sorting_1">1073</td>
</tr><tr id="pkg_flowCHIC" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/flowCHIC.html">flowCHIC</a></td>
	<td class=" ">Author: Joachim Schumann</td>
	<td class=" ">Analyze flow cytometric data using histogram information</td>
	<td class="  sorting_1">1073</td>
</tr><tr id="pkg_gCrisprTools" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/gCrisprTools.html">gCrisprTools</a></td>
	<td class=" ">Peter Haverty</td>
	<td class=" ">Suite of Functions for Pooled Crispr Screen QC and Analysis</td>
	<td class="  sorting_1">1073</td>
</tr><tr id="pkg_GenRank" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/GenRank.html">GenRank</a></td>
	<td class=" ">Chakravarthi Kanduri</td>
	<td class=" ">Candidate gene prioritization based on convergent evidence</td>
	<td class="  sorting_1">1073</td>
</tr><tr id="pkg_GSRI" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/GSRI.html">GSRI</a></td>
	<td class=" ">Julian Gehring</td>
	<td class=" ">Gene Set Regulation Index</td>
	<td class="  sorting_1">1073</td>
</tr><tr id="pkg_hierGWAS" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/hierGWAS.html">hierGWAS</a></td>
	<td class=" ">Laura Buzdugan</td>
	<td class=" ">Asessing statistical significance in predictive GWA studies</td>
	<td class="  sorting_1">1073</td>
</tr><tr id="pkg_idiogram" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/idiogram.html">idiogram</a></td>
	<td class=" ">Karl J. Dykema</td>
	<td class=" ">idiogram</td>
	<td class="  sorting_1">1073</td>
</tr><tr id="pkg_IntEREst" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/IntEREst.html">IntEREst</a></td>
	<td class=" ">Ali Oghabian, Mikko Frilander</td>
	<td class=" ">Intron-Exon Retention Estimator</td>
	<td class="  sorting_1">1073</td>
</tr><tr id="pkg_isomiRs" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/isomiRs.html">isomiRs</a></td>
	<td class=" ">Lorena Pantano</td>
	<td class=" ">Analyze isomiRs and miRNAs from small RNA-seq</td>
	<td class="  sorting_1">1073</td>
</tr><tr id="pkg_logitT" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/logitT.html">logitT</a></td>
	<td class=" ">Tobias Guennel</td>
	<td class=" ">logit-t Package</td>
	<td class="  sorting_1">1073</td>
</tr><tr id="pkg_mBPCR" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/mBPCR.html">mBPCR</a></td>
	<td class=" ">P.M.V. Rancoita</td>
	<td class=" ">Bayesian Piecewise Constant Regression for DNA copy number estimation</td>
	<td class="  sorting_1">1073</td>
</tr><tr id="pkg_MiPP" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/MiPP.html">MiPP</a></td>
	<td class=" ">Sukwoo Kim</td>
	<td class=" ">Misclassification Penalized Posterior Classification</td>
	<td class="  sorting_1">1073</td>
</tr><tr id="pkg_miRNAmeConverter" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/miRNAmeConverter.html">miRNAmeConverter</a></td>
	<td class=" ">Stefan J. Haunsberger</td>
	<td class=" ">Convert miRNA Names to Different miRBase Versions</td>
	<td class="  sorting_1">1073</td>
</tr><tr id="pkg_MmPalateMiRNA" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/MmPalateMiRNA.html">MmPalateMiRNA</a></td>
	<td class=" ">Guy Brock</td>
	<td class=" ">Murine Palate miRNA Expression Analysis</td>
	<td class="  sorting_1">1073</td>
</tr><tr id="pkg_MODA" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/MODA.html">MODA</a></td>
	<td class=" ">Dong Li</td>
	<td class=" ">MODA: MOdule Differential Analysis for weighted gene co-expression network</td>
	<td class="  sorting_1">1073</td>
</tr><tr id="pkg_nnNorm" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/nnNorm.html">nnNorm</a></td>
	<td class=" ">Adi Laurentiu Tarca</td>
	<td class=" ">Spatial and intensity based normalization of cDNA microarray data based on robust neural nets</td>
	<td class="  sorting_1">1073</td>
</tr><tr id="pkg_paircompviz" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/paircompviz.html">paircompviz</a></td>
	<td class=" ">Michal Burda</td>
	<td class=" ">Multiple comparison test visualization</td>
	<td class="  sorting_1">1073</td>
</tr><tr id="pkg_PANR" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/PANR.html">PANR</a></td>
	<td class=" ">Xin Wang</td>
	<td class=" ">Posterior association networks and functional modules inferred from rich phenotypes of gene perturbations</td>
	<td class="  sorting_1">1073</td>
</tr><tr id="pkg_PanVizGenerator" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/PanVizGenerator.html">PanVizGenerator</a></td>
	<td class=" ">Thomas Lin Pedersen</td>
	<td class=" ">Generate PanViz visualisations from your pangenome</td>
	<td class="  sorting_1">1073</td>
</tr><tr id="pkg_podkat" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/podkat.html">podkat</a></td>
	<td class=" ">Ulrich Bodenhofer</td>
	<td class=" ">Position-Dependent Kernel Association Test</td>
	<td class="  sorting_1">1073</td>
</tr><tr id="pkg_PPInfer" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/PPInfer.html">PPInfer</a></td>
	<td class=" ">Dongmin Jung</td>
	<td class=" ">Inferring functionally related proteins using protein interaction networks</td>
	<td class="  sorting_1">1073</td>
</tr><tr id="pkg_proFIA" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/proFIA.html">proFIA</a></td>
	<td class=" ">Alexis Delabriere</td>
	<td class=" ">Preprocessing of FIA-HRMS data</td>
	<td class="  sorting_1">1073</td>
</tr><tr id="pkg_prot2D" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/prot2D.html">prot2D</a></td>
	<td class=" ">Sebastien Artigaud</td>
	<td class=" ">Statistical Tools for volume data from 2D Gel Electrophoresis</td>
	<td class="  sorting_1">1073</td>
</tr><tr id="pkg_ProteomicsAnnotationHubData" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/ProteomicsAnnotationHubData.html">ProteomicsAnnotationHubData</a></td>
	<td class=" ">Laurent Gatto</td>
	<td class=" ">Transform public proteomics data resources into Bioconductor Data Structures</td>
	<td class="  sorting_1">1073</td>
</tr><tr id="pkg_pvac" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/pvac.html">pvac</a></td>
	<td class=" ">Jun Lu, Pierre R. Bushel</td>
	<td class=" ">PCA-based gene filtering for Affymetrix arrays</td>
	<td class="  sorting_1">1073</td>
</tr><tr id="pkg_RImmPort" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/RImmPort.html">RImmPort</a></td>
	<td class=" ">Zicheng Hu, Ravi Shankar</td>
	<td class=" ">RImmPort: Enabling Ready-for-analysis Immunology Research Data</td>
	<td class="  sorting_1">1073</td>
</tr><tr id="pkg_Rmagpie" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/Rmagpie.html">Rmagpie</a></td>
	<td class=" ">Camille Maumet</td>
	<td class=" ">MicroArray Gene-expression-based Program In Error rate estimation</td>
	<td class="  sorting_1">1073</td>
</tr><tr id="pkg_rnaseqcomp" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/rnaseqcomp.html">rnaseqcomp</a></td>
	<td class=" ">Mingxiang Teng</td>
	<td class=" ">Benchmarks for RNA-seq Quantification Pipelines</td>
	<td class="  sorting_1">1073</td>
</tr><tr id="pkg_spkTools" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/spkTools.html">spkTools</a></td>
	<td class=" ">Matthew N McCall</td>
	<td class=" ">Methods for Spike-in Arrays</td>
	<td class="  sorting_1">1073</td>
</tr><tr id="pkg_spliceSites" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/spliceSites.html">spliceSites</a></td>
	<td class=" ">Wolfgang Kaisers</td>
	<td class=" ">A bioconductor package for exploration of alignment gap positions from RNA-seq data</td>
	<td class="  sorting_1">1073</td>
</tr><tr id="pkg_spotSegmentation" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/spotSegmentation.html">spotSegmentation</a></td>
	<td class=" ">Chris Fraley</td>
	<td class=" ">Microarray Spot Segmentation and Gridding for Blocks of Microarray Spots</td>
	<td class="  sorting_1">1073</td>
</tr><tr id="pkg_Starr" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/Starr.html">Starr</a></td>
	<td class=" ">Benedikt Zacher</td>
	<td class=" ">Simple tiling array analysis of Affymetrix ChIP-chip data</td>
	<td class="  sorting_1">1073</td>
</tr><tr id="pkg_wiggleplotr" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/wiggleplotr.html">wiggleplotr</a></td>
	<td class=" ">Kaur Alasoo</td>
	<td class=" ">Make read coverage plots from BigWig files</td>
	<td class="  sorting_1">1073</td>
</tr><tr id="pkg_anota2seq" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/anota2seq.html">anota2seq</a></td>
	<td class=" ">Christian Oertlin, Julie Lorent</td>
	<td class=" ">Generally applicable transcriptome-wide analysis of translational efficiency using anota2seq</td>
	<td class="  sorting_1">1115</td>
</tr><tr id="pkg_CAnD" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/CAnD.html">CAnD</a></td>
	<td class=" ">Caitlin McHugh</td>
	<td class=" ">Perform Chromosomal Ancestry Differences (CAnD) Analyses</td>
	<td class="  sorting_1">1115</td>
</tr><tr id="pkg_diffGeneAnalysis" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/diffGeneAnalysis.html">diffGeneAnalysis</a></td>
	<td class=" ">Choudary Jagarlamudi</td>
	<td class=" ">Performs differential gene expression Analysis</td>
	<td class="  sorting_1">1115</td>
</tr><tr id="pkg_DrugVsDisease" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/DrugVsDisease.html">DrugVsDisease</a></td>
	<td class=" ">j. Saez-Rodriguez</td>
	<td class=" ">Comparison of disease and drug profiles using Gene set Enrichment Analysis</td>
	<td class="  sorting_1">1115</td>
</tr><tr id="pkg_flowBeads" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/flowBeads.html">flowBeads</a></td>
	<td class=" ">Nikolas Pontikos</td>
	<td class=" ">flowBeads: Analysis of flow bead data</td>
	<td class="  sorting_1">1115</td>
</tr><tr id="pkg_GSCA" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/GSCA.html">GSCA</a></td>
	<td class=" ">Zhicheng Ji</td>
	<td class=" ">GSCA: Gene Set Context Analysis</td>
	<td class="  sorting_1">1115</td>
</tr><tr id="pkg_HDTD" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/HDTD.html">HDTD</a></td>
	<td class=" ">Anestis Touloumis</td>
	<td class=" ">Statistical Inference about the Mean Matrix and the Covariance Matrices in High-Dimensional Transposable Data (HDTD)</td>
	<td class="  sorting_1">1115</td>
</tr><tr id="pkg_ImpulseDE" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/ImpulseDE.html">ImpulseDE</a></td>
	<td class=" ">Jil Sander, Nir Yosef</td>
	<td class=" ">Detection of DE genes in time series data using impulse models</td>
	<td class="  sorting_1">1115</td>
</tr><tr id="pkg_IVAS" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/IVAS.html">IVAS</a></td>
	<td class=" ">Seonggyun Han</td>
	<td class=" ">Identification of genetic Variants affecting Alternative Splicing</td>
	<td class="  sorting_1">1115</td>
</tr><tr id="pkg_joda" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/joda.html">joda</a></td>
	<td class=" ">Ewa Szczurek</td>
	<td class=" ">JODA algorithm for quantifying gene deregulation using knowledge</td>
	<td class="  sorting_1">1115</td>
</tr><tr id="pkg_lapmix" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/lapmix.html">lapmix</a></td>
	<td class=" ">Yann Ruffieux</td>
	<td class=" ">Laplace Mixture Model in Microarray Experiments</td>
	<td class="  sorting_1">1115</td>
</tr><tr id="pkg_metavizr" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/metavizr.html">metavizr</a></td>
	<td class=" ">Hector Corrada Bravo</td>
	<td class=" ">R Interface to the metaviz web app for interactive metagenomics data analysis and visualization</td>
	<td class="  sorting_1">1115</td>
</tr><tr id="pkg_OLINgui" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/OLINgui.html">OLINgui</a></td>
	<td class=" ">Matthias Futschik</td>
	<td class=" ">Graphical user interface for OLIN</td>
	<td class="  sorting_1">1115</td>
</tr><tr id="pkg_progeny" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/progeny.html">progeny</a></td>
	<td class=" ">Michael Schubert</td>
	<td class=" ">Pathway RespOnsive GENes for activity inference from gene expression</td>
	<td class="  sorting_1">1115</td>
</tr><tr id="pkg_QuaternaryProd" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/QuaternaryProd.html">QuaternaryProd</a></td>
	<td class=" ">Carl Tony Fakhry</td>
	<td class=" ">Computes the Quaternary Dot Product Scoring Statistic for Signed and Unsigned Causal Graphs</td>
	<td class="  sorting_1">1115</td>
</tr><tr id="pkg_ramwas" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/ramwas.html">ramwas</a></td>
	<td class=" ">Andrey A Shabalin</td>
	<td class=" ">Fast Methylome-Wide Association Study Pipeline for Enrichment Platforms</td>
	<td class="  sorting_1">1115</td>
</tr><tr id="pkg_RJMCMCNucleosomes" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/RJMCMCNucleosomes.html">RJMCMCNucleosomes</a></td>
	<td class=" ">Astrid Deschnes</td>
	<td class=" ">Bayesian hierarchical model for genome-wide nucleosome positioning with high-throughput short-read data (MNase-Seq)</td>
	<td class="  sorting_1">1115</td>
</tr><tr id="pkg_RTopper" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/RTopper.html">RTopper</a></td>
	<td class=" ">Luigi Marchionni</td>
	<td class=" ">This package is designed to perform Gene Set Analysis across multiple genomic platforms</td>
	<td class="  sorting_1">1115</td>
</tr><tr id="pkg_SIM" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/SIM.html">SIM</a></td>
	<td class=" ">Renee X. de Menezes</td>
	<td class=" ">Integrated Analysis on two human genomic datasets</td>
	<td class="  sorting_1">1115</td>
</tr><tr id="pkg_SNPhood" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/SNPhood.html">SNPhood</a></td>
	<td class=" ">Christian Arnold</td>
	<td class=" ">SNPhood: Investigate, quantify and visualise the epigenomic neighbourhood of SNPs using NGS data</td>
	<td class="  sorting_1">1115</td>
</tr><tr id="pkg_SpacePAC" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/SpacePAC.html">SpacePAC</a></td>
	<td class=" ">Gregory Ryslik</td>
	<td class=" ">Identification of Mutational Clusters in 3D Protein Space via Simulation.</td>
	<td class="  sorting_1">1115</td>
</tr><tr id="pkg_splineTimeR" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/splineTimeR.html">splineTimeR</a></td>
	<td class=" ">Herbert Braselmann</td>
	<td class=" ">Time-course differential gene expression data analysis using spline regression models followed by gene association network reconstruction</td>
	<td class="  sorting_1">1115</td>
</tr><tr id="pkg_sRAP" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/sRAP.html">sRAP</a></td>
	<td class=" ">Charles Warden</td>
	<td class=" ">Simplified RNA-Seq Analysis Pipeline</td>
	<td class="  sorting_1">1115</td>
</tr><tr id="pkg_sscore" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/sscore.html">sscore</a></td>
	<td class=" ">Richard Kennedy</td>
	<td class=" ">S-Score Algorithm for Affymetrix Oligonucleotide Microarrays</td>
	<td class="  sorting_1">1115</td>
</tr><tr id="pkg_TMixClust" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/TMixClust.html">TMixClust</a></td>
	<td class=" ">Monica Golumbeanu</td>
	<td class=" ">Time Series Clustering of Gene Expression with Gaussian Mixed-Effects Models and Smoothing Splines</td>
	<td class="  sorting_1">1115</td>
</tr><tr id="pkg_TransView" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/TransView.html">TransView</a></td>
	<td class=" ">Julius Muller</td>
	<td class=" ">Read density map construction and accession. Visualization of ChIPSeq and RNASeq data sets</td>
	<td class="  sorting_1">1115</td>
</tr><tr id="pkg_annmap" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/annmap.html">annmap</a></td>
	<td class=" ">Chris Wirth</td>
	<td class=" ">Genome annotation and visualisation package pertaining to Affymetrix arrays and NGS analysis.</td>
	<td class="  sorting_1">1141</td>
</tr><tr id="pkg_Director" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/Director.html">Director</a></td>
	<td class=" ">Katherine Icay</td>
	<td class=" ">A dynamic visualization tool of multi-level data</td>
	<td class="  sorting_1">1141</td>
</tr><tr id="pkg_DupChecker" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/DupChecker.html">DupChecker</a></td>
	<td class=" ">"Quanhu SHENG"</td>
	<td class=" ">a package for checking high-throughput genomic data redundancy in meta-analysis</td>
	<td class="  sorting_1">1141</td>
</tr><tr id="pkg_fCI" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/fCI.html">fCI</a></td>
	<td class=" ">Shaojun Tang</td>
	<td class=" ">f-divergence Cutoff Index for Differential Expression Analysis in Transcriptomics and Proteomics</td>
	<td class="  sorting_1">1141</td>
</tr><tr id="pkg_gcapc" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/gcapc.html">gcapc</a></td>
	<td class=" ">Mingxiang Teng</td>
	<td class=" ">GC Aware Peak Caller</td>
	<td class="  sorting_1">1141</td>
</tr><tr id="pkg_GeneBreak" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/GeneBreak.html">GeneBreak</a></td>
	<td class=" ">Evert van den Broek</td>
	<td class=" ">Gene Break Detection</td>
	<td class="  sorting_1">1141</td>
</tr><tr id="pkg_GeneGeneInteR" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/GeneGeneInteR.html">GeneGeneInteR</a></td>
	<td class=" ">Mathieu Emily, Magalie Houee-Bigot</td>
	<td class=" ">Tools for Testing Gene-Gene Interaction at the Gene Level</td>
	<td class="  sorting_1">1141</td>
</tr><tr id="pkg_genotypeeval" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/genotypeeval.html">genotypeeval</a></td>
	<td class=" ">Jennifer Tom</td>
	<td class=" ">QA/QC of a gVCF or VCF file</td>
	<td class="  sorting_1">1141</td>
</tr><tr id="pkg_gprege" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/gprege.html">gprege</a></td>
	<td class=" ">Alfredo Kalaitzis</td>
	<td class=" ">Gaussian Process Ranking and Estimation of Gene Expression time-series</td>
	<td class="  sorting_1">1141</td>
</tr><tr id="pkg_iChip" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/iChip.html">iChip</a></td>
	<td class=" ">Qianxing Mo</td>
	<td class=" ">Bayesian Modeling of ChIP-chip Data Through Hidden Ising Models</td>
	<td class="  sorting_1">1141</td>
</tr><tr id="pkg_LPEadj" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/LPEadj.html">LPEadj</a></td>
	<td class=" ">Carl Murie</td>
	<td class=" ">A correction of the local pooled error (LPE) method to replace the asymptotic variance adjustment with an unbiased adjustment based on sample size.</td>
	<td class="  sorting_1">1141</td>
</tr><tr id="pkg_macat" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/macat.html">macat</a></td>
	<td class=" ">Joern Toedling</td>
	<td class=" ">MicroArray Chromosome Analysis Tool</td>
	<td class="  sorting_1">1141</td>
</tr><tr id="pkg_MANOR" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/MANOR.html">MANOR</a></td>
	<td class=" ">Pierre Neuvial</td>
	<td class=" ">CGH Micro-Array NORmalization</td>
	<td class="  sorting_1">1141</td>
</tr><tr id="pkg_maskBAD" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/maskBAD.html">maskBAD</a></td>
	<td class=" ">Michael Dannemann</td>
	<td class=" ">Masking probes with binding affinity differences</td>
	<td class="  sorting_1">1141</td>
</tr><tr id="pkg_massiR" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/massiR.html">massiR</a></td>
	<td class=" ">Sam Buckberry</td>
	<td class=" ">massiR: MicroArray Sample Sex Identifier</td>
	<td class="  sorting_1">1141</td>
</tr><tr id="pkg_mcaGUI" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/mcaGUI.html">mcaGUI</a></td>
	<td class=" ">Wade K. Copeland</td>
	<td class=" ">Microbial Community Analysis GUI</td>
	<td class="  sorting_1">1141</td>
</tr><tr id="pkg_MetCirc" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/MetCirc.html">MetCirc</a></td>
	<td class=" ">Thomas Naake</td>
	<td class=" ">Navigating mass spectral similarity in high-resolution MS/MS metabolomics data</td>
	<td class="  sorting_1">1141</td>
</tr><tr id="pkg_miRBaseConverter" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/miRBaseConverter.html">miRBaseConverter</a></td>
	<td class=" ">Taosheng Xu</td>
	<td class=" ">A comprehensive and high-efficiency tool for converting and retrieving the information of miRNAs in different miRBase versions</td>
	<td class="  sorting_1">1141</td>
</tr><tr id="pkg_NetSAM" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/NetSAM.html">NetSAM</a></td>
	<td class=" ">Bing Zhang</td>
	<td class=" ">Network Seriation And Modularization</td>
	<td class="  sorting_1">1141</td>
</tr><tr id="pkg_PROMISE" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/PROMISE.html">PROMISE</a></td>
	<td class=" ">Stan Pounds, Xueyuan Cao</td>
	<td class=" ">PRojection Onto the Most Interesting Statistical Evidence</td>
	<td class="  sorting_1">1141</td>
</tr><tr id="pkg_PSEA" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/PSEA.html">PSEA</a></td>
	<td class=" ">Alexandre Kuhn</td>
	<td class=" ">Population-Specific Expression Analysis.</td>
	<td class="  sorting_1">1141</td>
</tr><tr id="pkg_RareVariantVis" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/RareVariantVis.html">RareVariantVis</a></td>
	<td class=" ">Tomasz Stokowy</td>
	<td class=" ">A suite for analysis of rare genomic variants in whole genome sequencing data</td>
	<td class="  sorting_1">1141</td>
</tr><tr id="pkg_sagenhaft" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/sagenhaft.html">sagenhaft</a></td>
	<td class=" ">Tim Beissbarth</td>
	<td class=" ">Collection of functions for reading and comparing SAGE libraries</td>
	<td class="  sorting_1">1141</td>
</tr><tr id="pkg_similaRpeak" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/similaRpeak.html">similaRpeak</a></td>
	<td class=" ">Astrid Deschenes</td>
	<td class=" ">Metrics to estimate a level of similarity between two ChIP-Seq profiles</td>
	<td class="  sorting_1">1141</td>
</tr><tr id="pkg_simulatorZ" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/simulatorZ.html">simulatorZ</a></td>
	<td class=" ">Yuqing Zhang</td>
	<td class=" ">Simulator for Collections of Independent Genomic Data Sets</td>
	<td class="  sorting_1">1141</td>
</tr><tr id="pkg_SLGI" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/SLGI.html">SLGI</a></td>
	<td class=" ">Nolwenn Le Meur</td>
	<td class=" ">Synthetic Lethal Genetic Interaction</td>
	<td class="  sorting_1">1141</td>
</tr><tr id="pkg_stepNorm" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/stepNorm.html">stepNorm</a></td>
	<td class=" ">Yuanyuan Xiao</td>
	<td class=" ">Stepwise normalization functions for cDNA microarrays</td>
	<td class="  sorting_1">1141</td>
</tr><tr id="pkg_Streamer" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/Streamer.html">Streamer</a></td>
	<td class=" ">Martin Morgan</td>
	<td class=" ">Enabling stream processing of large files</td>
	<td class="  sorting_1">1141</td>
</tr><tr id="pkg_timescape" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/timescape.html">timescape</a></td>
	<td class=" ">Maia Smith</td>
	<td class=" ">Patient Clonal Timescapes</td>
	<td class="  sorting_1">1141</td>
</tr><tr id="pkg_tofsims" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/tofsims.html">tofsims</a></td>
	<td class=" ">Lorenz Gerber</td>
	<td class=" ">Import, process and analysis of Time-of-Flight Secondary Ion Mass Spectrometry (ToF-SIMS) imaging data</td>
	<td class="  sorting_1">1141</td>
</tr><tr id="pkg_transcriptR" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/transcriptR.html">transcriptR</a></td>
	<td class=" ">Armen R. Karapetyan</td>
	<td class=" ">An Integrative Tool for ChIP- And RNA-Seq Based Primary Transcripts Detection and Quantification</td>
	<td class="  sorting_1">1141</td>
</tr><tr id="pkg_TurboNorm" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/TurboNorm.html">TurboNorm</a></td>
	<td class=" ">Maarten van Iterson</td>
	<td class=" ">A fast scatterplot smoother suitable for microarray normalization</td>
	<td class="  sorting_1">1141</td>
</tr><tr id="pkg_waveTiling" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/waveTiling.html">waveTiling</a></td>
	<td class=" ">Kristof De Beuf</td>
	<td class=" ">Wavelet-Based Models for Tiling Array Transcriptome Analysis</td>
	<td class="  sorting_1">1141</td>
</tr><tr id="pkg_bigmemoryExtras" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/bigmemoryExtras.html">bigmemoryExtras</a></td>
	<td class=" ">Peter M. Haverty</td>
	<td class=" ">An extension of the bigmemory package with added safety, convenience, and a factor class</td>
	<td class="  sorting_1">1174</td>
</tr><tr id="pkg_CancerInSilico" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/CancerInSilico.html">CancerInSilico</a></td>
	<td class=" ">Thomas D. Sherman, Elana J. Fertig</td>
	<td class=" ">An R interface for computational modeling of tumor progression</td>
	<td class="  sorting_1">1174</td>
</tr><tr id="pkg_CCPROMISE" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/CCPROMISE.html">CCPROMISE</a></td>
	<td class=" ">Xueyuan Cao</td>
	<td class=" ">PROMISE analysis with Canonical Correlation for Two Forms of High Dimensional Genetic Data</td>
	<td class="  sorting_1">1174</td>
</tr><tr id="pkg_clusterSeq" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/clusterSeq.html">clusterSeq</a></td>
	<td class=" ">Thomas J. Hardcastle</td>
	<td class=" ">Clustering of high-throughput sequencing data by identifying co-expression patterns</td>
	<td class="  sorting_1">1174</td>
</tr><tr id="pkg_consensusOV" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/consensusOV.html">consensusOV</a></td>
	<td class=" ">Benjamin Haibe-Kains</td>
	<td class=" ">Gene expression-based subtype classification for high-grade serous ovarian cancer</td>
	<td class="  sorting_1">1174</td>
</tr><tr id="pkg_deltaGseg" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/deltaGseg.html">deltaGseg</a></td>
	<td class=" ">Diana Low</td>
	<td class=" ">deltaGseg</td>
	<td class="  sorting_1">1174</td>
</tr><tr id="pkg_dks" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/dks.html">dks</a></td>
	<td class=" ">Jeffrey T. Leek</td>
	<td class=" ">The double Kolmogorov-Smirnov package for evaluating multiple testing procedures.</td>
	<td class="  sorting_1">1174</td>
</tr><tr id="pkg_FunChIP" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/FunChIP.html">FunChIP</a></td>
	<td class=" ">Alice Parodi</td>
	<td class=" ">Clustering and Alignment of ChIP-Seq peaks based on their shapes</td>
	<td class="  sorting_1">1174</td>
</tr><tr id="pkg_gcatest" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/gcatest.html">gcatest</a></td>
	<td class=" ">Wei Hao, John D. Storey</td>
	<td class=" ">Genotype Conditional Association TEST</td>
	<td class="  sorting_1">1174</td>
</tr><tr id="pkg_GEM" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/GEM.html">GEM</a></td>
	<td class=" ">Hong Pan</td>
	<td class=" ">GEM: fast association study for the interplay of Gene, Environment and Methylation</td>
	<td class="  sorting_1">1174</td>
</tr><tr id="pkg_geneplast" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/geneplast.html">geneplast</a></td>
	<td class=" ">Mauro Castro</td>
	<td class=" ">Evolutionary and plasticity analysis of orthologous groups</td>
	<td class="  sorting_1">1174</td>
</tr><tr id="pkg_globalSeq" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/globalSeq.html">globalSeq</a></td>
	<td class=" ">Armin Rauschenberger</td>
	<td class=" ">Global Test for Counts</td>
	<td class="  sorting_1">1174</td>
</tr><tr id="pkg_ibh" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/ibh.html">ibh</a></td>
	<td class=" ">Kircicegi Korkmaz</td>
	<td class=" ">Interaction Based Homogeneity for Evaluating Gene Lists</td>
	<td class="  sorting_1">1174</td>
</tr><tr id="pkg_inveRsion" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/inveRsion.html">inveRsion</a></td>
	<td class=" ">Alejandro Caceres</td>
	<td class=" ">Inversions in genotype data</td>
	<td class="  sorting_1">1174</td>
</tr><tr id="pkg_LINC" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/LINC.html">LINC</a></td>
	<td class=" ">Manuel Goepferich</td>
	<td class=" ">co-expression of lincRNAs and protein-coding genes</td>
	<td class="  sorting_1">1174</td>
</tr><tr id="pkg_MADSEQ" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/MADSEQ.html">MADSEQ</a></td>
	<td class=" ">Yu Kong</td>
	<td class=" ">Mosaic Aneuploidy Detection and Quantification using Massive Parallel Sequencing Data</td>
	<td class="  sorting_1">1174</td>
</tr><tr id="pkg_MCbiclust" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/MCbiclust.html">MCbiclust</a></td>
	<td class=" ">Robert Bentham</td>
	<td class=" ">Massive correlating biclusters for gene expression data and associated methods</td>
	<td class="  sorting_1">1174</td>
</tr><tr id="pkg_MGFR" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/MGFR.html">MGFR</a></td>
	<td class=" ">Khadija El Amrani</td>
	<td class=" ">Marker Gene Finder in RNA-seq data</td>
	<td class="  sorting_1">1174</td>
</tr><tr id="pkg_ndexr" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/ndexr.html">ndexr</a></td>
	<td class=" ">Florian Auer</td>
	<td class=" ">NDEx R client library</td>
	<td class="  sorting_1">1174</td>
</tr><tr id="pkg_OPWeight" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/OPWeight.html">OPWeight</a></td>
	<td class=" ">Mohamad Hasan</td>
	<td class=" ">Optimal p-value weighting with independent information</td>
	<td class="  sorting_1">1174</td>
</tr><tr id="pkg_pickgene" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/pickgene.html">pickgene</a></td>
	<td class=" ">Brian S. Yandell</td>
	<td class=" ">Adaptive Gene Picking for Microarray Expression Data Analysis</td>
	<td class="  sorting_1">1174</td>
</tr><tr id="pkg_PING" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/PING.html">PING</a></td>
	<td class=" ">Renan Sauteraud</td>
	<td class=" ">Probabilistic inference for Nucleosome Positioning with MNase-based or Sonicated Short-read Data</td>
	<td class="  sorting_1">1174</td>
</tr><tr id="pkg_Polyfit" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/Polyfit.html">Polyfit</a></td>
	<td class=" ">Conrad Burden</td>
	<td class=" ">Add-on to DESeq to improve p-values and q-values</td>
	<td class="  sorting_1">1174</td>
</tr><tr id="pkg_ReQON" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/ReQON.html">ReQON</a></td>
	<td class=" ">Christopher Cabanski</td>
	<td class=" ">Recalibrating Quality Of Nucleotides</td>
	<td class="  sorting_1">1174</td>
</tr><tr id="pkg_rgsepd" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/rgsepd.html">rgsepd</a></td>
	<td class=" ">Karl Stamm</td>
	<td class=" ">Gene Set Enrichment / Projection Displays</td>
	<td class="  sorting_1">1174</td>
</tr><tr id="pkg_Rnits" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/Rnits.html">Rnits</a></td>
	<td class=" ">Dipen P. Sangurdekar</td>
	<td class=" ">R Normalization and Inference of Time Series data</td>
	<td class="  sorting_1">1174</td>
</tr><tr id="pkg_traseR" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/traseR.html">traseR</a></td>
	<td class=" ">li chen</td>
	<td class=" ">GWAS trait-associated SNP enrichment analyses in genomic intervals</td>
	<td class="  sorting_1">1174</td>
</tr><tr id="pkg_UNDO" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/UNDO.html">UNDO</a></td>
	<td class=" ">Niya Wang</td>
	<td class=" ">Unsupervised Deconvolution of Tumor-Stromal Mixed Expressions</td>
	<td class="  sorting_1">1174</td>
</tr><tr id="pkg_cellbaseR" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/cellbaseR.html">cellbaseR</a></td>
	<td class=" ">Mohammed OE Abdallah</td>
	<td class=" ">Querying annotation data from the high performance Cellbase web</td>
	<td class="  sorting_1">1202</td>
</tr><tr id="pkg_CINdex" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/CINdex.html">CINdex</a></td>
	<td class=" ">Yuriy Gusev</td>
	<td class=" ">Chromosome Instability Index</td>
	<td class="  sorting_1">1202</td>
</tr><tr id="pkg_diffuStats" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/diffuStats.html">diffuStats</a></td>
	<td class=" ">Sergio Picart-Armada</td>
	<td class=" ">Diffusion scores on biological networks</td>
	<td class="  sorting_1">1202</td>
</tr><tr id="pkg_doppelgangR" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/doppelgangR.html">doppelgangR</a></td>
	<td class=" ">Levi Waldron</td>
	<td class=" ">Identify likely duplicate samples from genomic or meta-data</td>
	<td class="  sorting_1">1202</td>
</tr><tr id="pkg_fCCAC" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/fCCAC.html">fCCAC</a></td>
	<td class=" ">Pedro Madrigal</td>
	<td class=" ">functional Canonical Correlation Analysis to evaluate Covariance between nucleic acid sequencing datasets</td>
	<td class="  sorting_1">1202</td>
</tr><tr id="pkg_geneXtendeR" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/geneXtendeR.html">geneXtendeR</a></td>
	<td class=" ">Bohdan Khomtchouk</td>
	<td class=" ">Optimized Functional Annotation Of ChIP-seq Data</td>
	<td class="  sorting_1">1202</td>
</tr><tr id="pkg_goSTAG" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/goSTAG.html">goSTAG</a></td>
	<td class=" ">Brian D. Bennett</td>
	<td class=" ">A tool to use GO Subtrees to Tag and Annotate Genes within a set</td>
	<td class="  sorting_1">1202</td>
</tr><tr id="pkg_InPAS" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/InPAS.html">InPAS</a></td>
	<td class=" ">Jianhong Ou, Lihua Julie Zhu</td>
	<td class=" ">Identification of Novel alternative PolyAdenylation Sites (PAS)</td>
	<td class="  sorting_1">1202</td>
</tr><tr id="pkg_MatrixRider" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/MatrixRider.html">MatrixRider</a></td>
	<td class=" ">Elena Grassi</td>
	<td class=" ">Obtain total affinity and occupancies for binding site matrices on a given sequence</td>
	<td class="  sorting_1">1202</td>
</tr><tr id="pkg_mfa" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/mfa.html">mfa</a></td>
	<td class=" ">Kieran Campbell</td>
	<td class=" ">Bayesian hierarchical mixture of factor analyzers for modelling genomic bifurcations</td>
	<td class="  sorting_1">1202</td>
</tr><tr id="pkg_MGFM" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/MGFM.html">MGFM</a></td>
	<td class=" ">Khadija El Amrani</td>
	<td class=" ">Marker Gene Finder in Microarray gene expression data</td>
	<td class="  sorting_1">1202</td>
</tr><tr id="pkg_MIGSA" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/MIGSA.html">MIGSA</a></td>
	<td class=" ">Juan C. Rodriguez</td>
	<td class=" ">Massive and Integrative Gene Set Analysis</td>
	<td class="  sorting_1">1202</td>
</tr><tr id="pkg_MinimumDistance" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/MinimumDistance.html">MinimumDistance</a></td>
	<td class=" ">Robert B Scharpf</td>
	<td class=" ">A Package for De Novo CNV Detection in Case-Parent Trios</td>
	<td class="  sorting_1">1202</td>
</tr><tr id="pkg_NGScopy" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/NGScopy.html">NGScopy</a></td>
	<td class=" ">Xiaobei Zhao</td>
	<td class=" ">NGScopy: Detection of Copy Number Variations in Next Generation Sequencing sequencing</td>
	<td class="  sorting_1">1202</td>
</tr><tr id="pkg_PCpheno" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/PCpheno.html">PCpheno</a></td>
	<td class=" ">Nolwenn Le Meur</td>
	<td class=" ">Phenotypes and cellular organizational units</td>
	<td class="  sorting_1">1202</td>
</tr><tr id="pkg_phenopath" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/phenopath.html">phenopath</a></td>
	<td class=" ">Kieran Campbell</td>
	<td class=" ">Genomic trajectories with heterogeneous genetic and environmental backgrounds</td>
	<td class="  sorting_1">1202</td>
</tr><tr id="pkg_QuartPAC" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/QuartPAC.html">QuartPAC</a></td>
	<td class=" ">Gregory Ryslik</td>
	<td class=" ">Identification of mutational clusters in protein quaternary structures.</td>
	<td class="  sorting_1">1202</td>
</tr><tr id="pkg_RLMM" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/RLMM.html">RLMM</a></td>
	<td class=" ">Nusrat Rabbee</td>
	<td class=" ">A Genotype Calling Algorithm for Affymetrix SNP Arrays</td>
	<td class="  sorting_1">1202</td>
</tr><tr id="pkg_rMAT" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/rMAT.html">rMAT</a></td>
	<td class=" ">Arnaud Droit and Raphael Gottardo</td>
	<td class=" ">R implementation from MAT program to normalize and analyze tiling arrays and ChIP-chip data.</td>
	<td class="  sorting_1">1202</td>
</tr><tr id="pkg_RmiR" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/RmiR.html">RmiR</a></td>
	<td class=" ">Francesco Favero</td>
	<td class=" ">Package to work with miRNAs and miRNA targets with R</td>
	<td class="  sorting_1">1202</td>
</tr><tr id="pkg_RNAinteract" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/RNAinteract.html">RNAinteract</a></td>
	<td class=" ">Bernd Fischer</td>
	<td class=" ">Estimate Pairwise Interactions from multidimensional features</td>
	<td class="  sorting_1">1202</td>
</tr><tr id="pkg_RTNduals" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/RTNduals.html">RTNduals</a></td>
	<td class=" ">Mauro Castro, Clarice Groeneveld</td>
	<td class=" ">Analysis of co-regulation and inference of 'dual regulons'</td>
	<td class="  sorting_1">1202</td>
</tr><tr id="pkg_rTRMui" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/rTRMui.html">rTRMui</a></td>
	<td class=" ">Diego Diez</td>
	<td class=" ">A shiny user interface for rTRM</td>
	<td class="  sorting_1">1202</td>
</tr><tr id="pkg_SigCheck" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/SigCheck.html">SigCheck</a></td>
	<td class=" ">Rory Stark</td>
	<td class=" ">Check a gene signature's prognostic performance against random signatures, known signatures, and permuted data/metadata</td>
	<td class="  sorting_1">1202</td>
</tr><tr id="pkg_sights" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/sights.html">sights</a></td>
	<td class=" ">Elika Garg</td>
	<td class=" ">Statistics and dIagnostic Graphs for HTS</td>
	<td class="  sorting_1">1202</td>
</tr><tr id="pkg_slalom" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/slalom.html">slalom</a></td>
	<td class=" ">Davis McCarthy</td>
	<td class=" ">Factorial Latent Variable Modeling of Single-Cell RNA-Seq Data</td>
	<td class="  sorting_1">1202</td>
</tr><tr id="pkg_SNAGEE" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/SNAGEE.html">SNAGEE</a></td>
	<td class=" ">David Venet</td>
	<td class=" ">Signal-to-Noise applied to Gene Expression Experiments</td>
	<td class="  sorting_1">1202</td>
</tr><tr id="pkg_ssviz" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/ssviz.html">ssviz</a></td>
	<td class=" ">Diana Low</td>
	<td class=" ">A small RNA-seq visualizer and analysis toolkit</td>
	<td class="  sorting_1">1202</td>
</tr><tr id="pkg_StarBioTrek" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/StarBioTrek.html">StarBioTrek</a></td>
	<td class=" ">Claudia Cava</td>
	<td class=" ">StarBioTrek</td>
	<td class="  sorting_1">1202</td>
</tr><tr id="pkg_tRanslatome" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/tRanslatome.html">tRanslatome</a></td>
	<td class=" ">Toma Tebaldi, Erik Dassi</td>
	<td class=" ">Comparison between multiple levels of gene expression</td>
	<td class="  sorting_1">1202</td>
</tr><tr id="pkg_TSSi" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/TSSi.html">TSSi</a></td>
	<td class=" ">Julian Gehring</td>
	<td class=" ">Transcription Start Site Identification</td>
	<td class="  sorting_1">1202</td>
</tr><tr id="pkg_twoddpcr" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/twoddpcr.html">twoddpcr</a></td>
	<td class=" ">Anthony Chiu</td>
	<td class=" ">Classify 2-d Droplet Digital PCR (ddPCR) data and quantify the number of starting molecules</td>
	<td class="  sorting_1">1202</td>
</tr><tr id="pkg_BiocSklearn" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/BiocSklearn.html">BiocSklearn</a></td>
	<td class=" ">VJ Carey</td>
	<td class=" ">interface to python sklearn via Rstudio reticulate</td>
	<td class="  sorting_1">1234</td>
</tr><tr id="pkg_contiBAIT" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/contiBAIT.html">contiBAIT</a></td>
	<td class=" ">Kieran O'Neill</td>
	<td class=" ">Improves Early Build Genome Assemblies using Strand-Seq Data</td>
	<td class="  sorting_1">1234</td>
</tr><tr id="pkg_covRNA" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/covRNA.html">covRNA</a></td>
	<td class=" ">Lara Urban</td>
	<td class=" ">Multivariate Analysis of Transcriptomic Data</td>
	<td class="  sorting_1">1234</td>
</tr><tr id="pkg_DMRScan" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/DMRScan.html">DMRScan</a></td>
	<td class=" ">Christian M Page</td>
	<td class=" ">Detection of Differentially Methylated Regions</td>
	<td class="  sorting_1">1234</td>
</tr><tr id="pkg_geecc" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/geecc.html">geecc</a></td>
	<td class=" ">Markus Boenn</td>
	<td class=" ">Gene Set Enrichment Analysis Extended to Contingency Cubes</td>
	<td class="  sorting_1">1234</td>
</tr><tr id="pkg_geneRxCluster" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/geneRxCluster.html">geneRxCluster</a></td>
	<td class=" ">Charles Berry</td>
	<td class=" ">gRx Differential Clustering</td>
	<td class="  sorting_1">1234</td>
</tr><tr id="pkg_GEWIST" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/GEWIST.html">GEWIST</a></td>
	<td class=" ">Wei Q. Deng</td>
	<td class=" ">Gene Environment Wide Interaction Search Threshold</td>
	<td class="  sorting_1">1234</td>
</tr><tr id="pkg_GRridge" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/GRridge.html">GRridge</a></td>
	<td class=" ">Mark A. van de Wiel</td>
	<td class=" ">Better prediction by use of co-data: Adaptive group-regularized ridge regression</td>
	<td class="  sorting_1">1234</td>
</tr><tr id="pkg_iGC" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/iGC.html">iGC</a></td>
	<td class=" ">Liang-Bo Wang</td>
	<td class=" ">An integrated analysis package of Gene expression and Copy number alteration</td>
	<td class="  sorting_1">1234</td>
</tr><tr id="pkg_LedPred" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/LedPred.html">LedPred</a></td>
	<td class=" ">Aitor Gonzalez</td>
	<td class=" ">Learning from DNA to Predict Enhancers</td>
	<td class="  sorting_1">1234</td>
</tr><tr id="pkg_lpNet" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/lpNet.html">lpNet</a></td>
	<td class=" ">Lars Kaderali</td>
	<td class=" ">Linear Programming Model for Network Inference</td>
	<td class="  sorting_1">1234</td>
</tr><tr id="pkg_MantelCorr" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/MantelCorr.html">MantelCorr</a></td>
	<td class=" ">Brian Steinmeyer</td>
	<td class=" ">Compute Mantel Cluster Correlations</td>
	<td class="  sorting_1">1234</td>
</tr><tr id="pkg_mAPKL" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/mAPKL.html">mAPKL</a></td>
	<td class=" ">Argiris Sakellariou</td>
	<td class=" ">A Hybrid Feature Selection method for gene expression data</td>
	<td class="  sorting_1">1234</td>
</tr><tr id="pkg_MEAL" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/MEAL.html">MEAL</a></td>
	<td class=" ">Carlos Ruiz-Arenas</td>
	<td class=" ">Perform methylation analysis</td>
	<td class="  sorting_1">1234</td>
</tr><tr id="pkg_MeasurementError.cor" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/MeasurementError.cor.html">MeasurementError.cor</a></td>
	<td class=" ">Beiying Ding</td>
	<td class=" ">Measurement Error model estimate for correlation coefficient</td>
	<td class="  sorting_1">1234</td>
</tr><tr id="pkg_MiChip" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/MiChip.html">MiChip</a></td>
	<td class=" ">Jonathon Blake</td>
	<td class=" ">MiChip Parsing and Summarizing Functions</td>
	<td class="  sorting_1">1234</td>
</tr><tr id="pkg_MMDiff2" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/MMDiff2.html">MMDiff2</a></td>
	<td class=" ">Gabriele Schweikert</td>
	<td class=" ">Statistical Testing for ChIP-Seq data sets</td>
	<td class="  sorting_1">1234</td>
</tr><tr id="pkg_nucleoSim" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/nucleoSim.html">nucleoSim</a></td>
	<td class=" ">Astrid Deschenes</td>
	<td class=" ">Generate synthetic nucleosome maps</td>
	<td class="  sorting_1">1234</td>
</tr><tr id="pkg_pathVar" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/pathVar.html">pathVar</a></td>
	<td class=" ">Samuel Zimmerman</td>
	<td class=" ">Methods to Find Pathways with Significantly Different Variability</td>
	<td class="  sorting_1">1234</td>
</tr><tr id="pkg_plw" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/plw.html">plw</a></td>
	<td class=" ">Magnus Astrand</td>
	<td class=" ">Probe level Locally moderated Weighted t-tests.</td>
	<td class="  sorting_1">1234</td>
</tr><tr id="pkg_prebs" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/prebs.html">prebs</a></td>
	<td class=" ">Karolis Uziela</td>
	<td class=" ">Probe region expression estimation for RNA-seq data for improved microarray comparability</td>
	<td class="  sorting_1">1234</td>
</tr><tr id="pkg_Prize" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/Prize.html">Prize</a></td>
	<td class=" ">Daryanaz Dargahi</td>
	<td class=" ">Prize: an R package for prioritization estimation based on analytic hierarchy process</td>
	<td class="  sorting_1">1234</td>
</tr><tr id="pkg_qsea" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/qsea.html">qsea</a></td>
	<td class=" ">Matthias Lienhard</td>
	<td class=" ">IP-seq data analysis and vizualization</td>
	<td class="  sorting_1">1234</td>
</tr><tr id="pkg_RCASPAR" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/RCASPAR.html">RCASPAR</a></td>
	<td class=" ">Douaa Mugahid, Lars Kaderali</td>
	<td class=" ">A package for survival time prediction based on a piecewise baseline hazard Cox regression model.</td>
	<td class="  sorting_1">1234</td>
</tr><tr id="pkg_recoup" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/recoup.html">recoup</a></td>
	<td class=" ">Panagiotis Moulos</td>
	<td class=" ">An R package for the creation of complex genomic profile plots</td>
	<td class="  sorting_1">1234</td>
</tr><tr id="pkg_regsplice" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/regsplice.html">regsplice</a></td>
	<td class=" ">Lukas M. Weber</td>
	<td class=" ">L1-regularization based methods for detection of differential splicing</td>
	<td class="  sorting_1">1234</td>
</tr><tr id="pkg_RIVER" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/RIVER.html">RIVER</a></td>
	<td class=" ">Yungil Kim</td>
	<td class=" ">R package for RIVER (RNA-Informed Variant Effect on Regulation)</td>
	<td class="  sorting_1">1234</td>
</tr><tr id="pkg_rRDP" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/rRDP.html">rRDP</a></td>
	<td class=" ">Michael Hahsler</td>
	<td class=" ">Interface to the RDP Classifier</td>
	<td class="  sorting_1">1234</td>
</tr><tr id="pkg_SemDist" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/SemDist.html">SemDist</a></td>
	<td class=" ">Ian Gonzalez</td>
	<td class=" ">Information Accretion-based Function Predictor Evaluation</td>
	<td class="  sorting_1">1234</td>
</tr><tr id="pkg_splicegear" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/splicegear.html">splicegear</a></td>
	<td class=" ">Laurent Gautier</td>
	<td class=" ">splicegear</td>
	<td class="  sorting_1">1234</td>
</tr><tr id="pkg_SQUADD" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/SQUADD.html">SQUADD</a></td>
	<td class=" ">Martial Sankar</td>
	<td class=" ">Add-on of the SQUAD Software</td>
	<td class="  sorting_1">1234</td>
</tr><tr id="pkg_trigger" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/trigger.html">trigger</a></td>
	<td class=" ">John D. Storey</td>
	<td class=" ">Transcriptional Regulatory Inference from Genetics of Gene ExpRession</td>
	<td class="  sorting_1">1234</td>
</tr><tr id="pkg_ANF" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/ANF.html">ANF</a></td>
	<td class=" ">Tianle Ma</td>
	<td class=" ">Affinity Network Fusion for Complex Patient Clustering</td>
	<td class="  sorting_1">1266</td>
</tr><tr id="pkg_ASAFE" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/ASAFE.html">ASAFE</a></td>
	<td class=" ">Qian Zhang</td>
	<td class=" ">Ancestry Specific Allele Frequency Estimation</td>
	<td class="  sorting_1">1266</td>
</tr><tr id="pkg_BayesKnockdown" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/BayesKnockdown.html">BayesKnockdown</a></td>
	<td class=" ">William Chad Young</td>
	<td class=" ">BayesKnockdown: Posterior Probabilities for Edges from Knockdown Data</td>
	<td class="  sorting_1">1266</td>
</tr><tr id="pkg_BUMHMM" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/BUMHMM.html">BUMHMM</a></td>
	<td class=" ">Alina Selega</td>
	<td class=" ">Computational pipeline for computing probability of modification from structure probing experiment data</td>
	<td class="  sorting_1">1266</td>
</tr><tr id="pkg_CellMapper" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/CellMapper.html">CellMapper</a></td>
	<td class=" ">Brad Nelms</td>
	<td class=" ">Predict genes expressed selectively in specific cell types</td>
	<td class="  sorting_1">1266</td>
</tr><tr id="pkg_EBSEA" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/EBSEA.html">EBSEA</a></td>
	<td class=" ">Arfa Mehmood</td>
	<td class=" ">Exon Based Strategy for Expression Analysis of genes</td>
	<td class="  sorting_1">1266</td>
</tr><tr id="pkg_eegc" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/eegc.html">eegc</a></td>
	<td class=" ">Xiaoyuan Zhou</td>
	<td class=" ">Engineering Evaluation by Gene Categorization (eegc)</td>
	<td class="  sorting_1">1266</td>
</tr><tr id="pkg_GA4GHclient" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/GA4GHclient.html">GA4GHclient</a></td>
	<td class=" ">Welliton Souza</td>
	<td class=" ">A Bioconductor package for accessing GA4GH API data servers</td>
	<td class="  sorting_1">1266</td>
</tr><tr id="pkg_GA4GHshiny" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/GA4GHshiny.html">GA4GHshiny</a></td>
	<td class=" ">Welliton Souza</td>
	<td class=" ">Shiny application for interacting with GA4GH-based data servers</td>
	<td class="  sorting_1">1266</td>
</tr><tr id="pkg_iASeq" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/iASeq.html">iASeq</a></td>
	<td class=" ">Yingying Wei</td>
	<td class=" ">iASeq: integrating multiple sequencing datasets for detecting allele-specific events</td>
	<td class="  sorting_1">1266</td>
</tr><tr id="pkg_lol" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/lol.html">lol</a></td>
	<td class=" ">Yinyin Yuan</td>
	<td class=" ">Lots Of Lasso</td>
	<td class="  sorting_1">1266</td>
</tr><tr id="pkg_M3D" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/M3D.html">M3D</a></td>
	<td class=" ">Tom Mayo</td>
	<td class=" ">Identifies differentially methylated regions across testing groups</td>
	<td class="  sorting_1">1266</td>
</tr><tr id="pkg_matchBox" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/matchBox.html">matchBox</a></td>
	<td class=" ">Luigi Marchionni, Anuj Gupta</td>
	<td class=" ">Utilities to compute, compare, and plot the agreement between ordered vectors of features (ie. distinct genomic experiments). The package includes Correspondence-At-the-TOP (CAT) analysis.</td>
	<td class="  sorting_1">1266</td>
</tr><tr id="pkg_MEDME" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/MEDME.html">MEDME</a></td>
	<td class=" ">Mattia Pelizzola</td>
	<td class=" ">Modelling Experimental Data from MeDIP Enrichment</td>
	<td class="  sorting_1">1266</td>
</tr><tr id="pkg_metahdep" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/metahdep.html">metahdep</a></td>
	<td class=" ">John R. Stevens</td>
	<td class=" ">Hierarchical Dependence in Meta-Analysis</td>
	<td class="  sorting_1">1266</td>
</tr><tr id="pkg_methylInheritance" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/methylInheritance.html">methylInheritance</a></td>
	<td class=" ">Astrid Deschenes</td>
	<td class=" ">Permutation-Based Analysis associating Conserved Differentially Methylated Elements Across Multiple Generations to a Treatment Effect</td>
	<td class="  sorting_1">1266</td>
</tr><tr id="pkg_methylMnM" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/methylMnM.html">methylMnM</a></td>
	<td class=" ">Yan Zhou</td>
	<td class=" ">detect different methylation level (DMR)</td>
	<td class="  sorting_1">1266</td>
</tr><tr id="pkg_MIRA" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/MIRA.html">MIRA</a></td>
	<td class=" ">John Lawson</td>
	<td class=" ">Methylation-Based Inference of Regulatory Activity</td>
	<td class="  sorting_1">1266</td>
</tr><tr id="pkg_multiscan" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/multiscan.html">multiscan</a></td>
	<td class=" ">Mizanur Khondoker</td>
	<td class=" ">R package for combining multiple scans</td>
	<td class="  sorting_1">1266</td>
</tr><tr id="pkg_plrs" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/plrs.html">plrs</a></td>
	<td class=" ">Gwenael G.R. Leday to</td>
	<td class=" ">Piecewise Linear Regression Splines (PLRS) for the association between DNA copy number and gene expression</td>
	<td class="  sorting_1">1266</td>
</tr><tr id="pkg_plyranges" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/plyranges.html">plyranges</a></td>
	<td class=" ">Stuart Lee</td>
	<td class=" ">A fluent interface for manipulating GenomicRanges</td>
	<td class="  sorting_1">1266</td>
</tr><tr id="pkg_RefNet" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/RefNet.html">RefNet</a></td>
	<td class=" ">Paul Shannon</td>
	<td class=" ">A queryable collection of molecular interactions, from many sources</td>
	<td class="  sorting_1">1266</td>
</tr><tr id="pkg_RNAprobR" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/RNAprobR.html">RNAprobR</a></td>
	<td class=" ">Nikos Sidiropoulos</td>
	<td class=" ">An R package for analysis of massive parallel sequencing based RNA structure probing data</td>
	<td class="  sorting_1">1266</td>
</tr><tr id="pkg_banocc" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/banocc.html">banocc</a></td>
	<td class=" ">George Weingart, Curtis Huttenhower</td>
	<td class=" ">Bayesian ANalysis Of Compositional Covariance</td>
	<td class="  sorting_1">1289</td>
</tr><tr id="pkg_branchpointer" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/branchpointer.html">branchpointer</a></td>
	<td class=" ">Beth Signal</td>
	<td class=" ">Prediction of intronic splicing branchpoints</td>
	<td class="  sorting_1">1289</td>
</tr><tr id="pkg_CONFESS" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/CONFESS.html">CONFESS</a></td>
	<td class=" ">Diana LOW</td>
	<td class=" ">Cell OrderiNg by FluorEScence Signal</td>
	<td class="  sorting_1">1289</td>
</tr><tr id="pkg_dcGSA" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/dcGSA.html">dcGSA</a></td>
	<td class=" ">Jiehuan sun</td>
	<td class=" ">Distance-correlation based Gene Set Analysis for longitudinal gene expression profiles</td>
	<td class="  sorting_1">1289</td>
</tr><tr id="pkg_eudysbiome" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/eudysbiome.html">eudysbiome</a></td>
	<td class=" ">Xiaoyuan Zhou</td>
	<td class=" ">Cartesian plot and contingency test on 16S Microbial data</td>
	<td class="  sorting_1">1289</td>
</tr><tr id="pkg_garfield" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/garfield.html">garfield</a></td>
	<td class=" ">Valentina Iotchkova</td>
	<td class=" ">GWAS Analysis of Regulatory or Functional Information Enrichment with LD correction</td>
	<td class="  sorting_1">1289</td>
</tr><tr id="pkg_IdMappingAnalysis" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/IdMappingAnalysis.html">IdMappingAnalysis</a></td>
	<td class=" ">Alex Lisovich, Roger Day</td>
	<td class=" ">ID Mapping Analysis</td>
	<td class="  sorting_1">1289</td>
</tr><tr id="pkg_IdMappingRetrieval" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/IdMappingRetrieval.html">IdMappingRetrieval</a></td>
	<td class=" ">Alex Lisovich, Roger Day</td>
	<td class=" ">ID Mapping Data Retrieval</td>
	<td class="  sorting_1">1289</td>
</tr><tr id="pkg_INPower" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/INPower.html">INPower</a></td>
	<td class=" ">Bill Wheeler</td>
	<td class=" ">An R package for computing the number of susceptibility SNPs</td>
	<td class="  sorting_1">1289</td>
</tr><tr id="pkg_IWTomics" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/IWTomics.html">IWTomics</a></td>
	<td class=" ">Marzia A Cremona</td>
	<td class=" ">Interval-Wise Testing for Omics Data</td>
	<td class="  sorting_1">1289</td>
</tr><tr id="pkg_KCsmart" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/KCsmart.html">KCsmart</a></td>
	<td class=" ">Jorma de Ronde</td>
	<td class=" ">Multi sample aCGH analysis package using kernel convolution</td>
	<td class="  sorting_1">1289</td>
</tr><tr id="pkg_LowMACA" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/LowMACA.html">LowMACA</a></td>
	<td class=" ">Stefano de Pretis, Giorgio Melloni</td>
	<td class=" ">LowMACA - Low frequency Mutation Analysis via Consensus Alignment</td>
	<td class="  sorting_1">1289</td>
</tr><tr id="pkg_messina" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/messina.html">messina</a></td>
	<td class=" ">Mark Pinese</td>
	<td class=" ">Single-gene classifiers and outlier-resistant detection of differential expression for two-group and survival problems.</td>
	<td class="  sorting_1">1289</td>
</tr><tr id="pkg_MetaCyto" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/MetaCyto.html">MetaCyto</a></td>
	<td class=" ">Zicheng Hu</td>
	<td class=" ">MetaCyto: A package for meta-analysis of cytometry data</td>
	<td class="  sorting_1">1289</td>
</tr><tr id="pkg_MethTargetedNGS" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/MethTargetedNGS.html">MethTargetedNGS</a></td>
	<td class=" ">Muhammad Ahmer Jamil</td>
	<td class=" ">Perform Methylation Analysis on Next Generation Sequencing Data</td>
	<td class="  sorting_1">1289</td>
</tr><tr id="pkg_MoPS" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/MoPS.html">MoPS</a></td>
	<td class=" ">Philipp Eser</td>
	<td class=" ">MoPS - Model-based Periodicity Screening</td>
	<td class="  sorting_1">1289</td>
</tr><tr id="pkg_MultiMed" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/MultiMed.html">MultiMed</a></td>
	<td class=" ">Simina M. Boca</td>
	<td class=" ">Testing multiple biological mediators simultaneously</td>
	<td class="  sorting_1">1289</td>
</tr><tr id="pkg_OGSA" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/OGSA.html">OGSA</a></td>
	<td class=" ">Michael F. Ochs</td>
	<td class=" ">Outlier Gene Set Analysis</td>
	<td class="  sorting_1">1289</td>
</tr><tr id="pkg_plethy" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/plethy.html">plethy</a></td>
	<td class=" ">Daniel Bottomly</td>
	<td class=" ">R framework for exploration and analysis of respirometry data</td>
	<td class="  sorting_1">1289</td>
</tr><tr id="pkg_R3CPET" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/R3CPET.html">R3CPET</a></td>
	<td class=" ">Mohamed Nadhir Djekidel</td>
	<td class=" ">3CPET: Finding Co-factor Complexes in Chia-PET experiment using a Hierarchical Dirichlet Process</td>
	<td class="  sorting_1">1289</td>
</tr><tr id="pkg_reb" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/reb.html">reb</a></td>
	<td class=" ">Karl J. Dykema</td>
	<td class=" ">Regional Expression Biases</td>
	<td class="  sorting_1">1289</td>
</tr><tr id="pkg_rHVDM" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/rHVDM.html">rHVDM</a></td>
	<td class=" ">Martino Barenco</td>
	<td class=" ">Hidden Variable Dynamic Modeling</td>
	<td class="  sorting_1">1289</td>
</tr><tr id="pkg_sapFinder" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/sapFinder.html">sapFinder</a></td>
	<td class=" ">Shaohang Xu, Bo Wen</td>
	<td class=" ">A package for variant peptides detection and visualization in shotgun proteomics.</td>
	<td class="  sorting_1">1289</td>
</tr><tr id="pkg_scsR" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/scsR.html">scsR</a></td>
	<td class=" ">Andrea Franceschini, Roger Meier, Christian von Mering</td>
	<td class=" ">SiRNA correction for seed mediated off-target effect</td>
	<td class="  sorting_1">1289</td>
</tr><tr id="pkg_SEPA" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/SEPA.html">SEPA</a></td>
	<td class=" ">Zhicheng Ji</td>
	<td class=" ">SEPA</td>
	<td class="  sorting_1">1289</td>
</tr><tr id="pkg_SigFuge" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/SigFuge.html">SigFuge</a></td>
	<td class=" ">Patrick Kimes</td>
	<td class=" ">SigFuge</td>
	<td class="  sorting_1">1289</td>
</tr><tr id="pkg_staRank" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/staRank.html">staRank</a></td>
	<td class=" ">Juliane Siebourg</td>
	<td class=" ">Stability Ranking</td>
	<td class="  sorting_1">1289</td>
</tr><tr id="pkg_TVTB" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/TVTB.html">TVTB</a></td>
	<td class=" ">Kevin Rue-Albrecht</td>
	<td class=" ">TVTB: The VCF Tool Box</td>
	<td class="  sorting_1">1289</td>
</tr><tr id="pkg_TypeInfo" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/TypeInfo.html">TypeInfo</a></td>
	<td class=" ">Duncan Temple Lang</td>
	<td class=" ">Optional Type Specification Prototype</td>
	<td class="  sorting_1">1289</td>
</tr><tr id="pkg_unifiedWMWqPCR" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/unifiedWMWqPCR.html">unifiedWMWqPCR</a></td>
	<td class=" ">Joris Meys</td>
	<td class=" ">Unified Wilcoxon-Mann Whitney Test for testing differential expression in qPCR data</td>
	<td class="  sorting_1">1289</td>
</tr><tr id="pkg_VegaMC" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/VegaMC.html">VegaMC</a></td>
	<td class=" ">Sandro Morganella</td>
	<td class=" ">VegaMC: A Package Implementing a Variational Piecewise Smooth Model for Identification of Driver Chromosomal Imbalances in Cancer</td>
	<td class="  sorting_1">1289</td>
</tr><tr id="pkg_XDE" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/XDE.html">XDE</a></td>
	<td class=" ">Robert Scharpf</td>
	<td class=" ">XDE: a Bayesian hierarchical model for cross-study analysis of differential gene expression</td>
	<td class="  sorting_1">1289</td>
</tr><tr id="pkg_xmapbridge" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/xmapbridge.html">xmapbridge</a></td>
	<td class=" ">Chris Wirth</td>
	<td class=" ">Export plotting files to the xmapBridge for visualisation in X:Map</td>
	<td class="  sorting_1">1289</td>
</tr><tr id="pkg_DMCHMM" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/DMCHMM.html">DMCHMM</a></td>
	<td class=" ">Farhad Shokoohi</td>
	<td class=" ">Differentially Methylated CpG using Hidden Markov Model</td>
	<td class="  sorting_1">1322</td>
</tr><tr id="pkg_geneAttribution" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/geneAttribution.html">geneAttribution</a></td>
	<td class=" ">Arthur Wuster</td>
	<td class=" ">Identification of candidate genes associated with genetic variation</td>
	<td class="  sorting_1">1322</td>
</tr><tr id="pkg_GSALightning" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/GSALightning.html">GSALightning</a></td>
	<td class=" ">Billy Heung Wing Chang</td>
	<td class=" ">Fast Permutation-based Gene Set Analysis</td>
	<td class="  sorting_1">1322</td>
</tr><tr id="pkg_HilbertVisGUI" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/HilbertVisGUI.html">HilbertVisGUI</a></td>
	<td class=" ">Simon Anders</td>
	<td class=" ">HilbertVisGUI</td>
	<td class="  sorting_1">1322</td>
</tr><tr id="pkg_iSEE" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/iSEE.html">iSEE</a></td>
	<td class=" ">Charlotte Soneson</td>
	<td class=" ">Interactive SummarizedExperiment Explorer</td>
	<td class="  sorting_1">1322</td>
</tr><tr id="pkg_ISoLDE" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/ISoLDE.html">ISoLDE</a></td>
	<td class=" ">Christelle Reyns</td>
	<td class=" ">Integrative Statistics of alleLe Dependent Expression</td>
	<td class="  sorting_1">1322</td>
</tr><tr id="pkg_kimod" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/kimod.html">kimod</a></td>
	<td class=" ">M L Zingaretti</td>
	<td class=" ">A k-tables approach to integrate multiple Omics-Data</td>
	<td class="  sorting_1">1322</td>
</tr><tr id="pkg_M3C" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/M3C.html">M3C</a></td>
	<td class=" ">Christopher John</td>
	<td class=" ">Monte Carlo Reference-based Consensus Clustering</td>
	<td class="  sorting_1">1322</td>
</tr><tr id="pkg_miRcomp" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/miRcomp.html">miRcomp</a></td>
	<td class=" ">Matthew N. McCall</td>
	<td class=" ">Tools to assess and compare miRNA expression estimatation methods</td>
	<td class="  sorting_1">1322</td>
</tr><tr id="pkg_mitoODE" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/mitoODE.html">mitoODE</a></td>
	<td class=" ">Gregoire Pau</td>
	<td class=" ">Implementation of the differential equation model described in "Dynamical modelling of phenotypes in a genome-wide RNAi live-cell imaging assay"</td>
	<td class="  sorting_1">1322</td>
</tr><tr id="pkg_mpra" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/mpra.html">mpra</a></td>
	<td class=" ">Leslie Myint</td>
	<td class=" ">Analyze massively parallel reporter assays</td>
	<td class="  sorting_1">1322</td>
</tr><tr id="pkg_NTW" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/NTW.html">NTW</a></td>
	<td class=" ">Yuanhua Liu</td>
	<td class=" ">Predict gene network using an Ordinary Differential Equation (ODE) based method</td>
	<td class="  sorting_1">1322</td>
</tr><tr id="pkg_occugene" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/occugene.html">occugene</a></td>
	<td class=" ">Oliver Will</td>
	<td class=" ">Functions for Multinomial Occupancy Distribution</td>
	<td class="  sorting_1">1322</td>
</tr><tr id="pkg_PCAN" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/PCAN.html">PCAN</a></td>
	<td class=" ">Matthew Page and Patrice Godard</td>
	<td class=" ">Phenotype Consensus ANalysis (PCAN)</td>
	<td class="  sorting_1">1322</td>
</tr><tr id="pkg_proteinProfiles" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/proteinProfiles.html">proteinProfiles</a></td>
	<td class=" ">Julian Gehring</td>
	<td class=" ">Protein Profiling</td>
	<td class="  sorting_1">1322</td>
</tr><tr id="pkg_RBM" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/RBM.html">RBM</a></td>
	<td class=" ">Dongmei Li</td>
	<td class=" ">RBM: a R package for microarray and RNA-Seq data analysis</td>
	<td class="  sorting_1">1322</td>
</tr><tr id="pkg_rqt" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/rqt.html">rqt</a></td>
	<td class=" ">Ilya Y. Zhbannikov</td>
	<td class=" ">rqt: utilities for gene-level meta-analysis</td>
	<td class="  sorting_1">1322</td>
</tr><tr id="pkg_sampleClassifier" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/sampleClassifier.html">sampleClassifier</a></td>
	<td class=" ">Khadija El Amrani</td>
	<td class=" ">Sample Classifier</td>
	<td class="  sorting_1">1322</td>
</tr><tr id="pkg_SimBindProfiles" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/SimBindProfiles.html">SimBindProfiles</a></td>
	<td class=" ">Bettina Fischer</td>
	<td class=" ">Similar Binding Profiles</td>
	<td class="  sorting_1">1322</td>
</tr><tr id="pkg_sscu" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/sscu.html">sscu</a></td>
	<td class=" ">Yu Sun</td>
	<td class=" ">Strength of Selected Codon Usage</td>
	<td class="  sorting_1">1322</td>
</tr><tr id="pkg_ToPASeq" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/ToPASeq.html">ToPASeq</a></td>
	<td class=" ">Ludwig Geistlinger</td>
	<td class=" ">Topology-based pathway analysis of RNA-seq data</td>
	<td class="  sorting_1">1322</td>
</tr><tr id="pkg_triform" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/triform.html">triform</a></td>
	<td class=" ">Thomas Carroll</td>
	<td class=" ">Triform finds enriched regions (peaks) in transcription factor ChIP-sequencing data</td>
	<td class="  sorting_1">1322</td>
</tr><tr id="pkg_Vega" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/Vega.html">Vega</a></td>
	<td class=" ">Sandro Morganella</td>
	<td class=" ">An R package for copy number data segmentation</td>
	<td class="  sorting_1">1322</td>
</tr><tr id="pkg_bnbc" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/bnbc.html">bnbc</a></td>
	<td class=" ">Kipper Fletez-Brant</td>
	<td class=" ">Bandwise normalization and batch correction of Hi-C data</td>
	<td class="  sorting_1">1345</td>
</tr><tr id="pkg_chromswitch" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/chromswitch.html">chromswitch</a></td>
	<td class=" ">Selin Jessa</td>
	<td class=" ">An R package to detect chromatin state switches from epigenomic data</td>
	<td class="  sorting_1">1345</td>
</tr><tr id="pkg_covEB" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/covEB.html">covEB</a></td>
	<td class=" ">C. Pacini</td>
	<td class=" ">Empirical Bayes estimate of block diagonal covariance matrices</td>
	<td class="  sorting_1">1345</td>
</tr><tr id="pkg_eiR" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/eiR.html">eiR</a></td>
	<td class=" ">Thomas Girke</td>
	<td class=" ">Accelerated similarity searching of small molecules</td>
	<td class="  sorting_1">1345</td>
</tr><tr id="pkg_epiNEM" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/epiNEM.html">epiNEM</a></td>
	<td class=" ">Martin Pirkl</td>
	<td class=" ">epiNEM</td>
	<td class="  sorting_1">1345</td>
</tr><tr id="pkg_GISPA" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/GISPA.html">GISPA</a></td>
	<td class=" ">Bhakti Dwivedi</td>
	<td class=" ">GISPA: Method for Gene Integrated Set Profile Analysis</td>
	<td class="  sorting_1">1345</td>
</tr><tr id="pkg_iBMQ" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/iBMQ.html">iBMQ</a></td>
	<td class=" ">Greg Imholte</td>
	<td class=" ">integrated Bayesian Modeling of eQTL data</td>
	<td class="  sorting_1">1345</td>
</tr><tr id="pkg_IMAS" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/IMAS.html">IMAS</a></td>
	<td class=" ">Seonggyun Han</td>
	<td class=" ">Integrative analysis of Multi-omics data for Alternative Splicing</td>
	<td class="  sorting_1">1345</td>
</tr><tr id="pkg_Imetagene" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/Imetagene.html">Imetagene</a></td>
	<td class=" ">Audrey Lemacon</td>
	<td class=" ">A graphical interface for the metagene package</td>
	<td class="  sorting_1">1345</td>
</tr><tr id="pkg_IMPCdata" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/IMPCdata.html">IMPCdata</a></td>
	<td class=" ">Jeremy Mason</td>
	<td class=" ">Retrieves data from IMPC database</td>
	<td class="  sorting_1">1345</td>
</tr><tr id="pkg_mapscape" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/mapscape.html">mapscape</a></td>
	<td class=" ">Maia Smith</td>
	<td class=" ">mapscape</td>
	<td class="  sorting_1">1345</td>
</tr><tr id="pkg_MSstatsQC" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/MSstatsQC.html">MSstatsQC</a></td>
	<td class=" ">Eralp Dogu</td>
	<td class=" ">Longitudinal system suitability monitoring and quality control for proteomic experiments</td>
	<td class="  sorting_1">1345</td>
</tr><tr id="pkg_multiOmicsViz" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/multiOmicsViz.html">multiOmicsViz</a></td>
	<td class=" ">Jing Wang</td>
	<td class=" ">Plot the effect of one omics data on other omics data along the chromosome</td>
	<td class="  sorting_1">1345</td>
</tr><tr id="pkg_panelcn.mops" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/panelcn.mops.html">panelcn.mops</a></td>
	<td class=" ">Gundula Povysil</td>
	<td class=" ">CNV detection tool for targeted NGS panel data</td>
	<td class="  sorting_1">1345</td>
</tr><tr id="pkg_parglms" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/parglms.html">parglms</a></td>
	<td class=" ">VJ Carey</td>
	<td class=" ">support for parallelized estimation of GLMs/GEEs</td>
	<td class="  sorting_1">1345</td>
</tr><tr id="pkg_ProCoNA" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/ProCoNA.html">ProCoNA</a></td>
	<td class=" ">David L Gibbs</td>
	<td class=" ">Protein co-expression network analysis (ProCoNA).</td>
	<td class="  sorting_1">1345</td>
</tr><tr id="pkg_RITAN" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/RITAN.html">RITAN</a></td>
	<td class=" ">Michael Zimmermann</td>
	<td class=" ">Rapid Integration of Term Annotation and Network resources</td>
	<td class="  sorting_1">1345</td>
</tr><tr id="pkg_SMAP" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/SMAP.html">SMAP</a></td>
	<td class=" ">Robin Andersson</td>
	<td class=" ">A Segmental Maximum A Posteriori Approach to Array-CGH Copy Number Profiling</td>
	<td class="  sorting_1">1345</td>
</tr><tr id="pkg_SwathXtend" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/SwathXtend.html">SwathXtend</a></td>
	<td class=" ">Jemma Wu</td>
	<td class=" ">SWATH extended library generation and statistical data analysis</td>
	<td class="  sorting_1">1345</td>
</tr><tr id="pkg_synlet" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/synlet.html">synlet</a></td>
	<td class=" ">Chunxuan Shao</td>
	<td class=" ">Hits Selection for Synthetic Lethal RNAi Screen Data</td>
	<td class="  sorting_1">1345</td>
</tr><tr id="pkg_TIN" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/TIN.html">TIN</a></td>
	<td class=" ">Bjarne Johannessen</td>
	<td class=" ">Transcriptome instability analysis</td>
	<td class="  sorting_1">1345</td>
</tr><tr id="pkg_facopy" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/facopy.html">facopy</a></td>
	<td class=" ">David Mosen-Ansorena</td>
	<td class=" ">Feature-based association and gene-set enrichment for copy number alteration analysis in cancer</td>
	<td class="  sorting_1">1366</td>
</tr><tr id="pkg_iterClust" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/iterClust.html">iterClust</a></td>
	<td class=" ">Hongxu Ding</td>
	<td class=" ">Iterative Clustering</td>
	<td class="  sorting_1">1366</td>
</tr><tr id="pkg_methimpute" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/methimpute.html">methimpute</a></td>
	<td class=" ">Aaron Taudt</td>
	<td class=" ">Imputation-guided re-construction of complete methylomes from WGBS data</td>
	<td class="  sorting_1">1366</td>
</tr><tr id="pkg_methInheritSim" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/methInheritSim.html">methInheritSim</a></td>
	<td class=" ">Pascal Belleau</td>
	<td class=" ">Simulating Whole-Genome Inherited Bisulphite Sequencing Data</td>
	<td class="  sorting_1">1366</td>
</tr><tr id="pkg_netprioR" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/netprioR.html">netprioR</a></td>
	<td class=" ">Fabian Schmich</td>
	<td class=" ">A model for network-based prioritisation of genes</td>
	<td class="  sorting_1">1366</td>
</tr><tr id="pkg_odseq" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/odseq.html">odseq</a></td>
	<td class=" ">Jos Jimnez</td>
	<td class=" ">Outlier detection in multiple sequence alignments</td>
	<td class="  sorting_1">1366</td>
</tr><tr id="pkg_oppar" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/oppar.html">oppar</a></td>
	<td class=" ">Soroor Hediyeh zadeh</td>
	<td class=" ">Outlier profile and pathway analysis in R</td>
	<td class="  sorting_1">1366</td>
</tr><tr id="pkg_pbcmc" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/pbcmc.html">pbcmc</a></td>
	<td class=" ">Cristobal Fresno</td>
	<td class=" ">Permutation-Based Confidence for Molecular Classification</td>
	<td class="  sorting_1">1366</td>
</tr><tr id="pkg_RGraph2js" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/RGraph2js.html">RGraph2js</a></td>
	<td class=" ">Stephane Cano</td>
	<td class=" ">Convert a Graph into a D3js Script</td>
	<td class="  sorting_1">1366</td>
</tr><tr id="pkg_RTNsurvival" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/RTNsurvival.html">RTNsurvival</a></td>
	<td class=" ">Clarice Groeneveld, Mauro A. A. Castro</td>
	<td class=" ">Survival analysis using transcriptional networks inferred by the RTN package</td>
	<td class="  sorting_1">1366</td>
</tr><tr id="pkg_semisup" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/semisup.html">semisup</a></td>
	<td class=" ">Armin Rauschenberger</td>
	<td class=" ">Semi-Supervised Mixture Model</td>
	<td class="  sorting_1">1366</td>
</tr><tr id="pkg_sigsquared" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/sigsquared.html">sigsquared</a></td>
	<td class=" ">UnJin Lee</td>
	<td class=" ">Gene signature generation for functionally validated signaling pathways</td>
	<td class="  sorting_1">1366</td>
</tr><tr id="pkg_spikeLI" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/spikeLI.html">spikeLI</a></td>
	<td class=" ">Enrico Carlon</td>
	<td class=" ">Affymetrix Spike-in Langmuir Isotherm Data Analysis Tool</td>
	<td class="  sorting_1">1366</td>
</tr><tr id="pkg_SPLINTER" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/SPLINTER.html">SPLINTER</a></td>
	<td class=" ">Diana Low</td>
	<td class=" ">Splice Interpreter Of Transcripts</td>
	<td class="  sorting_1">1366</td>
</tr><tr id="pkg_SVAPLSseq" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/SVAPLSseq.html">SVAPLSseq</a></td>
	<td class=" ">Sutirtha Chakraborty</td>
	<td class=" ">SVAPLSseq-An R package to estimate the hidden factors of unwanted variability and adjust for them to enable a more powerful and accurate differential expression analysis based on RNAseq data</td>
	<td class="  sorting_1">1366</td>
</tr><tr id="pkg_SVM2CRM" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/SVM2CRM.html">SVM2CRM</a></td>
	<td class=" ">Guidantonio Malagoli Tagliazucchi</td>
	<td class=" ">SVM2CRM: support vector machine for cis-regulatory elements detections</td>
	<td class="  sorting_1">1366</td>
</tr><tr id="pkg_SwimR" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/SwimR.html">SwimR</a></td>
	<td class=" ">Randy Blakely</td>
	<td class=" ">SwimR: A Suite of Analytical Tools for Quantification of C. elegans Swimming Behavior</td>
	<td class="  sorting_1">1366</td>
</tr><tr id="pkg_ternarynet" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/ternarynet.html">ternarynet</a></td>
	<td class=" ">Matthew N. McCall</td>
	<td class=" ">Ternary Network Estimation</td>
	<td class="  sorting_1">1366</td>
</tr><tr id="pkg_TnT" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/TnT.html">TnT</a></td>
	<td class=" ">Jialin Ma</td>
	<td class=" ">Interactive Visualization for Genomic Features</td>
	<td class="  sorting_1">1366</td>
</tr><tr id="pkg_uSORT" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/uSORT.html">uSORT</a></td>
	<td class=" ">Hao Chen</td>
	<td class=" ">uSORT: A self-refining ordering pipeline for gene selection</td>
	<td class="  sorting_1">1366</td>
</tr><tr id="pkg_cbaf" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/cbaf.html">cbaf</a></td>
	<td class=" ">Arman Shahrisa</td>
	<td class=" ">Multiple automated functions for cbioportal.org</td>
	<td class="  sorting_1">1386</td>
</tr><tr id="pkg_flowTime" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/flowTime.html">flowTime</a></td>
	<td class=" ">R. Clay Wright</td>
	<td class=" ">Annotation and analysis of biological dynamical systems using flow cytometry</td>
	<td class="  sorting_1">1386</td>
</tr><tr id="pkg_GAprediction" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/GAprediction.html">GAprediction</a></td>
	<td class=" ">Jon Bohlin</td>
	<td class=" ">Prediction of gestational age with Illumina HumanMethylation450 data</td>
	<td class="  sorting_1">1386</td>
</tr><tr id="pkg_GeneGA" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/GeneGA.html">GeneGA</a></td>
	<td class=" ">Zhenpeng Li</td>
	<td class=" ">Design gene based on both mRNA secondary structure and codon usage bias using Genetic algorithm</td>
	<td class="  sorting_1">1386</td>
</tr><tr id="pkg_genphen" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/genphen.html">genphen</a></td>
	<td class=" ">Simo Kitanovski</td>
	<td class=" ">A tool for quantification of associations between genotypes and phenotypes with Bayesian inference and statistical learning techniques</td>
	<td class="  sorting_1">1386</td>
</tr><tr id="pkg_methyvim" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/methyvim.html">methyvim</a></td>
	<td class=" ">Nima Hejazi</td>
	<td class=" ">Targeted, Robust, and Model-free Differential Methylation Analysis</td>
	<td class="  sorting_1">1386</td>
</tr><tr id="pkg_miRsponge" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/miRsponge.html">miRsponge</a></td>
	<td class=" ">Junpeng Zhang</td>
	<td class=" ">Identification and analysis of miRNA sponge interaction networks and modules</td>
	<td class="  sorting_1">1386</td>
</tr><tr id="pkg_netReg" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/netReg.html">netReg</a></td>
	<td class=" ">Simon Dirmeier</td>
	<td class=" ">Network-Regularized Regression Models</td>
	<td class="  sorting_1">1386</td>
</tr><tr id="pkg_normalize450K" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/normalize450K.html">normalize450K</a></td>
	<td class=" ">Jonathan Alexander Heiss</td>
	<td class=" ">Preprocessing of Illumina Infinium 450K data</td>
	<td class="  sorting_1">1386</td>
</tr><tr id="pkg_PathwaySplice" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/PathwaySplice.html">PathwaySplice</a></td>
	<td class=" ">Aimin Yan</td>
	<td class=" ">An R Package for Unbiased Splicing Pathway Analysis</td>
	<td class="  sorting_1">1386</td>
</tr><tr id="pkg_pmm" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/pmm.html">pmm</a></td>
	<td class=" ">Anna Drewek</td>
	<td class=" ">Parallel Mixed Model</td>
	<td class="  sorting_1">1386</td>
</tr><tr id="pkg_SISPA" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/SISPA.html">SISPA</a></td>
	<td class=" ">Bhakti Dwivedi</td>
	<td class=" ">SISPA: Method for Sample Integrated Set Profile Analysis</td>
	<td class="  sorting_1">1386</td>
</tr><tr id="pkg_STROMA4" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/STROMA4.html">STROMA4</a></td>
	<td class=" ">Sadiq Saleh</td>
	<td class=" ">Assign Properties to TNBC Patients</td>
	<td class="  sorting_1">1386</td>
</tr><tr id="pkg_ChIPanalyser" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/ChIPanalyser.html">ChIPanalyser</a></td>
	<td class=" ">Patrick C.N. Martin</td>
	<td class=" ">ChIPanalyser: Predicting Transcription Factor Binding Sites</td>
	<td class="  sorting_1">1399</td>
</tr><tr id="pkg_gCMAP" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/gCMAP.html">gCMAP</a></td>
	<td class=" ">Thomas Sandmann</td>
	<td class=" ">Tools for Connectivity Map-like analyses</td>
	<td class="  sorting_1">1399</td>
</tr><tr id="pkg_geneClassifiers" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/geneClassifiers.html">geneClassifiers</a></td>
	<td class=" ">R Kuiper</td>
	<td class=" ">Application of gene classifiers</td>
	<td class="  sorting_1">1399</td>
</tr><tr id="pkg_MPFE" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/MPFE.html">MPFE</a></td>
	<td class=" ">Conrad Burden</td>
	<td class=" ">Estimation of the amplicon methylation pattern distribution from bisulphite sequencing data</td>
	<td class="  sorting_1">1399</td>
</tr><tr id="pkg_NADfinder" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/NADfinder.html">NADfinder</a></td>
	<td class=" ">Jianhong Ou, Lihua Julie Zhu</td>
	<td class=" ">Call wide peaks for sequencing data</td>
	<td class="  sorting_1">1399</td>
</tr><tr id="pkg_Onassis" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/Onassis.html">Onassis</a></td>
	<td class=" ">Eugenia Galeota</td>
	<td class=" ">OnASSIs Ontology Annotation and Semantic SImilarity software</td>
	<td class="  sorting_1">1399</td>
</tr><tr id="pkg_seqcombo" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/seqcombo.html">seqcombo</a></td>
	<td class=" ">Guangchuang Yu</td>
	<td class=" ">Visualization Tool for Sequence Recombination and Reassortment</td>
	<td class="  sorting_1">1399</td>
</tr><tr id="pkg_skewr" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/skewr.html">skewr</a></td>
	<td class=" ">Ryan Putney</td>
	<td class=" ">Visualize Intensities Produced by Illumina's Human Methylation 450k BeadChip</td>
	<td class="  sorting_1">1399</td>
</tr><tr id="pkg_sparseDOSSA" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/sparseDOSSA.html">sparseDOSSA</a></td>
	<td class=" ">Boyu Ren, Emma Schwager, George Weingart</td>
	<td class=" ">Sparse Data Observations for Simulating Synthetic Abundance</td>
	<td class="  sorting_1">1399</td>
</tr><tr id="pkg_Uniquorn" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/Uniquorn.html">Uniquorn</a></td>
	<td class=" ">'Raik Otto'</td>
	<td class=" ">Identification of cancer cell lines based on their weighted mutational/ variational fingerprint</td>
	<td class="  sorting_1">1399</td>
</tr><tr id="pkg_vtpnet" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/vtpnet.html">vtpnet</a></td>
	<td class=" ">VJ Carey</td>
	<td class=" ">variant-transcription factor-phenotype networks</td>
	<td class="  sorting_1">1399</td>
</tr><tr id="pkg_ChIPXpress" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/ChIPXpress.html">ChIPXpress</a></td>
	<td class=" ">George Wu</td>
	<td class=" ">ChIPXpress: enhanced transcription factor target gene identification from ChIP-seq and ChIP-chip data using publicly available gene expression profiles</td>
	<td class="  sorting_1">1410</td>
</tr><tr id="pkg_dmrseq" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/dmrseq.html">dmrseq</a></td>
	<td class=" ">Keegan Korthauer</td>
	<td class=" ">Detection and inference of differentially methylated regions from Whole Genome Bisulfite Sequencing</td>
	<td class="  sorting_1">1410</td>
</tr><tr id="pkg_epivizrChart" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/epivizrChart.html">epivizrChart</a></td>
	<td class=" ">Hector Corrada Bravo</td>
	<td class=" ">R interface to epiviz web components</td>
	<td class="  sorting_1">1410</td>
</tr><tr id="pkg_funtooNorm" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/funtooNorm.html">funtooNorm</a></td>
	<td class=" ">Kathleen Klein</td>
	<td class=" ">Normalization Procedure for Infinium HumanMethylation450 BeadChip Kit</td>
	<td class="  sorting_1">1410</td>
</tr><tr id="pkg_MaxContrastProjection" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/MaxContrastProjection.html">MaxContrastProjection</a></td>
	<td class=" ">Jan Sauer</td>
	<td class=" ">Perform a maximum contrast projection of 3D images along the z-dimension into 2D</td>
	<td class="  sorting_1">1410</td>
</tr><tr id="pkg_MethPed" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/MethPed.html">MethPed</a></td>
	<td class=" ">Helena Carn</td>
	<td class=" ">A DNA methylation classifier tool for the identification of pediatric brain tumor subtypes</td>
	<td class="  sorting_1">1410</td>
</tr><tr id="pkg_mimager" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/mimager.html">mimager</a></td>
	<td class=" ">Aaron Wolen</td>
	<td class=" ">mimager: The Microarray Imager</td>
	<td class="  sorting_1">1410</td>
</tr><tr id="pkg_msgbsR" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/msgbsR.html">msgbsR</a></td>
	<td class=" ">Benjamin Mayne</td>
	<td class=" ">msgbsR: methylation sensitive genotyping by sequencing (MS-GBS) R functions</td>
	<td class="  sorting_1">1410</td>
</tr><tr id="pkg_nudge" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/nudge.html">nudge</a></td>
	<td class=" ">N. Dean</td>
	<td class=" ">Normal Uniform Differential Gene Expression detection</td>
	<td class="  sorting_1">1410</td>
</tr><tr id="pkg_SRGnet" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/SRGnet.html">SRGnet</a></td>
	<td class=" ">Isar Nassiri</td>
	<td class=" ">SRGnet: An R package for studying synergistic response to gene mutations from transcriptomics data from transcriptomics data</td>
	<td class="  sorting_1">1410</td>
</tr><tr id="pkg_transcriptogramer" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/transcriptogramer.html">transcriptogramer</a></td>
	<td class=" ">Diego Morais</td>
	<td class=" ">Transcriptional analysis based on transcriptograms</td>
	<td class="  sorting_1">1410</td>
</tr><tr id="pkg_ClusterJudge" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/ClusterJudge.html">ClusterJudge</a></td>
	<td class=" ">Adrian Pasculescu</td>
	<td class=" ">Judging Quality of Clustering Methods using Mutual Information</td>
	<td class="  sorting_1">1421</td>
</tr><tr id="pkg_explorase" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/explorase.html">explorase</a></td>
	<td class=" ">Michael Lawrence</td>
	<td class=" ">GUI for exploratory data analysis of systems biology data</td>
	<td class="  sorting_1">1421</td>
</tr><tr id="pkg_MBAmethyl" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/MBAmethyl.html">MBAmethyl</a></td>
	<td class=" ">Tao Wang</td>
	<td class=" ">Model-based analysis of DNA methylation data</td>
	<td class="  sorting_1">1421</td>
</tr><tr id="pkg_ontoProc" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/ontoProc.html">ontoProc</a></td>
	<td class=" ">VJ Carey</td>
	<td class=" ">processing of ontologies of anatomy, cell lines, and so on</td>
	<td class="  sorting_1">1421</td>
</tr><tr id="pkg_profileScoreDist" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/profileScoreDist.html">profileScoreDist</a></td>
	<td class=" ">Paal O. Westermark</td>
	<td class=" ">Profile score distributions</td>
	<td class="  sorting_1">1421</td>
</tr><tr id="pkg_swfdr" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/swfdr.html">swfdr</a></td>
	<td class=" ">Simina M. Boca, Jeffrey T. Leek</td>
	<td class=" ">Science-wise false discovery rate and proportion of true null hypotheses estimation</td>
	<td class="  sorting_1">1421</td>
</tr><tr id="pkg_Doscheda" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/Doscheda.html">Doscheda</a></td>
	<td class=" ">Bruno Contrino</td>
	<td class=" ">A DownStream Chemo-Proteomics Analysis Pipeline</td>
	<td class="  sorting_1">1427</td>
</tr><tr id="pkg_drawProteins" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/drawProteins.html">drawProteins</a></td>
	<td class=" ">Paul Brennan</td>
	<td class=" ">Package to Draw Protein Schematics from Uniprot API output</td>
	<td class="  sorting_1">1427</td>
</tr><tr id="pkg_gaucho" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/gaucho.html">gaucho</a></td>
	<td class=" ">Alex Murison, Christopher Wardell</td>
	<td class=" ">Genetic Algorithms for Understanding Clonal Heterogeneity and Ordering</td>
	<td class="  sorting_1">1427</td>
</tr><tr id="pkg_GMRP" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/GMRP.html">GMRP</a></td>
	<td class=" ">Yuan-De Tan</td>
	<td class=" ">GWAS-based Mendelian Randomization and Path Analyses</td>
	<td class="  sorting_1">1427</td>
</tr><tr id="pkg_hiReadsProcessor" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/hiReadsProcessor.html">hiReadsProcessor</a></td>
	<td class=" ">Nirav V Malani</td>
	<td class=" ">Functions to process LM-PCR reads from 454/Illumina data</td>
	<td class="  sorting_1">1427</td>
</tr><tr id="pkg_IntramiRExploreR" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/IntramiRExploreR.html">IntramiRExploreR</a></td>
	<td class=" ">Surajit Bhattacharya</td>
	<td class=" ">Predicting Targets for Drosophila Intragenic miRNAs</td>
	<td class="  sorting_1">1427</td>
</tr><tr id="pkg_pathprint" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/pathprint.html">pathprint</a></td>
	<td class=" ">Sokratis Kariotis</td>
	<td class=" ">Pathway fingerprinting for analysis of gene expression arrays</td>
	<td class="  sorting_1">1427</td>
</tr><tr id="pkg_rexposome" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/rexposome.html">rexposome</a></td>
	<td class=" ">Carles Hernandez-Ferrer</td>
	<td class=" ">Exposome exploration and outcome data analysis</td>
	<td class="  sorting_1">1427</td>
</tr><tr id="pkg_topdownr" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/topdownr.html">topdownr</a></td>
	<td class=" ">Sebastian Gibb</td>
	<td class=" ">Investigation of Fragmentation Conditions in Top-Down Proteomics</td>
	<td class="  sorting_1">1427</td>
</tr><tr id="pkg_pcxn" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/pcxn.html">pcxn</a></td>
	<td class=" ">Sokratis Kariotis</td>
	<td class=" ">Exploring, analyzing and visualizing functions utilizing the pcxnData package</td>
	<td class="  sorting_1">1436</td>
</tr><tr id="pkg_pgca" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/pgca.html">pgca</a></td>
	<td class=" ">Gabriela Cohen-Freue</td>
	<td class=" ">PGCA: An Algorithm to Link Protein Groups Created from MS/MS Data</td>
	<td class="  sorting_1">1436</td>
</tr><tr id="pkg_phosphonormalizer" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/phosphonormalizer.html">phosphonormalizer</a></td>
	<td class=" ">Sohrab Saraei</td>
	<td class=" ">Compensates for the bias introduced by median normalization in phosphoproteomics</td>
	<td class="  sorting_1">1436</td>
</tr><tr id="pkg_PROPS" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/PROPS.html">PROPS</a></td>
	<td class=" ">Lichy Han</td>
	<td class=" ">PRObabilistic Pathway Score (PROPS)</td>
	<td class="  sorting_1">1436</td>
</tr><tr id="pkg_SeqSQC" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/SeqSQC.html">SeqSQC</a></td>
	<td class=" ">Qian Liu</td>
	<td class=" ">A bioconductor package for sample quality check with next generation sequencing data</td>
	<td class="  sorting_1">1436</td>
</tr><tr id="pkg_GOfuncR" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/GOfuncR.html">GOfuncR</a></td>
	<td class=" ">Steffi Grote</td>
	<td class=" ">Gene ontology enrichment using FUNC</td>
	<td class="  sorting_1">1441</td>
</tr><tr id="pkg_MBttest" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/MBttest.html">MBttest</a></td>
	<td class=" ">Yuan-De Tan</td>
	<td class=" ">Multiple Beta t-Tests</td>
	<td class="  sorting_1">1441</td>
</tr><tr id="pkg_openPrimeRui" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/openPrimeRui.html">openPrimeRui</a></td>
	<td class=" ">Matthias Dring</td>
	<td class=" ">Shiny Application for Multiplex PCR Primer Design and Analysis</td>
	<td class="  sorting_1">1441</td>
</tr><tr id="pkg_POST" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/POST.html">POST</a></td>
	<td class=" ">Xueyuan Cao</td>
	<td class=" ">Projection onto Orthogonal Space Testing for High Dimensional Data</td>
	<td class="  sorting_1">1441</td>
</tr><tr id="pkg_RVS" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/RVS.html">RVS</a></td>
	<td class=" ">Thomas Sherman</td>
	<td class=" ">Computes estimates of the probability of related individuals sharing a rare variant</td>
	<td class="  sorting_1">1441</td>
</tr><tr id="pkg_scoreInvHap" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/scoreInvHap.html">scoreInvHap</a></td>
	<td class=" ">Carlos Ruiz</td>
	<td class=" ">Get inversion status in predefined regions</td>
	<td class="  sorting_1">1441</td>
</tr><tr id="pkg_SPONGE" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/SPONGE.html">SPONGE</a></td>
	<td class=" ">Markus List</td>
	<td class=" ">Sparse Partial Correlations On Gene Expression</td>
	<td class="  sorting_1">1441</td>
</tr><tr id="pkg_oncomix" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/oncomix.html">oncomix</a></td>
	<td class=" ">Daniel Pique</td>
	<td class=" ">Identifying Genes Overexpressed in Subsets of Tumors from Tumor-Normal mRNA Expression Data</td>
	<td class="  sorting_1">1448</td>
</tr><tr id="pkg_seqCAT" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/seqCAT.html">seqCAT</a></td>
	<td class=" ">Erik Fasterius</td>
	<td class=" ">High Throughput Sequencing Cell Authentication Toolkit</td>
	<td class="  sorting_1">1448</td>
</tr><tr id="pkg_TFARM" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/TFARM.html">TFARM</a></td>
	<td class=" ">Liuba Nausicaa Martino</td>
	<td class=" ">Transcription Factors Association Rules Miner</td>
	<td class="  sorting_1">1448</td>
</tr><tr id="pkg_GSEABenchmarkeR" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/GSEABenchmarkeR.html">GSEABenchmarkeR</a></td>
	<td class=" ">Ludwig Geistlinger</td>
	<td class=" ">Reproducible GSEA Benchmarking</td>
	<td class="  sorting_1">1451</td>
</tr><tr id="pkg_ivygapSE" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/ivygapSE.html">ivygapSE</a></td>
	<td class=" ">VJ Carey</td>
	<td class=" ">A SummarizedExperiment for Ivy-GAP data</td>
	<td class="  sorting_1">1451</td>
</tr><tr id="pkg_miRmine" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/miRmine.html">miRmine</a></td>
	<td class=" ">Dusan Randjelovic</td>
	<td class=" ">Data package with miRNA-seq datasets from miRmine database as RangedSummarizedExperiment</td>
	<td class="  sorting_1">1451</td>
</tr><tr id="pkg_vulcan" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/vulcan.html">vulcan</a></td>
	<td class=" ">Federico M. Giorgi</td>
	<td class=" ">VirtUaL ChIP-Seq data Analysis using Networks</td>
	<td class="  sorting_1">1451</td>
</tr><tr id="pkg_Scale4C" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/Scale4C.html">Scale4C</a></td>
	<td class=" ">Carolin Walter</td>
	<td class=" ">Scale4C: an R/Bioconductor package for scale-space transformation of 4C-seq data</td>
	<td class="  sorting_1">1455</td>
</tr><tr id="pkg_tenXplore" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/tenXplore.html">tenXplore</a></td>
	<td class=" ">VJ Carey</td>
	<td class=" ">ontological exploration of scRNA-seq of 1.3 million mouse neurons from 10x genomics</td>
	<td class="  sorting_1">1455</td>
</tr><tr id="pkg_TFHAZ" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/TFHAZ.html">TFHAZ</a></td>
	<td class=" ">Alberto Marchesi</td>
	<td class=" ">Transcription Factor High Accumulation Zones</td>
	<td class="  sorting_1">1455</td>
</tr><tr id="pkg_diffcyt" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/diffcyt.html">diffcyt</a></td>
	<td class=" ">Lukas M. Weber</td>
	<td class=" ">Differential discovery in high-dimensional cytometry via high-resolution clustering</td>
	<td class="  sorting_1">1458</td>
</tr><tr id="pkg_oneSENSE" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/oneSENSE.html">oneSENSE</a></td>
	<td class=" ">Yong Kee Tan</td>
	<td class=" ">One-Dimensional Soli-Expression by Nonlinear Stochastic Embedding (OneSENSE)</td>
	<td class="  sorting_1">1458</td>
</tr><tr id="pkg_omicRexposome" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/omicRexposome.html">omicRexposome</a></td>
	<td class=" ">Carles Hernandez-Ferrer</td>
	<td class=" ">Exposome and omic data associatin and integration analysis</td>
	<td class="  sorting_1">1460</td>
</tr><tr id="pkg_ArrayExpressHTS" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/ArrayExpressHTS.html">ArrayExpressHTS</a></td>
	<td class=" ">Angela Goncalves, Andrew Tikhonov</td>
	<td class=" ">ArrayExpress High Throughput Sequencing Processing Pipeline</td>
	<td class="  sorting_1">1461</td>
</tr><tr id="pkg_samExploreR" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/samExploreR.html">samExploreR</a></td>
	<td class=" ">shailesh tripathi</td>
	<td class=" ">samExploreR package: high-performance read summarisation to count vectors with avaliability of sequencing depth reduction simulation</td>
	<td class="  sorting_1">1461</td>
</tr><tr id="pkg_singleCellTK" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/singleCellTK.html">singleCellTK</a></td>
	<td class=" ">David Jenkins</td>
	<td class=" ">Interactive Analysis of Single Cell RNA-Seq Data</td>
	<td class="  sorting_1">1461</td>
</tr><tr id="pkg_trena" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/trena.html">trena</a></td>
	<td class=" ">Paul Shannon</td>
	<td class=" ">Fit transcriptional regulatory networks using gene expression, priors, machine learning</td>
	<td class="  sorting_1">1461</td>
</tr><tr id="pkg_decontam" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/decontam.html">decontam</a></td>
	<td class=" ">Benjamin Callahan</td>
	<td class=" ">Identify Contaminants in Marker-gene and Metagenomics Sequencing Data</td>
	<td class="  sorting_1">1465</td>
</tr><tr id="pkg_DEsingle" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/DEsingle.html">DEsingle</a></td>
	<td class=" ">Zhun Miao</td>
	<td class=" ">DEsingle for detecting three types of differential expression in single-cell RNA-seq data</td>
	<td class="  sorting_1">1466</td>
</tr><tr id="pkg_IrisSpatialFeatures" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/IrisSpatialFeatures.html">IrisSpatialFeatures</a></td>
	<td class=" ">Daniel Gusenleitner</td>
	<td class=" ">A package to extract spatial features based on multiplex IF images</td>
	<td class="  sorting_1">1467</td>
</tr><tr id="pkg_TissueEnrich" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/TissueEnrich.html">TissueEnrich</a></td>
	<td class=" ">Ashish Jain</td>
	<td class=" ">Tissue-specific gene enrichment analysis</td>
	<td class="  sorting_1">1467</td>
</tr><tr id="pkg_vidger" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/vidger.html">vidger</a></td>
	<td class=" ">Brandon Monier</td>
	<td class=" ">Create rapid visualizations of RNAseq data in R</td>
	<td class="  sorting_1">1467</td>
</tr><tr id="pkg_predictionet" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/predictionet.html">predictionet</a></td>
	<td class=" ">Benjamin Haibe-Kains, Catharina Olsen</td>
	<td class=" ">Inference for predictive networks designed for (but not limited to) genomic data</td>
	<td class="  sorting_1">1470</td>
</tr><tr id="pkg_rSFFreader" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/rSFFreader.html">rSFFreader</a></td>
	<td class=" ">Matt Settles</td>
	<td class=" ">rSFFreader reads in sff files generated by Roche 454 and Life Sciences Ion Torrent sequencers</td>
	<td class="  sorting_1">1470</td>
</tr><tr id="pkg_HTSeqGenie" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/HTSeqGenie.html">HTSeqGenie</a></td>
	<td class=" ">Jens Reeder</td>
	<td class=" ">A NGS analysis pipeline.</td>
	<td class="  sorting_1">1472</td>
</tr><tr id="pkg_rWikiPathways" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/rWikiPathways.html">rWikiPathways</a></td>
	<td class=" ">Egon Willighagen, Alexander Pico</td>
	<td class=" ">rWikiPathways - R client library for the WikiPathways API</td>
	<td class="  sorting_1">1472</td>
</tr><tr id="pkg_BGmix" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/BGmix.html">BGmix</a></td>
	<td class=" ">Alex Lewin</td>
	<td class=" ">Bayesian models for differential gene expression</td>
	<td class="  sorting_1">1474</td>
</tr><tr id="pkg_MAGeCKFlute" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/MAGeCKFlute.html">MAGeCKFlute</a></td>
	<td class=" ">Wubing Zhang</td>
	<td class=" ">Integrative analysis pipeline for pooled CRISPR functional genetic screens</td>
	<td class="  sorting_1">1474</td>
</tr><tr id="pkg_hmdbQuery" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/hmdbQuery.html">hmdbQuery</a></td>
	<td class=" ">VJ Carey</td>
	<td class=" ">utilities for exploration of human metabolome database</td>
	<td class="  sorting_1">1476</td>
</tr><tr id="pkg_bcSeq" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/bcSeq.html">bcSeq</a></td>
	<td class=" ">Jiaxing Lin</td>
	<td class=" ">Fast Sequence Mapping in High-Throughput shRNA and CRISPR Screens</td>
	<td class="  sorting_1">1477</td>
</tr><tr id="pkg_gCMAPWeb" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/gCMAPWeb.html">gCMAPWeb</a></td>
	<td class=" ">Thomas Sandmann</td>
	<td class=" ">A web interface for gene-set enrichment analyses</td>
	<td class="  sorting_1">1477</td>
</tr><tr id="pkg_ORFik" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/ORFik.html">ORFik</a></td>
	<td class=" ">Kornel Labun</td>
	<td class=" ">Open Reading Frames in Genomics</td>
	<td class="  sorting_1">1477</td>
</tr><tr id="pkg_Trendy" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/Trendy.html">Trendy</a></td>
	<td class=" ">Rhonda Bacher</td>
	<td class=" ">Breakpoint analysis of time-course expression data</td>
	<td class="  sorting_1">1480</td>
</tr><tr id="pkg_ChIPSeqSpike" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/ChIPSeqSpike.html">ChIPSeqSpike</a></td>
	<td class=" ">Nicolas Descostes</td>
	<td class=" ">ChIP-Seq data scaling according to spike-in control</td>
	<td class="  sorting_1">1481</td>
</tr><tr id="pkg_SICtools" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/SICtools.html">SICtools</a></td>
	<td class=" ">Xiaobin Xing</td>
	<td class=" ">Find SNV/Indel differences between two bam files with near relationship</td>
	<td class="  sorting_1">1481</td>
</tr><tr id="pkg_BiocOncoTK" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/BiocOncoTK.html">BiocOncoTK</a></td>
	<td class=" ">VJ Carey</td>
	<td class=" ">Bioconductor components for general cancer genomics</td>
	<td class="  sorting_1">1483</td>
</tr><tr id="pkg_TFutils" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/TFutils.html">TFutils</a></td>
	<td class=" ">Shweta Gopaulakrishnan</td>
	<td class=" ">TFutils</td>
	<td class="  sorting_1">1483</td>
</tr><tr id="pkg_ASICS" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/ASICS.html">ASICS</a></td>
	<td class=" ">Galle Lefort</td>
	<td class=" ">Automatic Statistical Identification in Complex Spectra</td>
	<td class="  sorting_1">1485</td>
</tr><tr id="pkg_BioNetStat" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/BioNetStat.html">BioNetStat</a></td>
	<td class=" ">Vinicius Jardim</td>
	<td class=" ">Biological Network Analysis</td>
	<td class="  sorting_1">1485</td>
</tr><tr id="pkg_CAGEfightR" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/CAGEfightR.html">CAGEfightR</a></td>
	<td class=" ">Malte Thodberg</td>
	<td class=" ">Analysis of Cap Analysis of Gene Expression (CAGE) data using Bioconductor</td>
	<td class="  sorting_1">1487</td>
</tr><tr id="pkg_ccfindR" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/ccfindR.html">ccfindR</a></td>
	<td class=" ">Jun Woo</td>
	<td class=" ">Cancer Clone Finder</td>
	<td class="  sorting_1">1487</td>
</tr><tr id="pkg_diffcoexp" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/diffcoexp.html">diffcoexp</a></td>
	<td class=" ">Wenbin Wei</td>
	<td class=" ">Differential Co-expression Analysis</td>
	<td class="  sorting_1">1487</td>
</tr><tr id="pkg_loci2path" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/loci2path.html">loci2path</a></td>
	<td class=" ">Tianlei Xu</td>
	<td class=" ">Loci2path: regulatory annotation of genomic intervals based on tissue-specific expression QTLs</td>
	<td class="  sorting_1">1487</td>
</tr><tr id="pkg_MetaNeighbor" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/MetaNeighbor.html">MetaNeighbor</a></td>
	<td class=" ">Manthan Shah</td>
	<td class=" ">Single cell replicability analysis</td>
	<td class="  sorting_1">1487</td>
</tr><tr id="pkg_RGMQL" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/RGMQL.html">RGMQL</a></td>
	<td class=" ">Simone Pallotta</td>
	<td class=" ">GenoMetric Query Language for R/Bioconductor</td>
	<td class="  sorting_1">1487</td>
</tr><tr id="pkg_signet" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/signet.html">signet</a></td>
	<td class=" ">Alexandre Gouy</td>
	<td class=" ">signet: Selection Inference in Gene NETworks</td>
	<td class="  sorting_1">1487</td>
</tr><tr id="pkg_hipathia" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/hipathia.html">hipathia</a></td>
	<td class=" ">Marta R. Hidalgo</td>
	<td class=" ">HiPathia: High-throughput Pathway Analysis</td>
	<td class="  sorting_1">1494</td>
</tr><tr id="pkg_kissDE" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/kissDE.html">kissDE</a></td>
	<td class=" ">Aurlie Siberchicot</td>
	<td class=" ">Retrieves Condition-Specific Variants in RNA-Seq Data</td>
	<td class="  sorting_1">1494</td>
</tr><tr id="pkg_netSmooth" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/netSmooth.html">netSmooth</a></td>
	<td class=" ">Jonathan Ronen</td>
	<td class=" ">Network smoothing for scRNAseq</td>
	<td class="  sorting_1">1494</td>
</tr><tr id="pkg_slingshot" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/slingshot.html">slingshot</a></td>
	<td class=" ">Kelly Street</td>
	<td class=" ">Tools for ordering single-cell sequencing</td>
	<td class="  sorting_1">1494</td>
</tr><tr id="pkg_SparseSignatures" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/SparseSignatures.html">SparseSignatures</a></td>
	<td class=" ">Luca De Sano</td>
	<td class=" ">SparseSignatures</td>
	<td class="  sorting_1">1494</td>
</tr><tr id="pkg_adaptest" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/adaptest.html">adaptest</a></td>
	<td class=" ">Weixin Cai</td>
	<td class=" ">Data-Adaptive Statistics for High-Dimensional Multiple Testing</td>
	<td class="  sorting_1">1499</td>
</tr><tr id="pkg_BEARscc" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/BEARscc.html">BEARscc</a></td>
	<td class=" ">Benjamin Schuster-Boeckler</td>
	<td class=" ">BEARscc (Bayesian ERCC Assesstment of Robustness of Single Cell Clusters)</td>
	<td class="  sorting_1">1499</td>
</tr><tr id="pkg_BiFET" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/BiFET.html">BiFET</a></td>
	<td class=" ">Ahrim Youn</td>
	<td class=" ">Bias-free Footprint Enrichment Test</td>
	<td class="  sorting_1">1499</td>
</tr><tr id="pkg_DEComplexDisease" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/DEComplexDisease.html">DEComplexDisease</a></td>
	<td class=" ">Guofeng Meng</td>
	<td class=" ">A tool for differential expression analysis and DEGs based investigation to complex diseases by bi-clustering analysis</td>
	<td class="  sorting_1">1499</td>
</tr><tr id="pkg_FELLA" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/FELLA.html">FELLA</a></td>
	<td class=" ">Sergio Picart-Armada</td>
	<td class=" ">Interpretation and enrichment for metabolomics data</td>
	<td class="  sorting_1">1499</td>
</tr><tr id="pkg_iteremoval" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/iteremoval.html">iteremoval</a></td>
	<td class=" ">Jiacheng Chuan</td>
	<td class=" ">Iteration removal method for feature selection</td>
	<td class="  sorting_1">1499</td>
</tr><tr id="pkg_LineagePulse" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/LineagePulse.html">LineagePulse</a></td>
	<td class=" ">David S Fischer</td>
	<td class=" ">Differential expression analysis and model fitting for single-cell RNA-seq data</td>
	<td class="  sorting_1">1499</td>
</tr><tr id="pkg_martini" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/martini.html">martini</a></td>
	<td class=" ">Hector Climente-Gonzalez</td>
	<td class=" ">GWAS Incorporating Networks</td>
	<td class="  sorting_1">1499</td>
</tr><tr id="pkg_PowerExplorer" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/PowerExplorer.html">PowerExplorer</a></td>
	<td class=" ">Xu Qiao</td>
	<td class=" ">Power Estimation Tool for RNA-Seq and proteomics data</td>
	<td class="  sorting_1">1499</td>
</tr><tr id="pkg_ddPCRclust" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/ddPCRclust.html">ddPCRclust</a></td>
	<td class=" ">Benedikt G. Brink</td>
	<td class=" ">Clustering algorithm for ddPCR data</td>
	<td class="  sorting_1">1508</td>
</tr><tr id="pkg_DominoEffect" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/DominoEffect.html">DominoEffect</a></td>
	<td class=" ">Marija Buljan, Peter Blattmann</td>
	<td class=" ">Identification and Annotation of Protein Hotspot Residues</td>
	<td class="  sorting_1">1508</td>
</tr><tr id="pkg_GateFinder" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/GateFinder.html">GateFinder</a></td>
	<td class=" ">Nima Aghaeepour</td>
	<td class=" ">Projection-based Gating Strategy Optimization for Flow and Mass Cytometry</td>
	<td class="  sorting_1">1508</td>
</tr><tr id="pkg_iCNV" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/iCNV.html">iCNV</a></td>
	<td class=" ">Zilu Zhou</td>
	<td class=" ">Integrated Copy Number Variation detection</td>
	<td class="  sorting_1">1508</td>
</tr><tr id="pkg_igvR" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/igvR.html">igvR</a></td>
	<td class=" ">Paul Shannon</td>
	<td class=" ">igvR: integrative genomics viewer</td>
	<td class="  sorting_1">1508</td>
</tr><tr id="pkg_mCSEA" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/mCSEA.html">mCSEA</a></td>
	<td class=" ">Jordi Martorell-Marugn</td>
	<td class=" ">Methylated CpGs Set Enrichment Analysis</td>
	<td class="  sorting_1">1508</td>
</tr><tr id="pkg_CTDquerier" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/CTDquerier.html">CTDquerier</a></td>
	<td class=" ">Carles Hernandez-Ferrer</td>
	<td class=" ">Package for CTDbase data query, visualization and downstream analysis</td>
	<td class="  sorting_1">1514</td>
</tr><tr id="pkg_EnhancedVolcano" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/EnhancedVolcano.html">EnhancedVolcano</a></td>
	<td class=" ">Kevin Blighe</td>
	<td class=" ">Publication-ready volcano plots with enhanced colouring and labeling</td>
	<td class="  sorting_1">1514</td>
</tr><tr id="pkg_gep2pep" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/gep2pep.html">gep2pep</a></td>
	<td class=" ">Francesco Napolitano</td>
	<td class=" ">Creation and Analysis of Pathway Expression Profiles (PEPs)</td>
	<td class="  sorting_1">1514</td>
</tr><tr id="pkg_gsean" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/gsean.html">gsean</a></td>
	<td class=" ">Dongmin Jung</td>
	<td class=" ">Gene Set Enrichment Analysis with Networks</td>
	<td class="  sorting_1">1514</td>
</tr><tr id="pkg_Sconify" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/Sconify.html">Sconify</a></td>
	<td class=" ">Tyler J Burns</td>
	<td class=" ">A toolkit for performing KNN-based statistics for flow and mass cytometry data</td>
	<td class="  sorting_1">1514</td>
</tr><tr id="pkg_SummarizedBenchmark" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/SummarizedBenchmark.html">SummarizedBenchmark</a></td>
	<td class=" ">Alejandro Reyes, Patrick Kimes</td>
	<td class=" ">Classes and methods for performing benchmark comparisons</td>
	<td class="  sorting_1">1514</td>
</tr><tr id="pkg_TFEA.ChIP" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/TFEA.ChIP.html">TFEA.ChIP</a></td>
	<td class=" ">Laura Puente Santamara</td>
	<td class=" ">Analyze Transcription Factor Enrichment</td>
	<td class="  sorting_1">1514</td>
</tr><tr id="pkg_CellScore" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/CellScore.html">CellScore</a></td>
	<td class=" ">Nancy Mah</td>
	<td class=" ">Tool for Evaluation of Cell Identity from Transcription Profiles</td>
	<td class="  sorting_1">1521</td>
</tr><tr id="pkg_CHARGE" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/CHARGE.html">CHARGE</a></td>
	<td class=" ">Benjamin Mayne</td>
	<td class=" ">CHARGE: CHromosome Assessment in R from Gene Expression data</td>
	<td class="  sorting_1">1521</td>
</tr><tr id="pkg_CytoDx" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/CytoDx.html">CytoDx</a></td>
	<td class=" ">Zicheng Hu</td>
	<td class=" ">Robust prediction of clinical outcomes using cytometry data without cell gating</td>
	<td class="  sorting_1">1521</td>
</tr><tr id="pkg_GDSArray" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/GDSArray.html">GDSArray</a></td>
	<td class=" ">Qian Liu</td>
	<td class=" ">Representing GDS files as array-like objects</td>
	<td class="  sorting_1">1521</td>
</tr><tr id="pkg_mdp" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/mdp.html">mdp</a></td>
	<td class=" ">Helder Nakaya</td>
	<td class=" ">Molecular Degree of Perturbation calculates scores for transcriptome data samples based on their perturbation from controls</td>
	<td class="  sorting_1">1521</td>
</tr><tr id="pkg_omicsPrint" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/omicsPrint.html">omicsPrint</a></td>
	<td class=" ">Davy Cats</td>
	<td class=" ">Cross omic genetic fingerprinting</td>
	<td class="  sorting_1">1521</td>
</tr><tr id="pkg_singscore" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/singscore.html">singscore</a></td>
	<td class=" ">Ruqian Lyu</td>
	<td class=" ">Rank-based single-sample gene set scoring method</td>
	<td class="  sorting_1">1521</td>
</tr><tr id="pkg_DEScan2" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/DEScan2.html">DEScan2</a></td>
	<td class=" ">Dario Righelli</td>
	<td class=" ">Differential Enrichment Scan 2</td>
	<td class="  sorting_1">1528</td>
</tr><tr id="pkg_omicplotR" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/omicplotR.html">omicplotR</a></td>
	<td class=" ">Daniel Giguere</td>
	<td class=" ">Visual Exploration of Omic Datasets Using a Shiny App</td>
	<td class="  sorting_1">1528</td>
</tr><tr id="pkg_perturbatr" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/perturbatr.html">perturbatr</a></td>
	<td class=" ">Simon Dirmeier</td>
	<td class=" ">Statistical Analysis of High-Throughput Genetic Perturbation Screens</td>
	<td class="  sorting_1">1528</td>
</tr><tr id="pkg_phantasus" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/phantasus.html">phantasus</a></td>
	<td class=" ">Alexey Sergushichev</td>
	<td class=" ">Visual and interactive gene expression analysis</td>
	<td class="  sorting_1">1528</td>
</tr><tr id="pkg_pogos" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/pogos.html">pogos</a></td>
	<td class=" ">VJ Carey</td>
	<td class=" ">PharmacOGenomics Ontology Support</td>
	<td class="  sorting_1">1528</td>
</tr><tr id="pkg_RandomWalkRestartMH" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/RandomWalkRestartMH.html">RandomWalkRestartMH</a></td>
	<td class=" ">Alberto Valdeolivas Urbelz</td>
	<td class=" ">Random walk with restart on multiplex and heterogeneous Networks</td>
	<td class="  sorting_1">1528</td>
</tr><tr id="pkg_seqsetvis" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/seqsetvis.html">seqsetvis</a></td>
	<td class=" ">Joseph R Boyd</td>
	<td class=" ">Set Based Visualizations for Next-Gen Sequencing Data</td>
	<td class="  sorting_1">1528</td>
</tr><tr id="pkg_SIAMCAT" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/SIAMCAT.html">SIAMCAT</a></td>
	<td class=" ">Konrad Zych</td>
	<td class=" ">Statistical Inference of Associations between Microbial Communities And host phenoTypes</td>
	<td class="  sorting_1">1528</td>
</tr><tr id="pkg_srnadiff" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/srnadiff.html">srnadiff</a></td>
	<td class=" ">Matthias Zytnicki</td>
	<td class=" ">Differential Expression of Small RNA-Seq</td>
	<td class="  sorting_1">1528</td>
</tr><tr id="pkg_GARS" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/GARS.html">GARS</a></td>
	<td class=" ">Mattia Chiesa</td>
	<td class=" ">GARS: Genetic Algorithm for the identification of Robust Subsets of variables in high-dimensional and challenging datasets</td>
	<td class="  sorting_1">1537</td>
</tr><tr id="pkg_GeneStructureTools" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/GeneStructureTools.html">GeneStructureTools</a></td>
	<td class=" ">Beth Signal</td>
	<td class=" ">Tools for spliced gene structure manipulation and analysis</td>
	<td class="  sorting_1">1537</td>
</tr><tr id="pkg_IMMAN" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/IMMAN.html">IMMAN</a></td>
	<td class=" ">Minoo Ashtiani</td>
	<td class=" ">Interlog protein network reconstruction by Mapping and Mining ANalysis</td>
	<td class="  sorting_1">1537</td>
</tr><tr id="pkg_MACPET" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/MACPET.html">MACPET</a></td>
	<td class=" ">Ioannis Vardaxis</td>
	<td class=" ">Model based analysis for paired-end data</td>
	<td class="  sorting_1">1537</td>
</tr><tr id="pkg_powerTCR" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/powerTCR.html">powerTCR</a></td>
	<td class=" ">Hillary Koch</td>
	<td class=" ">Model-Based Comparative Analysis of the TCR Repertoire</td>
	<td class="  sorting_1">1537</td>
</tr><tr id="pkg_Rgin" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/Rgin.html">Rgin</a></td>
	<td class=" ">Hector Climente</td>
	<td class=" ">gin in R</td>
	<td class="  sorting_1">1537</td>
</tr><tr id="pkg_scFeatureFilter" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/scFeatureFilter.html">scFeatureFilter</a></td>
	<td class=" ">Guillaume Devailly</td>
	<td class=" ">A correlation-based method for quality filtering of single-cell RNAseq data</td>
	<td class="  sorting_1">1537</td>
</tr><tr id="pkg_scmeth" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/scmeth.html">scmeth</a></td>
	<td class=" ">Divy Kangeyan</td>
	<td class=" ">Functions to conduct quality control analysis in methylation data</td>
	<td class="  sorting_1">1537</td>
</tr><tr id="pkg_SDAMS" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/SDAMS.html">SDAMS</a></td>
	<td class=" ">Yuntong Li</td>
	<td class=" ">Differential Abundant Analysis for Metabolomics and Proteomics Data</td>
	<td class="  sorting_1">1537</td>
</tr><tr id="pkg_InTAD" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/InTAD.html">InTAD</a></td>
	<td class=" ">Konstantin Okonechnikov</td>
	<td class=" ">Search for correlation between epigenetic signals and gene expression in TADs</td>
	<td class="  sorting_1">1546</td>
</tr><tr id="pkg_MDTS" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/MDTS.html">MDTS</a></td>
	<td class=" ">Jack M.. Fu</td>
	<td class=" ">Detection of de novo deletion in targeted sequencing trios</td>
	<td class="  sorting_1">1546</td>
</tr><tr id="pkg_MSstatsQCgui" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/MSstatsQCgui.html">MSstatsQCgui</a></td>
	<td class=" ">Eralp Dogu</td>
	<td class=" ">A graphical user interface for MSstatsQC package</td>
	<td class="  sorting_1">1546</td>
</tr><tr id="pkg_sevenC" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/sevenC.html">sevenC</a></td>
	<td class=" ">Jonas Ibn-Salem</td>
	<td class=" ">Computational Chromosome Conformation Capture by Correlation of ChIP-seq at CTCF motifs</td>
	<td class="  sorting_1">1546</td>
</tr><tr id="pkg_TTMap" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/TTMap.html">TTMap</a></td>
	<td class=" ">Rachel Jeitziner</td>
	<td class=" ">Two-Tier Mapper: a clustering tool based on topological data analysis</td>
	<td class="  sorting_1">1546</td>
</tr><tr id="pkg_RSeqAn" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/RSeqAn.html">RSeqAn</a></td>
	<td class=" ">August Guang</td>
	<td class=" ">R SeqAn</td>
	<td class="  sorting_1">1551</td>
</tr><tr id="pkg_SEPIRA" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/SEPIRA.html">SEPIRA</a></td>
	<td class=" ">Yuting Chen</td>
	<td class=" ">Systems EPigenomics Inference of Regulatory Activity</td>
	<td class="  sorting_1">1551</td>
</tr><tr id="pkg_missRows" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/missRows.html">missRows</a></td>
	<td class=" ">Gonzalez Ignacio</td>
	<td class=" ">Handling Missing Individuals in Multi-Omics Data Integration</td>
	<td class="  sorting_1">1553</td>
</tr><tr id="pkg_RNAdecay" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/RNAdecay.html">RNAdecay</a></td>
	<td class=" ">Reed Sorenson</td>
	<td class=" ">Maximum Likelihood Decay Modeling of RNA Degradation Data</td>
	<td class="  sorting_1">1553</td>
</tr><tr id="pkg_tRNAscanImport" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/tRNAscanImport.html">tRNAscanImport</a></td>
	<td class=" ">Felix GM Ernst</td>
	<td class=" ">Imports a tRNAscan-SE result file as GRanges object</td>
	<td class="  sorting_1">1553</td>
</tr><tr id="pkg_OmaDB" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/OmaDB.html">OmaDB</a></td>
	<td class=" ">Klara Kaleb, Adrian Altenhoff</td>
	<td class=" ">R wrapper for the OMA REST API</td>
	<td class="  sorting_1">1556</td>
</tr><tr id="pkg_TxRegInfra" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/TxRegInfra.html">TxRegInfra</a></td>
	<td class=" ">VJ Carey</td>
	<td class=" ">Metadata management for multiomic specification of transcriptional regulatory networks</td>
	<td class="  sorting_1">1556</td>
</tr><tr id="pkg_ChIC" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/ChIC.html">ChIC</a></td>
	<td class=" ">Carmen Maria Livi</td>
	<td class=" ">Quality Control Pipeline for ChIP-Seq Data</td>
	<td class="  sorting_1">1558</td>
</tr><tr id="pkg_sesame" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/sesame.html">sesame</a></td>
	<td class=" ">Wanding Zhou</td>
	<td class=" ">Tools For Analyzing Illumina Infinium DNA Methylation Arrays</td>
	<td class="  sorting_1">1559</td>
</tr><tr id="pkg_CAMTHC" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/CAMTHC.html">CAMTHC</a></td>
	<td class=" ">Lulu Chen</td>
	<td class=" ">Convex Analysis of Mixtures for Tissue Heterogeneity Characterization</td>
	<td class="  sorting_1">1560</td>
</tr><tr id="pkg_maser" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/maser.html">maser</a></td>
	<td class=" ">Diogo Veiga</td>
	<td class=" ">Mapping Alternative Splicing Events to pRoteins</td>
	<td class="  sorting_1">1560</td>
</tr><tr id="pkg_cicero" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/cicero.html">cicero</a></td>
	<td class=" ">Hannah Pliner</td>
	<td class=" ">Predict cis-co-accessibility from single-cell chromatin accessibility data</td>
	<td class="  sorting_1">1562</td>
</tr><tr id="pkg_sigFeature" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/sigFeature.html">sigFeature</a></td>
	<td class=" ">Pijush Das Developer</td>
	<td class=" ">sigFeature: Significant feature selection using SVM-RFE &amp; t-statistic</td>
	<td class="  sorting_1">1563</td>
</tr><tr id="pkg_tximeta" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/tximeta.html">tximeta</a></td>
	<td class=" ">Michael Love</td>
	<td class=" ">Transcript Quantification Import with Automatic Metadata</td>
	<td class="  sorting_1">1564</td>
</tr><tr id="pkg_FastqCleaner" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/FastqCleaner.html">FastqCleaner</a></td>
	<td class=" ">Leandro Roser</td>
	<td class=" ">A Shiny Application for Quality Control, Filtering and Trimming of FASTQ Files</td>
	<td class="  sorting_1">1565</td>
</tr><tr id="pkg_gwasurvivr" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/gwasurvivr.html">gwasurvivr</a></td>
	<td class=" ">Abbas Rizvi</td>
	<td class=" ">gwasurvivr: an R package for genome wide survival analysis</td>
	<td class="  sorting_1">1565</td>
</tr><tr id="pkg_glmSparseNet" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/glmSparseNet.html">glmSparseNet</a></td>
	<td class=" ">Andr Verssimo</td>
	<td class=" ">Network Centrality Metrics for Elastic-Net Regularized Models</td>
	<td class="  sorting_1">1567</td>
</tr><tr id="pkg_methylGSA" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/methylGSA.html">methylGSA</a></td>
	<td class=" ">Xu Ren</td>
	<td class=" ">Gene Set Analysis Using the Outcome of Differential Methylation</td>
	<td class="  sorting_1">1567</td>
</tr><tr id="pkg_OUTRIDER" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/OUTRIDER.html">OUTRIDER</a></td>
	<td class=" ">Christian Mertes</td>
	<td class=" ">OUTRIDER - OUTlier in RNA-Seq fInDER</td>
	<td class="  sorting_1">1567</td>
</tr><tr id="pkg_CellTrails" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/CellTrails.html">CellTrails</a></td>
	<td class=" ">Daniel Ellwanger</td>
	<td class=" ">Reconstruction, visualization and analysis of branching trajectories</td>
	<td class="  sorting_1">1570</td>
</tr><tr id="pkg_icetea" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/icetea.html">icetea</a></td>
	<td class=" ">Vivek Bhardwaj</td>
	<td class=" ">Integrating Cap Enrichment with Transcript Expression Analysis</td>
	<td class="  sorting_1">1570</td>
</tr><tr id="pkg_NormalyzerDE" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/NormalyzerDE.html">NormalyzerDE</a></td>
	<td class=" ">Jakob Willforss</td>
	<td class=" ">Evaluation of normalization methods and calculation of differential expression analysis statistics</td>
	<td class="  sorting_1">1570</td>
</tr><tr id="pkg_qPLEXanalyzer" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/qPLEXanalyzer.html">qPLEXanalyzer</a></td>
	<td class=" ">Ashley Sawle</td>
	<td class=" ">Tools for qPLEX-RIME data analysis</td>
	<td class="  sorting_1">1570</td>
</tr><tr id="pkg_Rmmquant" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/Rmmquant.html">Rmmquant</a></td>
	<td class=" ">Zytnicki Matthias</td>
	<td class=" ">RNA-Seq multi-mapping Reads Quantification Tool</td>
	<td class="  sorting_1">1570</td>
</tr><tr id="pkg_slinky" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/slinky.html">slinky</a></td>
	<td class=" ">Eric J. Kort</td>
	<td class=" ">Putting the fun in LINCS L1000 data analysis</td>
	<td class="  sorting_1">1570</td>
</tr><tr id="pkg_BiocPkgTools" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/BiocPkgTools.html">BiocPkgTools</a></td>
	<td class=" ">Sean Davis</td>
	<td class=" ">Collection of simple tools for learning about Bioc Packages</td>
	<td class="  sorting_1">1576</td>
</tr><tr id="pkg_brainImageR" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/brainImageR.html">brainImageR</a></td>
	<td class=" ">Sara B Linker</td>
	<td class=" ">A Framework for visualizing gene set enrichment throughout neurodevelopment</td>
	<td class="  sorting_1">1576</td>
</tr><tr id="pkg_coRdon" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/coRdon.html">coRdon</a></td>
	<td class=" ">Anamaria Elek</td>
	<td class=" ">Codon Usage Analysis and Prediction of Gene Expressivity</td>
	<td class="  sorting_1">1576</td>
</tr><tr id="pkg_DEqMS" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/DEqMS.html">DEqMS</a></td>
	<td class=" ">Yafeng Zhu</td>
	<td class=" ">a tool to perform statistical analysis of differential protein expression for quantitative proteomics data.</td>
	<td class="  sorting_1">1576</td>
</tr><tr id="pkg_iasva" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/iasva.html">iasva</a></td>
	<td class=" ">Donghyung Lee, Anthony Cheng</td>
	<td class=" ">Iteratively Adjusted Surrogate Variable Analysis</td>
	<td class="  sorting_1">1576</td>
</tr><tr id="pkg_NBSplice" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/NBSplice.html">NBSplice</a></td>
	<td class=" ">Gabriela Merino</td>
	<td class=" ">Negative Binomial Models to detect Differential Splicing</td>
	<td class="  sorting_1">1576</td>
</tr><tr id="pkg_Ularcirc" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/Ularcirc.html">Ularcirc</a></td>
	<td class=" ">David Humphreys</td>
	<td class=" ">Shiny app for canonical and back splicing analysis (i.e. circular and mRNA analysis)</td>
	<td class="  sorting_1">1576</td>
</tr><tr id="pkg_ACE" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/ACE.html">ACE</a></td>
	<td class=" ">Jos B Poell</td>
	<td class=" ">Absolute Copy Number Estimation from Low-coverage Whole Genome Sequencing</td>
	<td class="  sorting_1">1583</td>
</tr><tr id="pkg_appreci8R" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/appreci8R.html">appreci8R</a></td>
	<td class=" ">Sarah Sandmann</td>
	<td class=" ">appreci8R: an R/Bioconductor package for filtering SNVs and short indels with high sensitivity and high PPV</td>
	<td class="  sorting_1">1583</td>
</tr><tr id="pkg_BUScorrect" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/BUScorrect.html">BUScorrect</a></td>
	<td class=" ">Xiangyu Luo</td>
	<td class=" ">Batch Effects Correction with Unknown Subtypes</td>
	<td class="  sorting_1">1583</td>
</tr><tr id="pkg_celaref" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/celaref.html">celaref</a></td>
	<td class=" ">Sarah Williams</td>
	<td class=" ">Single-cell RNAseq cell cluster labelling by reference</td>
	<td class="  sorting_1">1583</td>
</tr><tr id="pkg_condcomp" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/condcomp.html">condcomp</a></td>
	<td class=" ">Diogo P. P. Branco</td>
	<td class=" ">Condition Comparison in scRNA-seq Data</td>
	<td class="  sorting_1">1583</td>
</tr><tr id="pkg_GeneAccord" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/GeneAccord.html">GeneAccord</a></td>
	<td class=" ">Ariane L. Moore</td>
	<td class=" ">Detection of clonally exclusive gene or pathway pairs in a cohort of cancer patients</td>
	<td class="  sorting_1">1583</td>
</tr><tr id="pkg_GIGSEA" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/GIGSEA.html">GIGSEA</a></td>
	<td class=" ">Shijia Zhu</td>
	<td class=" ">Genotype Imputed Gene Set Enrichment Analysis</td>
	<td class="  sorting_1">1583</td>
</tr><tr id="pkg_ipdDb" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/ipdDb.html">ipdDb</a></td>
	<td class=" ">Steffen Klasberg</td>
	<td class=" ">IPD IMGT/HLA and IPD KIR database for Homo sapiens</td>
	<td class="  sorting_1">1583</td>
</tr><tr id="pkg_plotGrouper" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/plotGrouper.html">plotGrouper</a></td>
	<td class=" ">John D. Gagnon</td>
	<td class=" ">Shiny app GUI wrapper for ggplot with built-in statistical analysis</td>
	<td class="  sorting_1">1583</td>
</tr><tr id="pkg_artMS" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/artMS.html">artMS</a></td>
	<td class=" ">David Jimenez-Morales</td>
	<td class=" ">Analytical R tools for Mass Spectrometry</td>
	<td class="  sorting_1">1592</td>
</tr><tr id="pkg_countsimQC" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/countsimQC.html">countsimQC</a></td>
	<td class=" ">Charlotte Soneson</td>
	<td class=" ">Compare Characteristic Features of Count Data Sets</td>
	<td class="  sorting_1">1592</td>
</tr><tr id="pkg_ERSSA" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/ERSSA.html">ERSSA</a></td>
	<td class=" ">Zixuan Shao</td>
	<td class=" ">Empirical RNA-seq Sample Size Analysis</td>
	<td class="  sorting_1">1592</td>
</tr><tr id="pkg_HPAanalyze" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/HPAanalyze.html">HPAanalyze</a></td>
	<td class=" ">Anh Nhat Tran</td>
	<td class=" ">Retrieve and analyze data from the Human Protein Atlas</td>
	<td class="  sorting_1">1592</td>
</tr><tr id="pkg_INDEED" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/INDEED.html">INDEED</a></td>
	<td class=" ">Zhenzhi Li, Yiming Zuo</td>
	<td class=" ">An Implementation of Integrated Differential Expression and Differential Network Analysis for Biomarker Candidate Selection</td>
	<td class="  sorting_1">1592</td>
</tr><tr id="pkg_LRBaseDbi" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/LRBaseDbi.html">LRBaseDbi</a></td>
	<td class=" ">Koki Tsuyuzaki</td>
	<td class=" ">DBI to construct LRBase-related package</td>
	<td class="  sorting_1">1592</td>
</tr><tr id="pkg_MetNet" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/MetNet.html">MetNet</a></td>
	<td class=" ">Thomas Naake</td>
	<td class=" ">Inferring metabolic networks from untargeted high-resolution mass spectrometry data</td>
	<td class="  sorting_1">1592</td>
</tr><tr id="pkg_miRSM" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/miRSM.html">miRSM</a></td>
	<td class=" ">Junpeng Zhang</td>
	<td class=" ">Inferring miRNA sponge modules by integrating expression data and miRNA-target binding information</td>
	<td class="  sorting_1">1592</td>
</tr><tr id="pkg_mlm4omics" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/mlm4omics.html">mlm4omics</a></td>
	<td class=" ">Irene SL Zeng</td>
	<td class=" ">Multilevel Model for Multivariate Responses with Missing Values</td>
	<td class="  sorting_1">1592</td>
</tr><tr id="pkg_MSstatsTMT" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/MSstatsTMT.html">MSstatsTMT</a></td>
	<td class=" ">Ting Huang</td>
	<td class=" ">Protein Significance Analysis in shotgun mass spectrometry-based proteomic experiments with tandem mass tag (TMT) labeling</td>
	<td class="  sorting_1">1592</td>
</tr><tr id="pkg_onlineFDR" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/onlineFDR.html">onlineFDR</a></td>
	<td class=" ">David Robertson</td>
	<td class=" ">Online FDR control</td>
	<td class="  sorting_1">1592</td>
</tr><tr id="pkg_sparsenetgls" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/sparsenetgls.html">sparsenetgls</a></td>
	<td class=" ">Irene Zeng</td>
	<td class=" ">Using Gaussian graphical structue learning estimation in generalized least squared regression for multivariate normal regression</td>
	<td class="  sorting_1">1592</td>
</tr><tr id="pkg_universalmotif" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/universalmotif.html">universalmotif</a></td>
	<td class=" ">Benjamin Jean-Marie Tremblay</td>
	<td class=" ">Import, Modify, and Export Motifs with R</td>
	<td class="  sorting_1">1592</td>
</tr><tr id="pkg_abseqR" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/abseqR.html">abseqR</a></td>
	<td class=" ">JiaHong Fong</td>
	<td class=" ">Reporting and data analysis functionalities for Rep-Seq datasets of antibody libraries</td>
	<td class="  sorting_1">1605</td>
</tr><tr id="pkg_bayNorm" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/bayNorm.html">bayNorm</a></td>
	<td class=" ">Wenhao Tang</td>
	<td class=" ">Single-cell RNA sequencing data normalization</td>
	<td class="  sorting_1">1605</td>
</tr><tr id="pkg_breakpointR" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/breakpointR.html">breakpointR</a></td>
	<td class=" ">David Porubsky</td>
	<td class=" ">Find breakpoints in Strand-seq data</td>
	<td class="  sorting_1">1605</td>
</tr><tr id="pkg_COCOA" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/COCOA.html">COCOA</a></td>
	<td class=" ">John Lawson</td>
	<td class=" ">Coordinate Covariation Analysis</td>
	<td class="  sorting_1">1605</td>
</tr><tr id="pkg_consensus" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/consensus.html">consensus</a></td>
	<td class=" ">Tim Peters</td>
	<td class=" ">Cross-platform consensus analysis of genomic measurements via interlaboratory testing method</td>
	<td class="  sorting_1">1605</td>
</tr><tr id="pkg_FCBF" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/FCBF.html">FCBF</a></td>
	<td class=" ">Tiago Lubiana</td>
	<td class=" ">Fast Correlation Based Filter for Feature Selection</td>
	<td class="  sorting_1">1605</td>
</tr><tr id="pkg_FoldGO" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/FoldGO.html">FoldGO</a></td>
	<td class=" ">Daniil Wiebe</td>
	<td class=" ">Package for Fold-specific GO Terms Recognition</td>
	<td class="  sorting_1">1605</td>
</tr><tr id="pkg_gpart" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/gpart.html">gpart</a></td>
	<td class=" ">Sun Ah Kim</td>
	<td class=" ">Human genome partitioning of dense sequencing data by identifying haplotype blocks</td>
	<td class="  sorting_1">1605</td>
</tr><tr id="pkg_hierinf" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/hierinf.html">hierinf</a></td>
	<td class=" ">Claude Renaux</td>
	<td class=" ">Hierarchical Inference</td>
	<td class="  sorting_1">1605</td>
</tr><tr id="pkg_IsoCorrectoR" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/IsoCorrectoR.html">IsoCorrectoR</a></td>
	<td class=" ">Christian Kohler</td>
	<td class=" ">Correction for natural isotope abundance and tracer purity in MS and MS/MS data from stable isotope labeling experiments</td>
	<td class="  sorting_1">1605</td>
</tr><tr id="pkg_LoomExperiment" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/LoomExperiment.html">LoomExperiment</a></td>
	<td class=" ">Bioconductor Package Maintainer</td>
	<td class=" ">LoomExperiment container</td>
	<td class="  sorting_1">1605</td>
</tr><tr id="pkg_MetID" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/MetID.html">MetID</a></td>
	<td class=" ">Zhenzhi Li</td>
	<td class=" ">Network-based prioritization of putative metabolite IDs</td>
	<td class="  sorting_1">1605</td>
</tr><tr id="pkg_multiHiCcompare" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/multiHiCcompare.html">multiHiCcompare</a></td>
	<td class=" ">John Stansfield</td>
	<td class=" ">Normalize and detect differences between Hi-C datasets when replicates of each experimental condition are available</td>
	<td class="  sorting_1">1605</td>
</tr><tr id="pkg_nuCpos" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/nuCpos.html">nuCpos</a></td>
	<td class=" ">Hiroaki Kato</td>
	<td class=" ">An R package for prediction of nucleosome positions</td>
	<td class="  sorting_1">1605</td>
</tr><tr id="pkg_primirTSS" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/primirTSS.html">primirTSS</a></td>
	<td class=" ">Pumin Li</td>
	<td class=" ">Prediction of pri-miRNA Transcription Start Site</td>
	<td class="  sorting_1">1605</td>
</tr><tr id="pkg_QSutils" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/QSutils.html">QSutils</a></td>
	<td class=" ">Mercedes Guerrero-Murillo</td>
	<td class=" ">Quasispecies Diversity</td>
	<td class="  sorting_1">1605</td>
</tr><tr id="pkg_REBET" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/REBET.html">REBET</a></td>
	<td class=" ">Bill Wheeler</td>
	<td class=" ">The subREgion-based BurdEn Test (REBET)</td>
	<td class="  sorting_1">1605</td>
</tr><tr id="pkg_scruff" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/scruff.html">scruff</a></td>
	<td class=" ">Zhe Wang</td>
	<td class=" ">Single Cell RNA-Seq UMI Filtering Facilitator (scruff)</td>
	<td class="  sorting_1">1605</td>
</tr><tr id="pkg_SIMD" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/SIMD.html">SIMD</a></td>
	<td class=" ">Jiadi Zhu</td>
	<td class=" ">Statistical Inferences with MeDIP-seq Data (SIMD) to infer the methylation level for each CpG site</td>
	<td class="  sorting_1">1605</td>
</tr><tr id="pkg_strandCheckR" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/strandCheckR.html">strandCheckR</a></td>
	<td class=" ">Thu-Hien To</td>
	<td class=" ">Calculate strandness information of a bam file</td>
	<td class="  sorting_1">1605</td>
</tr><tr id="pkg_TimeSeriesExperiment" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/TimeSeriesExperiment.html">TimeSeriesExperiment</a></td>
	<td class=" ">Lan Huong Nguyen</td>
	<td class=" ">Analysis for short time-series data</td>
	<td class="  sorting_1">1605</td>
</tr><tr id="pkg_tRNA" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/tRNA.html">tRNA</a></td>
	<td class=" ">Felix GM Ernst</td>
	<td class=" ">tRNA: Analyzing tRNA sequences and structures</td>
	<td class="  sorting_1">1605</td>
</tr><tr id="pkg_XINA" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/XINA.html">XINA</a></td>
	<td class=" ">Lang Ho Lee and Sasha A. Singh</td>
	<td class=" ">Multiplexes isobaric mass tagged-based kinetics data for network analysis</td>
	<td class="  sorting_1">1605</td>
</tr><tr id="pkg_AffiXcan" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/AffiXcan.html">AffiXcan</a></td>
	<td class=" ">Alessandro Lussana</td>
	<td class=" ">A functional approach to impute Genetically Regulated Expression</td>
	<td class="  sorting_1">1628</td>
</tr><tr id="pkg_AssessORF" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/AssessORF.html">AssessORF</a></td>
	<td class=" ">Deepank Korandla</td>
	<td class=" ">Assess Gene Predictions Using Proteomics and Evolutionary Conservation</td>
	<td class="  sorting_1">1628</td>
</tr><tr id="pkg_BDMMAcorrect" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/BDMMAcorrect.html">BDMMAcorrect</a></td>
	<td class=" ">ZHENWEI DAI</td>
	<td class=" ">Meta-analysis for the metagenomic read counts data from different cohorts</td>
	<td class="  sorting_1">1628</td>
</tr><tr id="pkg_compartmap" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/compartmap.html">compartmap</a></td>
	<td class=" ">Benjamin Johnson</td>
	<td class=" ">A/B compartment inference from ATAC-seq and methylation array data</td>
	<td class="  sorting_1">1628</td>
</tr><tr id="pkg_consensusDE" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/consensusDE.html">consensusDE</a></td>
	<td class=" ">Ashley J. Waardenberg</td>
	<td class=" ">RNA-seq analysis using multiple algorithms</td>
	<td class="  sorting_1">1628</td>
</tr><tr id="pkg_cTRAP" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/cTRAP.html">cTRAP</a></td>
	<td class=" ">Nuno Saraiva-Agostinho</td>
	<td class=" ">Identification of candidate causal perturbations from differential gene expression data</td>
	<td class="  sorting_1">1628</td>
</tr><tr id="pkg_ExCluster" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/ExCluster.html">ExCluster</a></td>
	<td class=" ">R. Matthew Tanner</td>
	<td class=" ">ExCluster robustly detects differentially expressed exons between two conditions of RNA-seq data, requiring at least two independent biological replicates per condition</td>
	<td class="  sorting_1">1628</td>
</tr><tr id="pkg_HiCBricks" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/HiCBricks.html">HiCBricks</a></td>
	<td class=" ">Koustav Pal</td>
	<td class=" ">Framework for Storing and Accessing Hi-C Data Through HDF Files</td>
	<td class="  sorting_1">1628</td>
</tr><tr id="pkg_HIREewas" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/HIREewas.html">HIREewas</a></td>
	<td class=" ">Xiangyu Luo</td>
	<td class=" ">Detection of cell-type-specific risk-CpG sites in epigenome-wide association studies</td>
	<td class="  sorting_1">1628</td>
</tr><tr id="pkg_KinSwingR" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/KinSwingR.html">KinSwingR</a></td>
	<td class=" ">Ashley J. Waardenberg</td>
	<td class=" ">KinSwingR: network-based kinase activity prediction</td>
	<td class="  sorting_1">1628</td>
</tr><tr id="pkg_levi" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/levi.html">levi</a></td>
	<td class=" ">Jose Luiz Rybarczyk Filho</td>
	<td class=" ">Landscape Expression Visualization Interface</td>
	<td class="  sorting_1">1628</td>
</tr><tr id="pkg_MPRAnalyze" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/MPRAnalyze.html">MPRAnalyze</a></td>
	<td class=" ">Tal Ashuach</td>
	<td class=" ">Statistical Analysis of MPRA data</td>
	<td class="  sorting_1">1628</td>
</tr><tr id="pkg_NeighborNet" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/NeighborNet.html">NeighborNet</a></td>
	<td class=" ">Sahar Ansari</td>
	<td class=" ">Neighbor_net analysis</td>
	<td class="  sorting_1">1628</td>
</tr><tr id="pkg_OMICsPCA" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/OMICsPCA.html">OMICsPCA</a></td>
	<td class=" ">Subhadeep Das</td>
	<td class=" ">An R package for quantitative integration and analysis of multiple omics assays from heterogeneous samples</td>
	<td class="  sorting_1">1628</td>
</tr><tr id="pkg_PepsNMR" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/PepsNMR.html">PepsNMR</a></td>
	<td class=" ">Manon Martin</td>
	<td class=" ">Pre-process 1H-NMR FID signals</td>
	<td class="  sorting_1">1628</td>
</tr><tr id="pkg_RNASeqR" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/RNASeqR.html">RNASeqR</a></td>
	<td class=" ">Kuan-Hao Chao</td>
	<td class=" ">RNASeqR: RNA-Seq workflow for case-control study</td>
	<td class="  sorting_1">1628</td>
</tr><tr id="pkg_SCBN" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/SCBN.html">SCBN</a></td>
	<td class=" ">Yan Zhou</td>
	<td class=" ">A novel normalization method and differential expression analysis of RNA-seq d between different species</td>
	<td class="  sorting_1">1628</td>
</tr><tr id="pkg_transite" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/transite.html">transite</a></td>
	<td class=" ">Konstantin Krismer</td>
	<td class=" ">RNA-binding protein motif analysis</td>
	<td class="  sorting_1">1628</td>
</tr><tr id="pkg_tRNAdbImport" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/tRNAdbImport.html">tRNAdbImport</a></td>
	<td class=" ">Felix GM Ernst</td>
	<td class=" ">Importing from tRNAdb and mitotRNAdb as GRanges objects</td>
	<td class="  sorting_1">1628</td>
</tr><tr id="pkg_Wrench" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/Wrench.html">Wrench</a></td>
	<td class=" ">Hector Corrada Bravo</td>
	<td class=" ">Wrench normalization for sparse count data</td>
	<td class="  sorting_1">1628</td>
</tr><tr id="pkg_MTseeker" class="row_even">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/MTseeker.html">MTseeker</a></td>
	<td class=" ">Tim Triche</td>
	<td class=" ">Bioconductor Tools for Human Mitochondrial Variant Analysis</td>
	<td class="  sorting_1">1648</td>
</tr><tr id="pkg_ProteoMM" class="row_odd">
	<td class=" "><a href="https://www.bioconductor.org/packages/release/bioc/html/ProteoMM.html">ProteoMM</a></td>
	<td class=" ">Yuliya V Karpievitch</td>
	<td class=" ">Multi-Dataset Model-based Differential Expression Proteomics Analysis Platform</td>
	<td class="  sorting_1">1648</td>
</tr></tbody></table></div></div><div class="dataTables_info" id="biocViews_package_table_info">Showing 1 to 1,649 of 1,649 entries</div><div class="dataTables_paginate paging_two_button" id="biocViews_package_table_paginate"><a class="paginate_disabled_previous" tabindex="0" role="button" id="biocViews_package_table_previous" aria-controls="biocViews_package_table">Previous</a><a class="paginate_disabled_next" tabindex="0" role="button" id="biocViews_package_table_next" aria-controls="biocViews_package_table">Next</a></div></div>
</div>
        <div id="below_tree" class="biocViewsBelowTree"></div>
    </div>
</div>





	

      </div>

      <div id="SiteGlobalFooter" class="SiteGlobalFooter">

  <span class="global_footer_logo_bc noprint"><img src="./Bioconductor - BiocViews_files/logo_global_footer_bioconductor.gif" height="51" width="154" alt=""></span>

  <div class="attribution_copyright">
    <p>Contact us: <a href="https://support.bioconductor.org/">support.bioconductor.org</a></p>
    <p>Copyright  2003 - 2019, Bioconductor</p>
  </div>

  <a name="site-map"></a> <!-- accessibility anchor -->
  <ul class="footer_nav_list noprint">
    <li class="footer_nav_list_element footer_nav_list_element_0">
      <b><a href="https://www.bioconductor.org/index.html">Home</a></b>
    </li>
    <li class="footer_nav_list_element footer_nav_list_element_1">
      <ul>
        <li><b><a href="https://www.bioconductor.org/install/index.html">Install</a></b></li>
        <li><a href="https://www.bioconductor.org/install/index.html#install-R">Install R</a></li>
        <li><a href="https://www.bioconductor.org/install/index.html#find-bioconductor-packages">Find Bioconductor Packages</a></li>
        <li><a href="https://www.bioconductor.org/install/index.html#install-bioconductor-packages">Install Bioconductor Packages</a></li>
        <li><a href="https://www.bioconductor.org/install/index.html#update-bioconductor-packages">Update Bioconductor Packages</a></li>
      </ul>
    </li>
    <li class="footer_nav_list_element footer_nav_list_element_2">
      <ul>
        <li><b><a href="https://www.bioconductor.org/help/index.html">Help</a></b></li>
        <li><a href="https://www.bioconductor.org/packages/release/workflows/">Workflows</a></li>
        <li><a href="https://www.bioconductor.org/help/package-vignettes/">Package Vignettes</a></li>
        <li><a href="https://www.bioconductor.org/help/faq/">FAQ</a></li>
        <li><a href="https://www.bioconductor.org/help/support/">Support</a></li>
        <li><a href="http://cran.r-project.org/">Using R</a></li>
        <li><a href="https://www.bioconductor.org/help/course-materials/">Courses</a></li>
        <li><a href="https://www.bioconductor.org/help/publications/">Publications</a></li>
        <li><a href="https://www.bioconductor.org/help/bioconductor-cloud-ami/">Cloud AMI</a></li>
        <li><a href="https://www.bioconductor.org/help/community/">Community Resources</a></li>

      </ul>
    </li>
    <li class="footer_nav_list_element footer_nav_list_element_3">
      <ul>
        <li><b><a href="https://www.bioconductor.org/developers/index.html">Developers</a></b></li>
		<li><a href="https://www.bioconductor.org/developers/package-guidelines/">Package Guidelines</a></li>
		<li><a href="https://www.bioconductor.org/developers/package-submission/">Package Submission</a></li>
		<li><a href="https://www.bioconductor.org/developers/release-schedule/">Release Schedule</a></li>
		<li><a href="https://www.bioconductor.org/developers/source-control/">Source Control</a></li>
      </ul>
    </li>
    <li class="footer_nav_list_element footer_nav_list_element_4">
      <ul>
        <li><b><a href="https://www.bioconductor.org/about/index.html">About</a></b></li>
        <li><a href="https://www.bioconductor.org/about/advisory-board/">Advisory Board</a></li>
        <li><a href="https://www.bioconductor.org/about/annual-reports/">Annual Reports</a></li>
        <li><a href="https://www.bioconductor.org/about/core-team/">Core Team</a></li>
        <li><a href="https://www.bioconductor.org/about/mirrors/">Mirrors</a></li>
        <li><a href="https://www.bioconductor.org/about/related-projects/">Related Projects</a></li>
      </ul>
    </li>
  </ul>
  <br style="clear:both">
  &nbsp;

</div>


<div id="SiteMasthead" class="SiteMasthead">

  <a name="site-navigation"></a> <!-- accessibility anchor -->

  <span class="logo_vanity_bar noprint"></span>

  <a href="https://www.bioconductor.org/">
	<img src="./Bioconductor - BiocViews_files/logo_bioconductor.gif" border="0" class="masthead_logo" height="78" width="260" alt="Bioconductor - open source software for bioinformatics">
  </a>

  <div id="SiteMastheadRight" class="SiteMastheadRight">
    <div id="SiteMastheadRightBackground" class="SiteMastheadRightBackground">

      <a name="site-search"></a> <!-- accessibility anchor -->
      <form class="site_search" id="search_form" method="GET" action="https://www.bioconductor.org/help/search/index.html">
        Search: <input id="q" name="q">
      </form>

      <ul class="masthead_nav noprint">
        <li class="masthead_nav_element" id="masthead_nav_element_1">
          <a href="https://www.bioconductor.org/">Home</a>
        </li>
        <li class="masthead_nav_element" id="masthead_nav_element_2">
          <a href="https://www.bioconductor.org/install/">Install</a>
        </li>
        <li class="masthead_nav_element" id="masthead_nav_element_3">
          <a href="https://www.bioconductor.org/help/">Help</a>
        </li>
        <li class="masthead_nav_element" id="masthead_nav_element_4">
          <a href="https://www.bioconductor.org/developers/">Developers</a>
        </li>
        <li class="masthead_nav_element" id="masthead_nav_element_5">
          <a href="https://www.bioconductor.org/about/">About</a>
        </li>
      </ul>

    </div>
  </div>

</div>

    </div>
  

<div id="jstree-marker" style="display: none;"></div><div id="jstree-marker-line" style="display: none;"></div><div id="vakata-contextmenu"></div><ul class="ui-autocomplete ui-front ui-menu ui-widget ui-widget-content ui-corner-all" id="ui-id-1" tabindex="0" style="display: none;"></ul></body></html>